\hypertarget{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils}{\section{broad.\+pda.\+seq.\+alignment.\+Alignment\+Utils Class Reference}
\label{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils}\index{broad.\+pda.\+seq.\+alignment.\+Alignment\+Utils@{broad.\+pda.\+seq.\+alignment.\+Alignment\+Utils}}
}


Collaboration diagram for broad.\+pda.\+seq.\+alignment.\+Alignment\+Utils\+:
\subsection*{Static Public Member Functions}
\begin{DoxyCompactItemize}
\item 
static void \hyperlink{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a50fe3d7fac421c85d50a737755922da6}{main} (String\mbox{[}$\,$\mbox{]} args)  throws Exception 
\item 
static \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model}{Continuous\+Data\+Alignment\+Model} \hyperlink{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a01f53b98811de06325a819ab01cfbfc3}{load\+Alignment\+Data} (String alignment\+File)  throws I\+O\+Exception 
\item 
static \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model}{Continuous\+Data\+Alignment\+Model} \hyperlink{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a5b9a1f05e9bf43422e95723e0f5b1008}{load\+Alignment\+Data} (String alignment\+File, boolean load\+Chromosome\+Stats)  throws I\+O\+Exception 
\item 
static \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model}{Continuous\+Data\+Alignment\+Model} \hyperlink{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a3f38eadc7a254793812d0223e325fca2}{load\+Alignment\+Data} (String alignment\+File, boolean load\+Chromosome\+Stats, int min\+Mapping\+Quality)  throws I\+O\+Exception 
\item 
static \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model}{Continuous\+Data\+Alignment\+Model} \hyperlink{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a22bf5bebedcbaad5461419a3b9476fc9}{load\+Alignment\+Data} (String alignment\+File, boolean load\+Chromosome\+Stats, int min\+Mapping\+Quality, boolean remove\+Duplicate\+Flags)  throws I\+O\+Exception 
\item 
static \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model}{Continuous\+Data\+Alignment\+Model} \hyperlink{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_ad098b0d98d37c3e070b667882b9f3132}{load\+Alignment\+Data} (String alignment\+File, boolean load\+Chromosome\+Stats, boolean remove\+Duplicate\+Flags, boolean weigh\+Read\+Counts)  throws I\+O\+Exception 
\item 
static \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model}{Continuous\+Data\+Alignment\+Model} \hyperlink{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_ae407b9daa11a4ab33155c1c221dac929}{load\+Alignment\+Data} (String alignment\+File, boolean load\+Chromosome\+Stats, int min\+Mapping\+Quality, boolean remove\+Duplicate\+Flags, boolean weigh\+Read\+Counts)  throws I\+O\+Exception 
\item 
static \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model}{Continuous\+Data\+Alignment\+Model} \hyperlink{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a26be40597246795138bcf13da2ea1b9b}{load\+Alignment\+Data} (String alignment\+File, boolean load\+Chromosome\+Stats, int min\+Mapping\+Quality, boolean remove\+Duplicate\+Flags, boolean weigh\+Read\+Counts, String strand)  throws I\+O\+Exception 
\item 
static \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model}{Continuous\+Data\+Alignment\+Model} \hyperlink{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a4c8b2eacf9a07b744b6da702067a6ba4}{load\+Alignment\+Data} (String alignment\+File, boolean load\+Chromosome\+Stats, int min\+Mapping\+Quality, boolean remove\+Duplicate\+Flags, boolean weigh\+Read\+Counts, String strand, boolean load\+Pairs\+As\+Fragments)  throws I\+O\+Exception 
\item 
static void \hyperlink{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a6c939ca293f7fbfd141d81c1534a8461}{score\+Annotations} (\hyperlink{classbroad_1_1core_1_1annotation_1_1_b_e_d_reader}{B\+E\+D\+Reader} annotation\+Reader, \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model}{Continuous\+Data\+Alignment\+Model} data)  throws I\+O\+Exception 
\item 
static Map$<$ String, Integer $>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_aaaae3d6e8f7f451a772f432a8414f4a2}{get\+Sizes\+From\+S\+A\+M} (String sam\+File\+Name)
\item 
static Gene \hyperlink{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a88b4e4f173dd90784c6fe117156773c6}{S\+A\+M\+Format\+Full\+B\+E\+D} (S\+A\+M\+Record sam)
\end{DoxyCompactItemize}


\subsection{Member Function Documentation}
\hypertarget{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_aaaae3d6e8f7f451a772f432a8414f4a2}{\index{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}!get\+Sizes\+From\+S\+A\+M@{get\+Sizes\+From\+S\+A\+M}}
\index{get\+Sizes\+From\+S\+A\+M@{get\+Sizes\+From\+S\+A\+M}!broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}}
\subsubsection[{get\+Sizes\+From\+S\+A\+M}]{\setlength{\rightskip}{0pt plus 5cm}static Map$<$String, Integer$>$ broad.\+pda.\+seq.\+alignment.\+Alignment\+Utils.\+get\+Sizes\+From\+S\+A\+M (
\begin{DoxyParamCaption}
\item[{String}]{sam\+File\+Name}
\end{DoxyParamCaption}
)\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_aaaae3d6e8f7f451a772f432a8414f4a2}
\hypertarget{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a01f53b98811de06325a819ab01cfbfc3}{\index{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}!load\+Alignment\+Data@{load\+Alignment\+Data}}
\index{load\+Alignment\+Data@{load\+Alignment\+Data}!broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}}
\subsubsection[{load\+Alignment\+Data}]{\setlength{\rightskip}{0pt plus 5cm}static {\bf Continuous\+Data\+Alignment\+Model} broad.\+pda.\+seq.\+alignment.\+Alignment\+Utils.\+load\+Alignment\+Data (
\begin{DoxyParamCaption}
\item[{String}]{alignment\+File}
\end{DoxyParamCaption}
) throws I\+O\+Exception\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a01f53b98811de06325a819ab01cfbfc3}


Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a5b9a1f05e9bf43422e95723e0f5b1008}{\index{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}!load\+Alignment\+Data@{load\+Alignment\+Data}}
\index{load\+Alignment\+Data@{load\+Alignment\+Data}!broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}}
\subsubsection[{load\+Alignment\+Data}]{\setlength{\rightskip}{0pt plus 5cm}static {\bf Continuous\+Data\+Alignment\+Model} broad.\+pda.\+seq.\+alignment.\+Alignment\+Utils.\+load\+Alignment\+Data (
\begin{DoxyParamCaption}
\item[{String}]{alignment\+File, }
\item[{boolean}]{load\+Chromosome\+Stats}
\end{DoxyParamCaption}
) throws I\+O\+Exception\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a5b9a1f05e9bf43422e95723e0f5b1008}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a3f38eadc7a254793812d0223e325fca2}{\index{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}!load\+Alignment\+Data@{load\+Alignment\+Data}}
\index{load\+Alignment\+Data@{load\+Alignment\+Data}!broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}}
\subsubsection[{load\+Alignment\+Data}]{\setlength{\rightskip}{0pt plus 5cm}static {\bf Continuous\+Data\+Alignment\+Model} broad.\+pda.\+seq.\+alignment.\+Alignment\+Utils.\+load\+Alignment\+Data (
\begin{DoxyParamCaption}
\item[{String}]{alignment\+File, }
\item[{boolean}]{load\+Chromosome\+Stats, }
\item[{int}]{min\+Mapping\+Quality}
\end{DoxyParamCaption}
) throws I\+O\+Exception\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a3f38eadc7a254793812d0223e325fca2}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a22bf5bebedcbaad5461419a3b9476fc9}{\index{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}!load\+Alignment\+Data@{load\+Alignment\+Data}}
\index{load\+Alignment\+Data@{load\+Alignment\+Data}!broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}}
\subsubsection[{load\+Alignment\+Data}]{\setlength{\rightskip}{0pt plus 5cm}static {\bf Continuous\+Data\+Alignment\+Model} broad.\+pda.\+seq.\+alignment.\+Alignment\+Utils.\+load\+Alignment\+Data (
\begin{DoxyParamCaption}
\item[{String}]{alignment\+File, }
\item[{boolean}]{load\+Chromosome\+Stats, }
\item[{int}]{min\+Mapping\+Quality, }
\item[{boolean}]{remove\+Duplicate\+Flags}
\end{DoxyParamCaption}
) throws I\+O\+Exception\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a22bf5bebedcbaad5461419a3b9476fc9}
\begin{DoxyAuthor}{Author}
\+: skadri 
\end{DoxyAuthor}

\begin{DoxyParams}{Parameters}
{\em alignment\+File} & Name of alignment file \\
\hline
{\em load\+Chromosome\+Stats} & File with chromosome stats \\
\hline
{\em min\+Mapping\+Quality} & Mapping quality threshold \\
\hline
{\em remove\+Duplicate\+Flags} & Set to true if P\+C\+R duplicates should be ignored \\
\hline
\end{DoxyParams}
\begin{DoxyReturn}{Returns}

\end{DoxyReturn}

\begin{DoxyExceptions}{Exceptions}
{\em I\+O\+Exception} & \\
\hline
\end{DoxyExceptions}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_ad098b0d98d37c3e070b667882b9f3132}{\index{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}!load\+Alignment\+Data@{load\+Alignment\+Data}}
\index{load\+Alignment\+Data@{load\+Alignment\+Data}!broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}}
\subsubsection[{load\+Alignment\+Data}]{\setlength{\rightskip}{0pt plus 5cm}static {\bf Continuous\+Data\+Alignment\+Model} broad.\+pda.\+seq.\+alignment.\+Alignment\+Utils.\+load\+Alignment\+Data (
\begin{DoxyParamCaption}
\item[{String}]{alignment\+File, }
\item[{boolean}]{load\+Chromosome\+Stats, }
\item[{boolean}]{remove\+Duplicate\+Flags, }
\item[{boolean}]{weigh\+Read\+Counts}
\end{DoxyParamCaption}
) throws I\+O\+Exception\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_ad098b0d98d37c3e070b667882b9f3132}
\begin{DoxyAuthor}{Author}
\+: skadri 
\end{DoxyAuthor}

\begin{DoxyParams}{Parameters}
{\em alignment\+File} & Name of alignment file \\
\hline
{\em load\+Chromosome\+Stats} & File with chromosome stats \\
\hline
{\em remove\+Duplicate\+Flags} & Set to true if P\+C\+R duplicates should be ignored \\
\hline
{\em weigh\+Read\+Counts} & Set to true if read counts will be weighed by the N\+H flag \\
\hline
\end{DoxyParams}
\begin{DoxyReturn}{Returns}

\end{DoxyReturn}

\begin{DoxyExceptions}{Exceptions}
{\em I\+O\+Exception} & \\
\hline
\end{DoxyExceptions}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_ae407b9daa11a4ab33155c1c221dac929}{\index{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}!load\+Alignment\+Data@{load\+Alignment\+Data}}
\index{load\+Alignment\+Data@{load\+Alignment\+Data}!broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}}
\subsubsection[{load\+Alignment\+Data}]{\setlength{\rightskip}{0pt plus 5cm}static {\bf Continuous\+Data\+Alignment\+Model} broad.\+pda.\+seq.\+alignment.\+Alignment\+Utils.\+load\+Alignment\+Data (
\begin{DoxyParamCaption}
\item[{String}]{alignment\+File, }
\item[{boolean}]{load\+Chromosome\+Stats, }
\item[{int}]{min\+Mapping\+Quality, }
\item[{boolean}]{remove\+Duplicate\+Flags, }
\item[{boolean}]{weigh\+Read\+Counts}
\end{DoxyParamCaption}
) throws I\+O\+Exception\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_ae407b9daa11a4ab33155c1c221dac929}
\begin{DoxyAuthor}{Author}
\+: skadri 
\end{DoxyAuthor}

\begin{DoxyParams}{Parameters}
{\em alignment\+File} & Name of alignment file \\
\hline
{\em load\+Chromosome\+Stats} & File with chromosome stats \\
\hline
{\em min\+Mapping\+Quality} & Mapping quality threshold = will be set to 0.\+0 if weigh\+Read\+Counts is true \\
\hline
{\em remove\+Duplicate\+Flags} & Set to true if P\+C\+R duplicates should be ignored \\
\hline
{\em weigh\+Read\+Counts} & Set to true if read counts will be weighed by the N\+H flag \\
\hline
\end{DoxyParams}
\begin{DoxyReturn}{Returns}

\end{DoxyReturn}

\begin{DoxyExceptions}{Exceptions}
{\em I\+O\+Exception} & \\
\hline
\end{DoxyExceptions}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a26be40597246795138bcf13da2ea1b9b}{\index{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}!load\+Alignment\+Data@{load\+Alignment\+Data}}
\index{load\+Alignment\+Data@{load\+Alignment\+Data}!broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}}
\subsubsection[{load\+Alignment\+Data}]{\setlength{\rightskip}{0pt plus 5cm}static {\bf Continuous\+Data\+Alignment\+Model} broad.\+pda.\+seq.\+alignment.\+Alignment\+Utils.\+load\+Alignment\+Data (
\begin{DoxyParamCaption}
\item[{String}]{alignment\+File, }
\item[{boolean}]{load\+Chromosome\+Stats, }
\item[{int}]{min\+Mapping\+Quality, }
\item[{boolean}]{remove\+Duplicate\+Flags, }
\item[{boolean}]{weigh\+Read\+Counts, }
\item[{String}]{strand}
\end{DoxyParamCaption}
) throws I\+O\+Exception\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a26be40597246795138bcf13da2ea1b9b}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a4c8b2eacf9a07b744b6da702067a6ba4}{\index{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}!load\+Alignment\+Data@{load\+Alignment\+Data}}
\index{load\+Alignment\+Data@{load\+Alignment\+Data}!broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}}
\subsubsection[{load\+Alignment\+Data}]{\setlength{\rightskip}{0pt plus 5cm}static {\bf Continuous\+Data\+Alignment\+Model} broad.\+pda.\+seq.\+alignment.\+Alignment\+Utils.\+load\+Alignment\+Data (
\begin{DoxyParamCaption}
\item[{String}]{alignment\+File, }
\item[{boolean}]{load\+Chromosome\+Stats, }
\item[{int}]{min\+Mapping\+Quality, }
\item[{boolean}]{remove\+Duplicate\+Flags, }
\item[{boolean}]{weigh\+Read\+Counts, }
\item[{String}]{strand, }
\item[{boolean}]{load\+Pairs\+As\+Fragments}
\end{DoxyParamCaption}
) throws I\+O\+Exception\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a4c8b2eacf9a07b744b6da702067a6ba4}
\hypertarget{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a50fe3d7fac421c85d50a737755922da6}{\index{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}!main@{main}}
\index{main@{main}!broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}}
\subsubsection[{main}]{\setlength{\rightskip}{0pt plus 5cm}static void broad.\+pda.\+seq.\+alignment.\+Alignment\+Utils.\+main (
\begin{DoxyParamCaption}
\item[{String\mbox{[}$\,$\mbox{]}}]{args}
\end{DoxyParamCaption}
) throws Exception\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a50fe3d7fac421c85d50a737755922da6}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a88b4e4f173dd90784c6fe117156773c6}{\index{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}!S\+A\+M\+Format\+Full\+B\+E\+D@{S\+A\+M\+Format\+Full\+B\+E\+D}}
\index{S\+A\+M\+Format\+Full\+B\+E\+D@{S\+A\+M\+Format\+Full\+B\+E\+D}!broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}}
\subsubsection[{S\+A\+M\+Format\+Full\+B\+E\+D}]{\setlength{\rightskip}{0pt plus 5cm}static Gene broad.\+pda.\+seq.\+alignment.\+Alignment\+Utils.\+S\+A\+M\+Format\+Full\+B\+E\+D (
\begin{DoxyParamCaption}
\item[{S\+A\+M\+Record}]{sam}
\end{DoxyParamCaption}
)\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a88b4e4f173dd90784c6fe117156773c6}
\hypertarget{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a6c939ca293f7fbfd141d81c1534a8461}{\index{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}!score\+Annotations@{score\+Annotations}}
\index{score\+Annotations@{score\+Annotations}!broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils@{broad\+::pda\+::seq\+::alignment\+::\+Alignment\+Utils}}
\subsubsection[{score\+Annotations}]{\setlength{\rightskip}{0pt plus 5cm}static void broad.\+pda.\+seq.\+alignment.\+Alignment\+Utils.\+score\+Annotations (
\begin{DoxyParamCaption}
\item[{{\bf B\+E\+D\+Reader}}]{annotation\+Reader, }
\item[{{\bf Continuous\+Data\+Alignment\+Model}}]{data}
\end{DoxyParamCaption}
) throws I\+O\+Exception\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a6c939ca293f7fbfd141d81c1534a8461}


The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
java/broad/pda/seq/alignment/\hyperlink{_alignment_utils_8java}{Alignment\+Utils.\+java}\end{DoxyCompactItemize}
