\hypertarget{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor}{\section{broad.\+core.\+multiplealignment.\+Alignment\+Extractor Class Reference}
\label{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor}\index{broad.\+core.\+multiplealignment.\+Alignment\+Extractor@{broad.\+core.\+multiplealignment.\+Alignment\+Extractor}}
}


Collaboration diagram for broad.\+core.\+multiplealignment.\+Alignment\+Extractor\+:
\subsection*{Static Public Member Functions}
\begin{DoxyCompactItemize}
\item 
static void \hyperlink{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_a48d2b8b5d5fdbcc0caae3d2e29e049f2}{main} (String\mbox{[}$\,$\mbox{]} args)  throws I\+O\+Exception, Illegal\+Access\+Exception, Parse\+Exception 
\item 
static void \hyperlink{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_a953ec88a63df9ac98e99814792206a06}{extract\+G\+F\+F\+Regions} (String alignment\+File, String output\+File, String gff, String in\+Format, String out\+Format, String\mbox{[}$\,$\mbox{]} seqs)  throws I\+O\+Exception, Parse\+Exception 
\end{DoxyCompactItemize}
\subsection*{Static Public Attributes}
\begin{DoxyCompactItemize}
\item 
static Array\+List$<$ String $>$ \hyperlink{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_a6ca872097ad7ccf33de84c89dfbd7482}{degenerate\+Codon\+Starts}
\item 
static Array\+List$<$ String $>$ \hyperlink{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_ae2657f7145b67391c691ff34cf8b1b79}{two\+Fold\+Degenerate\+Codons}
\item 
static Array\+List$<$ String $>$ \hyperlink{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_a4e9a8a4f7ce3eb57cd3f1e92ba30bacb}{three\+Fold\+Degenerate\+Codons}
\item 
static Array\+List$<$ String $>$ \hyperlink{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_a97d857eef06955fef14372779a2e7e0f}{non\+Degenerate\+Codons}
\item 
static Hash\+Map$<$ String, String $>$ \hyperlink{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_a26170c7f5b1f3f17effd444c1ddcd9f4}{codon\+To\+Aminoacid}
\item 
static String \hyperlink{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_a3a667e09923b9b2c3de38373900c8151}{U\+S\+A\+G\+E}
\end{DoxyCompactItemize}


\subsection{Member Function Documentation}
\hypertarget{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_a953ec88a63df9ac98e99814792206a06}{\index{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor@{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor}!extract\+G\+F\+F\+Regions@{extract\+G\+F\+F\+Regions}}
\index{extract\+G\+F\+F\+Regions@{extract\+G\+F\+F\+Regions}!broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor@{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor}}
\subsubsection[{extract\+G\+F\+F\+Regions}]{\setlength{\rightskip}{0pt plus 5cm}static void broad.\+core.\+multiplealignment.\+Alignment\+Extractor.\+extract\+G\+F\+F\+Regions (
\begin{DoxyParamCaption}
\item[{String}]{alignment\+File, }
\item[{String}]{output\+File, }
\item[{String}]{gff, }
\item[{String}]{in\+Format, }
\item[{String}]{out\+Format, }
\item[{String\mbox{[}$\,$\mbox{]}}]{seqs}
\end{DoxyParamCaption}
) throws I\+O\+Exception, {\bf Parse\+Exception}\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_a953ec88a63df9ac98e99814792206a06}


Here is the call graph for this function\+:




Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_a48d2b8b5d5fdbcc0caae3d2e29e049f2}{\index{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor@{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor}!main@{main}}
\index{main@{main}!broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor@{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor}}
\subsubsection[{main}]{\setlength{\rightskip}{0pt plus 5cm}static void broad.\+core.\+multiplealignment.\+Alignment\+Extractor.\+main (
\begin{DoxyParamCaption}
\item[{String\mbox{[}$\,$\mbox{]}}]{args}
\end{DoxyParamCaption}
) throws I\+O\+Exception, Illegal\+Access\+Exception, {\bf Parse\+Exception}\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_a48d2b8b5d5fdbcc0caae3d2e29e049f2}


Here is the call graph for this function\+:




\subsection{Member Data Documentation}
\hypertarget{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_a26170c7f5b1f3f17effd444c1ddcd9f4}{\index{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor@{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor}!codon\+To\+Aminoacid@{codon\+To\+Aminoacid}}
\index{codon\+To\+Aminoacid@{codon\+To\+Aminoacid}!broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor@{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor}}
\subsubsection[{codon\+To\+Aminoacid}]{\setlength{\rightskip}{0pt plus 5cm}Hash\+Map$<$String, String$>$ broad.\+core.\+multiplealignment.\+Alignment\+Extractor.\+codon\+To\+Aminoacid\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_a26170c7f5b1f3f17effd444c1ddcd9f4}
\hypertarget{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_a6ca872097ad7ccf33de84c89dfbd7482}{\index{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor@{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor}!degenerate\+Codon\+Starts@{degenerate\+Codon\+Starts}}
\index{degenerate\+Codon\+Starts@{degenerate\+Codon\+Starts}!broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor@{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor}}
\subsubsection[{degenerate\+Codon\+Starts}]{\setlength{\rightskip}{0pt plus 5cm}Array\+List$<$String$>$ broad.\+core.\+multiplealignment.\+Alignment\+Extractor.\+degenerate\+Codon\+Starts\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_a6ca872097ad7ccf33de84c89dfbd7482}
\hypertarget{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_a97d857eef06955fef14372779a2e7e0f}{\index{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor@{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor}!non\+Degenerate\+Codons@{non\+Degenerate\+Codons}}
\index{non\+Degenerate\+Codons@{non\+Degenerate\+Codons}!broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor@{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor}}
\subsubsection[{non\+Degenerate\+Codons}]{\setlength{\rightskip}{0pt plus 5cm}Array\+List$<$String$>$ broad.\+core.\+multiplealignment.\+Alignment\+Extractor.\+non\+Degenerate\+Codons\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_a97d857eef06955fef14372779a2e7e0f}
\hypertarget{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_a4e9a8a4f7ce3eb57cd3f1e92ba30bacb}{\index{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor@{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor}!three\+Fold\+Degenerate\+Codons@{three\+Fold\+Degenerate\+Codons}}
\index{three\+Fold\+Degenerate\+Codons@{three\+Fold\+Degenerate\+Codons}!broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor@{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor}}
\subsubsection[{three\+Fold\+Degenerate\+Codons}]{\setlength{\rightskip}{0pt plus 5cm}Array\+List$<$String$>$ broad.\+core.\+multiplealignment.\+Alignment\+Extractor.\+three\+Fold\+Degenerate\+Codons\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_a4e9a8a4f7ce3eb57cd3f1e92ba30bacb}
\hypertarget{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_ae2657f7145b67391c691ff34cf8b1b79}{\index{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor@{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor}!two\+Fold\+Degenerate\+Codons@{two\+Fold\+Degenerate\+Codons}}
\index{two\+Fold\+Degenerate\+Codons@{two\+Fold\+Degenerate\+Codons}!broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor@{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor}}
\subsubsection[{two\+Fold\+Degenerate\+Codons}]{\setlength{\rightskip}{0pt plus 5cm}Array\+List$<$String$>$ broad.\+core.\+multiplealignment.\+Alignment\+Extractor.\+two\+Fold\+Degenerate\+Codons\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_ae2657f7145b67391c691ff34cf8b1b79}
\hypertarget{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_a3a667e09923b9b2c3de38373900c8151}{\index{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor@{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor}!U\+S\+A\+G\+E@{U\+S\+A\+G\+E}}
\index{U\+S\+A\+G\+E@{U\+S\+A\+G\+E}!broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor@{broad\+::core\+::multiplealignment\+::\+Alignment\+Extractor}}
\subsubsection[{U\+S\+A\+G\+E}]{\setlength{\rightskip}{0pt plus 5cm}String broad.\+core.\+multiplealignment.\+Alignment\+Extractor.\+U\+S\+A\+G\+E\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor_a3a667e09923b9b2c3de38373900c8151}
{\bfseries Initial value\+:}
\begin{DoxyCode}
= \textcolor{stringliteral}{"Usage: AlignmentExtractor TASK=<task\_num> <task\_args>\(\backslash\)n"} +
    \textcolor{stringliteral}{"\(\backslash\)tTasks:\(\backslash\)n"} +
    \textcolor{stringliteral}{"\(\backslash\)t\(\backslash\)t1. Extract Regions: IN=<Multiple alignment file> OUT=<output file> ANNOTGFF=<GFF annotation file,
       sequence name should match one of the aligned sequence IDs> INFORMAT=<[FASTA]/EXON/PHYLIP/SEQPHYLIP>
       OUTFORMAT<[FASTA]/EXON/PHYLIP> SEQS=<comma separated list of sequence ids>\(\backslash\)n"} +
    \textcolor{stringliteral}{"\(\backslash\)t\(\backslash\)t2. Extract 4D sites from exon alignment: IN=<Exon multiple alignment> OUT=<output file>
       INFORMAT=<input format (default FASTA)> OUTFORMAT=<output format (default FASTA)>\(\backslash\)n "} +
    \textcolor{stringliteral}{"\(\backslash\)t\(\backslash\)t3. Extract Conserved sites from exon alignment: IN=<Exon multiple alignment> OUT=<output alignment
       file> INFORMAT=<input format (default FASTA)> OUTFORMAT=<output format (default FASTA)>\(\backslash\)n"}+
    \textcolor{stringliteral}{"\(\backslash\)t\(\backslash\)t4. Extract region IN=<Multiple alignment file> OUT=<output file> REGION=<start..end>
       REF=<reference> INFORMAT=<[FASTA]/EXON/PHYLIP> OUTFORMAT<[FASTA]/EXON/PHYLIP/SEQPHYLIP> \(\backslash\)n"} +
    \textcolor{stringliteral}{"\(\backslash\)t\(\backslash\)t5. Convert alignment file IN=<Multiple Alignment File> OUT=<output file>
       INFORMAT=<[FASTA]/EXON/PHYLIP> OUTFORMAT<FASTA/EXON/[PHYLIP]/SEQPHYLIP> [-compress <if the output alignment should be reference gap
       free>] \(\backslash\)n"} +
    \textcolor{stringliteral}{"\(\backslash\)t\(\backslash\)t6. Introduce Random Gaps -in <Multiple Alignment File default is standard input> -out <output
       gapped alignment defalut is standard out> -informat <[FASTA]/EXON/PHYLIP> -outformat
       <FASTA/EXON/[PHYLIP]/SEQPHYLIP> -maxGaps <maximum number of gaps> [-ref <reference if no gaps should be introduced in reference>]\(\backslash\)n"} +
    \textcolor{stringliteral}{"\(\backslash\)t\(\backslash\)t7. Permute Columns -in <Multiple Alignment File default is standard input> -out <output gapped
       alignment defalut is standard out> -informat <[FASTA]/EXON/PHYLIP> -outformat <FASTA/EXON/[PHYLIP]/SEQPHYLIP> "}
       +
    \textcolor{stringliteral}{"\(\backslash\)n\(\backslash\)t8. Sample Columns from Alignment -in <Multiple Alignment File default is standard input> -out
       <output gapped alignment defalut is standard out> -informat <[FASTA]/EXON/PHYLIP> -outformat
       <[FASTA]/EXON/PHYLIP/SEQPHYLIP> -cols <Number of columns to sample> [-consecutiveCols <Sample consecutive columns rather than
       single ones>]"} +
    \textcolor{stringliteral}{"\(\backslash\)n\(\backslash\)t9. Combined alignments -informat <[FASTA], PHYLIP, SEQPHYLIP> -outformat <[FASTA], PHYLIP,
       SEQPHYLIP> -in <Standard input of file with a list of alignment files> -out <standard output or file name>
       [-maxColumns <If specified, and the combined alignment is larger than this value, a sampled alignmnet will be
       generated instead of the combined one>]"} +
    \textcolor{stringliteral}{"\(\backslash\)n"}
\end{DoxyCode}


The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
java/broad/core/multiplealignment/\hyperlink{_alignment_extractor_8java}{Alignment\+Extractor.\+java}\end{DoxyCompactItemize}
