\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data}{\section{broad.\+pda.\+seq.\+protection.\+Sample\+Data Class Reference}
\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data}\index{broad.\+pda.\+seq.\+protection.\+Sample\+Data@{broad.\+pda.\+seq.\+protection.\+Sample\+Data}}
}


Inheritance diagram for broad.\+pda.\+seq.\+protection.\+Sample\+Data\+:


Collaboration diagram for broad.\+pda.\+seq.\+protection.\+Sample\+Data\+:
\subsection*{Public Member Functions}
\begin{DoxyCompactItemize}
\item 
\hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_acbea9965c1d9f8b120e1510e326c5b96}{Sample\+Data} (String bam\+File, boolean \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a1a603b2dd39ba9c8b54b3552a3053783}{first\+Read\+Transcription\+Strand}, Map$<$ String, Collection$<$ Gene $>$$>$ genes, int window, int step, double expression\+Cutoff, boolean exp\+By\+Scan\+Pval, boolean fully\+Contained)  throws I\+O\+Exception 
\item 
int \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a8fe3a848dd59a9a072680c711fa86766}{hash\+Code} ()
\item 
void \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a2013287af35974ad48c4cd57b099ac4b}{set\+Expression\+Scan\+Pvalue\+Cutoff} (double expression\+Scan\+Pval\+Cutoff)
\item 
double \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a4087ee5f33ea5ecc977f4fdf4483804e}{get\+Expression\+Scan\+Pvalue\+Cutoff} ()
\item 
double \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_aa2bf68f6d49af41a18f1d4a1277c36f4}{get\+Gene\+Count} (Gene gene)
\item 
double \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a0795b41a9a096104df31f5b6ba2c9154}{get\+Gene\+Scan\+Pval} (Gene gene)
\item 
double \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a7d17b21cbf2a2368cc637a7debb8089e}{get\+Gene\+Average\+Coverage} (Gene gene)
\item 
boolean \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_af2a3f7b805b2fea8605b821b53f56958}{is\+Expressed} (Gene gene)
\item 
double \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_afd2283807ea84e0cc2da07c108f6ca98}{get\+Window\+Count} (Gene gene, Annotation window)
\item 
boolean \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a1a603b2dd39ba9c8b54b3552a3053783}{first\+Read\+Transcription\+Strand} ()
\item 
Map$<$ Annotation, \\*
Scan\+Statistic\+Score $>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a1adf639ddb5cd1b53ab3ff48f9f74ea9}{get\+Window\+Scores} (Gene gene)
\item 
boolean \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_af69246249c44eabd4349626e639a44c6}{got\+Window\+Scores\+From\+File} ()
\item 
double \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a2f18a4bad14841ddbb440c21a281703a}{get\+Enrichment\+Over\+Gene} (Gene gene, Annotation window)
\item 
Scan\+Statistic\+Score \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a477fb361142dde07116598599bb7ef52}{score\+Window\+With\+Fragment\+Length\+Filter} (Gene gene, Annotation window)
\item 
Scan\+Statistic\+Score \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ac476ccc56fab96d17419127a8cf313a1}{score\+Window} (Gene gene, Annotation window)
\item 
Scan\+Statistic\+Score \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ae556c14ccb3cb7b63182adf72e3630d7}{score\+Window} (Gene gene, Annotation window, double count)
\item 
Logger \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ab31cc1f558419ad028782703b015c489}{get\+Logger} ()
\item 
String \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a9284f2dc44109171457dcc0417d8520a}{get\+Sample\+Name} ()
\item 
String \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a3a3c58c0def5a809f7949a2f441e815d}{get\+Original\+Bam\+File} ()
\item 
void \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a77cf4e55d644cb09fa376f9d8b41c538}{set\+Sample\+Name} (String name)
\item 
int \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ab775331fb5bee8667165c78ca5a105ba}{get\+Window\+Size} ()
\item 
int \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a225b80f975a281ea6c4d99482a20ba0e}{get\+Step\+Size} ()
\item 
Transcriptome\+Space\+Alignment\+Model \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a7e1478400941779a43e0776cd90733bb}{get\+Data} ()
\item 
Transcriptome\+Space\+Alignment\+Model \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a896c9168d0e4ca469e94d60554bdf1d9}{get\+Fragment\+Length\+Filter\+Data} ()
\end{DoxyCompactItemize}
\subsection*{Static Public Member Functions}
\begin{DoxyCompactItemize}
\item 
static Annotation \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a8d27ed6393a7f774100c706c5a1b9b8d}{trim\+Max\+Contiguous} (Annotation window, List$<$ Double $>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ae06d8e63d6ddf7093085beba12b22e8a}{data}, double quantile)
\end{DoxyCompactItemize}
\subsection*{Protected Attributes}
\begin{DoxyCompactItemize}
\item 
String \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a5b0011ca04c2ee52ec32f8508b82ae99}{sample\+Name}
\item 
Transcriptome\+Space\+Alignment\+Model \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ae06d8e63d6ddf7093085beba12b22e8a}{data}
\item 
Transcriptome\+Space\+Alignment\+Model \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a214cced67761f0090ba326708620d3be}{max\+Fragment\+Length\+Data}
\item 
Map$<$ Gene, Scan\+Statistic\+Score $>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_af611a295e308f4f0d0849a80139ad7ff}{gene\+Scores}
\item 
Map$<$ Gene, Double $>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ad517adb3cfab02f8f65bc10818ebca71}{gene\+Avg\+Coverage}
\item 
Map$<$ Gene, Map$<$ Annotation, \\*
Scan\+Statistic\+Score $>$ $>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a6a53d104b93427859b6fd2c4ffbfdf6d}{window\+Scores}
\item 
int \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_af155a04e70a9feafd812aa6135f5b732}{window\+Size}
\item 
int \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ab5fe21e8281091d1be8de525e17d8a3a}{step\+Size}
\item 
Map$<$ String, Collection$<$ Gene $>$ $>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a8b1f58470538adcd26f47252bdf94232}{genes\+By\+Chr}
\item 
Map$<$ String, Gene $>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a528cf3b7c628a3f4a567bd33ba75ca0f}{genes\+By\+Name}
\item 
double \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ae1a282e95f82d5974d5e3c4189a81f8c}{expression\+Cutoff\+Value}
\item 
boolean \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a0f463fa018b0f1cc4495a3dfdd2e9ef5}{expression\+By\+Scan\+Pval}
\item 
boolean \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ac971b6d1585808b15cc2ef8807225bf3}{fully\+Contained\+Reads}
\end{DoxyCompactItemize}
\subsection*{Static Protected Attributes}
\begin{DoxyCompactItemize}
\item 
static Logger \hyperlink{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a4f08fb5a371b51daf757152c83863ac2}{logger} = Logger.\+get\+Logger(Sample\+Data.\+class.\+get\+Name())
\end{DoxyCompactItemize}


\subsection{Detailed Description}
\begin{DoxyAuthor}{Author}
prussell 
\end{DoxyAuthor}


\subsection{Constructor \& Destructor Documentation}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_acbea9965c1d9f8b120e1510e326c5b96}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!Sample\+Data@{Sample\+Data}}
\index{Sample\+Data@{Sample\+Data}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{Sample\+Data}]{\setlength{\rightskip}{0pt plus 5cm}broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+Sample\+Data (
\begin{DoxyParamCaption}
\item[{String}]{bam\+File, }
\item[{boolean}]{first\+Read\+Transcription\+Strand, }
\item[{Map$<$ String, Collection$<$ Gene $>$$>$}]{genes, }
\item[{int}]{window, }
\item[{int}]{step, }
\item[{double}]{expression\+Cutoff, }
\item[{boolean}]{exp\+By\+Scan\+Pval, }
\item[{boolean}]{fully\+Contained}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_acbea9965c1d9f8b120e1510e326c5b96}

\begin{DoxyParams}{Parameters}
{\em bam\+File} & Bam file \\
\hline
{\em first\+Read\+Transcription\+Strand} & Whether read 1 is in direction of transcription \\
\hline
{\em genes} & Genes by chromosome \\
\hline
{\em window} & Window size \\
\hline
{\em step} & Step size \\
\hline
{\em expression\+Cutoff} & P value cutoff for scan test of gene expression \\
\hline
{\em exp\+By\+Scan\+Pval} & Expression is assessed by scan P value. If false, uses average read depth \\
\hline
{\em fully\+Contained} & Count fully contained reads only \\
\hline
\end{DoxyParams}

\begin{DoxyExceptions}{Exceptions}
{\em I\+O\+Exception} & \\
\hline
\end{DoxyExceptions}


Here is the call graph for this function\+:




\subsection{Member Function Documentation}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a1a603b2dd39ba9c8b54b3552a3053783}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!first\+Read\+Transcription\+Strand@{first\+Read\+Transcription\+Strand}}
\index{first\+Read\+Transcription\+Strand@{first\+Read\+Transcription\+Strand}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{first\+Read\+Transcription\+Strand}]{\setlength{\rightskip}{0pt plus 5cm}boolean broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+first\+Read\+Transcription\+Strand (
\begin{DoxyParamCaption}
{}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a1a603b2dd39ba9c8b54b3552a3053783}
Whether read 1 is in direction of transcription \begin{DoxyReturn}{Returns}
True iff first read transcription strand is true 
\end{DoxyReturn}


Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a7e1478400941779a43e0776cd90733bb}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!get\+Data@{get\+Data}}
\index{get\+Data@{get\+Data}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{get\+Data}]{\setlength{\rightskip}{0pt plus 5cm}Transcriptome\+Space\+Alignment\+Model broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+get\+Data (
\begin{DoxyParamCaption}
{}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a7e1478400941779a43e0776cd90733bb}
Write the window scores to a file for future use 
\begin{DoxyParams}{Parameters}
{\em m} & Multi sample binding site caller that this belongs to \\
\hline
\end{DoxyParams}

\begin{DoxyExceptions}{Exceptions}
{\em I\+O\+Exception} & Get the alignment data \\
\hline
\end{DoxyExceptions}
\begin{DoxyReturn}{Returns}
Alignment model 
\end{DoxyReturn}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a2f18a4bad14841ddbb440c21a281703a}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!get\+Enrichment\+Over\+Gene@{get\+Enrichment\+Over\+Gene}}
\index{get\+Enrichment\+Over\+Gene@{get\+Enrichment\+Over\+Gene}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{get\+Enrichment\+Over\+Gene}]{\setlength{\rightskip}{0pt plus 5cm}double broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+get\+Enrichment\+Over\+Gene (
\begin{DoxyParamCaption}
\item[{Gene}]{gene, }
\item[{Annotation}]{window}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a2f18a4bad14841ddbb440c21a281703a}
Get enrichment of a window over a gene 
\begin{DoxyParams}{Parameters}
{\em gene} & The gene \\
\hline
{\em window} & Window contained in the gene \\
\hline
\end{DoxyParams}
\begin{DoxyReturn}{Returns}
Enrichment of window over gene background 
\end{DoxyReturn}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a4087ee5f33ea5ecc977f4fdf4483804e}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!get\+Expression\+Scan\+Pvalue\+Cutoff@{get\+Expression\+Scan\+Pvalue\+Cutoff}}
\index{get\+Expression\+Scan\+Pvalue\+Cutoff@{get\+Expression\+Scan\+Pvalue\+Cutoff}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{get\+Expression\+Scan\+Pvalue\+Cutoff}]{\setlength{\rightskip}{0pt plus 5cm}double broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+get\+Expression\+Scan\+Pvalue\+Cutoff (
\begin{DoxyParamCaption}
{}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a4087ee5f33ea5ecc977f4fdf4483804e}
Get genome wide scan P value cutoff for expression of transcript \begin{DoxyReturn}{Returns}
P value cutoff for transcript expression against genomic background 
\end{DoxyReturn}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a896c9168d0e4ca469e94d60554bdf1d9}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!get\+Fragment\+Length\+Filter\+Data@{get\+Fragment\+Length\+Filter\+Data}}
\index{get\+Fragment\+Length\+Filter\+Data@{get\+Fragment\+Length\+Filter\+Data}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{get\+Fragment\+Length\+Filter\+Data}]{\setlength{\rightskip}{0pt plus 5cm}Transcriptome\+Space\+Alignment\+Model broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+get\+Fragment\+Length\+Filter\+Data (
\begin{DoxyParamCaption}
{}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a896c9168d0e4ca469e94d60554bdf1d9}
Get the alignment data filtered by max fragment length \begin{DoxyReturn}{Returns}
Alignment model 
\end{DoxyReturn}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a7d17b21cbf2a2368cc637a7debb8089e}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!get\+Gene\+Average\+Coverage@{get\+Gene\+Average\+Coverage}}
\index{get\+Gene\+Average\+Coverage@{get\+Gene\+Average\+Coverage}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{get\+Gene\+Average\+Coverage}]{\setlength{\rightskip}{0pt plus 5cm}double broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+get\+Gene\+Average\+Coverage (
\begin{DoxyParamCaption}
\item[{Gene}]{gene}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a7d17b21cbf2a2368cc637a7debb8089e}
Get average coverage of gene Get score from cache or calculate and cache score 
\begin{DoxyParams}{Parameters}
{\em gene} & The gene \\
\hline
\end{DoxyParams}
\begin{DoxyReturn}{Returns}
The average coverage of the gene 
\end{DoxyReturn}


Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_aa2bf68f6d49af41a18f1d4a1277c36f4}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!get\+Gene\+Count@{get\+Gene\+Count}}
\index{get\+Gene\+Count@{get\+Gene\+Count}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{get\+Gene\+Count}]{\setlength{\rightskip}{0pt plus 5cm}double broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+get\+Gene\+Count (
\begin{DoxyParamCaption}
\item[{Gene}]{gene}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_aa2bf68f6d49af41a18f1d4a1277c36f4}
Get number of fragments mapping to gene Get score from cache or calculate and cache score 
\begin{DoxyParams}{Parameters}
{\em gene} & The gene \\
\hline
\end{DoxyParams}
\begin{DoxyReturn}{Returns}
The number of fragments mapping to the gene 
\end{DoxyReturn}


Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a0795b41a9a096104df31f5b6ba2c9154}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!get\+Gene\+Scan\+Pval@{get\+Gene\+Scan\+Pval}}
\index{get\+Gene\+Scan\+Pval@{get\+Gene\+Scan\+Pval}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{get\+Gene\+Scan\+Pval}]{\setlength{\rightskip}{0pt plus 5cm}double broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+get\+Gene\+Scan\+Pval (
\begin{DoxyParamCaption}
\item[{Gene}]{gene}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a0795b41a9a096104df31f5b6ba2c9154}
Get coordinate space wide scan P value of number of fragments mapping to the gene Get score from cache or calculate and cache score 
\begin{DoxyParams}{Parameters}
{\em gene} & The gene \\
\hline
\end{DoxyParams}
\begin{DoxyReturn}{Returns}
The scan P value of the number of fragments mapping to the gene with respect to teh coordinate space 
\end{DoxyReturn}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ab31cc1f558419ad028782703b015c489}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!get\+Logger@{get\+Logger}}
\index{get\+Logger@{get\+Logger}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{get\+Logger}]{\setlength{\rightskip}{0pt plus 5cm}Logger broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+get\+Logger (
\begin{DoxyParamCaption}
{}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ab31cc1f558419ad028782703b015c489}
Get the logger \begin{DoxyReturn}{Returns}
The logger 
\end{DoxyReturn}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a3a3c58c0def5a809f7949a2f441e815d}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!get\+Original\+Bam\+File@{get\+Original\+Bam\+File}}
\index{get\+Original\+Bam\+File@{get\+Original\+Bam\+File}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{get\+Original\+Bam\+File}]{\setlength{\rightskip}{0pt plus 5cm}String broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+get\+Original\+Bam\+File (
\begin{DoxyParamCaption}
{}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a3a3c58c0def5a809f7949a2f441e815d}
Get name of original bam file \begin{DoxyReturn}{Returns}
Bam file name 
\end{DoxyReturn}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a9284f2dc44109171457dcc0417d8520a}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!get\+Sample\+Name@{get\+Sample\+Name}}
\index{get\+Sample\+Name@{get\+Sample\+Name}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{get\+Sample\+Name}]{\setlength{\rightskip}{0pt plus 5cm}String broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+get\+Sample\+Name (
\begin{DoxyParamCaption}
{}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a9284f2dc44109171457dcc0417d8520a}
Get the sample name \begin{DoxyReturn}{Returns}
Sample name 
\end{DoxyReturn}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a225b80f975a281ea6c4d99482a20ba0e}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!get\+Step\+Size@{get\+Step\+Size}}
\index{get\+Step\+Size@{get\+Step\+Size}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{get\+Step\+Size}]{\setlength{\rightskip}{0pt plus 5cm}int broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+get\+Step\+Size (
\begin{DoxyParamCaption}
{}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a225b80f975a281ea6c4d99482a20ba0e}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_afd2283807ea84e0cc2da07c108f6ca98}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!get\+Window\+Count@{get\+Window\+Count}}
\index{get\+Window\+Count@{get\+Window\+Count}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{get\+Window\+Count}]{\setlength{\rightskip}{0pt plus 5cm}double broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+get\+Window\+Count (
\begin{DoxyParamCaption}
\item[{Gene}]{gene, }
\item[{Annotation}]{window}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_afd2283807ea84e0cc2da07c108f6ca98}
Get number of fragments mapping to gene window Get score from cache or calculate and cache score 
\begin{DoxyParams}{Parameters}
{\em gene} & The gene \\
\hline
{\em window} & The window \\
\hline
\end{DoxyParams}
\begin{DoxyReturn}{Returns}
The number of fragments mapping to the gene 
\end{DoxyReturn}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a1adf639ddb5cd1b53ab3ff48f9f74ea9}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!get\+Window\+Scores@{get\+Window\+Scores}}
\index{get\+Window\+Scores@{get\+Window\+Scores}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{get\+Window\+Scores}]{\setlength{\rightskip}{0pt plus 5cm}Map$<$Annotation, Scan\+Statistic\+Score$>$ broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+get\+Window\+Scores (
\begin{DoxyParamCaption}
\item[{Gene}]{gene}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a1adf639ddb5cd1b53ab3ff48f9f74ea9}
Get scores for each window in the gene 
\begin{DoxyParams}{Parameters}
{\em gene} & The gene \\
\hline
\end{DoxyParams}
\begin{DoxyReturn}{Returns}
The scores by window 
\end{DoxyReturn}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ab775331fb5bee8667165c78ca5a105ba}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!get\+Window\+Size@{get\+Window\+Size}}
\index{get\+Window\+Size@{get\+Window\+Size}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{get\+Window\+Size}]{\setlength{\rightskip}{0pt plus 5cm}int broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+get\+Window\+Size (
\begin{DoxyParamCaption}
{}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ab775331fb5bee8667165c78ca5a105ba}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_af69246249c44eabd4349626e639a44c6}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!got\+Window\+Scores\+From\+File@{got\+Window\+Scores\+From\+File}}
\index{got\+Window\+Scores\+From\+File@{got\+Window\+Scores\+From\+File}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{got\+Window\+Scores\+From\+File}]{\setlength{\rightskip}{0pt plus 5cm}boolean broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+got\+Window\+Scores\+From\+File (
\begin{DoxyParamCaption}
{}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_af69246249c44eabd4349626e639a44c6}
Whether the window scores were read from an existing file \begin{DoxyReturn}{Returns}
Whether the window scores were read from an existing file 
\end{DoxyReturn}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a8fe3a848dd59a9a072680c711fa86766}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!hash\+Code@{hash\+Code}}
\index{hash\+Code@{hash\+Code}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{hash\+Code}]{\setlength{\rightskip}{0pt plus 5cm}int broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+hash\+Code (
\begin{DoxyParamCaption}
{}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a8fe3a848dd59a9a072680c711fa86766}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_af2a3f7b805b2fea8605b821b53f56958}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!is\+Expressed@{is\+Expressed}}
\index{is\+Expressed@{is\+Expressed}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{is\+Expressed}]{\setlength{\rightskip}{0pt plus 5cm}boolean broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+is\+Expressed (
\begin{DoxyParamCaption}
\item[{Gene}]{gene}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_af2a3f7b805b2fea8605b821b53f56958}
Whether the gene is significantly expressed in the sample 
\begin{DoxyParams}{Parameters}
{\em gene} & The gene \\
\hline
\end{DoxyParams}
\begin{DoxyReturn}{Returns}
Whether the gene is expressed at the given significance level 
\end{DoxyReturn}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ac476ccc56fab96d17419127a8cf313a1}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!score\+Window@{score\+Window}}
\index{score\+Window@{score\+Window}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{score\+Window}]{\setlength{\rightskip}{0pt plus 5cm}Scan\+Statistic\+Score broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+score\+Window (
\begin{DoxyParamCaption}
\item[{Gene}]{gene, }
\item[{Annotation}]{window}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ac476ccc56fab96d17419127a8cf313a1}
Get scan statistic score for a specified window 
\begin{DoxyParams}{Parameters}
{\em gene} & Parent gene \\
\hline
{\em window} & The window \\
\hline
\end{DoxyParams}
\begin{DoxyReturn}{Returns}
Scan statistic score for window with global stats referring to parent transcript 
\end{DoxyReturn}


Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ae556c14ccb3cb7b63182adf72e3630d7}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!score\+Window@{score\+Window}}
\index{score\+Window@{score\+Window}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{score\+Window}]{\setlength{\rightskip}{0pt plus 5cm}Scan\+Statistic\+Score broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+score\+Window (
\begin{DoxyParamCaption}
\item[{Gene}]{gene, }
\item[{Annotation}]{window, }
\item[{double}]{count}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ae556c14ccb3cb7b63182adf72e3630d7}
Get scan statistic score for a specified window 
\begin{DoxyParams}{Parameters}
{\em gene} & Parent gene \\
\hline
{\em window} & The window \\
\hline
{\em count} & The number to use as the region count in score \\
\hline
\end{DoxyParams}
\begin{DoxyReturn}{Returns}
Scan statistic score for window with global stats referring to parent transcript 
\end{DoxyReturn}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a477fb361142dde07116598599bb7ef52}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!score\+Window\+With\+Fragment\+Length\+Filter@{score\+Window\+With\+Fragment\+Length\+Filter}}
\index{score\+Window\+With\+Fragment\+Length\+Filter@{score\+Window\+With\+Fragment\+Length\+Filter}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{score\+Window\+With\+Fragment\+Length\+Filter}]{\setlength{\rightskip}{0pt plus 5cm}Scan\+Statistic\+Score broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+score\+Window\+With\+Fragment\+Length\+Filter (
\begin{DoxyParamCaption}
\item[{Gene}]{gene, }
\item[{Annotation}]{window}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a477fb361142dde07116598599bb7ef52}
Get scan statistic score for a window based on filtering fragments overlapping the window by size 
\begin{DoxyParams}{Parameters}
{\em gene} & The gene \\
\hline
{\em window} & The window \\
\hline
\end{DoxyParams}
\begin{DoxyReturn}{Returns}
Scan statistic score for window with global stats referring to parent transcript 
\end{DoxyReturn}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a2013287af35974ad48c4cd57b099ac4b}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!set\+Expression\+Scan\+Pvalue\+Cutoff@{set\+Expression\+Scan\+Pvalue\+Cutoff}}
\index{set\+Expression\+Scan\+Pvalue\+Cutoff@{set\+Expression\+Scan\+Pvalue\+Cutoff}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{set\+Expression\+Scan\+Pvalue\+Cutoff}]{\setlength{\rightskip}{0pt plus 5cm}void broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+set\+Expression\+Scan\+Pvalue\+Cutoff (
\begin{DoxyParamCaption}
\item[{double}]{expression\+Scan\+Pval\+Cutoff}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a2013287af35974ad48c4cd57b099ac4b}
Set genome wide scan P value cutoff for expression of transcript 
\begin{DoxyParams}{Parameters}
{\em expression\+Scan\+Pval\+Cutoff} & P value cutoff for transcript expression against genomic background \\
\hline
\end{DoxyParams}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a77cf4e55d644cb09fa376f9d8b41c538}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!set\+Sample\+Name@{set\+Sample\+Name}}
\index{set\+Sample\+Name@{set\+Sample\+Name}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{set\+Sample\+Name}]{\setlength{\rightskip}{0pt plus 5cm}void broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+set\+Sample\+Name (
\begin{DoxyParamCaption}
\item[{String}]{name}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a77cf4e55d644cb09fa376f9d8b41c538}
Set the sample name 
\begin{DoxyParams}{Parameters}
{\em name} & Sample name \\
\hline
\end{DoxyParams}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a8d27ed6393a7f774100c706c5a1b9b8d}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!trim\+Max\+Contiguous@{trim\+Max\+Contiguous}}
\index{trim\+Max\+Contiguous@{trim\+Max\+Contiguous}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{trim\+Max\+Contiguous}]{\setlength{\rightskip}{0pt plus 5cm}static Annotation broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+trim\+Max\+Contiguous (
\begin{DoxyParamCaption}
\item[{Annotation}]{window, }
\item[{List$<$ Double $>$}]{data, }
\item[{double}]{quantile}
\end{DoxyParamCaption}
)\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a8d27ed6393a7f774100c706c5a1b9b8d}
Trim the region to max contiguous subregion above a certain quantile 
\begin{DoxyParams}{Parameters}
{\em window} & The region \\
\hline
{\em data} & Position level list of counts within the region \\
\hline
{\em quantile} & Quantile for trim max contiguous \\
\hline
\end{DoxyParams}
\begin{DoxyReturn}{Returns}
Trimmed region 
\end{DoxyReturn}


\subsection{Member Data Documentation}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ae06d8e63d6ddf7093085beba12b22e8a}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!data@{data}}
\index{data@{data}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{data}]{\setlength{\rightskip}{0pt plus 5cm}Transcriptome\+Space\+Alignment\+Model broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+data\hspace{0.3cm}{\ttfamily [protected]}}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ae06d8e63d6ddf7093085beba12b22e8a}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a0f463fa018b0f1cc4495a3dfdd2e9ef5}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!expression\+By\+Scan\+Pval@{expression\+By\+Scan\+Pval}}
\index{expression\+By\+Scan\+Pval@{expression\+By\+Scan\+Pval}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{expression\+By\+Scan\+Pval}]{\setlength{\rightskip}{0pt plus 5cm}boolean broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+expression\+By\+Scan\+Pval\hspace{0.3cm}{\ttfamily [protected]}}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a0f463fa018b0f1cc4495a3dfdd2e9ef5}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ae1a282e95f82d5974d5e3c4189a81f8c}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!expression\+Cutoff\+Value@{expression\+Cutoff\+Value}}
\index{expression\+Cutoff\+Value@{expression\+Cutoff\+Value}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{expression\+Cutoff\+Value}]{\setlength{\rightskip}{0pt plus 5cm}double broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+expression\+Cutoff\+Value\hspace{0.3cm}{\ttfamily [protected]}}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ae1a282e95f82d5974d5e3c4189a81f8c}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ac971b6d1585808b15cc2ef8807225bf3}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!fully\+Contained\+Reads@{fully\+Contained\+Reads}}
\index{fully\+Contained\+Reads@{fully\+Contained\+Reads}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{fully\+Contained\+Reads}]{\setlength{\rightskip}{0pt plus 5cm}boolean broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+fully\+Contained\+Reads\hspace{0.3cm}{\ttfamily [protected]}}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ac971b6d1585808b15cc2ef8807225bf3}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ad517adb3cfab02f8f65bc10818ebca71}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!gene\+Avg\+Coverage@{gene\+Avg\+Coverage}}
\index{gene\+Avg\+Coverage@{gene\+Avg\+Coverage}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{gene\+Avg\+Coverage}]{\setlength{\rightskip}{0pt plus 5cm}Map$<$Gene, Double$>$ broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+gene\+Avg\+Coverage\hspace{0.3cm}{\ttfamily [protected]}}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ad517adb3cfab02f8f65bc10818ebca71}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a8b1f58470538adcd26f47252bdf94232}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!genes\+By\+Chr@{genes\+By\+Chr}}
\index{genes\+By\+Chr@{genes\+By\+Chr}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{genes\+By\+Chr}]{\setlength{\rightskip}{0pt plus 5cm}Map$<$String, Collection$<$Gene$>$ $>$ broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+genes\+By\+Chr\hspace{0.3cm}{\ttfamily [protected]}}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a8b1f58470538adcd26f47252bdf94232}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a528cf3b7c628a3f4a567bd33ba75ca0f}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!genes\+By\+Name@{genes\+By\+Name}}
\index{genes\+By\+Name@{genes\+By\+Name}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{genes\+By\+Name}]{\setlength{\rightskip}{0pt plus 5cm}Map$<$String, Gene$>$ broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+genes\+By\+Name\hspace{0.3cm}{\ttfamily [protected]}}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a528cf3b7c628a3f4a567bd33ba75ca0f}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_af611a295e308f4f0d0849a80139ad7ff}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!gene\+Scores@{gene\+Scores}}
\index{gene\+Scores@{gene\+Scores}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{gene\+Scores}]{\setlength{\rightskip}{0pt plus 5cm}Map$<$Gene, Scan\+Statistic\+Score$>$ broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+gene\+Scores\hspace{0.3cm}{\ttfamily [protected]}}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_af611a295e308f4f0d0849a80139ad7ff}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a4f08fb5a371b51daf757152c83863ac2}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!logger@{logger}}
\index{logger@{logger}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{logger}]{\setlength{\rightskip}{0pt plus 5cm}Logger broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+logger = Logger.\+get\+Logger(Sample\+Data.\+class.\+get\+Name())\hspace{0.3cm}{\ttfamily [static]}, {\ttfamily [protected]}}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a4f08fb5a371b51daf757152c83863ac2}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a214cced67761f0090ba326708620d3be}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!max\+Fragment\+Length\+Data@{max\+Fragment\+Length\+Data}}
\index{max\+Fragment\+Length\+Data@{max\+Fragment\+Length\+Data}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{max\+Fragment\+Length\+Data}]{\setlength{\rightskip}{0pt plus 5cm}Transcriptome\+Space\+Alignment\+Model broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+max\+Fragment\+Length\+Data\hspace{0.3cm}{\ttfamily [protected]}}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a214cced67761f0090ba326708620d3be}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a5b0011ca04c2ee52ec32f8508b82ae99}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!sample\+Name@{sample\+Name}}
\index{sample\+Name@{sample\+Name}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{sample\+Name}]{\setlength{\rightskip}{0pt plus 5cm}String broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+sample\+Name\hspace{0.3cm}{\ttfamily [protected]}}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a5b0011ca04c2ee52ec32f8508b82ae99}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ab5fe21e8281091d1be8de525e17d8a3a}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!step\+Size@{step\+Size}}
\index{step\+Size@{step\+Size}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{step\+Size}]{\setlength{\rightskip}{0pt plus 5cm}int broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+step\+Size\hspace{0.3cm}{\ttfamily [protected]}}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ab5fe21e8281091d1be8de525e17d8a3a}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a6a53d104b93427859b6fd2c4ffbfdf6d}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!window\+Scores@{window\+Scores}}
\index{window\+Scores@{window\+Scores}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{window\+Scores}]{\setlength{\rightskip}{0pt plus 5cm}Map$<$Gene, Map$<$Annotation, Scan\+Statistic\+Score$>$ $>$ broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+window\+Scores\hspace{0.3cm}{\ttfamily [protected]}}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a6a53d104b93427859b6fd2c4ffbfdf6d}
\hypertarget{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_af155a04e70a9feafd812aa6135f5b732}{\index{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}!window\+Size@{window\+Size}}
\index{window\+Size@{window\+Size}!broad\+::pda\+::seq\+::protection\+::\+Sample\+Data@{broad\+::pda\+::seq\+::protection\+::\+Sample\+Data}}
\subsubsection[{window\+Size}]{\setlength{\rightskip}{0pt plus 5cm}int broad.\+pda.\+seq.\+protection.\+Sample\+Data.\+window\+Size\hspace{0.3cm}{\ttfamily [protected]}}}\label{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_af155a04e70a9feafd812aa6135f5b732}


The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
java/broad/pda/seq/protection/\hyperlink{protection_2_sample_data_8java}{Sample\+Data.\+java}\end{DoxyCompactItemize}
