\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model}{\section{broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model Class Reference}
\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model}\index{broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model@{broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model}}
}


Collaboration diagram for broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model\+:
\subsection*{Public Member Functions}
\begin{DoxyCompactItemize}
\item 
\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a6fa2acc5a5e70dcb0b7d9f36c35c5f57}{Continuous\+Data\+Alignment\+Model} (\hyperlink{interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model}{Alignment\+Data\+Model} data)  throws I\+O\+Exception
\item 
\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a954fd7962b745c8952dc27204ebe6252}{Continuous\+Data\+Alignment\+Model} (\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} data)  throws I\+O\+Exception
\item 
\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_adec2aa3724e353ed1f7678e17cb8e74a}{Continuous\+Data\+Alignment\+Model} (\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} data, int E\+F, int min\+Splice\+Support)  throws I\+O\+Exception
\item 
\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a3cbdd60567c630a9741fb1b187aa8f2b}{Continuous\+Data\+Alignment\+Model} (\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} data, File\mbox{[}$\,$\mbox{]} mask\+Files, int min\+Splice\+Support)  throws I\+O\+Exception
\item 
\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ad0d8dfa7ae363e2d07df5fd31ac3b3e2}{Continuous\+Data\+Alignment\+Model} (\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} data, File\mbox{[}$\,$\mbox{]} mask\+Files, int E\+F, int min\+Splice\+Support, \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} paired\+Data, \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} strand\+Specific\+Reads)  throws I\+O\+Exception
\item 
\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a75f19ed69ba5c917dd852d1c7c8ade65}{Continuous\+Data\+Alignment\+Model} (\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} data, File\mbox{[}$\,$\mbox{]} mask\+Files, int E\+F, int min\+Splice\+Support, \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} paired\+Data, \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} strand\+Specific\+Reads, boolean upweight)  throws I\+O\+Exception
\item 
\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a0f083a972a3995000d0a0b3e146ddd1b}{Continuous\+Data\+Alignment\+Model} (\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} data, Map$<$ String, Integer $>$ mask\+File\+Data, int E\+F, int min\+Splice\+Support, \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} paired\+Data, \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} strand\+Specific\+Reads)  throws I\+O\+Exception
\item 
\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a9966e36fdc254e2ae504e1ec8aa3f756}{Continuous\+Data\+Alignment\+Model} (\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} data, Map$<$ String, Integer $>$ mask\+File\+Data, int E\+F, int min\+Splice\+Support, \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} paired\+Data, \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} strand\+Specific\+Reads, boolean up\+Weight)  throws I\+O\+Exception
\item 
\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_abfa387c5108127e9167555282c98ed05}{Continuous\+Data\+Alignment\+Model} (\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} data, Map$<$ String, Integer $>$ mask\+File\+Data, int E\+F, int min\+Splice\+Support, \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} paired\+Data, \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} strand\+Specific\+Reads, int chunk\+Size, boolean up\+Weight\+Splices)  throws I\+O\+Exception
\item 
\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a9f8a43ee6ff09a8f6e30f8608844085f}{Continuous\+Data\+Alignment\+Model} (\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} data, File\mbox{[}$\,$\mbox{]} mask\+Files, int E\+F, int min\+Splice\+Support, \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} paired\+Data)  throws I\+O\+Exception
\item 
\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ab3be2b171b13a6d8844ca6cc4415533d}{Continuous\+Data\+Alignment\+Model} (\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} data, File\mbox{[}$\,$\mbox{]} mask\+Files, int E\+F, int min\+Splice\+Support)  throws I\+O\+Exception
\item 
void \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ae9180b76d6c1bc1ea8d6024a2b08c5db}{set\+Find\+Max\+Contiguous} (boolean find\+Max\+Contiguous)
\item 
void \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_aa17cb50bb398dc7770859377e8d9771e}{set\+Trim\+Ends} (boolean trim\+Ends)
\item 
void \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ab7ab620d4f4c1b25acc716d83a8d72b6}{set\+Min\+Contguous\+Segment\+Size} (int min\+Size)
\item 
void \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a8c1d4aef55169c5f5cc897edec41c1ff}{set\+Alpha} (double alpha)
\item 
void \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_af559e50d0cbcc7525afd106c4bb93418}{set\+Up\+Weight\+Splices} (boolean set\+Flag)
\item 
void \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a61ed951795b265ba619f938e715ca1ab}{set\+Min\+Number\+Of\+Splices} (int min\+Splice\+Support)
\item 
void \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a1e3aa5641fa5aa9963d9a3b05870012a}{set\+Extension\+Factor} (int extension\+Factor)
\item 
void \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ac019bf49be802c3b62d3a3055104c365}{set\+Mask\+File\+Data} (Map$<$ String, Integer $>$ masked\+Region\+Map)
\item 
void \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a13f7d8efa23552e98f798e9deec570de}{set\+Minimum\+Mapping\+Quality} (double min\+Map\+Qual)
\item 
void \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a05c9b50b32fd0cdfee049966ae7b9c1e}{set\+Trim\+Quantile} (double quantile)
\item 
\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a5f78dee2ade4678e5ac2d04576c83b6d}{get\+Data} ()
\item 
double \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a16fcd43426d01ba29a6e50d8a36ab38f}{get\+Lambda} (String chr)  throws I\+O\+Exception
\item 
double \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a68033365b74f1d7755eb6e712f47f863}{get\+Sum} (String chr)  throws I\+O\+Exception
\item 
double \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_affc935b97a13f0fc855cd5c1878dc8b2}{get\+Number\+Of\+Reads} (String chr)  throws I\+O\+Exception
\item 
double \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_adba4f6a27c198a96fcfa79a643fca0a3}{get\+Number\+Markers} (String chr)  throws I\+O\+Exception
\item 
double \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a5b689c01b77461487789d172e013a3f6}{get\+Number\+Of\+B\+Ps} ()
\item 
int \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a8e4c4c86479ee745866baef364fdd6fe}{get\+Count} (Collection$<$ Annotation $>$ Annotation)  throws I\+O\+Exception 
\item 
int\mbox{[}$\,$\mbox{]} \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_aaee0b9274c46753d21f44590c58c072c}{get\+Gene\+Counts} (Collection$<$ Gene $>$ genes)  throws I\+O\+Exception
\item 
int\mbox{[}$\,$\mbox{]} \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a76ef156f96e86fe1eea8b45e94d22f02}{get\+Gene\+Counts\+Without\+Dataset\+Totals} (Collection$<$ Gene $>$ genes)  throws I\+O\+Exception
\item 
double\mbox{[}$\,$\mbox{]} \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_acd9c8501fd75af7cb102cd6886cbe15b}{get\+Counts\+Per\+Bp} (Annotation align, Interval\+Tree$<$ Alignment $>$ tree)
\item 
List$<$ Double $>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a2f5c2e47fe3a39cb416bfddf280e2c7f}{get\+Counts\+Per\+Bp} (Gene gene, Interval\+Tree$<$ Alignment $>$ tree)
\item 
Map$<$ Gene, double\mbox{[}$\,$\mbox{]}$>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_aef09a54b699b7e8130f80515431e250c}{get\+Gene\+Expression} (Collection$<$ Gene $>$ genes)  throws I\+O\+Exception
\item 
int \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a24fc6c0cea13276596ad51dbab778d38}{get\+Counts} (Map$<$ String, Interval\+Tree$<$ Annotation $>$$>$ gene\+Tree, Collection$<$ String $>$ multimappers)  throws I\+O\+Exception
\item 
int \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_aadb1c1dfc12a73268e0552fb99e01024}{get\+Chromosome\+Length} (String chr)
\item 
Map$<$ String, Integer $>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_af1b3646c528368d2b2af71fb820350a6}{get\+Chromosome\+Lengths} ()
\item 
double\mbox{[}$\,$\mbox{]} \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_af7d90fa931951069416e3ad235e99f37}{get\+Data\+For\+Alignment} (Annotation align)  throws I\+O\+Exception
\item 
List$<$ Double $>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a00cf5fa4275d363fa77bf05454e7909c}{get\+Data\+For\+Gene} (Gene gene)  throws I\+O\+Exception
\item 
Map$<$ Gene, List$<$ Double $>$ $>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a0a1e9f474a59ee99a9c548092d35bf5b}{get\+Data\+For\+Gene} (Collection$<$ Gene $>$ Annotation)  throws I\+O\+Exception
\item 
\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a1db375ffca9153b8143df70c0bece395}{get\+Paired\+Data} ()
\item 
Collection$<$ Annotation $>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a9055da2c6c85869da10e9bff9c0fbac9}{scan} (int\mbox{[}$\,$\mbox{]} window\+Sizes, double alpha)  throws I\+O\+Exception
\item 
Collection$<$ Annotation $>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a854b44de12136762faa57412f7b47465}{scan} (int\mbox{[}$\,$\mbox{]} window\+Sizes, double alpha, String chr)  throws I\+O\+Exception
\item 
void \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a783abf082b1ebce3361a143d40764b04}{scan\+Genome} (\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model}{Continuous\+Data\+Alignment\+Model} data2, int window\+Size, String chr, File\+Writer writer, boolean filter\+Significance)  throws I\+O\+Exception
\item 
double\mbox{[}$\,$\mbox{]} \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_acde220ce2b79ad3a0cb0f0493eb52a2f}{scan\+P\+Rate} (Gene gene)  throws I\+O\+Exception
\item 
double\mbox{[}$\,$\mbox{]} \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ac64273677f9810b6e1a3598e1678b343}{scan\+P\+Rate} (Gene gene, Interval\+Tree$<$ Alignment $>$ tree)  throws I\+O\+Exception
\item 
double\mbox{[}$\,$\mbox{]} \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a2e6ae03ce97022cd39bfc2102c0580bd}{scan\+P\+Rate} (Gene gene, Interval\+Tree$<$ Alignment $>$ tree, double local\+Lambda)  throws I\+O\+Exception
\item 
double\mbox{[}$\,$\mbox{]} \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a3e7ef2b826b64f787457935fdb71e558}{scan\+P\+Rate} (Gene gene, Interval\+Tree$<$ Alignment $>$ tree, double local\+Lambda, int num\+Markers)  throws I\+O\+Exception
\item 
double\mbox{[}$\,$\mbox{]} \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_afdb0844bc8468e30a3a2b2cd55a13e62}{scan\+Region} (int fixed\+Width, \hyperlink{interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation}{Lightweight\+Genomic\+Annotation} region)  throws I\+O\+Exception
\item 
double\mbox{[}$\,$\mbox{]} \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a8cffd238bd6070c45d5bd0bb72b43aeb}{score\+Segment} (Annotation align)  throws I\+O\+Exception
\item 
Map$<$ Annotation, double\mbox{[}$\,$\mbox{]}$>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a7009b0bc1a05ca28f02a06831778168b}{score\+Segments} (Collection$<$ Annotation $>$ set)  throws I\+O\+Exception
\item 
Map$<$ Annotation, double\mbox{[}$\,$\mbox{]}$>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a594e568f254f787301c484cb79d443cc}{score\+Segments} (Collection$<$ Annotation $>$ set, String chr\+To\+Use)  throws I\+O\+Exception 
\item 
Map$<$ Annotation, double\mbox{[}$\,$\mbox{]}$>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a562be161ea6de2e8a1b104bdbd6d449b}{score\+Segments} (Collection$<$ Annotation $>$ set, String chr\+To\+Use, boolean ignore\+Alpha)  throws I\+O\+Exception
\item 
double\mbox{[}$\,$\mbox{]} \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a11cd7ab6673b37e19f9cfe066130ba20}{score\+Gene} (Gene align)  throws I\+O\+Exception 
\item 
double\mbox{[}$\,$\mbox{]} \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a3fedd5b944ae861cb5006ed403af033e}{score\+Gene} (Gene align, double local\+Lambda)  throws I\+O\+Exception 
\item 
\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score}{Gene\+Score} \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a0d920606dbcf6437d4e4cf8f3e415156}{score\+Gene} (Gene align, Interval\+Tree$<$ Alignment $>$ tree, double local\+Lambda)  throws I\+O\+Exception 
\item 
\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score}{Gene\+Score} \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a9bdf34088aadf81c3eabae435fe67767}{score\+Gene} (Gene align, Interval\+Tree$<$ Alignment $>$ tree, double local\+Lambda, int num\+Markers)  throws I\+O\+Exception 
\item 
Map$<$ Gene, double\mbox{[}$\,$\mbox{]}$>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_af1e5000926013a041e9597fb3a2dc5fe}{score\+Genes} (Collection$<$ Gene $>$ set, String chr\+To\+Use, Interval\+Tree$<$ Alignment $>$ tree)  throws I\+O\+Exception
\item 
\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score}{Gene\+Score} \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a3d06acc7e76ca7cbf52b2e6f41ad2bb1}{get\+Peak} (Gene gene, Gene start\+Codon, Interval\+Tree$<$ Alignment $>$ tree)  throws I\+O\+Exception 
\item 
Gene \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a067da57de440289d5bc6d7a4dbe5427d}{get\+Peak} (Gene gene, Gene start\+Codon)  throws I\+O\+Exception 
\item 
\hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ac1217728072eb3635f3e7dea450af701}{get\+Alignment\+Data\+Model\+Stats} ()
\item 
double\mbox{[}$\,$\mbox{]} \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a775a3a33946ec4040d85a2b3ba738ab9}{score\+Gene} (Gene align, Interval\+Tree$<$ Alignment $>$ tree)  throws I\+O\+Exception 
\item 
Map$<$ Gene, double\mbox{[}$\,$\mbox{]}$>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_affe8308674627334d5a187dde355b468}{score\+Genes} (Collection$<$ Gene $>$ genes)  throws I\+O\+Exception
\item 
Map$<$ Gene, double\mbox{[}$\,$\mbox{]}$>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a37b049c25b8723b27209f0cac8a9ab0f}{score\+Genes} (Collection$<$ Gene $>$ set, String chr\+To\+Use)  throws I\+O\+Exception
\item 
Map$<$ Gene, double\mbox{[}$\,$\mbox{]}$>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_aedf7ee948f8f74ee7e3f8c926bd77444}{score\+Genes} (Collection$<$ Gene $>$ set, String chr\+To\+Use, int size)  throws I\+O\+Exception
\item 
void \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a8fe3ed4e57d504db94f449a29385b098}{score\+Genes} (\hyperlink{classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser}{B\+E\+D\+File\+Parser} genes)  throws I\+O\+Exception
\item 
Map$<$ \hyperlink{classbroad_1_1pda_1_1seq_1_1graph_1_1_path}{Path}, double\mbox{[}$\,$\mbox{]}$>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_af620a01c0d3be891bdd277470553e666}{score\+Paths\+For\+Only\+R\+P\+K\+M} (Collection$<$ \hyperlink{classbroad_1_1pda_1_1seq_1_1graph_1_1_path}{Path} $>$ paths)
\item 
Map$<$ \hyperlink{classbroad_1_1pda_1_1seq_1_1graph_1_1_path}{Path}, double\mbox{[}$\,$\mbox{]}$>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a9e84cfdbc07c6b49f3e35b092726714f}{score\+Paths} (Collection$<$ \hyperlink{classbroad_1_1pda_1_1seq_1_1graph_1_1_path}{Path} $>$ paths, double lambda)  throws I\+O\+Exception
\item 
double \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a31cfc1d1c12f0c7ebece09e6c5bd4f43}{count} (Annotation align)  throws I\+O\+Exception
\item 
Gene \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a859581141e70dd3ad658aa3fad127d32}{trim\+Max\+Contiguous} (Gene region, double quantile)  throws I\+O\+Exception 
\item 
List$<$ Gene $>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a5374e1e5b261ff43e6da700967631b83}{filter\+Genes\+By\+Expression} (Map$<$ Gene, List$<$ Double $>$$>$ counts\+Map)
\item 
Gene \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a1c47ee216d5f3910c63d01275e0a0634}{trim\+Gene\+Ends} (Gene gene, double quantile)  throws I\+O\+Exception
\item 
void \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a1b6a5a28227911fd8ce4552dee2ccd36}{write\+Connections} (String save, \hyperlink{classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t}{Chromosome\+With\+Bubbles\+J\+Graph\+T} graph)  throws I\+O\+Exception
\item 
boolean \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a507f17b9636367cfc4dfa6e8bf25e7f6}{has\+Data\+For\+Chromosome} (String chr)
\item 
void \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a05b31a3d41397a1ff0e91b110e06e0ec}{reset\+Tree\+Cache} ()
\item 
double \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a5970adb73f2d1e0fd26b3334c982bbda}{C\+V} (Gene gene, Interval\+Tree$<$ Alignment $>$ tree)  throws I\+O\+Exception
\item 
Collection$<$ Annotation $>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a663b9546e782ac0dbded4f9d375dc8a3}{permute\+Regions} (Annotation align)  throws I\+O\+Exception
\item 
Collection$<$ Annotation $>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a767afcf399cb81b942f8bab5bd87fdaa}{raw\+Regions} (Annotation align)  throws I\+O\+Exception
\item 
int \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ad2358621505b89beef76afb69d366021}{get\+Bases\+Covered} (Annotation region)  throws I\+O\+Exception 
\item 
int \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_aa510bb71e2a3dc533af6f6188a6fa6af}{get\+Bases\+Covered} (Annotation region, int E\+F)  throws I\+O\+Exception 
\end{DoxyCompactItemize}
\subsection*{Static Public Member Functions}
\begin{DoxyCompactItemize}
\item 
static int\mbox{[}$\,$\mbox{]} \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a87118dfc0d49b68daca8cd351ae3bcab}{get\+Widths} (String str)
\item 
static Map$<$ \hyperlink{classbroad_1_1pda_1_1seq_1_1graph_1_1_path}{Path}, double\mbox{[}$\,$\mbox{]}$>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a1353042a53391b6ba89d333fdd5fab11}{score\+Paths} (Collection$<$ \hyperlink{classbroad_1_1pda_1_1seq_1_1graph_1_1_path}{Path} $>$ paths, \hyperlink{classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t}{Chromosome\+With\+Bubbles\+J\+Graph\+T} graph, double alpha)
\item 
static Map$<$ \hyperlink{classbroad_1_1pda_1_1seq_1_1graph_1_1_path}{Path}, double\mbox{[}$\,$\mbox{]}$>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a8c3eca6b8e9d8db29f3e0a8a7f25207b}{score\+Paths} (Collection$<$ \hyperlink{classbroad_1_1pda_1_1seq_1_1graph_1_1_path}{Path} $>$ paths, double lambda, double num\+Markers, double num\+Reads, double R\+P\+K\+M\+Contant, Map$<$ Annotation, Double $>$ local\+Rates, double alpha)
\item 
static double \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a58a72f78575d4e504c55b46c487d25c7}{calculate\+P\+Val} (int k, double lambda, double w, double T)
\item 
static void \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ab9f9fcd2fb7cfe1b353af62698759f73}{write\+Full\+B\+E\+D} (String save, Map$<$ Gene, double\mbox{[}$\,$\mbox{]}$>$ map, double alpha)  throws I\+O\+Exception
\item 
static void \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a2e67d6ebf44bb95cdb4548a6f75d6c4a}{write\+Full\+B\+E\+D} (String save, Collection$<$ Gene $>$ segments)  throws I\+O\+Exception
\item 
static void \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a8777cdf9066d2f8f1cdd818d9e728d69}{main} (String\mbox{[}$\,$\mbox{]} args)  throws Exception 
\item 
static Collection$<$ Gene $>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a2fc579cf99a725d9637eaf4891a4fd11}{filter\+Paths} (Map$<$ \hyperlink{classbroad_1_1pda_1_1seq_1_1graph_1_1_path}{Path}, double\mbox{[}$\,$\mbox{]}$>$ scores, double alpha2)
\item 
static Map$<$ String, Integer $>$ \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a039655b2e536f7a9728f3697c14c4505}{parse\+Mask\+Files} (File\mbox{[}$\,$\mbox{]} files)  throws I\+O\+Exception
\end{DoxyCompactItemize}
\subsection*{Static Public Attributes}
\begin{DoxyCompactItemize}
\item 
static int \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ada8b56f8e48ecc0df55f92f0a4e3a537}{D\+E\+F\+A\+U\+L\+T\+\_\+\+M\+I\+N\+\_\+\+M\+A\+P\+P\+I\+N\+G\+\_\+\+Q\+U\+A\+L\+I\+T\+Y} = 5
\item 
static int \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_afcf03553918b3c7cd3b2c7d183b35c97}{D\+E\+F\+A\+U\+L\+T\+\_\+\+I\+N\+S\+E\+R\+T\+\_\+\+S\+I\+Z\+E\+\_\+\+F\+U\+D\+G\+E} = 20
\item 
static double \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a1c80e71adece28e35fee890de6ed2304}{D\+E\+F\+A\+U\+L\+T\+\_\+\+I\+N\+S\+\_\+\+S\+I\+Z\+E\+\_\+\+P\+V\+A\+L} = 0.\+05
\item 
static double \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ad2e9aa31d5024257b5417f729ac5aa6e}{D\+E\+F\+A\+U\+L\+T\+\_\+\+M\+A\+K\+E\+\_\+\+G\+E\+N\+E\+\_\+\+O\+V\+E\+R\+L\+A\+P} = 0.\+3
\end{DoxyCompactItemize}


\subsection{Constructor \& Destructor Documentation}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a6fa2acc5a5e70dcb0b7d9f36c35c5f57}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!Continuous\+Data\+Alignment\+Model@{Continuous\+Data\+Alignment\+Model}}
\index{Continuous\+Data\+Alignment\+Model@{Continuous\+Data\+Alignment\+Model}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{Continuous\+Data\+Alignment\+Model}]{\setlength{\rightskip}{0pt plus 5cm}broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+Continuous\+Data\+Alignment\+Model (
\begin{DoxyParamCaption}
\item[{{\bf Alignment\+Data\+Model}}]{data}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a6fa2acc5a5e70dcb0b7d9f36c35c5f57}
Constructor to build the data alignment model 
\begin{DoxyParams}{Parameters}
{\em data} & the alignment data model that stores the read data \\
\hline
\end{DoxyParams}

\begin{DoxyExceptions}{Exceptions}
{\em I\+O\+Exception} & \\
\hline
\end{DoxyExceptions}


Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a954fd7962b745c8952dc27204ebe6252}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!Continuous\+Data\+Alignment\+Model@{Continuous\+Data\+Alignment\+Model}}
\index{Continuous\+Data\+Alignment\+Model@{Continuous\+Data\+Alignment\+Model}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{Continuous\+Data\+Alignment\+Model}]{\setlength{\rightskip}{0pt plus 5cm}broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+Continuous\+Data\+Alignment\+Model (
\begin{DoxyParamCaption}
\item[{{\bf Alignment\+Data\+Model\+Stats}}]{data}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a954fd7962b745c8952dc27204ebe6252}
Constructor to build the data alignment model 
\begin{DoxyParams}{Parameters}
{\em data} & the alignment data model that computes the statistics and stores the read data \\
\hline
\end{DoxyParams}

\begin{DoxyExceptions}{Exceptions}
{\em I\+O\+Exception} & \\
\hline
\end{DoxyExceptions}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_adec2aa3724e353ed1f7678e17cb8e74a}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!Continuous\+Data\+Alignment\+Model@{Continuous\+Data\+Alignment\+Model}}
\index{Continuous\+Data\+Alignment\+Model@{Continuous\+Data\+Alignment\+Model}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{Continuous\+Data\+Alignment\+Model}]{\setlength{\rightskip}{0pt plus 5cm}broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+Continuous\+Data\+Alignment\+Model (
\begin{DoxyParamCaption}
\item[{{\bf Alignment\+Data\+Model\+Stats}}]{data, }
\item[{int}]{E\+F, }
\item[{int}]{min\+Splice\+Support}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_adec2aa3724e353ed1f7678e17cb8e74a}
Constructor to build the data alignment model 
\begin{DoxyParams}{Parameters}
{\em data} & the alignment data model that computes the statistics and stores the read data \\
\hline
{\em E\+F} & an extension factor used to extend reads \\
\hline
{\em min\+Splice\+Support} & the minimum number of spliced reads needed to add a connection in the graph \\
\hline
\end{DoxyParams}

\begin{DoxyExceptions}{Exceptions}
{\em I\+O\+Exception} & \\
\hline
\end{DoxyExceptions}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a3cbdd60567c630a9741fb1b187aa8f2b}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!Continuous\+Data\+Alignment\+Model@{Continuous\+Data\+Alignment\+Model}}
\index{Continuous\+Data\+Alignment\+Model@{Continuous\+Data\+Alignment\+Model}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{Continuous\+Data\+Alignment\+Model}]{\setlength{\rightskip}{0pt plus 5cm}broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+Continuous\+Data\+Alignment\+Model (
\begin{DoxyParamCaption}
\item[{{\bf Alignment\+Data\+Model\+Stats}}]{data, }
\item[{File\mbox{[}$\,$\mbox{]}}]{mask\+Files, }
\item[{int}]{min\+Splice\+Support}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a3cbdd60567c630a9741fb1b187aa8f2b}
Constructor to build the data alignment model 
\begin{DoxyParams}{Parameters}
{\em data} & the alignment data model that computes the statistics and stores the read data \\
\hline
{\em mask\+Files} & the mask files \\
\hline
{\em min\+Splice\+Support} & the minimum number of spliced reads needed to add a connection in the graph \\
\hline
\end{DoxyParams}

\begin{DoxyExceptions}{Exceptions}
{\em I\+O\+Exception} & \\
\hline
\end{DoxyExceptions}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ad0d8dfa7ae363e2d07df5fd31ac3b3e2}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!Continuous\+Data\+Alignment\+Model@{Continuous\+Data\+Alignment\+Model}}
\index{Continuous\+Data\+Alignment\+Model@{Continuous\+Data\+Alignment\+Model}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{Continuous\+Data\+Alignment\+Model}]{\setlength{\rightskip}{0pt plus 5cm}broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+Continuous\+Data\+Alignment\+Model (
\begin{DoxyParamCaption}
\item[{{\bf Alignment\+Data\+Model\+Stats}}]{data, }
\item[{File\mbox{[}$\,$\mbox{]}}]{mask\+Files, }
\item[{int}]{E\+F, }
\item[{int}]{min\+Splice\+Support, }
\item[{{\bf Alignment\+Data\+Model\+Stats}}]{paired\+Data, }
\item[{{\bf Alignment\+Data\+Model\+Stats}}]{strand\+Specific\+Reads}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ad0d8dfa7ae363e2d07df5fd31ac3b3e2}
Constructor to build the data alignment model 
\begin{DoxyParams}{Parameters}
{\em data} & the alignment data model that computes the statistics and stores the read data \\
\hline
{\em mask\+Files} & the mask files \\
\hline
{\em E\+F} & an extension factor used to extend reads \\
\hline
{\em min\+Splice\+Support} & the minimum number of spliced reads needed to add a connection in the graph \\
\hline
{\em paired\+Data} & the alignment data model that stores the stats for paired end reads and the paired end read data \\
\hline
{\em strand\+Specific\+Reads} & an alignment data model for the strand specific reads \\
\hline
\end{DoxyParams}

\begin{DoxyExceptions}{Exceptions}
{\em I\+O\+Exception} & \\
\hline
\end{DoxyExceptions}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a75f19ed69ba5c917dd852d1c7c8ade65}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!Continuous\+Data\+Alignment\+Model@{Continuous\+Data\+Alignment\+Model}}
\index{Continuous\+Data\+Alignment\+Model@{Continuous\+Data\+Alignment\+Model}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{Continuous\+Data\+Alignment\+Model}]{\setlength{\rightskip}{0pt plus 5cm}broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+Continuous\+Data\+Alignment\+Model (
\begin{DoxyParamCaption}
\item[{{\bf Alignment\+Data\+Model\+Stats}}]{data, }
\item[{File\mbox{[}$\,$\mbox{]}}]{mask\+Files, }
\item[{int}]{E\+F, }
\item[{int}]{min\+Splice\+Support, }
\item[{{\bf Alignment\+Data\+Model\+Stats}}]{paired\+Data, }
\item[{{\bf Alignment\+Data\+Model\+Stats}}]{strand\+Specific\+Reads, }
\item[{boolean}]{upweight}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a75f19ed69ba5c917dd852d1c7c8ade65}
Constructor to build the data alignment model 
\begin{DoxyParams}{Parameters}
{\em data} & the alignment data model that computes the statistics and stores the read data \\
\hline
{\em mask\+Files} & the mask files \\
\hline
{\em E\+F} & an extension factor used to extend reads \\
\hline
{\em min\+Splice\+Support} & the minimum number of spliced reads needed to add a connection in the graph \\
\hline
{\em paired\+Data} & the alignment data model that stores the stats for paired end reads and the paired end read data \\
\hline
{\em strand\+Specific\+Reads} & an alignment data model for the strand specific reads \\
\hline
{\em chunk\+Size} & the size of the cache chunk used \\
\hline
\end{DoxyParams}

\begin{DoxyExceptions}{Exceptions}
{\em I\+O\+Exception} & \\
\hline
\end{DoxyExceptions}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a0f083a972a3995000d0a0b3e146ddd1b}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!Continuous\+Data\+Alignment\+Model@{Continuous\+Data\+Alignment\+Model}}
\index{Continuous\+Data\+Alignment\+Model@{Continuous\+Data\+Alignment\+Model}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{Continuous\+Data\+Alignment\+Model}]{\setlength{\rightskip}{0pt plus 5cm}broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+Continuous\+Data\+Alignment\+Model (
\begin{DoxyParamCaption}
\item[{{\bf Alignment\+Data\+Model\+Stats}}]{data, }
\item[{Map$<$ String, Integer $>$}]{mask\+File\+Data, }
\item[{int}]{E\+F, }
\item[{int}]{min\+Splice\+Support, }
\item[{{\bf Alignment\+Data\+Model\+Stats}}]{paired\+Data, }
\item[{{\bf Alignment\+Data\+Model\+Stats}}]{strand\+Specific\+Reads}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a0f083a972a3995000d0a0b3e146ddd1b}
Constructor to build the data alignment model 
\begin{DoxyParams}{Parameters}
{\em data} & the alignment data model that computes the statistics and stores the read data \\
\hline
{\em mask\+File\+Data} & a map of all unalignable regions of the genome \\
\hline
{\em E\+F} & an extension factor used to extend reads \\
\hline
{\em min\+Splice\+Support} & the minimum number of spliced reads needed to add a connection in the graph \\
\hline
{\em paired\+Data} & the alignment data model that stores the stats for paired end reads and the paired end read data \\
\hline
{\em strand\+Specific\+Reads} & an alignment data model for the strand specific reads \\
\hline
\end{DoxyParams}

\begin{DoxyExceptions}{Exceptions}
{\em I\+O\+Exception} & \\
\hline
\end{DoxyExceptions}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a9966e36fdc254e2ae504e1ec8aa3f756}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!Continuous\+Data\+Alignment\+Model@{Continuous\+Data\+Alignment\+Model}}
\index{Continuous\+Data\+Alignment\+Model@{Continuous\+Data\+Alignment\+Model}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{Continuous\+Data\+Alignment\+Model}]{\setlength{\rightskip}{0pt plus 5cm}broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+Continuous\+Data\+Alignment\+Model (
\begin{DoxyParamCaption}
\item[{{\bf Alignment\+Data\+Model\+Stats}}]{data, }
\item[{Map$<$ String, Integer $>$}]{mask\+File\+Data, }
\item[{int}]{E\+F, }
\item[{int}]{min\+Splice\+Support, }
\item[{{\bf Alignment\+Data\+Model\+Stats}}]{paired\+Data, }
\item[{{\bf Alignment\+Data\+Model\+Stats}}]{strand\+Specific\+Reads, }
\item[{boolean}]{up\+Weight}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a9966e36fdc254e2ae504e1ec8aa3f756}
Constructor to build the data alignment model 
\begin{DoxyParams}{Parameters}
{\em data} & the alignment data model that computes the statistics and stores the read data \\
\hline
{\em mask\+File\+Data} & a map of all unalignable regions of the genome \\
\hline
{\em E\+F} & an extension factor used to extend reads \\
\hline
{\em min\+Splice\+Support} & the minimum number of spliced reads needed to add a connection in the graph \\
\hline
{\em paired\+Data} & the alignment data model that stores the stats for paired end reads and the paired end read data \\
\hline
{\em strand\+Specific\+Reads} & an alignment data model for the strand specific reads \\
\hline
{\em up\+Weight\+Splices} & whether to weight spliced reads based on their proportioncoverage scores \\
\hline
\end{DoxyParams}

\begin{DoxyExceptions}{Exceptions}
{\em I\+O\+Exception} & \\
\hline
\end{DoxyExceptions}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_abfa387c5108127e9167555282c98ed05}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!Continuous\+Data\+Alignment\+Model@{Continuous\+Data\+Alignment\+Model}}
\index{Continuous\+Data\+Alignment\+Model@{Continuous\+Data\+Alignment\+Model}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{Continuous\+Data\+Alignment\+Model}]{\setlength{\rightskip}{0pt plus 5cm}broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+Continuous\+Data\+Alignment\+Model (
\begin{DoxyParamCaption}
\item[{{\bf Alignment\+Data\+Model\+Stats}}]{data, }
\item[{Map$<$ String, Integer $>$}]{mask\+File\+Data, }
\item[{int}]{E\+F, }
\item[{int}]{min\+Splice\+Support, }
\item[{{\bf Alignment\+Data\+Model\+Stats}}]{paired\+Data, }
\item[{{\bf Alignment\+Data\+Model\+Stats}}]{strand\+Specific\+Reads, }
\item[{int}]{chunk\+Size, }
\item[{boolean}]{up\+Weight\+Splices}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_abfa387c5108127e9167555282c98ed05}
Constructor to build the data alignment model 
\begin{DoxyParams}{Parameters}
{\em data} & the alignment data model that computes the statistics and stores the read data \\
\hline
{\em mask\+File\+Data} & a map of all unalignable regions of the genome \\
\hline
{\em E\+F} & an extension factor used to extend reads \\
\hline
{\em min\+Splice\+Support} & the minimum number of spliced reads needed to add a connection in the graph \\
\hline
{\em paired\+Data} & the alignment data model that stores the stats for paired end reads and the paired end read data \\
\hline
{\em strand\+Specific\+Reads} & an alignment data model for the strand specific reads \\
\hline
{\em chunk\+Size} & the size of the cache chunk used \\
\hline
{\em up\+Weight\+Splices} & whether to weight spliced reads based on their proportion when computing coverage scores \\
\hline
\end{DoxyParams}

\begin{DoxyExceptions}{Exceptions}
{\em I\+O\+Exception} & \\
\hline
\end{DoxyExceptions}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a9f8a43ee6ff09a8f6e30f8608844085f}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!Continuous\+Data\+Alignment\+Model@{Continuous\+Data\+Alignment\+Model}}
\index{Continuous\+Data\+Alignment\+Model@{Continuous\+Data\+Alignment\+Model}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{Continuous\+Data\+Alignment\+Model}]{\setlength{\rightskip}{0pt plus 5cm}broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+Continuous\+Data\+Alignment\+Model (
\begin{DoxyParamCaption}
\item[{{\bf Alignment\+Data\+Model\+Stats}}]{data, }
\item[{File\mbox{[}$\,$\mbox{]}}]{mask\+Files, }
\item[{int}]{E\+F, }
\item[{int}]{min\+Splice\+Support, }
\item[{{\bf Alignment\+Data\+Model\+Stats}}]{paired\+Data}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a9f8a43ee6ff09a8f6e30f8608844085f}
Constructor to build the data alignment model 
\begin{DoxyParams}{Parameters}
{\em data} & the alignment data model that computes the statistics and stores the read data \\
\hline
{\em mask\+Files} & the mask files \\
\hline
{\em E\+F} & an extension factor used to extend reads \\
\hline
{\em min\+Splice\+Support} & the minimum number of spliced reads needed to add a connection in the graph \\
\hline
{\em paired\+Data} & the alignment data model that stores the stats for paired end reads and the paired end read data \\
\hline
\end{DoxyParams}

\begin{DoxyExceptions}{Exceptions}
{\em I\+O\+Exception} & \\
\hline
\end{DoxyExceptions}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ab3be2b171b13a6d8844ca6cc4415533d}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!Continuous\+Data\+Alignment\+Model@{Continuous\+Data\+Alignment\+Model}}
\index{Continuous\+Data\+Alignment\+Model@{Continuous\+Data\+Alignment\+Model}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{Continuous\+Data\+Alignment\+Model}]{\setlength{\rightskip}{0pt plus 5cm}broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+Continuous\+Data\+Alignment\+Model (
\begin{DoxyParamCaption}
\item[{{\bf Alignment\+Data\+Model\+Stats}}]{data, }
\item[{File\mbox{[}$\,$\mbox{]}}]{mask\+Files, }
\item[{int}]{E\+F, }
\item[{int}]{min\+Splice\+Support}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ab3be2b171b13a6d8844ca6cc4415533d}
Constructor to build the data alignment model 
\begin{DoxyParams}{Parameters}
{\em data} & the alignment data model that computes the statistics and stores the read data \\
\hline
{\em mask\+Files} & the mask files \\
\hline
{\em E\+F} & an extension factor used to extend reads \\
\hline
{\em min\+Splice\+Support} & the minimum number of spliced reads needed to add a connection in the graph \\
\hline
\end{DoxyParams}

\begin{DoxyExceptions}{Exceptions}
{\em I\+O\+Exception} & \\
\hline
\end{DoxyExceptions}


\subsection{Member Function Documentation}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a58a72f78575d4e504c55b46c487d25c7}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!calculate\+P\+Val@{calculate\+P\+Val}}
\index{calculate\+P\+Val@{calculate\+P\+Val}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{calculate\+P\+Val}]{\setlength{\rightskip}{0pt plus 5cm}static double broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+calculate\+P\+Val (
\begin{DoxyParamCaption}
\item[{int}]{k, }
\item[{double}]{lambda, }
\item[{double}]{w, }
\item[{double}]{T}
\end{DoxyParamCaption}
)\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a58a72f78575d4e504c55b46c487d25c7}


Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a31cfc1d1c12f0c7ebece09e6c5bd4f43}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!count@{count}}
\index{count@{count}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{count}]{\setlength{\rightskip}{0pt plus 5cm}double broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+count (
\begin{DoxyParamCaption}
\item[{Annotation}]{align}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a31cfc1d1c12f0c7ebece09e6c5bd4f43}


Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a5970adb73f2d1e0fd26b3334c982bbda}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!C\+V@{C\+V}}
\index{C\+V@{C\+V}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{C\+V}]{\setlength{\rightskip}{0pt plus 5cm}double broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+C\+V (
\begin{DoxyParamCaption}
\item[{Gene}]{gene, }
\item[{Interval\+Tree$<$ Alignment $>$}]{tree}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a5970adb73f2d1e0fd26b3334c982bbda}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a5374e1e5b261ff43e6da700967631b83}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!filter\+Genes\+By\+Expression@{filter\+Genes\+By\+Expression}}
\index{filter\+Genes\+By\+Expression@{filter\+Genes\+By\+Expression}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{filter\+Genes\+By\+Expression}]{\setlength{\rightskip}{0pt plus 5cm}List$<$Gene$>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+filter\+Genes\+By\+Expression (
\begin{DoxyParamCaption}
\item[{Map$<$ Gene, List$<$ Double $>$$>$}]{counts\+Map}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a5374e1e5b261ff43e6da700967631b83}
This function will filter the genes in the input Map by a number of constraints on the total number of reads on the genes
\begin{DoxyEnumerate}
\item All genes with expression $<$ 5.\+0 read counts will be removed
\item All genes with expression in the 75th quantile of the remaining expressed genes will be returned 
\begin{DoxyParams}{Parameters}
{\em counts\+Map} & A map of Ref\+Seq\+Gene to a List of counts along the length of the gene \\
\hline
\end{DoxyParams}
\begin{DoxyReturn}{Returns}

\end{DoxyReturn}

\end{DoxyEnumerate}\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a2fc579cf99a725d9637eaf4891a4fd11}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!filter\+Paths@{filter\+Paths}}
\index{filter\+Paths@{filter\+Paths}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{filter\+Paths}]{\setlength{\rightskip}{0pt plus 5cm}static Collection$<$Gene$>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+filter\+Paths (
\begin{DoxyParamCaption}
\item[{Map$<$ {\bf Path}, double\mbox{[}$\,$\mbox{]}$>$}]{scores, }
\item[{double}]{alpha2}
\end{DoxyParamCaption}
)\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a2fc579cf99a725d9637eaf4891a4fd11}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ac1217728072eb3635f3e7dea450af701}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Alignment\+Data\+Model\+Stats@{get\+Alignment\+Data\+Model\+Stats}}
\index{get\+Alignment\+Data\+Model\+Stats@{get\+Alignment\+Data\+Model\+Stats}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Alignment\+Data\+Model\+Stats}]{\setlength{\rightskip}{0pt plus 5cm}{\bf Alignment\+Data\+Model\+Stats} broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Alignment\+Data\+Model\+Stats (
\begin{DoxyParamCaption}
{}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ac1217728072eb3635f3e7dea450af701}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ad2358621505b89beef76afb69d366021}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Bases\+Covered@{get\+Bases\+Covered}}
\index{get\+Bases\+Covered@{get\+Bases\+Covered}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Bases\+Covered}]{\setlength{\rightskip}{0pt plus 5cm}int broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Bases\+Covered (
\begin{DoxyParamCaption}
\item[{Annotation}]{region}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ad2358621505b89beef76afb69d366021}


Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_aa510bb71e2a3dc533af6f6188a6fa6af}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Bases\+Covered@{get\+Bases\+Covered}}
\index{get\+Bases\+Covered@{get\+Bases\+Covered}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Bases\+Covered}]{\setlength{\rightskip}{0pt plus 5cm}int broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Bases\+Covered (
\begin{DoxyParamCaption}
\item[{Annotation}]{region, }
\item[{int}]{E\+F}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_aa510bb71e2a3dc533af6f6188a6fa6af}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_aadb1c1dfc12a73268e0552fb99e01024}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Chromosome\+Length@{get\+Chromosome\+Length}}
\index{get\+Chromosome\+Length@{get\+Chromosome\+Length}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Chromosome\+Length}]{\setlength{\rightskip}{0pt plus 5cm}int broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Chromosome\+Length (
\begin{DoxyParamCaption}
\item[{String}]{chr}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_aadb1c1dfc12a73268e0552fb99e01024}


Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_af1b3646c528368d2b2af71fb820350a6}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Chromosome\+Lengths@{get\+Chromosome\+Lengths}}
\index{get\+Chromosome\+Lengths@{get\+Chromosome\+Lengths}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Chromosome\+Lengths}]{\setlength{\rightskip}{0pt plus 5cm}Map$<$String, Integer$>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Chromosome\+Lengths (
\begin{DoxyParamCaption}
{}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_af1b3646c528368d2b2af71fb820350a6}


Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a8e4c4c86479ee745866baef364fdd6fe}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Count@{get\+Count}}
\index{get\+Count@{get\+Count}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Count}]{\setlength{\rightskip}{0pt plus 5cm}int broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Count (
\begin{DoxyParamCaption}
\item[{Collection$<$ Annotation $>$}]{Annotation}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a8e4c4c86479ee745866baef364fdd6fe}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a24fc6c0cea13276596ad51dbab778d38}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Counts@{get\+Counts}}
\index{get\+Counts@{get\+Counts}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Counts}]{\setlength{\rightskip}{0pt plus 5cm}int broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Counts (
\begin{DoxyParamCaption}
\item[{Map$<$ String, Interval\+Tree$<$ Annotation $>$$>$}]{gene\+Tree, }
\item[{Collection$<$ String $>$}]{multimappers}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a24fc6c0cea13276596ad51dbab778d38}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_acd9c8501fd75af7cb102cd6886cbe15b}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Counts\+Per\+Bp@{get\+Counts\+Per\+Bp}}
\index{get\+Counts\+Per\+Bp@{get\+Counts\+Per\+Bp}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Counts\+Per\+Bp}]{\setlength{\rightskip}{0pt plus 5cm}double \mbox{[}$\,$\mbox{]} broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Counts\+Per\+Bp (
\begin{DoxyParamCaption}
\item[{Annotation}]{align, }
\item[{Interval\+Tree$<$ Alignment $>$}]{tree}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_acd9c8501fd75af7cb102cd6886cbe15b}


Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a2f5c2e47fe3a39cb416bfddf280e2c7f}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Counts\+Per\+Bp@{get\+Counts\+Per\+Bp}}
\index{get\+Counts\+Per\+Bp@{get\+Counts\+Per\+Bp}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Counts\+Per\+Bp}]{\setlength{\rightskip}{0pt plus 5cm}List$<$Double$>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Counts\+Per\+Bp (
\begin{DoxyParamCaption}
\item[{Gene}]{gene, }
\item[{Interval\+Tree$<$ Alignment $>$}]{tree}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a2f5c2e47fe3a39cb416bfddf280e2c7f}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a5f78dee2ade4678e5ac2d04576c83b6d}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Data@{get\+Data}}
\index{get\+Data@{get\+Data}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Data}]{\setlength{\rightskip}{0pt plus 5cm}{\bf Alignment\+Data\+Model\+Stats} broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Data (
\begin{DoxyParamCaption}
{}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a5f78dee2ade4678e5ac2d04576c83b6d}
Get the underlying \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} object \begin{DoxyReturn}{Returns}
the \hyperlink{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}{Alignment\+Data\+Model\+Stats} object 
\end{DoxyReturn}


Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_af7d90fa931951069416e3ad235e99f37}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Data\+For\+Alignment@{get\+Data\+For\+Alignment}}
\index{get\+Data\+For\+Alignment@{get\+Data\+For\+Alignment}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Data\+For\+Alignment}]{\setlength{\rightskip}{0pt plus 5cm}double \mbox{[}$\,$\mbox{]} broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Data\+For\+Alignment (
\begin{DoxyParamCaption}
\item[{Annotation}]{align}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_af7d90fa931951069416e3ad235e99f37}
This function returns an array of type double with counts for each base pair across the genomic region of the gene \begin{DoxyAuthor}{Author}
skadri 
\end{DoxyAuthor}

\begin{DoxyParams}{Parameters}
{\em align} & \\
\hline
\end{DoxyParams}
\begin{DoxyReturn}{Returns}

\end{DoxyReturn}

\begin{DoxyExceptions}{Exceptions}
{\em I\+O\+Exception} & \\
\hline
\end{DoxyExceptions}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a00cf5fa4275d363fa77bf05454e7909c}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Data\+For\+Gene@{get\+Data\+For\+Gene}}
\index{get\+Data\+For\+Gene@{get\+Data\+For\+Gene}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Data\+For\+Gene}]{\setlength{\rightskip}{0pt plus 5cm}List$<$Double$>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Data\+For\+Gene (
\begin{DoxyParamCaption}
\item[{Gene}]{gene}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a00cf5fa4275d363fa77bf05454e7909c}


Here is the call graph for this function\+:




Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a0a1e9f474a59ee99a9c548092d35bf5b}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Data\+For\+Gene@{get\+Data\+For\+Gene}}
\index{get\+Data\+For\+Gene@{get\+Data\+For\+Gene}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Data\+For\+Gene}]{\setlength{\rightskip}{0pt plus 5cm}Map$<$Gene, List$<$Double$>$ $>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Data\+For\+Gene (
\begin{DoxyParamCaption}
\item[{Collection$<$ Gene $>$}]{Annotation}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a0a1e9f474a59ee99a9c548092d35bf5b}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_aaee0b9274c46753d21f44590c58c072c}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Gene\+Counts@{get\+Gene\+Counts}}
\index{get\+Gene\+Counts@{get\+Gene\+Counts}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Gene\+Counts}]{\setlength{\rightskip}{0pt plus 5cm}int \mbox{[}$\,$\mbox{]} broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Gene\+Counts (
\begin{DoxyParamCaption}
\item[{Collection$<$ Gene $>$}]{genes}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_aaee0b9274c46753d21f44590c58c072c}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a76ef156f96e86fe1eea8b45e94d22f02}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Gene\+Counts\+Without\+Dataset\+Totals@{get\+Gene\+Counts\+Without\+Dataset\+Totals}}
\index{get\+Gene\+Counts\+Without\+Dataset\+Totals@{get\+Gene\+Counts\+Without\+Dataset\+Totals}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Gene\+Counts\+Without\+Dataset\+Totals}]{\setlength{\rightskip}{0pt plus 5cm}int \mbox{[}$\,$\mbox{]} broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Gene\+Counts\+Without\+Dataset\+Totals (
\begin{DoxyParamCaption}
\item[{Collection$<$ Gene $>$}]{genes}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a76ef156f96e86fe1eea8b45e94d22f02}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_aef09a54b699b7e8130f80515431e250c}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Gene\+Expression@{get\+Gene\+Expression}}
\index{get\+Gene\+Expression@{get\+Gene\+Expression}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Gene\+Expression}]{\setlength{\rightskip}{0pt plus 5cm}Map$<$Gene, double\mbox{[}$\,$\mbox{]}$>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Gene\+Expression (
\begin{DoxyParamCaption}
\item[{Collection$<$ Gene $>$}]{genes}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_aef09a54b699b7e8130f80515431e250c}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a16fcd43426d01ba29a6e50d8a36ab38f}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Lambda@{get\+Lambda}}
\index{get\+Lambda@{get\+Lambda}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Lambda}]{\setlength{\rightskip}{0pt plus 5cm}double broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Lambda (
\begin{DoxyParamCaption}
\item[{String}]{chr}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a16fcd43426d01ba29a6e50d8a36ab38f}
Get an estimate of the expected read coverage per base 
\begin{DoxyParams}{Parameters}
{\em chr} & chromosome to retrieve \\
\hline
\end{DoxyParams}
\begin{DoxyReturn}{Returns}
lambda 
\end{DoxyReturn}

\begin{DoxyExceptions}{Exceptions}
{\em I\+O\+Exception} & \\
\hline
\end{DoxyExceptions}


Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_adba4f6a27c198a96fcfa79a643fca0a3}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Number\+Markers@{get\+Number\+Markers}}
\index{get\+Number\+Markers@{get\+Number\+Markers}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Number\+Markers}]{\setlength{\rightskip}{0pt plus 5cm}double broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Number\+Markers (
\begin{DoxyParamCaption}
\item[{String}]{chr}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_adba4f6a27c198a96fcfa79a643fca0a3}
Gets the number of allowable bases per chr 
\begin{DoxyParams}{Parameters}
{\em chr} & chromosome to retrieve \\
\hline
\end{DoxyParams}
\begin{DoxyReturn}{Returns}
total number of allowable bases 
\end{DoxyReturn}


Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a5b689c01b77461487789d172e013a3f6}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Number\+Of\+B\+Ps@{get\+Number\+Of\+B\+Ps}}
\index{get\+Number\+Of\+B\+Ps@{get\+Number\+Of\+B\+Ps}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Number\+Of\+B\+Ps}]{\setlength{\rightskip}{0pt plus 5cm}double broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Number\+Of\+B\+Ps (
\begin{DoxyParamCaption}
{}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a5b689c01b77461487789d172e013a3f6}
Get total number of bases \begin{DoxyReturn}{Returns}
total number of bases 
\end{DoxyReturn}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_affc935b97a13f0fc855cd5c1878dc8b2}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Number\+Of\+Reads@{get\+Number\+Of\+Reads}}
\index{get\+Number\+Of\+Reads@{get\+Number\+Of\+Reads}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Number\+Of\+Reads}]{\setlength{\rightskip}{0pt plus 5cm}double broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Number\+Of\+Reads (
\begin{DoxyParamCaption}
\item[{String}]{chr}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_affc935b97a13f0fc855cd5c1878dc8b2}
Gets the total number of reads per chr 
\begin{DoxyParams}{Parameters}
{\em chr} & chromosome to retrieve \\
\hline
\end{DoxyParams}
\begin{DoxyReturn}{Returns}
total number of reads 
\end{DoxyReturn}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a1db375ffca9153b8143df70c0bece395}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Paired\+Data@{get\+Paired\+Data}}
\index{get\+Paired\+Data@{get\+Paired\+Data}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Paired\+Data}]{\setlength{\rightskip}{0pt plus 5cm}{\bf Alignment\+Data\+Model\+Stats} broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Paired\+Data (
\begin{DoxyParamCaption}
{}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a1db375ffca9153b8143df70c0bece395}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a3d06acc7e76ca7cbf52b2e6f41ad2bb1}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Peak@{get\+Peak}}
\index{get\+Peak@{get\+Peak}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Peak}]{\setlength{\rightskip}{0pt plus 5cm}{\bf Gene\+Score} broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Peak (
\begin{DoxyParamCaption}
\item[{Gene}]{gene, }
\item[{Gene}]{start\+Codon, }
\item[{Interval\+Tree$<$ Alignment $>$}]{tree}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a3d06acc7e76ca7cbf52b2e6f41ad2bb1}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a067da57de440289d5bc6d7a4dbe5427d}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Peak@{get\+Peak}}
\index{get\+Peak@{get\+Peak}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Peak}]{\setlength{\rightskip}{0pt plus 5cm}Gene broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Peak (
\begin{DoxyParamCaption}
\item[{Gene}]{gene, }
\item[{Gene}]{start\+Codon}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a067da57de440289d5bc6d7a4dbe5427d}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a68033365b74f1d7755eb6e712f47f863}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Sum@{get\+Sum}}
\index{get\+Sum@{get\+Sum}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Sum}]{\setlength{\rightskip}{0pt plus 5cm}double broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Sum (
\begin{DoxyParamCaption}
\item[{String}]{chr}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a68033365b74f1d7755eb6e712f47f863}
Gets the total number of reads per chr 
\begin{DoxyParams}{Parameters}
{\em chr} & chromosome to retrieve \\
\hline
\end{DoxyParams}
\begin{DoxyReturn}{Returns}
total number of reads 
\end{DoxyReturn}


Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a87118dfc0d49b68daca8cd351ae3bcab}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!get\+Widths@{get\+Widths}}
\index{get\+Widths@{get\+Widths}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{get\+Widths}]{\setlength{\rightskip}{0pt plus 5cm}static int \mbox{[}$\,$\mbox{]} broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+get\+Widths (
\begin{DoxyParamCaption}
\item[{String}]{str}
\end{DoxyParamCaption}
)\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a87118dfc0d49b68daca8cd351ae3bcab}


Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a507f17b9636367cfc4dfa6e8bf25e7f6}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!has\+Data\+For\+Chromosome@{has\+Data\+For\+Chromosome}}
\index{has\+Data\+For\+Chromosome@{has\+Data\+For\+Chromosome}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{has\+Data\+For\+Chromosome}]{\setlength{\rightskip}{0pt plus 5cm}boolean broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+has\+Data\+For\+Chromosome (
\begin{DoxyParamCaption}
\item[{String}]{chr}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a507f17b9636367cfc4dfa6e8bf25e7f6}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a8777cdf9066d2f8f1cdd818d9e728d69}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!main@{main}}
\index{main@{main}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{main}]{\setlength{\rightskip}{0pt plus 5cm}static void broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+main (
\begin{DoxyParamCaption}
\item[{String\mbox{[}$\,$\mbox{]}}]{args}
\end{DoxyParamCaption}
) throws Exception\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a8777cdf9066d2f8f1cdd818d9e728d69}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a039655b2e536f7a9728f3697c14c4505}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!parse\+Mask\+Files@{parse\+Mask\+Files}}
\index{parse\+Mask\+Files@{parse\+Mask\+Files}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{parse\+Mask\+Files}]{\setlength{\rightskip}{0pt plus 5cm}static Map$<$String, Integer$>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+parse\+Mask\+Files (
\begin{DoxyParamCaption}
\item[{File\mbox{[}$\,$\mbox{]}}]{files}
\end{DoxyParamCaption}
) throws I\+O\+Exception\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a039655b2e536f7a9728f3697c14c4505}


Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a663b9546e782ac0dbded4f9d375dc8a3}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!permute\+Regions@{permute\+Regions}}
\index{permute\+Regions@{permute\+Regions}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{permute\+Regions}]{\setlength{\rightskip}{0pt plus 5cm}Collection$<$Annotation$>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+permute\+Regions (
\begin{DoxyParamCaption}
\item[{Annotation}]{align}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a663b9546e782ac0dbded4f9d375dc8a3}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a767afcf399cb81b942f8bab5bd87fdaa}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!raw\+Regions@{raw\+Regions}}
\index{raw\+Regions@{raw\+Regions}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{raw\+Regions}]{\setlength{\rightskip}{0pt plus 5cm}Collection$<$Annotation$>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+raw\+Regions (
\begin{DoxyParamCaption}
\item[{Annotation}]{align}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a767afcf399cb81b942f8bab5bd87fdaa}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a05b31a3d41397a1ff0e91b110e06e0ec}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!reset\+Tree\+Cache@{reset\+Tree\+Cache}}
\index{reset\+Tree\+Cache@{reset\+Tree\+Cache}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{reset\+Tree\+Cache}]{\setlength{\rightskip}{0pt plus 5cm}void broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+reset\+Tree\+Cache (
\begin{DoxyParamCaption}
{}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a05b31a3d41397a1ff0e91b110e06e0ec}


Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a9055da2c6c85869da10e9bff9c0fbac9}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!scan@{scan}}
\index{scan@{scan}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{scan}]{\setlength{\rightskip}{0pt plus 5cm}Collection$<$Annotation$>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+scan (
\begin{DoxyParamCaption}
\item[{int\mbox{[}$\,$\mbox{]}}]{window\+Sizes, }
\item[{double}]{alpha}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a9055da2c6c85869da10e9bff9c0fbac9}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a854b44de12136762faa57412f7b47465}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!scan@{scan}}
\index{scan@{scan}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{scan}]{\setlength{\rightskip}{0pt plus 5cm}Collection$<$Annotation$>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+scan (
\begin{DoxyParamCaption}
\item[{int\mbox{[}$\,$\mbox{]}}]{window\+Sizes, }
\item[{double}]{alpha, }
\item[{String}]{chr}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a854b44de12136762faa57412f7b47465}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a783abf082b1ebce3361a143d40764b04}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!scan\+Genome@{scan\+Genome}}
\index{scan\+Genome@{scan\+Genome}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{scan\+Genome}]{\setlength{\rightskip}{0pt plus 5cm}void broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+scan\+Genome (
\begin{DoxyParamCaption}
\item[{{\bf Continuous\+Data\+Alignment\+Model}}]{data2, }
\item[{int}]{window\+Size, }
\item[{String}]{chr, }
\item[{File\+Writer}]{writer, }
\item[{boolean}]{filter\+Significance}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a783abf082b1ebce3361a143d40764b04}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_acde220ce2b79ad3a0cb0f0493eb52a2f}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!scan\+P\+Rate@{scan\+P\+Rate}}
\index{scan\+P\+Rate@{scan\+P\+Rate}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{scan\+P\+Rate}]{\setlength{\rightskip}{0pt plus 5cm}double \mbox{[}$\,$\mbox{]} broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+scan\+P\+Rate (
\begin{DoxyParamCaption}
\item[{Gene}]{gene}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_acde220ce2b79ad3a0cb0f0493eb52a2f}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ac64273677f9810b6e1a3598e1678b343}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!scan\+P\+Rate@{scan\+P\+Rate}}
\index{scan\+P\+Rate@{scan\+P\+Rate}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{scan\+P\+Rate}]{\setlength{\rightskip}{0pt plus 5cm}double \mbox{[}$\,$\mbox{]} broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+scan\+P\+Rate (
\begin{DoxyParamCaption}
\item[{Gene}]{gene, }
\item[{Interval\+Tree$<$ Alignment $>$}]{tree}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ac64273677f9810b6e1a3598e1678b343}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a2e6ae03ce97022cd39bfc2102c0580bd}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!scan\+P\+Rate@{scan\+P\+Rate}}
\index{scan\+P\+Rate@{scan\+P\+Rate}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{scan\+P\+Rate}]{\setlength{\rightskip}{0pt plus 5cm}double \mbox{[}$\,$\mbox{]} broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+scan\+P\+Rate (
\begin{DoxyParamCaption}
\item[{Gene}]{gene, }
\item[{Interval\+Tree$<$ Alignment $>$}]{tree, }
\item[{double}]{local\+Lambda}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a2e6ae03ce97022cd39bfc2102c0580bd}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a3e7ef2b826b64f787457935fdb71e558}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!scan\+P\+Rate@{scan\+P\+Rate}}
\index{scan\+P\+Rate@{scan\+P\+Rate}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{scan\+P\+Rate}]{\setlength{\rightskip}{0pt plus 5cm}double \mbox{[}$\,$\mbox{]} broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+scan\+P\+Rate (
\begin{DoxyParamCaption}
\item[{Gene}]{gene, }
\item[{Interval\+Tree$<$ Alignment $>$}]{tree, }
\item[{double}]{local\+Lambda, }
\item[{int}]{num\+Markers}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a3e7ef2b826b64f787457935fdb71e558}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_afdb0844bc8468e30a3a2b2cd55a13e62}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!scan\+Region@{scan\+Region}}
\index{scan\+Region@{scan\+Region}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{scan\+Region}]{\setlength{\rightskip}{0pt plus 5cm}double \mbox{[}$\,$\mbox{]} broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+scan\+Region (
\begin{DoxyParamCaption}
\item[{int}]{fixed\+Width, }
\item[{{\bf Lightweight\+Genomic\+Annotation}}]{region}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_afdb0844bc8468e30a3a2b2cd55a13e62}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a11cd7ab6673b37e19f9cfe066130ba20}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!score\+Gene@{score\+Gene}}
\index{score\+Gene@{score\+Gene}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{score\+Gene}]{\setlength{\rightskip}{0pt plus 5cm}double \mbox{[}$\,$\mbox{]} broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+score\+Gene (
\begin{DoxyParamCaption}
\item[{Gene}]{align}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a11cd7ab6673b37e19f9cfe066130ba20}

\begin{DoxyParams}{Parameters}
{\em align} & \\
\hline
\end{DoxyParams}
\begin{DoxyReturn}{Returns}
\{calculate\+P\+Val(new Double(sum).int\+Value(), local\+Lambda, gene\+Length, get\+Number\+Markers(chr)), enrich, sum, avg\+Coverage, rpkm, local\+Lambda, gene\+Length,nominal\+P, fully\+Contained\} 
\end{DoxyReturn}

\begin{DoxyExceptions}{Exceptions}
{\em I\+O\+Exception} & \\
\hline
\end{DoxyExceptions}


Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a3fedd5b944ae861cb5006ed403af033e}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!score\+Gene@{score\+Gene}}
\index{score\+Gene@{score\+Gene}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{score\+Gene}]{\setlength{\rightskip}{0pt plus 5cm}double \mbox{[}$\,$\mbox{]} broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+score\+Gene (
\begin{DoxyParamCaption}
\item[{Gene}]{align, }
\item[{double}]{local\+Lambda}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a3fedd5b944ae861cb5006ed403af033e}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a0d920606dbcf6437d4e4cf8f3e415156}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!score\+Gene@{score\+Gene}}
\index{score\+Gene@{score\+Gene}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{score\+Gene}]{\setlength{\rightskip}{0pt plus 5cm}{\bf Gene\+Score} broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+score\+Gene (
\begin{DoxyParamCaption}
\item[{Gene}]{align, }
\item[{Interval\+Tree$<$ Alignment $>$}]{tree, }
\item[{double}]{local\+Lambda}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a0d920606dbcf6437d4e4cf8f3e415156}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a9bdf34088aadf81c3eabae435fe67767}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!score\+Gene@{score\+Gene}}
\index{score\+Gene@{score\+Gene}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{score\+Gene}]{\setlength{\rightskip}{0pt plus 5cm}{\bf Gene\+Score} broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+score\+Gene (
\begin{DoxyParamCaption}
\item[{Gene}]{align, }
\item[{Interval\+Tree$<$ Alignment $>$}]{tree, }
\item[{double}]{local\+Lambda, }
\item[{int}]{num\+Markers}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a9bdf34088aadf81c3eabae435fe67767}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a775a3a33946ec4040d85a2b3ba738ab9}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!score\+Gene@{score\+Gene}}
\index{score\+Gene@{score\+Gene}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{score\+Gene}]{\setlength{\rightskip}{0pt plus 5cm}double \mbox{[}$\,$\mbox{]} broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+score\+Gene (
\begin{DoxyParamCaption}
\item[{Gene}]{align, }
\item[{Interval\+Tree$<$ Alignment $>$}]{tree}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a775a3a33946ec4040d85a2b3ba738ab9}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_af1e5000926013a041e9597fb3a2dc5fe}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!score\+Genes@{score\+Genes}}
\index{score\+Genes@{score\+Genes}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{score\+Genes}]{\setlength{\rightskip}{0pt plus 5cm}Map$<$Gene, double\mbox{[}$\,$\mbox{]}$>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+score\+Genes (
\begin{DoxyParamCaption}
\item[{Collection$<$ Gene $>$}]{set, }
\item[{String}]{chr\+To\+Use, }
\item[{Interval\+Tree$<$ Alignment $>$}]{tree}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_af1e5000926013a041e9597fb3a2dc5fe}


Here is the call graph for this function\+:




Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_affe8308674627334d5a187dde355b468}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!score\+Genes@{score\+Genes}}
\index{score\+Genes@{score\+Genes}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{score\+Genes}]{\setlength{\rightskip}{0pt plus 5cm}Map$<$Gene, double\mbox{[}$\,$\mbox{]}$>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+score\+Genes (
\begin{DoxyParamCaption}
\item[{Collection$<$ Gene $>$}]{genes}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_affe8308674627334d5a187dde355b468}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a37b049c25b8723b27209f0cac8a9ab0f}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!score\+Genes@{score\+Genes}}
\index{score\+Genes@{score\+Genes}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{score\+Genes}]{\setlength{\rightskip}{0pt plus 5cm}Map$<$Gene, double\mbox{[}$\,$\mbox{]}$>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+score\+Genes (
\begin{DoxyParamCaption}
\item[{Collection$<$ Gene $>$}]{set, }
\item[{String}]{chr\+To\+Use}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a37b049c25b8723b27209f0cac8a9ab0f}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_aedf7ee948f8f74ee7e3f8c926bd77444}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!score\+Genes@{score\+Genes}}
\index{score\+Genes@{score\+Genes}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{score\+Genes}]{\setlength{\rightskip}{0pt plus 5cm}Map$<$Gene, double\mbox{[}$\,$\mbox{]}$>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+score\+Genes (
\begin{DoxyParamCaption}
\item[{Collection$<$ Gene $>$}]{set, }
\item[{String}]{chr\+To\+Use, }
\item[{int}]{size}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_aedf7ee948f8f74ee7e3f8c926bd77444}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a8fe3ed4e57d504db94f449a29385b098}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!score\+Genes@{score\+Genes}}
\index{score\+Genes@{score\+Genes}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{score\+Genes}]{\setlength{\rightskip}{0pt plus 5cm}void broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+score\+Genes (
\begin{DoxyParamCaption}
\item[{{\bf B\+E\+D\+File\+Parser}}]{genes}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a8fe3ed4e57d504db94f449a29385b098}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a9e84cfdbc07c6b49f3e35b092726714f}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!score\+Paths@{score\+Paths}}
\index{score\+Paths@{score\+Paths}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{score\+Paths}]{\setlength{\rightskip}{0pt plus 5cm}Map$<${\bf Path}, double\mbox{[}$\,$\mbox{]}$>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+score\+Paths (
\begin{DoxyParamCaption}
\item[{Collection$<$ {\bf Path} $>$}]{paths, }
\item[{double}]{lambda}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a9e84cfdbc07c6b49f3e35b092726714f}


Here is the call graph for this function\+:




Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a1353042a53391b6ba89d333fdd5fab11}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!score\+Paths@{score\+Paths}}
\index{score\+Paths@{score\+Paths}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{score\+Paths}]{\setlength{\rightskip}{0pt plus 5cm}static Map$<${\bf Path}, double\mbox{[}$\,$\mbox{]}$>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+score\+Paths (
\begin{DoxyParamCaption}
\item[{Collection$<$ {\bf Path} $>$}]{paths, }
\item[{{\bf Chromosome\+With\+Bubbles\+J\+Graph\+T}}]{graph, }
\item[{double}]{alpha}
\end{DoxyParamCaption}
)\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a1353042a53391b6ba89d333fdd5fab11}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a8c3eca6b8e9d8db29f3e0a8a7f25207b}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!score\+Paths@{score\+Paths}}
\index{score\+Paths@{score\+Paths}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{score\+Paths}]{\setlength{\rightskip}{0pt plus 5cm}static Map$<${\bf Path}, double\mbox{[}$\,$\mbox{]}$>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+score\+Paths (
\begin{DoxyParamCaption}
\item[{Collection$<$ {\bf Path} $>$}]{paths, }
\item[{double}]{lambda, }
\item[{double}]{num\+Markers, }
\item[{double}]{num\+Reads, }
\item[{double}]{R\+P\+K\+M\+Contant, }
\item[{Map$<$ Annotation, Double $>$}]{local\+Rates, }
\item[{double}]{alpha}
\end{DoxyParamCaption}
)\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a8c3eca6b8e9d8db29f3e0a8a7f25207b}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_af620a01c0d3be891bdd277470553e666}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!score\+Paths\+For\+Only\+R\+P\+K\+M@{score\+Paths\+For\+Only\+R\+P\+K\+M}}
\index{score\+Paths\+For\+Only\+R\+P\+K\+M@{score\+Paths\+For\+Only\+R\+P\+K\+M}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{score\+Paths\+For\+Only\+R\+P\+K\+M}]{\setlength{\rightskip}{0pt plus 5cm}Map$<${\bf Path}, double\mbox{[}$\,$\mbox{]}$>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+score\+Paths\+For\+Only\+R\+P\+K\+M (
\begin{DoxyParamCaption}
\item[{Collection$<$ {\bf Path} $>$}]{paths}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_af620a01c0d3be891bdd277470553e666}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a8cffd238bd6070c45d5bd0bb72b43aeb}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!score\+Segment@{score\+Segment}}
\index{score\+Segment@{score\+Segment}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{score\+Segment}]{\setlength{\rightskip}{0pt plus 5cm}double \mbox{[}$\,$\mbox{]} broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+score\+Segment (
\begin{DoxyParamCaption}
\item[{Annotation}]{align}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a8cffd238bd6070c45d5bd0bb72b43aeb}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a7009b0bc1a05ca28f02a06831778168b}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!score\+Segments@{score\+Segments}}
\index{score\+Segments@{score\+Segments}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{score\+Segments}]{\setlength{\rightskip}{0pt plus 5cm}Map$<$Annotation, double\mbox{[}$\,$\mbox{]}$>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+score\+Segments (
\begin{DoxyParamCaption}
\item[{Collection$<$ Annotation $>$}]{set}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a7009b0bc1a05ca28f02a06831778168b}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a594e568f254f787301c484cb79d443cc}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!score\+Segments@{score\+Segments}}
\index{score\+Segments@{score\+Segments}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{score\+Segments}]{\setlength{\rightskip}{0pt plus 5cm}Map$<$Annotation, double\mbox{[}$\,$\mbox{]}$>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+score\+Segments (
\begin{DoxyParamCaption}
\item[{Collection$<$ Annotation $>$}]{set, }
\item[{String}]{chr\+To\+Use}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a594e568f254f787301c484cb79d443cc}


Here is the call graph for this function\+:




Here is the caller graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a562be161ea6de2e8a1b104bdbd6d449b}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!score\+Segments@{score\+Segments}}
\index{score\+Segments@{score\+Segments}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{score\+Segments}]{\setlength{\rightskip}{0pt plus 5cm}Map$<$Annotation, double\mbox{[}$\,$\mbox{]}$>$ broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+score\+Segments (
\begin{DoxyParamCaption}
\item[{Collection$<$ Annotation $>$}]{set, }
\item[{String}]{chr\+To\+Use, }
\item[{boolean}]{ignore\+Alpha}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a562be161ea6de2e8a1b104bdbd6d449b}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a8c1d4aef55169c5f5cc897edec41c1ff}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!set\+Alpha@{set\+Alpha}}
\index{set\+Alpha@{set\+Alpha}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{set\+Alpha}]{\setlength{\rightskip}{0pt plus 5cm}void broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+set\+Alpha (
\begin{DoxyParamCaption}
\item[{double}]{alpha}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a8c1d4aef55169c5f5cc897edec41c1ff}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a1e3aa5641fa5aa9963d9a3b05870012a}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!set\+Extension\+Factor@{set\+Extension\+Factor}}
\index{set\+Extension\+Factor@{set\+Extension\+Factor}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{set\+Extension\+Factor}]{\setlength{\rightskip}{0pt plus 5cm}void broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+set\+Extension\+Factor (
\begin{DoxyParamCaption}
\item[{int}]{extension\+Factor}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a1e3aa5641fa5aa9963d9a3b05870012a}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ae9180b76d6c1bc1ea8d6024a2b08c5db}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!set\+Find\+Max\+Contiguous@{set\+Find\+Max\+Contiguous}}
\index{set\+Find\+Max\+Contiguous@{set\+Find\+Max\+Contiguous}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{set\+Find\+Max\+Contiguous}]{\setlength{\rightskip}{0pt plus 5cm}void broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+set\+Find\+Max\+Contiguous (
\begin{DoxyParamCaption}
\item[{boolean}]{find\+Max\+Contiguous}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ae9180b76d6c1bc1ea8d6024a2b08c5db}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ac019bf49be802c3b62d3a3055104c365}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!set\+Mask\+File\+Data@{set\+Mask\+File\+Data}}
\index{set\+Mask\+File\+Data@{set\+Mask\+File\+Data}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{set\+Mask\+File\+Data}]{\setlength{\rightskip}{0pt plus 5cm}void broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+set\+Mask\+File\+Data (
\begin{DoxyParamCaption}
\item[{Map$<$ String, Integer $>$}]{masked\+Region\+Map}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ac019bf49be802c3b62d3a3055104c365}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ab7ab620d4f4c1b25acc716d83a8d72b6}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!set\+Min\+Contguous\+Segment\+Size@{set\+Min\+Contguous\+Segment\+Size}}
\index{set\+Min\+Contguous\+Segment\+Size@{set\+Min\+Contguous\+Segment\+Size}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{set\+Min\+Contguous\+Segment\+Size}]{\setlength{\rightskip}{0pt plus 5cm}void broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+set\+Min\+Contguous\+Segment\+Size (
\begin{DoxyParamCaption}
\item[{int}]{min\+Size}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ab7ab620d4f4c1b25acc716d83a8d72b6}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a13f7d8efa23552e98f798e9deec570de}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!set\+Minimum\+Mapping\+Quality@{set\+Minimum\+Mapping\+Quality}}
\index{set\+Minimum\+Mapping\+Quality@{set\+Minimum\+Mapping\+Quality}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{set\+Minimum\+Mapping\+Quality}]{\setlength{\rightskip}{0pt plus 5cm}void broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+set\+Minimum\+Mapping\+Quality (
\begin{DoxyParamCaption}
\item[{double}]{min\+Map\+Qual}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a13f7d8efa23552e98f798e9deec570de}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a61ed951795b265ba619f938e715ca1ab}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!set\+Min\+Number\+Of\+Splices@{set\+Min\+Number\+Of\+Splices}}
\index{set\+Min\+Number\+Of\+Splices@{set\+Min\+Number\+Of\+Splices}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{set\+Min\+Number\+Of\+Splices}]{\setlength{\rightskip}{0pt plus 5cm}void broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+set\+Min\+Number\+Of\+Splices (
\begin{DoxyParamCaption}
\item[{int}]{min\+Splice\+Support}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a61ed951795b265ba619f938e715ca1ab}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_aa17cb50bb398dc7770859377e8d9771e}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!set\+Trim\+Ends@{set\+Trim\+Ends}}
\index{set\+Trim\+Ends@{set\+Trim\+Ends}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{set\+Trim\+Ends}]{\setlength{\rightskip}{0pt plus 5cm}void broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+set\+Trim\+Ends (
\begin{DoxyParamCaption}
\item[{boolean}]{trim\+Ends}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_aa17cb50bb398dc7770859377e8d9771e}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a05c9b50b32fd0cdfee049966ae7b9c1e}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!set\+Trim\+Quantile@{set\+Trim\+Quantile}}
\index{set\+Trim\+Quantile@{set\+Trim\+Quantile}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{set\+Trim\+Quantile}]{\setlength{\rightskip}{0pt plus 5cm}void broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+set\+Trim\+Quantile (
\begin{DoxyParamCaption}
\item[{double}]{quantile}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a05c9b50b32fd0cdfee049966ae7b9c1e}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_af559e50d0cbcc7525afd106c4bb93418}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!set\+Up\+Weight\+Splices@{set\+Up\+Weight\+Splices}}
\index{set\+Up\+Weight\+Splices@{set\+Up\+Weight\+Splices}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{set\+Up\+Weight\+Splices}]{\setlength{\rightskip}{0pt plus 5cm}void broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+set\+Up\+Weight\+Splices (
\begin{DoxyParamCaption}
\item[{boolean}]{set\+Flag}
\end{DoxyParamCaption}
)}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_af559e50d0cbcc7525afd106c4bb93418}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a1c47ee216d5f3910c63d01275e0a0634}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!trim\+Gene\+Ends@{trim\+Gene\+Ends}}
\index{trim\+Gene\+Ends@{trim\+Gene\+Ends}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{trim\+Gene\+Ends}]{\setlength{\rightskip}{0pt plus 5cm}Gene broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+trim\+Gene\+Ends (
\begin{DoxyParamCaption}
\item[{Gene}]{gene, }
\item[{double}]{quantile}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a1c47ee216d5f3910c63d01275e0a0634}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a859581141e70dd3ad658aa3fad127d32}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!trim\+Max\+Contiguous@{trim\+Max\+Contiguous}}
\index{trim\+Max\+Contiguous@{trim\+Max\+Contiguous}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{trim\+Max\+Contiguous}]{\setlength{\rightskip}{0pt plus 5cm}Gene broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+trim\+Max\+Contiguous (
\begin{DoxyParamCaption}
\item[{Gene}]{region, }
\item[{double}]{quantile}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a859581141e70dd3ad658aa3fad127d32}
Trim a region to a contiguous subregion with max number of positions having coverage above the quantile 
\begin{DoxyParams}{Parameters}
{\em region} & The original region \\
\hline
{\em quantile} & The coverage quantile of the original region \\
\hline
\end{DoxyParams}
\begin{DoxyReturn}{Returns}
The contiguous subregion with maximal number of positions having coverage above the quantile of the original region or null if can't trim 
\end{DoxyReturn}

\begin{DoxyExceptions}{Exceptions}
{\em I\+O\+Exception} & \\
\hline
\end{DoxyExceptions}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a1b6a5a28227911fd8ce4552dee2ccd36}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!write\+Connections@{write\+Connections}}
\index{write\+Connections@{write\+Connections}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{write\+Connections}]{\setlength{\rightskip}{0pt plus 5cm}void broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+write\+Connections (
\begin{DoxyParamCaption}
\item[{String}]{save, }
\item[{{\bf Chromosome\+With\+Bubbles\+J\+Graph\+T}}]{graph}
\end{DoxyParamCaption}
) throws I\+O\+Exception}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a1b6a5a28227911fd8ce4552dee2ccd36}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ab9f9fcd2fb7cfe1b353af62698759f73}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!write\+Full\+B\+E\+D@{write\+Full\+B\+E\+D}}
\index{write\+Full\+B\+E\+D@{write\+Full\+B\+E\+D}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{write\+Full\+B\+E\+D}]{\setlength{\rightskip}{0pt plus 5cm}static void broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+write\+Full\+B\+E\+D (
\begin{DoxyParamCaption}
\item[{String}]{save, }
\item[{Map$<$ Gene, double\mbox{[}$\,$\mbox{]}$>$}]{map, }
\item[{double}]{alpha}
\end{DoxyParamCaption}
) throws I\+O\+Exception\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ab9f9fcd2fb7cfe1b353af62698759f73}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a2e67d6ebf44bb95cdb4548a6f75d6c4a}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!write\+Full\+B\+E\+D@{write\+Full\+B\+E\+D}}
\index{write\+Full\+B\+E\+D@{write\+Full\+B\+E\+D}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{write\+Full\+B\+E\+D}]{\setlength{\rightskip}{0pt plus 5cm}static void broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+write\+Full\+B\+E\+D (
\begin{DoxyParamCaption}
\item[{String}]{save, }
\item[{Collection$<$ Gene $>$}]{segments}
\end{DoxyParamCaption}
) throws I\+O\+Exception\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a2e67d6ebf44bb95cdb4548a6f75d6c4a}


\subsection{Member Data Documentation}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a1c80e71adece28e35fee890de6ed2304}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!D\+E\+F\+A\+U\+L\+T\+\_\+\+I\+N\+S\+\_\+\+S\+I\+Z\+E\+\_\+\+P\+V\+A\+L@{D\+E\+F\+A\+U\+L\+T\+\_\+\+I\+N\+S\+\_\+\+S\+I\+Z\+E\+\_\+\+P\+V\+A\+L}}
\index{D\+E\+F\+A\+U\+L\+T\+\_\+\+I\+N\+S\+\_\+\+S\+I\+Z\+E\+\_\+\+P\+V\+A\+L@{D\+E\+F\+A\+U\+L\+T\+\_\+\+I\+N\+S\+\_\+\+S\+I\+Z\+E\+\_\+\+P\+V\+A\+L}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{D\+E\+F\+A\+U\+L\+T\+\_\+\+I\+N\+S\+\_\+\+S\+I\+Z\+E\+\_\+\+P\+V\+A\+L}]{\setlength{\rightskip}{0pt plus 5cm}double broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+D\+E\+F\+A\+U\+L\+T\+\_\+\+I\+N\+S\+\_\+\+S\+I\+Z\+E\+\_\+\+P\+V\+A\+L = 0.\+05\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_a1c80e71adece28e35fee890de6ed2304}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_afcf03553918b3c7cd3b2c7d183b35c97}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!D\+E\+F\+A\+U\+L\+T\+\_\+\+I\+N\+S\+E\+R\+T\+\_\+\+S\+I\+Z\+E\+\_\+\+F\+U\+D\+G\+E@{D\+E\+F\+A\+U\+L\+T\+\_\+\+I\+N\+S\+E\+R\+T\+\_\+\+S\+I\+Z\+E\+\_\+\+F\+U\+D\+G\+E}}
\index{D\+E\+F\+A\+U\+L\+T\+\_\+\+I\+N\+S\+E\+R\+T\+\_\+\+S\+I\+Z\+E\+\_\+\+F\+U\+D\+G\+E@{D\+E\+F\+A\+U\+L\+T\+\_\+\+I\+N\+S\+E\+R\+T\+\_\+\+S\+I\+Z\+E\+\_\+\+F\+U\+D\+G\+E}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{D\+E\+F\+A\+U\+L\+T\+\_\+\+I\+N\+S\+E\+R\+T\+\_\+\+S\+I\+Z\+E\+\_\+\+F\+U\+D\+G\+E}]{\setlength{\rightskip}{0pt plus 5cm}int broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+D\+E\+F\+A\+U\+L\+T\+\_\+\+I\+N\+S\+E\+R\+T\+\_\+\+S\+I\+Z\+E\+\_\+\+F\+U\+D\+G\+E = 20\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_afcf03553918b3c7cd3b2c7d183b35c97}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ad2e9aa31d5024257b5417f729ac5aa6e}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!D\+E\+F\+A\+U\+L\+T\+\_\+\+M\+A\+K\+E\+\_\+\+G\+E\+N\+E\+\_\+\+O\+V\+E\+R\+L\+A\+P@{D\+E\+F\+A\+U\+L\+T\+\_\+\+M\+A\+K\+E\+\_\+\+G\+E\+N\+E\+\_\+\+O\+V\+E\+R\+L\+A\+P}}
\index{D\+E\+F\+A\+U\+L\+T\+\_\+\+M\+A\+K\+E\+\_\+\+G\+E\+N\+E\+\_\+\+O\+V\+E\+R\+L\+A\+P@{D\+E\+F\+A\+U\+L\+T\+\_\+\+M\+A\+K\+E\+\_\+\+G\+E\+N\+E\+\_\+\+O\+V\+E\+R\+L\+A\+P}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{D\+E\+F\+A\+U\+L\+T\+\_\+\+M\+A\+K\+E\+\_\+\+G\+E\+N\+E\+\_\+\+O\+V\+E\+R\+L\+A\+P}]{\setlength{\rightskip}{0pt plus 5cm}double broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+D\+E\+F\+A\+U\+L\+T\+\_\+\+M\+A\+K\+E\+\_\+\+G\+E\+N\+E\+\_\+\+O\+V\+E\+R\+L\+A\+P = 0.\+3\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ad2e9aa31d5024257b5417f729ac5aa6e}
\hypertarget{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ada8b56f8e48ecc0df55f92f0a4e3a537}{\index{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}!D\+E\+F\+A\+U\+L\+T\+\_\+\+M\+I\+N\+\_\+\+M\+A\+P\+P\+I\+N\+G\+\_\+\+Q\+U\+A\+L\+I\+T\+Y@{D\+E\+F\+A\+U\+L\+T\+\_\+\+M\+I\+N\+\_\+\+M\+A\+P\+P\+I\+N\+G\+\_\+\+Q\+U\+A\+L\+I\+T\+Y}}
\index{D\+E\+F\+A\+U\+L\+T\+\_\+\+M\+I\+N\+\_\+\+M\+A\+P\+P\+I\+N\+G\+\_\+\+Q\+U\+A\+L\+I\+T\+Y@{D\+E\+F\+A\+U\+L\+T\+\_\+\+M\+I\+N\+\_\+\+M\+A\+P\+P\+I\+N\+G\+\_\+\+Q\+U\+A\+L\+I\+T\+Y}!broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model@{broad\+::pda\+::seq\+::segmentation\+::\+Continuous\+Data\+Alignment\+Model}}
\subsubsection[{D\+E\+F\+A\+U\+L\+T\+\_\+\+M\+I\+N\+\_\+\+M\+A\+P\+P\+I\+N\+G\+\_\+\+Q\+U\+A\+L\+I\+T\+Y}]{\setlength{\rightskip}{0pt plus 5cm}int broad.\+pda.\+seq.\+segmentation.\+Continuous\+Data\+Alignment\+Model.\+D\+E\+F\+A\+U\+L\+T\+\_\+\+M\+I\+N\+\_\+\+M\+A\+P\+P\+I\+N\+G\+\_\+\+Q\+U\+A\+L\+I\+T\+Y = 5\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model_ada8b56f8e48ecc0df55f92f0a4e3a537}


The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
java/broad/pda/seq/segmentation/\hyperlink{_continuous_data_alignment_model_8java}{Continuous\+Data\+Alignment\+Model.\+java}\end{DoxyCompactItemize}
