\hypertarget{classbroad_1_1core_1_1motif_1_1_p_w_m_utils}{\section{broad.\+core.\+motif.\+P\+W\+M\+Utils Class Reference}
\label{classbroad_1_1core_1_1motif_1_1_p_w_m_utils}\index{broad.\+core.\+motif.\+P\+W\+M\+Utils@{broad.\+core.\+motif.\+P\+W\+M\+Utils}}
}


Collaboration diagram for broad.\+core.\+motif.\+P\+W\+M\+Utils\+:
\subsection*{Static Public Member Functions}
\begin{DoxyCompactItemize}
\item 
static void \hyperlink{classbroad_1_1core_1_1motif_1_1_p_w_m_utils_a17646e545879c7d2061fbc4cad0bf834}{main} (String\mbox{[}$\,$\mbox{]} args)  throws Illegal\+Argument\+Exception, Exception 
\item 
static double\mbox{[}$\,$\mbox{]} \hyperlink{classbroad_1_1core_1_1motif_1_1_p_w_m_utils_ade2a53589ad91434fd0a187a7befd8b4}{compute\+Sequence\+B\+G\+Sequences} (Map$<$ String, \hyperlink{classbroad_1_1core_1_1sequence_1_1_sequence}{Sequence} $>$ chrs\+By\+Name, Annotation\+Reader$<$?extends \hyperlink{interfacebroad_1_1core_1_1annotation_1_1_genomic_annotation}{Genomic\+Annotation} $>$ reader)  throws I\+O\+Exception 
\item 
static Map\\*
$<$ \hyperlink{classbroad_1_1core_1_1motif_1_1_position_weight_matrix}{Position\+Weight\+Matrix}, \\*
\hyperlink{interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation}{Lightweight\+Genomic\+Annotation} $>$ \hyperlink{classbroad_1_1core_1_1motif_1_1_p_w_m_utils_a901bd6475f321d8416ddbf47d03d32b4}{scan\+Region} (int shuffles, String prefix, double min\+Score\+To\+Write, boolean max\+Only, String outdir, List$<$ \hyperlink{classbroad_1_1core_1_1motif_1_1_position_weight_matrix}{Position\+Weight\+Matrix} $>$ pwms, String chr, Annotation annot, double\mbox{[}$\,$\mbox{]} bg\+Nucleotide\+Freqs, short\mbox{[}$\,$\mbox{]} enc\+Chr\+Seq)  throws I\+O\+Exception 
\end{DoxyCompactItemize}
\subsection*{Static Public Attributes}
\begin{DoxyCompactItemize}
\item 
static final String \hyperlink{classbroad_1_1core_1_1motif_1_1_p_w_m_utils_aff73c275f71b26881df0ee1d11924665}{U\+S\+A\+G\+E}
\end{DoxyCompactItemize}


\subsection{Member Function Documentation}
\hypertarget{classbroad_1_1core_1_1motif_1_1_p_w_m_utils_ade2a53589ad91434fd0a187a7befd8b4}{\index{broad\+::core\+::motif\+::\+P\+W\+M\+Utils@{broad\+::core\+::motif\+::\+P\+W\+M\+Utils}!compute\+Sequence\+B\+G\+Sequences@{compute\+Sequence\+B\+G\+Sequences}}
\index{compute\+Sequence\+B\+G\+Sequences@{compute\+Sequence\+B\+G\+Sequences}!broad\+::core\+::motif\+::\+P\+W\+M\+Utils@{broad\+::core\+::motif\+::\+P\+W\+M\+Utils}}
\subsubsection[{compute\+Sequence\+B\+G\+Sequences}]{\setlength{\rightskip}{0pt plus 5cm}static double \mbox{[}$\,$\mbox{]} broad.\+core.\+motif.\+P\+W\+M\+Utils.\+compute\+Sequence\+B\+G\+Sequences (
\begin{DoxyParamCaption}
\item[{Map$<$ String, {\bf Sequence} $>$}]{chrs\+By\+Name, }
\item[{Annotation\+Reader$<$?extends {\bf Genomic\+Annotation} $>$}]{reader}
\end{DoxyParamCaption}
) throws I\+O\+Exception\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1core_1_1motif_1_1_p_w_m_utils_ade2a53589ad91434fd0a187a7befd8b4}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1core_1_1motif_1_1_p_w_m_utils_a17646e545879c7d2061fbc4cad0bf834}{\index{broad\+::core\+::motif\+::\+P\+W\+M\+Utils@{broad\+::core\+::motif\+::\+P\+W\+M\+Utils}!main@{main}}
\index{main@{main}!broad\+::core\+::motif\+::\+P\+W\+M\+Utils@{broad\+::core\+::motif\+::\+P\+W\+M\+Utils}}
\subsubsection[{main}]{\setlength{\rightskip}{0pt plus 5cm}static void broad.\+core.\+motif.\+P\+W\+M\+Utils.\+main (
\begin{DoxyParamCaption}
\item[{String\mbox{[}$\,$\mbox{]}}]{args}
\end{DoxyParamCaption}
) throws Illegal\+Argument\+Exception, Exception\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1core_1_1motif_1_1_p_w_m_utils_a17646e545879c7d2061fbc4cad0bf834}


Here is the call graph for this function\+:


\hypertarget{classbroad_1_1core_1_1motif_1_1_p_w_m_utils_a901bd6475f321d8416ddbf47d03d32b4}{\index{broad\+::core\+::motif\+::\+P\+W\+M\+Utils@{broad\+::core\+::motif\+::\+P\+W\+M\+Utils}!scan\+Region@{scan\+Region}}
\index{scan\+Region@{scan\+Region}!broad\+::core\+::motif\+::\+P\+W\+M\+Utils@{broad\+::core\+::motif\+::\+P\+W\+M\+Utils}}
\subsubsection[{scan\+Region}]{\setlength{\rightskip}{0pt plus 5cm}static Map$<${\bf Position\+Weight\+Matrix}, {\bf Lightweight\+Genomic\+Annotation} $>$ broad.\+core.\+motif.\+P\+W\+M\+Utils.\+scan\+Region (
\begin{DoxyParamCaption}
\item[{int}]{shuffles, }
\item[{String}]{prefix, }
\item[{double}]{min\+Score\+To\+Write, }
\item[{boolean}]{max\+Only, }
\item[{String}]{outdir, }
\item[{List$<$ {\bf Position\+Weight\+Matrix} $>$}]{pwms, }
\item[{String}]{chr, }
\item[{Annotation}]{annot, }
\item[{double\mbox{[}$\,$\mbox{]}}]{bg\+Nucleotide\+Freqs, }
\item[{short\mbox{[}$\,$\mbox{]}}]{enc\+Chr\+Seq}
\end{DoxyParamCaption}
) throws I\+O\+Exception\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1core_1_1motif_1_1_p_w_m_utils_a901bd6475f321d8416ddbf47d03d32b4}


Here is the call graph for this function\+:




Here is the caller graph for this function\+:




\subsection{Member Data Documentation}
\hypertarget{classbroad_1_1core_1_1motif_1_1_p_w_m_utils_aff73c275f71b26881df0ee1d11924665}{\index{broad\+::core\+::motif\+::\+P\+W\+M\+Utils@{broad\+::core\+::motif\+::\+P\+W\+M\+Utils}!U\+S\+A\+G\+E@{U\+S\+A\+G\+E}}
\index{U\+S\+A\+G\+E@{U\+S\+A\+G\+E}!broad\+::core\+::motif\+::\+P\+W\+M\+Utils@{broad\+::core\+::motif\+::\+P\+W\+M\+Utils}}
\subsubsection[{U\+S\+A\+G\+E}]{\setlength{\rightskip}{0pt plus 5cm}final String broad.\+core.\+motif.\+P\+W\+M\+Utils.\+U\+S\+A\+G\+E\hspace{0.3cm}{\ttfamily [static]}}}\label{classbroad_1_1core_1_1motif_1_1_p_w_m_utils_aff73c275f71b26881df0ee1d11924665}
{\bfseries Initial value\+:}
\begin{DoxyCode}
= \textcolor{stringliteral}{"Usage: ArrayDesignUtilities TASK=<task\_num> <task\_args>\(\backslash\)n"} +
    \textcolor{stringliteral}{"\(\backslash\)tTasks:\(\backslash\)n"} +
    \textcolor{stringliteral}{"\(\backslash\)t\(\backslash\)t1. Compute log likelohood score thresholds for a given PWM set. This task will compute the
       distribution of scores then output thresholds for commonly used quantiles. -pwm <File containing one or more PWM
       matrices> -pA <new equilibrium probability for A>  -pC <new equilibrium probability for C>  -pG <new
       equilibrium probability for G>  -pT <new equilibrium probability for T> -out <file name for score quantile cutoff per
       pwm file or standard out if none is specified> -outdir <output directory or current directory if this
       parameter is not specified> [-prefix <A prefix for each pwm distribution file>]> "} +
    \textcolor{stringliteral}{"\(\backslash\)n\(\backslash\)t\(\backslash\)tscan. Scan a set a set of genomic regions for PWM ocurrencens -pwm <File containing one or more
       PWM matrices> -regions <Genomic Annotations> -seqdir <Sequence directory> -maxOnly <If only the maximal hit
       per region should be reported> -outdir <Output directory to write output file> [-minScoreToWrite <Filter
       output by minimum score, reudces output file size> -prefix <Prefix to use for each PWM hit file> -shuffles
       <Number of pwm shuffles to asses significance>]"}+
    \textcolor{stringliteral}{"\(\backslash\)n\(\backslash\)t\(\backslash\)tpermute. Permute a PWM matrix -pwm <File contining one or more PWM matrics> -out <Output file
       with permuted matrices or standard out in not specified>"}+
    \textcolor{stringliteral}{"\(\backslash\)n\(\backslash\)t\(\backslash\)treverse. Reverse-compliment a PWM -pwm <File contining one or more PWM matrics> -out <Output
       file with permuted matrices or standard out in not specified>"} +
    \textcolor{stringliteral}{"\(\backslash\)n\(\backslash\)t\(\backslash\)tscanGenome. Scan the genome for PWM ocurrencens -pwm <File containing one or more PWM matrices>
       -seqdir <Sequence directory> -outdir <Output directory to write output file> -minScoreToWrite <Filter output
       by minimum score, reudces output file size> [-prefix <Prefix to use for each PWM hit file> -shuffles
       <Number of pwm shuffles to asses significance>]"}+
    \textcolor{stringliteral}{"\(\backslash\)n"}
\end{DoxyCode}


The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
java/broad/core/motif/\hyperlink{_p_w_m_utils_8java}{P\+W\+M\+Utils.\+java}\end{DoxyCompactItemize}
