\documentclass[twoside]{book}

% Packages required by doxygen
\usepackage{calc}
\usepackage{doxygen}
\usepackage{graphicx}
\usepackage[utf8]{inputenc}
\usepackage{makeidx}
\usepackage{multicol}
\usepackage{multirow}
\usepackage{fixltx2e}
\PassOptionsToPackage{warn}{textcomp}
\usepackage{textcomp}
\usepackage[nointegrals]{wasysym}
\usepackage[table]{xcolor}

% Font selection
\usepackage[T1]{fontenc}
\usepackage{mathptmx}
\usepackage[scaled=.90]{helvet}
\usepackage{courier}
\usepackage{amssymb}
\usepackage{sectsty}
\renewcommand{\familydefault}{\sfdefault}
\allsectionsfont{%
  \fontseries{bc}\selectfont%
  \color{darkgray}%
}
\renewcommand{\DoxyLabelFont}{%
  \fontseries{bc}\selectfont%
  \color{darkgray}%
}
\newcommand{\+}{\discretionary{\mbox{\scriptsize$\hookleftarrow$}}{}{}}

% Page & text layout
\usepackage{geometry}
\geometry{%
  a4paper,%
  top=2.5cm,%
  bottom=2.5cm,%
  left=2.5cm,%
  right=2.5cm%
}
\tolerance=750
\hfuzz=15pt
\hbadness=750
\setlength{\emergencystretch}{15pt}
\setlength{\parindent}{0cm}
\setlength{\parskip}{0.2cm}
\makeatletter
\renewcommand{\paragraph}{%
  \@startsection{paragraph}{4}{0ex}{-1.0ex}{1.0ex}{%
    \normalfont\normalsize\bfseries\SS@parafont%
  }%
}
\renewcommand{\subparagraph}{%
  \@startsection{subparagraph}{5}{0ex}{-1.0ex}{1.0ex}{%
    \normalfont\normalsize\bfseries\SS@subparafont%
  }%
}
\makeatother

% Headers & footers
\usepackage{fancyhdr}
\pagestyle{fancyplain}
\fancyhead[LE]{\fancyplain{}{\bfseries\thepage}}
\fancyhead[CE]{\fancyplain{}{}}
\fancyhead[RE]{\fancyplain{}{\bfseries\leftmark}}
\fancyhead[LO]{\fancyplain{}{\bfseries\rightmark}}
\fancyhead[CO]{\fancyplain{}{}}
\fancyhead[RO]{\fancyplain{}{\bfseries\thepage}}
\fancyfoot[LE]{\fancyplain{}{}}
\fancyfoot[CE]{\fancyplain{}{}}
\fancyfoot[RE]{\fancyplain{}{\bfseries\scriptsize Generated on Fri Aug 15 2014 15\+:11\+:16 for E\+S\+A\+T by Doxygen }}
\fancyfoot[LO]{\fancyplain{}{\bfseries\scriptsize Generated on Fri Aug 15 2014 15\+:11\+:16 for E\+S\+A\+T by Doxygen }}
\fancyfoot[CO]{\fancyplain{}{}}
\fancyfoot[RO]{\fancyplain{}{}}
\renewcommand{\footrulewidth}{0.4pt}
\renewcommand{\chaptermark}[1]{%
  \markboth{#1}{}%
}
\renewcommand{\sectionmark}[1]{%
  \markright{\thesection\ #1}%
}

% Indices & bibliography
\usepackage{natbib}
\usepackage[titles]{tocloft}
\setcounter{tocdepth}{3}
\setcounter{secnumdepth}{5}
\makeindex

% Hyperlinks (required, but should be loaded last)
\usepackage{ifpdf}
\ifpdf
  \usepackage[pdftex,pagebackref=true]{hyperref}
\else
  \usepackage[ps2pdf,pagebackref=true]{hyperref}
\fi
\hypersetup{%
  colorlinks=true,%
  linkcolor=blue,%
  citecolor=blue,%
  unicode%
}

% Custom commands
\newcommand{\clearemptydoublepage}{%
  \newpage{\pagestyle{empty}\cleardoublepage}%
}


%===== C O N T E N T S =====

\begin{document}

% Titlepage & ToC
\hypersetup{pageanchor=false,
             bookmarks=true,
             bookmarksnumbered=true,
             pdfencoding=unicode
            }
\pagenumbering{roman}
\begin{titlepage}
\vspace*{7cm}
\begin{center}%
{\Large E\+S\+A\+T }\\
\vspace*{1cm}
{\large Generated by Doxygen 1.8.7}\\
\vspace*{0.5cm}
{\small Fri Aug 15 2014 15:11:16}\\
\end{center}
\end{titlepage}
\clearemptydoublepage
\tableofcontents
\clearemptydoublepage
\pagenumbering{arabic}
\hypersetup{pageanchor=true}

%--- Begin generated contents ---
\chapter{Deprecated List}
\label{deprecated}
\hypertarget{deprecated}{}
\input{deprecated}
\chapter{Namespace Index}
\input{namespaces}
\chapter{Hierarchical Index}
\input{hierarchy}
\chapter{Class Index}
\input{annotated}
\chapter{File Index}
\input{files}
\chapter{Namespace Documentation}
\input{namespacebroad}
\input{namespacebroad_1_1core}
\input{namespacebroad_1_1core_1_1annotation}
\input{namespacebroad_1_1core_1_1datastructures}
\input{namespacebroad_1_1core_1_1error}
\input{namespacebroad_1_1core_1_1hmm}
\input{namespacebroad_1_1core_1_1math}
\input{namespacebroad_1_1core_1_1motif}
\input{namespacebroad_1_1core_1_1motif_1_1meme}
\input{namespacebroad_1_1core_1_1motif_1_1misc}
\input{namespacebroad_1_1core_1_1multiplealignment}
\input{namespacebroad_1_1core_1_1overlaputils}
\input{namespacebroad_1_1core_1_1parser}
\input{namespacebroad_1_1core_1_1primer3}
\input{namespacebroad_1_1core_1_1sequence}
\input{namespacebroad_1_1core_1_1siphy}
\input{namespacebroad_1_1core_1_1siphy_1_1tools}
\input{namespacebroad_1_1core_1_1siphy_1_1tools_1_1conservation}
\input{namespacebroad_1_1core_1_1util}
\input{namespacebroad_1_1pda}
\input{namespacebroad_1_1pda_1_1annotation}
\input{namespacebroad_1_1pda_1_1countreads}
\input{namespacebroad_1_1pda_1_1datastructures}
\input{namespacebroad_1_1pda_1_1differential_expression}
\input{namespacebroad_1_1pda_1_1enrichment}
\input{namespacebroad_1_1pda_1_1feature}
\input{namespacebroad_1_1pda_1_1feature_1_1genome}
\input{namespacebroad_1_1pda_1_1gene}
\input{namespacebroad_1_1pda_1_1geneexpression}
\input{namespacebroad_1_1pda_1_1geneexpression_1_1agilent}
\input{namespacebroad_1_1pda_1_1geneexpression_1_1agilent_1_1analysis}
\input{namespacebroad_1_1pda_1_1geneexpression_1_1clustering}
\input{namespacebroad_1_1pda_1_1rap}
\input{namespacebroad_1_1pda_1_1rap_1_1misc}
\input{namespacebroad_1_1pda_1_1rnai}
\input{namespacebroad_1_1pda_1_1samtools}
\input{namespacebroad_1_1pda_1_1seq}
\input{namespacebroad_1_1pda_1_1seq_1_1alignment}
\input{namespacebroad_1_1pda_1_1seq_1_1chip}
\input{namespacebroad_1_1pda_1_1seq_1_1clip}
\input{namespacebroad_1_1pda_1_1seq_1_1fastq}
\input{namespacebroad_1_1pda_1_1seq_1_1graph}
\input{namespacebroad_1_1pda_1_1seq_1_1protection}
\input{namespacebroad_1_1pda_1_1seq_1_1rap}
\input{namespacebroad_1_1pda_1_1seq_1_1rap_1_1rna}
\input{namespacebroad_1_1pda_1_1seq_1_1segmentation}
\input{namespacebroad_1_1pda_1_1seq_1_1segmentation_1_1utils}
\input{namespacebroad_1_1pda_1_1tools}
\input{namespacebroad_1_1projection}
\input{namespacebroad_1_1projection_1_1gsa}
\input{namespacejaligner}
\input{namespacejaligner_1_1formats}
\input{namespacejaligner_1_1matrix}
\input{namespacejaligner_1_1util}
\input{namespace_jama}
\input{namespace_jama_1_1util}
\input{namespacenextgen}
\input{namespacenextgen_1_1core}
\input{namespacenextgen_1_1core_1_1read_filters}
\input{namespaceumms}
\input{namespaceumms_1_1core}
\input{namespaceumms_1_1core_1_1alignment}
\input{namespaceumms_1_1core_1_1annotation}
\input{namespaceumms_1_1core_1_1annotation_1_1filter}
\input{namespaceumms_1_1core_1_1coordinatesystem}
\input{namespaceumms_1_1core_1_1exception}
\input{namespaceumms_1_1core_1_1feature}
\input{namespaceumms_1_1core_1_1general}
\input{namespaceumms_1_1core_1_1model}
\input{namespaceumms_1_1core_1_1model_1_1score}
\input{namespaceumms_1_1core_1_1readers}
\input{namespaceumms_1_1core_1_1read_filters}
\input{namespaceumms_1_1core_1_1scripture}
\input{namespaceumms_1_1core_1_1scripture_1_1statistics}
\input{namespaceumms_1_1core_1_1utils}
\input{namespaceumms_1_1core_1_1writers}
\input{namespaceumms_1_1esat}
\chapter{Class Documentation}
\input{classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_absolute_pearson_function}
\input{classumms_1_1core_1_1annotation_1_1_abstract_annotation}
\input{classumms_1_1core_1_1annotation_1_1_abstract_annotation_collection_3_01_t_01extends_01_annotation_01_4}
\input{classbroad_1_1core_1_1sequence_1_1_abstract_fasta_parser}
\input{classumms_1_1core_1_1model_1_1score_1_1_abstract_list_score}
\input{classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment}
\input{classumms_1_1core_1_1model_1_1score_1_1_window_processor_3_01_s_01extends_01_window_score_01_4_1ca0a3751200684ea73fe33d33dcb315e}
\input{classumms_1_1core_1_1model_1_1score_1_1_window_score_1_1_abstract_window_score}
\input{classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_array_stats}
\input{classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_probe}
\input{classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_to_g_c_t}
\input{classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_utils}
\input{classjaligner_1_1_alignment}
\input{interfaceumms_1_1core_1_1alignment_1_1_alignment}
\input{classumms_1_1core_1_1model_1_1_alignment_model_1_1_alignment_count}
\input{classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_1_1_alignment_count}
\input{interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model}
\input{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats}
\input{classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor}
\input{classumms_1_1core_1_1model_1_1_alignment_model}
\input{classumms_1_1core_1_1alignment_1_1_alignment_pair}
\input{classbroad_1_1pda_1_1datastructures_1_1_alignments}
\input{classbroad_1_1pda_1_1datastructures_1_1_alignments_reader}
\input{classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils}
\input{classbroad_1_1core_1_1motif_1_1meme_1_1_a_m_e_result_reader}
\input{interfaceumms_1_1core_1_1annotation_1_1_annotation}
\input{interfaceumms_1_1core_1_1annotation_1_1_annotation_collection_3_01_t_01extends_01_annotation_01_4}
\input{classbroad_1_1pda_1_1countreads_1_1_annotation_counts}
\input{interfacebroad_1_1core_1_1annotation_1_1_annotation_factory_3_01_t_01extends_01_lightweight_genomic_annotation_01_4}
\input{classbroad_1_1core_1_1annotation_1_1_annotation_factory_factory}
\input{classumms_1_1core_1_1annotation_1_1_annotation_file_reader}
\input{interfacebroad_1_1core_1_1annotation_1_1_annotation_handler}
\input{classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4}
\input{classumms_1_1core_1_1coordinatesystem_1_1_annotation_out_of_bounds_exception}
\input{classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4}
\input{classbroad_1_1core_1_1annotation_1_1_annotation_reader_factory}
\input{classumms_1_1core_1_1utils_1_1_annotation_utils}
\input{classbroad_1_1core_1_1annotation_1_1_annotation_utils}
\input{classbroad_1_1core_1_1math_1_1_a_n_o_v_a}
\input{classbroad_1_1core_1_1siphy_1_1tools_1_1conservation_1_1_average_branch_length}
\input{classbroad_1_1core_1_1hmm_1_1_bad_model_exception}
\input{classumms_1_1core_1_1annotation_1_1_basic_annotation}
\input{classbroad_1_1core_1_1annotation_1_1_basic_annotation_reader}
\input{classbroad_1_1core_1_1annotation_1_1_basic_genomic_annotation}
\input{classbroad_1_1core_1_1annotation_1_1_basic_genomic_annotation_reader}
\input{classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation}
\input{classumms_1_1core_1_1model_1_1score_1_1_basic_scan_statistic_score}
\input{classbroad_1_1core_1_1annotation_1_1_basic_two_subject_annotation}
\input{classbroad_1_1pda_1_1seq_1_1protection_1_1_batched_multi_sample_scan_peak_caller}
\input{classbroad_1_1pda_1_1differential_expression_1_1_batch_norm}
\input{classbroad_1_1core_1_1siphy_1_1tools_1_1conservation_1_1_batch_submit_estimate_exon}
\input{classbroad_1_1core_1_1annotation_1_1_b_e_d}
\input{classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser}
\input{classbroad_1_1core_1_1annotation_1_1_b_e_d_graph}
\input{classbroad_1_1core_1_1annotation_1_1_b_e_d_graph_reader}
\input{classbroad_1_1core_1_1annotation_1_1_b_e_d_reader}
\input{classumms_1_1core_1_1model_1_1score_1_1_binomial_enrichment_score}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1_build_ratio_null_distribution}
\input{classumms_1_1core_1_1scripture_1_1_build_scripture_coordinate_space}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1_calculate_coverage_stats}
\input{classjaligner_1_1_cell}
\input{class_jama_1_1_cholesky_decomposition}
\input{classbroad_1_1pda_1_1feature_1_1genome_1_1_chromosome}
\input{classumms_1_1core_1_1alignment_1_1_chromosome_inconsistency_exception}
\input{classbroad_1_1core_1_1util_1_1_chromosome_string_comparator}
\input{classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t}
\input{classbroad_1_1pda_1_1differential_expression_1_1_cis_vs_trans}
\input{classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_classify_agilent_linc_probes}
\input{classumms_1_1core_1_1general_1_1_closeable_filter_iterator_3_01_t_01_4}
\input{classjaligner_1_1formats_1_1_c_l_u_s_t_a_l}
\input{classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_cluster}
\input{interfacebroad_1_1pda_1_1geneexpression_1_1clustering_1_1_cluster_distance_function}
\input{classbroad_1_1core_1_1util_1_1_c_l_util}
\input{classbroad_1_1core_1_1util_1_1_collapse_by_intersection}
\input{classumms_1_1esat_1_1_collapsed_gene_lengths}
\input{classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_collapse_probes_by_min_p_value}
\input{classbroad_1_1pda_1_1differential_expression_1_1_collapse_projections}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1_collect_annotation_enrichments}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1rna_1_1_collect_rna_seq_metrics}
\input{classbroad_1_1core_1_1math_1_1_combination_generator}
\input{classbroad_1_1core_1_1parser_1_1_command_line_parser}
\input{classjaligner_1_1util_1_1_commons}
\input{classumms_1_1core_1_1annotation_1_1_compound_interval}
\input{classbroad_1_1core_1_1motif_1_1misc_1_1_compute_conserved_motifs}
\input{classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_compute_correlation_of_replicates}
\input{classbroad_1_1pda_1_1seq_1_1chip_1_1_compute_enrichments}
\input{classbroad_1_1core_1_1math_1_1_compute_f_d_r}
\input{classbroad_1_1pda_1_1geneexpression_1_1_compute_hypergeomtric_enrichment}
\input{classbroad_1_1core_1_1siphy_1_1tools_1_1conservation_1_1_compute_kmers_for_region}
\input{classbroad_1_1core_1_1siphy_1_1tools_1_1conservation_1_1_compute_max_percentile_of_elements}
\input{classbroad_1_1core_1_1primer3_1_1_compute_m_i_r_score}
\input{classbroad_1_1core_1_1primer3_1_1_compute_original_score}
\input{classbroad_1_1core_1_1siphy_1_1tools_1_1conservation_1_1_compute_percentage_of_kmers}
\input{classumms_1_1core_1_1scripture_1_1statistics_1_1_connect_disconnected_transcripts}
\input{classbroad_1_1core_1_1siphy_1_1_conservation_utils}
\input{classbroad_1_1core_1_1siphy_1_1_conserved_p_w_m_scanner}
\input{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model}
\input{classbroad_1_1pda_1_1differential_expression_1_1_convert_to_discrete_matrix}
\input{interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space}
\input{classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_correlation_agilent_nanostring}
\input{classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_correlation_between_replicates}
\input{classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_correlation_matrix}
\input{classbroad_1_1pda_1_1geneexpression_1_1_count_nums}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1_count_probes}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1_count_reads}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1rna_1_1_count_reads}
\input{classumms_1_1core_1_1model_1_1score_1_1_count_score}
\input{classumms_1_1core_1_1model_1_1score_1_1_coverage_score}
\input{classbroad_1_1core_1_1primer3_1_1_cross_junction_p_c_r_designer}
\input{classbroad_1_1projection_1_1gsa_1_1_define_gene_signatures}
\input{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_define_source_and_sink_from_paired_ends}
\input{classbroad_1_1pda_1_1differential_expression_1_1_differential_expression}
\input{classbroad_1_1pda_1_1differential_expression_1_1_differential_scoring}
\input{classjaligner_1_1_directions}
\input{classbroad_1_1pda_1_1feature_1_1genome_1_1_directory_installed_genome_assembly}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1_distance_to_b_e_d}
\input{classbroad_1_1core_1_1math_1_1_distribution}
\input{enumbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t_1_1_edge_source_type}
\input{class_jama_1_1_eigenvalue_decomposition}
\input{classbroad_1_1core_1_1math_1_1_empirical_distribution}
\input{classumms_1_1core_1_1model_1_1score_1_1_end_in_window_count_score}
\input{classbroad_1_1pda_1_1enrichment_1_1_enrichment_metric}
\input{classbroad_1_1pda_1_1enrichment_1_1_enrichment_utils}
\input{classumms_1_1esat_1_1_e_s_a_t}
\input{classumms_1_1esat_1_1_e_s_a_t_distance_calculator}
\input{classbroad_1_1core_1_1siphy_1_1tools_1_1conservation_1_1_estimate_omega_per_exon}
\input{classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_euclidean_distance}
\input{classbroad_1_1core_1_1siphy_1_1_evolutionary_model}
\input{classbroad_1_1core_1_1siphy_1_1_evolutionary_model_parameters}
\input{classbroad_1_1pda_1_1geneexpression_1_1_expression_experiment_info}
\input{classbroad_1_1projection_1_1gsa_1_1_extract_genes}
\input{classbroad_1_1core_1_1sequence_1_1_extractor}
\input{classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_extract_samples_and_label}
\input{classbroad_1_1pda_1_1rnai_1_1_extract_sequence}
\input{interfaceumms_1_1core_1_1general_1_1_tabbed_reader_1_1_factory_3_01_t_01_4}
\input{classjaligner_1_1formats_1_1_f_a_s_t_a}
\input{classbroad_1_1pda_1_1seq_1_1clip_1_1_fasta_extend_peaks}
\input{interfacebroad_1_1core_1_1sequence_1_1_fasta_handler}
\input{classbroad_1_1core_1_1multiplealignment_1_1_fasta_multiple_alignment_i_o}
\input{classbroad_1_1core_1_1sequence_1_1_fasta_parser}
\input{classbroad_1_1core_1_1sequence_1_1_fasta_quality_binary_i_o}
\input{interfacebroad_1_1core_1_1sequence_1_1_fasta_quality_handler}
\input{classbroad_1_1core_1_1sequence_1_1_fasta_quality_index}
\input{classbroad_1_1core_1_1sequence_1_1_fasta_quality_parser}
\input{classbroad_1_1core_1_1sequence_1_1_fasta_sequence_i_o}
\input{classbroad_1_1pda_1_1countreads_1_1_fastq_library_stats}
\input{classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_parser}
\input{classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_sequence}
\input{classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_splitter_by_barcode}
\input{classbroad_1_1core_1_1math_1_1_f_d_r_distribution}
\input{interfacebroad_1_1core_1_1annotation_1_1_feature}
\input{classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment}
\input{classumms_1_1core_1_1model_1_1_alignment_model_1_1_filtered_iterator}
\input{classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_1_1_filtered_iterator}
\input{classbroad_1_1pda_1_1samtools_1_1_filter_fastq_reads}
\input{classbroad_1_1pda_1_1geneexpression_1_1_filter_g_c_t}
\input{classbroad_1_1pda_1_1geneexpression_1_1_filter_g_c_t_file}
\input{classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_filter_matrix_by_min_cutoff}
\input{classbroad_1_1pda_1_1samtools_1_1_filter_paired_reads_by_m_a_q}
\input{classbroad_1_1pda_1_1seq_1_1chip_1_1_find_diffrential_enriched_regions}
\input{classbroad_1_1pda_1_1rap_1_1misc_1_1_find_enriched_depleted_regions}
\input{classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_find_probes_for_gene_set}
\input{classbroad_1_1core_1_1math_1_1_fisher_combined_probability_test}
\input{interfacebroad_1_1core_1_1siphy_1_1_fit}
\input{classbroad_1_1pda_1_1differential_expression_1_1_fixed_control_differential_expression}
\input{classbroad_1_1pda_1_1differential_expression_1_1_fold_significance}
\input{classjaligner_1_1formats_1_1_format}
\input{classjaligner_1_1formats_1_1_format_factory}
\input{classumms_1_1core_1_1alignment_1_1_fragment_alignment}
\input{classumms_1_1core_1_1annotation_1_1filter_1_1_fully_contained_filter}
\input{classbroad_1_1pda_1_1gene_1_1functional_annotation}
\input{classbroad_1_1core_1_1datastructures_1_1_interval_tree_3_01_v_01_4_1_1_fwd_iterator}
\input{classbroad_1_1pda_1_1differential_expression_1_1_f_w_e_r_distribution}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1_g_c_content}
\input{classumms_1_1core_1_1annotation_1_1_gene}
\input{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_counts}
\input{classbroad_1_1pda_1_1geneexpression_1_1gene_exp_utils}
\input{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model}
\input{classbroad_1_1core_1_1datastructures_1_1_generic_rooted_tree_3_01_t_01_4}
\input{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score}
\input{classbroad_1_1projection_1_1gsa_1_1_gene_set_enrichment}
\input{classbroad_1_1core_1_1overlaputils_1_1_gene_set_intersect}
\input{classbroad_1_1pda_1_1gene_1_1_gene_tools}
\input{classumms_1_1core_1_1coordinatesystem_1_1_transcriptome_space_1_1_gene_tree}
\input{classumms_1_1core_1_1feature_1_1_gene_window}
\input{classbroad_1_1pda_1_1gene_1_1_gene_with_isoforms}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1_genome_command_line_program}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1_genome_scoring_program}
\input{classbroad_1_1pda_1_1seq_1_1clip_1_1_genome_space_sample_data}
\input{classbroad_1_1pda_1_1seq_1_1protection_1_1_genome_space_sample_data}
\input{classumms_1_1core_1_1feature_1_1_genome_window}
\input{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_genome_with_gaps}
\input{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_genome_with_gaps2}
\input{interfacebroad_1_1core_1_1annotation_1_1_genomic_annotation}
\input{interfacebroad_1_1core_1_1annotation_1_1_genomic_annotation_filter_3_01_t_01extends_01_lightweight_genomic_annotation_01_4}
\input{classumms_1_1core_1_1coordinatesystem_1_1_genomic_space}
\input{classnextgen_1_1core_1_1read_filters_1_1_genomic_span_filter}
\input{classbroad_1_1core_1_1annotation_1_1_g_f_f}
\input{classbroad_1_1core_1_1annotation_1_1_g_f_f_reader}
\input{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_global_splice_filter}
\input{classbroad_1_1pda_1_1geneexpression_1_1_global_stats}
\input{classbroad_1_1core_1_1util_1_1_g_m_t_parser}
\input{classbroad_1_1pda_1_1differential_expression_1_1_grab_probes}
\input{classbroad_1_1pda_1_1geneexpression_1_1_grab_specific_columns}
\input{classbroad_1_1pda_1_1differential_expression_1_1_g_s_a_projection}
\input{classbroad_1_1projection_1_1gsa_1_1_g_s_a_scoring}
\input{classbroad_1_1projection_1_1gsa_1_1_gsa_utils}
\input{classbroad_1_1pda_1_1annotation_1_1_g_t_f_file_parser}
\input{classbroad_1_1core_1_1primer3_1_1_hairpin_kmer}
\input{classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_hierarchical_clustering}
\input{classbroad_1_1core_1_1datastructures_1_1_illegal_tree_operation_exception}
\input{classumms_1_1core_1_1exception_1_1_incomplete_method_implementation_exception}
\input{interfacebroad_1_1core_1_1datastructures_1_1_interval}
\input{classbroad_1_1core_1_1datastructures_1_1_interval_tree_3_01_v_01_4}
\input{classbroad_1_1core_1_1primer3_1_1is_p_c_r_like}
\input{classumms_1_1core_1_1model_1_1_j_c_s_alignment_model}
\input{classbroad_1_1core_1_1datastructures_1_1_j_c_s_cache}
\input{classbroad_1_1core_1_1datastructures_1_1_j_c_s_cache_1_1_j_c_s_node_iterator}
\input{classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_kmeans}
\input{classbroad_1_1core_1_1math_1_1_k_s_test}
\input{classumms_1_1core_1_1model_1_1score_1_1_length_score}
\input{interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation}
\input{classbroad_1_1pda_1_1datastructures_1_1_location_aware_matrix}
\input{classbroad_1_1pda_1_1annotation_1_1_locus}
\input{classbroad_1_1core_1_1math_1_1_loess}
\input{class_jama_1_1_l_u_decomposition}
\input{classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_alignment}
\input{classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_i_o}
\input{classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_make_c_l_s_files}
\input{classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_make_c_l_s_from_description}
\input{classbroad_1_1pda_1_1differential_expression_1_1_make_positional_gene_sets}
\input{classbroad_1_1core_1_1math_1_1_mann_whitney}
\input{classbroad_1_1core_1_1math_1_1_mann_whitney_permutations}
\input{classumms_1_1core_1_1read_filters_1_1_mapping_quality_filter}
\input{classbroad_1_1core_1_1hmm_1_1_markov_model_3_01_t_01_4}
\input{interfacebroad_1_1core_1_1hmm_1_1_markov_state_3_01_t_01_4}
\input{classjaligner_1_1_markups}
\input{classumms_1_1core_1_1coordinatesystem_1_1_masked_genomic_space}
\input{class_jama_1_1util_1_1_maths}
\input{classbroad_1_1core_1_1math_1_1_math_util}
\input{classjaligner_1_1matrix_1_1_matrices_compartor}
\input{classjaligner_1_1matrix_1_1_matrix}
\input{class_jama_1_1_matrix}
\input{classjaligner_1_1matrix_1_1_matrix_generator}
\input{classjaligner_1_1matrix_1_1_matrix_loader_exception}
\input{classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers}
\input{classbroad_1_1core_1_1datastructures_1_1_matrix_with_header_utils}
\input{classbroad_1_1core_1_1annotation_1_1_maximum_contiguous_subsequence}
\input{classbroad_1_1core_1_1motif_1_1meme_1_1_meme_motif}
\input{classbroad_1_1core_1_1motif_1_1meme_1_1_meme_result_reader}
\input{classbroad_1_1pda_1_1differential_expression_1_1_merge_g_s_e_a_output_into_matrix}
\input{classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment}
\input{classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment_factory}
\input{interfacebroad_1_1core_1_1multiplealignment_1_1_multiple_alignment_i_o}
\input{classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment_i_o_factory}
\input{classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller}
\input{classumms_1_1core_1_1model_1_1score_1_1_multi_score}
\input{classjaligner_1_1_needleman_wunsch}
\input{classjaligner_1_1_needleman_wunsch_gotoh}
\input{classbroad_1_1pda_1_1gene_1_1_neighbor_analysis}
\input{classbroad_1_1pda_1_1gene_1_1_neighbors_null_model}
\input{classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score}
\input{classumms_1_1core_1_1model_1_1_alignment_model_1_1_node_iterator}
\input{classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_1_1_node_iterator}
\input{classumms_1_1core_1_1scripture_1_1_oriented_chromosome_transcript_graph}
\input{classumms_1_1core_1_1annotation_1_1filter_1_1_overlap_filter}
\input{classbroad_1_1core_1_1datastructures_1_1_interval_tree_3_01_v_01_4_1_1_overlap_iterator}
\input{classbroad_1_1pda_1_1geneexpression_1_1_pa_g_e_java}
\input{classjaligner_1_1formats_1_1_pair}
\input{classbroad_1_1core_1_1datastructures_1_1_pair_3_01_t1_01_4}
\input{classnextgen_1_1core_1_1read_filters_1_1_paired_and_proper_filter}
\input{classumms_1_1core_1_1alignment_1_1_paired_end_alignment_factory}
\input{classumms_1_1core_1_1read_filters_1_1_paired_end_filter}
\input{classumms_1_1core_1_1readers_1_1_paired_end_reader}
\input{classbroad_1_1pda_1_1samtools_1_1_paired_end_read_iterator}
\input{classbroad_1_1pda_1_1samtools_1_1_paired_end_t_d_f}
\input{classumms_1_1core_1_1writers_1_1_paired_end_writer}
\input{classumms_1_1core_1_1alignment_1_1_paired_read_alignment}
\input{classbroad_1_1core_1_1error_1_1_parse_exception}
\input{classbroad_1_1core_1_1util_1_1_parse_g_c_t_file}
\input{classbroad_1_1pda_1_1seq_1_1graph_1_1_path}
\input{classumms_1_1core_1_1read_filters_1_1_p_c_r_duplicate_filter}
\input{classbroad_1_1pda_1_1seq_1_1chip_1_1_peak_filter}
\input{classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_pearson_distance}
\input{classbroad_1_1core_1_1math_1_1_permutation_distribution}
\input{classumms_1_1core_1_1coordinatesystem_1_1_permutation_not_found_exception}
\input{classbroad_1_1pda_1_1differential_expression_1_1_permutations}
\input{classbroad_1_1core_1_1siphy_1_1_phastcons_reader}
\input{classbroad_1_1core_1_1multiplealignment_1_1_phylip_interleaved_multiple_alignment_i_o}
\input{classbroad_1_1core_1_1multiplealignment_1_1_phylip_sequencial_multiple_alignment_i_o}
\input{classbroad_1_1core_1_1siphy_1_1_pi_h_m_m}
\input{classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1_plot_aggregate_regions}
\input{classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score}
\input{classbroad_1_1core_1_1motif_1_1_position_weight_column}
\input{classbroad_1_1core_1_1motif_1_1_position_weight_matrix}
\input{classbroad_1_1core_1_1motif_1_1_position_weight_matrix_i_o}
\input{classbroad_1_1core_1_1siphy_1_1_position_weight_matrix_model}
\input{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_precompute_data_alignment_stats}
\input{classumms_1_1core_1_1general_1_1_predicates}
\input{classbroad_1_1pda_1_1geneexpression_1_1_preprocess_data_set}
\input{classbroad_1_1core_1_1primer3_1_1_primer3_exception}
\input{interfacebroad_1_1core_1_1primer3_1_1_primer3_picking_summary}
\input{interfacebroad_1_1core_1_1primer3_1_1_primer3_primer_picking_result}
\input{classbroad_1_1core_1_1primer3_1_1_primer_utils}
\input{classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1analysis_1_1_process_dataset}
\input{classumms_1_1core_1_1annotation_1_1_p_s_l_record}
\input{classbroad_1_1pda_1_1tools_1_1_p_s_l_x_to_s_a_m}
\input{classbroad_1_1core_1_1motif_1_1_p_w_m_utils}
\input{classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_q_c_agilent_results}
\input{class_jama_1_1_q_r_decomposition}
\input{classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_rank_order}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1_r_a_p_peak_caller}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1_ratio_permutation_peak_caller}
\input{classumms_1_1core_1_1model_1_1score_1_1_ratio_score}
\input{interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_read_count_normalizer}
\input{classumms_1_1core_1_1read_filters_1_1_read_ends_in_filter}
\input{interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_read_filter_3_01_t_01_4}
\input{classbroad_1_1pda_1_1seq_1_1protection_1_1_read_simulator}
\input{classbroad_1_1pda_1_1seq_1_1protection_1_1_read_simulator2}
\input{classumms_1_1core_1_1read_filters_1_1_read_starts_in_filter}
\input{classnextgen_1_1core_1_1read_filters_1_1_reads_to_reconstruct_filter}
\input{classbroad_1_1pda_1_1seq_1_1segmentation_1_1utils_1_1_reconstruction_comparer}
\input{classbroad_1_1core_1_1annotation_1_1_repeat_masker_annotation}
\input{classbroad_1_1core_1_1annotation_1_1_repeat_masker_annotation_reader}
\input{classbroad_1_1core_1_1primer3_1_1_restriction_enzyme}
\input{classbroad_1_1core_1_1datastructures_1_1_reversible_iterator_3_01_e_01_4}
\input{classbroad_1_1core_1_1datastructures_1_1_interval_tree_3_01_v_01_4_1_1_rev_iterator}
\input{classbroad_1_1pda_1_1rnai_1_1_r_n_ai_gene_annotation}
\input{classbroad_1_1pda_1_1differential_expression_1_1_run_all_g_s_e_a_batch}
\input{classbroad_1_1pda_1_1differential_expression_1_1_run_differential_expression}
\input{classbroad_1_1pda_1_1differential_expression_1_1_run_g_s_e_a}
\input{classumms_1_1core_1_1exception_1_1_runtime_i_o_exception}
\input{classbroad_1_1pda_1_1differential_expression_1_1_run_time_series}
\input{classumms_1_1core_1_1read_filters_1_1_same_orientation_filter}
\input{classbroad_1_1pda_1_1samtools_1_1_s_a_m_fragment_writer}
\input{classbroad_1_1pda_1_1samtools_1_1_s_a_m_paired_end_file_reader}
\input{classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data}
\input{classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data}
\input{interfacebroad_1_1core_1_1siphy_1_1_scaled_window_parser_listener}
\input{classbroad_1_1core_1_1math_1_1_scan_statistics}
\input{classumms_1_1core_1_1model_1_1score_1_1_scan_statistic_score}
\input{classbroad_1_1pda_1_1geneexpression_1_1_score_diff_genes}
\input{classbroad_1_1core_1_1motif_1_1misc_1_1_score_motif_instances}
\input{classbroad_1_1core_1_1motif_1_1_search_exception}
\input{classbroad_1_1core_1_1sequence_1_1_sequence}
\input{classjaligner_1_1_sequence}
\input{classbroad_1_1core_1_1motif_1_1_sequence_motif}
\input{classbroad_1_1core_1_1motif_1_1_sequence_motif_cluster}
\input{classjaligner_1_1util_1_1_sequence_parser}
\input{classjaligner_1_1util_1_1_sequence_parser_exception}
\input{classbroad_1_1core_1_1sequence_1_1_sequence_region}
\input{classbroad_1_1core_1_1sequence_1_1_sequence_region_window}
\input{classbroad_1_1core_1_1sequence_1_1_sequence_utils}
\input{classbroad_1_1core_1_1annotation_1_1_short_b_e_d}
\input{classbroad_1_1core_1_1annotation_1_1_short_b_e_d_reader}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1_shuffle_b_a_m}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1_shuffle_b_e_d}
\input{classumms_1_1core_1_1coordinatesystem_1_1_shuffled_iterator_3_01_t_01extends_01_annotation_01_4}
\input{classumms_1_1core_1_1alignment_1_1_single_end_alignment}
\input{classumms_1_1core_1_1annotation_1_1_single_interval}
\input{class_jama_1_1_singular_value_decomposition}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1_skellam_k_l_peak_caller}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1_skellam_score}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1_slide_and_calculate}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1_slide_and_count}
\input{classumms_1_1core_1_1alignment_1_1_smatch_like}
\input{classumms_1_1core_1_1alignment_1_1_smith_waterman_alignment}
\input{classjaligner_1_1_smith_waterman_gotoh}
\input{classumms_1_1core_1_1read_filters_1_1_spliced_read_filter}
\input{classbroad_1_1pda_1_1seq_1_1fastq_1_1_split_and_trim_index}
\input{classbroad_1_1pda_1_1differential_expression_1_1_split_by_name}
\input{classbroad_1_1pda_1_1seq_1_1fastq_1_1_split_fastq_by_index}
\input{classbroad_1_1core_1_1siphy_1_1tools_1_1conservation_1_1_split_file}
\input{classumms_1_1core_1_1model_1_1score_1_1_start_in_window_count_score}
\input{classbroad_1_1core_1_1siphy_1_1_stationary_distribution_i_o}
\input{classbroad_1_1core_1_1math_1_1_statistics}
\input{classbroad_1_1pda_1_1seq_1_1alignment_1_1_stored_alignment_stats}
\input{enumumms_1_1core_1_1annotation_1_1_annotation_1_1_strand}
\input{classbroad_1_1core_1_1parser_1_1_string_parser}
\input{classumms_1_1core_1_1model_1_1score_1_1_sum_score}
\input{classumms_1_1core_1_1general_1_1_tabbed_reader}
\input{classjaligner_1_1_traceback_node}
\input{classbroad_1_1core_1_1sequence_1_1_transcribed_regions}
\input{classbroad_1_1core_1_1sequence_1_1_transcribed_sequence_composition}
\input{classumms_1_1core_1_1coordinatesystem_1_1_transcript_in_genomic_space}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1rna_1_1_transcriptome_command_line_program}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1rna_1_1_transcriptome_scoring_program}
\input{classumms_1_1core_1_1coordinatesystem_1_1_transcriptome_space}
\input{classbroad_1_1pda_1_1gene_1_1transcripts_null_model}
\input{classbroad_1_1core_1_1siphy_1_1_tree_scaler}
\input{classbroad_1_1core_1_1siphy_1_1_tree_scaler_i_o}
\input{classbroad_1_1pda_1_1samtools_1_1_trim_fastq_reads}
\input{classbroad_1_1pda_1_1seq_1_1protection_1_1_two_sample_scan_skellam_peak_caller}
\input{interfacebroad_1_1core_1_1annotation_1_1_two_subject_annotation}
\input{classbroad_1_1core_1_1siphy_1_1_unable_to_fit_exception}
\input{classbroad_1_1core_1_1math_1_1_unfair_die}
\input{classumms_1_1core_1_1read_filters_1_1_unique_mapped_reads_filter}
\input{classbroad_1_1pda_1_1seq_1_1rap_1_1_unmark_duplicates}
\input{classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_1_1_unordered_gene_pair}
\input{classbroad_1_1core_1_1math_1_1_wilcox2_sample}
\input{interfaceumms_1_1core_1_1feature_1_1_window}
\input{classumms_1_1core_1_1model_1_1score_1_1_window_all_score}
\input{interfaceumms_1_1core_1_1model_1_1score_1_1_window_processor_3_01_s_01extends_01_window_score_01_4}
\input{interfaceumms_1_1core_1_1model_1_1score_1_1_window_score}
\input{classumms_1_1core_1_1model_1_1score_1_1_window_score_iterator_3_01_t_01extends_01_window_score_01_4}
\input{classbroad_1_1pda_1_1seq_1_1protection_1_1_window_scores}
\input{classbroad_1_1core_1_1sequence_1_1_window_slider}
\input{classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_1_1_wrap_alignment_count_iterator}
\input{classumms_1_1core_1_1model_1_1_alignment_model_1_1_wrap_alignment_count_iterator}
\input{classbroad_1_1pda_1_1seq_1_1segmentation_1_1_wrapped_i_g_v_iterator}
\input{classbroad_1_1pda_1_1samtools_1_1_wrapped_iterator}
\input{classumms_1_1core_1_1alignment_1_1_single_end_alignment_1_1_wrap_sam_record}
\chapter{File Documentation}
\input{_annotation_factory_8java}
\input{_annotation_factory_factory_8java}
\input{_annotation_handler_8java}
\input{_annotation_reader_8java}
\input{_annotation_reader_factory_8java}
\input{broad_2core_2annotation_2_annotation_utils_8java}
\input{umms_2core_2utils_2_annotation_utils_8java}
\input{_basic_annotation_reader_8java}
\input{_basic_genomic_annotation_8java}
\input{_basic_genomic_annotation_reader_8java}
\input{_basic_lightweight_annotation_8java}
\input{_basic_two_subject_annotation_8java}
\input{_b_e_d_8java}
\input{_b_e_d_graph_8java}
\input{_b_e_d_graph_reader_8java}
\input{_b_e_d_reader_8java}
\input{_feature_8java}
\input{_genomic_annotation_8java}
\input{_genomic_annotation_filter_8java}
\input{_g_f_f_8java}
\input{_g_f_f_reader_8java}
\input{_lightweight_genomic_annotation_8java}
\input{_maximum_contiguous_subsequence_8java}
\input{_repeat_masker_annotation_8java}
\input{_repeat_masker_annotation_reader_8java}
\input{_short_b_e_d_8java}
\input{_short_b_e_d_reader_8java}
\input{_two_subject_annotation_8java}
\input{_generic_rooted_tree_8java}
\input{_illegal_tree_operation_exception_8java}
\input{_interval_8java}
\input{_interval_tree_8java}
\input{_j_c_s_cache_8java}
\input{_matrix_with_headers_8java}
\input{_matrix_with_header_utils_8java}
\input{broad_2core_2datastructures_2_pair_8java}
\input{jaligner_2formats_2_pair_8java}
\input{_reversible_iterator_8java}
\input{_parse_exception_8java}
\input{_bad_model_exception_8java}
\input{_markov_model_8java}
\input{_markov_state_8java}
\input{_a_n_o_v_a_8java}
\input{_combination_generator_8java}
\input{_compute_f_d_r_8java}
\input{_distribution_8java}
\input{_empirical_distribution_8java}
\input{_f_d_r_distribution_8java}
\input{_fisher_combined_probability_test_8java}
\input{_k_s_test_8java}
\input{_loess_8java}
\input{_mann_whitney_8java}
\input{_mann_whitney_permutations_8java}
\input{_math_util_8java}
\input{_permutation_distribution_8java}
\input{_scan_statistics_8java}
\input{_statistics_8java}
\input{_unfair_die_8java}
\input{_wilcox2_sample_8java}
\input{_a_m_e_result_reader_8java}
\input{_meme_motif_8java}
\input{_meme_result_reader_8java}
\input{_compute_conserved_motifs_8java}
\input{_score_motif_instances_8java}
\input{_position_weight_column_8java}
\input{_position_weight_matrix_8java}
\input{_position_weight_matrix_i_o_8java}
\input{_p_w_m_utils_8java}
\input{_search_exception_8java}
\input{_sequence_motif_8java}
\input{_sequence_motif_cluster_8java}
\input{_alignment_extractor_8java}
\input{_fasta_multiple_alignment_i_o_8java}
\input{_m_a_f_alignment_8java}
\input{_m_a_f_i_o_8java}
\input{_multiple_alignment_8java}
\input{_multiple_alignment_factory_8java}
\input{_multiple_alignment_i_o_8java}
\input{_multiple_alignment_i_o_factory_8java}
\input{_phylip_interleaved_multiple_alignment_i_o_8java}
\input{_phylip_sequencial_multiple_alignment_i_o_8java}
\input{_gene_set_intersect_8java}
\input{_command_line_parser_8java}
\input{_string_parser_8java}
\input{_compute_m_i_r_score_8java}
\input{_compute_original_score_8java}
\input{_cross_junction_p_c_r_designer_8java}
\input{_hairpin_kmer_8java}
\input{is_p_c_r_like_8java}
\input{_primer3_exception_8java}
\input{_primer3_picking_summary_8java}
\input{_primer3_primer_picking_result_8java}
\input{_primer_utils_8java}
\input{_restriction_enzyme_8java}
\input{_abstract_fasta_parser_8java}
\input{_extractor_8java}
\input{_fasta_handler_8java}
\input{_fasta_parser_8java}
\input{_fasta_quality_binary_i_o_8java}
\input{_fasta_quality_handler_8java}
\input{_fasta_quality_index_8java}
\input{_fasta_quality_parser_8java}
\input{_fasta_sequence_i_o_8java}
\input{broad_2core_2sequence_2_sequence_8java}
\input{jaligner_2_sequence_8java}
\input{_sequence_region_8java}
\input{_sequence_region_window_8java}
\input{_sequence_utils_8java}
\input{_transcribed_regions_8java}
\input{_transcribed_sequence_composition_8java}
\input{_window_slider_8java}
\input{_conservation_utils_8java}
\input{_conserved_p_w_m_scanner_8java}
\input{_evolutionary_model_8java}
\input{_evolutionary_model_parameters_8java}
\input{_fit_8java}
\input{_phastcons_reader_8java}
\input{_pi_h_m_m_8java}
\input{_position_weight_matrix_model_8java}
\input{_scaled_window_parser_listener_8java}
\input{_stationary_distribution_i_o_8java}
\input{_average_branch_length_8java}
\input{_batch_submit_estimate_exon_8java}
\input{_compute_kmers_for_region_8java}
\input{_compute_max_percentile_of_elements_8java}
\input{_compute_percentage_of_kmers_8java}
\input{_estimate_omega_per_exon_8java}
\input{_split_file_8java}
\input{_tree_scaler_8java}
\input{_tree_scaler_i_o_8java}
\input{_unable_to_fit_exception_8java}
\input{_chromosome_string_comparator_8java}
\input{_c_l_util_8java}
\input{_collapse_by_intersection_8java}
\input{_g_m_t_parser_8java}
\input{_parse_g_c_t_file_8java}
\input{_b_e_d_file_parser_8java}
\input{_g_t_f_file_parser_8java}
\input{_locus_8java}
\input{_annotation_counts_8java}
\input{_fastq_library_stats_8java}
\input{_alignments_8java}
\input{_alignments_reader_8java}
\input{_location_aware_matrix_8java}
\input{_batch_norm_8java}
\input{_cis_vs_trans_8java}
\input{_collapse_projections_8java}
\input{_convert_to_discrete_matrix_8java}
\input{_differential_expression_8java}
\input{_differential_scoring_8java}
\input{_fixed_control_differential_expression_8java}
\input{_fold_significance_8java}
\input{_f_w_e_r_distribution_8java}
\input{_grab_probes_8java}
\input{_g_s_a_projection_8java}
\input{_make_positional_gene_sets_8java}
\input{_merge_g_s_e_a_output_into_matrix_8java}
\input{_permutations_8java}
\input{_run_all_g_s_e_a_batch_8java}
\input{_run_differential_expression_8java}
\input{_run_g_s_e_a_8java}
\input{_run_time_series_8java}
\input{_split_by_name_8java}
\input{_enrichment_metric_8java}
\input{_enrichment_utils_8java}
\input{_chromosome_8java}
\input{_directory_installed_genome_assembly_8java}
\input{functional_annotation_8java}
\input{_gene_tools_8java}
\input{_gene_with_isoforms_8java}
\input{_neighbor_analysis_8java}
\input{_neighbors_null_model_8java}
\input{transcripts_null_model_8java}
\input{_agilent_array_stats_8java}
\input{_agilent_probe_8java}
\input{_agilent_to_g_c_t_8java}
\input{_agilent_utils_8java}
\input{_process_dataset_8java}
\input{_classify_agilent_linc_probes_8java}
\input{_collapse_probes_by_min_p_value_8java}
\input{_correlation_agilent_nanostring_8java}
\input{_correlation_between_replicates_8java}
\input{_extract_samples_and_label_8java}
\input{_filter_matrix_by_min_cutoff_8java}
\input{_find_probes_for_gene_set_8java}
\input{_make_c_l_s_files_8java}
\input{_make_c_l_s_from_description_8java}
\input{_q_c_agilent_results_8java}
\input{_absolute_pearson_function_8java}
\input{_cluster_8java}
\input{_cluster_distance_function_8java}
\input{_compute_correlation_of_replicates_8java}
\input{_correlation_matrix_8java}
\input{_euclidean_distance_8java}
\input{_hierarchical_clustering_8java}
\input{_kmeans_8java}
\input{_pearson_distance_8java}
\input{_rank_order_8java}
\input{_compute_hypergeomtric_enrichment_8java}
\input{_count_nums_8java}
\input{_expression_experiment_info_8java}
\input{_filter_g_c_t_8java}
\input{_filter_g_c_t_file_8java}
\input{gene_exp_utils_8java}
\input{_global_stats_8java}
\input{_grab_specific_columns_8java}
\input{_pa_g_e_java_8java}
\input{_preprocess_data_set_8java}
\input{_score_diff_genes_8java}
\input{_find_enriched_depleted_regions_8java}
\input{_extract_sequence_8java}
\input{_r_n_ai_gene_annotation_8java}
\input{_filter_fastq_reads_8java}
\input{_filter_paired_reads_by_m_a_q_8java}
\input{_paired_end_read_iterator_8java}
\input{_paired_end_t_d_f_8java}
\input{_s_a_m_fragment_writer_8java}
\input{_s_a_m_paired_end_file_reader_8java}
\input{_trim_fastq_reads_8java}
\input{_wrapped_iterator_8java}
\input{_alignment_utils_8java}
\input{_stored_alignment_stats_8java}
\input{_compute_enrichments_8java}
\input{_find_diffrential_enriched_regions_8java}
\input{_peak_filter_8java}
\input{_fasta_extend_peaks_8java}
\input{clip_2_genome_space_sample_data_8java}
\input{protection_2_genome_space_sample_data_8java}
\input{_pileup_8java}
\input{clip_2_sample_data_8java}
\input{protection_2_sample_data_8java}
\input{_fastq_parser_8java}
\input{_fastq_sequence_8java}
\input{_fastq_splitter_by_barcode_8java}
\input{_split_and_trim_index_8java}
\input{_split_fastq_by_index_8java}
\input{_chromosome_with_bubbles_j_graph_t_8java}
\input{_path_8java}
\input{_batched_multi_sample_scan_peak_caller_8java}
\input{_multi_sample_scan_peak_caller_8java}
\input{_read_simulator_8java}
\input{_read_simulator2_8java}
\input{_two_sample_scan_skellam_peak_caller_8java}
\input{_window_scores_8java}
\input{_build_ratio_null_distribution_8java}
\input{_calculate_coverage_stats_8java}
\input{_collect_annotation_enrichments_8java}
\input{_count_probes_8java}
\input{_count_reads_8java}
\input{rna_2_count_reads_8java}
\input{_distance_to_b_e_d_8java}
\input{_g_c_content_8java}
\input{_genome_command_line_program_8java}
\input{_genome_scoring_program_8java}
\input{_plot_aggregate_regions_8java}
\input{_r_a_p_peak_caller_8java}
\input{_ratio_permutation_peak_caller_8java}
\input{_collect_rna_seq_metrics_8java}
\input{_transcriptome_command_line_program_8java}
\input{_transcriptome_scoring_program_8java}
\input{_shuffle_b_a_m_8java}
\input{_shuffle_b_e_d_8java}
\input{_skellam_k_l_peak_caller_8java}
\input{_skellam_score_8java}
\input{_slide_and_calculate_8java}
\input{_slide_and_count_8java}
\input{_unmark_duplicates_8java}
\input{_alignment_data_model_8java}
\input{_alignment_data_model_stats_8java}
\input{_continuous_data_alignment_model_8java}
\input{_define_source_and_sink_from_paired_ends_8java}
\input{_gene_counts_8java}
\input{_generic_alignment_data_model_8java}
\input{_gene_score_8java}
\input{_genome_with_gaps_8java}
\input{_genome_with_gaps2_8java}
\input{_global_splice_filter_8java}
\input{_precompute_data_alignment_stats_8java}
\input{_read_count_normalizer_8java}
\input{_read_filter_8java}
\input{_reconstruction_comparer_8java}
\input{_wrapped_i_g_v_iterator_8java}
\input{_p_s_l_x_to_s_a_m_8java}
\input{_define_gene_signatures_8java}
\input{_extract_genes_8java}
\input{_gene_set_enrichment_8java}
\input{_g_s_a_scoring_8java}
\input{_gsa_utils_8java}
\input{jaligner_2_alignment_8java}
\input{umms_2core_2alignment_2_alignment_8java}
\input{_cell_8java}
\input{_directions_8java}
\input{_c_l_u_s_t_a_l_8java}
\input{_f_a_s_t_a_8java}
\input{_format_8java}
\input{_format_factory_8java}
\input{_markups_8java}
\input{_matrices_compartor_8java}
\input{jaligner_2matrix_2_matrix_8java}
\input{_jama_2_matrix_8java}
\input{_matrix_generator_8java}
\input{_matrix_loader_exception_8java}
\input{_needleman_wunsch_8java}
\input{_needleman_wunsch_gotoh_8java}
\input{_smith_waterman_gotoh_8java}
\input{_traceback_node_8java}
\input{_commons_8java}
\input{_sequence_parser_8java}
\input{_sequence_parser_exception_8java}
\input{_cholesky_decomposition_8java}
\input{_eigenvalue_decomposition_8java}
\input{_l_u_decomposition_8java}
\input{_q_r_decomposition_8java}
\input{_singular_value_decomposition_8java}
\input{_maths_8java}
\input{_abstract_paired_end_alignment_8java}
\input{_alignment_pair_8java}
\input{_chromosome_inconsistency_exception_8java}
\input{_feature_sequence_alignment_8java}
\input{_fragment_alignment_8java}
\input{_paired_end_alignment_factory_8java}
\input{_paired_read_alignment_8java}
\input{_single_end_alignment_8java}
\input{_smatch_like_8java}
\input{_smith_waterman_alignment_8java}
\input{_abstract_annotation_8java}
\input{_abstract_annotation_collection_8java}
\input{_annotation_8java}
\input{_annotation_collection_8java}
\input{_annotation_file_reader_8java}
\input{_annotation_list_8java}
\input{_basic_annotation_8java}
\input{_compound_interval_8java}
\input{_fully_contained_filter_8java}
\input{_overlap_filter_8java}
\input{_gene_8java}
\input{_p_s_l_record_8java}
\input{_single_interval_8java}
\input{_annotation_out_of_bounds_exception_8java}
\input{_coordinate_space_8java}
\input{_genomic_space_8java}
\input{_masked_genomic_space_8java}
\input{_permutation_not_found_exception_8java}
\input{_shuffled_iterator_8java}
\input{_transcript_in_genomic_space_8java}
\input{_transcriptome_space_8java}
\input{_incomplete_method_implementation_exception_8java}
\input{_runtime_i_o_exception_8java}
\input{_gene_window_8java}
\input{_genome_window_8java}
\input{_window_8java}
\input{_closeable_filter_iterator_8java}
\input{_predicates_8java}
\input{_tabbed_reader_8java}
\input{_alignment_model_8java}
\input{_j_c_s_alignment_model_8java}
\input{_abstract_list_score_8java}
\input{_basic_scan_statistic_score_8java}
\input{_binomial_enrichment_score_8java}
\input{_count_score_8java}
\input{_coverage_score_8java}
\input{_end_in_window_count_score_8java}
\input{_length_score_8java}
\input{_multi_score_8java}
\input{_new_ratio_score_8java}
\input{_poisson_enrichment_score_8java}
\input{_ratio_score_8java}
\input{_scan_statistic_score_8java}
\input{_start_in_window_count_score_8java}
\input{_sum_score_8java}
\input{_window_all_score_8java}
\input{_window_processor_8java}
\input{_window_score_8java}
\input{_window_score_iterator_8java}
\input{_paired_end_reader_8java}
\input{_genomic_span_filter_8java}
\input{_mapping_quality_filter_8java}
\input{_paired_and_proper_filter_8java}
\input{_paired_end_filter_8java}
\input{_p_c_r_duplicate_filter_8java}
\input{_read_ends_in_filter_8java}
\input{_read_starts_in_filter_8java}
\input{_reads_to_reconstruct_filter_8java}
\input{_same_orientation_filter_8java}
\input{_spliced_read_filter_8java}
\input{_unique_mapped_reads_filter_8java}
\input{_build_scripture_coordinate_space_8java}
\input{_oriented_chromosome_transcript_graph_8java}
\input{_connect_disconnected_transcripts_8java}
\input{_paired_end_writer_8java}
\input{_collapsed_gene_lengths_8java}
\input{_e_s_a_t_8java}
\input{_e_s_a_t_distance_calculator_8java}
%--- End generated contents ---

% Index
\newpage
\phantomsection
\addcontentsline{toc}{chapter}{Index}
\printindex

\end{document}
