<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=9"/>
<meta name="generator" content="Doxygen 1.8.7"/>
<title>ESAT: Member List</title>
<link href="tabs.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="jquery.js"></script>
<script type="text/javascript" src="dynsections.js"></script>
<link href="search/search.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="search/search.js"></script>
<script type="text/javascript">
  $(document).ready(function() { searchBox.OnSelectItem(0); });
</script>
<link href="doxygen.css" rel="stylesheet" type="text/css" />
</head>
<body>
<div id="top"><!-- do not remove this div, it is closed by doxygen! -->
<div id="titlearea">
<table cellspacing="0" cellpadding="0">
 <tbody>
 <tr style="height: 56px;">
  <td style="padding-left: 0.5em;">
   <div id="projectname">ESAT
   </div>
  </td>
 </tr>
 </tbody>
</table>
</div>
<!-- end header part -->
<!-- Generated by Doxygen 1.8.7 -->
<script type="text/javascript">
var searchBox = new SearchBox("searchBox", "search",false,'Search');
</script>
  <div id="navrow1" class="tabs">
    <ul class="tablist">
      <li><a href="index.html"><span>Main&#160;Page</span></a></li>
      <li><a href="pages.html"><span>Related&#160;Pages</span></a></li>
      <li><a href="namespaces.html"><span>Packages</span></a></li>
      <li class="current"><a href="annotated.html"><span>Classes</span></a></li>
      <li><a href="files.html"><span>Files</span></a></li>
      <li>
        <div id="MSearchBox" class="MSearchBoxInactive">
        <span class="left">
          <img id="MSearchSelect" src="search/mag_sel.png"
               onmouseover="return searchBox.OnSearchSelectShow()"
               onmouseout="return searchBox.OnSearchSelectHide()"
               alt=""/>
          <input type="text" id="MSearchField" value="Search" accesskey="S"
               onfocus="searchBox.OnSearchFieldFocus(true)" 
               onblur="searchBox.OnSearchFieldFocus(false)" 
               onkeyup="searchBox.OnSearchFieldChange(event)"/>
          </span><span class="right">
            <a id="MSearchClose" href="javascript:searchBox.CloseResultsWindow()"><img id="MSearchCloseImg" border="0" src="search/close.png" alt=""/></a>
          </span>
        </div>
      </li>
    </ul>
  </div>
  <div id="navrow2" class="tabs2">
    <ul class="tablist">
      <li><a href="annotated.html"><span>Class&#160;List</span></a></li>
      <li><a href="classes.html"><span>Class&#160;Index</span></a></li>
      <li><a href="inherits.html"><span>Class&#160;Hierarchy</span></a></li>
      <li><a href="functions.html"><span>Class&#160;Members</span></a></li>
    </ul>
  </div>
<!-- window showing the filter options -->
<div id="MSearchSelectWindow"
     onmouseover="return searchBox.OnSearchSelectShow()"
     onmouseout="return searchBox.OnSearchSelectHide()"
     onkeydown="return searchBox.OnSearchSelectKey(event)">
<a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(0)"><span class="SelectionMark">&#160;</span>All</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(1)"><span class="SelectionMark">&#160;</span>Classes</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(2)"><span class="SelectionMark">&#160;</span>Namespaces</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(3)"><span class="SelectionMark">&#160;</span>Files</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(4)"><span class="SelectionMark">&#160;</span>Functions</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(5)"><span class="SelectionMark">&#160;</span>Variables</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(6)"><span class="SelectionMark">&#160;</span>Enumerator</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(7)"><span class="SelectionMark">&#160;</span>Pages</a></div>

<!-- iframe showing the search results (closed by default) -->
<div id="MSearchResultsWindow">
<iframe src="javascript:void(0)" frameborder="0" 
        name="MSearchResults" id="MSearchResults">
</iframe>
</div>

<div id="nav-path" class="navpath">
  <ul>
<li class="navelem"><a class="el" href="namespacebroad.html">broad</a></li><li class="navelem"><a class="el" href="namespacebroad_1_1pda.html">pda</a></li><li class="navelem"><a class="el" href="namespacebroad_1_1pda_1_1seq.html">seq</a></li><li class="navelem"><a class="el" href="namespacebroad_1_1pda_1_1seq_1_1segmentation.html">segmentation</a></li><li class="navelem"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">GenericAlignmentDataModel</a></li>  </ul>
</div>
</div><!-- top -->
<div class="header">
  <div class="headertitle">
<div class="title">broad.pda.seq.segmentation.GenericAlignmentDataModel Member List</div>  </div>
</div><!--header-->
<div class="contents">

<p>This is the complete list of members for <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>, including all inherited members.</p>
<table class="directory">
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a7bc3e012c4981ea00d6314e100cdaf7f">clearFullIntervalTreeAsAlignmentsCached</a>(String chr)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a85bb4ba62f07d9f29d04a0736d0aaa90">countReads</a>(Alignment record)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a30b218ffe2c70e0bdfd2b45de8fc3488">GenericAlignmentDataModel</a>(String filename, String lengthFile, boolean upweightSplices, double minMappingQuality, boolean removeDuplicatesFlag, boolean weighReadCounts, String strand, boolean loadPairsAsFragments)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a1bebfb4c118c9a69e2ba56bb14c7d618">GenericAlignmentDataModel</a>(String filename, String lengthFile, boolean upweightSplices, double minMappingQuality)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a9ca34700cabbaa3686e62cf37036dcdc">GenericAlignmentDataModel</a>(String filename, String lengthFile)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a50aa57688bb19188b77755c54cc1f8a5">GenericAlignmentDataModel</a>(String filename, String lengthFile, int minMappingQuality)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#acdae4c4c918d464beabb81dbdbefb5e1">GenericAlignmentDataModel</a>(String filename, String lengthFile, boolean upweightSplices)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a52ecfee3e2f26fd09ad3900f2fdd2e34">GenericAlignmentDataModel</a>(String filename, String lengthFile, boolean upweightSplices, int minMappingQuality, boolean removeDuplicatesFlag)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#af7fda420f4bea42ab1fb0b1063a75b48">GenericAlignmentDataModel</a>(String filename, String lengthFile, int minMappingQuality, boolean removeDuplicatesFlag)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a9db56d4a5f753f4819e3fc7806bb6de2">GenericAlignmentDataModel</a>(String filename, String lengthFile, boolean removeDuplicatesFlag, boolean weighReadCounts)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a2a80f15f2e55db41c31577789f285d04">GenericAlignmentDataModel</a>(String filename, String lengthFile, boolean upweightSplices, boolean removeDuplicatesFlag, boolean weighReadCounts)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a9f762cd68e9e3e98fad83a1e3732d5b0">GenericAlignmentDataModel</a>(String filename, String lengthFile, boolean upweightSplices, int minMappingQuality, boolean removeDuplicatesFlag, boolean weighReadCounts)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#adac8c2d23887427c364ec8a8bfd27f97">GenericAlignmentDataModel</a>(String filename, String lengthFile, boolean upweightSplices, int minMappingQuality, boolean removeDuplicatesFlag, boolean weighReadCounts, String strand)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a70112d8fb66b00e90e3e1c537ff152ce">getAnnotationOverlappingRegion</a>(Annotation align)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a103697fa11dcbb0127954861e92cc6a9">getBasesCovered</a>(Annotation region, int EF)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a4581ee080760a52c56aa2fec9315367b">getBasesCoveredPerAlignment</a>(Gene gene, IntervalTree&lt; Alignment &gt; tree, int EF)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a02490a8b8be93112f028ad4a6e2f0fed">getChrLength</a>(String chr)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a198d95d9393812ebeba0432f8ca4b840">getChrLengths</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a48905bee11e5ea811f31c97ddeb6901b">getChromosomeLength</a>(String chr)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a8400d889b8c00660927484a5182399d6">getChromosomeLengths</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#aa2afa5c2affb8696efa547372286322f">getChromosomeReadIterator</a>(String chromosome)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a24ff73af0c763387c20a8357c83fd5fc">getChrReadIterator</a>(String chromosome)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#ac3347ac18f32f257dbe64feb8e004663">getChunkEnd</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a0b0c6a88acbace9be14ddf06c5a7953c">getChunkStart</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a211c0c020c8dcba98e1e4aae65572077">getCounts</a>(String chr)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#af11ced151db31ab9e95d75cabeed3615">getCountsOfUniqueOverlappers</a>(Annotation target, Annotation exclude, IntervalTree&lt; Alignment &gt; tree, int EF)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a5fa26ea2bd8c08e3b846edf6c5341b93">getCountsOfUniqueOverlappers</a>(Annotation target, Gene exclude, IntervalTree&lt; Alignment &gt; tree, int EF)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#aa0e60a4a8182d340bd443c08618b0f99">getCountsOnBase</a>(String chr, int index)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a381f6ff8922c20cea08b090aae2b24a3">getCountsOnWindow</a>(String chr, int startIndex, int endIndex)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#aa6cb75b59a4ff7425b03ea133699040c">getCountsPerAlignment</a>(Annotation align, int EF)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a440f4aad672f0f7fdcc89f97743eb13d">getCountsPerAlignment</a>(Annotation align)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#ab125dc0da7450a18ed1c9c05efdc2cb1">getCountsPerAlignment</a>(Annotation align, IntervalTree&lt; Alignment &gt; tree, int EF)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#abdcc9dba34ae9960c29fefd5b631212e">getCountsPerAlignment</a>(Gene gene, IntervalTree&lt; Alignment &gt; tree, int EF)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#addf3cd1305b424740cda98268a4db541">getCountsPerAlignment</a>(Annotation[] Annotation, IntervalTree&lt; Alignment &gt; tree, int EF)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#ae1caac367e554184ec514b839a0cdee1">getCountsPerAlignment</a>(Annotation[] Annotation, int EF)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a1613e249e2e63d9e5a51b96400f67a1e">getCountsPerAlignment</a>(Annotation align, Map&lt; String, IntervalTree&lt; Annotation &gt;&gt; goodExonTree, IntervalTree&lt; Alignment &gt; tree, int extensionFactor)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a569f25df717cd394c8e5258355f75eed">getCountsPerAlignmentForPolyA</a>(Annotation align, int EF)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#aaaf8aeeaf2f81e37ef6949ecf3cff75f">getCountsPerAlignmentFullyContained</a>(Gene gene, IntervalTree&lt; Alignment &gt; tree, int EF)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a73535e3b231500350fcd80a35d53ff6b">getCountsPerAlignmentUncached</a>(String chr, int startIndex, int endIndex)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a90f008b83d90599a72bec79bc7be3dde">getCountsPerAlignmentWithSameEndpointForPolyA</a>(Annotation align, int EF, boolean polyAIsOnRight, int typicalOriginalAlignmentLength, int maxDistanceFromOriginalAlignmentEndpoint)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#aa7f82dd0b8dd66f73c9148b4ec533022">getCountsPerAnnotationtranded</a>(Gene gene, IntervalTree&lt; Alignment &gt; tree, int EF, String orientation)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a96929246b2df0c287be2d8607bc464b4">getCountsWithinExons</a>(Annotation align, IntervalTree&lt; Alignment &gt; tree, int EF)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a1939e58a7f5fc0af87ba5f898b872527">getCountsWithinExons</a>(Annotation align, Iterator&lt; Alignment &gt; iter, int EF)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a951f6057002a5fdcd049130d0d84219f">getExonAnnotationOverlappingRegion</a>(Annotation align)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a86e56d9913a237ac2fc17e354c46f708">getExonAnnotationOverlappingRegion</a>(IntervalTree&lt; Alignment &gt; tree)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a031ef42bb8a258dea08353faad7132ab">getFullIntervalTreeAsAnnotation</a>(String chr)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#aa63431326e357fb7e0bbd28695bfaf18">getGeneCounts</a>(Gene gene, int extensionFactor)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a920d82b5606fe25f56d4075f8119a573">getIntervalTree</a>(String chr, int start, int end)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a5ebeb6683a55b7472d9eca5fbed51375">getIntervalTreeCached</a>(String chr, int start, int end)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a60cddc275b39ab087bb34488651be35d">getIntervalTreeTruncated</a>(String chr, int start, int end)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a211158851e9a4f2ca31f4a556235de13">getIntervalTreeTruncatedCached</a>(String chromosome, int start, int end)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a0e9eeee0305815d3a7ac767e59c76867">getMinimumMappingQuality</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#ad4da7c417bd21633701d18ad20aa4389">getModelFilePath</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a1374d685b7b526b17623120e12012bd7">getNumberOfSplicedReads</a>(Annotation align)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#af225e707111af7c04b07e174bed0a306">getNumberOfSplicedReads</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#ac8781a27f3bba71092316dfbf9583b0b">getOverlappingRegions</a>(String chr)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#abfcc396fc6a74370074144b1ba648520">getPeak</a>(Gene gene, Gene startCodon, IntervalTree&lt; Alignment &gt; tree, int EF)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a74fc9bfd7375f0d90c457fcf9467d464">getReadIterator</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a670fc28ef7e6a79a1b36defb40658274">getReadIterator</a>(Annotation region)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a01edf28e0888aa2d53593fe9fa2ed1dd">getSamHeader</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#af0986112fd4ad0ec8bc544fead171a11">getScorePerAlignmentFromCache</a>(Gene window, IntervalTree&lt; Alignment &gt; chunkAlignmentTree, int extensionFactor)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a8487a4859373b0233fddf580f65af3d8">getSplicedReadExons</a>(String chr)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a6c0827a9ebdbec0ab9de892d1fe4bf03">getSplicedReads</a>(Annotation align)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a10c5e0ff9ad53491189eab3ed4fc7f03">getSplicedReads</a>(Annotation align, final int minIntronSize, final int maxIntronSize, int minNumIntrons, Sequence chrSeq)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#ab257a6d436a758401c55cbd8047c4d3d">getSplicedReads</a>(Annotation align, final int minIntronSize, final int maxIntronSize)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a34454203ec0e39208d65393220cb001b">getSplicedReads</a>(Annotation region, Predicate&lt; Annotation &gt; filter)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#aecacb369e6f69e1c166ced2b45cc80ab">getSplicedReads</a>(Annotation region, Collection&lt; Predicate&lt; Annotation &gt;&gt; filters, int minNumIntrons)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a805775951a0cfcc82f6a3dda04a85975">getSplicedReads</a>(Annotation region, Collection&lt; Predicate&lt; Annotation &gt;&gt; filters, int minNumIntrons, Sequence chrSeq)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a5bd1630184c564e9be0e255a383ea104">getSplicedReads</a>(Annotation region, Collection&lt; Predicate&lt; Annotation &gt;&gt; filters)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a1ba854e11f56bc2c1c8ca0bae29f0f81">getSplicedReads</a>(Annotation region, Collection&lt; Predicate&lt; Annotation &gt;&gt; filters, Sequence chrSeq)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a913396744d34fca92e67a2bf082c41a5">getSplicedReads</a>(int minIntronSize, int maxIntronSize)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#adc6649ca4f14fe2b1fc6f1ce5c716e95">broad::pda::seq::segmentation::AlignmentDataModel.getSplicedReads</a>(Annotation align, int minIntronSize, int maxIntronSize)</td><td class="entry"><a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html">broad.pda.seq.segmentation.AlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#afbcb83cbc75cfd98c8d5a0d444c9d299">getSpliceWeightFactor</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a64c03e96c06f7fbf55f48868b5bbf9de">getTotalNumberOfMappedReads</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a19917c689438c87d9e198f03beaf3da4">getTotalNumberOfMappedStrandedReads</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a96786fd0c7c2c14c2d305af98a1bbb8f">getTotalNumberOfReads</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a283db1d20013b9089f0a56cc642b7a10">getTotalNumberOfStrandedReads</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a21d5bb2538d3f702949c8cb54ac05d72">getWeight</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#aafaa32d7cc1b7d1909a3fccdafd1616b">hasNextExon</a>(String chr)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a77358e51336c799241530fb90e0c1bf3">isNegativeStranded</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#ae494c574fcc812082712b4436f4761f1">isPositiveStranded</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a86e1f6ed98d7f04a2b86a861fc91c8a7">isStranded</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a5875f9097a1103a86034ffef4aa252cf">loadedPairsAsFragments</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a8232fdb916a2bd53811f2d4b82be8c18">passes</a>(Gene gene, IntervalTree&lt; Alignment &gt; tree, int EF, double cutoff)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a85034612037598e915a95f1780cd3e10">passesCutoff</a>(Annotation align, IntervalTree&lt; Alignment &gt; tree, int EF, double threshold)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#ad8e9479d6a969f7038d4c18cbc697fcb">resetTreeCache</a>(String chr)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a1dd1f96e6b2ebf962e29448ada8327c4">resetTreeCache</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#aba1491c9625cddbed80a8f61845cb1f9">restartIterator</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#aad7c99994370aad03b934eb60423656c">setChunkSize</a>(int chunkSize)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a7ff259314b1bd4ec3898bccdad3e7433">setExtensionFactor</a>(int extensionFactor)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a9759d16bc1da489233ea978b698f2de1">setFirstRead</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a90eb0e32ff0c94bf137992c067e8fd38">setMinimumMappingQuality</a>(double minimumMappingQual)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#adebf2567676fec9bf5dda66d8bc2da6d">setNegativeStranded</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#abcb395b283983b86e7f3f3789c275cf8">setNormalizationStrategy</a>(ReadCountNormalizer normalizer)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a361c64731d551b0e40d3f90c35051ed5">setPositiveStranded</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a5ea0c67a41a6d61ae674784525937fd7">setRemoveDuplicatesFlag</a>(boolean flag)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a33f1d19373f11699e701a22fc857712b">setSecondRead</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a789507649e0152371c17a07d4027ad0b">setWeighReadCountsFlag</a>(boolean flag)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr class="even"><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#ad54df59e18f0462b6a9256846b1a363c">unsetStranded</a>()</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
  <tr><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a3130ad4e253569b2e659ed30dc71d9c9">updateGeneByFirstCounts</a>(Gene gene, IntervalTree&lt; Alignment &gt; tree, int EF)</td><td class="entry"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html">broad.pda.seq.segmentation.GenericAlignmentDataModel</a></td><td class="entry"></td></tr>
</table></div><!-- contents -->
<!-- start footer part -->
<hr class="footer"/><address class="footer"><small>
Generated on Fri Aug 15 2014 15:11:04 for ESAT by &#160;<a href="http://www.doxygen.org/index.html">
<img class="footer" src="doxygen.png" alt="doxygen"/>
</a> 1.8.7
</small></address>
</body>
</html>
