<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=9"/>
<meta name="generator" content="Doxygen 1.8.7"/>
<title>ESAT: umms.core.model.JCSAlignmentModel Class Reference</title>
<link href="tabs.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="jquery.js"></script>
<script type="text/javascript" src="dynsections.js"></script>
<link href="search/search.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="search/search.js"></script>
<script type="text/javascript">
  $(document).ready(function() { searchBox.OnSelectItem(0); });
</script>
<link href="doxygen.css" rel="stylesheet" type="text/css" />
</head>
<body>
<div id="top"><!-- do not remove this div, it is closed by doxygen! -->
<div id="titlearea">
<table cellspacing="0" cellpadding="0">
 <tbody>
 <tr style="height: 56px;">
  <td style="padding-left: 0.5em;">
   <div id="projectname">ESAT
   </div>
  </td>
 </tr>
 </tbody>
</table>
</div>
<!-- end header part -->
<!-- Generated by Doxygen 1.8.7 -->
<script type="text/javascript">
var searchBox = new SearchBox("searchBox", "search",false,'Search');
</script>
  <div id="navrow1" class="tabs">
    <ul class="tablist">
      <li><a href="index.html"><span>Main&#160;Page</span></a></li>
      <li><a href="pages.html"><span>Related&#160;Pages</span></a></li>
      <li><a href="namespaces.html"><span>Packages</span></a></li>
      <li class="current"><a href="annotated.html"><span>Classes</span></a></li>
      <li><a href="files.html"><span>Files</span></a></li>
      <li>
        <div id="MSearchBox" class="MSearchBoxInactive">
        <span class="left">
          <img id="MSearchSelect" src="search/mag_sel.png"
               onmouseover="return searchBox.OnSearchSelectShow()"
               onmouseout="return searchBox.OnSearchSelectHide()"
               alt=""/>
          <input type="text" id="MSearchField" value="Search" accesskey="S"
               onfocus="searchBox.OnSearchFieldFocus(true)" 
               onblur="searchBox.OnSearchFieldFocus(false)" 
               onkeyup="searchBox.OnSearchFieldChange(event)"/>
          </span><span class="right">
            <a id="MSearchClose" href="javascript:searchBox.CloseResultsWindow()"><img id="MSearchCloseImg" border="0" src="search/close.png" alt=""/></a>
          </span>
        </div>
      </li>
    </ul>
  </div>
  <div id="navrow2" class="tabs2">
    <ul class="tablist">
      <li><a href="annotated.html"><span>Class&#160;List</span></a></li>
      <li><a href="classes.html"><span>Class&#160;Index</span></a></li>
      <li><a href="inherits.html"><span>Class&#160;Hierarchy</span></a></li>
      <li><a href="functions.html"><span>Class&#160;Members</span></a></li>
    </ul>
  </div>
<!-- window showing the filter options -->
<div id="MSearchSelectWindow"
     onmouseover="return searchBox.OnSearchSelectShow()"
     onmouseout="return searchBox.OnSearchSelectHide()"
     onkeydown="return searchBox.OnSearchSelectKey(event)">
<a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(0)"><span class="SelectionMark">&#160;</span>All</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(1)"><span class="SelectionMark">&#160;</span>Classes</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(2)"><span class="SelectionMark">&#160;</span>Namespaces</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(3)"><span class="SelectionMark">&#160;</span>Files</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(4)"><span class="SelectionMark">&#160;</span>Functions</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(5)"><span class="SelectionMark">&#160;</span>Variables</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(6)"><span class="SelectionMark">&#160;</span>Enumerator</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(7)"><span class="SelectionMark">&#160;</span>Pages</a></div>

<!-- iframe showing the search results (closed by default) -->
<div id="MSearchResultsWindow">
<iframe src="javascript:void(0)" frameborder="0" 
        name="MSearchResults" id="MSearchResults">
</iframe>
</div>

<div id="nav-path" class="navpath">
  <ul>
<li class="navelem"><a class="el" href="namespaceumms.html">umms</a></li><li class="navelem"><a class="el" href="namespaceumms_1_1core.html">core</a></li><li class="navelem"><a class="el" href="namespaceumms_1_1core_1_1model.html">model</a></li><li class="navelem"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html">JCSAlignmentModel</a></li>  </ul>
</div>
</div><!-- top -->
<div class="header">
  <div class="summary">
<a href="#nested-classes">Classes</a> &#124;
<a href="#pub-methods">Public Member Functions</a> &#124;
<a href="#pub-static-methods">Static Public Member Functions</a> &#124;
<a href="#pro-attribs">Protected Attributes</a> &#124;
<a href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model-members.html">List of all members</a>  </div>
  <div class="headertitle">
<div class="title">umms.core.model.JCSAlignmentModel Class Reference</div>  </div>
</div><!--header-->
<div class="contents">
<div class="dynheader">
Inheritance diagram for umms.core.model.JCSAlignmentModel:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model__inherit__graph.png" border="0" usemap="#umms_8core_8model_8_j_c_s_alignment_model_inherit__map" alt="Inheritance graph"/></div>
<map name="umms_8core_8model_8_j_c_s_alignment_model_inherit__map" id="umms_8core_8model_8_j_c_s_alignment_model_inherit__map">
</map>
<center><span class="legend">[<a href="graph_legend.html">legend</a>]</span></center></div>
<div class="dynheader">
Collaboration diagram for umms.core.model.JCSAlignmentModel:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model__coll__graph.png" border="0" usemap="#umms_8core_8model_8_j_c_s_alignment_model_coll__map" alt="Collaboration graph"/></div>
<map name="umms_8core_8model_8_j_c_s_alignment_model_coll__map" id="umms_8core_8model_8_j_c_s_alignment_model_coll__map">
</map>
<center><span class="legend">[<a href="graph_legend.html">legend</a>]</span></center></div>
<table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="nested-classes"></a>
Classes</h2></td></tr>
<tr class="memitem:"><td class="memItemLeft" align="right" valign="top">class &#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_1_1_alignment_count.html">AlignmentCount</a></td></tr>
<tr class="separator:"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:"><td class="memItemLeft" align="right" valign="top">class &#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_1_1_filtered_iterator.html">FilteredIterator</a></td></tr>
<tr class="separator:"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:"><td class="memItemLeft" align="right" valign="top">class &#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_1_1_node_iterator.html">NodeIterator</a></td></tr>
<tr class="separator:"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:"><td class="memItemLeft" align="right" valign="top">class &#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_1_1_wrap_alignment_count_iterator.html">WrapAlignmentCountIterator</a></td></tr>
<tr class="separator:"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table><table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="pub-methods"></a>
Public Member Functions</h2></td></tr>
<tr class="memitem:ac92de131b08dd4133de55ac37f9f2505"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ac92de131b08dd4133de55ac37f9f2505">JCSAlignmentModel</a> (String bamFile, <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a> <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aa04751a0c366de794593489168749245">coordinateSpace</a>, Collection&lt; Predicate&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&gt; readFilters, boolean readOrCreatePairedEndBam)</td></tr>
<tr class="separator:ac92de131b08dd4133de55ac37f9f2505"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a338f7c8478f4737764d86f3c0ff9f15f"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a338f7c8478f4737764d86f3c0ff9f15f">JCSAlignmentModel</a> ()</td></tr>
<tr class="separator:a338f7c8478f4737764d86f3c0ff9f15f"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a68a93aed6e17b615fe3eb3bc2e731ae4"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a68a93aed6e17b615fe3eb3bc2e731ae4">JCSAlignmentModel</a> (String bamFile, <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a> <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aa04751a0c366de794593489168749245">coordinateSpace</a>, Collection&lt; Predicate&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&gt; readFilters, boolean readOrCreatePairedEndBam, TranscriptionRead transcriptionRead)</td></tr>
<tr class="separator:a68a93aed6e17b615fe3eb3bc2e731ae4"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a9ec427083c3bf3d33bd7247d31dc27c4"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a9ec427083c3bf3d33bd7247d31dc27c4">JCSAlignmentModel</a> (String bamFile, <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a> <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aa04751a0c366de794593489168749245">coordinateSpace</a>, Collection&lt; Predicate&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&gt; readFilters, boolean readOrCreatePairedEndBam, TranscriptionRead transcriptionRead, boolean fragment)</td></tr>
<tr class="separator:a9ec427083c3bf3d33bd7247d31dc27c4"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a22e820b0ad7413162f4c590e2e421fa5"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a22e820b0ad7413162f4c590e2e421fa5">JCSAlignmentModel</a> (String bamFile, <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a> <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aa04751a0c366de794593489168749245">coordinateSpace</a>, Collection&lt; Predicate&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&gt; readFilters, boolean readOrCreatePairedEndBam, TranscriptionRead transcriptionRead, String maskedRegionFile)</td></tr>
<tr class="separator:a22e820b0ad7413162f4c590e2e421fa5"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:aa4dc34aa180164bacaba93edc59d21ab"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aa4dc34aa180164bacaba93edc59d21ab">JCSAlignmentModel</a> (String bamFile, <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a> <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aa04751a0c366de794593489168749245">coordinateSpace</a>, Collection&lt; Predicate&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&gt; readFilters, boolean readOrCreatePairedEndBam, TranscriptionRead transcriptionRead, boolean fragment, String maskedRegionFile)</td></tr>
<tr class="separator:aa4dc34aa180164bacaba93edc59d21ab"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a741215ab27a69a4bd736cb039603e917"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a741215ab27a69a4bd736cb039603e917">JCSAlignmentModel</a> (String bamFile, <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a> <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aa04751a0c366de794593489168749245">coordinateSpace</a>, Collection&lt; Predicate&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&gt; readFilters)</td></tr>
<tr class="separator:a741215ab27a69a4bd736cb039603e917"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a50cb20d42d4d51f5625ec32c7f8d9cbe"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a50cb20d42d4d51f5625ec32c7f8d9cbe">JCSAlignmentModel</a> (String bamFile, <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a> <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aa04751a0c366de794593489168749245">coordinateSpace</a>, Collection&lt; Predicate&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&gt; readFilters, TranscriptionRead transcriptionRead)</td></tr>
<tr class="separator:a50cb20d42d4d51f5625ec32c7f8d9cbe"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a0d686d3511221c1bd0d25c877851bc9e"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a0d686d3511221c1bd0d25c877851bc9e">JCSAlignmentModel</a> (String bamFile, <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a> <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aa04751a0c366de794593489168749245">coordinateSpace</a>, boolean readOrCreatePairedEndBam)</td></tr>
<tr class="separator:a0d686d3511221c1bd0d25c877851bc9e"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a9150a136cbc88ddba57213b40e3ac742"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a9150a136cbc88ddba57213b40e3ac742">JCSAlignmentModel</a> (String bamFile, <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a> <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aa04751a0c366de794593489168749245">coordinateSpace</a>, boolean readOrCreatePairedEndBam, TranscriptionRead transcriptionRead)</td></tr>
<tr class="separator:a9150a136cbc88ddba57213b40e3ac742"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:affacfff09298f0a81c61b4e8f483bcd0"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#affacfff09298f0a81c61b4e8f483bcd0">JCSAlignmentModel</a> (String bamFile, <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a> <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aa04751a0c366de794593489168749245">coordinateSpace</a>)</td></tr>
<tr class="separator:affacfff09298f0a81c61b4e8f483bcd0"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a68255203d5baed7c72be12af9d65df2e"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a68255203d5baed7c72be12af9d65df2e">JCSAlignmentModel</a> (String bamFile, <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a> <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aa04751a0c366de794593489168749245">coordinateSpace</a>, TranscriptionRead transcriptionRead)</td></tr>
<tr class="separator:a68255203d5baed7c72be12af9d65df2e"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a8004a75b7c533c9e93db4ddb0fe8f48e"><td class="memItemLeft" align="right" valign="top">SAMFileHeader&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a8004a75b7c533c9e93db4ddb0fe8f48e">getHeader</a> ()</td></tr>
<tr class="separator:a8004a75b7c533c9e93db4ddb0fe8f48e"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a6bba3f6088806514166bc1109d3c6209"><td class="memItemLeft" align="right" valign="top">void&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a6bba3f6088806514166bc1109d3c6209">computeGlobalStats</a> ()</td></tr>
<tr class="separator:a6bba3f6088806514166bc1109d3c6209"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a0d7ac4c1ecb691d9e0207d40fca590f0"><td class="memItemLeft" align="right" valign="top">CloseableIterator&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a0d7ac4c1ecb691d9e0207d40fca590f0">getReadIterator</a> ()</td></tr>
<tr class="separator:a0d7ac4c1ecb691d9e0207d40fca590f0"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ae6c96c699d2e49d813d52eb0c039a277"><td class="memItemLeft" align="right" valign="top">CloseableIterator&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ae6c96c699d2e49d813d52eb0c039a277">getReadIterator</a> (<a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> region)</td></tr>
<tr class="separator:ae6c96c699d2e49d813d52eb0c039a277"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:adee8ec95f5509ff13521ef6d73c9188e"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#adee8ec95f5509ff13521ef6d73c9188e">getRefSequenceCounts</a> (String refName)</td></tr>
<tr class="separator:adee8ec95f5509ff13521ef6d73c9188e"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a135b96822b05a710277803ceb4a3771a"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a135b96822b05a710277803ceb4a3771a">getCount</a> (<a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> window, boolean fullyContained)</td></tr>
<tr class="separator:a135b96822b05a710277803ceb4a3771a"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a59b33d5479e284503ad2fdc5eea86958"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a59b33d5479e284503ad2fdc5eea86958">getCount</a> ()</td></tr>
<tr class="separator:a59b33d5479e284503ad2fdc5eea86958"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a22ec460cb86df8a22b94808295324c82"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a22ec460cb86df8a22b94808295324c82">getCount</a> (CloseableIterator&lt; <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_1_1_alignment_count.html">AlignmentCount</a> &gt; iter)</td></tr>
<tr class="separator:a22ec460cb86df8a22b94808295324c82"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:aca322da8e18f9ef5ceefbf4878c20005"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aca322da8e18f9ef5ceefbf4878c20005">getGlobalLambda</a> ()</td></tr>
<tr class="separator:aca322da8e18f9ef5ceefbf4878c20005"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a201240f2742a96b997444d5ee65575ef"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a201240f2742a96b997444d5ee65575ef">getGlobalLength</a> ()</td></tr>
<tr class="separator:a201240f2742a96b997444d5ee65575ef"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:acf2466e03b14876eba113f1f232b39d2"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#acf2466e03b14876eba113f1f232b39d2">getGlobalPairedFragments</a> ()</td></tr>
<tr class="separator:acf2466e03b14876eba113f1f232b39d2"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ac801ebc73dd4e2fc536214d62cfef81c"><td class="memItemLeft" align="right" valign="top"><a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html">EmpiricalDistribution</a>&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ac801ebc73dd4e2fc536214d62cfef81c">getReadSizeDistribution</a> (<a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a> coord, int maxSize, int numBins)</td></tr>
<tr class="separator:ac801ebc73dd4e2fc536214d62cfef81c"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a70d056fd6cb8a81c5181ec60dfba73eb"><td class="memItemLeft" align="right" valign="top"><a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html">EmpiricalDistribution</a>&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a70d056fd6cb8a81c5181ec60dfba73eb">getReadSizeDistribution</a> (<a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> region, <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a> coord, int maxSize, int numBins)</td></tr>
<tr class="separator:a70d056fd6cb8a81c5181ec60dfba73eb"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a971bf016be8fd846385d0bcf0fe38402"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a971bf016be8fd846385d0bcf0fe38402">getMedianReadSize</a> (<a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> region, <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a> coord, int maxSize, int numBins, boolean fullyContained)</td></tr>
<tr class="separator:a971bf016be8fd846385d0bcf0fe38402"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a687f68337411f48ebbb0aa5b3c3480d1"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a687f68337411f48ebbb0aa5b3c3480d1">getMedianReadSize</a> (<a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> region, <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a> coord, int maxSize, int numBins)</td></tr>
<tr class="separator:a687f68337411f48ebbb0aa5b3c3480d1"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:aa21dba22294a46e58e642e2383e66417"><td class="memItemLeft" align="right" valign="top"><a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html">EmpiricalDistribution</a>&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aa21dba22294a46e58e642e2383e66417">getReadSizeDistribution</a> (<a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> region, <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a> coord, int maxSize, int numBins, boolean fullyContained)</td></tr>
<tr class="separator:aa21dba22294a46e58e642e2383e66417"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a554eedd0b97cb7567223f951394082ea"><td class="memItemLeft" align="right" valign="top">CloseableIterator&lt; <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_1_1_alignment_count.html">AlignmentCount</a> &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a554eedd0b97cb7567223f951394082ea">getOverlappingReadCounts</a> (<a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> region, boolean fullyContained)</td></tr>
<tr class="separator:a554eedd0b97cb7567223f951394082ea"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:af4bf9a81aa77ef353a272037c28f9a37"><td class="memItemLeft" align="right" valign="top">CloseableIterator&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#af4bf9a81aa77ef353a272037c28f9a37">getOverlappingSplicedReads</a> (<a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> region, boolean fullyContained)</td></tr>
<tr class="separator:af4bf9a81aa77ef353a272037c28f9a37"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:af55ede2f0c4eff1338d4dec36ecbbc47"><td class="memItemLeft" align="right" valign="top">CloseableIterator&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#af55ede2f0c4eff1338d4dec36ecbbc47">getOverlappingSplicedReads</a> (String chr)</td></tr>
<tr class="separator:af55ede2f0c4eff1338d4dec36ecbbc47"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ad6006720efa61de32a105415b8f2f15e"><td class="memItemLeft" align="right" valign="top">CloseableIterator&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ad6006720efa61de32a105415b8f2f15e">getOverlappingReads</a> (String chr)</td></tr>
<tr class="separator:ad6006720efa61de32a105415b8f2f15e"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a1ac7103a830ade5c605fe96a00fe7c35"><td class="memItemLeft" align="right" valign="top">int&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a1ac7103a830ade5c605fe96a00fe7c35">getBasesCovered</a> (<a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> region, boolean fullyContained)</td></tr>
<tr class="separator:a1ac7103a830ade5c605fe96a00fe7c35"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a62dbb73b72268c223a90963ff37cc0ad"><td class="memItemLeft" align="right" valign="top">CloseableIterator&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a62dbb73b72268c223a90963ff37cc0ad">getPermutedAnnotations</a> (<a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> region)</td></tr>
<tr class="separator:a62dbb73b72268c223a90963ff37cc0ad"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a307b6ffe46f994204dae7864f70020aa"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a307b6ffe46f994204dae7864f70020aa">getGlobalNumReads</a> ()</td></tr>
<tr class="separator:a307b6ffe46f994204dae7864f70020aa"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a361f481309989ba63027bdfb4fa1ab7c"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a361f481309989ba63027bdfb4fa1ab7c">getGlobalNumReadsReferenceSeqs</a> ()</td></tr>
<tr class="separator:a361f481309989ba63027bdfb4fa1ab7c"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:af00895782a6099bc876a4226499e102d"><td class="memItemLeft" align="right" valign="top">boolean&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#af00895782a6099bc876a4226499e102d">isValid</a> (<a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> read)</td></tr>
<tr class="separator:af00895782a6099bc876a4226499e102d"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a80fd06717e0bda2031fd928406357902"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a80fd06717e0bda2031fd928406357902">getCount</a> (<a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> window)</td></tr>
<tr class="separator:a80fd06717e0bda2031fd928406357902"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:aa16f5a8f31e8a8bbf69119b241b51f28"><td class="memItemLeft" align="right" valign="top">boolean&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aa16f5a8f31e8a8bbf69119b241b51f28">containsReference</a> (String refName)</td></tr>
<tr class="separator:aa16f5a8f31e8a8bbf69119b241b51f28"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a224d0f54b1c732696c3d6397077e08af"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a224d0f54b1c732696c3d6397077e08af">getGlobalCount</a> ()</td></tr>
<tr class="separator:a224d0f54b1c732696c3d6397077e08af"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:acfc0598ca6ce97995a72e76fb5be02bf"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#acfc0598ca6ce97995a72e76fb5be02bf">getGlobalCountReferenceSeqs</a> ()</td></tr>
<tr class="separator:acfc0598ca6ce97995a72e76fb5be02bf"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:aab4abcef1912e2e0640854d32e4e0a1e"><td class="memItemLeft" align="right" valign="top">int&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aab4abcef1912e2e0640854d32e4e0a1e">size</a> ()</td></tr>
<tr class="separator:aab4abcef1912e2e0640854d32e4e0a1e"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:aca7b36716b9a6b085272fa783ce2195e"><td class="memItemLeft" align="right" valign="top">void&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aca7b36716b9a6b085272fa783ce2195e">addFilter</a> (Predicate&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt; filter)</td></tr>
<tr class="separator:aca7b36716b9a6b085272fa783ce2195e"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:afb4faf5978995a95b65f32085a634569"><td class="memItemLeft" align="right" valign="top">void&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#afb4faf5978995a95b65f32085a634569">addFilters</a> (Collection&lt; Predicate&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&gt; filters)</td></tr>
<tr class="separator:afb4faf5978995a95b65f32085a634569"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a2086f9de68630c7104ab1d43fde8d27a"><td class="memItemLeft" align="right" valign="top">WindowScoreIterator&lt; <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_count_score.html">CountScore</a> &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a2086f9de68630c7104ab1d43fde8d27a">scan</a> (<a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> region, int windowSize, int overlap)</td></tr>
<tr class="separator:a2086f9de68630c7104ab1d43fde8d27a"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ad53bee1540d7f533e110b8fdb37d9f23"><td class="memItemLeft" align="right" valign="top">long&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ad53bee1540d7f533e110b8fdb37d9f23">getRefSequenceLength</a> (String refName)</td></tr>
<tr class="separator:ad53bee1540d7f533e110b8fdb37d9f23"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a2dd83544006dcb719cf3f72198734c55"><td class="memItemLeft" align="right" valign="top">Iterator&lt;?extends <a class="el" href="interfaceumms_1_1core_1_1feature_1_1_window.html">Window</a> &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a2dd83544006dcb719cf3f72198734c55">windowIterator</a> (int windowSize, int overlap)</td></tr>
<tr class="separator:a2dd83544006dcb719cf3f72198734c55"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a660ec2a4de55d6ccc1652b33d64f80f3"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a660ec2a4de55d6ccc1652b33d64f80f3">getRefSequenceLambda</a> (String refName)</td></tr>
<tr class="separator:a660ec2a4de55d6ccc1652b33d64f80f3"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a3669cfd3c08b28363c1dabe962489d49"><td class="memItemLeft" align="right" valign="top">Map&lt; String, Integer &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a3669cfd3c08b28363c1dabe962489d49">getRefSequenceLengths</a> ()</td></tr>
<tr class="separator:a3669cfd3c08b28363c1dabe962489d49"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:aca51c172563488f6435e1cb096922c63"><td class="memItemLeft" align="right" valign="top"><a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a>&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aca51c172563488f6435e1cb096922c63">getCoordinateSpace</a> ()</td></tr>
<tr class="separator:aca51c172563488f6435e1cb096922c63"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a28738b736bcc5e5248ea58c39477a6c4"><td class="memItemLeft" align="right" valign="top">String&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a28738b736bcc5e5248ea58c39477a6c4">getBamFile</a> ()</td></tr>
<tr class="separator:a28738b736bcc5e5248ea58c39477a6c4"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:adb5028799b1713ae9d48a8dd64e3ec7c"><td class="memItemLeft" align="right" valign="top">int&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#adb5028799b1713ae9d48a8dd64e3ec7c">getBasesCovered</a> (<a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> region)</td></tr>
<tr class="separator:adb5028799b1713ae9d48a8dd64e3ec7c"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ab8fe2f798ebd01f580f084acd2b532aa"><td class="memItemLeft" align="right" valign="top">void&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ab8fe2f798ebd01f580f084acd2b532aa">removeFilter</a> (Predicate&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt; readFilter)</td></tr>
<tr class="separator:ab8fe2f798ebd01f580f084acd2b532aa"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a1d436b8676a6d0003d8dd57a4b73ed4b"><td class="memItemLeft" align="right" valign="top">int&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a1d436b8676a6d0003d8dd57a4b73ed4b">getIntronCounts</a> (<a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> intron)</td></tr>
<tr class="separator:a1d436b8676a6d0003d8dd57a4b73ed4b"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ac6831a54e791f8a59207c387ebda79cc"><td class="memItemLeft" align="right" valign="top">Map&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a>, Integer &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ac6831a54e791f8a59207c387ebda79cc">getOverlappingReadReplicateCounts</a> (<a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> region, boolean fullyContained)</td></tr>
<tr class="separator:ac6831a54e791f8a59207c387ebda79cc"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:add9b428e55a7f0a7619a08726b4acad5"><td class="memItemLeft" align="right" valign="top">CloseableIterator&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#add9b428e55a7f0a7619a08726b4acad5">getOverlappingReads</a> (<a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> region, boolean fullyContained)</td></tr>
<tr class="separator:add9b428e55a7f0a7619a08726b4acad5"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ace2f29aff75114c5a3cb64c867781006"><td class="memItemLeft" align="right" valign="top"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ace2f29aff75114c5a3cb64c867781006">getPeak</a> (<a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> parent, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> region, boolean fullyContained)</td></tr>
<tr class="separator:ace2f29aff75114c5a3cb64c867781006"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ada32da9c62bd9a077cc24cc9921576da"><td class="memItemLeft" align="right" valign="top">CloseableIterator&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ada32da9c62bd9a077cc24cc9921576da">getOverlappingAnnotations</a> (<a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> region, boolean fullyContained)</td></tr>
<tr class="separator:ada32da9c62bd9a077cc24cc9921576da"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ae41f39ab7b1d13fd6915e1d0405c78a7"><td class="memItemLeft" align="right" valign="top">CloseableIterator&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ae41f39ab7b1d13fd6915e1d0405c78a7">iterator</a> ()</td></tr>
<tr class="separator:ae41f39ab7b1d13fd6915e1d0405c78a7"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a1c7bbf1ce516ae5fca860b72ecebe159"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a1c7bbf1ce516ae5fca860b72ecebe159">getCountExcludingRegion</a> (<a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> region, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> excluded)</td></tr>
<tr class="separator:a1c7bbf1ce516ae5fca860b72ecebe159"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:af17880a4e617b7603bca490f64fa4cce"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#af17880a4e617b7603bca490f64fa4cce">getSplicedCount</a> (<a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_alignments.html">Alignments</a> alignments, boolean fullyContained)</td></tr>
<tr class="separator:af17880a4e617b7603bca490f64fa4cce"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a2e4fdd6d362d1ccd58c61d6c1bbeccb1"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a2e4fdd6d362d1ccd58c61d6c1bbeccb1">getAverageCountPerPositionWithThreshold</a> (<a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html">Gene</a> gene, int minCount)</td></tr>
<tr class="separator:a2e4fdd6d362d1ccd58c61d6c1bbeccb1"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ac64d0167e26483541e58e6794968629d"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ac64d0167e26483541e58e6794968629d">getAverageCountPerPosition</a> (<a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html">Gene</a> gene)</td></tr>
<tr class="separator:ac64d0167e26483541e58e6794968629d"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ab36d1b8f1a85207bbb5f2f4b1dd030d6"><td class="memItemLeft" align="right" valign="top">double[]&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ab36d1b8f1a85207bbb5f2f4b1dd030d6">getCountsPerPosition</a> (<a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html">Gene</a> gene)</td></tr>
<tr class="separator:ab36d1b8f1a85207bbb5f2f4b1dd030d6"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a03d3b9837dce06b17b424dd35e7bd012"><td class="memItemLeft" align="right" valign="top">List&lt; Double &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a03d3b9837dce06b17b424dd35e7bd012">getCountsStrandedPerPosition</a> (<a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> gene)</td></tr>
<tr class="separator:a03d3b9837dce06b17b424dd35e7bd012"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ad96d29e6f7416f7e54e0b44c1e22bfa9"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ad96d29e6f7416f7e54e0b44c1e22bfa9">getCountStrandedExcludingRegion</a> (<a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> region, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> excluded)</td></tr>
<tr class="separator:ad96d29e6f7416f7e54e0b44c1e22bfa9"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a6939ce3bb7ecb8dd46cc30bd0b88c492"><td class="memItemLeft" align="right" valign="top">AlignmentType&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a6939ce3bb7ecb8dd46cc30bd0b88c492">getAlignmentType</a> ()</td></tr>
<tr class="separator:a6939ce3bb7ecb8dd46cc30bd0b88c492"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table><table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="pub-static-methods"></a>
Static Public Member Functions</h2></td></tr>
<tr class="memitem:ac8aef318ba79841fc69adfbe53ebeede"><td class="memItemLeft" align="right" valign="top">static void&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ac8aef318ba79841fc69adfbe53ebeede">main</a> (String[] args)  throws IOException</td></tr>
<tr class="separator:ac8aef318ba79841fc69adfbe53ebeede"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table><table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="pro-attribs"></a>
Protected Attributes</h2></td></tr>
<tr class="memitem:aa04751a0c366de794593489168749245"><td class="memItemLeft" align="right" valign="top"><a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a>&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aa04751a0c366de794593489168749245">coordinateSpace</a></td></tr>
<tr class="separator:aa04751a0c366de794593489168749245"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table>
<a name="details" id="details"></a><h2 class="groupheader">Detailed Description</h2>
<div class="textblock"><p>Serves the purpose of the Generic DataAlignment model and the ContinuousDataAlignmentModel Will populate alignments If a paired end file has unmapped mates, it will be represented by a single end read If both mates are mapped it will be represented as paired end reads Therefore a single model might have both, use flag X to exclude or include only one type This will be a bridge to the analysis modules </p><dl class="section author"><dt>Author</dt><dd>skadri </dd></dl>
</div><h2 class="groupheader">Constructor &amp; Destructor Documentation</h2>
<a class="anchor" id="ac92de131b08dd4133de55ac37f9f2505"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">umms.core.model.JCSAlignmentModel.JCSAlignmentModel </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>bamFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a>&#160;</td>
          <td class="paramname"><em>coordinateSpace</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Collection&lt; Predicate&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&gt;&#160;</td>
          <td class="paramname"><em>readFilters</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>readOrCreatePairedEndBam</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Build with a BAM file Populate the alignment collection </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">bamFile</td><td></td></tr>
    <tr><td class="paramname">coordinateSpace</td><td></td></tr>
    <tr><td class="paramname"><a class="el" href="namespaceumms_1_1core_1_1read_filters.html">readFilters</a></td><td></td></tr>
  </table>
  </dd>
</dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td></td></tr>
  </table>
  </dd>
</dl>

</div>
</div>
<a class="anchor" id="a338f7c8478f4737764d86f3c0ff9f15f"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">umms.core.model.JCSAlignmentModel.JCSAlignmentModel </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a338f7c8478f4737764d86f3c0ff9f15f_icgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a338f7c8478f4737764d86f3c0ff9f15f_icgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a338f7c8478f4737764d86f3c0ff9f15f_icgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a338f7c8478f4737764d86f3c0ff9f15f_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a68a93aed6e17b615fe3eb3bc2e731ae4"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">umms.core.model.JCSAlignmentModel.JCSAlignmentModel </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>bamFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a>&#160;</td>
          <td class="paramname"><em>coordinateSpace</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Collection&lt; Predicate&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&gt;&#160;</td>
          <td class="paramname"><em>readFilters</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>readOrCreatePairedEndBam</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">TranscriptionRead&#160;</td>
          <td class="paramname"><em>transcriptionRead</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Populate the alignment collection </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">bamFile</td><td></td></tr>
    <tr><td class="paramname">coordinateSpace</td><td></td></tr>
    <tr><td class="paramname"><a class="el" href="namespaceumms_1_1core_1_1read_filters.html">readFilters</a></td><td></td></tr>
  </table>
  </dd>
</dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td></td></tr>
  </table>
  </dd>
</dl>

</div>
</div>
<a class="anchor" id="a9ec427083c3bf3d33bd7247d31dc27c4"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">umms.core.model.JCSAlignmentModel.JCSAlignmentModel </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>bamFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a>&#160;</td>
          <td class="paramname"><em>coordinateSpace</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Collection&lt; Predicate&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&gt;&#160;</td>
          <td class="paramname"><em>readFilters</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>readOrCreatePairedEndBam</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">TranscriptionRead&#160;</td>
          <td class="paramname"><em>transcriptionRead</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fragment</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a22e820b0ad7413162f4c590e2e421fa5"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">umms.core.model.JCSAlignmentModel.JCSAlignmentModel </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>bamFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a>&#160;</td>
          <td class="paramname"><em>coordinateSpace</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Collection&lt; Predicate&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&gt;&#160;</td>
          <td class="paramname"><em>readFilters</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>readOrCreatePairedEndBam</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">TranscriptionRead&#160;</td>
          <td class="paramname"><em>transcriptionRead</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>maskedRegionFile</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="aa4dc34aa180164bacaba93edc59d21ab"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">umms.core.model.JCSAlignmentModel.JCSAlignmentModel </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>bamFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a>&#160;</td>
          <td class="paramname"><em>coordinateSpace</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Collection&lt; Predicate&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&gt;&#160;</td>
          <td class="paramname"><em>readFilters</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>readOrCreatePairedEndBam</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">TranscriptionRead&#160;</td>
          <td class="paramname"><em>transcriptionRead</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fragment</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>maskedRegionFile</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Populate the alignment collection </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">bamFile</td><td></td></tr>
    <tr><td class="paramname">coordinateSpace</td><td></td></tr>
    <tr><td class="paramname"><a class="el" href="namespaceumms_1_1core_1_1read_filters.html">readFilters</a></td><td></td></tr>
  </table>
  </dd>
</dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td></td></tr>
  </table>
  </dd>
</dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aa4dc34aa180164bacaba93edc59d21ab_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aa4dc34aa180164bacaba93edc59d21ab_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aa4dc34aa180164bacaba93edc59d21ab_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aa4dc34aa180164bacaba93edc59d21ab_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a741215ab27a69a4bd736cb039603e917"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">umms.core.model.JCSAlignmentModel.JCSAlignmentModel </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>bamFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a>&#160;</td>
          <td class="paramname"><em>coordinateSpace</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Collection&lt; Predicate&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&gt;&#160;</td>
          <td class="paramname"><em>readFilters</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a50cb20d42d4d51f5625ec32c7f8d9cbe"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">umms.core.model.JCSAlignmentModel.JCSAlignmentModel </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>bamFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a>&#160;</td>
          <td class="paramname"><em>coordinateSpace</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Collection&lt; Predicate&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&gt;&#160;</td>
          <td class="paramname"><em>readFilters</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">TranscriptionRead&#160;</td>
          <td class="paramname"><em>transcriptionRead</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a0d686d3511221c1bd0d25c877851bc9e"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">umms.core.model.JCSAlignmentModel.JCSAlignmentModel </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>bamFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a>&#160;</td>
          <td class="paramname"><em>coordinateSpace</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>readOrCreatePairedEndBam</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a9150a136cbc88ddba57213b40e3ac742"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">umms.core.model.JCSAlignmentModel.JCSAlignmentModel </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>bamFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a>&#160;</td>
          <td class="paramname"><em>coordinateSpace</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>readOrCreatePairedEndBam</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">TranscriptionRead&#160;</td>
          <td class="paramname"><em>transcriptionRead</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="affacfff09298f0a81c61b4e8f483bcd0"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">umms.core.model.JCSAlignmentModel.JCSAlignmentModel </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>bamFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a>&#160;</td>
          <td class="paramname"><em>coordinateSpace</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a68255203d5baed7c72be12af9d65df2e"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">umms.core.model.JCSAlignmentModel.JCSAlignmentModel </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>bamFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a>&#160;</td>
          <td class="paramname"><em>coordinateSpace</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">TranscriptionRead&#160;</td>
          <td class="paramname"><em>transcriptionRead</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">

</div>
</div>
<h2 class="groupheader">Member Function Documentation</h2>
<a class="anchor" id="aca7b36716b9a6b085272fa783ce2195e"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">void umms.core.model.JCSAlignmentModel.addFilter </td>
          <td>(</td>
          <td class="paramtype">Predicate&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&#160;</td>
          <td class="paramname"><em>filter</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aca7b36716b9a6b085272fa783ce2195e_icgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aca7b36716b9a6b085272fa783ce2195e_icgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aca7b36716b9a6b085272fa783ce2195e_icgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aca7b36716b9a6b085272fa783ce2195e_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="afb4faf5978995a95b65f32085a634569"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">void umms.core.model.JCSAlignmentModel.addFilters </td>
          <td>(</td>
          <td class="paramtype">Collection&lt; Predicate&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&gt;&#160;</td>
          <td class="paramname"><em>filters</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_afb4faf5978995a95b65f32085a634569_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_afb4faf5978995a95b65f32085a634569_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_afb4faf5978995a95b65f32085a634569_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_afb4faf5978995a95b65f32085a634569_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a6bba3f6088806514166bc1109d3c6209"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">void umms.core.model.JCSAlignmentModel.computeGlobalStats </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Compute global stats and cache to a file for future use, or read from file if already cached </p>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a6bba3f6088806514166bc1109d3c6209_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a6bba3f6088806514166bc1109d3c6209_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a6bba3f6088806514166bc1109d3c6209_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a6bba3f6088806514166bc1109d3c6209_cgraph">
</map>
</div>
</p>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a6bba3f6088806514166bc1109d3c6209_icgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a6bba3f6088806514166bc1109d3c6209_icgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a6bba3f6088806514166bc1109d3c6209_icgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a6bba3f6088806514166bc1109d3c6209_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="aa16f5a8f31e8a8bbf69119b241b51f28"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">boolean umms.core.model.JCSAlignmentModel.containsReference </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>refName</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aa16f5a8f31e8a8bbf69119b241b51f28_icgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aa16f5a8f31e8a8bbf69119b241b51f28_icgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aa16f5a8f31e8a8bbf69119b241b51f28_icgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aa16f5a8f31e8a8bbf69119b241b51f28_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a6939ce3bb7ecb8dd46cc30bd0b88c492"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">AlignmentType umms.core.model.JCSAlignmentModel.getAlignmentType </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Returns the alignment type of the model </p><dl class="section return"><dt>Returns</dt><dd></dd></dl>

</div>
</div>
<a class="anchor" id="ac64d0167e26483541e58e6794968629d"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double umms.core.model.JCSAlignmentModel.getAverageCountPerPosition </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html">Gene</a>&#160;</td>
          <td class="paramname"><em>gene</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get the average position level coverage </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">gene</td><td>The gene </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>The average of all position counts in the gene </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ac64d0167e26483541e58e6794968629d_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ac64d0167e26483541e58e6794968629d_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ac64d0167e26483541e58e6794968629d_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ac64d0167e26483541e58e6794968629d_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a2e4fdd6d362d1ccd58c61d6c1bbeccb1"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double umms.core.model.JCSAlignmentModel.getAverageCountPerPositionWithThreshold </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html">Gene</a>&#160;</td>
          <td class="paramname"><em>gene</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>minCount</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get average position level coverage of only positions with count above a threshold (denominator excludes positions not passing threshold) </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">gene</td><td>The gene </td></tr>
    <tr><td class="paramname">minCount</td><td>Min coverage to include a position </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>The average of all position counts over threshold in the gene </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a2e4fdd6d362d1ccd58c61d6c1bbeccb1_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a2e4fdd6d362d1ccd58c61d6c1bbeccb1_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a2e4fdd6d362d1ccd58c61d6c1bbeccb1_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a2e4fdd6d362d1ccd58c61d6c1bbeccb1_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a28738b736bcc5e5248ea58c39477a6c4"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">String umms.core.model.JCSAlignmentModel.getBamFile </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a1ac7103a830ade5c605fe96a00fe7c35"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">int umms.core.model.JCSAlignmentModel.getBasesCovered </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>region</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fullyContained</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a1ac7103a830ade5c605fe96a00fe7c35_icgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a1ac7103a830ade5c605fe96a00fe7c35_icgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a1ac7103a830ade5c605fe96a00fe7c35_icgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a1ac7103a830ade5c605fe96a00fe7c35_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="adb5028799b1713ae9d48a8dd64e3ec7c"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">int umms.core.model.JCSAlignmentModel.getBasesCovered </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>region</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_adb5028799b1713ae9d48a8dd64e3ec7c_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_adb5028799b1713ae9d48a8dd64e3ec7c_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_adb5028799b1713ae9d48a8dd64e3ec7c_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_adb5028799b1713ae9d48a8dd64e3ec7c_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="aca51c172563488f6435e1cb096922c63"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname"><a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a> umms.core.model.JCSAlignmentModel.getCoordinateSpace </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a135b96822b05a710277803ceb4a3771a"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double umms.core.model.JCSAlignmentModel.getCount </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>window</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fullyContained</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Use the coordinate space to decide what is an overlapping read </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">window</td><td></td></tr>
    <tr><td class="paramname">fullyContained</td><td></td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd></dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a135b96822b05a710277803ceb4a3771a_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a135b96822b05a710277803ceb4a3771a_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a135b96822b05a710277803ceb4a3771a_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a135b96822b05a710277803ceb4a3771a_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a59b33d5479e284503ad2fdc5eea86958"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double umms.core.model.JCSAlignmentModel.getCount </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Use the coordinate space to decide what is an overlapping read Gets the count over the entire genome </p><dl class="section return"><dt>Returns</dt><dd></dd></dl>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a59b33d5479e284503ad2fdc5eea86958_icgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a59b33d5479e284503ad2fdc5eea86958_icgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a59b33d5479e284503ad2fdc5eea86958_icgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a59b33d5479e284503ad2fdc5eea86958_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a22ec460cb86df8a22b94808295324c82"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double umms.core.model.JCSAlignmentModel.getCount </td>
          <td>(</td>
          <td class="paramtype">CloseableIterator&lt; <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_1_1_alignment_count.html">AlignmentCount</a> &gt;&#160;</td>
          <td class="paramname"><em>iter</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a80fd06717e0bda2031fd928406357902"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double umms.core.model.JCSAlignmentModel.getCount </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>window</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a80fd06717e0bda2031fd928406357902_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a80fd06717e0bda2031fd928406357902_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a80fd06717e0bda2031fd928406357902_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a80fd06717e0bda2031fd928406357902_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a1c7bbf1ce516ae5fca860b72ecebe159"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double umms.core.model.JCSAlignmentModel.getCountExcludingRegion </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>region</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>excluded</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a1c7bbf1ce516ae5fca860b72ecebe159_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a1c7bbf1ce516ae5fca860b72ecebe159_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a1c7bbf1ce516ae5fca860b72ecebe159_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a1c7bbf1ce516ae5fca860b72ecebe159_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="ab36d1b8f1a85207bbb5f2f4b1dd030d6"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double [] umms.core.model.JCSAlignmentModel.getCountsPerPosition </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html">Gene</a>&#160;</td>
          <td class="paramname"><em>gene</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>This function returns an array list of counts for each position along the specified gene </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">gene</td><td></td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd></dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ab36d1b8f1a85207bbb5f2f4b1dd030d6_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ab36d1b8f1a85207bbb5f2f4b1dd030d6_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ab36d1b8f1a85207bbb5f2f4b1dd030d6_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ab36d1b8f1a85207bbb5f2f4b1dd030d6_cgraph">
</map>
</div>
</p>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ab36d1b8f1a85207bbb5f2f4b1dd030d6_icgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ab36d1b8f1a85207bbb5f2f4b1dd030d6_icgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ab36d1b8f1a85207bbb5f2f4b1dd030d6_icgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ab36d1b8f1a85207bbb5f2f4b1dd030d6_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a03d3b9837dce06b17b424dd35e7bd012"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">List&lt;Double&gt; umms.core.model.JCSAlignmentModel.getCountsStrandedPerPosition </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>gene</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>This function returns an array list of counts for each position along the specified gene </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">gene</td><td></td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd></dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a03d3b9837dce06b17b424dd35e7bd012_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a03d3b9837dce06b17b424dd35e7bd012_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a03d3b9837dce06b17b424dd35e7bd012_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a03d3b9837dce06b17b424dd35e7bd012_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="ad96d29e6f7416f7e54e0b44c1e22bfa9"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double umms.core.model.JCSAlignmentModel.getCountStrandedExcludingRegion </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>region</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>excluded</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ad96d29e6f7416f7e54e0b44c1e22bfa9_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ad96d29e6f7416f7e54e0b44c1e22bfa9_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ad96d29e6f7416f7e54e0b44c1e22bfa9_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ad96d29e6f7416f7e54e0b44c1e22bfa9_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a224d0f54b1c732696c3d6397077e08af"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double umms.core.model.JCSAlignmentModel.getGlobalCount </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a224d0f54b1c732696c3d6397077e08af_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a224d0f54b1c732696c3d6397077e08af_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a224d0f54b1c732696c3d6397077e08af_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a224d0f54b1c732696c3d6397077e08af_cgraph">
</map>
</div>
</p>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a224d0f54b1c732696c3d6397077e08af_icgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a224d0f54b1c732696c3d6397077e08af_icgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a224d0f54b1c732696c3d6397077e08af_icgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a224d0f54b1c732696c3d6397077e08af_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="acfc0598ca6ce97995a72e76fb5be02bf"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double umms.core.model.JCSAlignmentModel.getGlobalCountReferenceSeqs </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_acfc0598ca6ce97995a72e76fb5be02bf_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_acfc0598ca6ce97995a72e76fb5be02bf_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_acfc0598ca6ce97995a72e76fb5be02bf_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_acfc0598ca6ce97995a72e76fb5be02bf_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="aca322da8e18f9ef5ceefbf4878c20005"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double umms.core.model.JCSAlignmentModel.getGlobalLambda </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aca322da8e18f9ef5ceefbf4878c20005_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aca322da8e18f9ef5ceefbf4878c20005_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aca322da8e18f9ef5ceefbf4878c20005_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aca322da8e18f9ef5ceefbf4878c20005_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a201240f2742a96b997444d5ee65575ef"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double umms.core.model.JCSAlignmentModel.getGlobalLength </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a201240f2742a96b997444d5ee65575ef_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a201240f2742a96b997444d5ee65575ef_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a201240f2742a96b997444d5ee65575ef_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a201240f2742a96b997444d5ee65575ef_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a307b6ffe46f994204dae7864f70020aa"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double umms.core.model.JCSAlignmentModel.getGlobalNumReads </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get total number of reads </p><dl class="section return"><dt>Returns</dt><dd>Total read count </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a307b6ffe46f994204dae7864f70020aa_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a307b6ffe46f994204dae7864f70020aa_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a307b6ffe46f994204dae7864f70020aa_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a307b6ffe46f994204dae7864f70020aa_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a361f481309989ba63027bdfb4fa1ab7c"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double umms.core.model.JCSAlignmentModel.getGlobalNumReadsReferenceSeqs </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get total number of reads mapping to the coordinate space </p><dl class="section return"><dt>Returns</dt><dd>Total read count on coordinate space </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a361f481309989ba63027bdfb4fa1ab7c_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a361f481309989ba63027bdfb4fa1ab7c_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a361f481309989ba63027bdfb4fa1ab7c_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a361f481309989ba63027bdfb4fa1ab7c_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="acf2466e03b14876eba113f1f232b39d2"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double umms.core.model.JCSAlignmentModel.getGlobalPairedFragments </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_acf2466e03b14876eba113f1f232b39d2_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_acf2466e03b14876eba113f1f232b39d2_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_acf2466e03b14876eba113f1f232b39d2_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_acf2466e03b14876eba113f1f232b39d2_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a8004a75b7c533c9e93db4ddb0fe8f48e"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">SAMFileHeader umms.core.model.JCSAlignmentModel.getHeader </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Returns the SAM file header for this reader </p><dl class="section return"><dt>Returns</dt><dd></dd></dl>

</div>
</div>
<a class="anchor" id="a1d436b8676a6d0003d8dd57a4b73ed4b"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">int umms.core.model.JCSAlignmentModel.getIntronCounts </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>intron</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>This will count the number of spliced reads with exactly this intron </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">intron</td><td>the intron to count </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd></dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a1d436b8676a6d0003d8dd57a4b73ed4b_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a1d436b8676a6d0003d8dd57a4b73ed4b_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a1d436b8676a6d0003d8dd57a4b73ed4b_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a1d436b8676a6d0003d8dd57a4b73ed4b_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a971bf016be8fd846385d0bcf0fe38402"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double umms.core.model.JCSAlignmentModel.getMedianReadSize </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>region</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a>&#160;</td>
          <td class="paramname"><em>coord</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>maxSize</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>numBins</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fullyContained</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get the median of all read (or fragment) sizes in an annotation </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">region</td><td>The annotation in which to count read sizes </td></tr>
    <tr><td class="paramname">coord</td><td>Coordinate space in which to compute read sizes </td></tr>
    <tr><td class="paramname">maxSize</td><td>The maximum read size to check for </td></tr>
    <tr><td class="paramname">numBins</td><td>The number of bins for histogram </td></tr>
    <tr><td class="paramname">fullyContained</td><td>Count fully contained reads/fragments only </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Distribution of read sizes with respect to coordinate space </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a971bf016be8fd846385d0bcf0fe38402_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a971bf016be8fd846385d0bcf0fe38402_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a971bf016be8fd846385d0bcf0fe38402_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a971bf016be8fd846385d0bcf0fe38402_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a687f68337411f48ebbb0aa5b3c3480d1"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double umms.core.model.JCSAlignmentModel.getMedianReadSize </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>region</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a>&#160;</td>
          <td class="paramname"><em>coord</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>maxSize</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>numBins</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get the median of all read (or fragment) sizes in an annotation </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">region</td><td>The annotation in which to count read sizes </td></tr>
    <tr><td class="paramname">coord</td><td>Coordinate space in which to compute read sizes </td></tr>
    <tr><td class="paramname">maxSize</td><td>The maximum read size to check for </td></tr>
    <tr><td class="paramname">numBins</td><td>The number of bins for histogram </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Distribution of read sizes with respect to coordinate space </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a687f68337411f48ebbb0aa5b3c3480d1_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a687f68337411f48ebbb0aa5b3c3480d1_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a687f68337411f48ebbb0aa5b3c3480d1_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a687f68337411f48ebbb0aa5b3c3480d1_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="ada32da9c62bd9a077cc24cc9921576da"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">CloseableIterator&lt;<a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a>&gt; umms.core.model.JCSAlignmentModel.getOverlappingAnnotations </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>region</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fullyContained</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a554eedd0b97cb7567223f951394082ea"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">CloseableIterator&lt;<a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_1_1_alignment_count.html">AlignmentCount</a>&gt; umms.core.model.JCSAlignmentModel.getOverlappingReadCounts </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>region</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fullyContained</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Return the reads that overlap with this region in coordinate space If the Alignment model has a strand which is NOT unstranded, then returns stranded counts </p>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a554eedd0b97cb7567223f951394082ea_icgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a554eedd0b97cb7567223f951394082ea_icgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a554eedd0b97cb7567223f951394082ea_icgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a554eedd0b97cb7567223f951394082ea_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="ac6831a54e791f8a59207c387ebda79cc"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">Map&lt;<a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a>, Integer&gt; umms.core.model.JCSAlignmentModel.getOverlappingReadReplicateCounts </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>region</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fullyContained</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get number of replicates of each read overlapping the region </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">region</td><td>The region </td></tr>
    <tr><td class="paramname">fullyContained</td><td>Count fully contained reads only </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>For each unique read position, the number of replicates </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ac6831a54e791f8a59207c387ebda79cc_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ac6831a54e791f8a59207c387ebda79cc_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ac6831a54e791f8a59207c387ebda79cc_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ac6831a54e791f8a59207c387ebda79cc_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="ad6006720efa61de32a105415b8f2f15e"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">CloseableIterator&lt;<a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a>&gt; umms.core.model.JCSAlignmentModel.getOverlappingReads </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>chr</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ad6006720efa61de32a105415b8f2f15e_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ad6006720efa61de32a105415b8f2f15e_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ad6006720efa61de32a105415b8f2f15e_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ad6006720efa61de32a105415b8f2f15e_cgraph">
</map>
</div>
</p>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ad6006720efa61de32a105415b8f2f15e_icgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ad6006720efa61de32a105415b8f2f15e_icgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ad6006720efa61de32a105415b8f2f15e_icgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ad6006720efa61de32a105415b8f2f15e_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="add9b428e55a7f0a7619a08726b4acad5"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">CloseableIterator&lt;<a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a>&gt; umms.core.model.JCSAlignmentModel.getOverlappingReads </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>region</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fullyContained</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_add9b428e55a7f0a7619a08726b4acad5_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_add9b428e55a7f0a7619a08726b4acad5_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_add9b428e55a7f0a7619a08726b4acad5_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_add9b428e55a7f0a7619a08726b4acad5_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="af4bf9a81aa77ef353a272037c28f9a37"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">CloseableIterator&lt;<a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a>&gt; umms.core.model.JCSAlignmentModel.getOverlappingSplicedReads </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>region</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fullyContained</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Return the reads that overlap with this region in coordinate space </p>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_af4bf9a81aa77ef353a272037c28f9a37_icgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_af4bf9a81aa77ef353a272037c28f9a37_icgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_af4bf9a81aa77ef353a272037c28f9a37_icgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_af4bf9a81aa77ef353a272037c28f9a37_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="af55ede2f0c4eff1338d4dec36ecbbc47"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">CloseableIterator&lt;<a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a>&gt; umms.core.model.JCSAlignmentModel.getOverlappingSplicedReads </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>chr</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Return the reads that overlap with this region in coordinate space </p>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_af55ede2f0c4eff1338d4dec36ecbbc47_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_af55ede2f0c4eff1338d4dec36ecbbc47_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_af55ede2f0c4eff1338d4dec36ecbbc47_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_af55ede2f0c4eff1338d4dec36ecbbc47_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="ace2f29aff75114c5a3cb64c867781006"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a> umms.core.model.JCSAlignmentModel.getPeak </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>parent</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>region</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fullyContained</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get the span covered by all reads overlapping a region </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">parent</td><td>Parent annotation of the region of interest </td></tr>
    <tr><td class="paramname">region</td><td>The region of interest </td></tr>
    <tr><td class="paramname">fullyContained</td><td>Fully contained reads only </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>The subregion of the parent gene enclosing all reads overlapping the region or null if no overlappers </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ace2f29aff75114c5a3cb64c867781006_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ace2f29aff75114c5a3cb64c867781006_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ace2f29aff75114c5a3cb64c867781006_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ace2f29aff75114c5a3cb64c867781006_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a62dbb73b72268c223a90963ff37cc0ad"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">CloseableIterator&lt;<a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a>&gt; umms.core.model.JCSAlignmentModel.getPermutedAnnotations </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>region</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a62dbb73b72268c223a90963ff37cc0ad_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a62dbb73b72268c223a90963ff37cc0ad_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a62dbb73b72268c223a90963ff37cc0ad_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a62dbb73b72268c223a90963ff37cc0ad_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a0d7ac4c1ecb691d9e0207d40fca590f0"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">CloseableIterator&lt;<a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a>&gt; umms.core.model.JCSAlignmentModel.getReadIterator </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a0d7ac4c1ecb691d9e0207d40fca590f0_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a0d7ac4c1ecb691d9e0207d40fca590f0_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a0d7ac4c1ecb691d9e0207d40fca590f0_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a0d7ac4c1ecb691d9e0207d40fca590f0_cgraph">
</map>
</div>
</p>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a0d7ac4c1ecb691d9e0207d40fca590f0_icgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a0d7ac4c1ecb691d9e0207d40fca590f0_icgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a0d7ac4c1ecb691d9e0207d40fca590f0_icgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a0d7ac4c1ecb691d9e0207d40fca590f0_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="ae6c96c699d2e49d813d52eb0c039a277"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">CloseableIterator&lt;<a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a>&gt; umms.core.model.JCSAlignmentModel.getReadIterator </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>region</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ae6c96c699d2e49d813d52eb0c039a277_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ae6c96c699d2e49d813d52eb0c039a277_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ae6c96c699d2e49d813d52eb0c039a277_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ae6c96c699d2e49d813d52eb0c039a277_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="ac801ebc73dd4e2fc536214d62cfef81c"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname"><a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html">EmpiricalDistribution</a> umms.core.model.JCSAlignmentModel.getReadSizeDistribution </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a>&#160;</td>
          <td class="paramname"><em>coord</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>maxSize</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>numBins</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get the distribution of all read (or fragment) sizes with respect to a coordinate space </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">coord</td><td>The coordinate space </td></tr>
    <tr><td class="paramname">maxSize</td><td>The maximum read size to consider </td></tr>
    <tr><td class="paramname">numBins</td><td>The number of bins for histogram </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Distribution of read sizes with respect to coordinate space </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ac801ebc73dd4e2fc536214d62cfef81c_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ac801ebc73dd4e2fc536214d62cfef81c_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ac801ebc73dd4e2fc536214d62cfef81c_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ac801ebc73dd4e2fc536214d62cfef81c_cgraph">
</map>
</div>
</p>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ac801ebc73dd4e2fc536214d62cfef81c_icgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ac801ebc73dd4e2fc536214d62cfef81c_icgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ac801ebc73dd4e2fc536214d62cfef81c_icgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ac801ebc73dd4e2fc536214d62cfef81c_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a70d056fd6cb8a81c5181ec60dfba73eb"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname"><a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html">EmpiricalDistribution</a> umms.core.model.JCSAlignmentModel.getReadSizeDistribution </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>region</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a>&#160;</td>
          <td class="paramname"><em>coord</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>maxSize</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>numBins</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get the distribution of read (or fragment) sizes in an annotation </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">region</td><td>The annotation in which to count read sizes </td></tr>
    <tr><td class="paramname">coord</td><td>Coordinate space in which to compute read sizes </td></tr>
    <tr><td class="paramname">maxSize</td><td>The maximum read size to check for </td></tr>
    <tr><td class="paramname">numBins</td><td>The number of bins for histogram </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Distribution of read sizes with respect to coordinate space </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a70d056fd6cb8a81c5181ec60dfba73eb_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a70d056fd6cb8a81c5181ec60dfba73eb_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a70d056fd6cb8a81c5181ec60dfba73eb_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a70d056fd6cb8a81c5181ec60dfba73eb_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="aa21dba22294a46e58e642e2383e66417"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname"><a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html">EmpiricalDistribution</a> umms.core.model.JCSAlignmentModel.getReadSizeDistribution </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>region</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a>&#160;</td>
          <td class="paramname"><em>coord</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>maxSize</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>numBins</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fullyContained</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get the distribution of read (or fragment) sizes in an annotation </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">region</td><td>The annotation in which to count read sizes </td></tr>
    <tr><td class="paramname">coord</td><td>Coordinate space in which to compute read sizes </td></tr>
    <tr><td class="paramname">maxSize</td><td>The maximum read size to check for </td></tr>
    <tr><td class="paramname">numBins</td><td>The number of bins for histogram </td></tr>
    <tr><td class="paramname">fullyContained</td><td>Count fully contained reads/fragments only </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Distribution of read sizes with respect to coordinate space </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aa21dba22294a46e58e642e2383e66417_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aa21dba22294a46e58e642e2383e66417_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aa21dba22294a46e58e642e2383e66417_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aa21dba22294a46e58e642e2383e66417_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="adee8ec95f5509ff13521ef6d73c9188e"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double umms.core.model.JCSAlignmentModel.getRefSequenceCounts </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>refName</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>This function calculates the number of proper paired end reads over the entire coordinate space </p><dl class="section author"><dt>Author</dt><dd>skadri </dd></dl>
<dl class="section return"><dt>Returns</dt><dd>Returns the counts for the specified reference sequence </dd></dl>
<dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">refName</td><td></td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd></dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_adee8ec95f5509ff13521ef6d73c9188e_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_adee8ec95f5509ff13521ef6d73c9188e_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_adee8ec95f5509ff13521ef6d73c9188e_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_adee8ec95f5509ff13521ef6d73c9188e_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a660ec2a4de55d6ccc1652b33d64f80f3"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double umms.core.model.JCSAlignmentModel.getRefSequenceLambda </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>refName</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a660ec2a4de55d6ccc1652b33d64f80f3_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a660ec2a4de55d6ccc1652b33d64f80f3_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a660ec2a4de55d6ccc1652b33d64f80f3_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a660ec2a4de55d6ccc1652b33d64f80f3_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="ad53bee1540d7f533e110b8fdb37d9f23"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">long umms.core.model.JCSAlignmentModel.getRefSequenceLength </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>refName</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ad53bee1540d7f533e110b8fdb37d9f23_icgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ad53bee1540d7f533e110b8fdb37d9f23_icgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ad53bee1540d7f533e110b8fdb37d9f23_icgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ad53bee1540d7f533e110b8fdb37d9f23_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a3669cfd3c08b28363c1dabe962489d49"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">Map&lt;String, Integer&gt; umms.core.model.JCSAlignmentModel.getRefSequenceLengths </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get the lengths of the reference sequences </p><dl class="section return"><dt>Returns</dt><dd>Map associating each reference name with sequence length </dd></dl>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a3669cfd3c08b28363c1dabe962489d49_icgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a3669cfd3c08b28363c1dabe962489d49_icgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a3669cfd3c08b28363c1dabe962489d49_icgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_a3669cfd3c08b28363c1dabe962489d49_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="af17880a4e617b7603bca490f64fa4cce"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double umms.core.model.JCSAlignmentModel.getSplicedCount </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_alignments.html">Alignments</a>&#160;</td>
          <td class="paramname"><em>alignments</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fullyContained</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_af17880a4e617b7603bca490f64fa4cce_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_af17880a4e617b7603bca490f64fa4cce_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_af17880a4e617b7603bca490f64fa4cce_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_af17880a4e617b7603bca490f64fa4cce_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="af00895782a6099bc876a4226499e102d"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">boolean umms.core.model.JCSAlignmentModel.isValid </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a>&#160;</td>
          <td class="paramname"><em>read</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="ae41f39ab7b1d13fd6915e1d0405c78a7"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">CloseableIterator&lt;<a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a>&gt; umms.core.model.JCSAlignmentModel.iterator </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="ac8aef318ba79841fc69adfbe53ebeede"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">static void umms.core.model.JCSAlignmentModel.main </td>
          <td>(</td>
          <td class="paramtype">String[]&#160;</td>
          <td class="paramname"><em>args</em></td><td>)</td>
          <td> throws IOException</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">static</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ac8aef318ba79841fc69adfbe53ebeede_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ac8aef318ba79841fc69adfbe53ebeede_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ac8aef318ba79841fc69adfbe53ebeede_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_ac8aef318ba79841fc69adfbe53ebeede_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="ab8fe2f798ebd01f580f084acd2b532aa"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">void umms.core.model.JCSAlignmentModel.removeFilter </td>
          <td>(</td>
          <td class="paramtype">Predicate&lt; <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html">Alignment</a> &gt;&#160;</td>
          <td class="paramname"><em>readFilter</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Removes a read filter from the set </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">readFilter</td><td>to remove </td></tr>
  </table>
  </dd>
</dl>

</div>
</div>
<a class="anchor" id="a2086f9de68630c7104ab1d43fde8d27a"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">WindowScoreIterator&lt;<a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_count_score.html">CountScore</a>&gt; umms.core.model.JCSAlignmentModel.scan </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html">Annotation</a>&#160;</td>
          <td class="paramname"><em>region</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>windowSize</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>overlap</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="aab4abcef1912e2e0640854d32e4e0a1e"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">int umms.core.model.JCSAlignmentModel.size </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aab4abcef1912e2e0640854d32e4e0a1e_cgraph.png" border="0" usemap="#classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aab4abcef1912e2e0640854d32e4e0a1e_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aab4abcef1912e2e0640854d32e4e0a1e_cgraph" id="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_aab4abcef1912e2e0640854d32e4e0a1e_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a2dd83544006dcb719cf3f72198734c55"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">Iterator&lt;? extends <a class="el" href="interfaceumms_1_1core_1_1feature_1_1_window.html">Window</a>&gt; umms.core.model.JCSAlignmentModel.windowIterator </td>
          <td>(</td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>windowSize</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>overlap</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Iterate through the whole coordinate space in windows and score them using the provided window processor </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">windowSize</td><td>size of the windows to score </td></tr>
  </table>
  </dd>
</dl>

</div>
</div>
<h2 class="groupheader">Member Data Documentation</h2>
<a class="anchor" id="aa04751a0c366de794593489168749245"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname"><a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html">CoordinateSpace</a> umms.core.model.JCSAlignmentModel.coordinateSpace</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<hr/>The documentation for this class was generated from the following file:<ul>
<li>java/umms/core/model/<a class="el" href="_j_c_s_alignment_model_8java.html">JCSAlignmentModel.java</a></li>
</ul>
</div><!-- contents -->
<!-- start footer part -->
<hr class="footer"/><address class="footer"><small>
Generated on Fri Aug 15 2014 15:11:11 for ESAT by &#160;<a href="http://www.doxygen.org/index.html">
<img class="footer" src="doxygen.png" alt="doxygen"/>
</a> 1.8.7
</small></address>
</body>
</html>
