<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=9"/>
<meta name="generator" content="Doxygen 1.8.7"/>
<title>ESAT: broad.pda.seq.alignment.AlignmentUtils Class Reference</title>
<link href="tabs.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="jquery.js"></script>
<script type="text/javascript" src="dynsections.js"></script>
<link href="search/search.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="search/search.js"></script>
<script type="text/javascript">
  $(document).ready(function() { searchBox.OnSelectItem(0); });
</script>
<link href="doxygen.css" rel="stylesheet" type="text/css" />
</head>
<body>
<div id="top"><!-- do not remove this div, it is closed by doxygen! -->
<div id="titlearea">
<table cellspacing="0" cellpadding="0">
 <tbody>
 <tr style="height: 56px;">
  <td style="padding-left: 0.5em;">
   <div id="projectname">ESAT
   </div>
  </td>
 </tr>
 </tbody>
</table>
</div>
<!-- end header part -->
<!-- Generated by Doxygen 1.8.7 -->
<script type="text/javascript">
var searchBox = new SearchBox("searchBox", "search",false,'Search');
</script>
  <div id="navrow1" class="tabs">
    <ul class="tablist">
      <li><a href="index.html"><span>Main&#160;Page</span></a></li>
      <li><a href="pages.html"><span>Related&#160;Pages</span></a></li>
      <li><a href="namespaces.html"><span>Packages</span></a></li>
      <li class="current"><a href="annotated.html"><span>Classes</span></a></li>
      <li><a href="files.html"><span>Files</span></a></li>
      <li>
        <div id="MSearchBox" class="MSearchBoxInactive">
        <span class="left">
          <img id="MSearchSelect" src="search/mag_sel.png"
               onmouseover="return searchBox.OnSearchSelectShow()"
               onmouseout="return searchBox.OnSearchSelectHide()"
               alt=""/>
          <input type="text" id="MSearchField" value="Search" accesskey="S"
               onfocus="searchBox.OnSearchFieldFocus(true)" 
               onblur="searchBox.OnSearchFieldFocus(false)" 
               onkeyup="searchBox.OnSearchFieldChange(event)"/>
          </span><span class="right">
            <a id="MSearchClose" href="javascript:searchBox.CloseResultsWindow()"><img id="MSearchCloseImg" border="0" src="search/close.png" alt=""/></a>
          </span>
        </div>
      </li>
    </ul>
  </div>
  <div id="navrow2" class="tabs2">
    <ul class="tablist">
      <li><a href="annotated.html"><span>Class&#160;List</span></a></li>
      <li><a href="classes.html"><span>Class&#160;Index</span></a></li>
      <li><a href="inherits.html"><span>Class&#160;Hierarchy</span></a></li>
      <li><a href="functions.html"><span>Class&#160;Members</span></a></li>
    </ul>
  </div>
<!-- window showing the filter options -->
<div id="MSearchSelectWindow"
     onmouseover="return searchBox.OnSearchSelectShow()"
     onmouseout="return searchBox.OnSearchSelectHide()"
     onkeydown="return searchBox.OnSearchSelectKey(event)">
<a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(0)"><span class="SelectionMark">&#160;</span>All</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(1)"><span class="SelectionMark">&#160;</span>Classes</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(2)"><span class="SelectionMark">&#160;</span>Namespaces</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(3)"><span class="SelectionMark">&#160;</span>Files</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(4)"><span class="SelectionMark">&#160;</span>Functions</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(5)"><span class="SelectionMark">&#160;</span>Variables</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(6)"><span class="SelectionMark">&#160;</span>Enumerator</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(7)"><span class="SelectionMark">&#160;</span>Pages</a></div>

<!-- iframe showing the search results (closed by default) -->
<div id="MSearchResultsWindow">
<iframe src="javascript:void(0)" frameborder="0" 
        name="MSearchResults" id="MSearchResults">
</iframe>
</div>

<div id="nav-path" class="navpath">
  <ul>
<li class="navelem"><a class="el" href="namespacebroad.html">broad</a></li><li class="navelem"><a class="el" href="namespacebroad_1_1pda.html">pda</a></li><li class="navelem"><a class="el" href="namespacebroad_1_1pda_1_1seq.html">seq</a></li><li class="navelem"><a class="el" href="namespacebroad_1_1pda_1_1seq_1_1alignment.html">alignment</a></li><li class="navelem"><a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils.html">AlignmentUtils</a></li>  </ul>
</div>
</div><!-- top -->
<div class="header">
  <div class="summary">
<a href="#pub-static-methods">Static Public Member Functions</a> &#124;
<a href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils-members.html">List of all members</a>  </div>
  <div class="headertitle">
<div class="title">broad.pda.seq.alignment.AlignmentUtils Class Reference</div>  </div>
</div><!--header-->
<div class="contents">
<div class="dynheader">
Collaboration diagram for broad.pda.seq.alignment.AlignmentUtils:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils__coll__graph.png" border="0" usemap="#broad_8pda_8seq_8alignment_8_alignment_utils_coll__map" alt="Collaboration graph"/></div>
<map name="broad_8pda_8seq_8alignment_8_alignment_utils_coll__map" id="broad_8pda_8seq_8alignment_8_alignment_utils_coll__map">
</map>
<center><span class="legend">[<a href="graph_legend.html">legend</a>]</span></center></div>
<table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="pub-static-methods"></a>
Static Public Member Functions</h2></td></tr>
<tr class="memitem:a50fe3d7fac421c85d50a737755922da6"><td class="memItemLeft" align="right" valign="top">static void&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils.html#a50fe3d7fac421c85d50a737755922da6">main</a> (String[] args)  throws Exception </td></tr>
<tr class="separator:a50fe3d7fac421c85d50a737755922da6"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a01f53b98811de06325a819ab01cfbfc3"><td class="memItemLeft" align="right" valign="top">static <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html">ContinuousDataAlignmentModel</a>&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils.html#a01f53b98811de06325a819ab01cfbfc3">loadAlignmentData</a> (String alignmentFile)  throws IOException </td></tr>
<tr class="separator:a01f53b98811de06325a819ab01cfbfc3"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a5b9a1f05e9bf43422e95723e0f5b1008"><td class="memItemLeft" align="right" valign="top">static <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html">ContinuousDataAlignmentModel</a>&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils.html#a5b9a1f05e9bf43422e95723e0f5b1008">loadAlignmentData</a> (String alignmentFile, boolean loadChromosomeStats)  throws IOException </td></tr>
<tr class="separator:a5b9a1f05e9bf43422e95723e0f5b1008"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a3f38eadc7a254793812d0223e325fca2"><td class="memItemLeft" align="right" valign="top">static <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html">ContinuousDataAlignmentModel</a>&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils.html#a3f38eadc7a254793812d0223e325fca2">loadAlignmentData</a> (String alignmentFile, boolean loadChromosomeStats, int minMappingQuality)  throws IOException </td></tr>
<tr class="separator:a3f38eadc7a254793812d0223e325fca2"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a22bf5bebedcbaad5461419a3b9476fc9"><td class="memItemLeft" align="right" valign="top">static <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html">ContinuousDataAlignmentModel</a>&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils.html#a22bf5bebedcbaad5461419a3b9476fc9">loadAlignmentData</a> (String alignmentFile, boolean loadChromosomeStats, int minMappingQuality, boolean removeDuplicateFlags)  throws IOException </td></tr>
<tr class="separator:a22bf5bebedcbaad5461419a3b9476fc9"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ad098b0d98d37c3e070b667882b9f3132"><td class="memItemLeft" align="right" valign="top">static <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html">ContinuousDataAlignmentModel</a>&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils.html#ad098b0d98d37c3e070b667882b9f3132">loadAlignmentData</a> (String alignmentFile, boolean loadChromosomeStats, boolean removeDuplicateFlags, boolean weighReadCounts)  throws IOException </td></tr>
<tr class="separator:ad098b0d98d37c3e070b667882b9f3132"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ae407b9daa11a4ab33155c1c221dac929"><td class="memItemLeft" align="right" valign="top">static <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html">ContinuousDataAlignmentModel</a>&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils.html#ae407b9daa11a4ab33155c1c221dac929">loadAlignmentData</a> (String alignmentFile, boolean loadChromosomeStats, int minMappingQuality, boolean removeDuplicateFlags, boolean weighReadCounts)  throws IOException </td></tr>
<tr class="separator:ae407b9daa11a4ab33155c1c221dac929"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a26be40597246795138bcf13da2ea1b9b"><td class="memItemLeft" align="right" valign="top">static <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html">ContinuousDataAlignmentModel</a>&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils.html#a26be40597246795138bcf13da2ea1b9b">loadAlignmentData</a> (String alignmentFile, boolean loadChromosomeStats, int minMappingQuality, boolean removeDuplicateFlags, boolean weighReadCounts, String strand)  throws IOException </td></tr>
<tr class="separator:a26be40597246795138bcf13da2ea1b9b"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a4c8b2eacf9a07b744b6da702067a6ba4"><td class="memItemLeft" align="right" valign="top">static <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html">ContinuousDataAlignmentModel</a>&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils.html#a4c8b2eacf9a07b744b6da702067a6ba4">loadAlignmentData</a> (String alignmentFile, boolean loadChromosomeStats, int minMappingQuality, boolean removeDuplicateFlags, boolean weighReadCounts, String strand, boolean loadPairsAsFragments)  throws IOException </td></tr>
<tr class="separator:a4c8b2eacf9a07b744b6da702067a6ba4"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a6c939ca293f7fbfd141d81c1534a8461"><td class="memItemLeft" align="right" valign="top">static void&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils.html#a6c939ca293f7fbfd141d81c1534a8461">scoreAnnotations</a> (<a class="el" href="classbroad_1_1core_1_1annotation_1_1_b_e_d_reader.html">BEDReader</a> annotationReader, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html">ContinuousDataAlignmentModel</a> data)  throws IOException </td></tr>
<tr class="separator:a6c939ca293f7fbfd141d81c1534a8461"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:aaaae3d6e8f7f451a772f432a8414f4a2"><td class="memItemLeft" align="right" valign="top">static Map&lt; String, Integer &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils.html#aaaae3d6e8f7f451a772f432a8414f4a2">getSizesFromSAM</a> (String samFileName)</td></tr>
<tr class="separator:aaaae3d6e8f7f451a772f432a8414f4a2"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a88b4e4f173dd90784c6fe117156773c6"><td class="memItemLeft" align="right" valign="top">static Gene&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils.html#a88b4e4f173dd90784c6fe117156773c6">SAMFormatFullBED</a> (SAMRecord sam)</td></tr>
<tr class="separator:a88b4e4f173dd90784c6fe117156773c6"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table>
<h2 class="groupheader">Member Function Documentation</h2>
<a class="anchor" id="aaaae3d6e8f7f451a772f432a8414f4a2"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">static Map&lt;String, Integer&gt; broad.pda.seq.alignment.AlignmentUtils.getSizesFromSAM </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>samFileName</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">static</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a01f53b98811de06325a819ab01cfbfc3"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">static <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html">ContinuousDataAlignmentModel</a> broad.pda.seq.alignment.AlignmentUtils.loadAlignmentData </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>alignmentFile</em></td><td>)</td>
          <td> throws IOException</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">static</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a01f53b98811de06325a819ab01cfbfc3_icgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a01f53b98811de06325a819ab01cfbfc3_icgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a01f53b98811de06325a819ab01cfbfc3_icgraph" id="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a01f53b98811de06325a819ab01cfbfc3_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a5b9a1f05e9bf43422e95723e0f5b1008"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">static <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html">ContinuousDataAlignmentModel</a> broad.pda.seq.alignment.AlignmentUtils.loadAlignmentData </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>alignmentFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>loadChromosomeStats</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td> throws IOException</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">static</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a5b9a1f05e9bf43422e95723e0f5b1008_cgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a5b9a1f05e9bf43422e95723e0f5b1008_cgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a5b9a1f05e9bf43422e95723e0f5b1008_cgraph" id="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a5b9a1f05e9bf43422e95723e0f5b1008_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a3f38eadc7a254793812d0223e325fca2"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">static <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html">ContinuousDataAlignmentModel</a> broad.pda.seq.alignment.AlignmentUtils.loadAlignmentData </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>alignmentFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>loadChromosomeStats</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>minMappingQuality</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td> throws IOException</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">static</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a3f38eadc7a254793812d0223e325fca2_cgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a3f38eadc7a254793812d0223e325fca2_cgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a3f38eadc7a254793812d0223e325fca2_cgraph" id="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a3f38eadc7a254793812d0223e325fca2_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a22bf5bebedcbaad5461419a3b9476fc9"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">static <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html">ContinuousDataAlignmentModel</a> broad.pda.seq.alignment.AlignmentUtils.loadAlignmentData </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>alignmentFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>loadChromosomeStats</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>minMappingQuality</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>removeDuplicateFlags</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td> throws IOException</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">static</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">
<dl class="section author"><dt>Author</dt><dd>: skadri </dd></dl>
<dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">alignmentFile</td><td>Name of alignment file </td></tr>
    <tr><td class="paramname">loadChromosomeStats</td><td>File with chromosome stats </td></tr>
    <tr><td class="paramname">minMappingQuality</td><td>Mapping quality threshold </td></tr>
    <tr><td class="paramname">removeDuplicateFlags</td><td>Set to true if PCR duplicates should be ignored </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd></dd></dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td></td></tr>
  </table>
  </dd>
</dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a22bf5bebedcbaad5461419a3b9476fc9_cgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a22bf5bebedcbaad5461419a3b9476fc9_cgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a22bf5bebedcbaad5461419a3b9476fc9_cgraph" id="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a22bf5bebedcbaad5461419a3b9476fc9_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="ad098b0d98d37c3e070b667882b9f3132"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">static <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html">ContinuousDataAlignmentModel</a> broad.pda.seq.alignment.AlignmentUtils.loadAlignmentData </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>alignmentFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>loadChromosomeStats</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>removeDuplicateFlags</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>weighReadCounts</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td> throws IOException</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">static</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">
<dl class="section author"><dt>Author</dt><dd>: skadri </dd></dl>
<dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">alignmentFile</td><td>Name of alignment file </td></tr>
    <tr><td class="paramname">loadChromosomeStats</td><td>File with chromosome stats </td></tr>
    <tr><td class="paramname">removeDuplicateFlags</td><td>Set to true if PCR duplicates should be ignored </td></tr>
    <tr><td class="paramname">weighReadCounts</td><td>Set to true if read counts will be weighed by the NH flag </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd></dd></dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td></td></tr>
  </table>
  </dd>
</dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_ad098b0d98d37c3e070b667882b9f3132_cgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_ad098b0d98d37c3e070b667882b9f3132_cgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_ad098b0d98d37c3e070b667882b9f3132_cgraph" id="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_ad098b0d98d37c3e070b667882b9f3132_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="ae407b9daa11a4ab33155c1c221dac929"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">static <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html">ContinuousDataAlignmentModel</a> broad.pda.seq.alignment.AlignmentUtils.loadAlignmentData </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>alignmentFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>loadChromosomeStats</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>minMappingQuality</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>removeDuplicateFlags</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>weighReadCounts</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td> throws IOException</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">static</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">
<dl class="section author"><dt>Author</dt><dd>: skadri </dd></dl>
<dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">alignmentFile</td><td>Name of alignment file </td></tr>
    <tr><td class="paramname">loadChromosomeStats</td><td>File with chromosome stats </td></tr>
    <tr><td class="paramname">minMappingQuality</td><td>Mapping quality threshold = will be set to 0.0 if weighReadCounts is true </td></tr>
    <tr><td class="paramname">removeDuplicateFlags</td><td>Set to true if PCR duplicates should be ignored </td></tr>
    <tr><td class="paramname">weighReadCounts</td><td>Set to true if read counts will be weighed by the NH flag </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd></dd></dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td></td></tr>
  </table>
  </dd>
</dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_ae407b9daa11a4ab33155c1c221dac929_cgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_ae407b9daa11a4ab33155c1c221dac929_cgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_ae407b9daa11a4ab33155c1c221dac929_cgraph" id="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_ae407b9daa11a4ab33155c1c221dac929_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a26be40597246795138bcf13da2ea1b9b"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">static <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html">ContinuousDataAlignmentModel</a> broad.pda.seq.alignment.AlignmentUtils.loadAlignmentData </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>alignmentFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>loadChromosomeStats</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>minMappingQuality</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>removeDuplicateFlags</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>weighReadCounts</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>strand</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td> throws IOException</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">static</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a26be40597246795138bcf13da2ea1b9b_cgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a26be40597246795138bcf13da2ea1b9b_cgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a26be40597246795138bcf13da2ea1b9b_cgraph" id="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a26be40597246795138bcf13da2ea1b9b_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a4c8b2eacf9a07b744b6da702067a6ba4"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">static <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html">ContinuousDataAlignmentModel</a> broad.pda.seq.alignment.AlignmentUtils.loadAlignmentData </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>alignmentFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>loadChromosomeStats</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>minMappingQuality</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>removeDuplicateFlags</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>weighReadCounts</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>strand</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>loadPairsAsFragments</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td> throws IOException</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">static</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a50fe3d7fac421c85d50a737755922da6"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">static void broad.pda.seq.alignment.AlignmentUtils.main </td>
          <td>(</td>
          <td class="paramtype">String[]&#160;</td>
          <td class="paramname"><em>args</em></td><td>)</td>
          <td> throws Exception</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">static</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a50fe3d7fac421c85d50a737755922da6_cgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a50fe3d7fac421c85d50a737755922da6_cgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a50fe3d7fac421c85d50a737755922da6_cgraph" id="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils_a50fe3d7fac421c85d50a737755922da6_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a88b4e4f173dd90784c6fe117156773c6"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">static Gene broad.pda.seq.alignment.AlignmentUtils.SAMFormatFullBED </td>
          <td>(</td>
          <td class="paramtype">SAMRecord&#160;</td>
          <td class="paramname"><em>sam</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">static</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a6c939ca293f7fbfd141d81c1534a8461"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">static void broad.pda.seq.alignment.AlignmentUtils.scoreAnnotations </td>
          <td>(</td>
          <td class="paramtype"><a class="el" href="classbroad_1_1core_1_1annotation_1_1_b_e_d_reader.html">BEDReader</a>&#160;</td>
          <td class="paramname"><em>annotationReader</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype"><a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html">ContinuousDataAlignmentModel</a>&#160;</td>
          <td class="paramname"><em>data</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td> throws IOException</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">static</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<hr/>The documentation for this class was generated from the following file:<ul>
<li>java/broad/pda/seq/alignment/<a class="el" href="_alignment_utils_8java.html">AlignmentUtils.java</a></li>
</ul>
</div><!-- contents -->
<!-- start footer part -->
<hr class="footer"/><address class="footer"><small>
Generated on Fri Aug 15 2014 15:11:01 for ESAT by &#160;<a href="http://www.doxygen.org/index.html">
<img class="footer" src="doxygen.png" alt="doxygen"/>
</a> 1.8.7
</small></address>
</body>
</html>
