<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=9"/>
<meta name="generator" content="Doxygen 1.8.7"/>
<title>ESAT: broad.pda.seq.clip.SampleData Class Reference</title>
<link href="tabs.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="jquery.js"></script>
<script type="text/javascript" src="dynsections.js"></script>
<link href="search/search.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="search/search.js"></script>
<script type="text/javascript">
  $(document).ready(function() { searchBox.OnSelectItem(0); });
</script>
<link href="doxygen.css" rel="stylesheet" type="text/css" />
</head>
<body>
<div id="top"><!-- do not remove this div, it is closed by doxygen! -->
<div id="titlearea">
<table cellspacing="0" cellpadding="0">
 <tbody>
 <tr style="height: 56px;">
  <td style="padding-left: 0.5em;">
   <div id="projectname">ESAT
   </div>
  </td>
 </tr>
 </tbody>
</table>
</div>
<!-- end header part -->
<!-- Generated by Doxygen 1.8.7 -->
<script type="text/javascript">
var searchBox = new SearchBox("searchBox", "search",false,'Search');
</script>
  <div id="navrow1" class="tabs">
    <ul class="tablist">
      <li><a href="index.html"><span>Main&#160;Page</span></a></li>
      <li><a href="pages.html"><span>Related&#160;Pages</span></a></li>
      <li><a href="namespaces.html"><span>Packages</span></a></li>
      <li class="current"><a href="annotated.html"><span>Classes</span></a></li>
      <li><a href="files.html"><span>Files</span></a></li>
      <li>
        <div id="MSearchBox" class="MSearchBoxInactive">
        <span class="left">
          <img id="MSearchSelect" src="search/mag_sel.png"
               onmouseover="return searchBox.OnSearchSelectShow()"
               onmouseout="return searchBox.OnSearchSelectHide()"
               alt=""/>
          <input type="text" id="MSearchField" value="Search" accesskey="S"
               onfocus="searchBox.OnSearchFieldFocus(true)" 
               onblur="searchBox.OnSearchFieldFocus(false)" 
               onkeyup="searchBox.OnSearchFieldChange(event)"/>
          </span><span class="right">
            <a id="MSearchClose" href="javascript:searchBox.CloseResultsWindow()"><img id="MSearchCloseImg" border="0" src="search/close.png" alt=""/></a>
          </span>
        </div>
      </li>
    </ul>
  </div>
  <div id="navrow2" class="tabs2">
    <ul class="tablist">
      <li><a href="annotated.html"><span>Class&#160;List</span></a></li>
      <li><a href="classes.html"><span>Class&#160;Index</span></a></li>
      <li><a href="inherits.html"><span>Class&#160;Hierarchy</span></a></li>
      <li><a href="functions.html"><span>Class&#160;Members</span></a></li>
    </ul>
  </div>
<!-- window showing the filter options -->
<div id="MSearchSelectWindow"
     onmouseover="return searchBox.OnSearchSelectShow()"
     onmouseout="return searchBox.OnSearchSelectHide()"
     onkeydown="return searchBox.OnSearchSelectKey(event)">
<a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(0)"><span class="SelectionMark">&#160;</span>All</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(1)"><span class="SelectionMark">&#160;</span>Classes</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(2)"><span class="SelectionMark">&#160;</span>Namespaces</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(3)"><span class="SelectionMark">&#160;</span>Files</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(4)"><span class="SelectionMark">&#160;</span>Functions</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(5)"><span class="SelectionMark">&#160;</span>Variables</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(6)"><span class="SelectionMark">&#160;</span>Enumerator</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(7)"><span class="SelectionMark">&#160;</span>Pages</a></div>

<!-- iframe showing the search results (closed by default) -->
<div id="MSearchResultsWindow">
<iframe src="javascript:void(0)" frameborder="0" 
        name="MSearchResults" id="MSearchResults">
</iframe>
</div>

<div id="nav-path" class="navpath">
  <ul>
<li class="navelem"><a class="el" href="namespacebroad.html">broad</a></li><li class="navelem"><a class="el" href="namespacebroad_1_1pda.html">pda</a></li><li class="navelem"><a class="el" href="namespacebroad_1_1pda_1_1seq.html">seq</a></li><li class="navelem"><a class="el" href="namespacebroad_1_1pda_1_1seq_1_1clip.html">clip</a></li><li class="navelem"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html">SampleData</a></li>  </ul>
</div>
</div><!-- top -->
<div class="header">
  <div class="summary">
<a href="#pub-methods">Public Member Functions</a> &#124;
<a href="#pub-static-attribs">Static Public Attributes</a> &#124;
<a href="#pro-attribs">Protected Attributes</a> &#124;
<a href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data-members.html">List of all members</a>  </div>
  <div class="headertitle">
<div class="title">broad.pda.seq.clip.SampleData Class Reference</div>  </div>
</div><!--header-->
<div class="contents">
<div class="dynheader">
Inheritance diagram for broad.pda.seq.clip.SampleData:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data__inherit__graph.png" border="0" usemap="#broad_8pda_8seq_8clip_8_sample_data_inherit__map" alt="Inheritance graph"/></div>
<map name="broad_8pda_8seq_8clip_8_sample_data_inherit__map" id="broad_8pda_8seq_8clip_8_sample_data_inherit__map">
</map>
<center><span class="legend">[<a href="graph_legend.html">legend</a>]</span></center></div>
<div class="dynheader">
Collaboration diagram for broad.pda.seq.clip.SampleData:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data__coll__graph.png" border="0" usemap="#broad_8pda_8seq_8clip_8_sample_data_coll__map" alt="Collaboration graph"/></div>
<map name="broad_8pda_8seq_8clip_8_sample_data_coll__map" id="broad_8pda_8seq_8clip_8_sample_data_coll__map">
</map>
<center><span class="legend">[<a href="graph_legend.html">legend</a>]</span></center></div>
<table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="pub-methods"></a>
Public Member Functions</h2></td></tr>
<tr class="memitem:adcfd7ff578f7f31b52d759171e966003"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#adcfd7ff578f7f31b52d759171e966003">SampleData</a> (String bamFile, boolean <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a8bd77375c425d2175f5e63a85120b244">firstReadTranscriptionStrand</a>, Map&lt; String, Collection&lt; Gene &gt;&gt; genes, double expressionCutoff, boolean expByScanPval, boolean fullyContained)  throws IOException </td></tr>
<tr class="separator:adcfd7ff578f7f31b52d759171e966003"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:afec94a7510b50e455bc60c11c6ddc2ff"><td class="memItemLeft" align="right" valign="top">int&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#afec94a7510b50e455bc60c11c6ddc2ff">hashCode</a> ()</td></tr>
<tr class="separator:afec94a7510b50e455bc60c11c6ddc2ff"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a3f9e817ce92fd84f2b5756bdb5b97c36"><td class="memItemLeft" align="right" valign="top">void&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a3f9e817ce92fd84f2b5756bdb5b97c36">setExpressionScanPvalueCutoff</a> (double expressionScanPvalCutoff)</td></tr>
<tr class="separator:a3f9e817ce92fd84f2b5756bdb5b97c36"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ab1d89f168632a45a362704dafdbdc13c"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#ab1d89f168632a45a362704dafdbdc13c">getExpressionScanPvalueCutoff</a> ()</td></tr>
<tr class="separator:ab1d89f168632a45a362704dafdbdc13c"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ad90b439a428d5cf25c496d69b49437a4"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#ad90b439a428d5cf25c496d69b49437a4">getGeneCount</a> (Gene gene)</td></tr>
<tr class="separator:ad90b439a428d5cf25c496d69b49437a4"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:acacdbc70e2771b904d9611b653f18c28"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#acacdbc70e2771b904d9611b653f18c28">getFragmentLengthGeneCount</a> (Gene gene)</td></tr>
<tr class="separator:acacdbc70e2771b904d9611b653f18c28"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:aed882cc54637ce63c7c00e014fcd40cd"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#aed882cc54637ce63c7c00e014fcd40cd">getGeneScanPval</a> (Gene gene)</td></tr>
<tr class="separator:aed882cc54637ce63c7c00e014fcd40cd"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ae3f4ea198ba9415c26448706f02177bf"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#ae3f4ea198ba9415c26448706f02177bf">getGeneAverageCoverage</a> (Gene gene)</td></tr>
<tr class="separator:ae3f4ea198ba9415c26448706f02177bf"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a20bd1f5c944414c4305844a15c35f8a1"><td class="memItemLeft" align="right" valign="top">boolean&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a20bd1f5c944414c4305844a15c35f8a1">isExpressed</a> (Gene gene)</td></tr>
<tr class="separator:a20bd1f5c944414c4305844a15c35f8a1"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a8bd77375c425d2175f5e63a85120b244"><td class="memItemLeft" align="right" valign="top">boolean&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a8bd77375c425d2175f5e63a85120b244">firstReadTranscriptionStrand</a> ()</td></tr>
<tr class="separator:a8bd77375c425d2175f5e63a85120b244"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a83006fe4a56566ac14690adc205bf797"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a83006fe4a56566ac14690adc205bf797">getEnrichmentOverGene</a> (Gene gene, Annotation window)</td></tr>
<tr class="separator:a83006fe4a56566ac14690adc205bf797"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a2bfa33d3f02ba003618ad347cde46eda"><td class="memItemLeft" align="right" valign="top">Logger&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a2bfa33d3f02ba003618ad347cde46eda">getLogger</a> ()</td></tr>
<tr class="separator:a2bfa33d3f02ba003618ad347cde46eda"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a23c6423a51684f6f61cb445eec5d98d6"><td class="memItemLeft" align="right" valign="top">String&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a23c6423a51684f6f61cb445eec5d98d6">getSampleName</a> ()</td></tr>
<tr class="separator:a23c6423a51684f6f61cb445eec5d98d6"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ac929b442873bd023a32feb28dac3dbe9"><td class="memItemLeft" align="right" valign="top">String&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#ac929b442873bd023a32feb28dac3dbe9">getOriginalBamFile</a> ()</td></tr>
<tr class="separator:ac929b442873bd023a32feb28dac3dbe9"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a2e2bdce82785743a6814f003b3dc1df0"><td class="memItemLeft" align="right" valign="top">void&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a2e2bdce82785743a6814f003b3dc1df0">setSampleName</a> (String name)</td></tr>
<tr class="separator:a2e2bdce82785743a6814f003b3dc1df0"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:aa5acde7843960f0190b3a1ff9ae4b708"><td class="memItemLeft" align="right" valign="top">TranscriptomeSpaceAlignmentModel&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#aa5acde7843960f0190b3a1ff9ae4b708">getData</a> ()</td></tr>
<tr class="separator:aa5acde7843960f0190b3a1ff9ae4b708"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a36db29585dcedfafcb7169a660bef489"><td class="memItemLeft" align="right" valign="top">TranscriptomeSpaceAlignmentModel&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a36db29585dcedfafcb7169a660bef489">getFragmentLengthFilterData</a> ()</td></tr>
<tr class="separator:a36db29585dcedfafcb7169a660bef489"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table><table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="pub-static-attribs"></a>
Static Public Attributes</h2></td></tr>
<tr class="memitem:af64488a77382d0f1f7bde4c6909de2d5"><td class="memItemLeft" align="right" valign="top">static Logger&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#af64488a77382d0f1f7bde4c6909de2d5">logger</a> = Logger.getLogger(SampleData.class.getName())</td></tr>
<tr class="separator:af64488a77382d0f1f7bde4c6909de2d5"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table><table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="pro-attribs"></a>
Protected Attributes</h2></td></tr>
<tr class="memitem:a7c4644dba1de73c9baf1743c65628617"><td class="memItemLeft" align="right" valign="top">String&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a7c4644dba1de73c9baf1743c65628617">sampleName</a></td></tr>
<tr class="separator:a7c4644dba1de73c9baf1743c65628617"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:aaa5b43deb7143f3c845893faab23e33e"><td class="memItemLeft" align="right" valign="top">TranscriptomeSpaceAlignmentModel&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#aaa5b43deb7143f3c845893faab23e33e">data</a></td></tr>
<tr class="separator:aaa5b43deb7143f3c845893faab23e33e"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ae74526f9f7081cb45a51ac714bdb788e"><td class="memItemLeft" align="right" valign="top">TranscriptomeSpaceAlignmentModel&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#ae74526f9f7081cb45a51ac714bdb788e">maxFragmentLengthData</a></td></tr>
<tr class="separator:ae74526f9f7081cb45a51ac714bdb788e"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ab16a8cd9c6087c8086bdc63e4daae69d"><td class="memItemLeft" align="right" valign="top">Map&lt; Gene, ScanStatisticScore &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#ab16a8cd9c6087c8086bdc63e4daae69d">maxFragmentLengthGeneScores</a></td></tr>
<tr class="separator:ab16a8cd9c6087c8086bdc63e4daae69d"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a38dff572a12928939c8cefc3ad77b624"><td class="memItemLeft" align="right" valign="top">Map&lt; Gene, ScanStatisticScore &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a38dff572a12928939c8cefc3ad77b624">geneScores</a></td></tr>
<tr class="separator:a38dff572a12928939c8cefc3ad77b624"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:af41165298cda748416235bdaeef7ecf5"><td class="memItemLeft" align="right" valign="top">Map&lt; Gene, Double &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#af41165298cda748416235bdaeef7ecf5">geneAvgCoverage</a></td></tr>
<tr class="separator:af41165298cda748416235bdaeef7ecf5"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a8cd7634b21c85e6a4e79e65083932b28"><td class="memItemLeft" align="right" valign="top">Map&lt; String, Collection&lt; Gene &gt; &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a8cd7634b21c85e6a4e79e65083932b28">genesByChr</a></td></tr>
<tr class="separator:a8cd7634b21c85e6a4e79e65083932b28"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ab1ed0ab3b3ab3a672ba6fec27eb0905e"><td class="memItemLeft" align="right" valign="top">Map&lt; String, Gene &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#ab1ed0ab3b3ab3a672ba6fec27eb0905e">genesByName</a></td></tr>
<tr class="separator:ab1ed0ab3b3ab3a672ba6fec27eb0905e"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ad60c5254db251c508821fe4d85563c6c"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#ad60c5254db251c508821fe4d85563c6c">expressionCutoffValue</a></td></tr>
<tr class="separator:ad60c5254db251c508821fe4d85563c6c"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a728846c6f5131cd3a7b4acdcb682c4fa"><td class="memItemLeft" align="right" valign="top">boolean&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a728846c6f5131cd3a7b4acdcb682c4fa">expressionByScanPval</a></td></tr>
<tr class="separator:a728846c6f5131cd3a7b4acdcb682c4fa"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a9688cfb05381c9fbf875f8b329c89f9a"><td class="memItemLeft" align="right" valign="top">boolean&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a9688cfb05381c9fbf875f8b329c89f9a">fullyContainedReads</a></td></tr>
<tr class="separator:a9688cfb05381c9fbf875f8b329c89f9a"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table>
<a name="details" id="details"></a><h2 class="groupheader">Detailed Description</h2>
<div class="textblock"><dl class="section author"><dt>Author</dt><dd>shari </dd></dl>
</div><h2 class="groupheader">Constructor &amp; Destructor Documentation</h2>
<a class="anchor" id="adcfd7ff578f7f31b52d759171e966003"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">broad.pda.seq.clip.SampleData.SampleData </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>bamFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>firstReadTranscriptionStrand</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Map&lt; String, Collection&lt; Gene &gt;&gt;&#160;</td>
          <td class="paramname"><em>genes</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">double&#160;</td>
          <td class="paramname"><em>expressionCutoff</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>expByScanPval</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fullyContained</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td> throws IOException</td>
        </tr>
      </table>
</div><div class="memdoc">
<dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">bamFile</td><td>Bam file </td></tr>
    <tr><td class="paramname">firstReadTranscriptionStrand</td><td>Whether read 1 is in direction of transcription </td></tr>
    <tr><td class="paramname">genes</td><td>Genes by chromosome </td></tr>
    <tr><td class="paramname">window</td><td>Window size </td></tr>
    <tr><td class="paramname">step</td><td>Step size </td></tr>
    <tr><td class="paramname">expressionCutoff</td><td>P value cutoff for scan test of gene expression </td></tr>
    <tr><td class="paramname">expByScanPval</td><td>Expression is assessed by scan P value. If false, uses average read depth </td></tr>
    <tr><td class="paramname">fullyContained</td><td>Count fully contained reads only </td></tr>
  </table>
  </dd>
</dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td></td></tr>
  </table>
  </dd>
</dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data_adcfd7ff578f7f31b52d759171e966003_cgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data_adcfd7ff578f7f31b52d759171e966003_cgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data_adcfd7ff578f7f31b52d759171e966003_cgraph" id="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data_adcfd7ff578f7f31b52d759171e966003_cgraph">
</map>
</div>
</p>

</div>
</div>
<h2 class="groupheader">Member Function Documentation</h2>
<a class="anchor" id="a8bd77375c425d2175f5e63a85120b244"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">boolean broad.pda.seq.clip.SampleData.firstReadTranscriptionStrand </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Whether read 1 is in direction of transcription </p><dl class="section return"><dt>Returns</dt><dd>True iff first read transcription strand is true </dd></dl>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data_a8bd77375c425d2175f5e63a85120b244_icgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data_a8bd77375c425d2175f5e63a85120b244_icgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data_a8bd77375c425d2175f5e63a85120b244_icgraph" id="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data_a8bd77375c425d2175f5e63a85120b244_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="aa5acde7843960f0190b3a1ff9ae4b708"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">TranscriptomeSpaceAlignmentModel broad.pda.seq.clip.SampleData.getData </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Write the window scores to a file for future use </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">m</td><td>Multi sample binding site caller that this belongs to </td></tr>
  </table>
  </dd>
</dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td>Get the alignment data </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Alignment model </dd></dl>

</div>
</div>
<a class="anchor" id="a83006fe4a56566ac14690adc205bf797"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double broad.pda.seq.clip.SampleData.getEnrichmentOverGene </td>
          <td>(</td>
          <td class="paramtype">Gene&#160;</td>
          <td class="paramname"><em>gene</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Annotation&#160;</td>
          <td class="paramname"><em>window</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get enrichment of a window over a gene </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">gene</td><td>The gene </td></tr>
    <tr><td class="paramname">window</td><td>Window contained in the gene </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Enrichment of window over gene background </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data_a83006fe4a56566ac14690adc205bf797_cgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data_a83006fe4a56566ac14690adc205bf797_cgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data_a83006fe4a56566ac14690adc205bf797_cgraph" id="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data_a83006fe4a56566ac14690adc205bf797_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="ab1d89f168632a45a362704dafdbdc13c"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double broad.pda.seq.clip.SampleData.getExpressionScanPvalueCutoff </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get genome wide scan P value cutoff for expression of transcript </p><dl class="section return"><dt>Returns</dt><dd>P value cutoff for transcript expression against genomic background </dd></dl>

</div>
</div>
<a class="anchor" id="a36db29585dcedfafcb7169a660bef489"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">TranscriptomeSpaceAlignmentModel broad.pda.seq.clip.SampleData.getFragmentLengthFilterData </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get the alignment data filtered by max fragment length </p><dl class="section return"><dt>Returns</dt><dd>Alignment model </dd></dl>

</div>
</div>
<a class="anchor" id="acacdbc70e2771b904d9611b653f18c28"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double broad.pda.seq.clip.SampleData.getFragmentLengthGeneCount </td>
          <td>(</td>
          <td class="paramtype">Gene&#160;</td>
          <td class="paramname"><em>gene</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get number of fragments mapping to gene after fragment length filter Get score from cache or calculate and cache score </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">gene</td><td>The gene </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>The number of fragments mapping to the gene </dd></dl>

</div>
</div>
<a class="anchor" id="ae3f4ea198ba9415c26448706f02177bf"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double broad.pda.seq.clip.SampleData.getGeneAverageCoverage </td>
          <td>(</td>
          <td class="paramtype">Gene&#160;</td>
          <td class="paramname"><em>gene</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get average coverage of gene Get score from cache or calculate and cache score </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">gene</td><td>The gene </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>The average coverage of the gene </dd></dl>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data_ae3f4ea198ba9415c26448706f02177bf_icgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data_ae3f4ea198ba9415c26448706f02177bf_icgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data_ae3f4ea198ba9415c26448706f02177bf_icgraph" id="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data_ae3f4ea198ba9415c26448706f02177bf_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="ad90b439a428d5cf25c496d69b49437a4"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double broad.pda.seq.clip.SampleData.getGeneCount </td>
          <td>(</td>
          <td class="paramtype">Gene&#160;</td>
          <td class="paramname"><em>gene</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get number of fragments mapping to gene Get score from cache or calculate and cache score </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">gene</td><td>The gene </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>The number of fragments mapping to the gene </dd></dl>

</div>
</div>
<a class="anchor" id="aed882cc54637ce63c7c00e014fcd40cd"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double broad.pda.seq.clip.SampleData.getGeneScanPval </td>
          <td>(</td>
          <td class="paramtype">Gene&#160;</td>
          <td class="paramname"><em>gene</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get coordinate space wide scan P value of number of fragments mapping to the gene Get score from cache or calculate and cache score </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">gene</td><td>The gene </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>The scan P value of the number of fragments mapping to the gene with respect to teh coordinate space </dd></dl>

</div>
</div>
<a class="anchor" id="a2bfa33d3f02ba003618ad347cde46eda"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">Logger broad.pda.seq.clip.SampleData.getLogger </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get the logger </p><dl class="section return"><dt>Returns</dt><dd>The logger </dd></dl>

</div>
</div>
<a class="anchor" id="ac929b442873bd023a32feb28dac3dbe9"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">String broad.pda.seq.clip.SampleData.getOriginalBamFile </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get name of original bam file </p><dl class="section return"><dt>Returns</dt><dd>Bam file name </dd></dl>

</div>
</div>
<a class="anchor" id="a23c6423a51684f6f61cb445eec5d98d6"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">String broad.pda.seq.clip.SampleData.getSampleName </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get the sample name </p><dl class="section return"><dt>Returns</dt><dd>Sample name </dd></dl>

</div>
</div>
<a class="anchor" id="afec94a7510b50e455bc60c11c6ddc2ff"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">int broad.pda.seq.clip.SampleData.hashCode </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a20bd1f5c944414c4305844a15c35f8a1"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">boolean broad.pda.seq.clip.SampleData.isExpressed </td>
          <td>(</td>
          <td class="paramtype">Gene&#160;</td>
          <td class="paramname"><em>gene</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Whether the gene is significantly expressed in the sample </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">gene</td><td>The gene </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Whether the gene is expressed at the given significance level </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data_a20bd1f5c944414c4305844a15c35f8a1_cgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data_a20bd1f5c944414c4305844a15c35f8a1_cgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data_a20bd1f5c944414c4305844a15c35f8a1_cgraph" id="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data_a20bd1f5c944414c4305844a15c35f8a1_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a3f9e817ce92fd84f2b5756bdb5b97c36"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">void broad.pda.seq.clip.SampleData.setExpressionScanPvalueCutoff </td>
          <td>(</td>
          <td class="paramtype">double&#160;</td>
          <td class="paramname"><em>expressionScanPvalCutoff</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Set genome wide scan P value cutoff for expression of transcript </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">expressionScanPvalCutoff</td><td>P value cutoff for transcript expression against genomic background </td></tr>
  </table>
  </dd>
</dl>

</div>
</div>
<a class="anchor" id="a2e2bdce82785743a6814f003b3dc1df0"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">void broad.pda.seq.clip.SampleData.setSampleName </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>name</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Set the sample name </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">name</td><td>Sample name </td></tr>
  </table>
  </dd>
</dl>

</div>
</div>
<h2 class="groupheader">Member Data Documentation</h2>
<a class="anchor" id="aaa5b43deb7143f3c845893faab23e33e"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">TranscriptomeSpaceAlignmentModel broad.pda.seq.clip.SampleData.data</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a728846c6f5131cd3a7b4acdcb682c4fa"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">boolean broad.pda.seq.clip.SampleData.expressionByScanPval</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="ad60c5254db251c508821fe4d85563c6c"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">double broad.pda.seq.clip.SampleData.expressionCutoffValue</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a9688cfb05381c9fbf875f8b329c89f9a"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">boolean broad.pda.seq.clip.SampleData.fullyContainedReads</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="af41165298cda748416235bdaeef7ecf5"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">Map&lt;Gene, Double&gt; broad.pda.seq.clip.SampleData.geneAvgCoverage</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a8cd7634b21c85e6a4e79e65083932b28"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">Map&lt;String, Collection&lt;Gene&gt; &gt; broad.pda.seq.clip.SampleData.genesByChr</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="ab1ed0ab3b3ab3a672ba6fec27eb0905e"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">Map&lt;String, Gene&gt; broad.pda.seq.clip.SampleData.genesByName</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a38dff572a12928939c8cefc3ad77b624"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">Map&lt;Gene, ScanStatisticScore&gt; broad.pda.seq.clip.SampleData.geneScores</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="af64488a77382d0f1f7bde4c6909de2d5"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">Logger broad.pda.seq.clip.SampleData.logger = Logger.getLogger(SampleData.class.getName())</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">static</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="ae74526f9f7081cb45a51ac714bdb788e"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">TranscriptomeSpaceAlignmentModel broad.pda.seq.clip.SampleData.maxFragmentLengthData</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="ab16a8cd9c6087c8086bdc63e4daae69d"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">Map&lt;Gene, ScanStatisticScore&gt; broad.pda.seq.clip.SampleData.maxFragmentLengthGeneScores</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a7c4644dba1de73c9baf1743c65628617"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">String broad.pda.seq.clip.SampleData.sampleName</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<hr/>The documentation for this class was generated from the following file:<ul>
<li>java/broad/pda/seq/clip/<a class="el" href="clip_2_sample_data_8java.html">SampleData.java</a></li>
</ul>
</div><!-- contents -->
<!-- start footer part -->
<hr class="footer"/><address class="footer"><small>
Generated on Fri Aug 15 2014 15:11:02 for ESAT by &#160;<a href="http://www.doxygen.org/index.html">
<img class="footer" src="doxygen.png" alt="doxygen"/>
</a> 1.8.7
</small></address>
</body>
</html>
