<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=9"/>
<meta name="generator" content="Doxygen 1.8.7"/>
<title>ESAT: umms.core.alignment.FeatureSequenceAlignment Class Reference</title>
<link href="tabs.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="jquery.js"></script>
<script type="text/javascript" src="dynsections.js"></script>
<link href="search/search.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="search/search.js"></script>
<script type="text/javascript">
  $(document).ready(function() { searchBox.OnSelectItem(0); });
</script>
<link href="doxygen.css" rel="stylesheet" type="text/css" />
</head>
<body>
<div id="top"><!-- do not remove this div, it is closed by doxygen! -->
<div id="titlearea">
<table cellspacing="0" cellpadding="0">
 <tbody>
 <tr style="height: 56px;">
  <td style="padding-left: 0.5em;">
   <div id="projectname">ESAT
   </div>
  </td>
 </tr>
 </tbody>
</table>
</div>
<!-- end header part -->
<!-- Generated by Doxygen 1.8.7 -->
<script type="text/javascript">
var searchBox = new SearchBox("searchBox", "search",false,'Search');
</script>
  <div id="navrow1" class="tabs">
    <ul class="tablist">
      <li><a href="index.html"><span>Main&#160;Page</span></a></li>
      <li><a href="pages.html"><span>Related&#160;Pages</span></a></li>
      <li><a href="namespaces.html"><span>Packages</span></a></li>
      <li class="current"><a href="annotated.html"><span>Classes</span></a></li>
      <li><a href="files.html"><span>Files</span></a></li>
      <li>
        <div id="MSearchBox" class="MSearchBoxInactive">
        <span class="left">
          <img id="MSearchSelect" src="search/mag_sel.png"
               onmouseover="return searchBox.OnSearchSelectShow()"
               onmouseout="return searchBox.OnSearchSelectHide()"
               alt=""/>
          <input type="text" id="MSearchField" value="Search" accesskey="S"
               onfocus="searchBox.OnSearchFieldFocus(true)" 
               onblur="searchBox.OnSearchFieldFocus(false)" 
               onkeyup="searchBox.OnSearchFieldChange(event)"/>
          </span><span class="right">
            <a id="MSearchClose" href="javascript:searchBox.CloseResultsWindow()"><img id="MSearchCloseImg" border="0" src="search/close.png" alt=""/></a>
          </span>
        </div>
      </li>
    </ul>
  </div>
  <div id="navrow2" class="tabs2">
    <ul class="tablist">
      <li><a href="annotated.html"><span>Class&#160;List</span></a></li>
      <li><a href="classes.html"><span>Class&#160;Index</span></a></li>
      <li><a href="inherits.html"><span>Class&#160;Hierarchy</span></a></li>
      <li><a href="functions.html"><span>Class&#160;Members</span></a></li>
    </ul>
  </div>
<!-- window showing the filter options -->
<div id="MSearchSelectWindow"
     onmouseover="return searchBox.OnSearchSelectShow()"
     onmouseout="return searchBox.OnSearchSelectHide()"
     onkeydown="return searchBox.OnSearchSelectKey(event)">
<a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(0)"><span class="SelectionMark">&#160;</span>All</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(1)"><span class="SelectionMark">&#160;</span>Classes</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(2)"><span class="SelectionMark">&#160;</span>Namespaces</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(3)"><span class="SelectionMark">&#160;</span>Files</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(4)"><span class="SelectionMark">&#160;</span>Functions</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(5)"><span class="SelectionMark">&#160;</span>Variables</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(6)"><span class="SelectionMark">&#160;</span>Enumerator</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(7)"><span class="SelectionMark">&#160;</span>Pages</a></div>

<!-- iframe showing the search results (closed by default) -->
<div id="MSearchResultsWindow">
<iframe src="javascript:void(0)" frameborder="0" 
        name="MSearchResults" id="MSearchResults">
</iframe>
</div>

<div id="nav-path" class="navpath">
  <ul>
<li class="navelem"><a class="el" href="namespaceumms.html">umms</a></li><li class="navelem"><a class="el" href="namespaceumms_1_1core.html">core</a></li><li class="navelem"><a class="el" href="namespaceumms_1_1core_1_1alignment.html">alignment</a></li><li class="navelem"><a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment.html">FeatureSequenceAlignment</a></li>  </ul>
</div>
</div><!-- top -->
<div class="header">
  <div class="summary">
<a href="#nested-classes">Classes</a> &#124;
<a href="#pub-methods">Public Member Functions</a> &#124;
<a href="#pub-static-methods">Static Public Member Functions</a> &#124;
<a href="#pub-static-attribs">Static Public Attributes</a> &#124;
<a href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment-members.html">List of all members</a>  </div>
  <div class="headertitle">
<div class="title">umms.core.alignment.FeatureSequenceAlignment Class Reference</div>  </div>
</div><!--header-->
<div class="contents">
<table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="nested-classes"></a>
Classes</h2></td></tr>
<tr class="memitem:"><td class="memItemLeft" align="right" valign="top">class &#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_1_1_unordered_gene_pair.html">UnorderedGenePair</a></td></tr>
<tr class="separator:"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table><table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="pub-methods"></a>
Public Member Functions</h2></td></tr>
<tr class="memitem:ab35fcf26aae378b32764e85217562ba1"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment.html#ab35fcf26aae378b32764e85217562ba1">FeatureSequenceAlignment</a> (String featureBedFile, String genomeFasta)  throws IOException </td></tr>
<tr class="separator:ab35fcf26aae378b32764e85217562ba1"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a11d8045bdeea5ef20ae6437a1db80e96"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment.html#a11d8045bdeea5ef20ae6437a1db80e96">FeatureSequenceAlignment</a> (Collection&lt; <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html">Gene</a> &gt; genes, String genomeFasta)  throws IOException </td></tr>
<tr class="separator:a11d8045bdeea5ef20ae6437a1db80e96"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:aaaff665576cdbcf265e150ff167dfabb"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment.html#aaaff665576cdbcf265e150ff167dfabb">FeatureSequenceAlignment</a> (String featureBedFile, String genomeFasta, float matchScore, float mismatchScore, float gapOpenPenalty, float gapExtendPenalty)  throws IOException </td></tr>
<tr class="separator:aaaff665576cdbcf265e150ff167dfabb"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a33614e453036f115a7755c4d809cd768"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment.html#a33614e453036f115a7755c4d809cd768">FeatureSequenceAlignment</a> (Collection&lt; <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html">Gene</a> &gt; genes, Map&lt; String, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html">broad.core.sequence.Sequence</a> &gt; chrs, float matchScore, float mismatchScore, float gapOpenPenalty, float gapExtendPenalty)  throws IOException </td></tr>
<tr class="separator:a33614e453036f115a7755c4d809cd768"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:aec32e5a32e6a269fd93d9e62db8d5922"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment.html#aec32e5a32e6a269fd93d9e62db8d5922">FeatureSequenceAlignment</a> (Collection&lt; <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html">Gene</a> &gt; genes, String genomeFasta, float matchScore, float mismatchScore, float gapOpenPenalty, float gapExtendPenalty)  throws IOException </td></tr>
<tr class="separator:aec32e5a32e6a269fd93d9e62db8d5922"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a335945731d4a8ddcca3b130106172247"><td class="memItemLeft" align="right" valign="top">Map&lt; <a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_1_1_unordered_gene_pair.html">UnorderedGenePair</a>, <br class="typebreak" />
<a class="el" href="classjaligner_1_1_alignment.html">jaligner.Alignment</a> &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment.html#a335945731d4a8ddcca3b130106172247">getAllPairwiseSenseAlignments</a> (float minAlignLength, float minPctIdentity)</td></tr>
<tr class="separator:a335945731d4a8ddcca3b130106172247"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ab6151110c36c6f03905a49d791fd56f1"><td class="memItemLeft" align="right" valign="top">Map&lt; <a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_1_1_unordered_gene_pair.html">UnorderedGenePair</a>, <br class="typebreak" />
<a class="el" href="classjaligner_1_1_alignment.html">jaligner.Alignment</a> &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment.html#ab6151110c36c6f03905a49d791fd56f1">getAllPairwiseAntisenseAlignments</a> (float minAlignLength, float minPctIdentity)</td></tr>
<tr class="separator:ab6151110c36c6f03905a49d791fd56f1"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a61e3263283cf027e28d2b280b53aa5ea"><td class="memItemLeft" align="right" valign="top">void&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment.html#a61e3263283cf027e28d2b280b53aa5ea">writeAllSenseAlignmentsToBed</a> (String outBedFile, int minAlignLength, float minPctIdentity)  throws IOException </td></tr>
<tr class="separator:a61e3263283cf027e28d2b280b53aa5ea"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:adfc91ae05ecb83e8d09963f5ffe8e2ea"><td class="memItemLeft" align="right" valign="top">void&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment.html#adfc91ae05ecb83e8d09963f5ffe8e2ea">writeAllAntisenseAlignmentsToBed</a> (String outBedFile, int minAlignLength, float minPctIdentity)  throws IOException </td></tr>
<tr class="separator:adfc91ae05ecb83e8d09963f5ffe8e2ea"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:af19b7baf1e78ec683d89359f06fc4327"><td class="memItemLeft" align="right" valign="top">void&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment.html#af19b7baf1e78ec683d89359f06fc4327">writeAllSenseAlignmentsToBed</a> (String outBedFile, boolean append, int minAlignLength, float minPctIdentity)  throws IOException </td></tr>
<tr class="separator:af19b7baf1e78ec683d89359f06fc4327"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:aec99809f0d839cd75596dd97f5f33305"><td class="memItemLeft" align="right" valign="top">void&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment.html#aec99809f0d839cd75596dd97f5f33305">writeAllAntisenseAlignmentsToBed</a> (String outBedFile, boolean append, int minAlignLength, float minPctIdentity)  throws IOException </td></tr>
<tr class="separator:aec99809f0d839cd75596dd97f5f33305"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table><table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="pub-static-methods"></a>
Static Public Member Functions</h2></td></tr>
<tr class="memitem:a09afca590c8f7c9a6adcb2289b911fa5"><td class="memItemLeft" align="right" valign="top">static void&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment.html#a09afca590c8f7c9a6adcb2289b911fa5">main</a> (String[] args)  throws IOException </td></tr>
<tr class="separator:a09afca590c8f7c9a6adcb2289b911fa5"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table><table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="pub-static-attribs"></a>
Static Public Attributes</h2></td></tr>
<tr class="memitem:a632fe2de8b604500533be5ea8139473f"><td class="memItemLeft" align="right" valign="top">static float&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment.html#a632fe2de8b604500533be5ea8139473f">DEFAULT_MATCH_SCORE</a> = 5</td></tr>
<tr class="separator:a632fe2de8b604500533be5ea8139473f"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a21bd1298e50fa041cd652da22e37770a"><td class="memItemLeft" align="right" valign="top">static float&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment.html#a21bd1298e50fa041cd652da22e37770a">DEFAULT_MISMATCH_SCORE</a> = -4</td></tr>
<tr class="separator:a21bd1298e50fa041cd652da22e37770a"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:adb8e26dc3dd4ee0cdba1f3d7ae1bbfff"><td class="memItemLeft" align="right" valign="top">static float&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment.html#adb8e26dc3dd4ee0cdba1f3d7ae1bbfff">DEFAULT_GAP_OPEN_PENALTY</a> = 8</td></tr>
<tr class="separator:adb8e26dc3dd4ee0cdba1f3d7ae1bbfff"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a2a3d43f6bdf98dc290930f48c791fceb"><td class="memItemLeft" align="right" valign="top">static float&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment.html#a2a3d43f6bdf98dc290930f48c791fceb">DEFAULT_GAP_EXTEND_PENALTY</a> = 2</td></tr>
<tr class="separator:a2a3d43f6bdf98dc290930f48c791fceb"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table>
<a name="details" id="details"></a><h2 class="groupheader">Detailed Description</h2>
<div class="textblock"><dl class="section author"><dt>Author</dt><dd>prussell </dd></dl>
</div><h2 class="groupheader">Constructor &amp; Destructor Documentation</h2>
<a class="anchor" id="ab35fcf26aae378b32764e85217562ba1"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">umms.core.alignment.FeatureSequenceAlignment.FeatureSequenceAlignment </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>featureBedFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>genomeFasta</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td> throws IOException</td>
        </tr>
      </table>
</div><div class="memdoc">
<dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">featureBedFile</td><td>Bed file of features </td></tr>
    <tr><td class="paramname">genomeFasta</td><td>Genome fasta file </td></tr>
  </table>
  </dd>
</dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td></td></tr>
  </table>
  </dd>
</dl>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_ab35fcf26aae378b32764e85217562ba1_icgraph.png" border="0" usemap="#classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_ab35fcf26aae378b32764e85217562ba1_icgraph" alt=""/></div>
<map name="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_ab35fcf26aae378b32764e85217562ba1_icgraph" id="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_ab35fcf26aae378b32764e85217562ba1_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a11d8045bdeea5ef20ae6437a1db80e96"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">umms.core.alignment.FeatureSequenceAlignment.FeatureSequenceAlignment </td>
          <td>(</td>
          <td class="paramtype">Collection&lt; <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html">Gene</a> &gt;&#160;</td>
          <td class="paramname"><em>genes</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>genomeFasta</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td> throws IOException</td>
        </tr>
      </table>
</div><div class="memdoc">
<dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">genes</td><td>Set of features </td></tr>
    <tr><td class="paramname">genomeFasta</td><td>Genome fasta file </td></tr>
  </table>
  </dd>
</dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td></td></tr>
  </table>
  </dd>
</dl>

</div>
</div>
<a class="anchor" id="aaaff665576cdbcf265e150ff167dfabb"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">umms.core.alignment.FeatureSequenceAlignment.FeatureSequenceAlignment </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>featureBedFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>genomeFasta</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">float&#160;</td>
          <td class="paramname"><em>matchScore</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">float&#160;</td>
          <td class="paramname"><em>mismatchScore</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">float&#160;</td>
          <td class="paramname"><em>gapOpenPenalty</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">float&#160;</td>
          <td class="paramname"><em>gapExtendPenalty</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td> throws IOException</td>
        </tr>
      </table>
</div><div class="memdoc">
<dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">featureBedFile</td><td>Bed file of features </td></tr>
    <tr><td class="paramname">genomeFasta</td><td>Genome fasta file </td></tr>
    <tr><td class="paramname">matchScore</td><td>Match score for Smith Waterman </td></tr>
    <tr><td class="paramname">mismatchScore</td><td>Mismatch score for Smith Waterman </td></tr>
    <tr><td class="paramname">gapOpenPenalty</td><td>Gap open penalty for Smith Waterman </td></tr>
    <tr><td class="paramname">gapExtendPenalty</td><td>Gap extend penalty for Smith Waterman </td></tr>
  </table>
  </dd>
</dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td></td></tr>
  </table>
  </dd>
</dl>

</div>
</div>
<a class="anchor" id="a33614e453036f115a7755c4d809cd768"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">umms.core.alignment.FeatureSequenceAlignment.FeatureSequenceAlignment </td>
          <td>(</td>
          <td class="paramtype">Collection&lt; <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html">Gene</a> &gt;&#160;</td>
          <td class="paramname"><em>genes</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Map&lt; String, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html">broad.core.sequence.Sequence</a> &gt;&#160;</td>
          <td class="paramname"><em>chrs</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">float&#160;</td>
          <td class="paramname"><em>matchScore</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">float&#160;</td>
          <td class="paramname"><em>mismatchScore</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">float&#160;</td>
          <td class="paramname"><em>gapOpenPenalty</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">float&#160;</td>
          <td class="paramname"><em>gapExtendPenalty</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td> throws IOException</td>
        </tr>
      </table>
</div><div class="memdoc">
<dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">genes</td><td>The features </td></tr>
    <tr><td class="paramname">chrs</td><td>Map of chromosome names to sequences </td></tr>
    <tr><td class="paramname">matchScore</td><td>Match score for Smith Waterman </td></tr>
    <tr><td class="paramname">mismatchScore</td><td>Mismatch score for Smith Waterman </td></tr>
    <tr><td class="paramname">gapOpenPenalty</td><td>Gap open penalty for Smith Waterman </td></tr>
    <tr><td class="paramname">gapExtendPenalty</td><td>Gap extend penalty for Smith Waterman </td></tr>
  </table>
  </dd>
</dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td></td></tr>
  </table>
  </dd>
</dl>

</div>
</div>
<a class="anchor" id="aec32e5a32e6a269fd93d9e62db8d5922"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">umms.core.alignment.FeatureSequenceAlignment.FeatureSequenceAlignment </td>
          <td>(</td>
          <td class="paramtype">Collection&lt; <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html">Gene</a> &gt;&#160;</td>
          <td class="paramname"><em>genes</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>genomeFasta</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">float&#160;</td>
          <td class="paramname"><em>matchScore</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">float&#160;</td>
          <td class="paramname"><em>mismatchScore</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">float&#160;</td>
          <td class="paramname"><em>gapOpenPenalty</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">float&#160;</td>
          <td class="paramname"><em>gapExtendPenalty</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td> throws IOException</td>
        </tr>
      </table>
</div><div class="memdoc">
<dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">genes</td><td>Set of features </td></tr>
    <tr><td class="paramname">genomeFasta</td><td>Genome fasta file </td></tr>
    <tr><td class="paramname">matchScore</td><td>Match score for Smith Waterman </td></tr>
    <tr><td class="paramname">mismatchScore</td><td>Mismatch score for Smith Waterman </td></tr>
    <tr><td class="paramname">gapOpenPenalty</td><td>Gap open penalty for Smith Waterman </td></tr>
    <tr><td class="paramname">gapExtendPenalty</td><td>Gap extend penalty for Smith Waterman </td></tr>
  </table>
  </dd>
</dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td></td></tr>
  </table>
  </dd>
</dl>

</div>
</div>
<h2 class="groupheader">Member Function Documentation</h2>
<a class="anchor" id="ab6151110c36c6f03905a49d791fd56f1"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">Map&lt;<a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_1_1_unordered_gene_pair.html">UnorderedGenePair</a>, <a class="el" href="classjaligner_1_1_alignment.html">jaligner.Alignment</a>&gt; umms.core.alignment.FeatureSequenceAlignment.getAllPairwiseAntisenseAlignments </td>
          <td>(</td>
          <td class="paramtype">float&#160;</td>
          <td class="paramname"><em>minAlignLength</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">float&#160;</td>
          <td class="paramname"><em>minPctIdentity</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get all antisense direction alignments passing thresholds </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">minAlignLength</td><td>Min alignment length </td></tr>
    <tr><td class="paramname">minPctIdentity</td><td>Min percent identity </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>All antisense alignments of unordered gene pair passing criteria </dd></dl>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_ab6151110c36c6f03905a49d791fd56f1_icgraph.png" border="0" usemap="#classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_ab6151110c36c6f03905a49d791fd56f1_icgraph" alt=""/></div>
<map name="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_ab6151110c36c6f03905a49d791fd56f1_icgraph" id="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_ab6151110c36c6f03905a49d791fd56f1_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a335945731d4a8ddcca3b130106172247"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">Map&lt;<a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_1_1_unordered_gene_pair.html">UnorderedGenePair</a>, <a class="el" href="classjaligner_1_1_alignment.html">jaligner.Alignment</a>&gt; umms.core.alignment.FeatureSequenceAlignment.getAllPairwiseSenseAlignments </td>
          <td>(</td>
          <td class="paramtype">float&#160;</td>
          <td class="paramname"><em>minAlignLength</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">float&#160;</td>
          <td class="paramname"><em>minPctIdentity</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get all sense direction alignments passing thresholds </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">minAlignLength</td><td>Min alignment length </td></tr>
    <tr><td class="paramname">minPctIdentity</td><td>Min percent identity </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>All sense alignments of unordered gene pair passing criteria </dd></dl>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_a335945731d4a8ddcca3b130106172247_icgraph.png" border="0" usemap="#classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_a335945731d4a8ddcca3b130106172247_icgraph" alt=""/></div>
<map name="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_a335945731d4a8ddcca3b130106172247_icgraph" id="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_a335945731d4a8ddcca3b130106172247_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a09afca590c8f7c9a6adcb2289b911fa5"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">static void umms.core.alignment.FeatureSequenceAlignment.main </td>
          <td>(</td>
          <td class="paramtype">String[]&#160;</td>
          <td class="paramname"><em>args</em></td><td>)</td>
          <td> throws IOException</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">static</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">
<dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">args</td><td></td></tr>
  </table>
  </dd>
</dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td></td></tr>
  </table>
  </dd>
</dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_a09afca590c8f7c9a6adcb2289b911fa5_cgraph.png" border="0" usemap="#classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_a09afca590c8f7c9a6adcb2289b911fa5_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_a09afca590c8f7c9a6adcb2289b911fa5_cgraph" id="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_a09afca590c8f7c9a6adcb2289b911fa5_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="adfc91ae05ecb83e8d09963f5ffe8e2ea"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">void umms.core.alignment.FeatureSequenceAlignment.writeAllAntisenseAlignmentsToBed </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>outBedFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>minAlignLength</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">float&#160;</td>
          <td class="paramname"><em>minPctIdentity</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td> throws IOException</td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Write all pairwise alignments (antisense direction only) to bed file in genome coordinates </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">outBedFile</td><td>Output bed file </td></tr>
    <tr><td class="paramname">minAlignLength</td><td>Min alignment length to keep </td></tr>
    <tr><td class="paramname">minPctIdentity</td><td>Min percent identity to keep </td></tr>
  </table>
  </dd>
</dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td></td></tr>
  </table>
  </dd>
</dl>

</div>
</div>
<a class="anchor" id="aec99809f0d839cd75596dd97f5f33305"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">void umms.core.alignment.FeatureSequenceAlignment.writeAllAntisenseAlignmentsToBed </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>outBedFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>append</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>minAlignLength</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">float&#160;</td>
          <td class="paramname"><em>minPctIdentity</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td> throws IOException</td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Write all pairwise alignments (antisense direction only) to bed file in genome coordinates </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">outBedFile</td><td>Output bed file </td></tr>
    <tr><td class="paramname">append</td><td>Write to end of file rather than beginning </td></tr>
    <tr><td class="paramname">minAlignLength</td><td>Min alignment length to keep </td></tr>
    <tr><td class="paramname">minPctIdentity</td><td>Min percent identity to keep </td></tr>
  </table>
  </dd>
</dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td></td></tr>
  </table>
  </dd>
</dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_aec99809f0d839cd75596dd97f5f33305_cgraph.png" border="0" usemap="#classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_aec99809f0d839cd75596dd97f5f33305_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_aec99809f0d839cd75596dd97f5f33305_cgraph" id="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_aec99809f0d839cd75596dd97f5f33305_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a61e3263283cf027e28d2b280b53aa5ea"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">void umms.core.alignment.FeatureSequenceAlignment.writeAllSenseAlignmentsToBed </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>outBedFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>minAlignLength</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">float&#160;</td>
          <td class="paramname"><em>minPctIdentity</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td> throws IOException</td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Write all pairwise alignments (sense direction only) to bed file in genome coordinates </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">outBedFile</td><td>Output bed file </td></tr>
    <tr><td class="paramname">minAlignLength</td><td>Min alignment length to keep </td></tr>
    <tr><td class="paramname">minPctIdentity</td><td>Min percent identity to keep </td></tr>
  </table>
  </dd>
</dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td></td></tr>
  </table>
  </dd>
</dl>

</div>
</div>
<a class="anchor" id="af19b7baf1e78ec683d89359f06fc4327"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">void umms.core.alignment.FeatureSequenceAlignment.writeAllSenseAlignmentsToBed </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>outBedFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>append</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>minAlignLength</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">float&#160;</td>
          <td class="paramname"><em>minPctIdentity</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td> throws IOException</td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Write all pairwise alignments (sense direction only) to bed file in genome coordinates </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">outBedFile</td><td>Output bed file </td></tr>
    <tr><td class="paramname">append</td><td>Write to end of file rather than beginning </td></tr>
    <tr><td class="paramname">minAlignLength</td><td>Min alignment length to keep </td></tr>
    <tr><td class="paramname">minPctIdentity</td><td>Min percent identity to keep </td></tr>
  </table>
  </dd>
</dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td></td></tr>
  </table>
  </dd>
</dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_af19b7baf1e78ec683d89359f06fc4327_cgraph.png" border="0" usemap="#classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_af19b7baf1e78ec683d89359f06fc4327_cgraph" alt=""/></div>
<map name="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_af19b7baf1e78ec683d89359f06fc4327_cgraph" id="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_af19b7baf1e78ec683d89359f06fc4327_cgraph">
</map>
</div>
</p>

</div>
</div>
<h2 class="groupheader">Member Data Documentation</h2>
<a class="anchor" id="a2a3d43f6bdf98dc290930f48c791fceb"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">float umms.core.alignment.FeatureSequenceAlignment.DEFAULT_GAP_EXTEND_PENALTY = 2</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">static</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">
<p>Default gap extend penalty for Smith Waterman </p>

</div>
</div>
<a class="anchor" id="adb8e26dc3dd4ee0cdba1f3d7ae1bbfff"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">float umms.core.alignment.FeatureSequenceAlignment.DEFAULT_GAP_OPEN_PENALTY = 8</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">static</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">
<p>Default gap open penalty for Smith Waterman </p>

</div>
</div>
<a class="anchor" id="a632fe2de8b604500533be5ea8139473f"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">float umms.core.alignment.FeatureSequenceAlignment.DEFAULT_MATCH_SCORE = 5</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">static</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">
<p>Default match score for Smith Waterman </p>

</div>
</div>
<a class="anchor" id="a21bd1298e50fa041cd652da22e37770a"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">float umms.core.alignment.FeatureSequenceAlignment.DEFAULT_MISMATCH_SCORE = -4</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">static</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">
<p>Default mismatch score for Smith Waterman </p>

</div>
</div>
<hr/>The documentation for this class was generated from the following file:<ul>
<li>java/umms/core/alignment/<a class="el" href="_feature_sequence_alignment_8java.html">FeatureSequenceAlignment.java</a></li>
</ul>
</div><!-- contents -->
<!-- start footer part -->
<hr class="footer"/><address class="footer"><small>
Generated on Fri Aug 15 2014 15:11:07 for ESAT by &#160;<a href="http://www.doxygen.org/index.html">
<img class="footer" src="doxygen.png" alt="doxygen"/>
</a> 1.8.7
</small></address>
</body>
</html>
