<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=9"/>
<meta name="generator" content="Doxygen 1.8.7"/>
<title>ESAT: broad.pda.seq.protection.SampleData Class Reference</title>
<link href="tabs.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="jquery.js"></script>
<script type="text/javascript" src="dynsections.js"></script>
<link href="search/search.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="search/search.js"></script>
<script type="text/javascript">
  $(document).ready(function() { searchBox.OnSelectItem(0); });
</script>
<link href="doxygen.css" rel="stylesheet" type="text/css" />
</head>
<body>
<div id="top"><!-- do not remove this div, it is closed by doxygen! -->
<div id="titlearea">
<table cellspacing="0" cellpadding="0">
 <tbody>
 <tr style="height: 56px;">
  <td style="padding-left: 0.5em;">
   <div id="projectname">ESAT
   </div>
  </td>
 </tr>
 </tbody>
</table>
</div>
<!-- end header part -->
<!-- Generated by Doxygen 1.8.7 -->
<script type="text/javascript">
var searchBox = new SearchBox("searchBox", "search",false,'Search');
</script>
  <div id="navrow1" class="tabs">
    <ul class="tablist">
      <li><a href="index.html"><span>Main&#160;Page</span></a></li>
      <li><a href="pages.html"><span>Related&#160;Pages</span></a></li>
      <li><a href="namespaces.html"><span>Packages</span></a></li>
      <li class="current"><a href="annotated.html"><span>Classes</span></a></li>
      <li><a href="files.html"><span>Files</span></a></li>
      <li>
        <div id="MSearchBox" class="MSearchBoxInactive">
        <span class="left">
          <img id="MSearchSelect" src="search/mag_sel.png"
               onmouseover="return searchBox.OnSearchSelectShow()"
               onmouseout="return searchBox.OnSearchSelectHide()"
               alt=""/>
          <input type="text" id="MSearchField" value="Search" accesskey="S"
               onfocus="searchBox.OnSearchFieldFocus(true)" 
               onblur="searchBox.OnSearchFieldFocus(false)" 
               onkeyup="searchBox.OnSearchFieldChange(event)"/>
          </span><span class="right">
            <a id="MSearchClose" href="javascript:searchBox.CloseResultsWindow()"><img id="MSearchCloseImg" border="0" src="search/close.png" alt=""/></a>
          </span>
        </div>
      </li>
    </ul>
  </div>
  <div id="navrow2" class="tabs2">
    <ul class="tablist">
      <li><a href="annotated.html"><span>Class&#160;List</span></a></li>
      <li><a href="classes.html"><span>Class&#160;Index</span></a></li>
      <li><a href="inherits.html"><span>Class&#160;Hierarchy</span></a></li>
      <li><a href="functions.html"><span>Class&#160;Members</span></a></li>
    </ul>
  </div>
<!-- window showing the filter options -->
<div id="MSearchSelectWindow"
     onmouseover="return searchBox.OnSearchSelectShow()"
     onmouseout="return searchBox.OnSearchSelectHide()"
     onkeydown="return searchBox.OnSearchSelectKey(event)">
<a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(0)"><span class="SelectionMark">&#160;</span>All</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(1)"><span class="SelectionMark">&#160;</span>Classes</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(2)"><span class="SelectionMark">&#160;</span>Namespaces</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(3)"><span class="SelectionMark">&#160;</span>Files</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(4)"><span class="SelectionMark">&#160;</span>Functions</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(5)"><span class="SelectionMark">&#160;</span>Variables</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(6)"><span class="SelectionMark">&#160;</span>Enumerator</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(7)"><span class="SelectionMark">&#160;</span>Pages</a></div>

<!-- iframe showing the search results (closed by default) -->
<div id="MSearchResultsWindow">
<iframe src="javascript:void(0)" frameborder="0" 
        name="MSearchResults" id="MSearchResults">
</iframe>
</div>

<div id="nav-path" class="navpath">
  <ul>
<li class="navelem"><a class="el" href="namespacebroad.html">broad</a></li><li class="navelem"><a class="el" href="namespacebroad_1_1pda.html">pda</a></li><li class="navelem"><a class="el" href="namespacebroad_1_1pda_1_1seq.html">seq</a></li><li class="navelem"><a class="el" href="namespacebroad_1_1pda_1_1seq_1_1protection.html">protection</a></li><li class="navelem"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html">SampleData</a></li>  </ul>
</div>
</div><!-- top -->
<div class="header">
  <div class="summary">
<a href="#pub-methods">Public Member Functions</a> &#124;
<a href="#pub-static-methods">Static Public Member Functions</a> &#124;
<a href="#pro-attribs">Protected Attributes</a> &#124;
<a href="#pro-static-attribs">Static Protected Attributes</a> &#124;
<a href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data-members.html">List of all members</a>  </div>
  <div class="headertitle">
<div class="title">broad.pda.seq.protection.SampleData Class Reference</div>  </div>
</div><!--header-->
<div class="contents">
<div class="dynheader">
Inheritance diagram for broad.pda.seq.protection.SampleData:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data__inherit__graph.png" border="0" usemap="#broad_8pda_8seq_8protection_8_sample_data_inherit__map" alt="Inheritance graph"/></div>
<map name="broad_8pda_8seq_8protection_8_sample_data_inherit__map" id="broad_8pda_8seq_8protection_8_sample_data_inherit__map">
</map>
<center><span class="legend">[<a href="graph_legend.html">legend</a>]</span></center></div>
<div class="dynheader">
Collaboration diagram for broad.pda.seq.protection.SampleData:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data__coll__graph.png" border="0" usemap="#broad_8pda_8seq_8protection_8_sample_data_coll__map" alt="Collaboration graph"/></div>
<map name="broad_8pda_8seq_8protection_8_sample_data_coll__map" id="broad_8pda_8seq_8protection_8_sample_data_coll__map">
</map>
<center><span class="legend">[<a href="graph_legend.html">legend</a>]</span></center></div>
<table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="pub-methods"></a>
Public Member Functions</h2></td></tr>
<tr class="memitem:acbea9965c1d9f8b120e1510e326c5b96"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#acbea9965c1d9f8b120e1510e326c5b96">SampleData</a> (String bamFile, boolean <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a1a603b2dd39ba9c8b54b3552a3053783">firstReadTranscriptionStrand</a>, Map&lt; String, Collection&lt; Gene &gt;&gt; genes, int window, int step, double expressionCutoff, boolean expByScanPval, boolean fullyContained)  throws IOException </td></tr>
<tr class="separator:acbea9965c1d9f8b120e1510e326c5b96"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a8fe3a848dd59a9a072680c711fa86766"><td class="memItemLeft" align="right" valign="top">int&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a8fe3a848dd59a9a072680c711fa86766">hashCode</a> ()</td></tr>
<tr class="separator:a8fe3a848dd59a9a072680c711fa86766"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a2013287af35974ad48c4cd57b099ac4b"><td class="memItemLeft" align="right" valign="top">void&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a2013287af35974ad48c4cd57b099ac4b">setExpressionScanPvalueCutoff</a> (double expressionScanPvalCutoff)</td></tr>
<tr class="separator:a2013287af35974ad48c4cd57b099ac4b"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a4087ee5f33ea5ecc977f4fdf4483804e"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a4087ee5f33ea5ecc977f4fdf4483804e">getExpressionScanPvalueCutoff</a> ()</td></tr>
<tr class="separator:a4087ee5f33ea5ecc977f4fdf4483804e"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:aa2bf68f6d49af41a18f1d4a1277c36f4"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#aa2bf68f6d49af41a18f1d4a1277c36f4">getGeneCount</a> (Gene gene)</td></tr>
<tr class="separator:aa2bf68f6d49af41a18f1d4a1277c36f4"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a0795b41a9a096104df31f5b6ba2c9154"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a0795b41a9a096104df31f5b6ba2c9154">getGeneScanPval</a> (Gene gene)</td></tr>
<tr class="separator:a0795b41a9a096104df31f5b6ba2c9154"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a7d17b21cbf2a2368cc637a7debb8089e"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a7d17b21cbf2a2368cc637a7debb8089e">getGeneAverageCoverage</a> (Gene gene)</td></tr>
<tr class="separator:a7d17b21cbf2a2368cc637a7debb8089e"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:af2a3f7b805b2fea8605b821b53f56958"><td class="memItemLeft" align="right" valign="top">boolean&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#af2a3f7b805b2fea8605b821b53f56958">isExpressed</a> (Gene gene)</td></tr>
<tr class="separator:af2a3f7b805b2fea8605b821b53f56958"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:afd2283807ea84e0cc2da07c108f6ca98"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#afd2283807ea84e0cc2da07c108f6ca98">getWindowCount</a> (Gene gene, Annotation window)</td></tr>
<tr class="separator:afd2283807ea84e0cc2da07c108f6ca98"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a1a603b2dd39ba9c8b54b3552a3053783"><td class="memItemLeft" align="right" valign="top">boolean&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a1a603b2dd39ba9c8b54b3552a3053783">firstReadTranscriptionStrand</a> ()</td></tr>
<tr class="separator:a1a603b2dd39ba9c8b54b3552a3053783"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a1adf639ddb5cd1b53ab3ff48f9f74ea9"><td class="memItemLeft" align="right" valign="top">Map&lt; Annotation, <br class="typebreak" />
ScanStatisticScore &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a1adf639ddb5cd1b53ab3ff48f9f74ea9">getWindowScores</a> (Gene gene)</td></tr>
<tr class="separator:a1adf639ddb5cd1b53ab3ff48f9f74ea9"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:af69246249c44eabd4349626e639a44c6"><td class="memItemLeft" align="right" valign="top">boolean&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#af69246249c44eabd4349626e639a44c6">gotWindowScoresFromFile</a> ()</td></tr>
<tr class="separator:af69246249c44eabd4349626e639a44c6"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a2f18a4bad14841ddbb440c21a281703a"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a2f18a4bad14841ddbb440c21a281703a">getEnrichmentOverGene</a> (Gene gene, Annotation window)</td></tr>
<tr class="separator:a2f18a4bad14841ddbb440c21a281703a"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a477fb361142dde07116598599bb7ef52"><td class="memItemLeft" align="right" valign="top">ScanStatisticScore&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a477fb361142dde07116598599bb7ef52">scoreWindowWithFragmentLengthFilter</a> (Gene gene, Annotation window)</td></tr>
<tr class="separator:a477fb361142dde07116598599bb7ef52"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ac476ccc56fab96d17419127a8cf313a1"><td class="memItemLeft" align="right" valign="top">ScanStatisticScore&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#ac476ccc56fab96d17419127a8cf313a1">scoreWindow</a> (Gene gene, Annotation window)</td></tr>
<tr class="separator:ac476ccc56fab96d17419127a8cf313a1"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ae556c14ccb3cb7b63182adf72e3630d7"><td class="memItemLeft" align="right" valign="top">ScanStatisticScore&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#ae556c14ccb3cb7b63182adf72e3630d7">scoreWindow</a> (Gene gene, Annotation window, double count)</td></tr>
<tr class="separator:ae556c14ccb3cb7b63182adf72e3630d7"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ab31cc1f558419ad028782703b015c489"><td class="memItemLeft" align="right" valign="top">Logger&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#ab31cc1f558419ad028782703b015c489">getLogger</a> ()</td></tr>
<tr class="separator:ab31cc1f558419ad028782703b015c489"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a9284f2dc44109171457dcc0417d8520a"><td class="memItemLeft" align="right" valign="top">String&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a9284f2dc44109171457dcc0417d8520a">getSampleName</a> ()</td></tr>
<tr class="separator:a9284f2dc44109171457dcc0417d8520a"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a3a3c58c0def5a809f7949a2f441e815d"><td class="memItemLeft" align="right" valign="top">String&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a3a3c58c0def5a809f7949a2f441e815d">getOriginalBamFile</a> ()</td></tr>
<tr class="separator:a3a3c58c0def5a809f7949a2f441e815d"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a77cf4e55d644cb09fa376f9d8b41c538"><td class="memItemLeft" align="right" valign="top">void&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a77cf4e55d644cb09fa376f9d8b41c538">setSampleName</a> (String name)</td></tr>
<tr class="separator:a77cf4e55d644cb09fa376f9d8b41c538"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ab775331fb5bee8667165c78ca5a105ba"><td class="memItemLeft" align="right" valign="top">int&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#ab775331fb5bee8667165c78ca5a105ba">getWindowSize</a> ()</td></tr>
<tr class="separator:ab775331fb5bee8667165c78ca5a105ba"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a225b80f975a281ea6c4d99482a20ba0e"><td class="memItemLeft" align="right" valign="top">int&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a225b80f975a281ea6c4d99482a20ba0e">getStepSize</a> ()</td></tr>
<tr class="separator:a225b80f975a281ea6c4d99482a20ba0e"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a7e1478400941779a43e0776cd90733bb"><td class="memItemLeft" align="right" valign="top">TranscriptomeSpaceAlignmentModel&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a7e1478400941779a43e0776cd90733bb">getData</a> ()</td></tr>
<tr class="separator:a7e1478400941779a43e0776cd90733bb"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a896c9168d0e4ca469e94d60554bdf1d9"><td class="memItemLeft" align="right" valign="top">TranscriptomeSpaceAlignmentModel&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a896c9168d0e4ca469e94d60554bdf1d9">getFragmentLengthFilterData</a> ()</td></tr>
<tr class="separator:a896c9168d0e4ca469e94d60554bdf1d9"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table><table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="pub-static-methods"></a>
Static Public Member Functions</h2></td></tr>
<tr class="memitem:a8d27ed6393a7f774100c706c5a1b9b8d"><td class="memItemLeft" align="right" valign="top">static Annotation&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a8d27ed6393a7f774100c706c5a1b9b8d">trimMaxContiguous</a> (Annotation window, List&lt; Double &gt; <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#ae06d8e63d6ddf7093085beba12b22e8a">data</a>, double quantile)</td></tr>
<tr class="separator:a8d27ed6393a7f774100c706c5a1b9b8d"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table><table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="pro-attribs"></a>
Protected Attributes</h2></td></tr>
<tr class="memitem:a5b0011ca04c2ee52ec32f8508b82ae99"><td class="memItemLeft" align="right" valign="top">String&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a5b0011ca04c2ee52ec32f8508b82ae99">sampleName</a></td></tr>
<tr class="separator:a5b0011ca04c2ee52ec32f8508b82ae99"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ae06d8e63d6ddf7093085beba12b22e8a"><td class="memItemLeft" align="right" valign="top">TranscriptomeSpaceAlignmentModel&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#ae06d8e63d6ddf7093085beba12b22e8a">data</a></td></tr>
<tr class="separator:ae06d8e63d6ddf7093085beba12b22e8a"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a214cced67761f0090ba326708620d3be"><td class="memItemLeft" align="right" valign="top">TranscriptomeSpaceAlignmentModel&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a214cced67761f0090ba326708620d3be">maxFragmentLengthData</a></td></tr>
<tr class="separator:a214cced67761f0090ba326708620d3be"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:af611a295e308f4f0d0849a80139ad7ff"><td class="memItemLeft" align="right" valign="top">Map&lt; Gene, ScanStatisticScore &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#af611a295e308f4f0d0849a80139ad7ff">geneScores</a></td></tr>
<tr class="separator:af611a295e308f4f0d0849a80139ad7ff"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ad517adb3cfab02f8f65bc10818ebca71"><td class="memItemLeft" align="right" valign="top">Map&lt; Gene, Double &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#ad517adb3cfab02f8f65bc10818ebca71">geneAvgCoverage</a></td></tr>
<tr class="separator:ad517adb3cfab02f8f65bc10818ebca71"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a6a53d104b93427859b6fd2c4ffbfdf6d"><td class="memItemLeft" align="right" valign="top">Map&lt; Gene, Map&lt; Annotation, <br class="typebreak" />
ScanStatisticScore &gt; &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a6a53d104b93427859b6fd2c4ffbfdf6d">windowScores</a></td></tr>
<tr class="separator:a6a53d104b93427859b6fd2c4ffbfdf6d"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:af155a04e70a9feafd812aa6135f5b732"><td class="memItemLeft" align="right" valign="top">int&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#af155a04e70a9feafd812aa6135f5b732">windowSize</a></td></tr>
<tr class="separator:af155a04e70a9feafd812aa6135f5b732"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ab5fe21e8281091d1be8de525e17d8a3a"><td class="memItemLeft" align="right" valign="top">int&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#ab5fe21e8281091d1be8de525e17d8a3a">stepSize</a></td></tr>
<tr class="separator:ab5fe21e8281091d1be8de525e17d8a3a"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a8b1f58470538adcd26f47252bdf94232"><td class="memItemLeft" align="right" valign="top">Map&lt; String, Collection&lt; Gene &gt; &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a8b1f58470538adcd26f47252bdf94232">genesByChr</a></td></tr>
<tr class="separator:a8b1f58470538adcd26f47252bdf94232"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a528cf3b7c628a3f4a567bd33ba75ca0f"><td class="memItemLeft" align="right" valign="top">Map&lt; String, Gene &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a528cf3b7c628a3f4a567bd33ba75ca0f">genesByName</a></td></tr>
<tr class="separator:a528cf3b7c628a3f4a567bd33ba75ca0f"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ae1a282e95f82d5974d5e3c4189a81f8c"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#ae1a282e95f82d5974d5e3c4189a81f8c">expressionCutoffValue</a></td></tr>
<tr class="separator:ae1a282e95f82d5974d5e3c4189a81f8c"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a0f463fa018b0f1cc4495a3dfdd2e9ef5"><td class="memItemLeft" align="right" valign="top">boolean&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a0f463fa018b0f1cc4495a3dfdd2e9ef5">expressionByScanPval</a></td></tr>
<tr class="separator:a0f463fa018b0f1cc4495a3dfdd2e9ef5"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ac971b6d1585808b15cc2ef8807225bf3"><td class="memItemLeft" align="right" valign="top">boolean&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#ac971b6d1585808b15cc2ef8807225bf3">fullyContainedReads</a></td></tr>
<tr class="separator:ac971b6d1585808b15cc2ef8807225bf3"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table><table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="pro-static-attribs"></a>
Static Protected Attributes</h2></td></tr>
<tr class="memitem:a4f08fb5a371b51daf757152c83863ac2"><td class="memItemLeft" align="right" valign="top">static Logger&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a4f08fb5a371b51daf757152c83863ac2">logger</a> = Logger.getLogger(SampleData.class.getName())</td></tr>
<tr class="separator:a4f08fb5a371b51daf757152c83863ac2"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table>
<a name="details" id="details"></a><h2 class="groupheader">Detailed Description</h2>
<div class="textblock"><dl class="section author"><dt>Author</dt><dd>prussell </dd></dl>
</div><h2 class="groupheader">Constructor &amp; Destructor Documentation</h2>
<a class="anchor" id="acbea9965c1d9f8b120e1510e326c5b96"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">broad.pda.seq.protection.SampleData.SampleData </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>bamFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>firstReadTranscriptionStrand</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Map&lt; String, Collection&lt; Gene &gt;&gt;&#160;</td>
          <td class="paramname"><em>genes</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>window</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>step</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">double&#160;</td>
          <td class="paramname"><em>expressionCutoff</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>expByScanPval</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fullyContained</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td> throws IOException</td>
        </tr>
      </table>
</div><div class="memdoc">
<dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">bamFile</td><td>Bam file </td></tr>
    <tr><td class="paramname">firstReadTranscriptionStrand</td><td>Whether read 1 is in direction of transcription </td></tr>
    <tr><td class="paramname">genes</td><td>Genes by chromosome </td></tr>
    <tr><td class="paramname">window</td><td>Window size </td></tr>
    <tr><td class="paramname">step</td><td>Step size </td></tr>
    <tr><td class="paramname">expressionCutoff</td><td>P value cutoff for scan test of gene expression </td></tr>
    <tr><td class="paramname">expByScanPval</td><td>Expression is assessed by scan P value. If false, uses average read depth </td></tr>
    <tr><td class="paramname">fullyContained</td><td>Count fully contained reads only </td></tr>
  </table>
  </dd>
</dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td></td></tr>
  </table>
  </dd>
</dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_acbea9965c1d9f8b120e1510e326c5b96_cgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_acbea9965c1d9f8b120e1510e326c5b96_cgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_acbea9965c1d9f8b120e1510e326c5b96_cgraph" id="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_acbea9965c1d9f8b120e1510e326c5b96_cgraph">
</map>
</div>
</p>

</div>
</div>
<h2 class="groupheader">Member Function Documentation</h2>
<a class="anchor" id="a1a603b2dd39ba9c8b54b3552a3053783"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">boolean broad.pda.seq.protection.SampleData.firstReadTranscriptionStrand </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Whether read 1 is in direction of transcription </p><dl class="section return"><dt>Returns</dt><dd>True iff first read transcription strand is true </dd></dl>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a1a603b2dd39ba9c8b54b3552a3053783_icgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a1a603b2dd39ba9c8b54b3552a3053783_icgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a1a603b2dd39ba9c8b54b3552a3053783_icgraph" id="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a1a603b2dd39ba9c8b54b3552a3053783_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a7e1478400941779a43e0776cd90733bb"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">TranscriptomeSpaceAlignmentModel broad.pda.seq.protection.SampleData.getData </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Write the window scores to a file for future use </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">m</td><td>Multi sample binding site caller that this belongs to </td></tr>
  </table>
  </dd>
</dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td>Get the alignment data </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Alignment model </dd></dl>

</div>
</div>
<a class="anchor" id="a2f18a4bad14841ddbb440c21a281703a"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double broad.pda.seq.protection.SampleData.getEnrichmentOverGene </td>
          <td>(</td>
          <td class="paramtype">Gene&#160;</td>
          <td class="paramname"><em>gene</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Annotation&#160;</td>
          <td class="paramname"><em>window</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get enrichment of a window over a gene </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">gene</td><td>The gene </td></tr>
    <tr><td class="paramname">window</td><td>Window contained in the gene </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Enrichment of window over gene background </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a2f18a4bad14841ddbb440c21a281703a_cgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a2f18a4bad14841ddbb440c21a281703a_cgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a2f18a4bad14841ddbb440c21a281703a_cgraph" id="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a2f18a4bad14841ddbb440c21a281703a_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a4087ee5f33ea5ecc977f4fdf4483804e"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double broad.pda.seq.protection.SampleData.getExpressionScanPvalueCutoff </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get genome wide scan P value cutoff for expression of transcript </p><dl class="section return"><dt>Returns</dt><dd>P value cutoff for transcript expression against genomic background </dd></dl>

</div>
</div>
<a class="anchor" id="a896c9168d0e4ca469e94d60554bdf1d9"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">TranscriptomeSpaceAlignmentModel broad.pda.seq.protection.SampleData.getFragmentLengthFilterData </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get the alignment data filtered by max fragment length </p><dl class="section return"><dt>Returns</dt><dd>Alignment model </dd></dl>

</div>
</div>
<a class="anchor" id="a7d17b21cbf2a2368cc637a7debb8089e"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double broad.pda.seq.protection.SampleData.getGeneAverageCoverage </td>
          <td>(</td>
          <td class="paramtype">Gene&#160;</td>
          <td class="paramname"><em>gene</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get average coverage of gene Get score from cache or calculate and cache score </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">gene</td><td>The gene </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>The average coverage of the gene </dd></dl>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a7d17b21cbf2a2368cc637a7debb8089e_icgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a7d17b21cbf2a2368cc637a7debb8089e_icgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a7d17b21cbf2a2368cc637a7debb8089e_icgraph" id="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a7d17b21cbf2a2368cc637a7debb8089e_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="aa2bf68f6d49af41a18f1d4a1277c36f4"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double broad.pda.seq.protection.SampleData.getGeneCount </td>
          <td>(</td>
          <td class="paramtype">Gene&#160;</td>
          <td class="paramname"><em>gene</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get number of fragments mapping to gene Get score from cache or calculate and cache score </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">gene</td><td>The gene </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>The number of fragments mapping to the gene </dd></dl>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_aa2bf68f6d49af41a18f1d4a1277c36f4_icgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_aa2bf68f6d49af41a18f1d4a1277c36f4_icgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_aa2bf68f6d49af41a18f1d4a1277c36f4_icgraph" id="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_aa2bf68f6d49af41a18f1d4a1277c36f4_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a0795b41a9a096104df31f5b6ba2c9154"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double broad.pda.seq.protection.SampleData.getGeneScanPval </td>
          <td>(</td>
          <td class="paramtype">Gene&#160;</td>
          <td class="paramname"><em>gene</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get coordinate space wide scan P value of number of fragments mapping to the gene Get score from cache or calculate and cache score </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">gene</td><td>The gene </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>The scan P value of the number of fragments mapping to the gene with respect to teh coordinate space </dd></dl>

</div>
</div>
<a class="anchor" id="ab31cc1f558419ad028782703b015c489"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">Logger broad.pda.seq.protection.SampleData.getLogger </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get the logger </p><dl class="section return"><dt>Returns</dt><dd>The logger </dd></dl>

</div>
</div>
<a class="anchor" id="a3a3c58c0def5a809f7949a2f441e815d"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">String broad.pda.seq.protection.SampleData.getOriginalBamFile </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get name of original bam file </p><dl class="section return"><dt>Returns</dt><dd>Bam file name </dd></dl>

</div>
</div>
<a class="anchor" id="a9284f2dc44109171457dcc0417d8520a"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">String broad.pda.seq.protection.SampleData.getSampleName </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get the sample name </p><dl class="section return"><dt>Returns</dt><dd>Sample name </dd></dl>

</div>
</div>
<a class="anchor" id="a225b80f975a281ea6c4d99482a20ba0e"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">int broad.pda.seq.protection.SampleData.getStepSize </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="afd2283807ea84e0cc2da07c108f6ca98"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double broad.pda.seq.protection.SampleData.getWindowCount </td>
          <td>(</td>
          <td class="paramtype">Gene&#160;</td>
          <td class="paramname"><em>gene</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Annotation&#160;</td>
          <td class="paramname"><em>window</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get number of fragments mapping to gene window Get score from cache or calculate and cache score </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">gene</td><td>The gene </td></tr>
    <tr><td class="paramname">window</td><td>The window </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>The number of fragments mapping to the gene </dd></dl>

</div>
</div>
<a class="anchor" id="a1adf639ddb5cd1b53ab3ff48f9f74ea9"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">Map&lt;Annotation, ScanStatisticScore&gt; broad.pda.seq.protection.SampleData.getWindowScores </td>
          <td>(</td>
          <td class="paramtype">Gene&#160;</td>
          <td class="paramname"><em>gene</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get scores for each window in the gene </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">gene</td><td>The gene </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>The scores by window </dd></dl>

</div>
</div>
<a class="anchor" id="ab775331fb5bee8667165c78ca5a105ba"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">int broad.pda.seq.protection.SampleData.getWindowSize </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="af69246249c44eabd4349626e639a44c6"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">boolean broad.pda.seq.protection.SampleData.gotWindowScoresFromFile </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Whether the window scores were read from an existing file </p><dl class="section return"><dt>Returns</dt><dd>Whether the window scores were read from an existing file </dd></dl>

</div>
</div>
<a class="anchor" id="a8fe3a848dd59a9a072680c711fa86766"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">int broad.pda.seq.protection.SampleData.hashCode </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="af2a3f7b805b2fea8605b821b53f56958"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">boolean broad.pda.seq.protection.SampleData.isExpressed </td>
          <td>(</td>
          <td class="paramtype">Gene&#160;</td>
          <td class="paramname"><em>gene</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Whether the gene is significantly expressed in the sample </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">gene</td><td>The gene </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Whether the gene is expressed at the given significance level </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_af2a3f7b805b2fea8605b821b53f56958_cgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_af2a3f7b805b2fea8605b821b53f56958_cgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_af2a3f7b805b2fea8605b821b53f56958_cgraph" id="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_af2a3f7b805b2fea8605b821b53f56958_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="ac476ccc56fab96d17419127a8cf313a1"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">ScanStatisticScore broad.pda.seq.protection.SampleData.scoreWindow </td>
          <td>(</td>
          <td class="paramtype">Gene&#160;</td>
          <td class="paramname"><em>gene</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Annotation&#160;</td>
          <td class="paramname"><em>window</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get scan statistic score for a specified window </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">gene</td><td>Parent gene </td></tr>
    <tr><td class="paramname">window</td><td>The window </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Scan statistic score for window with global stats referring to parent transcript </dd></dl>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ac476ccc56fab96d17419127a8cf313a1_icgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ac476ccc56fab96d17419127a8cf313a1_icgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ac476ccc56fab96d17419127a8cf313a1_icgraph" id="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ac476ccc56fab96d17419127a8cf313a1_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="ae556c14ccb3cb7b63182adf72e3630d7"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">ScanStatisticScore broad.pda.seq.protection.SampleData.scoreWindow </td>
          <td>(</td>
          <td class="paramtype">Gene&#160;</td>
          <td class="paramname"><em>gene</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Annotation&#160;</td>
          <td class="paramname"><em>window</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">double&#160;</td>
          <td class="paramname"><em>count</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get scan statistic score for a specified window </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">gene</td><td>Parent gene </td></tr>
    <tr><td class="paramname">window</td><td>The window </td></tr>
    <tr><td class="paramname">count</td><td>The number to use as the region count in score </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Scan statistic score for window with global stats referring to parent transcript </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ae556c14ccb3cb7b63182adf72e3630d7_cgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ae556c14ccb3cb7b63182adf72e3630d7_cgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ae556c14ccb3cb7b63182adf72e3630d7_cgraph" id="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_ae556c14ccb3cb7b63182adf72e3630d7_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a477fb361142dde07116598599bb7ef52"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">ScanStatisticScore broad.pda.seq.protection.SampleData.scoreWindowWithFragmentLengthFilter </td>
          <td>(</td>
          <td class="paramtype">Gene&#160;</td>
          <td class="paramname"><em>gene</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Annotation&#160;</td>
          <td class="paramname"><em>window</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get scan statistic score for a window based on filtering fragments overlapping the window by size </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">gene</td><td>The gene </td></tr>
    <tr><td class="paramname">window</td><td>The window </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Scan statistic score for window with global stats referring to parent transcript </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a477fb361142dde07116598599bb7ef52_cgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a477fb361142dde07116598599bb7ef52_cgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a477fb361142dde07116598599bb7ef52_cgraph" id="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data_a477fb361142dde07116598599bb7ef52_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a2013287af35974ad48c4cd57b099ac4b"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">void broad.pda.seq.protection.SampleData.setExpressionScanPvalueCutoff </td>
          <td>(</td>
          <td class="paramtype">double&#160;</td>
          <td class="paramname"><em>expressionScanPvalCutoff</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Set genome wide scan P value cutoff for expression of transcript </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">expressionScanPvalCutoff</td><td>P value cutoff for transcript expression against genomic background </td></tr>
  </table>
  </dd>
</dl>

</div>
</div>
<a class="anchor" id="a77cf4e55d644cb09fa376f9d8b41c538"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">void broad.pda.seq.protection.SampleData.setSampleName </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>name</em></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Set the sample name </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">name</td><td>Sample name </td></tr>
  </table>
  </dd>
</dl>

</div>
</div>
<a class="anchor" id="a8d27ed6393a7f774100c706c5a1b9b8d"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">static Annotation broad.pda.seq.protection.SampleData.trimMaxContiguous </td>
          <td>(</td>
          <td class="paramtype">Annotation&#160;</td>
          <td class="paramname"><em>window</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">List&lt; Double &gt;&#160;</td>
          <td class="paramname"><em>data</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">double&#160;</td>
          <td class="paramname"><em>quantile</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">static</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">
<p>Trim the region to max contiguous subregion above a certain quantile </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">window</td><td>The region </td></tr>
    <tr><td class="paramname">data</td><td>Position level list of counts within the region </td></tr>
    <tr><td class="paramname">quantile</td><td>Quantile for trim max contiguous </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Trimmed region </dd></dl>

</div>
</div>
<h2 class="groupheader">Member Data Documentation</h2>
<a class="anchor" id="ae06d8e63d6ddf7093085beba12b22e8a"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">TranscriptomeSpaceAlignmentModel broad.pda.seq.protection.SampleData.data</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a0f463fa018b0f1cc4495a3dfdd2e9ef5"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">boolean broad.pda.seq.protection.SampleData.expressionByScanPval</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="ae1a282e95f82d5974d5e3c4189a81f8c"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">double broad.pda.seq.protection.SampleData.expressionCutoffValue</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="ac971b6d1585808b15cc2ef8807225bf3"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">boolean broad.pda.seq.protection.SampleData.fullyContainedReads</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="ad517adb3cfab02f8f65bc10818ebca71"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">Map&lt;Gene, Double&gt; broad.pda.seq.protection.SampleData.geneAvgCoverage</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a8b1f58470538adcd26f47252bdf94232"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">Map&lt;String, Collection&lt;Gene&gt; &gt; broad.pda.seq.protection.SampleData.genesByChr</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a528cf3b7c628a3f4a567bd33ba75ca0f"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">Map&lt;String, Gene&gt; broad.pda.seq.protection.SampleData.genesByName</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="af611a295e308f4f0d0849a80139ad7ff"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">Map&lt;Gene, ScanStatisticScore&gt; broad.pda.seq.protection.SampleData.geneScores</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a4f08fb5a371b51daf757152c83863ac2"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">Logger broad.pda.seq.protection.SampleData.logger = Logger.getLogger(SampleData.class.getName())</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">static</span><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a214cced67761f0090ba326708620d3be"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">TranscriptomeSpaceAlignmentModel broad.pda.seq.protection.SampleData.maxFragmentLengthData</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a5b0011ca04c2ee52ec32f8508b82ae99"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">String broad.pda.seq.protection.SampleData.sampleName</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="ab5fe21e8281091d1be8de525e17d8a3a"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">int broad.pda.seq.protection.SampleData.stepSize</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="a6a53d104b93427859b6fd2c4ffbfdf6d"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">Map&lt;Gene, Map&lt;Annotation, ScanStatisticScore&gt; &gt; broad.pda.seq.protection.SampleData.windowScores</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<a class="anchor" id="af155a04e70a9feafd812aa6135f5b732"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">int broad.pda.seq.protection.SampleData.windowSize</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">protected</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

</div>
</div>
<hr/>The documentation for this class was generated from the following file:<ul>
<li>java/broad/pda/seq/protection/<a class="el" href="protection_2_sample_data_8java.html">SampleData.java</a></li>
</ul>
</div><!-- contents -->
<!-- start footer part -->
<hr class="footer"/><address class="footer"><small>
Generated on Fri Aug 15 2014 15:11:03 for ESAT by &#160;<a href="http://www.doxygen.org/index.html">
<img class="footer" src="doxygen.png" alt="doxygen"/>
</a> 1.8.7
</small></address>
</body>
</html>
