<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=9"/>
<meta name="generator" content="Doxygen 1.8.7"/>
<title>ESAT: broad.pda.seq.clip.Pileup Class Reference</title>
<link href="tabs.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="jquery.js"></script>
<script type="text/javascript" src="dynsections.js"></script>
<link href="search/search.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="search/search.js"></script>
<script type="text/javascript">
  $(document).ready(function() { searchBox.OnSelectItem(0); });
</script>
<link href="doxygen.css" rel="stylesheet" type="text/css" />
</head>
<body>
<div id="top"><!-- do not remove this div, it is closed by doxygen! -->
<div id="titlearea">
<table cellspacing="0" cellpadding="0">
 <tbody>
 <tr style="height: 56px;">
  <td style="padding-left: 0.5em;">
   <div id="projectname">ESAT
   </div>
  </td>
 </tr>
 </tbody>
</table>
</div>
<!-- end header part -->
<!-- Generated by Doxygen 1.8.7 -->
<script type="text/javascript">
var searchBox = new SearchBox("searchBox", "search",false,'Search');
</script>
  <div id="navrow1" class="tabs">
    <ul class="tablist">
      <li><a href="index.html"><span>Main&#160;Page</span></a></li>
      <li><a href="pages.html"><span>Related&#160;Pages</span></a></li>
      <li><a href="namespaces.html"><span>Packages</span></a></li>
      <li class="current"><a href="annotated.html"><span>Classes</span></a></li>
      <li><a href="files.html"><span>Files</span></a></li>
      <li>
        <div id="MSearchBox" class="MSearchBoxInactive">
        <span class="left">
          <img id="MSearchSelect" src="search/mag_sel.png"
               onmouseover="return searchBox.OnSearchSelectShow()"
               onmouseout="return searchBox.OnSearchSelectHide()"
               alt=""/>
          <input type="text" id="MSearchField" value="Search" accesskey="S"
               onfocus="searchBox.OnSearchFieldFocus(true)" 
               onblur="searchBox.OnSearchFieldFocus(false)" 
               onkeyup="searchBox.OnSearchFieldChange(event)"/>
          </span><span class="right">
            <a id="MSearchClose" href="javascript:searchBox.CloseResultsWindow()"><img id="MSearchCloseImg" border="0" src="search/close.png" alt=""/></a>
          </span>
        </div>
      </li>
    </ul>
  </div>
  <div id="navrow2" class="tabs2">
    <ul class="tablist">
      <li><a href="annotated.html"><span>Class&#160;List</span></a></li>
      <li><a href="classes.html"><span>Class&#160;Index</span></a></li>
      <li><a href="inherits.html"><span>Class&#160;Hierarchy</span></a></li>
      <li><a href="functions.html"><span>Class&#160;Members</span></a></li>
    </ul>
  </div>
<!-- window showing the filter options -->
<div id="MSearchSelectWindow"
     onmouseover="return searchBox.OnSearchSelectShow()"
     onmouseout="return searchBox.OnSearchSelectHide()"
     onkeydown="return searchBox.OnSearchSelectKey(event)">
<a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(0)"><span class="SelectionMark">&#160;</span>All</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(1)"><span class="SelectionMark">&#160;</span>Classes</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(2)"><span class="SelectionMark">&#160;</span>Namespaces</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(3)"><span class="SelectionMark">&#160;</span>Files</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(4)"><span class="SelectionMark">&#160;</span>Functions</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(5)"><span class="SelectionMark">&#160;</span>Variables</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(6)"><span class="SelectionMark">&#160;</span>Enumerator</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(7)"><span class="SelectionMark">&#160;</span>Pages</a></div>

<!-- iframe showing the search results (closed by default) -->
<div id="MSearchResultsWindow">
<iframe src="javascript:void(0)" frameborder="0" 
        name="MSearchResults" id="MSearchResults">
</iframe>
</div>

<div id="nav-path" class="navpath">
  <ul>
<li class="navelem"><a class="el" href="namespacebroad.html">broad</a></li><li class="navelem"><a class="el" href="namespacebroad_1_1pda.html">pda</a></li><li class="navelem"><a class="el" href="namespacebroad_1_1pda_1_1seq.html">seq</a></li><li class="navelem"><a class="el" href="namespacebroad_1_1pda_1_1seq_1_1clip.html">clip</a></li><li class="navelem"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup.html">Pileup</a></li>  </ul>
</div>
</div><!-- top -->
<div class="header">
  <div class="summary">
<a href="#pub-methods">Public Member Functions</a> &#124;
<a href="#pub-static-methods">Static Public Member Functions</a> &#124;
<a href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup-members.html">List of all members</a>  </div>
  <div class="headertitle">
<div class="title">broad.pda.seq.clip.Pileup Class Reference</div>  </div>
</div><!--header-->
<div class="contents">
<div class="dynheader">
Collaboration diagram for broad.pda.seq.clip.Pileup:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup__coll__graph.png" border="0" usemap="#broad_8pda_8seq_8clip_8_pileup_coll__map" alt="Collaboration graph"/></div>
<map name="broad_8pda_8seq_8clip_8_pileup_coll__map" id="broad_8pda_8seq_8clip_8_pileup_coll__map">
</map>
<center><span class="legend">[<a href="graph_legend.html">legend</a>]</span></center></div>
<table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="pub-methods"></a>
Public Member Functions</h2></td></tr>
<tr class="memitem:a21189ad218d6b720b568c74b4270884d"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup.html#a21189ad218d6b720b568c74b4270884d">Pileup</a> (String bamFile, String annotationBedFile, String subregionBedFile, String genomeFasta, Collection&lt; Predicate&lt; Alignment &gt;&gt; readFilters)  throws IOException </td></tr>
<tr class="separator:a21189ad218d6b720b568c74b4270884d"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:af08c5950bd4da328783f0d86f773b847"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup.html#af08c5950bd4da328783f0d86f773b847">Pileup</a> (String bamFile, String fragmentEndWig, String annotationBedFile, String subregionBedFile, String genomeFasta, String regionOverlapperCountFile, String peakRegionBedFile, Collection&lt; Predicate&lt; Alignment &gt;&gt; readFilters)  throws IOException </td></tr>
<tr class="separator:af08c5950bd4da328783f0d86f773b847"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a3192137e746edd357a296f5d823f8e2f"><td class="memItemLeft" align="right" valign="top">Map&lt; Annotation, Collection<br class="typebreak" />
&lt; <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html">EmpiricalDistribution</a> &gt; &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup.html#a3192137e746edd357a296f5d823f8e2f">getShuffledPositionsAllRegions</a> (boolean fullyContained, Map&lt; String, Double &gt; nucProbs, int numPermutations)</td></tr>
<tr class="separator:a3192137e746edd357a296f5d823f8e2f"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a75369d564eda2373e41b0fd60451fda8"><td class="memItemLeft" align="right" valign="top">Map&lt; Annotation, Double &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup.html#a75369d564eda2373e41b0fd60451fda8">getScanPvalFragmentEndsMaxPositionAllRegionsBam</a> (boolean fullyContained, Map&lt; String, Double &gt; nucProbs, int numPermutations, boolean fivePrimeEnd)</td></tr>
<tr class="separator:a75369d564eda2373e41b0fd60451fda8"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a2f2ec59cc7fdb4ced9c11014070fa10a"><td class="memItemLeft" align="right" valign="top">Map&lt; Integer, Double &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup.html#a2f2ec59cc7fdb4ced9c11014070fa10a">getScanPvalsFragmentEndPileupsWig</a> (Annotation region, boolean fullyContained, Map&lt; String, Double &gt; nucProbs, int numPermutations)</td></tr>
<tr class="separator:a2f2ec59cc7fdb4ced9c11014070fa10a"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ae5bf54f9ea963c4d704bb7f2dd98c0ee"><td class="memItemLeft" align="right" valign="top">Map&lt; Annotation, Map&lt; Integer, <br class="typebreak" />
Double &gt; &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup.html#ae5bf54f9ea963c4d704bb7f2dd98c0ee">getScanPvalsFragmentEndPileupsAllRegionsWig</a> (boolean fullyContained, Map&lt; String, Double &gt; nucProbs, int numPermutations)</td></tr>
<tr class="separator:ae5bf54f9ea963c4d704bb7f2dd98c0ee"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:ac4e8a5dd5705c5e479d1ed70e1d1852f"><td class="memItemLeft" align="right" valign="top">void&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup.html#ac4e8a5dd5705c5e479d1ed70e1d1852f">writeScanPvalsFragmentEndPileupsAllRegionsWig</a> (String outTable, String outBed, double pvalCutoffForBed, boolean fullyContained, Map&lt; String, Double &gt; nucProbs, int numPermutations)  throws IOException </td></tr>
<tr class="separator:ac4e8a5dd5705c5e479d1ed70e1d1852f"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:af7726e161df33f1675b770110193078f"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup.html#af7726e161df33f1675b770110193078f">getScanPvalFragmentEndsMaxPositionBam</a> (Annotation region, boolean fullyContained, Map&lt; String, Double &gt; nucProbs, int numPermutations, boolean fivePrimeEnd)</td></tr>
<tr class="separator:af7726e161df33f1675b770110193078f"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a9da41d6224e8d98d57989bf58c16190a"><td class="memItemLeft" align="right" valign="top">Collection&lt; <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html">EmpiricalDistribution</a> &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup.html#a9da41d6224e8d98d57989bf58c16190a">getShuffledPositions</a> (Annotation region, boolean fullyContained, Map&lt; String, Double &gt; nucProbs, int numPermutations)</td></tr>
<tr class="separator:a9da41d6224e8d98d57989bf58c16190a"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a9d35efcf06921f02594aef226c6862d4"><td class="memItemLeft" align="right" valign="top"><a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html">EmpiricalDistribution</a>&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup.html#a9d35efcf06921f02594aef226c6862d4">getFragmentEndsFromBamFile</a> (Annotation region, boolean fullyContained, boolean fivePrimeEnd)</td></tr>
<tr class="separator:a9d35efcf06921f02594aef226c6862d4"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a25885ff6ffd5befe82971e505b35ce81"><td class="memItemLeft" align="right" valign="top">Map&lt; String, Collection&lt; Gene &gt; &gt;&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup.html#a25885ff6ffd5befe82971e505b35ce81">getAnnotation</a> ()</td></tr>
<tr class="separator:a25885ff6ffd5befe82971e505b35ce81"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table><table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="pub-static-methods"></a>
Static Public Member Functions</h2></td></tr>
<tr class="memitem:adb48168b821ea257a6510ed8af7cd09a"><td class="memItemLeft" align="right" valign="top">static void&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup.html#adb48168b821ea257a6510ed8af7cd09a">main</a> (String[] args)  throws IOException </td></tr>
<tr class="separator:adb48168b821ea257a6510ed8af7cd09a"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table>
<a name="details" id="details"></a><h2 class="groupheader">Detailed Description</h2>
<div class="textblock"><dl class="section author"><dt>Author</dt><dd>prussell </dd></dl>
</div><h2 class="groupheader">Constructor &amp; Destructor Documentation</h2>
<a class="anchor" id="a21189ad218d6b720b568c74b4270884d"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">broad.pda.seq.clip.Pileup.Pileup </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>bamFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>annotationBedFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>subregionBedFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>genomeFasta</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Collection&lt; Predicate&lt; Alignment &gt;&gt;&#160;</td>
          <td class="paramname"><em>readFilters</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td> throws IOException</td>
        </tr>
      </table>
</div><div class="memdoc">
<dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">bamFile</td><td>Bam file of alignments </td></tr>
    <tr><td class="paramname">annotationBedFile</td><td>Bed transcriptome annotation </td></tr>
    <tr><td class="paramname">genomeFasta</td><td>Fasta file of genome </td></tr>
    <tr><td class="paramname">readFilters</td><td>Read filters to use </td></tr>
  </table>
  </dd>
</dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td></td></tr>
  </table>
  </dd>
</dl>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a21189ad218d6b720b568c74b4270884d_icgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a21189ad218d6b720b568c74b4270884d_icgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a21189ad218d6b720b568c74b4270884d_icgraph" id="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a21189ad218d6b720b568c74b4270884d_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="af08c5950bd4da328783f0d86f773b847"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">broad.pda.seq.clip.Pileup.Pileup </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>bamFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>fragmentEndWig</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>annotationBedFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>subregionBedFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>genomeFasta</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>regionOverlapperCountFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>peakRegionBedFile</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Collection&lt; Predicate&lt; Alignment &gt;&gt;&#160;</td>
          <td class="paramname"><em>readFilters</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td> throws IOException</td>
        </tr>
      </table>
</div><div class="memdoc">
<dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">bamFile</td><td>Bam file of alignments </td></tr>
    <tr><td class="paramname">annotationBedFile</td><td>Bed transcriptome annotation </td></tr>
    <tr><td class="paramname">genomeFasta</td><td>Fasta file of genome </td></tr>
    <tr><td class="paramname">regionOverlapperCountFile</td><td>File of overlapper counts for regions (can be incomplete) </td></tr>
    <tr><td class="paramname">peakRegionBedFile</td><td>Bed file of peak region (subregion of the parent gene enclosing all reads overlapping the region) corresponding to each region. Bed name is the same as region name. </td></tr>
    <tr><td class="paramname">readFilters</td><td>Read filters to use </td></tr>
  </table>
  </dd>
</dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td></td></tr>
  </table>
  </dd>
</dl>

</div>
</div>
<h2 class="groupheader">Member Function Documentation</h2>
<a class="anchor" id="a25885ff6ffd5befe82971e505b35ce81"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">Map&lt;String, Collection&lt;Gene&gt; &gt; broad.pda.seq.clip.Pileup.getAnnotation </td>
          <td>(</td>
          <td class="paramname"></td><td>)</td>
          <td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get the transcriptome annotation </p><dl class="section return"><dt>Returns</dt><dd>The annotation </dd></dl>

</div>
</div>
<a class="anchor" id="a9d35efcf06921f02594aef226c6862d4"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname"><a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html">EmpiricalDistribution</a> broad.pda.seq.clip.Pileup.getFragmentEndsFromBamFile </td>
          <td>(</td>
          <td class="paramtype">Annotation&#160;</td>
          <td class="paramname"><em>region</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fullyContained</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fivePrimeEnd</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get an empirical distribution storing the fragment ends of all fragments overlapping the region Fragment ends are in transcript coordinates with respect to the parent transcript </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">parent</td><td>Parent gene </td></tr>
    <tr><td class="paramname">region</td><td>Region within parent to get pileup over </td></tr>
    <tr><td class="paramname">fullyContained</td><td>Only include fragments that are fully contained in the subregion </td></tr>
    <tr><td class="paramname">fivePrimeEnd</td><td>Report the 5' end of fragments (if false, reports 3' end) </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Distribution of fragment endpoints in transcript coordinates </dd></dl>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a9d35efcf06921f02594aef226c6862d4_icgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a9d35efcf06921f02594aef226c6862d4_icgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a9d35efcf06921f02594aef226c6862d4_icgraph" id="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a9d35efcf06921f02594aef226c6862d4_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a75369d564eda2373e41b0fd60451fda8"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">Map&lt;Annotation, Double&gt; broad.pda.seq.clip.Pileup.getScanPvalFragmentEndsMaxPositionAllRegionsBam </td>
          <td>(</td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fullyContained</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Map&lt; String, Double &gt;&#160;</td>
          <td class="paramname"><em>nucProbs</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>numPermutations</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fivePrimeEnd</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>For each region, find the highest fragment end pileup from the bam file. Randomly permute the fragments in a biased way according to background nucleotide probabilities. Get a scan P value for the real highest pileup in the region. </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">fullyContained</td><td>When computing number of fragments and peak region, include fully contained fragments only </td></tr>
    <tr><td class="paramname">nucProbs</td><td>Background nucleotide probabilities </td></tr>
    <tr><td class="paramname">numPermutations</td><td>Number of permutations per region </td></tr>
    <tr><td class="paramname">fivePrimeEnd</td><td>Use the 5' end of fragments (if false, use 3' end) </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Map of region to P value </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a75369d564eda2373e41b0fd60451fda8_cgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a75369d564eda2373e41b0fd60451fda8_cgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a75369d564eda2373e41b0fd60451fda8_cgraph" id="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a75369d564eda2373e41b0fd60451fda8_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="af7726e161df33f1675b770110193078f"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">double broad.pda.seq.clip.Pileup.getScanPvalFragmentEndsMaxPositionBam </td>
          <td>(</td>
          <td class="paramtype">Annotation&#160;</td>
          <td class="paramname"><em>region</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fullyContained</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Map&lt; String, Double &gt;&#160;</td>
          <td class="paramname"><em>nucProbs</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>numPermutations</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fivePrimeEnd</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Find the highest fragment end pileup in the region from the bam file. Randomly permute the fragments in a biased way according to background nucleotide probabilities. Get a scan P value for the real highest pileup in the region. </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">region</td><td>The region </td></tr>
    <tr><td class="paramname">fullyContained</td><td>When computing number of fragments and peak region, include fully contained fragments only </td></tr>
    <tr><td class="paramname">nucProbs</td><td>Background nucleotide probabilities </td></tr>
    <tr><td class="paramname">numPermutations</td><td>Number of permutations per region </td></tr>
    <tr><td class="paramname">fivePrimeEnd</td><td>Use the 5' end of fragments (if false, use 3' end) </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Map of region to P value </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_af7726e161df33f1675b770110193078f_cgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_af7726e161df33f1675b770110193078f_cgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_af7726e161df33f1675b770110193078f_cgraph" id="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_af7726e161df33f1675b770110193078f_cgraph">
</map>
</div>
</p>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_af7726e161df33f1675b770110193078f_icgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_af7726e161df33f1675b770110193078f_icgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_af7726e161df33f1675b770110193078f_icgraph" id="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_af7726e161df33f1675b770110193078f_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="ae5bf54f9ea963c4d704bb7f2dd98c0ee"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">Map&lt;Annotation, Map&lt;Integer, Double&gt; &gt; broad.pda.seq.clip.Pileup.getScanPvalsFragmentEndPileupsAllRegionsWig </td>
          <td>(</td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fullyContained</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Map&lt; String, Double &gt;&#160;</td>
          <td class="paramname"><em>nucProbs</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>numPermutations</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Randomly permute the fragments in a biased way according to background nucleotide probabilities. For each region get a scan P value for pileup height of each position in the region as read from wig file. </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">fullyContained</td><td>If computing peak regions from the bam file, include fully contained fragments only </td></tr>
    <tr><td class="paramname">nucProbs</td><td>Background nucleotide probabilities </td></tr>
    <tr><td class="paramname">numPermutations</td><td>Number of permutations per region </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Map of region to scan P value for each position in region </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_ae5bf54f9ea963c4d704bb7f2dd98c0ee_cgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_ae5bf54f9ea963c4d704bb7f2dd98c0ee_cgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_ae5bf54f9ea963c4d704bb7f2dd98c0ee_cgraph" id="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_ae5bf54f9ea963c4d704bb7f2dd98c0ee_cgraph">
</map>
</div>
</p>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_ae5bf54f9ea963c4d704bb7f2dd98c0ee_icgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_ae5bf54f9ea963c4d704bb7f2dd98c0ee_icgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_ae5bf54f9ea963c4d704bb7f2dd98c0ee_icgraph" id="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_ae5bf54f9ea963c4d704bb7f2dd98c0ee_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a2f2ec59cc7fdb4ced9c11014070fa10a"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">Map&lt;Integer, Double&gt; broad.pda.seq.clip.Pileup.getScanPvalsFragmentEndPileupsWig </td>
          <td>(</td>
          <td class="paramtype">Annotation&#160;</td>
          <td class="paramname"><em>region</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fullyContained</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Map&lt; String, Double &gt;&#160;</td>
          <td class="paramname"><em>nucProbs</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>numPermutations</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Randomly permute the fragments in a biased way according to background nucleotide probabilities. Get a scan P value for pileup height of each position in the region as read from wig file. </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">region</td><td>The region </td></tr>
    <tr><td class="paramname">fullyContained</td><td>If computing peak region from the bam file, include fully contained fragments only </td></tr>
    <tr><td class="paramname">nucProbs</td><td>Background nucleotide probabilities </td></tr>
    <tr><td class="paramname">numPermutations</td><td>Number of permutations </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Scan P value for each position in region </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a2f2ec59cc7fdb4ced9c11014070fa10a_cgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a2f2ec59cc7fdb4ced9c11014070fa10a_cgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a2f2ec59cc7fdb4ced9c11014070fa10a_cgraph" id="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a2f2ec59cc7fdb4ced9c11014070fa10a_cgraph">
</map>
</div>
</p>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a2f2ec59cc7fdb4ced9c11014070fa10a_icgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a2f2ec59cc7fdb4ced9c11014070fa10a_icgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a2f2ec59cc7fdb4ced9c11014070fa10a_icgraph" id="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a2f2ec59cc7fdb4ced9c11014070fa10a_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a9da41d6224e8d98d57989bf58c16190a"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">Collection&lt;<a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html">EmpiricalDistribution</a>&gt; broad.pda.seq.clip.Pileup.getShuffledPositions </td>
          <td>(</td>
          <td class="paramtype">Annotation&#160;</td>
          <td class="paramname"><em>region</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fullyContained</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Map&lt; String, Double &gt;&#160;</td>
          <td class="paramname"><em>nucProbs</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>numPermutations</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Get shuffled sets of positions within the span covered by fragments overlapping the region Each set contains a number of positions equal to the number of fragments overlapping the region Random shuffled positions are biased by nucleotide </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">region</td><td>Region within parent gene </td></tr>
    <tr><td class="paramname">fullyContained</td><td>When getting the span covered by fragments overlapping the region, include fully contained fragments only </td></tr>
    <tr><td class="paramname">nucProbs</td><td>Background probabilities of each nucleotide; must add up to 1 </td></tr>
    <tr><td class="paramname">numPermutations</td><td>Number of random sets to get </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Collection of random position sets; each set is presented in an empirical distribution </dd></dl>

<p><div class="dynheader">
Here is the caller graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a9da41d6224e8d98d57989bf58c16190a_icgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a9da41d6224e8d98d57989bf58c16190a_icgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a9da41d6224e8d98d57989bf58c16190a_icgraph" id="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a9da41d6224e8d98d57989bf58c16190a_icgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="a3192137e746edd357a296f5d823f8e2f"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">Map&lt;Annotation, Collection&lt;<a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html">EmpiricalDistribution</a>&gt; &gt; broad.pda.seq.clip.Pileup.getShuffledPositionsAllRegions </td>
          <td>(</td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fullyContained</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Map&lt; String, Double &gt;&#160;</td>
          <td class="paramname"><em>nucProbs</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>numPermutations</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
</div><div class="memdoc">
<p>For each region get several shufflings of the positions </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">fullyContained</td><td>When computing number of fragments and peak region, include fully contained fragments only </td></tr>
    <tr><td class="paramname">nucProbs</td><td>Background nucleotide probabilities </td></tr>
    <tr><td class="paramname">numPermutations</td><td>Number of permutations per region </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Map of region to collection of random permutations </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a3192137e746edd357a296f5d823f8e2f_cgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a3192137e746edd357a296f5d823f8e2f_cgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a3192137e746edd357a296f5d823f8e2f_cgraph" id="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_a3192137e746edd357a296f5d823f8e2f_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="adb48168b821ea257a6510ed8af7cd09a"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">static void broad.pda.seq.clip.Pileup.main </td>
          <td>(</td>
          <td class="paramtype">String[]&#160;</td>
          <td class="paramname"><em>args</em></td><td>)</td>
          <td> throws IOException</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">static</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">
<dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">args</td><td></td></tr>
  </table>
  </dd>
</dl>
<dl class="exception"><dt>Exceptions</dt><dd>
  <table class="exception">
    <tr><td class="paramname">IOException</td><td></td></tr>
  </table>
  </dd>
</dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_adb48168b821ea257a6510ed8af7cd09a_cgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_adb48168b821ea257a6510ed8af7cd09a_cgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_adb48168b821ea257a6510ed8af7cd09a_cgraph" id="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_adb48168b821ea257a6510ed8af7cd09a_cgraph">
</map>
</div>
</p>

</div>
</div>
<a class="anchor" id="ac4e8a5dd5705c5e479d1ed70e1d1852f"></a>
<div class="memitem">
<div class="memproto">
      <table class="memname">
        <tr>
          <td class="memname">void broad.pda.seq.clip.Pileup.writeScanPvalsFragmentEndPileupsAllRegionsWig </td>
          <td>(</td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>outTable</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">String&#160;</td>
          <td class="paramname"><em>outBed</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">double&#160;</td>
          <td class="paramname"><em>pvalCutoffForBed</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">boolean&#160;</td>
          <td class="paramname"><em>fullyContained</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">Map&lt; String, Double &gt;&#160;</td>
          <td class="paramname"><em>nucProbs</em>, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">int&#160;</td>
          <td class="paramname"><em>numPermutations</em>&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td> throws IOException</td>
        </tr>
      </table>
</div><div class="memdoc">
<p>Randomly permute the fragments in a biased way according to background nucleotide probabilities. For each region get a scan P value for pileup height of each position in the region as read from wig file. Write information to a table. </p><dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">outTable</td><td>Output table file </td></tr>
    <tr><td class="paramname">outBed</td><td>Bed file for significant positions </td></tr>
    <tr><td class="paramname">pvalCutoffForBed</td><td>P value cutoff for significant position </td></tr>
    <tr><td class="paramname">fullyContained</td><td>If computing peak regions from the bam file, include fully contained fragments only </td></tr>
    <tr><td class="paramname">nucProbs</td><td>Background nucleotide probabilities </td></tr>
    <tr><td class="paramname">numPermutations</td><td>Number of permutations per region </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Map of region to scan P value for each position in region </dd></dl>

<p><div class="dynheader">
Here is the call graph for this function:</div>
<div class="dyncontent">
<div class="center"><img src="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_ac4e8a5dd5705c5e479d1ed70e1d1852f_cgraph.png" border="0" usemap="#classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_ac4e8a5dd5705c5e479d1ed70e1d1852f_cgraph" alt=""/></div>
<map name="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_ac4e8a5dd5705c5e479d1ed70e1d1852f_cgraph" id="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup_ac4e8a5dd5705c5e479d1ed70e1d1852f_cgraph">
</map>
</div>
</p>

</div>
</div>
<hr/>The documentation for this class was generated from the following file:<ul>
<li>java/broad/pda/seq/clip/<a class="el" href="_pileup_8java.html">Pileup.java</a></li>
</ul>
</div><!-- contents -->
<!-- start footer part -->
<hr class="footer"/><address class="footer"><small>
Generated on Fri Aug 15 2014 15:11:01 for ESAT by &#160;<a href="http://www.doxygen.org/index.html">
<img class="footer" src="doxygen.png" alt="doxygen"/>
</a> 1.8.7
</small></address>
</body>
</html>
