<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=9"/>
<meta name="generator" content="Doxygen 1.8.7"/>
<title>ESAT: Class Members</title>
<link href="tabs.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="jquery.js"></script>
<script type="text/javascript" src="dynsections.js"></script>
<link href="search/search.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="search/search.js"></script>
<script type="text/javascript">
  $(document).ready(function() { searchBox.OnSelectItem(0); });
</script>
<link href="doxygen.css" rel="stylesheet" type="text/css" />
</head>
<body>
<div id="top"><!-- do not remove this div, it is closed by doxygen! -->
<div id="titlearea">
<table cellspacing="0" cellpadding="0">
 <tbody>
 <tr style="height: 56px;">
  <td style="padding-left: 0.5em;">
   <div id="projectname">ESAT
   </div>
  </td>
 </tr>
 </tbody>
</table>
</div>
<!-- end header part -->
<!-- Generated by Doxygen 1.8.7 -->
<script type="text/javascript">
var searchBox = new SearchBox("searchBox", "search",false,'Search');
</script>
  <div id="navrow1" class="tabs">
    <ul class="tablist">
      <li><a href="index.html"><span>Main&#160;Page</span></a></li>
      <li><a href="pages.html"><span>Related&#160;Pages</span></a></li>
      <li><a href="namespaces.html"><span>Packages</span></a></li>
      <li class="current"><a href="annotated.html"><span>Classes</span></a></li>
      <li><a href="files.html"><span>Files</span></a></li>
      <li>
        <div id="MSearchBox" class="MSearchBoxInactive">
        <span class="left">
          <img id="MSearchSelect" src="search/mag_sel.png"
               onmouseover="return searchBox.OnSearchSelectShow()"
               onmouseout="return searchBox.OnSearchSelectHide()"
               alt=""/>
          <input type="text" id="MSearchField" value="Search" accesskey="S"
               onfocus="searchBox.OnSearchFieldFocus(true)" 
               onblur="searchBox.OnSearchFieldFocus(false)" 
               onkeyup="searchBox.OnSearchFieldChange(event)"/>
          </span><span class="right">
            <a id="MSearchClose" href="javascript:searchBox.CloseResultsWindow()"><img id="MSearchCloseImg" border="0" src="search/close.png" alt=""/></a>
          </span>
        </div>
      </li>
    </ul>
  </div>
  <div id="navrow2" class="tabs2">
    <ul class="tablist">
      <li><a href="annotated.html"><span>Class&#160;List</span></a></li>
      <li><a href="classes.html"><span>Class&#160;Index</span></a></li>
      <li><a href="inherits.html"><span>Class&#160;Hierarchy</span></a></li>
      <li class="current"><a href="functions.html"><span>Class&#160;Members</span></a></li>
    </ul>
  </div>
  <div id="navrow3" class="tabs2">
    <ul class="tablist">
      <li class="current"><a href="functions.html"><span>All</span></a></li>
      <li><a href="functions_func.html"><span>Functions</span></a></li>
      <li><a href="functions_vars.html"><span>Variables</span></a></li>
      <li><a href="functions_eval.html"><span>Enumerator</span></a></li>
    </ul>
  </div>
  <div id="navrow4" class="tabs3">
    <ul class="tablist">
      <li><a href="functions.html#index_a"><span>a</span></a></li>
      <li><a href="functions_b.html#index_b"><span>b</span></a></li>
      <li><a href="functions_c.html#index_c"><span>c</span></a></li>
      <li><a href="functions_d.html#index_d"><span>d</span></a></li>
      <li><a href="functions_e.html#index_e"><span>e</span></a></li>
      <li><a href="functions_f.html#index_f"><span>f</span></a></li>
      <li class="current"><a href="functions_g.html#index_g"><span>g</span></a></li>
      <li><a href="functions_h.html#index_h"><span>h</span></a></li>
      <li><a href="functions_i.html#index_i"><span>i</span></a></li>
      <li><a href="functions_j.html#index_j"><span>j</span></a></li>
      <li><a href="functions_k.html#index_k"><span>k</span></a></li>
      <li><a href="functions_l.html#index_l"><span>l</span></a></li>
      <li><a href="functions_m.html#index_m"><span>m</span></a></li>
      <li><a href="functions_n.html#index_n"><span>n</span></a></li>
      <li><a href="functions_o.html#index_o"><span>o</span></a></li>
      <li><a href="functions_p.html#index_p"><span>p</span></a></li>
      <li><a href="functions_q.html#index_q"><span>q</span></a></li>
      <li><a href="functions_r.html#index_r"><span>r</span></a></li>
      <li><a href="functions_s.html#index_s"><span>s</span></a></li>
      <li><a href="functions_t.html#index_t"><span>t</span></a></li>
      <li><a href="functions_u.html#index_u"><span>u</span></a></li>
      <li><a href="functions_v.html#index_v"><span>v</span></a></li>
      <li><a href="functions_w.html#index_w"><span>w</span></a></li>
      <li><a href="functions_z.html#index_z"><span>z</span></a></li>
    </ul>
  </div>
</div><!-- top -->
<!-- window showing the filter options -->
<div id="MSearchSelectWindow"
     onmouseover="return searchBox.OnSearchSelectShow()"
     onmouseout="return searchBox.OnSearchSelectHide()"
     onkeydown="return searchBox.OnSearchSelectKey(event)">
<a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(0)"><span class="SelectionMark">&#160;</span>All</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(1)"><span class="SelectionMark">&#160;</span>Classes</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(2)"><span class="SelectionMark">&#160;</span>Namespaces</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(3)"><span class="SelectionMark">&#160;</span>Files</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(4)"><span class="SelectionMark">&#160;</span>Functions</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(5)"><span class="SelectionMark">&#160;</span>Variables</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(6)"><span class="SelectionMark">&#160;</span>Enumerator</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(7)"><span class="SelectionMark">&#160;</span>Pages</a></div>

<!-- iframe showing the search results (closed by default) -->
<div id="MSearchResultsWindow">
<iframe src="javascript:void(0)" frameborder="0" 
        name="MSearchResults" id="MSearchResults">
</iframe>
</div>

<div class="contents">
<div class="textblock">Here is a list of all class members with links to the classes they belong to:</div>

<h3><a class="anchor" id="index_g"></a>- g -</h3><ul>
<li>G_CODE
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#a9dc0680bd4e9988d3fd95e3b29247246">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>GAP
: <a class="el" href="classjaligner_1_1_alignment.html#a5578c8eb1e46b49cba1ba8745a1c9726">jaligner.Alignment</a>
, <a class="el" href="classjaligner_1_1_markups.html#aeeccc407eb80259ed2fba80e03c03b98">jaligner.Markups</a>
</li>
<li>GAP_CODE
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#ab6be2a517b748bbf3c4fb8da3be81920">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>gaps()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_chromosome.html#a21507b12eda19aa0795eb32013a276da">broad.pda.feature.genome.Chromosome</a>
</li>
<li>gcContent()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a38dde2f2faa86fc1e71dd44878e8b078">broad.core.sequence.Sequence</a>
</li>
<li>GENE
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_genome_command_line_program.html#a992519fafb24df6e37ef1cf39f9f2176">broad.pda.seq.rap.GenomeCommandLineProgram</a>
</li>
<li>Gene()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a548a0f54773a11fdf300e222dfe10408">umms.core.annotation.Gene</a>
</li>
<li>GENE_BED
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1rna_1_1_transcriptome_command_line_program.html#a4f25520daa1a78a924387bb781d17ca9">broad.pda.seq.rap.rna.TranscriptomeCommandLineProgram</a>
</li>
<li>geneAvgCoverage
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#af41165298cda748416235bdaeef7ecf5">broad.pda.seq.clip.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#ad517adb3cfab02f8f65bc10818ebca71">broad.pda.seq.protection.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_window_scores.html#a0e6068b9b298dfc039ca118061817e0b">broad.pda.seq.protection.WindowScores</a>
</li>
<li>GeneCounts()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_counts.html#accb85c6f30274901690afd89bc0c3cc4">broad.pda.seq.segmentation.GeneCounts</a>
</li>
<li>generate()
: <a class="el" href="classjaligner_1_1matrix_1_1_matrix_generator.html#a18ab01e74080f02a981c04830d299ea0">jaligner.matrix.MatrixGenerator</a>
</li>
<li>generateAlignment()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_pi_h_m_m.html#a6fecf2a6ea9381a221e9f59c1afb65e3">broad.core.siphy.PiHMM</a>
</li>
<li>generateAll5Mers()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a36a9672e2487d0c7a9166eb8870ec650">broad.core.sequence.Sequence</a>
</li>
<li>generateAllKmers()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#aef0026e345f4683e26c1cb306adb6388">broad.core.sequence.Sequence</a>
</li>
<li>generateHKYQMatrix()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model_parameters.html#aeb7a76e925192294fa01f8f5d1801743">broad.core.siphy.EvolutionaryModelParameters</a>
</li>
<li>generateRandom()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_j_c_s_cache.html#afad1ffec78cbe93d6b122122aa08bc33">broad.core.datastructures.JCSCache</a>
</li>
<li>generateRandomSequence()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a63de9839d1efc73e189632b5ede83778">broad.core.sequence.Sequence</a>
</li>
<li>generateSubregions()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_plot_aggregate_regions.html#a346e5ff0aec512e849bdba8343db1b25">broad.pda.seq.rap.PlotAggregateRegions</a>
</li>
<li>GenericAlignmentDataModel()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a30b218ffe2c70e0bdfd2b45de8fc3488">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>GenericRootedTree()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_generic_rooted_tree_3_01_t_01_4.html#a28e41e478bbd1260bb2f5c962186f01d">broad.core.datastructures.GenericRootedTree&lt; T &gt;</a>
</li>
<li>genes
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#a3ec4b745c679b9c175b6b1fb43100f09">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_two_sample_scan_skellam_peak_caller.html#a04a2a9a97512a37b8b24c458e11c6fe3">broad.pda.seq.protection.TwoSampleScanSkellamPeakCaller</a>
</li>
<li>genesByChr
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a8cd7634b21c85e6a4e79e65083932b28">broad.pda.seq.clip.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a8b1f58470538adcd26f47252bdf94232">broad.pda.seq.protection.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_window_scores.html#aa32bcf4ca6a247e092ca644a0b958237">broad.pda.seq.protection.WindowScores</a>
</li>
<li>genesByName
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#ab1ed0ab3b3ab3a672ba6fec27eb0905e">broad.pda.seq.clip.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a528cf3b7c628a3f4a567bd33ba75ca0f">broad.pda.seq.protection.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_window_scores.html#ad1359f7b732916942358a4e3fb15e57c">broad.pda.seq.protection.WindowScores</a>
</li>
<li>GeneScore()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score.html#a64719ea8716d20908ad1b91dc845e7fe">broad.pda.seq.segmentation.GeneScore</a>
</li>
<li>geneScores
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a38dff572a12928939c8cefc3ad77b624">broad.pda.seq.clip.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#af611a295e308f4f0d0849a80139ad7ff">broad.pda.seq.protection.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_window_scores.html#a6692ac429e881e1420d05811d1b3f04e">broad.pda.seq.protection.WindowScores</a>
</li>
<li>GeneSetEnrichment()
: <a class="el" href="classbroad_1_1projection_1_1gsa_1_1_gene_set_enrichment.html#aa2889ec963f3f565fd36e5758b575162">broad.projection.gsa.GeneSetEnrichment</a>
</li>
<li>geneSpanContains()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a218a0602d404d12635adef7d0830c131">umms.core.annotation.Gene</a>
</li>
<li>GeneWindow()
: <a class="el" href="classumms_1_1core_1_1feature_1_1_gene_window.html#af3c1e96401bd9c2e0befc2ae05cd6fba">umms.core.feature.GeneWindow</a>
</li>
<li>GeneWithIsoforms()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_gene_with_isoforms.html#a701d20a3c35b334ae246b381d545d4d2">broad.pda.gene.GeneWithIsoforms</a>
</li>
<li>GENOME
: <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_paired_end_t_d_f.html#a6512b89abaa0a824d76c6eb7308790f3">broad.pda.samtools.PairedEndTDF</a>
</li>
<li>GenomeSpaceSampleData()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_genome_space_sample_data.html#adc0e201ce593f6d21b1f7b32c907f6a5">broad.pda.seq.clip.GenomeSpaceSampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_genome_space_sample_data.html#ab048b235d662fc6a9ebee44e59259542">broad.pda.seq.protection.GenomeSpaceSampleData</a>
</li>
<li>GenomeWindow()
: <a class="el" href="classumms_1_1core_1_1feature_1_1_genome_window.html#a0801705077c7f6ff230ef4c26515a307">umms.core.feature.GenomeWindow</a>
</li>
<li>GenomeWithGaps()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_genome_with_gaps.html#ac90360d08f6ac2fb48ac1d1a7723e93a">broad.pda.seq.segmentation.GenomeWithGaps</a>
</li>
<li>GenomeWithGaps2()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_genome_with_gaps2.html#a65534cb5315d04bdcf2365aa0dc9a10a">broad.pda.seq.segmentation.GenomeWithGaps2</a>
</li>
<li>GenomicSpace()
: <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_genomic_space.html#accf612fa76269c990309129a2eaac792">umms.core.coordinatesystem.GenomicSpace</a>
</li>
<li>GenomicSpanFilter()
: <a class="el" href="classnextgen_1_1core_1_1read_filters_1_1_genomic_span_filter.html#a1a9584b156a2ef42874cd4a46e271c6c">nextgen.core.readFilters.GenomicSpanFilter</a>
</li>
<li>genomicToGenomicPositionWithOffset()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a24c4baf757d1fec374f8d7cfad9f1578">umms.core.annotation.Gene</a>
</li>
<li>genomicToTranscriptPosition()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a30c16cb2d10856f49e7d79d15f2da795">umms.core.annotation.Gene</a>
</li>
<li>geometricMean()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_statistics.html#a24e4eabeb0b090b80b2f0b802f1d9d46">broad.core.math.Statistics</a>
</li>
<li>geometricMeanMAD()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_statistics.html#a7efc9182cf4cdf0af3478207f65cb56b">broad.core.math.Statistics</a>
</li>
<li>get()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#aac7a2395aebc37ee1d7f5fa84b888ffb">broad.core.datastructures.MatrixWithHeaders</a>
, <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a36e88558b56d49a7742ff7bcc8841485">broad.pda.annotation.BEDFileParser</a>
, <a class="el" href="class_jama_1_1_matrix.html#a4570cadbd1f4ccf87e21ecbcfe5f471f">Jama.Matrix</a>
</li>
<li>get3PRegion()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a592f7b9ad8d50ffd4487ecc496469bb4">broad.pda.annotation.BEDFileParser</a>
</li>
<li>get3Prime()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a961f5187cacc679ceba5a4f68d8a5e37">broad.core.sequence.Sequence</a>
</li>
<li>get3PrimeExon()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a402966c4490f62bffd11bbdff30aef78">umms.core.annotation.Gene</a>
</li>
<li>get3PrimeGenomicDistance()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#aa60cb4760bf8cb9b33974db3e2ae91b7">umms.core.annotation.Gene</a>
</li>
<li>get3Utr()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#ac0999b66a6a587bcf737ed8101f2d371">umms.core.annotation.Gene</a>
</li>
<li>get3UTR()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a5da62e5fc6fc17a00f28b91a2d692c3e">umms.core.annotation.Gene</a>
</li>
<li>get3UTRGene()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#afb42d4f238f4c58d94b75b74472083df">umms.core.annotation.Gene</a>
</li>
<li>get3UtrIncludingIntrons()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#aae0325e4bc1a847f48930fe1c62952bd">umms.core.annotation.Gene</a>
</li>
<li>get5PrimeExon()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#aa9b9af2ec900d1fcf2cb4754a5bdccdd">umms.core.annotation.Gene</a>
</li>
<li>get5UTR()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#af8460b1e69cc22e216a0c59da1e9dc88">umms.core.annotation.Gene</a>
</li>
<li>get5Utr()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a207204cc1cf3f4e8b1db5ceba1d00d89">umms.core.annotation.Gene</a>
</li>
<li>get5UTRGene()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#ab02249dcebe27ea4d2dc72a8632e27ce">umms.core.annotation.Gene</a>
</li>
<li>get5UtrIncludingIntrons()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#aeacc46ef311ac493dc1c5f1d82190a8e">umms.core.annotation.Gene</a>
</li>
<li>getA()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_two_subject_annotation.html#a2974021dd7ee2e275ba5de25259c8609">broad.core.annotation.BasicTwoSubjectAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_two_subject_annotation.html#a61054032506058267657aaf0725666ec">broad.core.annotation.TwoSubjectAnnotation</a>
</li>
<li>getAbsFDR()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_f_d_r_distribution.html#a9c06a0aaa14de7471871fa375ca9be0a">broad.core.math.FDRDistribution</a>
</li>
<li>getAbsFDRMatrix()
: <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_differential_expression.html#a8f86ffcb6419034fe32b23fbb84dce91">broad.pda.differentialExpression.DifferentialExpression</a>
</li>
<li>getAbsoluteEnd()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a0dc2e7da9f2c51472a0e3eac713f832d">broad.core.sequence.SequenceRegion</a>
</li>
<li>getAbsoluteStart()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a856ad86d2f8a2bd5ccd5199ab7773518">broad.core.sequence.SequenceRegion</a>
</li>
<li>getAEnd()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_two_subject_annotation.html#a8b278c9ac7ef63c5f79ee5279587747b">broad.core.annotation.BasicTwoSubjectAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_two_subject_annotation.html#a52f210c4bb64c12dbd512cb9f1f4a3c2">broad.core.annotation.TwoSubjectAnnotation</a>
</li>
<li>getAlignDir()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix.html#ad0d1df880623b3436db55e212fe61486">broad.core.motif.PositionWeightMatrix</a>
</li>
<li>getAlignedSequence()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_alignment.html#a6cbc6fde4d47f8b1a74569021d844f29">broad.core.multiplealignment.MAFAlignment</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#a26518e06adc60b6632eda3cf6d5626be">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>getAlignedSequenceIds()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_alignment.html#a3b0364eae278b8e4a3524ed723cd563f">broad.core.multiplealignment.MAFAlignment</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#a23b43077f9943897e9389ddc4f403d81">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>getAlignedSequences()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_alignment.html#a435da7e3532bbc433ec02f970ecf3707">broad.core.multiplealignment.MAFAlignment</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#a9b23bea8a4c978acfe6f4a2202bc8f57">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>getAlignment()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#a6cf246f0e21368221a8bff05ff4d0412">broad.pda.seq.graph.Path</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_paired_end_alignment_factory.html#ab93527d3f4c3e18d3c030fcc0772f240">umms.core.alignment.PairedEndAlignmentFactory</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#ab24320099ec493ae869b31bececfa418">umms.core.annotation.Gene</a>
</li>
<li>getAlignmentChunkSize()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_conserved_p_w_m_scanner.html#a8cd3bf3df6a1524920dd4da4a16fa5d7">broad.core.siphy.ConservedPWMScanner</a>
</li>
<li>getAlignmentDataModelStats()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#ac1217728072eb3635f3e7dea450af701">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>getAlignmentEnd()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#af6f367d03b371f8c11272b50edeb2166">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#a9b406dc941d7555254f9b6dcf2ae48da">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a1128bfdff4a5057310ea9470da3b7b56">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>getAlignmentIdxSizeAtCreation()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_stored_alignment_stats.html#a763365a8a09b955eb2143410c6749620">broad.pda.seq.alignment.StoredAlignmentStats</a>
</li>
<li>getAlignmentSizeAtCreation()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_stored_alignment_stats.html#ac38a0828bb7815a9cad14128db297e55">broad.pda.seq.alignment.StoredAlignmentStats</a>
</li>
<li>getAlignmentStart()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a287e7b5c0d789621212961e9a2dac052">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#ac6591781d5421244ad6518a4554f1760">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a9f7d7609476d41c9cff235e581fa4ef1">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>getAlignmentType()
: <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a6939ce3bb7ecb8dd46cc30bd0b88c492">umms.core.model.JCSAlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1readers_1_1_paired_end_reader.html#acf45dbe8546d2d0b09d778ded28e3ae6">umms.core.readers.PairedEndReader</a>
</li>
<li>getAllChromosomes()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_directory_installed_genome_assembly.html#add74b3876e83af8631034564c062779f">broad.pda.feature.genome.DirectoryInstalledGenomeAssembly</a>
</li>
<li>getAllDataValues()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#a87b93a0a097bfd7fa7b330b77075108e">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getAllIsoformExonSet()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_gene_with_isoforms.html#a10f9995cb3f615e7f85924ed2804773e">broad.pda.gene.GeneWithIsoforms</a>
</li>
<li>getAllIsoformIntronSet()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_gene_with_isoforms.html#a05c8b5e950abd2bc006a016d34074a55">broad.pda.gene.GeneWithIsoforms</a>
</li>
<li>getAllIsoforms()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#a19c4c5f51f36903524cc17e2a0f88158">broad.pda.annotation.Locus</a>
, <a class="el" href="classbroad_1_1pda_1_1gene_1_1_gene_with_isoforms.html#a1cd8024fd5795d9f98202731d0d58ba2">broad.pda.gene.GeneWithIsoforms</a>
</li>
<li>getAllMerged()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#a20a429f963554d39be7c3a02a410b534">broad.pda.annotation.Locus</a>
</li>
<li>getAllNonRandomChromosomes()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_directory_installed_genome_assembly.html#a40ce13a9c4f63ea6766e1b7a4762693c">broad.pda.feature.genome.DirectoryInstalledGenomeAssembly</a>
</li>
<li>getAllPairwiseAntisenseAlignments()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment.html#ab6151110c36c6f03905a49d791fd56f1">umms.core.alignment.FeatureSequenceAlignment</a>
</li>
<li>getAllPairwiseSenseAlignments()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment.html#a335945731d4a8ddcca3b130106172247">umms.core.alignment.FeatureSequenceAlignment</a>
</li>
<li>getAllPaths()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a38e4b4f6ca631176279b4718e960af5b">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>getAllPositions()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#ae3bffcffd420d44f6c6dfe9add7f6278">umms.core.annotation.Gene</a>
</li>
<li>getAllPossibleAmplicons()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1is_p_c_r_like.html#ac8403ef497e218582841fafa82b78a3a">broad.core.primer3.isPCRLike</a>
</li>
<li>getAllPossibleTargets()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_smatch_like.html#aa1900953c5ea1d5ab406fa083557e752">umms.core.alignment.SmatchLike</a>
</li>
<li>getAlphabetSize()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_column.html#a629eb4fe6442a462bc5b5c7352576364">broad.core.motif.PositionWeightColumn</a>
, <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model.html#a8a5a312efd9ed9fa6d0b10ad27521f1f">broad.core.siphy.EvolutionaryModel</a>
</li>
<li>getAMap()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_loess.html#a068ab174f57253dd822cc82776aec672">broad.core.math.Loess</a>
</li>
<li>getAName()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_two_subject_annotation.html#a222bca0ebc983c0c2f6c338a1d5056cd">broad.core.annotation.BasicTwoSubjectAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_two_subject_annotation.html#a37a57d624fcbd8c2d9de07716aaa6330">broad.core.annotation.TwoSubjectAnnotation</a>
</li>
<li>getAnnotation()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#a68641405ddb243b4315928dd9defc511">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup.html#a25885ff6ffd5befe82971e505b35ce81">broad.pda.seq.clip.Pileup</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_score_1_1_abstract_window_score.html#a1005010d00c6d09f70b39da815c3ee38">umms.core.model.score.WindowScore.AbstractWindowScore</a>
, <a class="el" href="interfaceumms_1_1core_1_1model_1_1score_1_1_window_score.html#ade117fa7e14c8e6839af41f8a9f1cbb2">umms.core.model.score.WindowScore</a>
</li>
<li>getAnnotationDirectory()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_directory_installed_genome_assembly.html#a12c802ee5de2070d4f01fd38ffade65e">broad.pda.feature.genome.DirectoryInstalledGenomeAssembly</a>
</li>
<li>getAnnotationList()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#ade7e4808675687f787ac8af08c2adb53">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
, <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_genomic_annotation_reader.html#ab9dfb0f9f5faecc81d703550f7c1afed">broad.core.annotation.BasicGenomicAnnotationReader</a>
</li>
<li>getAnnotationOverlappingRegion()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a2007c077986e1cbc0b656ad30e2c7b50">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#acacd0e090239a957152168f4e217b675">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a70112d8fb66b00e90e3e1c537ff152ce">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getAnnotationSetMap()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a1fa902beff864404c2aa999389078f1c">broad.pda.annotation.BEDFileParser</a>
</li>
<li>getAnnotationsForSequence()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#ad15cb7da92cfed836d3e012b5301efb7">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
</li>
<li>getAntisense()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a2b294328ff90285bc6e1efce02b0217f">broad.core.sequence.Sequence</a>
</li>
<li>getArgString()
: <a class="el" href="classbroad_1_1core_1_1parser_1_1_command_line_parser.html#af859e4c1934b19df011c55aa27c3d9ee">broad.core.parser.CommandLineParser</a>
</li>
<li>getArray()
: <a class="el" href="class_jama_1_1_matrix.html#a5129a7c385e943684b43d4bc2daa2b65">Jama.Matrix</a>
</li>
<li>getArrayCopy()
: <a class="el" href="class_jama_1_1_matrix.html#a518944310c17f3315c0a226ab711e1bf">Jama.Matrix</a>
</li>
<li>getAsMatrix()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#a2d26c015e85b447b9f1aa004e42fe191">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>getAStart()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_two_subject_annotation.html#ab73cb7aae4bc1fc6202d7a46d69ee757">broad.core.annotation.BasicTwoSubjectAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_two_subject_annotation.html#a53788b14799a78deac390727cb63903e">broad.core.annotation.TwoSubjectAnnotation</a>
</li>
<li>getAttribute()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a6ed9ab1244c6e54bc43ca6cb24f084af">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#ae8f96c807f69c9e0d48077998f9f338f">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a0b0ebd5fb452d61925959b90e7f55932">umms.core.alignment.SingleEndAlignment</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a5b5f51555eb2e4dcc81fe17fc6957487">umms.core.annotation.Gene</a>
</li>
<li>getAttributes()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_g_f_f.html#a5e628e04590803dd8cfdbf6df7a3875c">broad.core.annotation.GFF</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#abe7756cedde6fd478befeadedf018778">umms.core.annotation.Gene</a>
</li>
<li>getAutosomeGenomeSize()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_directory_installed_genome_assembly.html#aef7d0ff9b0a4ecb57110d3a356f517b9">broad.pda.feature.genome.DirectoryInstalledGenomeAssembly</a>
</li>
<li>getAutosomeNumber()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_directory_installed_genome_assembly.html#a8e47c044e18a1d686806e8553965287c">broad.pda.feature.genome.DirectoryInstalledGenomeAssembly</a>
</li>
<li>getAutosomes()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_directory_installed_genome_assembly.html#af979446fb4d640dfbf1e24750523a3bb">broad.pda.feature.genome.DirectoryInstalledGenomeAssembly</a>
</li>
<li>getAverageCountPerPosition()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a1eedf20d46a117cc86b53632a8cff0cb">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ac64d0167e26483541e58e6794968629d">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getAverageCountPerPositionWithThreshold()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#afdbc71f66c4a8e02cf59c6525c668ad0">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a2e4fdd6d362d1ccd58c61d6c1bbeccb1">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getAverageCoverage()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#af4195a976a2cfe99cf89cb79a3175b9d">umms.core.annotation.AbstractAnnotationCollection&lt; T extends Annotation &gt;</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#a23abd386efd728e409b2f8cd18d49afa">umms.core.annotation.AnnotationCollection&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_basic_scan_statistic_score.html#aeb57f75075173202d9b7e0520c566cdd">umms.core.model.score.BasicScanStatisticScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#a414f3019395acb555de36ce124778e2b">umms.core.model.score.MultiScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#af7c1c57de963db1269ac73fdc35c6395">umms.core.model.score.PoissonEnrichmentScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_scan_statistic_score.html#a55e96afcf845400a7fbca4eb67874791">umms.core.model.score.ScanStatisticScore</a>
</li>
<li>getAverageScore()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#ab47a5d591255960b69742d1ea010a04d">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
</li>
<li>getAvergeNumberOfReads()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score.html#a45fb0a85f59876c037b4d0fdd6730cfe">broad.pda.seq.segmentation.GeneScore</a>
</li>
<li>getB()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_two_subject_annotation.html#a2769947c2df223c61063f225e204cb61">broad.core.annotation.BasicTwoSubjectAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_two_subject_annotation.html#a7e6486d879fc7a4a98f292ed8fc8ebee">broad.core.annotation.TwoSubjectAnnotation</a>
</li>
<li>getBackgroundNucleotideFreqs()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model_parameters.html#a94dcc6d7e8cf0e6c79b209d881d2f07a">broad.core.siphy.EvolutionaryModelParameters</a>
</li>
<li>getBackwardEdges()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#affa018ef5e00f853738fd8a21f32c949">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>getBackwardPaths()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#aeaf8d68a1b24e160b4ec49198d8029e5">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>getBamFile()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#aa7aa575e64161143170f8840d920e438">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a28738b736bcc5e5248ea58c39477a6c4">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getBaseCoverage()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#a3f3fec809e8eaacb1a90cc0895197ee4">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
</li>
<li>getBaseIUB()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_column.html#a82b21c847d46a9ad142aace82bc115d2">broad.core.motif.PositionWeightColumn</a>
</li>
<li>getBasesCovered()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#a66f9bfe25e08fc0309ac119b99c6464d">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
, <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a1f32348c427920a6dc9fbd373ce0ffce">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#ad2358621505b89beef76afb69d366021">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a103697fa11dcbb0127954861e92cc6a9">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#a752a3547528015029ac6526b47e455f1">umms.core.annotation.AbstractAnnotationCollection&lt; T extends Annotation &gt;</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#a92eaa2a64a83682f3e9afda6a5a0db41">umms.core.annotation.AnnotationCollection&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4.html#aee85b7f16327c1a40a143042c76d133c">umms.core.annotation.AnnotationList&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a8b901c92b5e45e9c69796e0691700128">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#adb5028799b1713ae9d48a8dd64e3ec7c">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getBasesCoveredPerAlignment()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#af1d8d47cc4bc7e9bb6448e97dcbbc507">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a4581ee080760a52c56aa2fec9315367b">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getBatch()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1_expression_experiment_info.html#a0976a22c2b4b93d161aebffa64b0d3c6">broad.pda.geneexpression.ExpressionExperimentInfo</a>
</li>
<li>getBedScore()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#ab014794b9c11bfc0942483bb6b145573">umms.core.annotation.Gene</a>
</li>
<li>getBEnd()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_two_subject_annotation.html#aac9158eb1a4e07482821f574b09e3e1e">broad.core.annotation.BasicTwoSubjectAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_two_subject_annotation.html#a7466b3990a5910edcfd2be466d775dc5">broad.core.annotation.TwoSubjectAnnotation</a>
</li>
<li>getBin()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#a12f62582e15df016e650156de658c99a">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getBinEnd()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#a5bb9a84df2962122db60c0ddb1bd1784">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getBinMidPoint()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#a57aa10b8a192dac9d2a4ac4e90547c61">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getBinNumber()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#a1382490334a88b8e154afb8733868f6c">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getBinStart()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#aa50df0c29126b3eeea6773ac1c8e5572">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getBlastHit()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_hairpin_kmer.html#a6137c907877131fdc99e2ac9e29d5c57">broad.core.primer3.HairpinKmer</a>
</li>
<li>getBlockIterator()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_alignment.html#a7e31cd45e98145f4638106f2d3a54986">broad.core.multiplealignment.MAFAlignment</a>
</li>
<li>getBlocks()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_b_e_d.html#a695ef7ec4841b35e0b5decd4c3fa1e79">broad.core.annotation.BED</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_genomic_annotation.html#a6ffd3db0115b9db800bafe115aba87ea">broad.core.annotation.GenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a9d1b1a86babcedd2002d68746ea727e3">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#a1fc089c9a77adad8c174d36bf2452489">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a59ef0358ee04952f6ddd2c2bd4e448f7">umms.core.annotation.BasicAnnotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_compound_interval.html#a78e9d5bb2219d80cfc01c9f483458dda">umms.core.annotation.CompoundInterval</a>
</li>
<li>getBlockSizeStarts()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_b_e_d.html#ac211b2014e51647c781e3556d072df57">broad.core.annotation.BED</a>
</li>
<li>getBName()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_two_subject_annotation.html#ad64cbcfbea531c45e3555f2d9e3f0ecb">broad.core.annotation.BasicTwoSubjectAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_two_subject_annotation.html#ad92db50fa9151210d578dd61148e0e4d">broad.core.annotation.TwoSubjectAnnotation</a>
</li>
<li>getBooleanArg()
: <a class="el" href="classbroad_1_1core_1_1parser_1_1_command_line_parser.html#a5776bb75022365597c4f147f1c4f04d0">broad.core.parser.CommandLineParser</a>
</li>
<li>getBStart()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_two_subject_annotation.html#aa6b5cecc4eb5cafcf177a4fba1d95577">broad.core.annotation.BasicTwoSubjectAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_two_subject_annotation.html#a90ea6b2f236446912c96c69fdc9f9925">broad.core.annotation.TwoSubjectAnnotation</a>
</li>
<li>getCDNAs()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_genome_with_gaps.html#a9cdbea49c2cee0000d19d5f390841d16">broad.pda.seq.segmentation.GenomeWithGaps</a>
</li>
<li>getCDS()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#ae414757cb1720273e4f4a3188978eb49">umms.core.annotation.Gene</a>
</li>
<li>getCDSEnd()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a13e77e99c8170151fc7ab481f7e33318">umms.core.annotation.Gene</a>
</li>
<li>getCDSRegion()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a2bbe2bac12a9b9f09bfb2962e8a72d67">umms.core.annotation.Gene</a>
</li>
<li>getCDSStart()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#aefe341def25928fca0a33427ed3e1bee">umms.core.annotation.Gene</a>
</li>
<li>getChildren()
: <a class="el" href="classumms_1_1core_1_1utils_1_1_annotation_utils.html#a51fc4a9a678764303e6d94afc89df218">umms.core.utils.AnnotationUtils</a>
</li>
<li>getChr()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a83746b3145789378233bc9fc66df4f46">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a389ee80e6e6d04b7cbd55b1c533378e4">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#a156e2c6d3a8d3d7d9b8f50af90d0a0c2">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a52638f714aa84c73b0ef79e02bd9390b">umms.core.alignment.SingleEndAlignment</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation.html#abe1735bf83ddb3aae45c734149c50eb1">umms.core.annotation.AbstractAnnotation</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#ab07edf07fd2182cd16bceb91ee6de35a">umms.core.annotation.Annotation</a>
</li>
<li>getChrLength()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a02490a8b8be93112f028ad4a6e2f0fed">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getChrLengths()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a198d95d9393812ebeba0432f8ca4b840">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getChrNum()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#af827d888eaeea5171833f59e83832776">umms.core.annotation.Gene</a>
</li>
<li>getChromosome()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#a3c3c393894b2c4d90334dda13a148eec">broad.core.annotation.BasicLightweightAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation.html#a11b7345e12c3842e8b12530dfdf0e40c">broad.core.annotation.LightweightGenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a91d1cdf7cb2c148bf44e2ba57320b205">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_directory_installed_genome_assembly.html#a51ba62097bffbb4e5fe307283bf12610">broad.pda.feature.genome.DirectoryInstalledGenomeAssembly</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#a00cd44d8c338e3e487e678d2858024f5">broad.pda.seq.graph.Path</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a4e81aeaa83886f64c780bbedbb88a839">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a739c55573225c24e736b25965ed01251">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>getChromosomeAnnotationMap()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#a643eb0fb496db61f15a334d67e3f8bc8">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
</li>
<li>getChromosomeBEDs()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#a0c4976c92588ad35e262e90ebca4382f">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
</li>
<li>getChromosomeIterator()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#a939e2df3892ffc1a1a2c0684e3a21875">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
, <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a4d664c67f8afe6a6dafc6ddeca84c7f9">broad.pda.annotation.BEDFileParser</a>
</li>
<li>getChromosomeLength()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a58e438f245b1e95a11dd93e38c611f49">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
, <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a36c6800345f6dd84eade58ce23662dfe">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#aadb1c1dfc12a73268e0552fb99e01024">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a48905bee11e5ea811f31c97ddeb6901b">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getChromosomeLengths()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#acac093d995c75987a65f4e44151a11e8">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a103a170d04336fada8cc29e45352f465">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#af1b3646c528368d2b2af71fb820350a6">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a8400d889b8c00660927484a5182399d6">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getChromosomeNames()
: <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html#ae60b7522b8aa973dd48bae33aef80ff0">umms.core.coordinatesystem.CoordinateSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_genomic_space.html#afbc8bbc4fdf502dc86864b70c01e942e">umms.core.coordinatesystem.GenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_masked_genomic_space.html#a6034d7102a4450c672379f7b8973b9b5">umms.core.coordinatesystem.MaskedGenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcript_in_genomic_space.html#a0bfff97f8c289cb3d0c264a7d28a7da3">umms.core.coordinatesystem.TranscriptInGenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcriptome_space.html#a1605208e15a7edb288bfce529040f314">umms.core.coordinatesystem.TranscriptomeSpace</a>
</li>
<li>getChromosomeNumber()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_chromosome.html#ac46b99bc0b629718335526ac89002f86">broad.pda.feature.genome.Chromosome</a>
</li>
<li>getChromosomeReadIterator()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a708fef9e7fe5b4b9fdb00b8f78885677">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#aa2afa5c2affb8696efa547372286322f">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getChromosomes()
: <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_location_aware_matrix.html#a2e1340ec85a65aa076a4f1deacd9c049">broad.pda.datastructures.LocationAwareMatrix</a>
</li>
<li>getChromosomeString()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#a713c14e80c68be53c7175c08a4042f10">broad.core.annotation.BasicLightweightAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation.html#a5ab11d683bcfed2572c3914ee5ba5a24">broad.core.annotation.LightweightGenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a7ff291e7f7540b7707f500210ad50803">broad.core.sequence.SequenceRegion</a>
</li>
<li>getChromosomeTree()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#afee522dcd0f10dd65290b9dea988ecca">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
</li>
<li>getChromosomeTreeMap()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#a6af994265d45e558896a30af64bf37f3">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
</li>
<li>getChrReadIterator()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a24ff73af0c763387c20a8357c83fd5fc">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getChrSequence()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_utils.html#a209552755e74b65ff900a09d25c8387a">broad.core.sequence.SequenceUtils</a>
</li>
<li>getChrSequencesFromFasta()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_fasta_sequence_i_o.html#a933ca0d02c96ba0321e334ad3431a6cd">broad.core.sequence.FastaSequenceIO</a>
</li>
<li>getChrSizesFile()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_utils.html#a8e5ce4d144fbbc7b0d274e410cb25921">broad.core.sequence.SequenceUtils</a>
</li>
<li>getChrTree()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#ad9062000e1e13b19313a3959db8605bb">broad.pda.annotation.BEDFileParser</a>
</li>
<li>getChunkEnd()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a95d31d5d49223e148254478e9961effb">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a8c55306897ad0667ece361ffd2732180">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#ac3347ac18f32f257dbe64feb8e004663">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getChunkStart()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#ac0d7d4cd9a3c4a866c3e6cdad9f60992">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#acb34d9f586298e5583d1c113ff1af6b8">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a0b0c6a88acbace9be14ddf06c5a7953c">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getCigarString()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a9fd8f352f4558d8bab1da3ec428479ed">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>getClosest()
: <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_location_aware_matrix.html#a568bcf953a7d2ae720f51a9b196106c2">broad.pda.datastructures.LocationAwareMatrix</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4.html#a6a8186197e84b8133853f701908d0e82">umms.core.annotation.AnnotationList&lt; T extends Annotation &gt;</a>
</li>
<li>getClosestDownstream()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a17134d7932aab01f1cb7783f60bbded9">broad.pda.annotation.BEDFileParser</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4.html#aa97e34b194d3815d34549626e350efbb">umms.core.annotation.AnnotationList&lt; T extends Annotation &gt;</a>
</li>
<li>getClosestNonOverlapping()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4.html#a0185730c127c1cf197b475ae30388f8d">umms.core.annotation.AnnotationList&lt; T extends Annotation &gt;</a>
</li>
<li>getClosestUpstream()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a01bdda303f0aa5c707342733e3bb09c2">broad.pda.annotation.BEDFileParser</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4.html#aa02f46b613567dbf91e5ebc0eeb42fc3">umms.core.annotation.AnnotationList&lt; T extends Annotation &gt;</a>
</li>
<li>getClusterInfo()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_kmeans.html#a5cc4e3945675d6a8cd1686737a4d916a">broad.pda.geneexpression.clustering.Kmeans</a>
</li>
<li>getCol()
: <a class="el" href="classjaligner_1_1_cell.html#adfb84252c1633090a51f6173d5c811f5">jaligner.Cell</a>
</li>
<li>getColoumnName()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a9fb0e8dc96101b151a5c40eac8c453e2">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>getColumn()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#ad71f3a9dbda2f1969161bf6cb8a176c1">broad.core.datastructures.MatrixWithHeaders</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_alignment.html#a5c0ddfc79c6d95bc1579f156f84f5608">broad.core.multiplealignment.MAFAlignment</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#ac6a2d951ce87a39003ad2190b35620d3">broad.core.multiplealignment.MultipleAlignment</a>
, <a class="el" href="class_jama_1_1_matrix.html#a679eb65734d2d2922d833761da542345">Jama.Matrix</a>
</li>
<li>getColumnDimension()
: <a class="el" href="class_jama_1_1_matrix.html#ab3d646f9fb632a02ebac93c9118ab16a">Jama.Matrix</a>
</li>
<li>getColumnNames()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a488a21c1993afb6381f59f5b133c8f1c">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>getColumnNum()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_probe.html#a84c1c947797737f342319467ea841253">broad.pda.geneexpression.agilent.AgilentProbe</a>
</li>
<li>getColumnPackedCopy()
: <a class="el" href="class_jama_1_1_matrix.html#a62efb942093464fcf4d8c2bd557d51fa">Jama.Matrix</a>
</li>
<li>getColumnsAsVector()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_alignment.html#a4a26c761129c3b99364a45c242f8e06b">broad.core.multiplealignment.MAFAlignment</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#a4fc36e3151292c38c16f65e792725894">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>getComaptibleBetweenAny2Sets()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#a605c39f1d3fdd54f354087b8b49fbbfb">broad.pda.annotation.Locus</a>
</li>
<li>getComaptibleIsoforms()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#a5efc45d71e297a4996655b451accd1a3">broad.pda.annotation.Locus</a>
</li>
<li>getCombinedPvalue()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_fisher_combined_probability_test.html#a2c696eeb20ff18e619787e0a4a60bfe4">broad.core.math.FisherCombinedProbabilityTest</a>
</li>
<li>getComplement()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_column.html#a3b8fc8d22e61e6e7d681c7e200dc4e5d">broad.core.motif.PositionWeightColumn</a>
</li>
<li>getConcatenatedSubAlignment()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#aca6da10d461c82b32cf2fcc585e17a37">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>getConsensus()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix.html#a13a21c2d909695d4bab505ca1743fb4e">broad.core.motif.PositionWeightMatrix</a>
</li>
<li>getConsensusSequence()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_sequence_motif.html#a782ae826cd35a956402df350ef408c70">broad.core.motif.SequenceMotif</a>
</li>
<li>getConsistencyProblems()
: <a class="el" href="classbroad_1_1core_1_1hmm_1_1_markov_model_3_01_t_01_4.html#a0f483ef7027735422579aa60a3f6c9d6">broad.core.hmm.MarkovModel&lt; T &gt;</a>
</li>
<li>getContainingSequenceId()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#addecf6e872fb99d85e68f6503a3eb9b2">broad.core.sequence.SequenceRegion</a>
</li>
<li>getControlClass()
: <a class="el" href="classbroad_1_1core_1_1util_1_1_parse_g_c_t_file.html#aa0d127bf87abedb2695db9e582fd2946">broad.core.util.ParseGCTFile</a>
</li>
<li>getCoordinateAtPosition()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_compound_interval.html#a68a22617660bba90254228b97720b1de">umms.core.annotation.CompoundInterval</a>
</li>
<li>getCoordinateSpace()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#a69120a86f6a3c8d0607b663301c438b2">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_genome_command_line_program.html#a59439a63626d967a2b7f781b83479a8e">broad.pda.seq.rap.GenomeCommandLineProgram</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_r_a_p_peak_caller.html#af18e84651f7d4671be2c7e01ebaadd60">broad.pda.seq.rap.RAPPeakCaller</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#a681707090e9b5e55e0b51a3ef791857c">umms.core.annotation.AnnotationCollection&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4.html#ad0aa1bdb87c57359d3bac419631905ed">umms.core.annotation.AnnotationList&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a411c3f236c894f9df07ad4001f315d5f">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aca51c172563488f6435e1cb096922c63">umms.core.model.JCSAlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_scan_statistic_score.html#a262bc285097a3dcd60b9c3d2ce81e0ed">umms.core.model.score.ScanStatisticScore</a>
</li>
<li>getCoreMotifs()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_sequence_motif.html#a8a590ae198b7351cd4e774b580c9f4ca">broad.core.motif.SequenceMotif</a>
</li>
<li>getCount()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a8e4c4c86479ee745866baef364fdd6fe">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#a260b940b8bbebeb3e04bd0e75b003d17">umms.core.annotation.AbstractAnnotationCollection&lt; T extends Annotation &gt;</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#a19932ce01d0745d0b2a391df45e06aa4">umms.core.annotation.AnnotationCollection&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4.html#a3fe337384000f17feddeb3146f7f7d2b">umms.core.annotation.AnnotationList&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model_1_1_alignment_count.html#a9e4220a3c68d71909bd8daec8361b067">umms.core.model.AlignmentModel.AlignmentCount</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a928a086e1ce54468cf0fd582e32f404d">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_1_1_alignment_count.html#a137f96ea81cbd7772c852627844ea01a">umms.core.model.JCSAlignmentModel.AlignmentCount</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a135b96822b05a710277803ceb4a3771a">umms.core.model.JCSAlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_abstract_list_score.html#a807a62ef44088c4262d7f6dd8ac837ae">umms.core.model.score.AbstractListScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_count_score.html#a906acf2141cba6fab150c85de37ca2b5">umms.core.model.score.CountScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_end_in_window_count_score.html#afe68efa9b172f5426ad85a65159b1099">umms.core.model.score.EndInWindowCountScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_start_in_window_count_score.html#a3518e08e029c8e6d0dcc8e00c31ec761">umms.core.model.score.StartInWindowCountScore</a>
</li>
<li>getCountByBinStart()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#abf937e871a1bfd38d2cfb30cb2641d68">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getCountExcludingRegion()
: <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#a176fffae65aa5c43dabb697cc3e7a961">umms.core.annotation.AnnotationCollection&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4.html#a3d4dc0c6465467606a23c0fd149328c0">umms.core.annotation.AnnotationList&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a6fc7f8dfbc07c036d0701237b917ec80">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a1c7bbf1ce516ae5fca860b72ecebe159">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getCountProcessor()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#a7e1567b53456c524612b5046601b7d3a">umms.core.annotation.AbstractAnnotationCollection&lt; T extends Annotation &gt;</a>
</li>
<li>getCounts()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a2119f620f0e7c5e9a784caa028b15131">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a6df3c21f752e4ac12630862b07c48ba3">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a24fc6c0cea13276596ad51dbab778d38">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a211c0c020c8dcba98e1e4aae65572077">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getCountScore()
: <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_alignments.html#adc34f93586f8533504447b613d1358d8">broad.pda.datastructures.Alignments</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a5acd3dbf90c17f7c1c8219aa30ba9891">umms.core.annotation.Gene</a>
</li>
<li>getCountsOfUniqueOverlappers()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#af4fb78bc4dc32a83dfe3e25dbc0d7143">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a72086a8b305719f0ae5b67a7ffd6ed49">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#af11ced151db31ab9e95d75cabeed3615">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getCountsOnBase()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a99f79714e857a21ead8341f53ddb1dd3">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#aa0e60a4a8182d340bd443c08618b0f99">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getCountsOnWindow()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a381f6ff8922c20cea08b090aae2b24a3">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getCountsPerAlignment()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a0eccb07bd96e7c6ba718bfd87aaaad1f">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#aa05959f24b019c2b1b124f6d9d7ca3cc">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#aa6cb75b59a4ff7425b03ea133699040c">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getCountsPerAlignmentForPolyA()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a569f25df717cd394c8e5258355f75eed">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getCountsPerAlignmentFullyContained()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a0f1acb29be16a343f4beb4498a7a2970">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#aaaf8aeeaf2f81e37ef6949ecf3cff75f">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getCountsPerAlignmentUncached()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a73535e3b231500350fcd80a35d53ff6b">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getCountsPerAlignmentWithSameEndpointForPolyA()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a90f008b83d90599a72bec79bc7be3dde">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getCountsPerAnnotationtranded()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a0be22f281d5cabb81da650083813ea30">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#aa7f82dd0b8dd66f73c9148b4ec533022">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getCountsPerBp()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#acd9c8501fd75af7cb102cd6886cbe15b">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>getCountsPerPosition()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a2ad06cbab73fda69b7f399d1980adde7">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ab36d1b8f1a85207bbb5f2f4b1dd030d6">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getCountsStrandedPerPosition()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a02809d43b5a67c1a9c8942b7d399c1a2">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a03d3b9837dce06b17b424dd35e7bd012">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getCountStrandedExcludingRegion()
: <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#a4bad301160c77622b557cef7d2f5a725">umms.core.annotation.AnnotationCollection&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4.html#a83d34811a344577d2fd00744f490b9ad">umms.core.annotation.AnnotationList&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a4314840666a311fe81b2f0f0340d0f7b">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ad96d29e6f7416f7e54e0b44c1e22bfa9">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getCountsWithinExons()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a700536b90c78342c4ce1f949155b1823">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a96929246b2df0c287be2d8607bc464b4">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getCoverage()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#a0404541c8aaadbef81224e9925bb3c2f">broad.pda.seq.graph.Path</a>
</li>
<li>getCtrlCoordSpace()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#ae9e83e53efa37b7da7494b6d331e5c42">umms.core.model.score.MultiScore</a>
</li>
<li>getCtrlCount()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_binomial_enrichment_score.html#ab288a6522eaff92e133e19ee8f369135">umms.core.model.score.BinomialEnrichmentScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#a963a7d6c8fd0ce0e23ffcf5c63f00c3a">umms.core.model.score.MultiScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#ab066acb3478a9f7a5069261e0d846589">umms.core.model.score.PoissonEnrichmentScore</a>
</li>
<li>getCtrlLambda()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#a1f99a2a5be0260419cfe1ded10e40edb">umms.core.model.score.PoissonEnrichmentScore</a>
</li>
<li>getCtrlRegionCount()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_binomial_enrichment_score.html#ac1e19e31d05cb7a9790e5916b59fb932">umms.core.model.score.BinomialEnrichmentScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#acd956420ecd37565d440797f56bf7018">umms.core.model.score.MultiScore</a>
</li>
<li>getCtrlRegionTotal()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#a0f66687c8022fb1211135f109cf7867e">umms.core.model.score.PoissonEnrichmentScore</a>
</li>
<li>getCtrlSampleLambda()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#a1dcb31681e206ca571e6b6c7339f5768">umms.core.model.score.PoissonEnrichmentScore</a>
</li>
<li>getCtrlTotal()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#a449a683166290cdd7054bad530407855">umms.core.model.score.PoissonEnrichmentScore</a>
</li>
<li>getCumulativeProbability()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#acaf3ac4a00eb20b5fa99729d6050c98b">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getCurrentBase()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_abstract_fasta_parser.html#a7435a2e90f3fbd8a2215aed2278b2d8f">broad.core.sequence.AbstractFastaParser</a>
</li>
<li>getCurrentQuality()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_fasta_quality_parser.html#a2ac902f3c6f63a9c51fb877e5d891bc9">broad.core.sequence.FastaQualityParser</a>
</li>
<li>getCurrentRelease()
: <a class="el" href="classjaligner_1_1util_1_1_commons.html#a355a1dea60e473b31ff8ad60bb56b994">jaligner.util.Commons</a>
</li>
<li>getCurrentSequenceId()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_abstract_fasta_parser.html#af8f2a134ede096d51ded5e3f2b9fc4fd">broad.core.sequence.AbstractFastaParser</a>
</li>
<li>getD()
: <a class="el" href="class_jama_1_1_eigenvalue_decomposition.html#a3120d34f74224c1093cc9bc03f245ee1">Jama.EigenvalueDecomposition</a>
</li>
<li>getData()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a00f4475ce4248cc4071968d1ea7ab06c">broad.core.datastructures.MatrixWithHeaders</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#aa5acde7843960f0190b3a1ff9ae4b708">broad.pda.seq.clip.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a7e1478400941779a43e0776cd90733bb">broad.pda.seq.protection.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a3eec646275a0b6cc0f3e4542c0ca5e0a">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a5f78dee2ade4678e5ac2d04576c83b6d">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>getDataForAlignment()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#af7d90fa931951069416e3ad235e99f37">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>getDataForGene()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a00cf5fa4275d363fa77bf05454e7909c">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>getDataModel()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_utils.html#aefce66d41ea513ac55e494811f725522">broad.core.sequence.SequenceUtils</a>
</li>
<li>getDataType1()
: <a class="el" href="classbroad_1_1pda_1_1enrichment_1_1_enrichment_metric.html#a1e902f9e0b8a955dd86fee8792754cf7">broad.pda.enrichment.EnrichmentMetric</a>
</li>
<li>getDataType2()
: <a class="el" href="classbroad_1_1pda_1_1enrichment_1_1_enrichment_metric.html#a8974577185db4df2da04ae324677fd5a">broad.pda.enrichment.EnrichmentMetric</a>
</li>
<li>getDataTypes()
: <a class="el" href="classbroad_1_1pda_1_1enrichment_1_1_enrichment_metric.html#ad6b6fa73f1d0d0da595ff99c4411bcea">broad.pda.enrichment.EnrichmentMetric</a>
</li>
<li>getDebugInfo()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#a14cdd68e8c1bc594f39badfc1c1bcde5">umms.core.model.score.MultiScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_scan_statistic_score.html#a392a0b75ab2921bd7b85361c63295f7c">umms.core.model.score.ScanStatisticScore</a>
</li>
<li>getDefaultFile()
: <a class="el" href="classumms_1_1core_1_1writers_1_1_paired_end_writer.html#a1c122086a58a6837457e7b63eef0cb14">umms.core.writers.PairedEndWriter</a>
</li>
<li>getDenominatorCount()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#a93b23e58ece71cc054f535d97722c709">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#a016968c6cc0959776ace843c2c784616">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#a9ae2b2657a9333aadf08d70bbe92d35e">umms.core.model.score.WindowAllScore</a>
</li>
<li>getDenominatorLambda()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#a7c40c0401bc3716b3925d55d406f494a">umms.core.model.score.WindowAllScore</a>
</li>
<li>getDenominatorRegionTotal()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#a5099065a98b37f2f8ced26c490b34815">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#ae3a38c4eee8dfbfad80c674ee621d549">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#abea956161e9e1cba7c6e23d64ef8cd2e">umms.core.model.score.WindowAllScore</a>
</li>
<li>getDenominatorRPKM()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#af6a7ce43a225b3ae9b37cdd7080ade0b">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#a3dcbd0b94b85adddb54348d075650260">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#ac3c013a76b2294982c6b2affdf809cd8">umms.core.model.score.WindowAllScore</a>
</li>
<li>getDenominatorScanPVal()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_skellam_score.html#a530e50d3eb5f0c690eafd388dfd8a529">broad.pda.seq.rap.SkellamScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#a8dce98d2a4b85d353527fff26f4f9bf2">umms.core.model.score.WindowAllScore</a>
</li>
<li>getDenominatorTotal()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#af100d5f2aa2c974efdbe258ab25df026">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#a97082aeb1ae49329276a789ba12124cb">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#afa9588551cc4ea71a624a58ca02724ac">umms.core.model.score.WindowAllScore</a>
</li>
<li>getDensity()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#af964172d81916b8a1ee2dbb6c2a2ff73">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getDepletionBinomialPValue()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#aa01e2e91cb8482d0d67493814b8afa4a">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#a493eb53da0b3502bea226ca41d6f69b7">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#ad3e4a93c84529f4ff80bffc5f2c0f369">umms.core.model.score.WindowAllScore</a>
</li>
<li>getDepletionBinomialScore()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#a3d251d4c7a5c760668e457c6a94f03f0">umms.core.model.score.RatioScore</a>
</li>
<li>getDescription()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_sequence.html#ab2a40eee8f89ade0fa9d3e202322b4ae">broad.pda.seq.fastq.FastqSequence</a>
, <a class="el" href="classjaligner_1_1_sequence.html#a02a715ccb253a4d7af4dd6f40c16b6e5">jaligner.Sequence</a>
</li>
<li>getDirection()
: <a class="el" href="classjaligner_1_1_traceback_node.html#a537c9f62575c32be38e3516886ed666e">jaligner.TracebackNode</a>
</li>
<li>getDistancesToClosest()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#a0c53608fb99256004374dd6b0bdd3185">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
</li>
<li>getDistanceTo()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#a944575391d1ef0c562e3beeb4d493925">broad.core.annotation.BasicLightweightAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation.html#a7b51d8f731634b924bd1fd8507acf416">broad.core.annotation.LightweightGenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a53a14391b4e8c69c4f13e7428f158d70">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation.html#ac8df393754cf49172aadad4b254e5fda">umms.core.annotation.AbstractAnnotation</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#a24f9c0d8338789f72d5a8be63affd084">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_single_interval.html#a50c6fe38bfa25042209be45b319185e8">umms.core.annotation.SingleInterval</a>
</li>
<li>getDistanceTo3primeNeighbor()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#ac1ada0b4c99e92fd6acad2f68b9e4f2d">broad.pda.annotation.Locus</a>
</li>
<li>getDistanceTo5primeNeighbor()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#aebe1e52c92b834e03198cd903f164244">broad.pda.annotation.Locus</a>
</li>
<li>getDistanceToClosest()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#aa3a5e5fc97cb560c6a4ff633f204147c">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4.html#a0c6fe27137e7b0773be5b822846a66e9">umms.core.annotation.AnnotationList&lt; T extends Annotation &gt;</a>
</li>
<li>getDistributionOfHistogramValues()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#aab00bd3b82aa7048a17a57fc2f05191b">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getDoubleArg()
: <a class="el" href="classbroad_1_1core_1_1parser_1_1_command_line_parser.html#a1a12805a038827bca9e5a41c5df97933">broad.core.parser.CommandLineParser</a>
</li>
<li>getDoublePivot()
: <a class="el" href="class_jama_1_1_l_u_decomposition.html#a69d68d7f0bcaa3b7bf005b1d8ce285c1">Jama.LUDecomposition</a>
</li>
<li>getdoubleronicCounts()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_counts.html#aee7288cdf0870ffaa24a151c24b81371">broad.pda.seq.segmentation.GeneCounts</a>
</li>
<li>getDuplicateReadFlag()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment_1_1_wrap_sam_record.html#a4c4da4b04c7efeca4b59e54d03a60d6c">umms.core.alignment.SingleEndAlignment.WrapSamRecord</a>
</li>
<li>getEdgeList()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#a835abd712872e0c724d0a1fd0cf35582">broad.pda.seq.graph.Path</a>
</li>
<li>getEdges()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#ad1aa68a9922bef912e0641e7221cd2b2">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#abd24e986520eea6377aee5fde8ed85a3">broad.pda.seq.graph.Path</a>
</li>
<li>getElements()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_phastcons_reader.html#a18553ee20ff9f58b6a4c3354a2361d05">broad.core.siphy.PhastconsReader</a>
</li>
<li>getEmissionLogProbability()
: <a class="el" href="interfacebroad_1_1core_1_1hmm_1_1_markov_state_3_01_t_01_4.html#a88fafaa54e0b0ffec841b7a9c3def406">broad.core.hmm.MarkovState&lt; T &gt;</a>
</li>
<li>getEmissionProbability()
: <a class="el" href="interfacebroad_1_1core_1_1hmm_1_1_markov_state_3_01_t_01_4.html#ad3cd6d7c27695c97397797dacc59d089">broad.core.hmm.MarkovState&lt; T &gt;</a>
</li>
<li>getEmpiricalMaxStatisticDistribution()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#ac2b22a7d56c4a3859521123ae662e1c0">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getEmptyEmpiricalDistribution()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_build_ratio_null_distribution.html#ab77aff24273c3f5aef0f719f669ea1be">broad.pda.seq.rap.BuildRatioNullDistribution</a>
</li>
<li>getEncodedSequence()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a67016a0976b792c79d5362da0dd58b11">broad.core.sequence.Sequence</a>
</li>
<li>getEnd()
: <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation.html#a68b4fd4cda3321ffd91dd427df9b24a7">broad.core.annotation.LightweightGenomicAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1datastructures_1_1_interval.html#a3ecab556a0bfd0b97a7ff64fe875f546">broad.core.datastructures.Interval</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a59b9f3e750cccc29016b2e5bc4372976">broad.core.sequence.Sequence</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a1f2cbaafc85940a5c2719243f4640249">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a0837c3a39bc327a0e2e249a92c98a1aa">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#a653aaa69bcea0d35aeba7c92d9a68bb5">broad.pda.seq.graph.Path</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a843812c2305e69d131f2892b4b30b68e">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#afac2284106875d0f138898ad59e399fb">umms.core.alignment.Alignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#a51310d48912558d549d21cbf92ab946a">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a3fe8028115f96664761b63a9d16926cf">umms.core.annotation.BasicAnnotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_compound_interval.html#acf6878737d4032c688557394d11c33ac">umms.core.annotation.CompoundInterval</a>
</li>
<li>getEndVertex()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#a531ea71c00d0f6aae528d2bc576ecd43">broad.pda.seq.graph.Path</a>
</li>
<li>getEnrichment()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score.html#a713c54de24705c9dcc2ac273b8d52d11">broad.pda.seq.segmentation.GeneScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_scan_statistic_score.html#a2b410f5e59321ad692a8493e019932f2">umms.core.model.score.ScanStatisticScore</a>
</li>
<li>getEnrichmentBinomialPValue()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#a536d07d5066c4e0ddfee6f73f3dfaac0">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#a33c05dfce5f5378a5abf2b8611b7efcd">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#a6e0eefcb0d6d9a2192030ff5daf093f4">umms.core.model.score.WindowAllScore</a>
</li>
<li>getEnrichmentBinomialScore()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#a359817dbe99da022226d340298d05dba">umms.core.model.score.RatioScore</a>
</li>
<li>getEnrichmentOverControl()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_binomial_enrichment_score.html#abe55eff1931234a966a48da40987b2a5">umms.core.model.score.BinomialEnrichmentScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#a9124cf6e020fd07b4864eff3f7353a53">umms.core.model.score.MultiScore</a>
</li>
<li>getEnrichmentOverGene()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a83006fe4a56566ac14690adc205bf797">broad.pda.seq.clip.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a2f18a4bad14841ddbb440c21a281703a">broad.pda.seq.protection.SampleData</a>
</li>
<li>getEntireChromosome()
: <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html#ac20a537edec229754e0bc46408714f3c">umms.core.coordinatesystem.CoordinateSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_genomic_space.html#aed8d04961988d74e80e84061d202d9eb">umms.core.coordinatesystem.GenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_masked_genomic_space.html#a1a71e4fed6519abf120e78f798ece9be">umms.core.coordinatesystem.MaskedGenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcript_in_genomic_space.html#a254306a08e9c478002e3c721a4c9c540">umms.core.coordinatesystem.TranscriptInGenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcriptome_space.html#a5b234ba584e16b4c9f6302ad41c40984">umms.core.coordinatesystem.TranscriptomeSpace</a>
</li>
<li>getEstimatedLibrarySize()
: <a class="el" href="classbroad_1_1pda_1_1countreads_1_1_fastq_library_stats.html#af0d4416c9e3a72bfc2ab99d44b2b0ab9">broad.pda.countreads.FastqLibraryStats</a>
</li>
<li>getEstimationWindow()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_tree_scaler_i_o.html#ab8e19762981c09c8139d16b5e6e9e752">broad.core.siphy.TreeScalerIO</a>
</li>
<li>getEval()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1meme_1_1_meme_motif.html#a9cb970391847d2a40e96b19fdeeec93c">broad.core.motif.meme.MemeMotif</a>
</li>
<li>getExactIsoform()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a4e515ca60fbce30a804ec9d1212e33da">broad.pda.annotation.BEDFileParser</a>
</li>
<li>getExonAnnotationOverlappingRegion()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a001c1b90c85bcd26c5ea425e6ac07987">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a3bf75c99feba84ec070c90b94549bb03">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a951f6057002a5fdcd049130d0d84219f">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getExonicCounts()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_counts.html#a04248417ed6fe97892675eaff535f16b">broad.pda.seq.segmentation.GeneCounts</a>
</li>
<li>getExons()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a162487714a4f65542311eae90cb2f30a">umms.core.annotation.Gene</a>
</li>
<li>getExonSet()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#aa06c5697c4ec73b9257a826ab1060149">umms.core.annotation.Gene</a>
</li>
<li>getExonsScores()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#ae227231b6f7c160b6bfd698582b38ab7">umms.core.annotation.Gene</a>
</li>
<li>getExonTree()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a676460934bac3cd7775637c8f43b11b8">umms.core.annotation.Gene</a>
</li>
<li>getExonUnion()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#a0cfa9aabc38cfea8f8f657cc656bba87">broad.pda.annotation.Locus</a>
</li>
<li>getExperimentBarcode()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1_expression_experiment_info.html#a46a19fd7392e8a56bd709311e8c86d9b">broad.pda.geneexpression.ExpressionExperimentInfo</a>
</li>
<li>getExperimentHybDate()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1_expression_experiment_info.html#a2ca0ee915a370cf16c1f1ef286bc666d">broad.pda.geneexpression.ExpressionExperimentInfo</a>
</li>
<li>getExperimentName()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1_expression_experiment_info.html#a521009d9236367aff6d71cf592e6bd65">broad.pda.geneexpression.ExpressionExperimentInfo</a>
</li>
<li>getExperimentPlatePosition()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1_expression_experiment_info.html#a6f8e2b5864eee31aee23c61ac20e60e6">broad.pda.geneexpression.ExpressionExperimentInfo</a>
</li>
<li>getExpressionScanPvalueCutoff()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#ab1d89f168632a45a362704dafdbdc13c">broad.pda.seq.clip.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a4087ee5f33ea5ecc977f4fdf4483804e">broad.pda.seq.protection.SampleData</a>
</li>
<li>getExtend()
: <a class="el" href="classjaligner_1_1_alignment.html#a2b48d1d472b71c05059c245a85077f25">jaligner.Alignment</a>
</li>
<li>getExtraFields()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#aa49a33ecfaa2a1b142313b8d07596c60">umms.core.annotation.Gene</a>
</li>
<li>getExtraScore()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#a29f10b679f86820e035e1f8071ce552d">broad.core.annotation.BasicLightweightAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation.html#a9f830dce45015975df71e4babf9d811b">broad.core.annotation.LightweightGenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#add4b041421355dc90ba6a47169d76797">broad.core.sequence.SequenceRegion</a>
</li>
<li>getExtraScores()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#a2730f74d634af4832bc9a7c4071316cb">broad.core.annotation.BasicLightweightAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation.html#a3b41240f5758c0c98f9238520bbb203b">broad.core.annotation.LightweightGenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a64d2b48939f4a42d9b1ba307ef7f8d55">broad.core.sequence.SequenceRegion</a>
</li>
<li>getFactory()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_factory_factory.html#aef96a679b4e6771e8990e90dc5694e2f">broad.core.annotation.AnnotationFactoryFactory</a>
</li>
<li>getFDR()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_f_d_r_distribution.html#af1aeb6050c5dee84f703411053e8539b">broad.core.math.FDRDistribution</a>
, <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_run_differential_expression.html#a3e1b89e0a26f78383d03fa7aab42ddfe">broad.pda.differentialExpression.RunDifferentialExpression</a>
</li>
<li>getFDRDistribution()
: <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_differential_expression.html#ac5dbba7888df9150b1deb6d6fa8aa0e7">broad.pda.differentialExpression.DifferentialExpression</a>
</li>
<li>getFDRMatrix()
: <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_differential_expression.html#a8c7a95c323869360d3d791b5a4e3ea47">broad.pda.differentialExpression.DifferentialExpression</a>
</li>
<li>getFeature()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_g_f_f.html#a32286be94741ff45ae9d4ec4caa7f742">broad.core.annotation.GFF</a>
</li>
<li>getFieldCount()
: <a class="el" href="classbroad_1_1core_1_1parser_1_1_string_parser.html#ad5b185a9ef1e230a65e14c45c59a185f">broad.core.parser.StringParser</a>
</li>
<li>getFileSeparator()
: <a class="el" href="classjaligner_1_1util_1_1_commons.html#a9b2cbfa8be21ef473f93c439f1b2fb48">jaligner.util.Commons</a>
</li>
<li>getFileSize()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_abstract_fasta_parser.html#a91fd66568021c7d79e2f04e8778f35ce">broad.core.sequence.AbstractFastaParser</a>
</li>
<li>getFinalMValues()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_loess.html#a362409d5ac365910ed12a2859a2442fe">broad.core.math.Loess</a>
</li>
<li>getFirstExon()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#ab45c89fb5975cdbc4b1aee97a531ac20">umms.core.annotation.Gene</a>
</li>
<li>getFirstFragmentPositionStranded()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a6a0100ecd7ccbbfa99b2fdbce249264b">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#aac4a1a7ab425219a683a4a43a07ce9d8">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#ac9e20c5185b1732fc1fbcd66c670cff0">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>getFirstGene()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_1_1_unordered_gene_pair.html#af4f9363590209bb087cba52a12d8a23a">umms.core.alignment.FeatureSequenceAlignment.UnorderedGenePair</a>
</li>
<li>getFirstIntersectLeftFromMedian()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#a3f2ef6d9169a542a9efa85d33b56f6d2">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getFirstIntersectRightFromMedian()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#ac8c6bfb7c1e20464cbf057b49a14d854">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getFirstIntron()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a9803bda081210c957bccfe1a205d0b5f">umms.core.annotation.Gene</a>
</li>
<li>getFirstMate()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a35a255d4328f1e1ad9c2374f5900ceab">umms.core.alignment.AbstractPairedEndAlignment</a>
</li>
<li>getFirstNBPs()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_sequence.html#ac112cfd094a9ce4d543a3a4bf96946bc">broad.pda.seq.fastq.FastqSequence</a>
</li>
<li>getFirstSequence()
: <a class="el" href="classbroad_1_1pda_1_1rnai_1_1_extract_sequence.html#a6adfa50f391f4e203f935ab99b8869b3">broad.pda.rnai.ExtractSequence</a>
</li>
<li>getFirstValue()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_g_f_f.html#a4bbeab1dc47ec4c302cfc3aca1fe9c53">broad.core.annotation.GFF</a>
</li>
<li>getFits()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_stationary_distribution_i_o.html#a3a8f733bc29d80c9e4705b3cb328edf8">broad.core.siphy.StationaryDistributionIO</a>
</li>
<li>getFivePrimeBases()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_genomic_annotation.html#a8ed2dc419963ef16a12b10e038b772f1">broad.core.annotation.BasicGenomicAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_genomic_annotation.html#a53951d58e6b04312e1ac28d0bb16f23c">broad.core.annotation.GenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#ad170cee390bc0e30111801aea4ea5522">broad.core.sequence.SequenceRegion</a>
</li>
<li>getFlagsAndValues()
: <a class="el" href="classbroad_1_1core_1_1parser_1_1_command_line_parser.html#af4f549277703f59966280847ebb07660">broad.core.parser.CommandLineParser</a>
</li>
<li>getFlankingBlocks()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a3e559cf112dcf53643c073df5bf7c1aa">umms.core.annotation.BasicAnnotation</a>
</li>
<li>getFloatArg()
: <a class="el" href="classbroad_1_1core_1_1parser_1_1_command_line_parser.html#a89de2b3e47d77c6dc09daa4a229f56c5">broad.core.parser.CommandLineParser</a>
</li>
<li>getFoldMatrix()
: <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_differential_expression.html#aacfa88764f952afa917880f1a44317b9">broad.pda.differentialExpression.DifferentialExpression</a>
</li>
<li>getFormat()
: <a class="el" href="classjaligner_1_1formats_1_1_format_factory.html#a2ec3d952de23c5cc649f03a804a1f739">jaligner.formats.FormatFactory</a>
</li>
<li>getFormats()
: <a class="el" href="classjaligner_1_1formats_1_1_format_factory.html#abe01131f3b213cf76e28c7ff1f137a4c">jaligner.formats.FormatFactory</a>
</li>
<li>getForwardEdges()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#afea269e94ab8a39ab2df12ff9d104062">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>getForwardTargetRegions()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_smatch_like.html#afd7abcc767e14a7bef6d6a7bfe26002f">umms.core.alignment.SmatchLike</a>
</li>
<li>getForwardTargets()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_smatch_like.html#a637526895697dd36d8bd7acc33be1b05">umms.core.alignment.SmatchLike</a>
</li>
<li>getFragment()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a210850190a8e4e98c2ce56c842f0b80e">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#afdfc69376b512b44b069c0391c6e81bb">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a4bfe393bea33fbdd9f36beb72d226afc">umms.core.alignment.SingleEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html#adebcb0b519d1c1d5d1b8bd23b9a7cad7">umms.core.coordinatesystem.CoordinateSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_genomic_space.html#af398b7d440b6c3d7090232b6755a67f3">umms.core.coordinatesystem.GenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_masked_genomic_space.html#a5e2c87159db2089d61cc9d3d7e438486">umms.core.coordinatesystem.MaskedGenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcript_in_genomic_space.html#a2a9774c8e5e1a638366899cf97fd2cde">umms.core.coordinatesystem.TranscriptInGenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcriptome_space.html#ac9fe2dc48421fe148ad8ccca4eabf08e">umms.core.coordinatesystem.TranscriptomeSpace</a>
</li>
<li>getFragmentEnd()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#af16e4a059527adbdf5ac66350cab42c5">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#a0853dfe9729b648e70ffa65be427c4ca">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a38409b7b5071dbc0c884b6b5412ed1e0">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>getFragmentEndsFromBamFile()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup.html#a9d35efcf06921f02594aef226c6862d4">broad.pda.seq.clip.Pileup</a>
</li>
<li>getFragmentLengthFilterData()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a36db29585dcedfafcb7169a660bef489">broad.pda.seq.clip.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a896c9168d0e4ca469e94d60554bdf1d9">broad.pda.seq.protection.SampleData</a>
</li>
<li>getFragmentLengthGeneCount()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#acacdbc70e2771b904d9611b653f18c28">broad.pda.seq.clip.SampleData</a>
</li>
<li>getFragmentMidpoint()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#aa013b3820d63aa91253df3d83bc24881">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#a3c16abce949288dfde66f350c55e3866">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#aec29a84cf1505c4eb312bf730388a4e2">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>getFragmentSize()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#ab6d963d0dd6bd87e8c98fadf6e185f2e">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#a91730892dd6c46db8004b9a59f7c8734">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a2afaffd724b67924c4eb259457307c76">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>getFragmentStart()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a64eda682e150d7be478dadb30ec43c56">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#a56d8823df478d6a953758d95a32f6844">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a8cc3012be5d52c863d56b719ded4f4d7">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>getFragmentStrand()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a7272dda055f8425c3f85302fd96cfa5b">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#a67d44004454daa9c4ad7fc4dc66c9929">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a884e160d9e77c62b6e4ffa26b46bb104">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>getFrame()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_g_f_f.html#a448010b0f277781ba06a15c3b371bed5">broad.core.annotation.GFF</a>
</li>
<li>getFullInfoString()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a0ceb54ae1082e7c7844698140ebb9985">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation.html#a745f96c6e9f859a8f9be365f5916c11d">umms.core.annotation.AbstractAnnotation</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#ab8039241bfdb54655ba3eebf3a50f14b">umms.core.annotation.Annotation</a>
</li>
<li>getFullIntervalTreeAsAnnotation()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a0c395230c930df1c25572915b1be34ca">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a031ef42bb8a258dea08353faad7132ab">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getFullPrintableAlignment()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_smith_waterman_alignment.html#ac0693d7813e33cc22a40fdeb4295a081">umms.core.alignment.SmithWatermanAlignment</a>
</li>
<li>getFullyContainedNumberOfReads()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score.html#a9cdb44154d8a541fab9bebbb26f3b566">broad.pda.seq.segmentation.GeneScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_basic_scan_statistic_score.html#a7608635112c824db827a73237c9e179e">umms.core.model.score.BasicScanStatisticScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_scan_statistic_score.html#ade40e9b117e0ed167285ec03445e5743">umms.core.model.score.ScanStatisticScore</a>
</li>
<li>getFWER()
: <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_f_w_e_r_distribution.html#a203c1c623f968e01d73d1bae77ceb799">broad.pda.differentialExpression.FWERDistribution</a>
</li>
<li>getFWERMatrix()
: <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_differential_expression.html#af277590130eb9dc8feeff74b45d20305">broad.pda.differentialExpression.DifferentialExpression</a>
</li>
<li>getGappedSeqsInWindowMatrix()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_conservation_utils.html#a9f75501f2ed4d9bbc09461319aa1729c">broad.core.siphy.ConservationUtils</a>
</li>
<li>getGappedSize()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#af9c644563d081f56c49ad26d114d15de">umms.core.annotation.Gene</a>
</li>
<li>getGaps()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_genome_with_gaps.html#a0a9b818dbd5a2021baf0a63be84e0140">broad.pda.seq.segmentation.GenomeWithGaps</a>
, <a class="el" href="classjaligner_1_1_alignment.html#af51939568ee574d67afd7f6e7b6efe96">jaligner.Alignment</a>
</li>
<li>getGaps1()
: <a class="el" href="classjaligner_1_1_alignment.html#aab382d680bfff5de44fc45f8b8b4a532">jaligner.Alignment</a>
</li>
<li>getGaps2()
: <a class="el" href="classjaligner_1_1_alignment.html#afaed289aea427146978a44e4f9256d9c">jaligner.Alignment</a>
</li>
<li>getGapsSize()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#ac902afc1bae6378611e184e43562aaf6">broad.core.sequence.Sequence</a>
</li>
<li>getGene()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a37e961395720da4c91b43ae3d09fc467">broad.pda.annotation.BEDFileParser</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#a259b3430738009d5c8f95374ad1ba225">umms.core.model.score.MultiScore</a>
</li>
<li>getGeneAverageCoverage()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#ae3f4ea198ba9415c26448706f02177bf">broad.pda.seq.clip.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a7d17b21cbf2a2368cc637a7debb8089e">broad.pda.seq.protection.SampleData</a>
</li>
<li>getGeneByName()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#abd060b4376a538313b22991bf1eab8a9">broad.pda.annotation.BEDFileParser</a>
</li>
<li>getGeneCount()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#ad90b439a428d5cf25c496d69b49437a4">broad.pda.seq.clip.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#aa2bf68f6d49af41a18f1d4a1277c36f4">broad.pda.seq.protection.SampleData</a>
</li>
<li>getGeneCounts()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#abcc14ff2c0bf3c7ecd36e170fa1e5518">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a5e1bbaff7cd24dcd934c4ad669a2d6b1">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#aaee0b9274c46753d21f44590c58c072c">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#aa63431326e357fb7e0bbd28695bfaf18">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getGeneCountsWithoutDatasetTotals()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a9e9325efeb3cd65178b97c0bc675229e">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a76ef156f96e86fe1eea8b45e94d22f02">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>getGeneDescription()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_probe.html#a7e391d1c45c21de7823a5d4d6a7ca8e6">broad.pda.geneexpression.agilent.AgilentProbe</a>
</li>
<li>getGeneExpression()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a5379bafb9acafd7fbb7f7d59ef8c9308">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#aef09a54b699b7e8130f80515431e250c">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>getGeneIDIsoMap()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_g_t_f_file_parser.html#ab209c5c9823b5dfefa59299591ea7781">broad.pda.annotation.GTFFileParser</a>
</li>
<li>getGeneLength()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score.html#ad840bce93ebbf0eb74ce7cbba4736dd4">broad.pda.seq.segmentation.GeneScore</a>
</li>
<li>getGeneName()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_probe.html#ac8356167b52231358f3df6955b98fc1a">broad.pda.geneexpression.agilent.AgilentProbe</a>
, <a class="el" href="classbroad_1_1pda_1_1rnai_1_1_r_n_ai_gene_annotation.html#ade4f0313595a5bddf0fef019788a8185">broad.pda.rnai.RNAiGeneAnnotation</a>
</li>
<li>getGenePaths()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a7466db78d94e9436f2a4850a32435de2">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>getGenePval()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_genome_space_sample_data.html#ad0f782f3334827febea9c93e01878ae6">broad.pda.seq.clip.GenomeSpaceSampleData</a>
</li>
<li>getGeneRandNeighbors()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_neighbors_null_model.html#ac1a835df4c21a06eaeeb92709f4d0c59">broad.pda.gene.NeighborsNullModel</a>
</li>
<li>getGenes()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_transcribed_regions.html#a90654b16eed66a82727aae8abe59587c">broad.core.sequence.TranscribedRegions</a>
</li>
<li>GetGenes()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a80cb8b476bfeebcffc34c37bdd809d1b">broad.pda.annotation.BEDFileParser</a>
</li>
<li>getGenes()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_genome_command_line_program.html#ab1d7c1f083bc030ded1d014283d73201">broad.pda.seq.rap.GenomeCommandLineProgram</a>
</li>
<li>getGeneScanPval()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#aed882cc54637ce63c7c00e014fcd40cd">broad.pda.seq.clip.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_genome_space_sample_data.html#ad2292944e532fa900f27933bc2cfc7a2">broad.pda.seq.protection.GenomeSpaceSampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a0795b41a9a096104df31f5b6ba2c9154">broad.pda.seq.protection.SampleData</a>
</li>
<li>getGenesPassingFDR()
: <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_differential_expression.html#a1567565f3a248b3bc60627681d017e1b">broad.pda.differentialExpression.DifferentialExpression</a>
, <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_run_differential_expression.html#aca16939cef4d396c0f7ff7fb4acb1bd1">broad.pda.differentialExpression.RunDifferentialExpression</a>
</li>
<li>GetGenesWithIsoforms()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a92159642dd29f98fafb73d8a023d7a48">broad.pda.annotation.BEDFileParser</a>
</li>
<li>getGenomeSequence()
: <a class="el" href="classbroad_1_1pda_1_1rnai_1_1_extract_sequence.html#aaf80072fdbceac344f758b410f9001b2">broad.pda.rnai.ExtractSequence</a>
</li>
<li>getGenomeSize()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_directory_installed_genome_assembly.html#a928bd63f7e8e0fc7ff953346a44410e8">broad.pda.feature.genome.DirectoryInstalledGenomeAssembly</a>
</li>
<li>getGenomicLength()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a0becc72afe78ac5896c56262ce9f93ae">umms.core.annotation.Gene</a>
</li>
<li>getGenomicRegionOverlappers()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a353c3e74a98e5161351e8d08b1abd553">broad.pda.annotation.BEDFileParser</a>
</li>
<li>getGenomicSpace()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1rna_1_1_transcriptome_command_line_program.html#aff5c6cc64d861821f530e4dc869fc3ed">broad.pda.seq.rap.rna.TranscriptomeCommandLineProgram</a>
</li>
<li>getGlobalCount()
: <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#a78b46d17f831900001bc45e2643ae69e">umms.core.annotation.AnnotationCollection&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4.html#a7f74f466dcdb91eb2a4a073c77362c86">umms.core.annotation.AnnotationList&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a1f2e9cd52c3f45ee20663e44bbe0fb37">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a224d0f54b1c732696c3d6397077e08af">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getGlobalCountReferenceSeqs()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a8371e9ed5766760c3f78ba545b006f57">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#acfc0598ca6ce97995a72e76fb5be02bf">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getGlobalEnrichment()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_basic_scan_statistic_score.html#a7d0693db16d22a5bd0f5b746fcdb2b43">umms.core.model.score.BasicScanStatisticScore</a>
</li>
<li>getGlobalLambda()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a0b3aca4d1a840f1519b7bf9798b95039">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aca322da8e18f9ef5ceefbf4878c20005">umms.core.model.JCSAlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_basic_scan_statistic_score.html#a6ebdc7a57a5a409e58a233de615036c2">umms.core.model.score.BasicScanStatisticScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#a31618efad19a0e51a2816e1d132aba07">umms.core.model.score.MultiScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_scan_statistic_score.html#aa11ef30139134a9dd645e3470ca94a40">umms.core.model.score.ScanStatisticScore</a>
</li>
<li>getGlobalLength()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a2d0ca3da4ecfd1c131cd501cd83534bf">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a201240f2742a96b997444d5ee65575ef">umms.core.model.JCSAlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_basic_scan_statistic_score.html#aea3afb594bc9f8eda886e0987285b504">umms.core.model.score.BasicScanStatisticScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#a1af2e8503a14f8ddf03000335df764c1">umms.core.model.score.MultiScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#ab7942d7e9a951cec179a9f9d32a90dae">umms.core.model.score.PoissonEnrichmentScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_scan_statistic_score.html#a3cf3f87c128b3fd35d5ce49ed496c43e">umms.core.model.score.ScanStatisticScore</a>
</li>
<li>getGlobalNumReads()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a6efebb99d6bf0d6eeca5059f81384fcf">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a307b6ffe46f994204dae7864f70020aa">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getGlobalNumReadsReferenceSeqs()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a100151cc831e5ac822dd97c04f5e7773">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a361f481309989ba63027bdfb4fa1ab7c">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getGlobalPairedFragments()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#aae475cceb8f8c8c85f7902f0a3e75e43">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#acf2466e03b14876eba113f1f232b39d2">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getGlobalRPKMConstant()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_stored_alignment_stats.html#abb5ea3c63cf501daffe23ab9081003ad">broad.pda.seq.alignment.StoredAlignmentStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a88e59059f08c53ca2d159afd4bbc5fe4">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
</li>
<li>getGraph()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#a2f308b5d2202905e9d6af414cc379096">broad.pda.seq.graph.Path</a>
</li>
<li>getGroup()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_g_f_f.html#acab364bbf9c317a31767e6b1adce7871">broad.core.annotation.GFF</a>
</li>
<li>getGroupIndexes()
: <a class="el" href="classbroad_1_1core_1_1util_1_1_parse_g_c_t_file.html#a212dc63e80e3d8c1ca091b4048914a17">broad.core.util.ParseGCTFile</a>
</li>
<li>getGTFAttr()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_g_f_f.html#a775eff0dc1b7fba00a90abe52abeecd7">broad.core.annotation.GFF</a>
</li>
<li>getGTFGeneTranscriptMap()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_g_t_f_file_parser.html#a70ad696e47d527d952c764208d0afc0d">broad.pda.annotation.GTFFileParser</a>
</li>
<li>getH()
: <a class="el" href="class_jama_1_1_q_r_decomposition.html#a700a752da9386dc6db76dcd03d93f756">Jama.QRDecomposition</a>
</li>
<li>getHeader()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_alignment.html#a2b163056c476a4b7ddf520d28a0dd68b">broad.core.multiplealignment.MAFAlignment</a>
, <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_s_a_m_paired_end_file_reader.html#aebc7516a7202903377c4e58b0b75ed33">broad.pda.samtools.SAMPairedEndFileReader</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a7c244aba0fae3e4a4affd9a2d018c888">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#a1535cacbe7e0b688eecfce4ea9490c2c">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a3187e99040ea901670c40d345e14ddf2">umms.core.alignment.SingleEndAlignment</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a8004a75b7c533c9e93db4ddb0fe8f48e">umms.core.model.JCSAlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1readers_1_1_paired_end_reader.html#a0053947945b1d82abb79ab943c78e0e9">umms.core.readers.PairedEndReader</a>
</li>
<li>getHistogram()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#a5b21df6b2ce3df19d871ae0ebd1b9cec">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getId()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_genomic_annotation.html#a38f97fda9ac408d296525789104eb96c">broad.core.annotation.BasicGenomicAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_genomic_annotation.html#a8a46707d926ff0aa1fa0104acc74358a">broad.core.annotation.GenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1motif_1_1_sequence_motif.html#af0f6b0982fd6158c8fe56e344a6b4589">broad.core.motif.SequenceMotif</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a5da71f38ac38e891746b5f747fe67213">broad.core.sequence.Sequence</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a3a81800b533cb2f0a880c5158741ebd0">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classbroad_1_1pda_1_1enrichment_1_1_enrichment_metric.html#aee144337542c922c3f9574ba37eebc1a">broad.pda.enrichment.EnrichmentMetric</a>
, <a class="el" href="classjaligner_1_1formats_1_1_format.html#aa048f21b9987629497d836c399da6747">jaligner.formats.Format</a>
, <a class="el" href="classjaligner_1_1matrix_1_1_matrix.html#ab59872fac3104e3061d65e8cd0e3a1fb">jaligner.matrix.Matrix</a>
, <a class="el" href="classjaligner_1_1_sequence.html#a1ce3b0e34788f6f27dc7312f12fa8191">jaligner.Sequence</a>
</li>
<li>getIdentity()
: <a class="el" href="classjaligner_1_1_alignment.html#a5b3b8597278c2e880627aa51b366742f">jaligner.Alignment</a>
</li>
<li>getImagEigenvalues()
: <a class="el" href="class_jama_1_1_eigenvalue_decomposition.html#adc0af6cb4a4a6c6f01903d979820d585">Jama.EigenvalueDecomposition</a>
</li>
<li>getIncidentEdges()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a445a09bc5b15d604661680071b926a38">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>getIndecesForRowDescription()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a4e4af6c5ca9bca86aefac4ab7d3dd026">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>getIndex()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_interval_tree_3_01_v_01_4.html#a87fa030b462c169a7cb7ec515d652170">broad.core.datastructures.IntervalTree&lt; V &gt;</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_alignment.html#ad3dca7634a26f2abfe8d5bbe74b5215c">broad.core.multiplealignment.MAFAlignment</a>
</li>
<li>getIndexFor()
: <a class="el" href="classbroad_1_1core_1_1parser_1_1_string_parser.html#a41df20ecf6b5981300aad15764b572ea">broad.core.parser.StringParser</a>
</li>
<li>getIndexGroup()
: <a class="el" href="classbroad_1_1core_1_1util_1_1_parse_g_c_t_file.html#a9147960feca9e43647c5691d6387709e">broad.core.util.ParseGCTFile</a>
</li>
<li>getInformationContent()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_column.html#ac3d42c98e9344071ac290237548d7337">broad.core.motif.PositionWeightColumn</a>
</li>
<li>getInitialStateTransitionProbabilities()
: <a class="el" href="classbroad_1_1core_1_1hmm_1_1_markov_model_3_01_t_01_4.html#abc493a6b5d103b9cf94675560e02328c">broad.core.hmm.MarkovModel&lt; T &gt;</a>
</li>
<li>getInOrUnorientedEdges()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#ac11519a79446df006a85aa51487d9171">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>getInstance()
: <a class="el" href="classjaligner_1_1formats_1_1_format_factory.html#a985e04230615fa0dd275cf1852da7824">jaligner.formats.FormatFactory</a>
</li>
<li>getIntArg()
: <a class="el" href="classbroad_1_1core_1_1parser_1_1_command_line_parser.html#a172b91cd4b699b9341366c6dcc1b3b8e">broad.core.parser.CommandLineParser</a>
</li>
<li>getIntervalBetweenReads()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a37c576222ddbb88a5afca64ba6049996">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#a8cf9661b7c5fb24a23d5ec2edff6009f">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a8df968b30698af73b5ef30c45a336098">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>getIntervalTree()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a93bf3d9d5f736f192972c45834bf1fd6">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#af39e2cbab15523f4ce749edfaef1a157">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a920d82b5606fe25f56d4075f8119a573">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getIntervalTreeCached()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a42694fa2b3aee5c5c2cdd9b4c09ea67c">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a81241fd2d415955c7f4081bf6fe3d01b">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a5ebeb6683a55b7472d9eca5fbed51375">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getIntervalTreeTruncated()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a60cddc275b39ab087bb34488651be35d">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getIntervalTreeTruncatedCached()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a5f38e79de08ea2c450625ee17b4ddf57">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#ac199784edf7f223bf6eac76c6ef23101">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a211158851e9a4f2ca31f4a556235de13">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getIntervalTreeWithIsoforoms()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a685ac4182dd2f2730f904b74fbb24cd9">broad.pda.annotation.BEDFileParser</a>
</li>
<li>getIntronCounts()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a623a25e976697a1773196a2cf5f4d349">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a1d436b8676a6d0003d8dd57a4b73ed4b">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getIntrons()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a2738757f1768d271c10b12e6f5a498d0">umms.core.annotation.Gene</a>
</li>
<li>getIntronsBlocks()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#aa6c38c831dfee6a932f6c6ebdab96384">umms.core.annotation.Gene</a>
</li>
<li>getIntronSet()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#aa0f15894f4ab4ed9112de6cab211876d">umms.core.annotation.Gene</a>
</li>
<li>getIntronTranscript()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a921d5aac3c0e8e5b5ff2de7328e5d68e">umms.core.annotation.Gene</a>
</li>
<li>getIOHelper()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#a9c520e0b975a9041f83a44e1900a4b19">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>getIsComaptibleBetweenAny2Sets()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#a9ba15abbca6aba31e688c956dc321b3c">broad.pda.annotation.Locus</a>
</li>
<li>getIsComaptibleBetweenSubsets()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#ae84317b60ac2dd9b071db1c9a1301a2c">broad.pda.annotation.Locus</a>
</li>
<li>getIsComaptibleWithRefPerSet()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#a51ec31985ddc969f74872a2eae4fbe80">broad.pda.annotation.Locus</a>
</li>
<li>getIsFirstMate()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#aaebe0666bfb535f6cf60beb167800351">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>getIsoformMap()
: <a class="el" href="classumms_1_1core_1_1scripture_1_1_build_scripture_coordinate_space.html#a7ba371be0c1c6194e87333cdeb125467">umms.core.scripture.BuildScriptureCoordinateSpace</a>
</li>
<li>getIsoforms()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#abedac8afb5f8441e6f0d1e54de1239e5">umms.core.annotation.Gene</a>
</li>
<li>getIsPartiallyComaptibleWithRefPerSet()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#adf5e06cedc03a242dc6916dccd077071">broad.pda.annotation.Locus</a>
</li>
<li>getJAlignerInfo()
: <a class="el" href="classjaligner_1_1util_1_1_commons.html#a980e171044753b15ecf040cf68f5a99d">jaligner.util.Commons</a>
</li>
<li>getKmerEndPosition()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_hairpin_kmer.html#a698c1efd7c65b2a97fb22ee4ce3e8ba3">broad.core.primer3.HairpinKmer</a>
</li>
<li>getKmerSequence()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_hairpin_kmer.html#a95de1ea0f84761e1ddd1550815d5d026">broad.core.primer3.HairpinKmer</a>
</li>
<li>getKmerStartPosition()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_hairpin_kmer.html#addcf0acb79aed88e1534f13b138856f4">broad.core.primer3.HairpinKmer</a>
</li>
<li>getKSEnrichments()
: <a class="el" href="classbroad_1_1projection_1_1gsa_1_1_gene_set_enrichment.html#ad05dad6526aebca3c9b6508be1b8402d">broad.projection.gsa.GeneSetEnrichment</a>
</li>
<li>getKSFDR()
: <a class="el" href="classbroad_1_1projection_1_1gsa_1_1_gene_set_enrichment.html#a9af8225e4a5a8eeabd5a164e35885226">broad.projection.gsa.GeneSetEnrichment</a>
</li>
<li>getKSScores()
: <a class="el" href="classbroad_1_1projection_1_1gsa_1_1_g_s_a_scoring.html#a693da560ed5ecea8aa701c81295307ad">broad.projection.gsa.GSAScoring</a>
</li>
<li>getKSStatistic()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#ac265c5a5ca84aebb7d37229cbb92df34">broad.core.math.EmpiricalDistribution</a>
, <a class="el" href="classbroad_1_1core_1_1math_1_1_statistics.html#a4ec55597e8c774dba8de302ae6fc00ea">broad.core.math.Statistics</a>
</li>
<li>getL()
: <a class="el" href="class_jama_1_1_cholesky_decomposition.html#adcf74030c9514f8f3037a2f373f8bf8d">Jama.CholeskyDecomposition</a>
, <a class="el" href="class_jama_1_1_l_u_decomposition.html#ac24a98b938dd29b58fa0e9db41618242">Jama.LUDecomposition</a>
</li>
<li>getLambda()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_stored_alignment_stats.html#a37580bd68902828d1a52309bbbc591ae">broad.pda.seq.alignment.StoredAlignmentStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a7d20d81fde6834294a8876c1e1a1c12d">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a1b29a81c6b1a093d16081d8e362835f0">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a16fcd43426d01ba29a6e50d8a36ab38f">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>getLargestParent()
: <a class="el" href="classumms_1_1core_1_1utils_1_1_annotation_utils.html#adef9249ccd39f5aa8d2fed2b91e3dcc8">umms.core.utils.AnnotationUtils</a>
</li>
<li>getLastExon()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a0564f0d47e80de0bcef1407df7645780">umms.core.annotation.Gene</a>
</li>
<li>getLastExons()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#a8a0b54365f9d635929ad29cc80f42cdf">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
</li>
<li>getLastFragmentPositionStranded()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a74011edfb3ccb6915bea34da9d73e3c7">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#a174877831cbae9325d73c7d09906ae3d">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a7ab034b249df1e27ea154af3993e55e7">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>getLastIntron()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a74728d0843f18d11420ec15bab53a2b1">umms.core.annotation.Gene</a>
</li>
<li>getLastModifiedAlignmentIdxTimeStampAtCreation()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_stored_alignment_stats.html#aca6ad81073610ff4847eac7720cc1f4d">broad.pda.seq.alignment.StoredAlignmentStats</a>
</li>
<li>getLastModifiedAlignmentTimeStampAtCreation()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_stored_alignment_stats.html#a1628a35b4c1ea864b79d03d850f09c6a">broad.pda.seq.alignment.StoredAlignmentStats</a>
</li>
<li>getLastModifiedStoredTimeStamp()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_stored_alignment_stats.html#ae4869d1416affe87711445a7d58286b0">broad.pda.seq.alignment.StoredAlignmentStats</a>
</li>
<li>getLeaves()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_generic_rooted_tree_3_01_t_01_4.html#afa523dabb98b86d540c8031b537b6096">broad.core.datastructures.GenericRootedTree&lt; T &gt;</a>
</li>
<li>getLeft()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_cluster.html#a9ff47b9d556ffc04b8922a5f043e6a0c">broad.pda.geneexpression.clustering.Cluster</a>
</li>
<li>getLeftHighInfoStart()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix.html#a2fe7ed8e3b005e43dbe66784519cd37d">broad.core.motif.PositionWeightMatrix</a>
</li>
<li>getLeftNeighbor()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_neighbor_analysis.html#a2da854d4918421bfe33bc4883b6d5edc">broad.pda.gene.NeighborAnalysis</a>
</li>
<li>getLength()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_genomic_annotation.html#a1572a98b3c0a66370168e7f0eaa524df">broad.core.annotation.BasicGenomicAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_genomic_annotation.html#a0b276757bb75b768a6c6a266a9daa96d">broad.core.annotation.GenomicAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1datastructures_1_1_interval.html#a96afcf6952ec287be3ee66d9dde58d75">broad.core.datastructures.Interval</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#ae1f565eae2b5a441424f232711a1da9b">broad.core.sequence.Sequence</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a73892ab0e9b260c5f518481a47fdba82">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_sequence.html#a6fd087d51a9fd2ab4083360cd019a9e9">broad.pda.seq.fastq.FastqSequence</a>
, <a class="el" href="classjaligner_1_1_alignment.html#a7258f6e5bf636f0ac59767f42f1c3d9d">jaligner.Alignment</a>
, <a class="el" href="classjaligner_1_1_traceback_node.html#a2439a53f26bb269c2813e352259e9cb6">jaligner.TracebackNode</a>
, <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html#a5a1a16d1a6043ea09efe1084b5a9c84c">umms.core.coordinatesystem.CoordinateSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_genomic_space.html#aff9d1bad2aa51c3896bc90fafd5c6a42">umms.core.coordinatesystem.GenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_masked_genomic_space.html#a02d96b1cd3cd75f0134b149eb566ed8b">umms.core.coordinatesystem.MaskedGenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcript_in_genomic_space.html#af5d0d7a6d07a15ffc4384c98379d58a0">umms.core.coordinatesystem.TranscriptInGenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcriptome_space_1_1_gene_tree.html#a35a0578cbfa063328839d269629deed0">umms.core.coordinatesystem.TranscriptomeSpace.GeneTree</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcriptome_space.html#a06905b39f4cf215f2f38d98f2bb9838f">umms.core.coordinatesystem.TranscriptomeSpace</a>
</li>
<li>getLengthOnReference()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a1f19800e926c265f58ad05065bdc9a88">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation.html#ae96fae62fc5573abad1e33cbcad29003">umms.core.annotation.AbstractAnnotation</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#a082064fbc61b16ca20c298473ed1b3cc">umms.core.annotation.Annotation</a>
</li>
<li>getLineSeparator()
: <a class="el" href="classjaligner_1_1util_1_1_commons.html#ad1b4cbeb53ca9107e3979a6a5ba1fe50">jaligner.util.Commons</a>
</li>
<li>getLlr()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1meme_1_1_meme_motif.html#a8cbf5b9f393210e55326d9c8b1ed87d2">broad.core.motif.meme.MemeMotif</a>
</li>
<li>getLocalLambda()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#a2d1abad788b58cac224885272cf117a7">broad.pda.seq.graph.Path</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score.html#ad50650eb826331cc8ac828b169f03482">broad.pda.seq.segmentation.GeneScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_basic_scan_statistic_score.html#a67c56ac3e65e0c7868f64c05a5f8816c">umms.core.model.score.BasicScanStatisticScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_scan_statistic_score.html#ade266a940c29a1911ff8252fdd6b1d73">umms.core.model.score.ScanStatisticScore</a>
</li>
<li>getLocalRate()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#adda80fdb6e6d20255389643e6cb4d3a2">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>getLocalRpkmConstant()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_stored_alignment_stats.html#a80285e1a1ca002d5a00188d8f0d3b161">broad.pda.seq.alignment.StoredAlignmentStats</a>
</li>
<li>getLocalSampleLambda()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#a655ac97fedec41bafae2d527cda19766">umms.core.model.score.PoissonEnrichmentScore</a>
</li>
<li>getLocationString()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#a768a87c705c0f6abc1025dc776486cc9">broad.core.annotation.BasicLightweightAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_genomic_annotation.html#a057591abba9d5aae2a201bd918ede655">broad.core.annotation.GenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#ab928ab467224a7fba4e8cc1585befd1f">broad.core.sequence.SequenceRegion</a>
</li>
<li>getLog2Ratio()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#a138a5c44c26357eede46db62a0030899">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#a8dc892f53be6a5499835d5e61ac919a7">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#accee1ff27647f8b6865ff770bb82c318">umms.core.model.score.WindowAllScore</a>
</li>
<li>getLog2RegionRatio()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#ab9b35ce818f5f17c1bdf64a60610dd82">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#a5c2c20f663cc34c2b54aa7358e8adf05">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#aad4afe76a5c7e1b09e4a7d4095862483">umms.core.model.score.WindowAllScore</a>
</li>
<li>getLogger()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a2bfa33d3f02ba003618ad347cde46eda">broad.pda.seq.clip.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#ab31cc1f558419ad028782703b015c489">broad.pda.seq.protection.SampleData</a>
</li>
<li>getLogLikelihood()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix.html#a0fab2425599830972c542de9832541c2">broad.core.motif.PositionWeightMatrix</a>
, <a class="el" href="classbroad_1_1core_1_1siphy_1_1_tree_scaler.html#aa28eca418572c1475df5d267edaa9487">broad.core.siphy.TreeScaler</a>
</li>
<li>getLogLikelihoodRatio()
: <a class="el" href="interfacebroad_1_1core_1_1siphy_1_1_fit.html#a01c537a969cdb5e701cdd196c504a0a7">broad.core.siphy.Fit</a>
</li>
<li>getLogProb()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_column.html#af17d5c4039831262d7716ce3c80c77cc">broad.core.motif.PositionWeightColumn</a>
</li>
<li>getLogProcessedSignal()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_probe.html#a910143b246897074f1b266c8e45d828c">broad.pda.geneexpression.agilent.AgilentProbe</a>
</li>
<li>getLogRatio()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_column.html#acc95c9927ec2eb0099868cd6cf56acfd">broad.core.motif.PositionWeightColumn</a>
</li>
<li>getLogRatioWeightedByNegEntropy()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_column.html#a585fabfe6c110fee519aa51ceda7fd81">broad.core.motif.PositionWeightColumn</a>
</li>
<li>getLongestIntornChainIso()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#aac5b8bb5955d907402dba1f158c9d266">broad.pda.annotation.Locus</a>
</li>
<li>getLongestPerfectMatch()
: <a class="el" href="classjaligner_1_1_alignment.html#a200ad943e56a07cbc196fed4844a4f0b">jaligner.Alignment</a>
</li>
<li>getLongestUngappedAlignment()
: <a class="el" href="classjaligner_1_1_alignment.html#ab441a0192a6807aea435daf33c38ad7a">jaligner.Alignment</a>
</li>
<li>getMap()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_loess.html#a8e87a9e6008cd998fab086f8477498e8">broad.core.math.Loess</a>
</li>
<li>getMappingQuality()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a0fa29adb0a3285800e542b174ef4084d">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#a3fd5709cfe830908e7e0660c91bd4daa">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a06ceaaa21a4b84604e2db3f7ef114402">umms.core.alignment.SingleEndAlignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment_1_1_wrap_sam_record.html#adf5f702667ceba749cf001eea76c57c3">umms.core.alignment.SingleEndAlignment.WrapSamRecord</a>
</li>
<li>getMarkupLine()
: <a class="el" href="classjaligner_1_1_alignment.html#a6915c636c16199e545f3aa04402d7f8c">jaligner.Alignment</a>
</li>
<li>getMaskedRegions()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_stored_alignment_stats.html#a7179b18f167549ff9d4275af659b18a6">broad.pda.seq.alignment.StoredAlignmentStats</a>
</li>
<li>getMaskedRegionsSize()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#aac01d478cd4d5a31d5d9a51b86cb9d2a">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
</li>
<li>getMatch()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_p_s_l_record.html#a94f1a2fcb6fec040cc1dbaf7e5e42c1e">umms.core.annotation.PSLRecord</a>
</li>
<li>getMatches()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1meme_1_1_meme_motif.html#a57076b3995d8d3ce052a9db400bdc663">broad.core.motif.meme.MemeMotif</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a69dcd323117fd945585a71236ffef9e9">broad.core.sequence.Sequence</a>
</li>
<li>getMatrices()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix_i_o.html#a0e1024aa1dbbef0403babbfba71cf449">broad.core.motif.PositionWeightMatrixIO</a>
</li>
<li>getMatrix()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_hierarchical_clustering.html#ad1e80a17182c37cd9689d8ab1514b7ab">broad.pda.geneexpression.clustering.HierarchicalClustering</a>
, <a class="el" href="classjaligner_1_1_alignment.html#a26e8a54855f80d1d43738ddfff3029a0">jaligner.Alignment</a>
, <a class="el" href="class_jama_1_1_matrix.html#aa60bbca27d77dc5c9ae697e83885a4d8">Jama.Matrix</a>
</li>
<li>getMax()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#aea21ebb0d17ee335d3defce2b97150d0">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getMaxExonsCovered()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#ab6466dfa65c9a7701bd066379301a01a">broad.pda.annotation.Locus</a>
</li>
<li>getMaxMeanEnrichments()
: <a class="el" href="classbroad_1_1projection_1_1gsa_1_1_gene_set_enrichment.html#a97f7913210461d02bffc3b0a9dc1f45a">broad.projection.gsa.GeneSetEnrichment</a>
</li>
<li>getMaxMeanFDR()
: <a class="el" href="classbroad_1_1projection_1_1gsa_1_1_gene_set_enrichment.html#a6c2ce0e6aee894ebaf20bc66d9c264eb">broad.projection.gsa.GeneSetEnrichment</a>
</li>
<li>getMaxMeanScores()
: <a class="el" href="classbroad_1_1projection_1_1gsa_1_1_g_s_a_scoring.html#a26f38757a8e66dcdb37549ff5fffb598">broad.projection.gsa.GSAScoring</a>
</li>
<li>getMaxPctCovered()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#ac7a725bf08ba1f907352e743037e453b">broad.pda.annotation.Locus</a>
</li>
<li>getMaxPctGenomeCovered()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#aa43b3954b74d63bec8ac72a52e55b89c">broad.pda.annotation.Locus</a>
</li>
<li>getMaxScoreAcrossAllIso()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#ab5b65d99190dfd997054dd358c800c5b">broad.pda.annotation.Locus</a>
</li>
<li>getMaxScorePerSet()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#a8aa215097798926bce3723075eb05056">broad.pda.annotation.Locus</a>
</li>
<li>getMean()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#a095b4739b56efa19f731b64baa974b95">broad.core.math.EmpiricalDistribution</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_abstract_list_score.html#a29fc1a2172e53228a768071d70dbabd1">umms.core.model.score.AbstractListScore</a>
</li>
<li>getMeanOverAllRows()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a62678d31173af3e2f6a070cabd32a2f5">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>getMedian()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_abstract_list_score.html#a9c3fcd18daa8ec8fffffa8191df3dbe3">umms.core.model.score.AbstractListScore</a>
</li>
<li>getMedianOfAllDataValues()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#a4aecfe15a0ee52e7486cb2d63cc82648">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getMedianOverAllRows()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#ac18f9bee90f0a4d6c7b14afe8b641d06">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>getMedianReadSize()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#ac8db2149ef23944b52d4fc6080b5fc48">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a687f68337411f48ebbb0aa5b3c3480d1">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getMembers()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_cluster.html#ace59db59ddd5fee84c54b0b37a2aa5a7">broad.pda.geneexpression.clustering.Cluster</a>
</li>
<li>getMerged()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#ab86206c127ee93f5aa36fdec5c7a7f30">broad.pda.annotation.Locus</a>
, <a class="el" href="classbroad_1_1pda_1_1gene_1_1_gene_with_isoforms.html#aceaafb564ece7b46650aac895a433a30">broad.pda.gene.GeneWithIsoforms</a>
</li>
<li>getMergedAnnotationMap()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#ab4cead55423bc92714e6761c144bc2a8">broad.pda.annotation.BEDFileParser</a>
</li>
<li>getMergedCopy()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#ae1e1428140172568cf8a2b3fcafd7a5f">broad.pda.annotation.BEDFileParser</a>
</li>
<li>getMessage()
: <a class="el" href="classbroad_1_1core_1_1hmm_1_1_bad_model_exception.html#ad5de3c940718e09f1621196b1232d5fd">broad.core.hmm.BadModelException</a>
</li>
<li>getMethod()
: <a class="el" href="classbroad_1_1pda_1_1enrichment_1_1_enrichment_metric.html#a256b5d9c11e0f925f9f4894b9a68ec39">broad.pda.enrichment.EnrichmentMetric</a>
</li>
<li>getMiddle()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#ae3d92f493e84b236a38aac116bff28d3">broad.core.annotation.BasicLightweightAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation.html#aa67e2c999b69ce3f322f88c4c8d67b3f">broad.core.annotation.LightweightGenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#aefecde331a2d5fc81cfc4e63c02cdc46">broad.core.sequence.SequenceRegion</a>
</li>
<li>getMidpoint()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a2349bf9ffd84ef978ecc06e0db1804d4">broad.core.sequence.SequenceRegion</a>
</li>
<li>getMidPoint()
: <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_alignments.html#acdc1bd0519bc4c35f4c563a6e100a066">broad.pda.datastructures.Alignments</a>
</li>
<li>getMidpoint()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation.html#a52c42de2db42cf6b85583c4d83db8b0e">umms.core.annotation.AbstractAnnotation</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#afae7bf9cbe54b2703f8f633d1fc1936c">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#acf96ec93c503abe161f40e346992fe68">umms.core.annotation.BasicAnnotation</a>
, <a class="el" href="classumms_1_1core_1_1utils_1_1_annotation_utils.html#a6b7be0a704f3217b3b882dd06b07bd0b">umms.core.utils.AnnotationUtils</a>
</li>
<li>getMidpointGenomicCoords()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#ac26fe7612bd2bafe2c58ffb088a3ddef">umms.core.annotation.Gene</a>
</li>
<li>getMin()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#a754b27b52a19d7593e342e06308f9ae9">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getMinimumMappingQuality()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a6bdc47c2740081d2eaa43039ed64b59e">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a8a6731855b1e14f18bde6403c224e48b">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a0e9eeee0305815d3a7ac767e59c76867">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getMinimumTreeLength()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_tree_scaler.html#a95e23fcd6918e57aa1de02c5a821721c">broad.core.siphy.TreeScaler</a>
</li>
<li>getMirScore()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_hairpin_kmer.html#ab176e0652dfb5364e98aa377c38789b1">broad.core.primer3.HairpinKmer</a>
</li>
<li>getModel()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_tree_scaler.html#a4f07812132f23c410d73c521776113df">broad.core.siphy.TreeScaler</a>
</li>
<li>getModelFilePath()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#ade12353a8750a5517fda8f337c9a4f13">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#ad4da7c417bd21633701d18ad20aa4389">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getModelParameters()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model.html#a8c96fbd3055894234ce347c0ad68e8eb">broad.core.siphy.EvolutionaryModel</a>
</li>
<li>getMotif()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_sequence_motif.html#ae325c3588311a2f9fdea517b81fdd670">broad.core.motif.SequenceMotif</a>
</li>
<li>getMotifNames()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1misc_1_1_compute_conserved_motifs.html#ab30591bcefe13076a6a388cfc5878bad">broad.core.motif.misc.ComputeConservedMotifs</a>
</li>
<li>getMotifs()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1meme_1_1_meme_result_reader.html#ac9c8a40eab0c87a4c28f78e92d88e8d9">broad.core.motif.meme.MemeResultReader</a>
</li>
<li>getName()
: <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_feature.html#a7cfda06d3ea4ba04b6295675cd0fb7ec">broad.core.annotation.Feature</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation.html#af3d71fb0469ff283d3a4c4b539f58406">broad.core.annotation.LightweightGenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_generic_rooted_tree_3_01_t_01_4.html#a56e6fc08eb824371e30273589d9280ad">broad.core.datastructures.GenericRootedTree&lt; T &gt;</a>
, <a class="el" href="interfacebroad_1_1core_1_1hmm_1_1_markov_state_3_01_t_01_4.html#ad499d0d3db6b9cafaf6c4207e329df67">broad.core.hmm.MarkovState&lt; T &gt;</a>
, <a class="el" href="classbroad_1_1core_1_1motif_1_1meme_1_1_meme_motif.html#aba49196c19dea4235afa8d3b226dcbc8">broad.core.motif.meme.MemeMotif</a>
, <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix.html#a9c4f72f0173e0b84b3e3a732d9de22fc">broad.core.motif.PositionWeightMatrix</a>
, <a class="el" href="classbroad_1_1core_1_1primer3_1_1_restriction_enzyme.html#afee2b972470c63791ff0240893c2a5d5">broad.core.primer3.RestrictionEnzyme</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#abe729e8af2a1d244f41ddf93373ca829">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_chromosome.html#ad3469d02b22deddedb0d3b6205c1797a">broad.pda.feature.genome.Chromosome</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_absolute_pearson_function.html#a8c669622988d3afdf82816398465a567">broad.pda.geneexpression.clustering.AbsolutePearsonFunction</a>
, <a class="el" href="interfacebroad_1_1pda_1_1geneexpression_1_1clustering_1_1_cluster_distance_function.html#a1dde0f531a84ff1dcf17e2b076274467">broad.pda.geneexpression.clustering.ClusterDistanceFunction</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_euclidean_distance.html#a6ffc4536e27248d555ff6acc9b44e614">broad.pda.geneexpression.clustering.EuclideanDistance</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_pearson_distance.html#a09f91dc23f9a81e0dca720848ce7ed57">broad.pda.geneexpression.clustering.PearsonDistance</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_sequence.html#a9916fe4f95c1f85be1b3d778fca6ff39">broad.pda.seq.fastq.FastqSequence</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a684dcbe674fa90359576b194050d3681">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#adf055d923b7239351f4dd5908a1eebeb">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a75317ca06eb03c9e96cf18862e652c35">umms.core.annotation.BasicAnnotation</a>
</li>
<li>getName1()
: <a class="el" href="classjaligner_1_1_alignment.html#adc95ebcd08efadf95a1c215d90100578">jaligner.Alignment</a>
</li>
<li>getName2()
: <a class="el" href="classjaligner_1_1_alignment.html#ad656149e70e23a2f007bb543d0d393d8">jaligner.Alignment</a>
</li>
<li>getNameGeneMap()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a728380042bb55b55bb02108870d15fa7">broad.pda.annotation.BEDFileParser</a>
</li>
<li>getNanostringProbeClasses()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a8d2c5b2225c666f78db0102072178051">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>getNanostringReplicateMap()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#afde32f255717ce83e2d2dd960cad6e23">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>getNeighborDistance()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_neighbor_analysis.html#a9121f0c290ec779569131366cf39f2d9">broad.pda.gene.NeighborAnalysis</a>
</li>
<li>getNeighbors()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_neighbor_analysis.html#a8b56a8b65222eacfd043c75f4ef99e00">broad.pda.gene.NeighborAnalysis</a>
</li>
<li>getNext()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_combination_generator.html#a7eff3bd69051ba5a14595b23b4dbe17b">broad.core.math.CombinationGenerator</a>
</li>
<li>getNextClosest()
: <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_location_aware_matrix.html#ac52ce4740d48e3ae918578ec24732327">broad.pda.datastructures.LocationAwareMatrix</a>
</li>
<li>getNextKmer()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix.html#a2e65801b95229c2fb5977890aeb26d58">broad.core.motif.PositionWeightMatrix</a>
</li>
<li>getNextRandom()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_combination_generator.html#a7053a13f87f5b58a1a012214f21cc692">broad.core.math.CombinationGenerator</a>
</li>
<li>getNextWindow()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_genome_with_gaps.html#a31b11e3fa798fbd8b22ad29a651e1526">broad.pda.seq.segmentation.GenomeWithGaps</a>
</li>
<li>getNodeCount()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#a831d48efb7c551f60eb6b26a18f6e8f2">broad.pda.seq.graph.Path</a>
</li>
<li>getNodeElements()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_generic_rooted_tree_3_01_t_01_4.html#ade1222f912c50f83e1870676bf3b8a80">broad.core.datastructures.GenericRootedTree&lt; T &gt;</a>
</li>
<li>getNodeLikelihoodParameterMap()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model.html#aa8f9f0fbedc8a9f768fb1bbbef27c8f8">broad.core.siphy.EvolutionaryModel</a>
</li>
<li>getNodePair()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a94688ea637f3ddea7183104a100f3d60">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
, <a class="el" href="classumms_1_1core_1_1scripture_1_1_oriented_chromosome_transcript_graph.html#a3e30facaf439c3a3b0730e1fbcdfcab2">umms.core.scripture.OrientedChromosomeTranscriptGraph</a>
</li>
<li>getNodes()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_generic_rooted_tree_3_01_t_01_4.html#a108d8b6e982aa44309a0c98d52d55a47">broad.core.datastructures.GenericRootedTree&lt; T &gt;</a>
</li>
<li>getNominalPValue()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score.html#aebdc322e453c50557063a98d27f8a203">broad.pda.seq.segmentation.GeneScore</a>
</li>
<li>getNominalPValueMatrix()
: <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_differential_expression.html#a4909eee42faab431a9c0fa9267f948b6">broad.pda.differentialExpression.DifferentialExpression</a>
</li>
<li>getNonControl1PercentileSignal()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_array_stats.html#a2a9bff75bfd2b26eeab456b313cb6264">broad.pda.geneexpression.agilent.AgilentArrayStats</a>
</li>
<li>getNonControl50PercentileSignal()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_array_stats.html#a1e4958b4239f5d8ac69f606376b47231">broad.pda.geneexpression.agilent.AgilentArrayStats</a>
</li>
<li>getNonControl99PercentileSignal()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_array_stats.html#a6e40e5ea62b4d36df8ed8e37e82fabb4">broad.pda.geneexpression.agilent.AgilentArrayStats</a>
</li>
<li>getNonRandomTotalSize()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_directory_installed_genome_assembly.html#aff762dacbc8fb6031f31388771d745cb">broad.pda.feature.genome.DirectoryInstalledGenomeAssembly</a>
</li>
<li>getNormalizationFactor()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_array_stats.html#a8aa9de3ef20fde93707fcf8a29d175e4">broad.pda.geneexpression.agilent.AgilentArrayStats</a>
</li>
<li>getNormalizedKSEnrichments()
: <a class="el" href="classbroad_1_1projection_1_1gsa_1_1_gene_set_enrichment.html#a4ee185cd70bea1b1eacdd5a84c9e9932">broad.projection.gsa.GeneSetEnrichment</a>
</li>
<li>getNormalizedMaxMeanEnrichments()
: <a class="el" href="classbroad_1_1projection_1_1gsa_1_1_gene_set_enrichment.html#ae718cd0b1c52ed4cc0ad006202b29444">broad.projection.gsa.GeneSetEnrichment</a>
</li>
<li>getNormalizedScore()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a4b2cdaf4a39c83bd8e6ae4f480ef7c76">umms.core.annotation.Gene</a>
</li>
<li>getNormalizedSignal()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_probe.html#a961eff4190abc4a65eb61944889a87fd">broad.pda.geneexpression.agilent.AgilentProbe</a>
</li>
<li>getNullDistDegreesOfFreedom()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_fisher_combined_probability_test.html#aef25df170428bcb78af34b00a5b8bb57">broad.core.math.FisherCombinedProbabilityTest</a>
</li>
<li>getNum3PrimeComplementarity()
: <a class="el" href="interfacebroad_1_1core_1_1primer3_1_1_primer3_picking_summary.html#af4b53c02396212c708a6859dd802d550">broad.core.primer3.Primer3PickingSummary</a>
</li>
<li>getNum3PrimerStability()
: <a class="el" href="interfacebroad_1_1core_1_1primer3_1_1_primer3_picking_summary.html#a95545d2c4807cd023ef5e4627fc1ca5c">broad.core.primer3.Primer3PickingSummary</a>
</li>
<li>getNumberColumns()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#ae7c7bc88c04efcaa8638ffea805bf209">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>getNumberMarkers()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#afe80f65d9956c7236d1565af5292d86b">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#adba4f6a27c198a96fcfa79a643fca0a3">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>getNumberOfAligningSequences()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#a342692470eca8bfd3fc0d8227f66e0e3">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>getNumberOfBackgroundNonUniformFeaturesOverlimit()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_array_stats.html#aae400a7eb9325d900a5b8760f2a8a645">broad.pda.geneexpression.agilent.AgilentArrayStats</a>
</li>
<li>getNumberOfBackgroundPopulationOverlimit()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_array_stats.html#aae25713abd84ba58b85fe5940ee6427b">broad.pda.geneexpression.agilent.AgilentArrayStats</a>
</li>
<li>getNumberOfBPs()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a5b689c01b77461487789d172e013a3f6">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>getNumberOfEdges()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#ab96f69a0d2cd728894d59a7b0b757efc">broad.pda.seq.graph.Path</a>
</li>
<li>getNumberOfFoundFeatures()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_array_stats.html#a62a9024ac1e09d46f264337a66186502">broad.pda.geneexpression.agilent.AgilentArrayStats</a>
</li>
<li>getNumberOfIsoforms()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a016957d14960c3e93a84c669c9096276">broad.pda.annotation.BEDFileParser</a>
, <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#ae17acc06fa08a70dba97d64b165b7260">broad.pda.annotation.Locus</a>
, <a class="el" href="classbroad_1_1pda_1_1gene_1_1_gene_with_isoforms.html#a3485eb306e62f1964875debc376a0276">broad.pda.gene.GeneWithIsoforms</a>
</li>
<li>getNumberOfMarkers()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a4c0a0f55287f464249e043a627acc561">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>getNumberOfMatches()
: <a class="el" href="classjaligner_1_1_alignment.html#aeb4d1e3f60d8ce729e23cba0351a6c0f">jaligner.Alignment</a>
</li>
<li>getNumberOfMatches2()
: <a class="el" href="classjaligner_1_1_alignment.html#a32636c678af76a2d435562d684679b92">jaligner.Alignment</a>
</li>
<li>getNumberOfNonUniformFeaturesOverlimit()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_array_stats.html#ab6bb263384208fefd612b57574a46291">broad.pda.geneexpression.agilent.AgilentArrayStats</a>
</li>
<li>getNumberOfPopulationOverlimit()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_array_stats.html#a2537e79268a9f357a46d9a28a4dafa35">broad.pda.geneexpression.agilent.AgilentArrayStats</a>
</li>
<li>getNumberOfReads()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_stored_alignment_stats.html#a2824bb30b0cef62811543540a07070f5">broad.pda.seq.alignment.StoredAlignmentStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a37fc1481c4cc33dd1f711370017c5231">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a961d4d29021dcee878de7c9641b3f6a3">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#affc935b97a13f0fc855cd5c1878dc8b2">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score.html#a0ff304f3db77f29bb3f7f1b4a57faa4b">broad.pda.seq.segmentation.GeneScore</a>
</li>
<li>getNumberOfReadsByChr()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a1492aaf55e384cd251aac1339f1a1e08">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
</li>
<li>getNumberOfSaturatedProbes()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_array_stats.html#aa55b8e9aa8418ad082d7edd4f02375ab">broad.pda.geneexpression.agilent.AgilentArrayStats</a>
</li>
<li>getNumberOfSequences()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_parser.html#a18db9a7a9677bdaa101504eb9654adc2">broad.pda.seq.fastq.FastqParser</a>
</li>
<li>getNumberOfSplicedReads()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#ad4132fd86a0976bb049930b0d02416b8">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a1374d685b7b526b17623120e12012bd7">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getNumberOfValuesLargerThan()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#ab66ebe060da4ef98cb0a57ef25180a81">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getNumberPositions()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a34660b307472e14d299bae95902346ef">umms.core.annotation.Gene</a>
</li>
<li>getNumberRows()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a991bbb49ec4c0a3dd409ebbc48975ccb">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>getNumCol()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix.html#a589b2f00b2f8f91f83aeb5852d941db3">broad.core.motif.PositionWeightMatrix</a>
</li>
<li>getNumColdMeltingTemp()
: <a class="el" href="interfacebroad_1_1core_1_1primer3_1_1_primer3_picking_summary.html#ac8ad89a533624b2f5d6f9752afd18cb8">broad.core.primer3.Primer3PickingSummary</a>
</li>
<li>getNumColumns()
: <a class="el" href="class_jama_1_1_matrix.html#a37f5c3aa789c5f938013453ee1890c5b">Jama.Matrix</a>
</li>
<li>getNumEndPosToErase()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix.html#afd512d123d1e3ac15a2a6294f160024a">broad.core.motif.PositionWeightMatrix</a>
</li>
<li>getNumeratorCount()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#a30acd35178f2104d19e57812869c583c">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#af7b4d14fef9b8f950b5b6d42d74db4bb">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#a87f4de8b66a065747c088ea070800a86">umms.core.model.score.WindowAllScore</a>
</li>
<li>getNumeratorLambda()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#aebf10de139ccc2bb6c5839758342290d">umms.core.model.score.WindowAllScore</a>
</li>
<li>getNumeratorRegionTotal()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#ac413fbfe0bff20f164ac39e730b00dab">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#a2dedcf22da864a46a6e805a398c79a63">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#aa84f99d749d254db6655f66a650f618a">umms.core.model.score.WindowAllScore</a>
</li>
<li>getNumeratorRPKM()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#a5895395a7e81a482257c8f28175ba67a">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#afe24f7cc069f72f5f01dd4dabe558bda">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#a7727a0c2ed1fcfd74a7bf986a344c419">umms.core.model.score.WindowAllScore</a>
</li>
<li>getNumeratorScanPVal()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_skellam_score.html#a7b931c83c7a5211fb236dddcb3a567f7">broad.pda.seq.rap.SkellamScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#affae49300346f55b174aa8ff6bb32820">umms.core.model.score.WindowAllScore</a>
</li>
<li>getNumeratorTotal()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#a450ef192f2f085afc2a59d0687772b35">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#acb1ab9baaeec7bc88420b065a09a4060">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#a6961f90f44a7f5b9a1b9d3bb88f26ebc">umms.core.model.score.WindowAllScore</a>
</li>
<li>getNumExons()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a1a7e633ec4a457829ae8f17223d5d58c">umms.core.annotation.Gene</a>
</li>
<li>getNumGC()
: <a class="el" href="interfacebroad_1_1core_1_1primer3_1_1_primer3_picking_summary.html#ac6b75a2de192bb0724f4aeee8efea571">broad.core.primer3.Primer3PickingSummary</a>
</li>
<li>getNumGCClampFailures()
: <a class="el" href="interfacebroad_1_1core_1_1primer3_1_1_primer3_picking_summary.html#a0c5563c75d36a76f591e2436500c02fc">broad.core.primer3.Primer3PickingSummary</a>
</li>
<li>getNumHotMeltingTemp()
: <a class="el" href="interfacebroad_1_1core_1_1primer3_1_1_primer3_picking_summary.html#abba1b8794f3326b08bfca5d433308fe3">broad.core.primer3.Primer3PickingSummary</a>
</li>
<li>getNumIsoformsPerSet()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#a7a094d4a22f912ce3659c27f063e573c">broad.pda.annotation.Locus</a>
</li>
<li>getNumLeft()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_combination_generator.html#aa4cbe2046427cc15d0418d9cedc366fc">broad.core.math.CombinationGenerator</a>
</li>
<li>getNumMarkers()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_stored_alignment_stats.html#ad5894bb9942d01028c5ea0c5bc7e2c1e">broad.pda.seq.alignment.StoredAlignmentStats</a>
</li>
<li>getNumMispriming()
: <a class="el" href="interfacebroad_1_1core_1_1primer3_1_1_primer3_picking_summary.html#a64ab4f797b5e0d7883d3fb74db99ba5c">broad.core.primer3.Primer3PickingSummary</a>
</li>
<li>getNumNs()
: <a class="el" href="interfacebroad_1_1core_1_1primer3_1_1_primer3_picking_summary.html#a19404fdd204f179bcb4fc58bc3b20900">broad.core.primer3.Primer3PickingSummary</a>
</li>
<li>getNumOk()
: <a class="el" href="interfacebroad_1_1core_1_1primer3_1_1_primer3_picking_summary.html#ab975ff3f9887d1538445df2d13d825a5">broad.core.primer3.Primer3PickingSummary</a>
</li>
<li>getNumOligosTested()
: <a class="el" href="interfacebroad_1_1core_1_1primer3_1_1_primer3_picking_summary.html#afd6699d6613f8994a5671568840ce984">broad.core.primer3.Primer3PickingSummary</a>
</li>
<li>getNumOverlappingAnnotations()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4.html#abdda73dbcc304eb45a3c7b63ed115d77">umms.core.annotation.AnnotationList&lt; T extends Annotation &gt;</a>
</li>
<li>getNumOverlappingExcludedRegions()
: <a class="el" href="interfacebroad_1_1core_1_1primer3_1_1_primer3_picking_summary.html#a0274c11847071145e95e07153084d6f4">broad.core.primer3.Primer3PickingSummary</a>
</li>
<li>getNumOverlappingTargets()
: <a class="el" href="interfacebroad_1_1core_1_1primer3_1_1_primer3_picking_summary.html#a93ebd478c3f76036ec80a335b04ced68">broad.core.primer3.Primer3PickingSummary</a>
</li>
<li>getNumPolyX()
: <a class="el" href="interfacebroad_1_1core_1_1primer3_1_1_primer3_picking_summary.html#a2f9cc932b398ae64414cf83c3280a82e">broad.core.primer3.Primer3PickingSummary</a>
</li>
<li>getNumPossibleKmers()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_sequence_motif.html#a6f6e246a25237a0b46d9a5f1a6ec5df2">broad.core.motif.SequenceMotif</a>
</li>
<li>getNumReads()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model_1_1_alignment_count.html#a073955ebe911c70f69f26009a4f056a2">umms.core.model.AlignmentModel.AlignmentCount</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_1_1_alignment_count.html#a968c72b34a071fb004304deb0b9c4410">umms.core.model.JCSAlignmentModel.AlignmentCount</a>
</li>
<li>getNumRows()
: <a class="el" href="class_jama_1_1_matrix.html#a4dd1acb3763939c5efb6c10c3121818b">Jama.Matrix</a>
</li>
<li>getNumSelfComplementarityAnywhere()
: <a class="el" href="interfacebroad_1_1core_1_1primer3_1_1_primer3_picking_summary.html#a7711605db5d3a253742f4509932b9daf">broad.core.primer3.Primer3PickingSummary</a>
</li>
<li>getNumSeqQuality()
: <a class="el" href="interfacebroad_1_1core_1_1primer3_1_1_primer3_picking_summary.html#a70e5c63abd3a583acff30b5b801eeeab">broad.core.primer3.Primer3PickingSummary</a>
</li>
<li>getNumStartPosToErase()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix.html#a493b3736cd07a224265e66cd1609624c">broad.core.motif.PositionWeightMatrix</a>
</li>
<li>getNumUniqueReads()
: <a class="el" href="classbroad_1_1pda_1_1countreads_1_1_fastq_library_stats.html#a3d80a8886d5085a020400bbb2401acfd">broad.pda.countreads.FastqLibraryStats</a>
</li>
<li>getOffset()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_abstract_fasta_parser.html#a7abf04fe7636662479e5f1d52e9fd250">broad.core.sequence.AbstractFastaParser</a>
</li>
<li>getOneRandomSamplePermutation()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#a513b0bf190da04d2d446f9d76e2f0da6">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>getOpen()
: <a class="el" href="classjaligner_1_1_alignment.html#a38930d997ad59fc254e83b8aee0a2cbd">jaligner.Alignment</a>
</li>
<li>getOppositeVertex()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a25152ad6fcc6df9039f78c2b408fc717">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>getOrCreatePairedEndFile()
: <a class="el" href="classumms_1_1core_1_1readers_1_1_paired_end_reader.html#a46eda7e5fccfc727a3164ced595a9384">umms.core.readers.PairedEndReader</a>
</li>
<li>getOrdered()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_cluster.html#a23e3c164ae7f07b42c2cd29985c0a548">broad.pda.geneexpression.clustering.Cluster</a>
</li>
<li>getOrderedAssembly()
: <a class="el" href="classumms_1_1core_1_1scripture_1_1statistics_1_1_connect_disconnected_transcripts.html#ad55264ba1c376afdc3c838f8cf4905d1">umms.core.scripture.statistics.ConnectDisconnectedTranscripts</a>
</li>
<li>getOrderedListCDT()
: <a class="el" href="classbroad_1_1core_1_1util_1_1_parse_g_c_t_file.html#aa656d8a652b1fee6d5bc43464a06719f">broad.core.util.ParseGCTFile</a>
</li>
<li>getOrientation()
: <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_genomic_annotation.html#a36373aa2575e846905204c09bf74f041">broad.core.annotation.GenomicAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation.html#a1179b529806740f0a0302f8cd1c44b2d">broad.core.annotation.LightweightGenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a0e94aaa43793d75bd5c34d0a7e71f1a1">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_transcribed_regions.html#a065fbc2f6f4b0de5564df618b9ca8376">broad.core.sequence.TranscribedRegions</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#ac58d8ef7f61e00ae2ee627c67c645595">broad.pda.seq.graph.Path</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a6c978b97041609bfee866d81de3db450">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#ad5125f9609856d2a1b8e788664551558">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#aa5ccd2fae3fd4bd937ba202512db8030">umms.core.annotation.BasicAnnotation</a>
</li>
<li>getOrientedEnd()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_genomic_annotation.html#a7928fb714b0f90833b6d5852ac6ff207">broad.core.annotation.BasicGenomicAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_genomic_annotation.html#aa4e0c451e63fc794926460facfe7732d">broad.core.annotation.GenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#af96253e8f85e32a8dfceb9eca62eb825">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation.html#a46f8f6a0ce8eae87fddbd2fa344ea050">umms.core.annotation.AbstractAnnotation</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#a6c505cf912abeadd9d3647d0ee719ed2">umms.core.annotation.Annotation</a>
</li>
<li>getOrientedLastExon()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a265d68946dbee7ee557fc7d3936246b1">umms.core.annotation.Gene</a>
</li>
<li>getOrientedStart()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_genomic_annotation.html#a54aa0c6d854d7f5935dbc70e9de0e2c9">broad.core.annotation.BasicGenomicAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_genomic_annotation.html#ac6b1fdda54ca70227d23b707813a247c">broad.core.annotation.GenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a3f0accf8e2e5997d5d20d5c7696abc5f">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation.html#a5c5ecd3b98fa57d233844b24ed04637d">umms.core.annotation.AbstractAnnotation</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#ad11e657218df42bd003f432797e92f34">umms.core.annotation.Annotation</a>
</li>
<li>getOriginalBamFile()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#ac929b442873bd023a32feb28dac3dbe9">broad.pda.seq.clip.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a3a3c58c0def5a809f7949a2f441e815d">broad.pda.seq.protection.SampleData</a>
</li>
<li>getOriginalMValues()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_loess.html#a4a20d061bf64133e8364fc0bdf5630ba">broad.core.math.Loess</a>
</li>
<li>getOriginalScore()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_hairpin_kmer.html#ab743db5421aed53935c4bc5ff9183012">broad.core.primer3.HairpinKmer</a>
</li>
<li>getOriginalSequence1()
: <a class="el" href="classjaligner_1_1_alignment.html#a0c12e63557ffe3811418054b100744bf">jaligner.Alignment</a>
</li>
<li>getOriginalSequence2()
: <a class="el" href="classjaligner_1_1_alignment.html#acfe767713eefb64f61d62632347ca04a">jaligner.Alignment</a>
</li>
<li>getOrphanNodes()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a971a1afc17edfd3822cf93cb00011161">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>getOrphanPaths()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#abfc48453fd3cd296b35eebd812cf471e">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
, <a class="el" href="classumms_1_1core_1_1scripture_1_1_oriented_chromosome_transcript_graph.html#a4d84a0c47daf982d58aa6057531e0294">umms.core.scripture.OrientedChromosomeTranscriptGraph</a>
</li>
<li>getOrphanVertices()
: <a class="el" href="classumms_1_1core_1_1scripture_1_1_oriented_chromosome_transcript_graph.html#a3d428fc8927b3ebb4051f4d3858cc0ac">umms.core.scripture.OrientedChromosomeTranscriptGraph</a>
</li>
<li>getOutOrUnorientedEdges()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#aceaf71f4821bdafa6bd63423762ef9fc">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>getOutputFile()
: <a class="el" href="classumms_1_1core_1_1writers_1_1_paired_end_writer.html#a5ca002632fb9f4f13cf7883b1d5fce66">umms.core.writers.PairedEndWriter</a>
</li>
<li>getOverlap()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#a37f37590e75f2cbbb1268a59b8a686f5">broad.core.annotation.BasicLightweightAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation.html#a0a033cc9a97af87142b83afa0317668c">broad.core.annotation.LightweightGenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1overlaputils_1_1_gene_set_intersect.html#af40f199698b8f8ae56e31ccfb162e5e5">broad.core.overlaputils.GeneSetIntersect</a>
, <a class="el" href="classbroad_1_1core_1_1primer3_1_1_hairpin_kmer.html#ae6410252588bf4ba7a0d87f866ee9b4c">broad.core.primer3.HairpinKmer</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#af2e91a26d22501beed8d7c3d5a7b5d26">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#af3b8df9d79d125c32fb4e9500662e819">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a23906f94da29ba7bd51a057b822fe916">umms.core.annotation.BasicAnnotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a1f2ec9d846b8abf619b001696215e828">umms.core.annotation.Gene</a>
</li>
<li>getOverlappers()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#a9105fc31a2ce3c17aae9804da9254702">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
, <a class="el" href="classbroad_1_1core_1_1siphy_1_1_stationary_distribution_i_o.html#a10be4a3b31d2fdb2d49acc1a8df3891e">broad.core.siphy.StationaryDistributionIO</a>
, <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a5ee44c0a7cd95e4d0c36e58f82efb8d2">broad.pda.annotation.BEDFileParser</a>
, <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_location_aware_matrix.html#a1a3c6c34c6d39eb5b262362d46a3521d">broad.pda.datastructures.LocationAwareMatrix</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#ae68cc738e565b9a1c21516d756e760dc">umms.core.annotation.Gene</a>
</li>
<li>getOverlappingAnnotationList()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4.html#a6f97cda4309af1c52286ca2933d0b3fb">umms.core.annotation.AnnotationList&lt; T extends Annotation &gt;</a>
</li>
<li>getOverlappingAnnotations()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#a7e46ae18a43e66b8da2b821853df3fd5">umms.core.annotation.AbstractAnnotationCollection&lt; T extends Annotation &gt;</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#a9a2bd5c17d881c7e3decce83c8e2e8e6">umms.core.annotation.AnnotationCollection&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4.html#ab6327322586c8fae041be25cf91facb1">umms.core.annotation.AnnotationList&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#af3b5cd3178779e5b2022e1739af7ef85">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ada32da9c62bd9a077cc24cc9921576da">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getOverlappingGenes()
: <a class="el" href="classumms_1_1core_1_1scripture_1_1_build_scripture_coordinate_space.html#ad01703d5c482fa9d72c9be36be080627">umms.core.scripture.BuildScriptureCoordinateSpace</a>
</li>
<li>getOverlappingNodes()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#ac04ef0dc00df0156632abeaf208c66fc">broad.pda.seq.graph.Path</a>
</li>
<li>getOverlappingReadCounts()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a499fbcef653f843914b3c053fd5d2aa5">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a554eedd0b97cb7567223f951394082ea">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getOverlappingReadReplicateCounts()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a8db29afa1455774d98060c04f76b6c1c">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ac6831a54e791f8a59207c387ebda79cc">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getOverlappingReads()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#af05b3d980c6e5c0bc831d5c4872748f8">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#add9b428e55a7f0a7619a08726b4acad5">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getOverlappingRegion()
: <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html#a41c8790447f098e8a88c44f8dd5c3b80">umms.core.coordinatesystem.CoordinateSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_genomic_space.html#a6634f9ac9b3c5e86166c004a63e74d6b">umms.core.coordinatesystem.GenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_masked_genomic_space.html#aec28f0413edd73d141901a8c70a3d443">umms.core.coordinatesystem.MaskedGenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcript_in_genomic_space.html#a7dea0503116c1b5bed03059d3fd6d564">umms.core.coordinatesystem.TranscriptInGenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcriptome_space.html#ae06f880bbb3b66420f0e9f70087ad8c5">umms.core.coordinatesystem.TranscriptomeSpace</a>
</li>
<li>getOverlappingRegions()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#ad367e7266ec3b29c10479ccf4cf09778">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#ac8781a27f3bba71092316dfbf9583b0b">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getOverlappingSplicedReads()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a6ab24fe8d449fe58682f3d4ce3470c27">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#af4bf9a81aa77ef353a272037c28f9a37">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getOverlappingVertices()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#aeeeb5f64b18f9125c5dcb3b392dbfb7c">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>getOverlapWithRandomizedPositions()
: <a class="el" href="classbroad_1_1core_1_1overlaputils_1_1_gene_set_intersect.html#ac93739572a200cb1ab294d3145f1e902">broad.core.overlaputils.GeneSetIntersect</a>
</li>
<li>getP()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_mann_whitney_permutations.html#a4a9dc8521da59a20c7773e167b25dd0e">broad.core.math.MannWhitneyPermutations</a>
, <a class="el" href="classbroad_1_1core_1_1math_1_1_wilcox2_sample.html#aa3f90a2d43669b4e83399e38a73a0d8d">broad.core.math.Wilcox2Sample</a>
</li>
<li>getPair1Parser()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_splitter_by_barcode.html#a9ece5d69acc4979a5c60929c483f5ede">broad.pda.seq.fastq.FastqSplitterByBarcode</a>
</li>
<li>getPair2Parser()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_splitter_by_barcode.html#a7f34e04a652de1adc982f14fc6482eac">broad.pda.seq.fastq.FastqSplitterByBarcode</a>
</li>
<li>getPairedData()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a1db375ffca9153b8143df70c0bece395">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>getPairedEndEdges()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#a0f08036276af74a69d99ba13a0a55a84">broad.pda.seq.graph.Path</a>
</li>
<li>getPairedEndFile()
: <a class="el" href="classumms_1_1core_1_1readers_1_1_paired_end_reader.html#afaaf969aed6aaa75928ba7e6e17586e7">umms.core.readers.PairedEndReader</a>
</li>
<li>getParameters()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model.html#a362676dce20e057dfa97bb7e60ea0886">broad.core.siphy.EvolutionaryModel</a>
, <a class="el" href="classbroad_1_1core_1_1util_1_1_c_l_util.html#a02adb4236d194361a68ee94ad366e14a">broad.core.util.CLUtil</a>
</li>
<li>getParents()
: <a class="el" href="classumms_1_1core_1_1utils_1_1_annotation_utils.html#ac03f2394577fbf5f2bf25d041f195868">umms.core.utils.AnnotationUtils</a>
</li>
<li>getPaths()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#ab63a00a2c99d961006c1aa684c9ff508">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
, <a class="el" href="classumms_1_1core_1_1scripture_1_1_build_scripture_coordinate_space.html#a21a8a4ecd7583dba621994b16570f72a">umms.core.scripture.BuildScriptureCoordinateSpace</a>
, <a class="el" href="classumms_1_1core_1_1scripture_1_1_oriented_chromosome_transcript_graph.html#a052d51d6c53b23588f9cec5df1af36b8">umms.core.scripture.OrientedChromosomeTranscriptGraph</a>
</li>
<li>getPeak()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a00f7056035916ce93d6b87a8b5c17cd6">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#acf64ec3269f39b7e56e9b0ec5eacec52">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a3d06acc7e76ca7cbf52b2e6f41ad2bb1">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#abfcc396fc6a74370074144b1ba648520">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a1f85a13c310238f87b145bcc8bfe20b2">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ace2f29aff75114c5a3cb64c867781006">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getPeakBedFileName()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#afea3110e51be144225c55f1138761a50">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>getPeakScores()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_collect_annotation_enrichments.html#a608e157802fd681c674d43c4ae81fa3b">broad.pda.seq.rap.CollectAnnotationEnrichments</a>
</li>
<li>getPercent3UTR()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#aadc804eb8e2ac39c271509a51b1091ef">umms.core.annotation.Gene</a>
</li>
<li>getPercentCDS()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#ac62559d5d9adc8f6b3f8ac9da7d41a54">umms.core.annotation.Gene</a>
</li>
<li>getPercentConsistent()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#a85749f3f13bf21ae8f40d6643a01867a">broad.pda.seq.graph.Path</a>
</li>
<li>getPercentDuplicated()
: <a class="el" href="classbroad_1_1pda_1_1countreads_1_1_fastq_library_stats.html#a6d681ea21ea795c5c29e87eaa1a3152a">broad.pda.countreads.FastqLibraryStats</a>
</li>
<li>getPercentFeaturesWellAboveBackground()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_array_stats.html#aef36284a6d205a39136e5aefa10438b3">broad.pda.geneexpression.agilent.AgilentArrayStats</a>
</li>
<li>getPercentGC()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_hairpin_kmer.html#a1fc153d6348e1a2f959d230b005c3f04">broad.core.primer3.HairpinKmer</a>
</li>
<li>getPercentIdentity()
: <a class="el" href="classjaligner_1_1_alignment.html#a14c3147b9f914f86e93eba1a7ccb86f1">jaligner.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_p_s_l_record.html#a23ee2735be5903fd7a082554d1bf6edf">umms.core.annotation.PSLRecord</a>
</li>
<li>getPercentOfValuesBetween()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#a3c7586beedfa27952720655075a5e939">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getPercentOfValuesLargerThan()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#a1b8be955520da05ba3b7a5074848fee2">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getPercentVariantsCovered()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_hairpin_kmer.html#a2c34bc14ef01361fe29d89f287ba937e">broad.core.primer3.HairpinKmer</a>
</li>
<li>getPermNum()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_neighbors_null_model.html#afe164ab163e240d3c81cb05928de934a">broad.pda.gene.NeighborsNullModel</a>
</li>
<li>getPermutationMatrix()
: <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_differential_expression.html#a7740449cb640715c9e7dd3eb62fdcaa7">broad.pda.differentialExpression.DifferentialExpression</a>
</li>
<li>getPermutedAnnotations()
: <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#a83cb095bf99eafd3075bdf8d75da422f">umms.core.annotation.AnnotationCollection&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4.html#a35723c6193710912989fdd204d40e3f1">umms.core.annotation.AnnotationList&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a50d046e38dea99f843039193c615cea8">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a62dbb73b72268c223a90963ff37cc0ad">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getPi()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model.html#a535a4ad36f0c8da1dd617be5cfed66a1">broad.core.siphy.EvolutionaryModel</a>
</li>
<li>getPIDToName()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a2f79f7b9a68bf397ebb2c5984c1bf4b3">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>getPivot()
: <a class="el" href="class_jama_1_1_l_u_decomposition.html#aa85e93ba287bc938514395c92c6c54cb">Jama.LUDecomposition</a>
</li>
<li>getPlateName()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1_expression_experiment_info.html#aef3d20dab34dc299f66fe686bb0fa1c1">broad.pda.geneexpression.ExpressionExperimentInfo</a>
</li>
<li>getPoissonLambda()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_two_sample_scan_skellam_peak_caller.html#a1fd8a00c157f21f8fde2572a7b70887d">broad.pda.seq.protection.TwoSampleScanSkellamPeakCaller</a>
</li>
<li>getPosition()
: <a class="el" href="interfacebroad_1_1core_1_1siphy_1_1_fit.html#acd7d4b4ccc02fec178920ac66a6322f4">broad.core.siphy.Fit</a>
</li>
<li>getPositionAtCoordinate()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_compound_interval.html#a24e2ec501a12d06192e9dc77fe038cb4">umms.core.annotation.CompoundInterval</a>
</li>
<li>getPositionAtReferenceCoordinate()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a814cf3f63d0af28f322e31ada59fc208">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation.html#a68f1acbc77b4fcf0eb0220a5ccd79281">umms.core.annotation.AbstractAnnotation</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#ad9091621cfc9cbcbdd479bb110977fde">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#ae69c53e957652448229c91fc0e56fa73">umms.core.annotation.BasicAnnotation</a>
</li>
<li>getPositionCountList()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#ae5b5c83c1ef855c59c804217e6522c95">umms.core.model.AlignmentModel</a>
</li>
<li>getPositionCountMap()
: <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcriptome_space.html#a7dc129bef498e547154553c2e6ccf067">umms.core.coordinatesystem.TranscriptomeSpace</a>
</li>
<li>getPreferredFileExtension()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_fasta_multiple_alignment_i_o.html#a7c484cb6c86e23c98dff7add42684dfc">broad.core.multiplealignment.FastaMultipleAlignmentIO</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_i_o.html#ae65f18d5f17bfc130ce07c8bc4f001d0">broad.core.multiplealignment.MAFIO</a>
, <a class="el" href="interfacebroad_1_1core_1_1multiplealignment_1_1_multiple_alignment_i_o.html#a188e4b85766ab0438ccb1f22a0eaa23a">broad.core.multiplealignment.MultipleAlignmentIO</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_phylip_interleaved_multiple_alignment_i_o.html#a5f071bdcdad8ea9a4dcafbbd8ebc9b2a">broad.core.multiplealignment.PhylipInterleavedMultipleAlignmentIO</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_phylip_sequencial_multiple_alignment_i_o.html#a2fa2d40e02fb7a1485a39bf5049507aa">broad.core.multiplealignment.PhylipSequencialMultipleAlignmentIO</a>
</li>
<li>getPrimerPickingSummary()
: <a class="el" href="interfacebroad_1_1core_1_1primer3_1_1_primer3_primer_picking_result.html#a6190f217893c55d20bdf4f726d568972">broad.core.primer3.Primer3PrimerPickingResult</a>
</li>
<li>getPrimers()
: <a class="el" href="interfacebroad_1_1core_1_1primer3_1_1_primer3_primer_picking_result.html#a561371902252808073e96d020f498e8e">broad.core.primer3.Primer3PrimerPickingResult</a>
</li>
<li>getPriorClosest()
: <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_location_aware_matrix.html#a96adb475ad10bd7346702f01bf66b379">broad.pda.datastructures.LocationAwareMatrix</a>
</li>
<li>getProbability()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#a8ebca08775010dcd36c0d4dacd19fb90">broad.core.math.EmpiricalDistribution</a>
, <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_column.html#a1018f31a29569c2ecf2350c363bd2c75">broad.core.motif.PositionWeightColumn</a>
</li>
<li>getProbename()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_probe.html#aa2762410b7558415f57027407bd3b664">broad.pda.geneexpression.agilent.AgilentProbe</a>
</li>
<li>getProcessedSignal()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_probe.html#a68212359aba098533a6e6896f0e62eb7">broad.pda.geneexpression.agilent.AgilentProbe</a>
</li>
<li>getPromoter()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a7547d62e9d7421183cf1aafc864a2e1a">umms.core.annotation.Gene</a>
</li>
<li>getPromoters()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a2f72490110162bff435fc33c1d011b9d">broad.pda.annotation.BEDFileParser</a>
</li>
<li>getPVal()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model.html#a897c4df0be1002ef9fda5180e2cf4f83">broad.core.siphy.EvolutionaryModel</a>
</li>
<li>getPValue()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#a4fe6ff857ebe0d3a9b3b1a132f8f0003">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getPvalue()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_mann_whitney.html#af7a5889f4e6283800d73e8828421705a">broad.core.math.MannWhitney</a>
</li>
<li>getPValue()
: <a class="el" href="classbroad_1_1pda_1_1enrichment_1_1_enrichment_metric.html#ad8d4cf2c011985b7211e5ffc31062266">broad.pda.enrichment.EnrichmentMetric</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#af48a609ac806f6adcc886dcaa4884fb7">umms.core.annotation.Gene</a>
</li>
<li>getPvalue()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_basic_scan_statistic_score.html#aeea11b84828ad35985eaf18ba6f76e1d">umms.core.model.score.BasicScanStatisticScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_binomial_enrichment_score.html#a61dd9a809363a54c793a85de1aeae4f7">umms.core.model.score.BinomialEnrichmentScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_count_score.html#af2747cc400041fdf14c61065b2339139">umms.core.model.score.CountScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#aaff32cc152311616f6c10cb55c7849c6">umms.core.model.score.MultiScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#a92527822d24cf2809729926cd4866935">umms.core.model.score.PoissonEnrichmentScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_scan_statistic_score.html#a716eb4cc7d4c585cfb543cc6ede98dc8">umms.core.model.score.ScanStatisticScore</a>
</li>
<li>getPValue2()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#ae214d3d72550b348d271bae4e790ab0c">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getPWM()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1meme_1_1_meme_motif.html#a6538d8c841ba837042a0747c1b49ee7d">broad.core.motif.meme.MemeMotif</a>
</li>
<li>getPWMiterator()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_conserved_p_w_m_scanner.html#aa7018f23306757554a82bec2d677b763">broad.core.siphy.ConservedPWMScanner</a>
</li>
<li>getQ()
: <a class="el" href="class_jama_1_1_q_r_decomposition.html#ae2f72dd0466a1500e0c01efbb5c8bfa7">Jama.QRDecomposition</a>
</li>
<li>getQualityBases()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_fasta_quality_binary_i_o.html#a7c627c4bd0d0b5a9c4ae2005fd908065">broad.core.sequence.FastaQualityBinaryIO</a>
</li>
<li>getQuantileOfBins()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#aff802bd0118bab3ce71b97ab04f2e604">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getR()
: <a class="el" href="class_jama_1_1_q_r_decomposition.html#a918a7ff1594770cd4ee02e67241ef768">Jama.QRDecomposition</a>
</li>
<li>getRandomAnnotation()
: <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcriptome_space_1_1_gene_tree.html#ad1ff961107c21cbfeeb4592ae11340cb">umms.core.coordinatesystem.TranscriptomeSpace.GeneTree</a>
</li>
<li>getRandomSets()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1transcripts_null_model.html#af798a0b8314aa0e8b874fb345af3ae44">broad.pda.gene.transcriptsNullModel</a>
</li>
<li>getRankedList()
: <a class="el" href="classbroad_1_1projection_1_1gsa_1_1_gene_set_enrichment.html#addffe4a50dc870c52627c3302bedda9a">broad.projection.gsa.GeneSetEnrichment</a>
</li>
<li>getRateMatrix()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model.html#a876a93aa4071613481f8e4366e66169d">broad.core.siphy.EvolutionaryModel</a>
, <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model_parameters.html#a92475abf78a66378148441778bb8a4df">broad.core.siphy.EvolutionaryModelParameters</a>
</li>
<li>getRatio()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#a241f72296f12599ebbc03f6b6babc9b6">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#a68c52e528d1195138124c85d638eba7c">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#a6486ae7c419b77fa62e91550bd87617b">umms.core.model.score.WindowAllScore</a>
</li>
<li>getRawDensity()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#a2cf0c99f0309dbc67192ddaa70bdaa1d">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getRead()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model_1_1_alignment_count.html#aa7049d2e624f409b1d7ecd20565035f9">umms.core.model.AlignmentModel.AlignmentCount</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_1_1_alignment_count.html#a1a5d47398fa623367619763af8bdcf6c">umms.core.model.JCSAlignmentModel.AlignmentCount</a>
</li>
<li>getRead1Trim3p()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_splitter_by_barcode.html#a2d02fb9d2e05002de4a9a04e7256af3f">broad.pda.seq.fastq.FastqSplitterByBarcode</a>
</li>
<li>getRead1Trim5p()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_splitter_by_barcode.html#a49f2f853bde9c90cacc3ad97708f4f56">broad.pda.seq.fastq.FastqSplitterByBarcode</a>
</li>
<li>getRead2Trim3p()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_splitter_by_barcode.html#a61ee9709c707ae1f7fdbb05dcb887b41">broad.pda.seq.fastq.FastqSplitterByBarcode</a>
</li>
<li>getRead2Trim5p()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_splitter_by_barcode.html#a3536f1626889354dd010de2206fbc32b">broad.pda.seq.fastq.FastqSplitterByBarcode</a>
</li>
<li>getReadAlignmentBlocks()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a54e350bcbc168434f78ab6c2417c5eb4">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#aa05560844de602aa9ad47c63c422d73d">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a8ec7d5ed9c1d86ea1bbabe2966e1cac9">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>getReadAlignments()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a83479cb561a7d49efef887022b69c1c7">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#a563a8db2ae65cace4f3192798de6b542">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#add75d5f74290c291d87c7179aea0eaa7">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>getReader()
: <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_s_a_m_paired_end_file_reader.html#ab3636dbaeb6c8f851d60084603b30907">broad.pda.samtools.SAMPairedEndFileReader</a>
</li>
<li>getReadIterator()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#acc6162ee6b18c5abdc5a8a83c0f6ad56">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#aed73c1bc87882923e37e9d2deab47ced">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a74fc9bfd7375f0d90c457fcf9467d464">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#ac1001577f4c2e5be527f1d9c4321d1a5">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ae6c96c699d2e49d813d52eb0c039a277">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getReadLength()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a7947c9515e3e23e7edf4e16249f3d394">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>getReadMates()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#adeb9200fe5412ef288d4715b1622323c">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#a727bd0d8dbdf53adc79c290dcefb860c">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#ace8024acd8620c905790d25ffeda35fa">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>getReadName()
: <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_filter_fastq_reads.html#a796ed4e03547113036c82d2d4525c427">broad.pda.samtools.FilterFastqReads</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a8c6173f422b8e037747b02f8d728efe4">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#a3163e7ed8d2f62170825865c5f708b22">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#ac1f86f82faccc55aac35209be5901b5b">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>getReads()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_j_c_s_cache.html#aba7253cf42c957d9e361c30f4ed4e1f9">broad.core.datastructures.JCSCache</a>
</li>
<li>getReadSequence()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a5411152f2e68b099413caddc9dba148f">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#a35e95edade9121503e38f1e5a82dbfd6">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#ad2498b099878ec8d8599a7e0cddc16c8">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>getReadSizeDistribution()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#ab634dd7e626def19bea7ecf401ca04a5">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ac801ebc73dd4e2fc536214d62cfef81c">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getReadString()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment_1_1_wrap_sam_record.html#a99563863c6d0389540ae3a48a415c675">umms.core.alignment.SingleEndAlignment.WrapSamRecord</a>
</li>
<li>getRealEigenvalues()
: <a class="el" href="class_jama_1_1_eigenvalue_decomposition.html#a90b0d4e795512f334c5294e15b31ea93">Jama.EigenvalueDecomposition</a>
</li>
<li>getReasonForUnsafe()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_stored_alignment_stats.html#af0536b5d93aa4aadb3385d91a53147ad">broad.pda.seq.alignment.StoredAlignmentStats</a>
</li>
<li>getReducedRepresentation()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_directory_installed_genome_assembly.html#abfa6a7b8afd3f5f5bddb699939db5e28">broad.pda.feature.genome.DirectoryInstalledGenomeAssembly</a>
</li>
<li>getReference()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#a56143bbb7c10c52fe379caf646baf3ec">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>getReferenceAnnotation()
: <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html#a481e6661291d17633dff9f06dd034b61">umms.core.coordinatesystem.CoordinateSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_genomic_space.html#a791e805d7bcff80b9992473d7f3ce166">umms.core.coordinatesystem.GenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_masked_genomic_space.html#aa30285093525e3869868a0d581bfc9ca">umms.core.coordinatesystem.MaskedGenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcript_in_genomic_space.html#aa29f8f9716891ceec997edb80632a8a6">umms.core.coordinatesystem.TranscriptInGenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcriptome_space.html#a57cde968a85677a101950c8af6691844">umms.core.coordinatesystem.TranscriptomeSpace</a>
</li>
<li>getReferenceAnnotations()
: <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html#a50d49a54862b5c651d50a5c035ae2fd6">umms.core.coordinatesystem.CoordinateSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_genomic_space.html#a9c50958fee98b103c73b8787d2818ac5">umms.core.coordinatesystem.GenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_masked_genomic_space.html#a2f24dd26327cbf0149e784c49cb6d033">umms.core.coordinatesystem.MaskedGenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcript_in_genomic_space.html#ae511e3d677e4ae595d484e2e822b8322">umms.core.coordinatesystem.TranscriptInGenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcriptome_space.html#a14f2ecdae9339f65f3bb1b94c1fc1778">umms.core.coordinatesystem.TranscriptomeSpace</a>
</li>
<li>getReferenceCoordinateAtPosition()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a7fced373c02877b366667388203c64f9">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation.html#afe9d6ee4987afb23a573988a3bb922fd">umms.core.annotation.AbstractAnnotation</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#a71683725c96fca10f0134a088643ec36">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a97c73f4d65360b36306293e770c16f2a">umms.core.annotation.BasicAnnotation</a>
</li>
<li>getReferenceEnd()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_alignment.html#ac3c532409f19dd413db6111dea5b1bd1">broad.core.multiplealignment.MAFAlignment</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#aadca4a586e660bfae64331b2f4448c30">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>getReferenceId()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#aae72327f5ed0497dd4c97e6a40c85b6a">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>getReferenceName()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a3ef8c9ab35ce4e0033488dfaa45fe5be">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#a2dd53f6e6ab4a573c0e93ef42e223b93">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#aabdddcc39fc1c200ede2a7b72987127a">umms.core.annotation.BasicAnnotation</a>
</li>
<li>getReferenceNames()
: <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html#ac00dc2e5986d067615d65301bc93a63b">umms.core.coordinatesystem.CoordinateSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_genomic_space.html#a8f4a8587b9f12cb6644d9b2dc320259f">umms.core.coordinatesystem.GenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_masked_genomic_space.html#a4e5bf7a2a3cb551e7135205b8d59f854">umms.core.coordinatesystem.MaskedGenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcript_in_genomic_space.html#a42cdf9b1c85d87e9868cc8edfcdb2687">umms.core.coordinatesystem.TranscriptInGenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcriptome_space.html#a863507c0d75e79e7565ca8e01dd03265">umms.core.coordinatesystem.TranscriptomeSpace</a>
</li>
<li>getReferenceStart()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_alignment.html#a6b6655c27333075b60992c0a509aef72">broad.core.multiplealignment.MAFAlignment</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#aaff6f54d392c84a823118338da1aca1c">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>getReferenceTranscript()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#ab0a21b52c4209efeb94b69cd092ab55f">broad.pda.annotation.Locus</a>
</li>
<li>getRefseqGeneName()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a3e3efc2ec470bfbe104afe5759187351">broad.core.sequence.Sequence</a>
</li>
<li>getRefseqRnaClassName()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a4946d4cdaad7a472b878a0fac3bdaf85">broad.core.sequence.Sequence</a>
</li>
<li>getRefseqSpeciesName()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a3da56d57c8b71c784b28caad5b810856">broad.core.sequence.Sequence</a>
</li>
<li>getRefSequenceCounts()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a11f67951abb4648fe109809c361856d2">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#adee8ec95f5509ff13521ef6d73c9188e">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getRefSequenceLambda()
: <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#adb362c994b2498ea94212730974394f1">umms.core.annotation.AnnotationCollection&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4.html#afadddf6f7e885a2ed9b908041871eeb8">umms.core.annotation.AnnotationList&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#ad2ca51fe2918d5a556aaa72fd0372e72">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a660ec2a4de55d6ccc1652b33d64f80f3">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getRefSequenceLength()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a3b33fe828a04581c1106d7179a1182a6">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ad53bee1540d7f533e110b8fdb37d9f23">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getRefSequenceLengths()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#aa8168915afb028e9db4ecaf28338a6e5">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a3669cfd3c08b28363c1dabe962489d49">umms.core.model.JCSAlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1readers_1_1_paired_end_reader.html#a43aff465a170901ce11a079f921065a1">umms.core.readers.PairedEndReader</a>
</li>
<li>getRegion()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a0b1f162c2ca4fafb7ff10dbb09f7a5f2">broad.core.sequence.Sequence</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#ab6f99ae47039c2a507e8fefd3d75105a">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region_window.html#a9c0e9ea5d3532cea285c2e222a904d61">broad.core.sequence.SequenceRegionWindow</a>
, <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_chromosome.html#a43a6cf2a9046106d068032d1533fc60e">broad.pda.feature.genome.Chromosome</a>
, <a class="el" href="classbroad_1_1pda_1_1rnai_1_1_r_n_ai_gene_annotation.html#a10dc03049c54ea74b351497700177f98">broad.pda.rnai.RNAiGeneAnnotation</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcriptome_space_1_1_gene_tree.html#a2812ecf6ec4ce7e66677e01961b3fb8a">umms.core.coordinatesystem.TranscriptomeSpace.GeneTree</a>
</li>
<li>getRegionDenominatorRPKM()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#a1584f84d271b371c3bb7ec7bfe7b0f48">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#ad98b3c84e8de43604cf72faad4218649">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#a3392c352a23ed36bc60920a66a7d12de">umms.core.model.score.WindowAllScore</a>
</li>
<li>getRegionEnd()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a8963e3dc03bcfbdf74a57bb016b64cfb">broad.core.sequence.SequenceRegion</a>
</li>
<li>getRegionLength()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_basic_scan_statistic_score.html#a1511649a2bcc63582bd0716df87fc2f8">umms.core.model.score.BasicScanStatisticScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_binomial_enrichment_score.html#af5a279f23479b5f987ccf67f5f40bd13">umms.core.model.score.BinomialEnrichmentScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#a42ee81eee6d3792d4ed816c5fa9a98f3">umms.core.model.score.MultiScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#a4374b417b6f63a0a0f43a5632cf579f3">umms.core.model.score.PoissonEnrichmentScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_scan_statistic_score.html#adb697c290aecf4aab182532829f75c6b">umms.core.model.score.ScanStatisticScore</a>
</li>
<li>getRegionNumeratorRPKM()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#a26a42d5b27ecffe6bc7be64b5986d8fb">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#a1cd82d1ae365cb76b06c021a571e65cf">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#a52d6f602bc5ff6a0df966f204fe93284">umms.core.model.score.WindowAllScore</a>
</li>
<li>getRegionRatio()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#a7d5afcbe79df879267e40808def116e9">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#ac9e25c283add9bccf9953ed67f6ad6de">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#a0836eff5c3d1631e4c502fe96b8ea068">umms.core.model.score.WindowAllScore</a>
</li>
<li>getRegions()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a183530d8cd29a589efc7a195e5b6c176">broad.core.sequence.Sequence</a>
, <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_chromosome.html#a5b74ba6deffd94310cc648006654ada0">broad.pda.feature.genome.Chromosome</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_genome_command_line_program.html#af3dfa4b958b37f13cfccbe10e0ff7310">broad.pda.seq.rap.GenomeCommandLineProgram</a>
</li>
<li>getRegionSet()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_genome_command_line_program.html#a5114bc9a602f32c1c0e918e90b1f6f00">broad.pda.seq.rap.GenomeCommandLineProgram</a>
</li>
<li>getRegionsMap()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_genome_command_line_program.html#a2a2ff42e2b1724397090595d17a073d2">broad.pda.seq.rap.GenomeCommandLineProgram</a>
</li>
<li>getRegionStart()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#aa2f6c6d9bcf5b511b084490c58867a15">broad.core.sequence.SequenceRegion</a>
</li>
<li>getRegionTotal()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_count_score.html#a067ecd35931600525a2fcd10c25d0f7a">umms.core.model.score.CountScore</a>
</li>
<li>getRelationship()
: <a class="el" href="interfacebroad_1_1core_1_1datastructures_1_1_interval.html#a630b2eae8e23cd00c870f4f3b51303c5">broad.core.datastructures.Interval</a>
</li>
<li>getRelativeGenomeLength()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a069b16c959ceace301b673830ab49e1e">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_genome_with_gaps2.html#adccda0fda9f0b5c1b81e1b794398a8b3">broad.pda.seq.segmentation.GenomeWithGaps2</a>
</li>
<li>getRelativePositions()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_hairpin_kmer.html#a78a3c96c2b34cc37b558f390512a074c">broad.core.primer3.HairpinKmer</a>
</li>
<li>getRelativeWindow()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_genome_with_gaps2.html#ada3ca77892b7f8cee297ba4554c2c672">broad.pda.seq.segmentation.GenomeWithGaps2</a>
</li>
<li>getRepeatClass()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_repeat_masker_annotation.html#a5d76873dcfe380e8e7fe5e5a41dbc313">broad.core.annotation.RepeatMaskerAnnotation</a>
</li>
<li>getRepeatFamily()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_repeat_masker_annotation.html#a61fc73a94c83c7665240a71192fd9af4">broad.core.annotation.RepeatMaskerAnnotation</a>
</li>
<li>getRepeatFile()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_directory_installed_genome_assembly.html#a0c140dd9235aa9464c0ff812d25309a2">broad.pda.feature.genome.DirectoryInstalledGenomeAssembly</a>
</li>
<li>getRepeatName()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_repeat_masker_annotation.html#aa6caca9cc5147ba8badb8d98276ba919">broad.core.annotation.RepeatMaskerAnnotation</a>
</li>
<li>getREST()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1_score_diff_genes.html#a862844df645eec8a57210f139b94e337">broad.pda.geneexpression.ScoreDiffGenes</a>
</li>
<li>getResults()
: <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_run_differential_expression.html#a1d50d6f1bd2701f326827fc3af431e8b">broad.pda.differentialExpression.RunDifferentialExpression</a>
</li>
<li>getReverseStrand()
: <a class="el" href="enumumms_1_1core_1_1annotation_1_1_annotation_1_1_strand.html#adbacee1553c65383a56cdb22a22d74ad">umms.core.annotation.Annotation.Strand</a>
</li>
<li>getReverseTargets()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_smatch_like.html#ae07909228a8cc278e182d7f148d269c1">umms.core.alignment.SmatchLike</a>
</li>
<li>getRgb()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_b_e_d.html#ac4731b050a7b8a172795a285d655f1c0">broad.core.annotation.BED</a>
</li>
<li>getRight()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_cluster.html#a245291eb22328a4df2c3dc7bbab90f92">broad.pda.geneexpression.clustering.Cluster</a>
</li>
<li>getRightHighInfoStart()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix.html#a7f0911cdd2e978b07ab07c680415dfda">broad.core.motif.PositionWeightMatrix</a>
</li>
<li>getRightNeighbor()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_neighbor_analysis.html#a8594ddc3a3a3ba948613c1084652c271">broad.pda.gene.NeighborAnalysis</a>
</li>
<li>getRoot()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_generic_rooted_tree_3_01_t_01_4.html#a6d1bc149b8ededfc36567a254cdcd892">broad.core.datastructures.GenericRootedTree&lt; T &gt;</a>
</li>
<li>getRow()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#acd1df44e15cd98a48871a0583c8334c7">broad.core.datastructures.MatrixWithHeaders</a>
, <a class="el" href="classjaligner_1_1_cell.html#a8c1d97f4faf109109676ca858b0fb026">jaligner.Cell</a>
, <a class="el" href="class_jama_1_1_matrix.html#a2abc91fe8b387894bda1ff7a793c7b9d">Jama.Matrix</a>
</li>
<li>getRowAnnotations()
: <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_location_aware_matrix.html#a67a96cdf0a6b10a54641fd63d1974f96">broad.pda.datastructures.LocationAwareMatrix</a>
</li>
<li>getRowDescriptions()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a9969b7c99c32f1319eb5818012489f90">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>getRowDimension()
: <a class="el" href="class_jama_1_1_matrix.html#a1696a4fbbbc49c0e4ac665d104b3af1d">Jama.Matrix</a>
</li>
<li>getRowLocationMap()
: <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_location_aware_matrix.html#a58c5ec3cf67ef67dcb837812f48b47f0">broad.pda.datastructures.LocationAwareMatrix</a>
</li>
<li>getRowName()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#ac2d2bab0736a2a92c7b1c6085aa93945">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>getRowNames()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a5ed12303338cc6bb92792d8fea575231">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>getRownum()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_probe.html#af6e945427031897c3c5e8e56d37f3c22">broad.pda.geneexpression.agilent.AgilentProbe</a>
</li>
<li>getRowPackedCopy()
: <a class="el" href="class_jama_1_1_matrix.html#a4740d11abbac942d0f5873e55e1846c0">Jama.Matrix</a>
</li>
<li>getRowPosition()
: <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_location_aware_matrix.html#affcb1adcdf0e72debfeb7ac5c3da1290">broad.pda.datastructures.LocationAwareMatrix</a>
</li>
<li>getRowTree()
: <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_location_aware_matrix.html#a74078bbee72db894231be70f8a2fd8b2">broad.pda.datastructures.LocationAwareMatrix</a>
</li>
<li>getRPK()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_count_score.html#a922204b474c2dd5aea50202778bc34b6">umms.core.model.score.CountScore</a>
</li>
<li>getRPKM()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score.html#a770599865ed2f8b51cfe6fc277c7a029">broad.pda.seq.segmentation.GeneScore</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#ae5ec245344aa9abd44a4592315b8a7dc">umms.core.annotation.Gene</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_count_score.html#ae4061b9bc8daf3b622533645ae73902c">umms.core.model.score.CountScore</a>
</li>
<li>getRPKMConstant()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a2cfac25b852dd32fc35288340e02e4b3">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a943dc7415136c3885385896e7173c5b9">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
</li>
<li>getRS8Score()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_hairpin_kmer.html#a41cdc00d4ed4473e37c7ee28020e6680">broad.core.primer3.HairpinKmer</a>
</li>
<li>getS()
: <a class="el" href="class_jama_1_1_singular_value_decomposition.html#a9806f3bbaf6404473ef1f19f58779cff">Jama.SingularValueDecomposition</a>
</li>
<li>getSAMEnd()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a167dc4949f97136d3a0c7c2ff24621c7">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation.html#abf8179bf648101ed67eca28075e73a9e">umms.core.annotation.AbstractAnnotation</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#ad3de070378091d8ae6cad2d496abba22">umms.core.annotation.Annotation</a>
</li>
<li>getSamHeader()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a01edf28e0888aa2d53593fe9fa2ed1dd">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getSampleCoordSpace()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#ac5835952a0c41b13c469808d426f2d25">umms.core.model.score.MultiScore</a>
</li>
<li>getSampleCount()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_binomial_enrichment_score.html#adbd5e9efd8b56ad20b551e68da1f9bd5">umms.core.model.score.BinomialEnrichmentScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#aafd9d746298556ecbd1b365d349ffe13">umms.core.model.score.MultiScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#a2ac79b7c4d121adf4abb4dde5fce19e8">umms.core.model.score.PoissonEnrichmentScore</a>
</li>
<li>getSampleLambda()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#a0c84aab1e90ecb51a6cbb37bc64b71cc">umms.core.model.score.PoissonEnrichmentScore</a>
</li>
<li>getSampleName()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a23c6423a51684f6f61cb445eec5d98d6">broad.pda.seq.clip.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a9284f2dc44109171457dcc0417d8520a">broad.pda.seq.protection.SampleData</a>
</li>
<li>getSampleRegionCount()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_binomial_enrichment_score.html#a74d614de501b2811ae0eb9b966837db1">umms.core.model.score.BinomialEnrichmentScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#ac53520acee3ce90901ec454a491fdb61">umms.core.model.score.MultiScore</a>
</li>
<li>getSampleRegionTotal()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#a301a50635af5074544e263e7eecd05bc">umms.core.model.score.PoissonEnrichmentScore</a>
</li>
<li>getSampleTotal()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#a957d28b38461eee75b99a6228d24a966">umms.core.model.score.PoissonEnrichmentScore</a>
</li>
<li>getSampleType()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1_expression_experiment_info.html#a08698d848b21586d3d5c78cd8f4769e3">broad.pda.geneexpression.ExpressionExperimentInfo</a>
</li>
<li>getSAMRecordIterator()
: <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_s_a_m_paired_end_file_reader.html#a445a2860d8414157045797d3a79fd057">broad.pda.samtools.SAMPairedEndFileReader</a>
</li>
<li>getSAMStart()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#ab8bdec52b791c752123dbcb2f853a648">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation.html#a282494d138b1c62cc03b6202d1ddddef">umms.core.annotation.AbstractAnnotation</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#a26f021a7fa89d0cfd22f2ff2b7801388">umms.core.annotation.Annotation</a>
</li>
<li>getSamString()
: <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_s_a_m_fragment_writer.html#a488dd5ae131e2829175cd3d200ceb1e0">broad.pda.samtools.SAMFragmentWriter</a>
</li>
<li>getSAMString()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a46e618a4d6e89a7b69e3ec5563016a57">umms.core.annotation.Gene</a>
</li>
<li>getSaturationSignalValue()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_array_stats.html#aca1a6da3303d719474c669f6da2ed6fb">broad.pda.geneexpression.agilent.AgilentArrayStats</a>
</li>
<li>getScaledWindows()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_tree_scaler_i_o.html#ab1c676fec1471f7b23d7d1d5199a3878">broad.core.siphy.TreeScalerIO</a>
</li>
<li>getScanDistribution()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_permutation_distribution.html#a1f13a6af358b1359c0d56833ceefac11">broad.core.math.PermutationDistribution</a>
</li>
<li>getScanPvalFragmentEndsMaxPositionAllRegionsBam()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup.html#a75369d564eda2373e41b0fd60451fda8">broad.pda.seq.clip.Pileup</a>
</li>
<li>getScanPvalFragmentEndsMaxPositionBam()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup.html#af7726e161df33f1675b770110193078f">broad.pda.seq.clip.Pileup</a>
</li>
<li>getScanPvalsFragmentEndPileupsAllRegionsWig()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup.html#ae5bf54f9ea963c4d704bb7f2dd98c0ee">broad.pda.seq.clip.Pileup</a>
</li>
<li>getScanPvalsFragmentEndPileupsWig()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup.html#a2f2ec59cc7fdb4ced9c11014070fa10a">broad.pda.seq.clip.Pileup</a>
</li>
<li>getScanPvalue()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_permutation_distribution.html#adeb49888f4ff5c275859a9a4f306c5e9">broad.core.math.PermutationDistribution</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score.html#a37af0b606cb0805c4f70a196bd2c3554">broad.pda.seq.segmentation.GeneScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_basic_scan_statistic_score.html#abed9c5b27dd0af8fa6bb652d2964ebd8">umms.core.model.score.BasicScanStatisticScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_scan_statistic_score.html#ab240e94386a0a48a6f32be20f15c6976">umms.core.model.score.ScanStatisticScore</a>
</li>
<li>getScore()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#a09f0b0f3e4eab4d2ed93ffeb7419646d">broad.core.annotation.BasicLightweightAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_feature.html#ac6ed3306f42ecda11fa3d05913c19c62">broad.core.annotation.Feature</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation.html#a1331acfee9aa44b1987937f227be0885">broad.core.annotation.LightweightGenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#a8b53ad42c6e6fb83173552a18b4f54f0">broad.core.multiplealignment.MultipleAlignment</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#ab4107ed17a9cefec0e99ba9d5481cced">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region_window.html#aa18a140d583dc9ba4a0b2dfa28164b27">broad.core.sequence.SequenceRegionWindow</a>
, <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_alignments.html#ad035b618294c2b18a553049446b814f5">broad.pda.datastructures.Alignments</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_cluster.html#aee0b7c8eeca7c5aa03efc1e130bd173d">broad.pda.geneexpression.clustering.Cluster</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#a04360d2166579e8b16be57484e375d9b">broad.pda.seq.graph.Path</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_skellam_score.html#a4b26980b244bbb121634509a36c37905">broad.pda.seq.rap.SkellamScore</a>
, <a class="el" href="classjaligner_1_1_alignment.html#a81ee36cb77de6f426175da2a60791d8a">jaligner.Alignment</a>
, <a class="el" href="classjaligner_1_1_cell.html#aee1cdb5e8f186b4f0c75c5bcedee998e">jaligner.Cell</a>
, <a class="el" href="classjaligner_1_1matrix_1_1_matrix.html#a6bd8bdab7fab62b94e8f53a81f27c87d">jaligner.matrix.Matrix</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#a117f218fd9f4b0bbe5e183647c9d4826">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a9701cbbb370ff99a7436068616bf7f27">umms.core.annotation.BasicAnnotation</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_abstract_list_score.html#a2927a97c83af8b901402b0c84c414a07">umms.core.model.score.AbstractListScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_count_score.html#a1308c7c7d75f339947c29575099d41c3">umms.core.model.score.CountScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_end_in_window_count_score.html#a54e9f7ec3a667641fab13fa15b162f7c">umms.core.model.score.EndInWindowCountScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#abc8dd462110ee0768011709ea84b9346">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#a65ac8a6fe0603eb99cd0ba1de0302097">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_start_in_window_count_score.html#ac19f44fe80ce3d2ad0c236bb62d67c5c">umms.core.model.score.StartInWindowCountScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_sum_score.html#af2ac743d89d631a5f6bf69d6bd74db8a">umms.core.model.score.SumScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#a43b3c23abd2ac3c3bd49691656048433">umms.core.model.score.WindowAllScore</a>
, <a class="el" href="interfaceumms_1_1core_1_1model_1_1score_1_1_window_score.html#a5232038f3397874511f1e8d83df72e97">umms.core.model.score.WindowScore</a>
</li>
<li>getScoredExons()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_gene_with_isoforms.html#a2338d4fc34ef3ca18b07870da25e8b4b">broad.pda.gene.GeneWithIsoforms</a>
</li>
<li>getScorePerAlignmentFromCache()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a60519992f1a090084dda7052debef056">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#ac91865896814b6d061a12ffa3d578ad3">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#af0986112fd4ad0ec8bc544fead171a11">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getScores()
: <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_alignments.html#af6439db4165fd6e08b0c97e5ed96ec5f">broad.pda.datastructures.Alignments</a>
, <a class="el" href="classjaligner_1_1matrix_1_1_matrix.html#a813895a2ef37875372d5edc9a6ff9e1b">jaligner.matrix.Matrix</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_abstract_list_score.html#a4b08f13c6d1ee4abe0d0bbb1e5987251">umms.core.model.score.AbstractListScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_basic_scan_statistic_score.html#a7a19d5c09d09314d3f947991c1e8d086">umms.core.model.score.BasicScanStatisticScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_scan_statistic_score.html#a16b8e6d7030540cc9f22c249cbc81cfe">umms.core.model.score.ScanStatisticScore</a>
</li>
<li>getScoreSum()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#a268ade3e42daed82b3f2db46ee3fd896">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
</li>
<li>getScoreWithNoTerminalGaps()
: <a class="el" href="classjaligner_1_1_alignment.html#a43a82c42d3220541cbc9c9eb53d49f8e">jaligner.Alignment</a>
</li>
<li>getSecondGene()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_1_1_unordered_gene_pair.html#a3ded680df55f81b0e2060155291bb872">umms.core.alignment.FeatureSequenceAlignment.UnorderedGenePair</a>
</li>
<li>getSecondMate()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#aeb289b5bce028d7819f8f893a44be774">umms.core.alignment.AbstractPairedEndAlignment</a>
</li>
<li>getSentinel()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_interval_tree_3_01_v_01_4.html#a3542cde71dcd6d21b8b693ebacd38172">broad.core.datastructures.IntervalTree&lt; V &gt;</a>
</li>
<li>getSequence()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_genomic_annotation.html#aaa7708e6458bd5b16eade5e931ed917a">broad.core.annotation.BasicGenomicAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_genomic_annotation.html#ae913593fc234e211b353d1bd6b475728">broad.core.annotation.GenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a263ddf2c510c260b61b07f9d642c5f81">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_alignments.html#af85738a8a7dc405405d467a343ff307b">broad.pda.datastructures.Alignments</a>
, <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_chromosome.html#af39ea4a445eef9e789359baf6d93688b">broad.pda.feature.genome.Chromosome</a>
, <a class="el" href="classbroad_1_1pda_1_1rnai_1_1_r_n_ai_gene_annotation.html#a74d1b76c153f85b5311dff362966a5fc">broad.pda.rnai.RNAiGeneAnnotation</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_sequence.html#a60e407f2f62ab4e72667fa19f9f7b6e8">broad.pda.seq.fastq.FastqSequence</a>
, <a class="el" href="classjaligner_1_1_sequence.html#a8f74fb0c13f60b0c80780bdb07792a48">jaligner.Sequence</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a52a5ba66770bdae96fe3d634c5749281">umms.core.annotation.Gene</a>
</li>
<li>getSequence1()
: <a class="el" href="classjaligner_1_1_alignment.html#a846af4ea8699097936db4fc7282ea727">jaligner.Alignment</a>
</li>
<li>getSequence2()
: <a class="el" href="classjaligner_1_1_alignment.html#af4b53d82f275712ec1b8e090f04a27f2">jaligner.Alignment</a>
</li>
<li>getSequenceAnnotationMap()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#a97f1a2884828d396023843755147022f">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
</li>
<li>getSequenceBases()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a416d299b6cd3017e291847079c55380f">broad.core.sequence.Sequence</a>
</li>
<li>getSequenceBuilder()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a7a3e2c4c228ea4ad140b89c9ac6baca3">broad.core.sequence.Sequence</a>
</li>
<li>getSequenceDir()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_directory_installed_genome_assembly.html#aed195a28d47db8336eedf61e0426b6cc">broad.pda.feature.genome.DirectoryInstalledGenomeAssembly</a>
</li>
<li>getSequenceForGene()
: <a class="el" href="classbroad_1_1pda_1_1rnai_1_1_extract_sequence.html#a59fc486013a478291335961ccf47b5dd">broad.pda.rnai.ExtractSequence</a>
</li>
<li>getSequenceName()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_g_f_f.html#a13830d58f52bb6a17c2232b737ea0baf">broad.core.annotation.GFF</a>
</li>
<li>getSequenceNames()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_fasta_sequence_i_o.html#add110f97ffec320326f68e49728a41a3">broad.core.sequence.FastaSequenceIO</a>
, <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_s_a_m_paired_end_file_reader.html#a1412dc9e8408dcbc581ca071488e53b5">broad.pda.samtools.SAMPairedEndFileReader</a>
, <a class="el" href="classumms_1_1core_1_1readers_1_1_paired_end_reader.html#a3126e03ea603485ecd7eb0f8dc3381a1">umms.core.readers.PairedEndReader</a>
</li>
<li>getSequenceObject()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a8a83f1fd0ee98494e18c36476155cec0">umms.core.annotation.Gene</a>
</li>
<li>getSequences()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_parser.html#ab6f48cad18b6a00b8be620714d07968b">broad.pda.seq.fastq.FastqParser</a>
</li>
<li>getSequenceUnoriented()
: <a class="el" href="classbroad_1_1pda_1_1rnai_1_1_extract_sequence.html#a28a4bd210bd1c447a44144c25ad2700a">broad.pda.rnai.ExtractSequence</a>
</li>
<li>getSetIsoforms()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#a93df766c9727f3ed89e79c321cef078a">broad.pda.annotation.Locus</a>
</li>
<li>getShortArm()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_chromosome.html#a520a5c2b9095f3aa4079c6992bce443c">broad.pda.feature.genome.Chromosome</a>
</li>
<li>getShuffledPositions()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup.html#a9da41d6224e8d98d57989bf58c16190a">broad.pda.seq.clip.Pileup</a>
</li>
<li>getShuffledPositionsAllRegions()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup.html#a3192137e746edd357a296f5d823f8e2f">broad.pda.seq.clip.Pileup</a>
</li>
<li>getSignificantGeneSets()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1_pa_g_e_java.html#a6f3931ef76b19b908e94c5ae859e056e">broad.pda.geneexpression.PaGEJava</a>
</li>
<li>getSilhouette()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_kmeans.html#ad1f0deb93ba760134439cc50caadee96">broad.pda.geneexpression.clustering.Kmeans</a>
</li>
<li>getSimilarity()
: <a class="el" href="classjaligner_1_1_alignment.html#ac7ffc2c5cc00467b766e2a9447420aaa">jaligner.Alignment</a>
</li>
<li>getSingleReadsSkipped()
: <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_paired_end_read_iterator.html#afa19ac960a85e684e8f5be020a5cf479">broad.pda.samtools.PairedEndReadIterator</a>
</li>
<li>getSingleSampleScanPeaks()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#a5f3dacfe2ed0ece8473039132fc28c64">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>getSingularValues()
: <a class="el" href="class_jama_1_1_singular_value_decomposition.html#a787358e07849a230154d02ed5009a9a1">Jama.SingularValueDecomposition</a>
</li>
<li>getSinkVertices()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a8e54ddc31c8dff5537cd771bbb0a10e7">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
, <a class="el" href="classumms_1_1core_1_1scripture_1_1_oriented_chromosome_transcript_graph.html#ade41013e156feeae401801be1ab08c03">umms.core.scripture.OrientedChromosomeTranscriptGraph</a>
</li>
<li>getSites()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1meme_1_1_meme_motif.html#a673b187a0caaae88d18e60e7ccd6c04d">broad.core.motif.meme.MemeMotif</a>
</li>
<li>getSize()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a2688217ce17cf4cdfc30c746380db0f8">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#aab1fa671b2c90cac37a657f5ead8e00a">broad.pda.seq.graph.Path</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation.html#a502d642ef35b9ef95d88156fe0c0114d">umms.core.annotation.AbstractAnnotation</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#a19aad320398985fa78572524f5dc4f88">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a9e791756126fb65d791fe6c3620535b8">umms.core.annotation.Gene</a>
, <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html#ab4c7b2a5fb8675ccf6e43e4fa7e0a7a7">umms.core.coordinatesystem.CoordinateSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_genomic_space.html#a426572ebde2b646b760564f610a8e618">umms.core.coordinatesystem.GenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_masked_genomic_space.html#a021ea20a50d2940b947c69a80ff0103b">umms.core.coordinatesystem.MaskedGenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcript_in_genomic_space.html#a1c62e35a3558e8f8bb40674d084b6143">umms.core.coordinatesystem.TranscriptInGenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcriptome_space.html#a0507bd6bd556e74bb734790344b6531e">umms.core.coordinatesystem.TranscriptomeSpace</a>
, <a class="el" href="classumms_1_1core_1_1feature_1_1_gene_window.html#a62c44cfc68a5776841b9c1405720051b">umms.core.feature.GeneWindow</a>
</li>
<li>getSizesFromSAM()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils.html#aaaae3d6e8f7f451a772f432a8414f4a2">broad.pda.seq.alignment.AlignmentUtils</a>
</li>
<li>getSkellamPvalue()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_two_sample_scan_skellam_peak_caller.html#afcdd15bdc7980bb781b093eae3951104">broad.pda.seq.protection.TwoSampleScanSkellamPeakCaller</a>
</li>
<li>getSkellamPValue()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_skellam_score.html#ac5d4b9bc60ea6e9598830f76d210bcd3">broad.pda.seq.rap.SkellamScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#ad917b97ffdb8a2f02f6122616c0de469">umms.core.model.score.WindowAllScore</a>
</li>
<li>getSlider()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#aed240f0dc08cc05ee521e34b48f04063">broad.core.sequence.Sequence</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a05fc6d4159079357daf6d1fb510ef3b7">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_window_slider.html#ab81009a0ac70a08af46a0ed2f2376da1">broad.core.sequence.WindowSlider</a>
</li>
<li>getSoftmaskedRegions()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a61503379b2f74c904e0a6d26f8829086">broad.core.sequence.Sequence</a>
</li>
<li>getSortedAndUniqueExons()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#afa5372b27652f6e5788049f428e6a7a8">umms.core.annotation.Gene</a>
</li>
<li>getSortedNodes()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#ae5c86ade2b8e8d09a6ed6bbe1a4791f0">broad.pda.seq.graph.Path</a>
</li>
<li>getSource()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_g_f_f.html#aa65cbde0f9e887df1a5e19199693c531">broad.core.annotation.GFF</a>
</li>
<li>getSourceAnnotations()
: <a class="el" href="classumms_1_1core_1_1feature_1_1_gene_window.html#a5cd5f91c2e7e2eaf0a4783d5a3746a01">umms.core.feature.GeneWindow</a>
, <a class="el" href="classumms_1_1core_1_1feature_1_1_genome_window.html#ae2b7673478fa32fd3a28611f0e384b8e">umms.core.feature.GenomeWindow</a>
, <a class="el" href="interfaceumms_1_1core_1_1feature_1_1_window.html#a1de74ce8b62b8f273dc20df20fd97f39">umms.core.feature.Window</a>
</li>
<li>getSourceVertices()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#ad770bae6ab4a2477e609fa450d5a9bdf">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
, <a class="el" href="classumms_1_1core_1_1scripture_1_1_oriented_chromosome_transcript_graph.html#a2d2b4ae004f53c19e4890cb3351be976">umms.core.scripture.OrientedChromosomeTranscriptGraph</a>
</li>
<li>getSpan()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_compound_interval.html#a8691646ad235799c26f89c2c9d5b964e">umms.core.annotation.CompoundInterval</a>
</li>
<li>getSpikeInDetectionLimit()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_array_stats.html#a95550a628a60efc460285b6d03be14e5">broad.pda.geneexpression.agilent.AgilentArrayStats</a>
</li>
<li>getSpliceConnections()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#ade05219b0093af953ff33e9a268b3224">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#a559c79b4aa45eeec28c95d5336dc096e">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#af9a4e3fc1eceb88258782e785dcc3d92">umms.core.alignment.SingleEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#a87f320553683b507e84278eb03ccacdb">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a8de58e046b212813b450d383e0f6c8e2">umms.core.annotation.BasicAnnotation</a>
</li>
<li>getSplicedCount()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a45b55e717dcb570d8a3853360c92f272">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#af17880a4e617b7603bca490f64fa4cce">umms.core.model.JCSAlignmentModel</a>
</li>
<li>getSplicedReadExons()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a5b3f5ae924195f85e3867fafcf12925a">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a8487a4859373b0233fddf580f65af3d8">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getSplicedReads()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a6e26fa1ddc97bcbca18b0c44313cd6a5">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a15b1d3b05b1003be0295c3245a4e131b">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#aecacb369e6f69e1c166ced2b45cc80ab">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getSpliceJunctionCoordinates()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#ad68c6ca26f7595432240b8b77b88253f">umms.core.annotation.Gene</a>
</li>
<li>getSpliceJunctionCoordinatesTree()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a86b6200a444bfcd32520596ae38b1db7">umms.core.annotation.Gene</a>
</li>
<li>getSpliceWeight()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a3e8f397fe2620928de69b3165ecc1fde">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#a92238c19edc6d4e647a0567e0619050c">broad.pda.seq.graph.Path</a>
</li>
<li>getSpliceWeightFactor()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a06594535ec2d118a84a9a10a0755d7fa">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a9b4d4c4860a95ac5d583333475b845ea">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#afbcb83cbc75cfd98c8d5a0d444c9d299">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getStandardDeviation()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#a0463b946ed45a8c92511ce684680e9bd">broad.core.math.EmpiricalDistribution</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_abstract_list_score.html#a29d42985031bef2fd4cc340eeea8fd55">umms.core.model.score.AbstractListScore</a>
</li>
<li>getStart()
: <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation.html#aa09c2220d181284077c5c020f2721b63">broad.core.annotation.LightweightGenomicAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1datastructures_1_1_interval.html#a3ae69808c01e3b50765b22724f18fe14">broad.core.datastructures.Interval</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#afcca12aead494125f57be12abe9e2adb">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a64465fbdfb57e6549113e47877c9b37a">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#ac65fac25077ec4dba79daa2fc9ad5682">broad.pda.seq.graph.Path</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a99f17f8d510ad2798e2c0b5b886df2bc">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#a62c8d52837fefe4dec37dc0a550751b9">umms.core.alignment.Alignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#a4fc0345d5a95dfb0bfd018cc0d9d400f">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a2f52387ec04cec413c4fea644ad6dd9f">umms.core.annotation.BasicAnnotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_compound_interval.html#a01a3c5e0cfce5f5a283c9ff4a7225490">umms.core.annotation.CompoundInterval</a>
</li>
<li>getStart1()
: <a class="el" href="classjaligner_1_1_alignment.html#acea04b1b3442fbbe9e18c1700dc97757">jaligner.Alignment</a>
</li>
<li>getStart2()
: <a class="el" href="classjaligner_1_1_alignment.html#a74e0e6fb22bf32202ef16001ce2ba9e8">jaligner.Alignment</a>
</li>
<li>getStartCodon()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a66af3f09f9f1adfc842397613d1e5ff7">umms.core.annotation.Gene</a>
</li>
<li>getStartVertex()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#a671e412fbad1e7845efc842e3c67e5c4">broad.pda.seq.graph.Path</a>
</li>
<li>getState()
: <a class="el" href="classbroad_1_1core_1_1hmm_1_1_markov_model_3_01_t_01_4.html#a3965e6fcde1614cb752ea55f1ed345b1">broad.core.hmm.MarkovModel&lt; T &gt;</a>
</li>
<li>getStates()
: <a class="el" href="classbroad_1_1core_1_1hmm_1_1_markov_model_3_01_t_01_4.html#a8308c5b89e679a387d92b5343a13af95">broad.core.hmm.MarkovModel&lt; T &gt;</a>
</li>
<li>getStateTransitionMatrix()
: <a class="el" href="classbroad_1_1core_1_1hmm_1_1_markov_model_3_01_t_01_4.html#a955cdf8737110eaf811de2a511bc4f30">broad.core.hmm.MarkovModel&lt; T &gt;</a>
</li>
<li>getStatistic()
: <a class="el" href="classbroad_1_1pda_1_1enrichment_1_1_enrichment_metric.html#a586064a2f01aadc6436173ac6ae8cb4f">broad.pda.enrichment.EnrichmentMetric</a>
</li>
<li>getStepSize()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a225b80f975a281ea6c4d99482a20ba0e">broad.pda.seq.protection.SampleData</a>
</li>
<li>getStrand()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_g_f_f.html#a55a7daa043957ff87c4fd2d93c5b7447">broad.core.annotation.GFF</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a59d6c478e423d1b674f4a650409cb4c1">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation.html#a3a129fd352007aff4aa44b2a785f6477">umms.core.annotation.AbstractAnnotation</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#ae43b57d64e5ca6c58123c695fa31bf34">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#ac53691d7b9efa13bc4ea3350d0ee83d3">umms.core.model.AlignmentModel</a>
</li>
<li>getString()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a8f04b65c662ee6fd8903543f0e621076">umms.core.annotation.Gene</a>
</li>
<li>getStringArg()
: <a class="el" href="classbroad_1_1core_1_1parser_1_1_command_line_parser.html#ab453d8b59548ccda31289abff8f44b37">broad.core.parser.CommandLineParser</a>
</li>
<li>getStringArray()
: <a class="el" href="classbroad_1_1core_1_1parser_1_1_string_parser.html#ac388a4e2475ca305f1cd9ca4600e4d50">broad.core.parser.StringParser</a>
</li>
<li>getStringListArg()
: <a class="el" href="classbroad_1_1core_1_1parser_1_1_command_line_parser.html#a6906c17b029cb5293d812b520065163a">broad.core.parser.CommandLineParser</a>
</li>
<li>getSubAlignment()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_alignment.html#a4bec03246b949c3d8a49cedfa062eaf0">broad.core.multiplealignment.MAFAlignment</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#ad8b69760ca48a1009539fd5fbf7c8c48">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>getSubAnnotationFromEnd()
: <a class="el" href="classumms_1_1esat_1_1_e_s_a_t.html#aeb4346e593752ad30a10e4c59026e85d">umms.esat.ESAT</a>
, <a class="el" href="classumms_1_1esat_1_1_e_s_a_t_distance_calculator.html#a8991dbfa8390362b7401aa10aa78a753">umms.esat.ESATDistanceCalculator</a>
</li>
<li>getSubAnnotationFromStart()
: <a class="el" href="classumms_1_1esat_1_1_e_s_a_t.html#a46150c22de56265a4175d196b4f50ae2">umms.esat.ESAT</a>
, <a class="el" href="classumms_1_1esat_1_1_e_s_a_t_distance_calculator.html#aa651a5a0773a942db8851c56201b1316">umms.esat.ESATDistanceCalculator</a>
</li>
<li>getSubAnnotationMidpointWithinAnnotation()
: <a class="el" href="classumms_1_1core_1_1utils_1_1_annotation_utils.html#aad340b093e32c429083e65070ee4a3ed">umms.core.utils.AnnotationUtils</a>
</li>
<li>getSubclusters()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_cluster.html#ad35ffe1b453cfec9527792ffd369f7f7">broad.pda.geneexpression.clustering.Cluster</a>
</li>
<li>getSubmethod()
: <a class="el" href="classbroad_1_1pda_1_1enrichment_1_1_enrichment_metric.html#a09fba0011238304a664db239acf77c95">broad.pda.enrichment.EnrichmentMetric</a>
</li>
<li>getSubSequence()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a1917327054e5ef6c6d5d3ef15c203f03">broad.core.sequence.Sequence</a>
</li>
<li>getSubsequence()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#ac6dee6a60810f7af86ac448ac18b721b">broad.core.sequence.Sequence</a>
</li>
<li>getSubSequence()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#ab03732869550e07296a989208b50c100">broad.core.sequence.Sequence</a>
</li>
<li>getSubsequence()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#aea523aacfd70e1301e61ce005db9fd68">broad.core.sequence.Sequence</a>
</li>
<li>getSubSequence()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a5af216e57b3a21de28ff79b81a51af25">broad.core.sequence.Sequence</a>
</li>
<li>getSum()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a804caa716e6be5b84f785eb80e90e5a9">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a68033365b74f1d7755eb6e712f47f863">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_abstract_list_score.html#a8427958d055d5c9504f5fa1c47f0401e">umms.core.model.score.AbstractListScore</a>
</li>
<li>getSummary()
: <a class="el" href="classjaligner_1_1_alignment.html#a3d250e0dcf2923a86b99818754e81e98">jaligner.Alignment</a>
</li>
<li>getSymbol()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_chromosome.html#a8600a6e4cefcd5e4d70a1ea20987d370">broad.pda.feature.genome.Chromosome</a>
</li>
<li>getTermGenes()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1functional_annotation.html#ac25fe9ebfabb92cd739c154cdc55d2e2">broad.pda.gene.functionalAnnotation</a>
</li>
<li>getTestStatistic()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_fisher_combined_probability_test.html#a7cee9fab03e9e054960df0e8fb4e500c">broad.core.math.FisherCombinedProbabilityTest</a>
</li>
<li>getTestStatisticMatrix()
: <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_differential_expression.html#aa9eba8e67806696f4ec056fd90158bce">broad.pda.differentialExpression.DifferentialExpression</a>
</li>
<li>getThickEnd()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_b_e_d.html#a536117fb2a3bfc5dd57afd96c783e04d">broad.core.annotation.BED</a>
</li>
<li>getThickStart()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_b_e_d.html#a1d5ccf3b546f5dd4fdc96c31fb0e599c">broad.core.annotation.BED</a>
</li>
<li>getThreePrimeBases()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_genomic_annotation.html#a0fb40ec2fb50f82f09ce0c80049a0e25">broad.core.annotation.BasicGenomicAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_genomic_annotation.html#af6fe274a67896a3b31bbc7959d1fe185">broad.core.annotation.GenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#af89ecd9fdc8acb1252e1366c4297d550">broad.core.sequence.SequenceRegion</a>
</li>
<li>getTimePoint()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1_expression_experiment_info.html#a43d0cde2352a1bf06b6b9de5c54c769e">broad.pda.geneexpression.ExpressionExperimentInfo</a>
</li>
<li>getTokens()
: <a class="el" href="classbroad_1_1core_1_1parser_1_1_string_parser.html#a3a99eaa42daf066177fc98c59b6200eb">broad.core.parser.StringParser</a>
</li>
<li>getTotal()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_combination_generator.html#a5c3b3f2855ca7ed1ceb85489d29d1ddb">broad.core.math.CombinationGenerator</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_count_score.html#a4668d5a19912f7f9bf52c6c3b8fe2b2c">umms.core.model.score.CountScore</a>
</li>
<li>getTotalCoverage()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#a6e0d575c3f9629a083419111507dc33a">umms.core.annotation.AbstractAnnotationCollection&lt; T extends Annotation &gt;</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#a643ab1265b4dcd6547c36401a116dd50">umms.core.annotation.AnnotationCollection&lt; T extends Annotation &gt;</a>
</li>
<li>getTotalDistanceFromNode()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_tree_scaler.html#add956377753d6a650493d0fae59994b0">broad.core.siphy.TreeScaler</a>
</li>
<li>getTotalInSources()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_tree_scaler_i_o.html#a166843bedad99e2fba9d6390a1abd79a">broad.core.siphy.TreeScalerIO</a>
</li>
<li>getTotalNumberOffeatures()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_array_stats.html#a53f31665c19fa87760eb99bc1339dd81">broad.pda.geneexpression.agilent.AgilentArrayStats</a>
</li>
<li>getTotalNumberOfMappedReads()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a64c03e96c06f7fbf55f48868b5bbf9de">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getTotalNumberOfMappedStrandedReads()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a19917c689438c87d9e198f03beaf3da4">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getTotalNumberOfReads()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a96786fd0c7c2c14c2d305af98a1bbb8f">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getTotalNumberOfStrandedReads()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a8735fdbc1316cf8f2612b1ee23242ed6">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a283db1d20013b9089f0a56cc642b7a10">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>getTotalObservations()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#afce47669b093234f08a4d04a527cc488">broad.core.math.EmpiricalDistribution</a>
</li>
<li>getTotalReads()
: <a class="el" href="classbroad_1_1pda_1_1countreads_1_1_fastq_library_stats.html#a927c0809077c9d8be5623e836a9f0179">broad.pda.countreads.FastqLibraryStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a61ea07f3dae047b5a31ec4f70414aae2">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
</li>
<li>getTrackInfo()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#ad3fef635ef16bb2001844d16ae4155f8">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
</li>
<li>getTranscribedBase()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_transcribed_regions.html#ad930ec63439d92b88207511295100f82">broad.core.sequence.TranscribedRegions</a>
</li>
<li>getTranscribedBaseAtBeginningPosition()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_transcribed_sequence_composition.html#a1ac885628c31fd901c8dbe3494de0e55">broad.core.sequence.TranscribedSequenceComposition</a>
</li>
<li>getTranscriptLength()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#ad286d775edabc4009f3590f93acd8e89">umms.core.annotation.Gene</a>
</li>
<li>getTranscriptName()
: <a class="el" href="classbroad_1_1pda_1_1rnai_1_1_r_n_ai_gene_annotation.html#a6a3926492ca39c3fe5e12a1857766642">broad.pda.rnai.RNAiGeneAnnotation</a>
</li>
<li>getTranscriptomeSpace()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1rna_1_1_transcriptome_command_line_program.html#aa322d62d8f06c7b19b6b3844f71b795b">broad.pda.seq.rap.rna.TranscriptomeCommandLineProgram</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcriptome_space.html#a009b95c213608b804918ed940e13ce2a">umms.core.coordinatesystem.TranscriptomeSpace</a>
</li>
<li>getTranscripts()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_hairpin_kmer.html#a9565c3976f7cb63bb06b2ff488f60e29">broad.core.primer3.HairpinKmer</a>
</li>
<li>getTranscriptSequence()
: <a class="el" href="classbroad_1_1pda_1_1rnai_1_1_r_n_ai_gene_annotation.html#a448603290312af2544d88d28fa60a7fe">broad.pda.rnai.RNAiGeneAnnotation</a>
</li>
<li>getTree()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model.html#ae37b5f8381618f817dfabd2370a7241f">broad.core.siphy.EvolutionaryModel</a>
, <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model_parameters.html#ae59fa06d452658b1914c03f7b5f4c6d7">broad.core.siphy.EvolutionaryModelParameters</a>
</li>
<li>getType()
: <a class="el" href="classjaligner_1_1_sequence.html#a4833766cfd049625955716ddc9aba95e">jaligner.Sequence</a>
</li>
<li>getU()
: <a class="el" href="class_jama_1_1_l_u_decomposition.html#a1e7898faf116eb7997e75eefa00f2d01">Jama.LUDecomposition</a>
, <a class="el" href="class_jama_1_1_singular_value_decomposition.html#aa5f3a3541dc98ad8b9abb4b3ba07f0fc">Jama.SingularValueDecomposition</a>
</li>
<li>getUn()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_directory_installed_genome_assembly.html#a604c602d0427720cdf86a6fd7a885743">broad.pda.feature.genome.DirectoryInstalledGenomeAssembly</a>
</li>
<li>getUngappedReferenceIslands()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#a565a11481850134152042482fb856421">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>getUngappedSequenceReferenceIslands()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#aee4c7a91d6db5c3983b862dad485adb5">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>getUngappedSize()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_chromosome.html#a226357672815a362698eee421cf21a2e">broad.pda.feature.genome.Chromosome</a>
</li>
<li>getUniqueNeighbors()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_neighbors_null_model.html#af5ac20d724aa6a95a38c47ffd71de94c">broad.pda.gene.NeighborsNullModel</a>
</li>
<li>getUniqueRepeatClasses()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_repeat_masker_annotation_reader.html#af6b0938d08085b5f8e915a8bab93f8e2">broad.core.annotation.RepeatMaskerAnnotationReader</a>
</li>
<li>getUniqueRepeatFamilies()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_repeat_masker_annotation_reader.html#aa575a5e82057feaaf50e66bf628028d8">broad.core.annotation.RepeatMaskerAnnotationReader</a>
</li>
<li>getUniqueRepeatNames()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_repeat_masker_annotation_reader.html#a82a9c1f12e02483e928455d343134e1a">broad.core.annotation.RepeatMaskerAnnotationReader</a>
</li>
<li>getUnmaskedLength()
: <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_genomic_space.html#a8f86806abc96d49a209969abee97be5e">umms.core.coordinatesystem.GenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcript_in_genomic_space.html#a9c7d62e75789b55cd3aef2087b832565">umms.core.coordinatesystem.TranscriptInGenomicSpace</a>
</li>
<li>getUnqComaptibleBetweenAny2Sets()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#a9505fde35b8cb334a7103cf2611ef899">broad.pda.annotation.Locus</a>
</li>
<li>getUnqComaptibleBetweenSubsets()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#a333584ce51e5c6f1a2f00c2ad0586fea">broad.pda.annotation.Locus</a>
</li>
<li>getUserDirectory()
: <a class="el" href="classjaligner_1_1util_1_1_commons.html#a42fc068e5fc2e61ed19cb267354bcc24">jaligner.util.Commons</a>
</li>
<li>getUtr3Counts()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_counts.html#a0d8da8b72e3ed8f24eea4ed967e5561a">broad.pda.seq.segmentation.GeneCounts</a>
</li>
<li>getUtr5Counts()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_counts.html#a0609588c72e5a46740859190a61abdd6">broad.pda.seq.segmentation.GeneCounts</a>
</li>
<li>getV()
: <a class="el" href="class_jama_1_1_eigenvalue_decomposition.html#a16cfd86fa74288c8d27536460d70df7b">Jama.EigenvalueDecomposition</a>
, <a class="el" href="class_jama_1_1_singular_value_decomposition.html#ad0e8cf5267ef9a61fa97324fc8649d8e">Jama.SingularValueDecomposition</a>
</li>
<li>getValidOutgoingEdges()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#af2b3ad769c7da3c808548c2099d8e3a1">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>getValue1()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_pair_3_01_t1_01_4.html#a8c03da02a99a2c76396fda801ceac218">broad.core.datastructures.Pair&lt; T1 &gt;</a>
</li>
<li>getValue2()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_pair_3_01_t1_01_4.html#a4c03600e24ad8920674cb2edf989400e">broad.core.datastructures.Pair&lt; T1 &gt;</a>
</li>
<li>getValues()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_g_f_f.html#a7c572e27e1de2c50c6dfefca340c39ea">broad.core.annotation.GFF</a>
, <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a5050674b9d1bbbd161a1bf876249b1f7">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>getVectorEncodedSequence()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a5127ade0f481eedbb87cdd6fde86f293">broad.core.sequence.Sequence</a>
</li>
<li>getWeight()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_column.html#a11cdf460bc71b967f82f8beff5425ee1">broad.core.motif.PositionWeightColumn</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#a204f471b1073913ec667dab2860cdd72">broad.pda.seq.graph.Path</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a21d5bb2538d3f702949c8cb54ac05d72">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#aaf11809a0fd53aa8ad485d6ab993d7c0">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#adc1158b384c343f66a3cccfee951b293">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a2deb3e94dfddeef2103d245c47686ee1">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>getWidth()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1meme_1_1_meme_motif.html#a172b6ebd50e0a9b55be788f21f006cd4">broad.core.motif.meme.MemeMotif</a>
</li>
<li>getWidths()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a87118dfc0d49b68daca8cd351ae3bcab">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>getWindowCount()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#afd2283807ea84e0cc2da07c108f6ca98">broad.pda.seq.protection.SampleData</a>
</li>
<li>getWindowIterator()
: <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html#a251edfa76046cde3077b86a670ac6945">umms.core.coordinatesystem.CoordinateSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_genomic_space.html#a92f232940d1bd938e6ead9c8780e8b17">umms.core.coordinatesystem.GenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_masked_genomic_space.html#ac115e2a6661d72819e378be1204833ec">umms.core.coordinatesystem.MaskedGenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcript_in_genomic_space.html#a8d2e25ee2f28cf4570798be66474b8d8">umms.core.coordinatesystem.TranscriptInGenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcriptome_space.html#abe2bf4806426c7074fb585f85d0632db">umms.core.coordinatesystem.TranscriptomeSpace</a>
</li>
<li>getWindowLength()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_basic_scan_statistic_score.html#a50cc80126179865c6817917bc58c3f43">umms.core.model.score.BasicScanStatisticScore</a>
</li>
<li>getWindowProcessor()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_genome_scoring_program.html#aeee35502ac7d566b7a32f2564fbf1a9b">broad.pda.seq.rap.GenomeScoringProgram</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1rna_1_1_transcriptome_scoring_program.html#a696ae45b0c3c70b0a160c8c3d8913f2f">broad.pda.seq.rap.rna.TranscriptomeScoringProgram</a>
</li>
<li>getWindows()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a296bd8542ee7397933d6dc58117f2bc0">umms.core.annotation.Gene</a>
, <a class="el" href="classumms_1_1core_1_1feature_1_1_gene_window.html#af40fab362891f30799d0680beea273c1">umms.core.feature.GeneWindow</a>
, <a class="el" href="classumms_1_1core_1_1feature_1_1_genome_window.html#aa03e0fb867f659eba10ca1144469063d">umms.core.feature.GenomeWindow</a>
, <a class="el" href="interfaceumms_1_1core_1_1feature_1_1_window.html#a8294e5f6fb17bf1f3354e543231ae5ec">umms.core.feature.Window</a>
</li>
<li>getWindowScoreIterator()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_slide_and_calculate.html#a34ca6d1e9be4800f5af03612256c29b8">broad.pda.seq.rap.SlideAndCalculate</a>
</li>
<li>getWindowScores()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a1adf639ddb5cd1b53ab3ff48f9f74ea9">broad.pda.seq.protection.SampleData</a>
</li>
<li>getWindowSize()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#ab775331fb5bee8667165c78ca5a105ba">broad.pda.seq.protection.SampleData</a>
</li>
<li>getX()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_directory_installed_genome_assembly.html#a3d2f23d85c86e9f373043c9bcd17422c">broad.pda.feature.genome.DirectoryInstalledGenomeAssembly</a>
</li>
<li>getZ()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_mann_whitney.html#a867f8251b41b686641161b5c8311eca4">broad.core.math.MannWhitney</a>
, <a class="el" href="classbroad_1_1core_1_1math_1_1_wilcox2_sample.html#a2fea6398accca8947137d78fd31761a7">broad.core.math.Wilcox2Sample</a>
</li>
<li>getZscore()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#a2ae5c95e4aca00cffcc7fae6031a165b">broad.core.math.EmpiricalDistribution</a>
</li>
<li>GFF()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_g_f_f.html#ab077cc4a68225601e6d411f49e76e434">broad.core.annotation.GFF</a>
</li>
<li>gffFactory
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_factory_factory.html#a1ca1d0758e6d3167487ecaa594aa46eb">broad.core.annotation.AnnotationFactoryFactory</a>
</li>
<li>GFFReader()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_g_f_f_reader.html#ac5e40ca3193905823e32a67a1c8dcb46">broad.core.annotation.GFFReader</a>
</li>
<li>GlobalSpliceFilter()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_global_splice_filter.html#a1b27b404efa3b13591e3bc5c801b5865">broad.pda.seq.segmentation.GlobalSpliceFilter</a>
</li>
<li>GlobalStats()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1_global_stats.html#ad709bbb0f08e17a6052b7f04afed0c97">broad.pda.geneexpression.GlobalStats</a>
</li>
<li>gotWindowScoresFromFile()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#af69246249c44eabd4349626e639a44c6">broad.pda.seq.protection.SampleData</a>
</li>
<li>GrabProbes()
: <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_grab_probes.html#ac08f0c2a1f9d34e317ac448659f1d07d">broad.pda.differentialExpression.GrabProbes</a>
</li>
<li>GSAProjection()
: <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_g_s_a_projection.html#a36ad523a42364f0dc316f7ecb70ea269">broad.pda.differentialExpression.GSAProjection</a>
</li>
<li>GSAScoring()
: <a class="el" href="classbroad_1_1projection_1_1gsa_1_1_g_s_a_scoring.html#a511a11c0388f2886143c2c638bcd2bf8">broad.projection.gsa.GSAScoring</a>
</li>
<li>GTFFileParser()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_g_t_f_file_parser.html#aa075326f19462c9810fcf619f87390bd">broad.pda.annotation.GTFFileParser</a>
</li>
<li>GTFWrite()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_g_t_f_file_parser.html#a6cf2e37db633eac20b4ff4eeb39f965c">broad.pda.annotation.GTFFileParser</a>
</li>
</ul>
</div><!-- contents -->
<!-- start footer part -->
<hr class="footer"/><address class="footer"><small>
Generated on Fri Aug 15 2014 15:11:16 for ESAT by &#160;<a href="http://www.doxygen.org/index.html">
<img class="footer" src="doxygen.png" alt="doxygen"/>
</a> 1.8.7
</small></address>
</body>
</html>
