<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=9"/>
<meta name="generator" content="Doxygen 1.8.7"/>
<title>ESAT: Class Members</title>
<link href="tabs.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="jquery.js"></script>
<script type="text/javascript" src="dynsections.js"></script>
<link href="search/search.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="search/search.js"></script>
<script type="text/javascript">
  $(document).ready(function() { searchBox.OnSelectItem(0); });
</script>
<link href="doxygen.css" rel="stylesheet" type="text/css" />
</head>
<body>
<div id="top"><!-- do not remove this div, it is closed by doxygen! -->
<div id="titlearea">
<table cellspacing="0" cellpadding="0">
 <tbody>
 <tr style="height: 56px;">
  <td style="padding-left: 0.5em;">
   <div id="projectname">ESAT
   </div>
  </td>
 </tr>
 </tbody>
</table>
</div>
<!-- end header part -->
<!-- Generated by Doxygen 1.8.7 -->
<script type="text/javascript">
var searchBox = new SearchBox("searchBox", "search",false,'Search');
</script>
  <div id="navrow1" class="tabs">
    <ul class="tablist">
      <li><a href="index.html"><span>Main&#160;Page</span></a></li>
      <li><a href="pages.html"><span>Related&#160;Pages</span></a></li>
      <li><a href="namespaces.html"><span>Packages</span></a></li>
      <li class="current"><a href="annotated.html"><span>Classes</span></a></li>
      <li><a href="files.html"><span>Files</span></a></li>
      <li>
        <div id="MSearchBox" class="MSearchBoxInactive">
        <span class="left">
          <img id="MSearchSelect" src="search/mag_sel.png"
               onmouseover="return searchBox.OnSearchSelectShow()"
               onmouseout="return searchBox.OnSearchSelectHide()"
               alt=""/>
          <input type="text" id="MSearchField" value="Search" accesskey="S"
               onfocus="searchBox.OnSearchFieldFocus(true)" 
               onblur="searchBox.OnSearchFieldFocus(false)" 
               onkeyup="searchBox.OnSearchFieldChange(event)"/>
          </span><span class="right">
            <a id="MSearchClose" href="javascript:searchBox.CloseResultsWindow()"><img id="MSearchCloseImg" border="0" src="search/close.png" alt=""/></a>
          </span>
        </div>
      </li>
    </ul>
  </div>
  <div id="navrow2" class="tabs2">
    <ul class="tablist">
      <li><a href="annotated.html"><span>Class&#160;List</span></a></li>
      <li><a href="classes.html"><span>Class&#160;Index</span></a></li>
      <li><a href="inherits.html"><span>Class&#160;Hierarchy</span></a></li>
      <li class="current"><a href="functions.html"><span>Class&#160;Members</span></a></li>
    </ul>
  </div>
  <div id="navrow3" class="tabs2">
    <ul class="tablist">
      <li class="current"><a href="functions.html"><span>All</span></a></li>
      <li><a href="functions_func.html"><span>Functions</span></a></li>
      <li><a href="functions_vars.html"><span>Variables</span></a></li>
      <li><a href="functions_eval.html"><span>Enumerator</span></a></li>
    </ul>
  </div>
  <div id="navrow4" class="tabs3">
    <ul class="tablist">
      <li><a href="functions.html#index_a"><span>a</span></a></li>
      <li><a href="functions_b.html#index_b"><span>b</span></a></li>
      <li class="current"><a href="functions_c.html#index_c"><span>c</span></a></li>
      <li><a href="functions_d.html#index_d"><span>d</span></a></li>
      <li><a href="functions_e.html#index_e"><span>e</span></a></li>
      <li><a href="functions_f.html#index_f"><span>f</span></a></li>
      <li><a href="functions_g.html#index_g"><span>g</span></a></li>
      <li><a href="functions_h.html#index_h"><span>h</span></a></li>
      <li><a href="functions_i.html#index_i"><span>i</span></a></li>
      <li><a href="functions_j.html#index_j"><span>j</span></a></li>
      <li><a href="functions_k.html#index_k"><span>k</span></a></li>
      <li><a href="functions_l.html#index_l"><span>l</span></a></li>
      <li><a href="functions_m.html#index_m"><span>m</span></a></li>
      <li><a href="functions_n.html#index_n"><span>n</span></a></li>
      <li><a href="functions_o.html#index_o"><span>o</span></a></li>
      <li><a href="functions_p.html#index_p"><span>p</span></a></li>
      <li><a href="functions_q.html#index_q"><span>q</span></a></li>
      <li><a href="functions_r.html#index_r"><span>r</span></a></li>
      <li><a href="functions_s.html#index_s"><span>s</span></a></li>
      <li><a href="functions_t.html#index_t"><span>t</span></a></li>
      <li><a href="functions_u.html#index_u"><span>u</span></a></li>
      <li><a href="functions_v.html#index_v"><span>v</span></a></li>
      <li><a href="functions_w.html#index_w"><span>w</span></a></li>
      <li><a href="functions_z.html#index_z"><span>z</span></a></li>
    </ul>
  </div>
</div><!-- top -->
<!-- window showing the filter options -->
<div id="MSearchSelectWindow"
     onmouseover="return searchBox.OnSearchSelectShow()"
     onmouseout="return searchBox.OnSearchSelectHide()"
     onkeydown="return searchBox.OnSearchSelectKey(event)">
<a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(0)"><span class="SelectionMark">&#160;</span>All</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(1)"><span class="SelectionMark">&#160;</span>Classes</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(2)"><span class="SelectionMark">&#160;</span>Namespaces</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(3)"><span class="SelectionMark">&#160;</span>Files</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(4)"><span class="SelectionMark">&#160;</span>Functions</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(5)"><span class="SelectionMark">&#160;</span>Variables</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(6)"><span class="SelectionMark">&#160;</span>Enumerator</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(7)"><span class="SelectionMark">&#160;</span>Pages</a></div>

<!-- iframe showing the search results (closed by default) -->
<div id="MSearchResultsWindow">
<iframe src="javascript:void(0)" frameborder="0" 
        name="MSearchResults" id="MSearchResults">
</iframe>
</div>

<div class="contents">
<div class="textblock">Here is a list of all class members with links to the classes they belong to:</div>

<h3><a class="anchor" id="index_c"></a>- c -</h3><ul>
<li>C_CODE
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#a2e2aba2b96a17a177bd95d466059031d">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>calcAllTermsEnrichment()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1functional_annotation.html#a5c2ba1d1acf76ae8f19dbbf79c775984">broad.pda.gene.functionalAnnotation</a>
</li>
<li>calcLightSilhouette()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_kmeans.html#acd4f82d43bf7e3096bbf31c13ffdc2ec">broad.pda.geneexpression.clustering.Kmeans</a>
</li>
<li>calcSilhouette()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_kmeans.html#a8e87cbb019573c9672027b8eb220e7c4">broad.pda.geneexpression.clustering.Kmeans</a>
</li>
<li>calcTermEnrichment()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1functional_annotation.html#aa3f877452e95fde1ff2eb5520e318856">broad.pda.gene.functionalAnnotation</a>
</li>
<li>calculateApproximatePVal()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_scan_statistics.html#a0bcb18905e2ec14223a7038d0293aae0">broad.core.math.ScanStatistics</a>
</li>
<li>calculateBamEnrichments()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_collect_annotation_enrichments.html#a526e950c4fd241ba6a4850d88a5931aa">broad.pda.seq.rap.CollectAnnotationEnrichments</a>
</li>
<li>calculateDiscreteAnnotationEnrichments()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_collect_annotation_enrichments.html#a21202bc9f043306a502cbe9c22299225">broad.pda.seq.rap.CollectAnnotationEnrichments</a>
</li>
<li>calculateGc()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_g_c_content.html#a2cdf64da9c909c928da3470d7bcb1ce2">broad.pda.seq.rap.GCContent</a>
</li>
<li>calculateGlobalPvalue()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_basic_scan_statistic_score.html#a4f291022ecc0497ec100456fec710ec0">umms.core.model.score.BasicScanStatisticScore</a>
</li>
<li>calculateLocalPvalue()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_basic_scan_statistic_score.html#a578eff1fcf43c1bc8d8079c7563d4e6d">umms.core.model.score.BasicScanStatisticScore</a>
</li>
<li>calculateNormalizationFactors()
: <a class="el" href="classumms_1_1esat_1_1_e_s_a_t.html#a36fc452b596f2408460fc28d50f98d4f">umms.esat.ESAT</a>
, <a class="el" href="classumms_1_1esat_1_1_e_s_a_t_distance_calculator.html#a82606cd46ae4f4570afac64befafb941">umms.esat.ESATDistanceCalculator</a>
</li>
<li>calculatePVal()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_scan_statistics.html#adfa56e760fadf1adcdf23c62062ff0e3">broad.core.math.ScanStatistics</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a58a72f78575d4e504c55b46c487d25c7">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_binomial_enrichment_score.html#a9ce874d7fa579be05ecc406e0bd104da">umms.core.model.score.BinomialEnrichmentScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#ae7e5653afa419e3870ebe81a5cdb36ab">umms.core.model.score.PoissonEnrichmentScore</a>
</li>
<li>calculateScore()
: <a class="el" href="classjaligner_1_1_alignment.html#a412518ac97945f4efeacaffb0a2db896">jaligner.Alignment</a>
</li>
<li>callKLPeaks()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_r_a_p_peak_caller.html#a4241a787df082c17f3e7a48545c83be9">broad.pda.seq.rap.RAPPeakCaller</a>
</li>
<li>callSkellamPeaks()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_r_a_p_peak_caller.html#ad88d001df75ac488d424b594cb168c7c">broad.pda.seq.rap.RAPPeakCaller</a>
</li>
<li>Cell()
: <a class="el" href="classjaligner_1_1_cell.html#a448af57dcf6f098bce46d87a23589b85">jaligner.Cell</a>
</li>
<li>centroidOf()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix.html#affcf048fc62fc2a720bc87b63b4028ee">broad.core.motif.PositionWeightMatrix</a>
</li>
<li>changeBackground()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model.html#ae948a8e429c783cd525b495186edfa16">broad.core.siphy.EvolutionaryModel</a>
</li>
<li>checkOrientation()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#ab57a595f55e990b71c2f71a86bec4043">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>checkScore()
: <a class="el" href="classjaligner_1_1_alignment.html#ab9ec4af0857b026b6a61ef95c5d62aad">jaligner.Alignment</a>
</li>
<li>chiSquareDistance()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#aee3d9ed2c0e1745431fa438ca0add5c1">broad.core.math.EmpiricalDistribution</a>
</li>
<li>chol()
: <a class="el" href="class_jama_1_1_matrix.html#aa8318d703898bcdad10235bf23566f2e">Jama.Matrix</a>
</li>
<li>CholeskyDecomposition()
: <a class="el" href="class_jama_1_1_cholesky_decomposition.html#a315b88758566a9852579b5138b7386f2">Jama.CholeskyDecomposition</a>
</li>
<li>Chromosome()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_chromosome.html#ac2cef93e257072afecedd00560e4a1ad">broad.pda.feature.genome.Chromosome</a>
</li>
<li>ChromosomeInconsistencyException()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_chromosome_inconsistency_exception.html#ac34fbdafac04c3b356c31ffd94c87638">umms.core.alignment.ChromosomeInconsistencyException</a>
</li>
<li>chromosomeLength()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a050728258c8438e309c1412caabcc130">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
</li>
<li>ChromosomeWithBubblesJGraphT()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#aac13df74ce52fd9243d7f79b623cb314">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>chunk()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#ae15813e860a2d317425e2520831aeb5a">broad.core.sequence.Sequence</a>
</li>
<li>CisVsTrans()
: <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_cis_vs_trans.html#a43583a09057da3787a0896aee63ec492">broad.pda.differentialExpression.CisVsTrans</a>
</li>
<li>ClassifyAgilentLincProbes()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_classify_agilent_linc_probes.html#a274f182a83c3032b230022d1789bafc1">broad.pda.geneexpression.agilent.ClassifyAgilentLincProbes</a>
</li>
<li>cleanIsoforms()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_gene_with_isoforms.html#a88948fa8b40ad89c03b68290d7680e89">broad.pda.gene.GeneWithIsoforms</a>
</li>
<li>cleanup()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_collect_annotation_enrichments.html#a36bfbc9033accd19863195a27a170eaa">broad.pda.seq.rap.CollectAnnotationEnrichments</a>
</li>
<li>clear()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#ab5bc07ea8ca501a2279d52841e8c8d64">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
, <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_interval_tree_3_01_v_01_4.html#a372b459843c74a5c78e19f6625051ef2">broad.core.datastructures.IntervalTree&lt; V &gt;</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_alignment.html#a3dbdfe66d00c1a1dd2abe810f50d8887">broad.core.multiplealignment.MAFAlignment</a>
, <a class="el" href="classbroad_1_1core_1_1parser_1_1_string_parser.html#aef4b94fbed7a71500a398fac0f7dd1de">broad.core.parser.StringParser</a>
, <a class="el" href="classbroad_1_1core_1_1siphy_1_1_tree_scaler_i_o.html#ac47c876b37855beabcb882191a1b3796">broad.core.siphy.TreeScalerIO</a>
, <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a771489368c29e46deb27d897cc86e89c">broad.pda.annotation.BEDFileParser</a>
</li>
<li>clearAttribute()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_g_f_f.html#a065fabc714731218d195f1e65031dab2">broad.core.annotation.GFF</a>
</li>
<li>clearFullIntervalTreeAsAlignmentsCached()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a79e7e291a3c7f8594234974c673c953b">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a7bc3e012c4981ea00d6314e100cdaf7f">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>CLIP_AT_BOUNDARIES
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_plot_aggregate_regions.html#a4407dcbebaf7f056ee5f1991bfad1a4a">broad.pda.seq.rap.PlotAggregateRegions</a>
</li>
<li>clone()
: <a class="el" href="class_jama_1_1_matrix.html#a07d5a928a5a152a6f508cae88ac07c7d">Jama.Matrix</a>
</li>
<li>cloneRegions()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#aafba9e3874d96126b9001f8ff9d815a2">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
</li>
<li>close()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_j_c_s_cache_1_1_j_c_s_node_iterator.html#ad84be5a706b26015491a523f6373e73e">broad.core.datastructures.JCSCache.JCSNodeIterator</a>
, <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_paired_end_read_iterator.html#a680e5116d46574fe21e8a0f9621fdcb2">broad.pda.samtools.PairedEndReadIterator</a>
, <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_s_a_m_paired_end_file_reader.html#a39580aacae64bebfdad06b166388c554">broad.pda.samtools.SAMPairedEndFileReader</a>
, <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_wrapped_iterator.html#aaa2c9f20f6e76125330e240265c57aa2">broad.pda.samtools.WrappedIterator</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_parser.html#a4dc28f7d389ce614c2f3bdcd2762f6b7">broad.pda.seq.fastq.FastqParser</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_wrapped_i_g_v_iterator.html#ac4ea627972346e3bfbccb77dd81793b3">broad.pda.seq.segmentation.WrappedIGVIterator</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_shuffled_iterator_3_01_t_01extends_01_annotation_01_4.html#ae137b35ce4ebdd00c1d3406f9f0b818f">umms.core.coordinatesystem.ShuffledIterator&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1general_1_1_closeable_filter_iterator_3_01_t_01_4.html#a4e5c6d33f6182d12d6ccca55ea651336">umms.core.general.CloseableFilterIterator&lt; T &gt;</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model_1_1_filtered_iterator.html#ad5b5430a14933e8043a5fdf003cdd369">umms.core.model.AlignmentModel.FilteredIterator</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model_1_1_node_iterator.html#a7c83272b03c1e7284a2683bddf41af35">umms.core.model.AlignmentModel.NodeIterator</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model_1_1_wrap_alignment_count_iterator.html#ade9d57988a39b3c0192c221425b04e8a">umms.core.model.AlignmentModel.WrapAlignmentCountIterator</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_1_1_filtered_iterator.html#ac131b52f17f049cb7ca31b1fae0918e1">umms.core.model.JCSAlignmentModel.FilteredIterator</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_1_1_node_iterator.html#a4ea6c85e5c0c2708ab2d534a4cf86f0c">umms.core.model.JCSAlignmentModel.NodeIterator</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model_1_1_wrap_alignment_count_iterator.html#a1d56d25dd74c8dc82b8699dc2f954c31">umms.core.model.JCSAlignmentModel.WrapAlignmentCountIterator</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_score_iterator_3_01_t_01extends_01_window_score_01_4.html#a6495f8883cc968e1260c819990b6c807">umms.core.model.score.WindowScoreIterator&lt; T extends WindowScore &gt;</a>
, <a class="el" href="classumms_1_1core_1_1readers_1_1_paired_end_reader.html#a042b674540f796c0631213592b167448">umms.core.readers.PairedEndReader</a>
, <a class="el" href="classumms_1_1core_1_1writers_1_1_paired_end_writer.html#a5d5be508ea6ea765331fd61ec7ddcbdc">umms.core.writers.PairedEndWriter</a>
</li>
<li>CloseableFilterIterator()
: <a class="el" href="classumms_1_1core_1_1general_1_1_closeable_filter_iterator_3_01_t_01_4.html#a9cdca39896a41041469d8c1c7910dbcd">umms.core.general.CloseableFilterIterator&lt; T &gt;</a>
</li>
<li>closeFilterRejectWriters()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#a85f7d561f66a351adc6a02682ba18685">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>closeTo1()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_math_util.html#aa7a9fefae878f5f2770712f45373260a">broad.core.math.MathUtil</a>
</li>
<li>CLUSTAL()
: <a class="el" href="classjaligner_1_1formats_1_1_c_l_u_s_t_a_l.html#a4b5118b8e370a08699645a1ff0571fd3">jaligner.formats.CLUSTAL</a>
</li>
<li>Cluster()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_cluster.html#aa7aac9aa7b19ceef6e3959d895002c83">broad.pda.geneexpression.clustering.Cluster</a>
</li>
<li>cluster()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_hierarchical_clustering.html#a54cae450cf75f5336ea4175a061207c5">broad.pda.geneexpression.clustering.HierarchicalClustering</a>
</li>
<li>CLUtil()
: <a class="el" href="classbroad_1_1core_1_1util_1_1_c_l_util.html#a6c1df173d24e2ada39900c07100ccd96">broad.core.util.CLUtil</a>
</li>
<li>codonToAminoacid
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor.html#a26170c7f5b1f3f17effd444c1ddcd9f4">broad.core.multiplealignment.AlignmentExtractor</a>
</li>
<li>colapse()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_gene_with_isoforms.html#a05c201b75ab2113fb78c65b986dbce5a">broad.pda.gene.GeneWithIsoforms</a>
</li>
<li>collapse()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#a3a84e443b2bd20e3ac8aa8c4e91eaad3">umms.core.annotation.AbstractAnnotationCollection&lt; T extends Annotation &gt;</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#a57ab6ba2dd1e9ade31e4e308d21c7eb9">umms.core.annotation.AnnotationCollection&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4.html#a0e7dc7676b6d0c5b561104b132c68350">umms.core.annotation.AnnotationList&lt; T extends Annotation &gt;</a>
</li>
<li>collapseByIntersection()
: <a class="el" href="classbroad_1_1core_1_1util_1_1_collapse_by_intersection.html#acca4135dc1d9cafa4a07b0be84b00e67">broad.core.util.CollapseByIntersection</a>
</li>
<li>CollapsedGeneLengths()
: <a class="el" href="classumms_1_1esat_1_1_collapsed_gene_lengths.html#aac7dc4890b2a7910293c614dcc11f39d">umms.esat.CollapsedGeneLengths</a>
</li>
<li>CollapseGenesByIntersection()
: <a class="el" href="classbroad_1_1core_1_1util_1_1_collapse_by_intersection.html#a36614cce291e2f9b03e50803b52a27ec">broad.core.util.CollapseByIntersection</a>
</li>
<li>collapseOverlappers()
: <a class="el" href="classumms_1_1core_1_1utils_1_1_annotation_utils.html#af14dd663711ec5bb378f70875f114e41">umms.core.utils.AnnotationUtils</a>
</li>
<li>CollapseProbesByMinPValue()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_collapse_probes_by_min_p_value.html#af33b2c853b3c87797106c98b06473036">broad.pda.geneexpression.agilent.CollapseProbesByMinPValue</a>
</li>
<li>CollapseProjections()
: <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_collapse_projections.html#a3b2683d1c6e24a83ff58965141a12c3b">broad.pda.differentialExpression.CollapseProjections</a>
</li>
<li>collapseSingleExonTranscripts()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a45ec37aa431467c0b765680ef53aefcc">broad.pda.annotation.BEDFileParser</a>
</li>
<li>collectResults()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_collect_annotation_enrichments.html#a06a833e59342a9dcf6df6ef6d283a794">broad.pda.seq.rap.CollectAnnotationEnrichments</a>
</li>
<li>columnDensityNormalization()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#ab7bca43861ef3ed38f85238a0cbaa0c3">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>columnDimension()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#ae61bb6176aa58f4102f3b9ca85a1581f">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>CombinationGenerator()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_combination_generator.html#a239cddbdffdb62bbff5c0e5f5fd43ac3">broad.core.math.CombinationGenerator</a>
</li>
<li>commandLineHasDebugFlag()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#a81cf63b0b2d082a3ee577570365edb68">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>commandLineOutDir()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#ad720ec027b5514deff88752afab91036">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>CommandLineParser()
: <a class="el" href="classbroad_1_1core_1_1parser_1_1_command_line_parser.html#aca48b6e3ce81bf026ce5dd86d0b39287">broad.core.parser.CommandLineParser</a>
</li>
<li>commaSeparatedStringToList()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_conservation_utils.html#a5fe0b616c5840dc5e6cd74bed82a1ee3">broad.core.siphy.ConservationUtils</a>
</li>
<li>compare()
: <a class="el" href="classbroad_1_1core_1_1util_1_1_chromosome_string_comparator.html#ae891652a81581a952d777fd185044baf">broad.core.util.ChromosomeStringComparator</a>
, <a class="el" href="classjaligner_1_1matrix_1_1_matrices_compartor.html#aa43bed41e2ff9421b77785f3390b5de8">jaligner.matrix.MatricesCompartor</a>
</li>
<li>compareNames()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_chromosome.html#ab5ed3e1b37710b6028bae9804c6ad67f">broad.pda.feature.genome.Chromosome</a>
</li>
<li>compareTo()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_genomic_annotation.html#af2945eb5b4eda18596aa43ee1ef6293c">broad.core.annotation.BasicGenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#a153aadccdd60d81d71286ab8185e0687">broad.core.annotation.BasicLightweightAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1primer3_1_1_hairpin_kmer.html#a94a68e49d68979288bb6eb0ef8d8984b">broad.core.primer3.HairpinKmer</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#ab5b37e429308cc060e9a3e1933c22592">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_chromosome.html#af500375109a20373d852e86fd8c5ed75">broad.pda.feature.genome.Chromosome</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#a70cb4c85f8d4cacee312306bee79508d">broad.pda.seq.graph.Path</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment_1_1_unordered_gene_pair.html#a2bf54d6e4db59e3149e03893df7d2644">umms.core.alignment.FeatureSequenceAlignment.UnorderedGenePair</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation.html#a387f03c8bfb61c496326838af22fa688">umms.core.annotation.AbstractAnnotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_compound_interval.html#ad6a78ea63b868ff8265595187b58a4e6">umms.core.annotation.CompoundInterval</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a3cd085a5501a376bdd6f929a7d1271b3">umms.core.annotation.Gene</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_single_interval.html#ab5cd161bf20b4b63196de28feba3a400">umms.core.annotation.SingleInterval</a>
, <a class="el" href="classumms_1_1core_1_1feature_1_1_gene_window.html#adb96ff8b83b9d068e49a93c5b2e52669">umms.core.feature.GeneWindow</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_count_score.html#a10ed7654d427b9467d72c30b21a0435a">umms.core.model.score.CountScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_end_in_window_count_score.html#aa9e48a00f68443a78aa022d0b0af17f7">umms.core.model.score.EndInWindowCountScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_start_in_window_count_score.html#a273b98b70ea1318c32a11c6e880d98b9">umms.core.model.score.StartInWindowCountScore</a>
</li>
<li>compareToAnnotation()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#ac9f43edf121d64bc95040d941729c939">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation.html#a6526db819c6e715c4b0dfe50a2750b11">umms.core.annotation.AbstractAnnotation</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#ad31140b58428e5ceb60d15118d1a1a28">umms.core.annotation.Annotation</a>
</li>
<li>compatible()
: <a class="el" href="classumms_1_1core_1_1scripture_1_1_build_scripture_coordinate_space.html#acf4c8c23eafcf330eff979c95da4aa10">umms.core.scripture.BuildScriptureCoordinateSpace</a>
</li>
<li>complement()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_compute_m_i_r_score.html#ae4b403abd67a0c76183f6d11a6e646cc">broad.core.primer3.ComputeMIRScore</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#ab208ef33c2a874e665a270a30a9a66b4">broad.core.sequence.Sequence</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#ac9aa1ca4eab54cf1f61c388b71baf63c">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#a0c9c81ea4fbb301fc88aea73a98e5d53">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a4e365680579c67297f191e9fcd804b02">umms.core.annotation.BasicAnnotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_compound_interval.html#af1eb8352a5ba9319729f9a18e8f4c7d8">umms.core.annotation.CompoundInterval</a>
</li>
<li>CompoundInterval()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_compound_interval.html#a11643c83ca967494810bdcb0bd7886a7">umms.core.annotation.CompoundInterval</a>
</li>
<li>compress()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_alignment.html#a0b37804e6ca4cc55157e60670ed39a26">broad.core.multiplealignment.MAFAlignment</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#ad475a9fa42f4aaa5fc128fd3688f9c38">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>compressMatrixByRowDescriptions()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a626d82bf2cfedf21c83407f04fc931ff">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>compute454Flow()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#aa07f336c9e608874def5dfb43f089258">broad.core.sequence.Sequence</a>
</li>
<li>computeAbsFoldMatrix()
: <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_differential_scoring.html#a0412d7c9e49d4aaffb91eda091b48a26">broad.pda.differentialExpression.DifferentialScoring</a>
</li>
<li>ComputeConservedMotifs()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1misc_1_1_compute_conserved_motifs.html#a05e3cb6dd1ab74dec72d5d98996a4019">broad.core.motif.misc.ComputeConservedMotifs</a>
</li>
<li>ComputeCorrelationOfReplicates()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_compute_correlation_of_replicates.html#a8a2c2ac12a1cdf0a1457b8f9bd597cb9">broad.pda.geneexpression.clustering.ComputeCorrelationOfReplicates</a>
</li>
<li>ComputeEnrichments()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1chip_1_1_compute_enrichments.html#a134d8d1648f3b6599adeb806fe595977">broad.pda.seq.chip.ComputeEnrichments</a>
</li>
<li>computeFoldMatrix()
: <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_differential_scoring.html#a3bab659a18e1c6ba69833a89bf6d0d4c">broad.pda.differentialExpression.DifferentialScoring</a>
</li>
<li>computeGCContent()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a6eb5265c9cdcde2244f7c5aac6497093">broad.core.sequence.Sequence</a>
</li>
<li>computeGCPercent()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_compute_original_score.html#a437266c034c972266b211060a846c495">broad.core.primer3.ComputeOriginalScore</a>
</li>
<li>computeGlobalStats()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#ae1bdf3b93c959ea459a07c8b781913c9">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a6bba3f6088806514166bc1109d3c6209">umms.core.model.JCSAlignmentModel</a>
</li>
<li>ComputeHypergeomtricEnrichment()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1_compute_hypergeomtric_enrichment.html#a9ea7926db744a08de985d2248c1cb072">broad.pda.geneexpression.ComputeHypergeomtricEnrichment</a>
</li>
<li>computeJMatrix()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model.html#a5db015308ac89579addc1e3db945dc14">broad.core.siphy.EvolutionaryModel</a>
</li>
<li>ComputeKmersForRegion()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1tools_1_1conservation_1_1_compute_kmers_for_region.html#a22724955ca2aa57424d9707b3693b3b7">broad.core.siphy.tools.conservation.ComputeKmersForRegion</a>
</li>
<li>computeLikelihood()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model.html#af0505fba93f6d822b09fd631b78bfadf">broad.core.siphy.EvolutionaryModel</a>
</li>
<li>ComputeMaxPercentileOfElements()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1tools_1_1conservation_1_1_compute_max_percentile_of_elements.html#a8b61f06dabc30bb30e88a05e338de30f">broad.core.siphy.tools.conservation.ComputeMaxPercentileOfElements</a>
</li>
<li>computeMaxTM()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_primer_utils.html#ac67f3e563a1546d6766907568512498f">broad.core.primer3.PrimerUtils</a>
</li>
<li>computeMIRScore()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_compute_m_i_r_score.html#af05e599f26cbdc0faed1fa397f5c4f3e">broad.core.primer3.ComputeMIRScore</a>
</li>
<li>computeOriginalScore()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_compute_original_score.html#ac0a5a237e3b52c5fdeaf7a74d1828986">broad.core.primer3.ComputeOriginalScore</a>
</li>
<li>computeOriginalScorePerKMer()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_compute_original_score.html#ad638c89f72ea2e8e1f4e602ca5b6ca26">broad.core.primer3.ComputeOriginalScore</a>
</li>
<li>computePathLogLikelihood()
: <a class="el" href="classbroad_1_1core_1_1hmm_1_1_markov_model_3_01_t_01_4.html#ab9e4009415f48fb35892f39e73544237">broad.core.hmm.MarkovModel&lt; T &gt;</a>
, <a class="el" href="classbroad_1_1core_1_1siphy_1_1_pi_h_m_m.html#a5ca38442b44e90860d8af48fc0aefe7f">broad.core.siphy.PiHMM</a>
</li>
<li>ComputePercentageOfKmers()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1tools_1_1conservation_1_1_compute_percentage_of_kmers.html#a9ea8017314f8811b0fff01396a0c90ed">broad.core.siphy.tools.conservation.ComputePercentageOfKmers</a>
</li>
<li>computePermutedZScores()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_statistics.html#a1b6c59d31a5f6102d779c99d737a0d81">broad.core.math.Statistics</a>
</li>
<li>computeProb()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_column.html#a43ec5b48d9467a142242c9e3c4e4863e">broad.core.motif.PositionWeightColumn</a>
</li>
<li>computeScore()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region_window.html#ab04991e1445c6bbeb35c2f167f52ce2d">broad.core.sequence.SequenceRegionWindow</a>
</li>
<li>computeScoreDistribution()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix.html#a8dbc8707e7fd3c12d2264d80ecf20d36">broad.core.motif.PositionWeightMatrix</a>
</li>
<li>computeSequenceBGSequences()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_p_w_m_utils.html#ade2a53589ad91434fd0a187a7befd8b4">broad.core.motif.PWMUtils</a>
</li>
<li>computeSequenceComposition()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#ae40c1d7033b0e09af85bcda1379f59c8">broad.core.sequence.Sequence</a>
</li>
<li>computeTestStatistics()
: <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_differential_scoring.html#abf56213c28a2bf6b919b42ebdbf2d003">broad.pda.differentialExpression.DifferentialScoring</a>
</li>
<li>computeTM()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_primer_utils.html#a425177255bddf89ea4e26fc81c5e2f11">broad.core.primer3.PrimerUtils</a>
</li>
<li>computeTM2()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_primer_utils.html#a47dd17b7c6c2a0a2d7a7e64c10a31807">broad.core.primer3.PrimerUtils</a>
</li>
<li>computeTransitions()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model.html#a02f88c2dc38c01a5923f064477daaa51">broad.core.siphy.EvolutionaryModel</a>
</li>
<li>concatAnnotationSet()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#adbf3eb7df259742adac2209dfd5699a1">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
</li>
<li>cond()
: <a class="el" href="class_jama_1_1_matrix.html#a531ae0d584b7861c5924efcc9d356741">Jama.Matrix</a>
, <a class="el" href="class_jama_1_1_singular_value_decomposition.html#a0193ef1cb3aa0361d897409a53a6e4c6">Jama.SingularValueDecomposition</a>
</li>
<li>confidenceByColumn()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#af1bdcc66fa4e72efe89060370302373c">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>ConnectDisconnectedTranscripts()
: <a class="el" href="classumms_1_1core_1_1scripture_1_1statistics_1_1_connect_disconnected_transcripts.html#a513b9e39e9fbcfc2db0ccd8135e5b02d">umms.core.scripture.statistics.ConnectDisconnectedTranscripts</a>
</li>
<li>connectVertexToGraph()
: <a class="el" href="classumms_1_1core_1_1scripture_1_1_oriented_chromosome_transcript_graph.html#a6da3579b3d2cd6a0de92d18839d2cac8">umms.core.scripture.OrientedChromosomeTranscriptGraph</a>
</li>
<li>ConservedPWMScanner()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_conserved_p_w_m_scanner.html#a0701148d8534b8806f9f49e27a15433f">broad.core.siphy.ConservedPWMScanner</a>
</li>
<li>constituentIntrons()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_gene_with_isoforms.html#a0edc29c33ccb0bd3514f8ad36230f15f">broad.pda.gene.GeneWithIsoforms</a>
</li>
<li>constituentIsoform()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_gene_with_isoforms.html#a7458b9e87f5562becd888d0c50f378f4">broad.pda.gene.GeneWithIsoforms</a>
</li>
<li>constructWithCopy()
: <a class="el" href="class_jama_1_1_matrix.html#ad900a4fa94f188bd56f4f89a20f30f0c">Jama.Matrix</a>
</li>
<li>containChr()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a821b1303f6cbbc2ecb4f127728c5babe">broad.pda.annotation.BEDFileParser</a>
</li>
<li>containedWithin()
: <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_alignments.html#addcedc7cdc38187c24a5bb5077cb9112">broad.pda.datastructures.Alignments</a>
</li>
<li>contains()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#a768dd90feef065f8b04a253c2d1c3e1e">broad.core.annotation.BasicLightweightAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation.html#a2987707b47ff3712738f13421c153f1f">broad.core.annotation.LightweightGenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_alignment.html#a822c6167e78e38c9234dc27ac2f6e869">broad.core.multiplealignment.MAFAlignment</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a175ad53de415a37d032a20ca3d5370ab">broad.core.sequence.Sequence</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#ad896c336b5e5144d7ddf11dd612a3093">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a7396c229435f01089a3c4280e93beb36">broad.pda.annotation.BEDFileParser</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#adfc487b0ab197c742bbbe39767272b79">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a25d8463822c9b3236fce57bef3e3cf2a">umms.core.annotation.BasicAnnotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_compound_interval.html#a3bf8c87e093e0b792f7f26b535070e48">umms.core.annotation.CompoundInterval</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_single_interval.html#ac4beddc176961e3571745e624e8b7935">umms.core.annotation.SingleInterval</a>
</li>
<li>containsAnnotation()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#afc5bc223c3e7bac8c5d2e8267f8c4725">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
</li>
<li>containsAttribute()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a872db95b096377da9d316bd91a2275e8">umms.core.annotation.Gene</a>
</li>
<li>containsColumn()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#ab9b0bd67e77fdf66dd7a583293c5d3cc">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>containsDataFor()
: <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_genomic_space.html#afff244ec46e14018c2e7d7114ec266cb">umms.core.coordinatesystem.GenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcript_in_genomic_space.html#ae3b26ae31c790de615ccb58067828583">umms.core.coordinatesystem.TranscriptInGenomicSpace</a>
</li>
<li>containsExactInterval()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_compound_interval.html#ae5614a7f4a1a316d34000d64aa5c9ce1">umms.core.annotation.CompoundInterval</a>
</li>
<li>containsFullLengthUngappedMatch()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_smith_waterman_alignment.html#a0a0fb30db091b27fd560c0e04b0f7d67">umms.core.alignment.SmithWatermanAlignment</a>
</li>
<li>containsInterval()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_compound_interval.html#a488f2416729954c1bda70925354f1f38">umms.core.annotation.CompoundInterval</a>
</li>
<li>containsReference()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a13cae6f7cc2b1c6830fa8adb2d410ee4">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aa16f5a8f31e8a8bbf69119b241b51f28">umms.core.model.JCSAlignmentModel</a>
</li>
<li>containsRow()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#aef05204bab76e833ceab87413b13c1df">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>contiguousEndSubSequenceOverMin()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_maximum_contiguous_subsequence.html#a8e7167f5b84cac82a44d3cddfc39976d">broad.core.annotation.MaximumContiguousSubsequence</a>
</li>
<li>contiguousStartSubSequenceOverMin()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_maximum_contiguous_subsequence.html#a340ad2ccb44aae075f97b65b41c60194">broad.core.annotation.MaximumContiguousSubsequence</a>
</li>
<li>CONTINUOUS_EXTENSION
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_collect_annotation_enrichments.html#af0f4ff2d7acf7fc97467f5e7b8443eab">broad.pda.seq.rap.CollectAnnotationEnrichments</a>
</li>
<li>ContinuousDataAlignmentModel()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#ad0d8dfa7ae363e2d07df5fd31ac3b3e2">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>CONTROL
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_build_ratio_null_distribution.html#a774eb640de6c6b27ffbff3ebaf9040d1">broad.pda.seq.rap.BuildRatioNullDistribution</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_genome_scoring_program.html#acfa8d6ffb03052c84e8766d0cd310470">broad.pda.seq.rap.GenomeScoringProgram</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_ratio_permutation_peak_caller.html#a1c4c5d03165fa9a6a3cbdbb11bc78916">broad.pda.seq.rap.RatioPermutationPeakCaller</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1rna_1_1_transcriptome_scoring_program.html#a90bcb800638d3bc689301daff0be1397">broad.pda.seq.rap.rna.TranscriptomeScoringProgram</a>
</li>
<li>controlSamples
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#aa8b0b700ab5285893b7c7d7bc0710941">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>convertInputToPairedEnd()
: <a class="el" href="classumms_1_1core_1_1writers_1_1_paired_end_writer.html#accda9db268398e2d01de77a971db2231">umms.core.writers.PairedEndWriter</a>
</li>
<li>ConvertToDiscreteMatrix()
: <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_convert_to_discrete_matrix.html#af4269b7d284553e5c252bd261125d7b7">broad.pda.differentialExpression.ConvertToDiscreteMatrix</a>
</li>
<li>convertToFasta()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_parser.html#a954f65e71af98855f3477d5b17320f18">broad.pda.seq.fastq.FastqParser</a>
</li>
<li>convertToNumberedFasta()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_parser.html#a2bbe196edee4649d6a62c569426cad04">broad.pda.seq.fastq.FastqParser</a>
</li>
<li>convertToPairedEndFragment()
: <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_s_a_m_paired_end_file_reader.html#ae86af5872ad93d8a8dcb34e3fa03510a">broad.pda.samtools.SAMPairedEndFileReader</a>
</li>
<li>COORD_SPACE
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_genome_command_line_program.html#a9b8d006a5a2595175626e6ca35697a9f">broad.pda.seq.rap.GenomeCommandLineProgram</a>
</li>
<li>coordinateSpace
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_genome_command_line_program.html#afc325af5b3b4a5698675575c3bc01ab7">broad.pda.seq.rap.GenomeCommandLineProgram</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4.html#aa4eb4081457dd28d136337f3447ad446">umms.core.annotation.AnnotationList&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a39ef275175b500df6548f8ec18949dfe">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aa04751a0c366de794593489168749245">umms.core.model.JCSAlignmentModel</a>
</li>
<li>copy()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#af8962575e65d9cdbba2bba970b3f263c">broad.core.datastructures.MatrixWithHeaders</a>
, <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix.html#a1793deced98122752214cacdfef0a1d4">broad.core.motif.PositionWeightMatrix</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#aa04f060027d77f46a1384c279bacacaa">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model.html#aa418445762f1cbcb395dac8c0effbad6">broad.core.siphy.EvolutionaryModel</a>
, <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model_parameters.html#a0734fcf5d3102b10ade0f7faf0f12177">broad.core.siphy.EvolutionaryModelParameters</a>
, <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a0449ae4d64ca760468587a0a03144864">broad.pda.annotation.BEDFileParser</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#a16cab9392b06f9a4860df6690ba24c07">broad.pda.seq.graph.Path</a>
, <a class="el" href="classjaligner_1_1_alignment.html#a8ec62ac2e46293fd8ecee14659e9a65e">jaligner.Alignment</a>
, <a class="el" href="class_jama_1_1_matrix.html#a571128c206168d38c707a43b41bb9661">Jama.Matrix</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#a88efc6a83878d7af34c4a20786df29eb">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#aafaa36eb94c730a1325e356a519e8e09">umms.core.annotation.BasicAnnotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a682fa39fb20128063873191be423331d">umms.core.annotation.Gene</a>
</li>
<li>CorrelationAgilentNanostring()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_correlation_agilent_nanostring.html#a0914250010bca3f7c62802ac8f547408">broad.pda.geneexpression.agilent.CorrelationAgilentNanostring</a>
</li>
<li>CorrelationBetweenReplicates()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_correlation_between_replicates.html#a2e355029abd1c0c68775a74ca66caeab">broad.pda.geneexpression.agilent.CorrelationBetweenReplicates</a>
</li>
<li>CorrelationMatrix()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_correlation_matrix.html#a4efd3fcf38be9ef8a6db2088f9f743ed">broad.pda.geneexpression.clustering.CorrelationMatrix</a>
</li>
<li>count()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a583c93381358136c1c916e4bcf195bcc">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a31cfc1d1c12f0c7ebece09e6c5bd4f43">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>countLessThanLambda()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_skellam_score.html#a15e62e4b9eb28988d625c51ef448002c">broad.pda.seq.rap.SkellamScore</a>
</li>
<li>CountNums()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1_count_nums.html#aa79da4306cbdc84b108afeb7c5bb95a2">broad.pda.geneexpression.CountNums</a>
</li>
<li>countOverlappers()
: <a class="el" href="classbroad_1_1core_1_1overlaputils_1_1_gene_set_intersect.html#a8fdf9a00a09d928137c033175041c778">broad.core.overlaputils.GeneSetIntersect</a>
</li>
<li>countReads()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a85bb4ba62f07d9f29d04a0736d0aaa90">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>CountScore()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_count_score.html#a0bf74019bebe256876b06ac7065092d9">umms.core.model.score.CountScore</a>
</li>
<li>countSoftMaskedBases()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a7832688b204b920fb48642f144c95606">broad.core.sequence.Sequence</a>
</li>
<li>countWithinExon()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a87bfcd3cfd5d4adaad76fd1b87ddc937">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
</li>
<li>covariance()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_statistics.html#adccf05d41f39445ef05d90e1b0d52b0e">broad.core.math.Statistics</a>
</li>
<li>CoverageScore()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_coverage_score.html#a6f6d0a2817cc70044542a6fd94a31143">umms.core.model.score.CoverageScore</a>
</li>
<li>create()
: <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_annotation_factory_3_01_t_01extends_01_lightweight_genomic_annotation_01_4.html#a34914aa67623922c30e7e456ee7d2415">broad.core.annotation.AnnotationFactory&lt; T extends LightweightGenomicAnnotation &gt;</a>
, <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_factory.html#a4aa8d21072631f53478e96aacf98b86b">broad.core.annotation.AnnotationReaderFactory</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment_factory.html#af8089270a8d570a1d60ee16701ad5fbe">broad.core.multiplealignment.MultipleAlignmentFactory</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment_i_o_factory.html#a22fd88edfcc24e1b02cd08c3d6f15951">broad.core.multiplealignment.MultipleAlignmentIOFactory</a>
, <a class="el" href="interfaceumms_1_1core_1_1general_1_1_tabbed_reader_1_1_factory_3_01_t_01_4.html#a7990dcd1dc832e9faaf84f598be8ea2a">umms.core.general.TabbedReader.Factory&lt; T &gt;</a>
</li>
<li>createAnnotation()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#af52bebd3414e0f94c87ebd3c8d6fa158">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
, <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_annotation_reader.html#acfd320987e51841554395a70294a065c">broad.core.annotation.BasicAnnotationReader</a>
, <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_genomic_annotation_reader.html#a987b7996be954efb377c5911a66d9e28">broad.core.annotation.BasicGenomicAnnotationReader</a>
, <a class="el" href="classbroad_1_1core_1_1annotation_1_1_b_e_d_graph_reader.html#a38845de37182e1215200b0e3772c540f">broad.core.annotation.BEDGraphReader</a>
, <a class="el" href="classbroad_1_1core_1_1annotation_1_1_b_e_d_reader.html#a65cd4de10f0de6ff4441c1ebce59f0d0">broad.core.annotation.BEDReader</a>
, <a class="el" href="classbroad_1_1core_1_1annotation_1_1_g_f_f_reader.html#a904ec7132145acee82ad588a7220bc9c">broad.core.annotation.GFFReader</a>
, <a class="el" href="classbroad_1_1core_1_1annotation_1_1_repeat_masker_annotation_reader.html#a28135a78c93863178ac6e4e06eb2e6fa">broad.core.annotation.RepeatMaskerAnnotationReader</a>
, <a class="el" href="classbroad_1_1core_1_1annotation_1_1_short_b_e_d_reader.html#a7540cff50bedf34657b60da407c3228e">broad.core.annotation.ShortBEDReader</a>
, <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_alignments_reader.html#a248dffdc961399ecbf3103abc357ee49">broad.pda.datastructures.AlignmentsReader</a>
</li>
<li>createAsciiFromBinary()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_fasta_quality_binary_i_o.html#ab97e7ca335721336cee440cb805fcc0e">broad.core.sequence.FastaQualityBinaryIO</a>
</li>
<li>createBinaryFromAscii()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_fasta_quality_binary_i_o.html#aa61d72cc22872856e9344310c57f2bff">broad.core.sequence.FastaQualityBinaryIO</a>
</li>
<li>createDefaultTwoStateChain()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_pi_h_m_m.html#aa83a4e23ba05718dd683cbe7b230464c">broad.core.siphy.PiHMM</a>
</li>
<li>createEdge()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#aa00423c00a6b114c4b1146278a8ec66d">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>createFromCommandArgs()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#ade2431eade4f45e77e73e36236cdc81e">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>createFromUCSC()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#a5c367cdb862fa07a36c30a41a1c27994">broad.core.annotation.BasicLightweightAnnotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a29f98089d7537358134502ac8f3edf79">umms.core.annotation.BasicAnnotation</a>
</li>
<li>createIndex()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_alignment.html#a8255b00582e0e79db7e00ab566790d19">broad.core.multiplealignment.MAFAlignment</a>
</li>
<li>createIndexFromQualityFile()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_fasta_quality_index.html#a0676d9e369196ce9eb7e7875006afe6a">broad.core.sequence.FastaQualityIndex</a>
</li>
<li>createIsoPWM()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix.html#a566b285edf5ae045f8af3ed12a0eb549">broad.core.motif.PositionWeightMatrix</a>
</li>
<li>createSizeFile()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_fasta_sequence_i_o.html#a68327781ab4dee01efe5ff346951f325">broad.core.sequence.FastaSequenceIO</a>
</li>
<li>createTwoStateChain()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_pi_h_m_m.html#a509a9478700a4bb63d2912a29d0b9bb4">broad.core.siphy.PiHMM</a>
</li>
<li>createUnloadedAlignment()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_i_o.html#a454109fd269c309a9a313abf27bed8b6">broad.core.multiplealignment.MAFIO</a>
</li>
<li>CrossRandModelsWithBed()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1transcripts_null_model.html#afdfbc5fab3964d33ecb497cec6a84d37">broad.pda.gene.transcriptsNullModel</a>
</li>
<li>cs
: <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_shuffled_iterator_3_01_t_01extends_01_annotation_01_4.html#a4919b67f1b738b7aee76c0905d23062e">umms.core.coordinatesystem.ShuffledIterator&lt; T extends Annotation &gt;</a>
</li>
<li>ctrl
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_window_scores.html#adbea63ef3f80374592309593aff3eeac">broad.pda.seq.protection.WindowScores</a>
</li>
<li>ctrlModel
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#ad2cbf957a05a1fb3e0337dcb4b95ab91">umms.core.model.score.MultiScore</a>
</li>
<li>currentBase
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_abstract_fasta_parser.html#a9d3b56bac67fec8a4d9b98cc852ecc54">broad.core.sequence.AbstractFastaParser</a>
</li>
<li>customCommandLineValidation()
: <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_paired_end_t_d_f.html#a02a6d35622f6b076ea3510fbd325bb3e">broad.pda.samtools.PairedEndTDF</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_collect_annotation_enrichments.html#a042956bda14093b8ce0f448921d6ba18">broad.pda.seq.rap.CollectAnnotationEnrichments</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_genome_command_line_program.html#a10e8c8b90dd0a384d3140710e7341051">broad.pda.seq.rap.GenomeCommandLineProgram</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_genome_scoring_program.html#a87fc28eeebb27b9b4ef0b2a9caa1cdf1">broad.pda.seq.rap.GenomeScoringProgram</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1rna_1_1_transcriptome_command_line_program.html#a835e3f71545e30ebe4e260f9ee846b1a">broad.pda.seq.rap.rna.TranscriptomeCommandLineProgram</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1rna_1_1_transcriptome_scoring_program.html#af631b5639f16680e19e7e3cceb1e909f">broad.pda.seq.rap.rna.TranscriptomeScoringProgram</a>
</li>
<li>CUTOFF
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_ratio_permutation_peak_caller.html#a4c6cfaac4d6b4f8ef4c76dcf8872319e">broad.pda.seq.rap.RatioPermutationPeakCaller</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_skellam_k_l_peak_caller.html#ae4c43ea4fba902de8ff4c8c0cc6b0ec0">broad.pda.seq.rap.SkellamKLPeakCaller</a>
</li>
<li>CV()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a5970adb73f2d1e0fd26b3334c982bbda">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
</ul>
</div><!-- contents -->
<!-- start footer part -->
<hr class="footer"/><address class="footer"><small>
Generated on Fri Aug 15 2014 15:11:16 for ESAT by &#160;<a href="http://www.doxygen.org/index.html">
<img class="footer" src="doxygen.png" alt="doxygen"/>
</a> 1.8.7
</small></address>
</body>
</html>
