<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=9"/>
<meta name="generator" content="Doxygen 1.8.7"/>
<title>ESAT: Class Members - Functions</title>
<link href="tabs.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="jquery.js"></script>
<script type="text/javascript" src="dynsections.js"></script>
<link href="search/search.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="search/search.js"></script>
<script type="text/javascript">
  $(document).ready(function() { searchBox.OnSelectItem(0); });
</script>
<link href="doxygen.css" rel="stylesheet" type="text/css" />
</head>
<body>
<div id="top"><!-- do not remove this div, it is closed by doxygen! -->
<div id="titlearea">
<table cellspacing="0" cellpadding="0">
 <tbody>
 <tr style="height: 56px;">
  <td style="padding-left: 0.5em;">
   <div id="projectname">ESAT
   </div>
  </td>
 </tr>
 </tbody>
</table>
</div>
<!-- end header part -->
<!-- Generated by Doxygen 1.8.7 -->
<script type="text/javascript">
var searchBox = new SearchBox("searchBox", "search",false,'Search');
</script>
  <div id="navrow1" class="tabs">
    <ul class="tablist">
      <li><a href="index.html"><span>Main&#160;Page</span></a></li>
      <li><a href="pages.html"><span>Related&#160;Pages</span></a></li>
      <li><a href="namespaces.html"><span>Packages</span></a></li>
      <li class="current"><a href="annotated.html"><span>Classes</span></a></li>
      <li><a href="files.html"><span>Files</span></a></li>
      <li>
        <div id="MSearchBox" class="MSearchBoxInactive">
        <span class="left">
          <img id="MSearchSelect" src="search/mag_sel.png"
               onmouseover="return searchBox.OnSearchSelectShow()"
               onmouseout="return searchBox.OnSearchSelectHide()"
               alt=""/>
          <input type="text" id="MSearchField" value="Search" accesskey="S"
               onfocus="searchBox.OnSearchFieldFocus(true)" 
               onblur="searchBox.OnSearchFieldFocus(false)" 
               onkeyup="searchBox.OnSearchFieldChange(event)"/>
          </span><span class="right">
            <a id="MSearchClose" href="javascript:searchBox.CloseResultsWindow()"><img id="MSearchCloseImg" border="0" src="search/close.png" alt=""/></a>
          </span>
        </div>
      </li>
    </ul>
  </div>
  <div id="navrow2" class="tabs2">
    <ul class="tablist">
      <li><a href="annotated.html"><span>Class&#160;List</span></a></li>
      <li><a href="classes.html"><span>Class&#160;Index</span></a></li>
      <li><a href="inherits.html"><span>Class&#160;Hierarchy</span></a></li>
      <li class="current"><a href="functions.html"><span>Class&#160;Members</span></a></li>
    </ul>
  </div>
  <div id="navrow3" class="tabs2">
    <ul class="tablist">
      <li><a href="functions.html"><span>All</span></a></li>
      <li class="current"><a href="functions_func.html"><span>Functions</span></a></li>
      <li><a href="functions_vars.html"><span>Variables</span></a></li>
      <li><a href="functions_eval.html"><span>Enumerator</span></a></li>
    </ul>
  </div>
  <div id="navrow4" class="tabs3">
    <ul class="tablist">
      <li><a href="functions_func.html#index_a"><span>a</span></a></li>
      <li><a href="functions_func_b.html#index_b"><span>b</span></a></li>
      <li><a href="functions_func_c.html#index_c"><span>c</span></a></li>
      <li><a href="functions_func_d.html#index_d"><span>d</span></a></li>
      <li><a href="functions_func_e.html#index_e"><span>e</span></a></li>
      <li><a href="functions_func_f.html#index_f"><span>f</span></a></li>
      <li><a href="functions_func_g.html#index_g"><span>g</span></a></li>
      <li><a href="functions_func_h.html#index_h"><span>h</span></a></li>
      <li><a href="functions_func_i.html#index_i"><span>i</span></a></li>
      <li><a href="functions_func_j.html#index_j"><span>j</span></a></li>
      <li><a href="functions_func_k.html#index_k"><span>k</span></a></li>
      <li><a href="functions_func_l.html#index_l"><span>l</span></a></li>
      <li class="current"><a href="functions_func_m.html#index_m"><span>m</span></a></li>
      <li><a href="functions_func_n.html#index_n"><span>n</span></a></li>
      <li><a href="functions_func_o.html#index_o"><span>o</span></a></li>
      <li><a href="functions_func_p.html#index_p"><span>p</span></a></li>
      <li><a href="functions_func_q.html#index_q"><span>q</span></a></li>
      <li><a href="functions_func_r.html#index_r"><span>r</span></a></li>
      <li><a href="functions_func_s.html#index_s"><span>s</span></a></li>
      <li><a href="functions_func_t.html#index_t"><span>t</span></a></li>
      <li><a href="functions_func_u.html#index_u"><span>u</span></a></li>
      <li><a href="functions_func_v.html#index_v"><span>v</span></a></li>
      <li><a href="functions_func_w.html#index_w"><span>w</span></a></li>
      <li><a href="functions_func_z.html#index_z"><span>z</span></a></li>
    </ul>
  </div>
</div><!-- top -->
<!-- window showing the filter options -->
<div id="MSearchSelectWindow"
     onmouseover="return searchBox.OnSearchSelectShow()"
     onmouseout="return searchBox.OnSearchSelectHide()"
     onkeydown="return searchBox.OnSearchSelectKey(event)">
<a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(0)"><span class="SelectionMark">&#160;</span>All</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(1)"><span class="SelectionMark">&#160;</span>Classes</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(2)"><span class="SelectionMark">&#160;</span>Namespaces</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(3)"><span class="SelectionMark">&#160;</span>Files</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(4)"><span class="SelectionMark">&#160;</span>Functions</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(5)"><span class="SelectionMark">&#160;</span>Variables</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(6)"><span class="SelectionMark">&#160;</span>Enumerator</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(7)"><span class="SelectionMark">&#160;</span>Pages</a></div>

<!-- iframe showing the search results (closed by default) -->
<div id="MSearchResultsWindow">
<iframe src="javascript:void(0)" frameborder="0" 
        name="MSearchResults" id="MSearchResults">
</iframe>
</div>

<div class="contents">
&#160;

<h3><a class="anchor" id="index_m"></a>- m -</h3><ul>
<li>MAD()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_statistics.html#aac411bc30e481714437119453e8c39b8">broad.core.math.Statistics</a>
</li>
<li>MAFAlignment()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_alignment.html#a226c288c376c8b47482d6ba6b60312a0">broad.core.multiplealignment.MAFAlignment</a>
</li>
<li>MAFIO()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_i_o.html#a5faaf6458bab8337b93511919f83724f">broad.core.multiplealignment.MAFIO</a>
</li>
<li>main()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_utils.html#ad54a460b148c5517dd615c7d1547538e">broad.core.annotation.AnnotationUtils</a>
, <a class="el" href="classbroad_1_1core_1_1annotation_1_1_maximum_contiguous_subsequence.html#a1762dd14d94c2702312ff3414a913153">broad.core.annotation.MaximumContiguousSubsequence</a>
, <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_header_utils.html#a4806083e1522a9664004ce2e4f1aea06">broad.core.datastructures.MatrixWithHeaderUtils</a>
, <a class="el" href="classbroad_1_1core_1_1math_1_1_a_n_o_v_a.html#a059aca814f49944efc77f329a488cef5">broad.core.math.ANOVA</a>
, <a class="el" href="classbroad_1_1core_1_1math_1_1_combination_generator.html#aac6651a64c2fc56baeae84f078fb6232">broad.core.math.CombinationGenerator</a>
, <a class="el" href="classbroad_1_1core_1_1math_1_1_empirical_distribution.html#ad8df7d1fa6fabdedac5fd2ce3a2b9bd8">broad.core.math.EmpiricalDistribution</a>
, <a class="el" href="classbroad_1_1core_1_1math_1_1_f_d_r_distribution.html#a794d06b3bd991c3a7f4205e70cac7fa4">broad.core.math.FDRDistribution</a>
, <a class="el" href="classbroad_1_1core_1_1math_1_1_loess.html#af04e63ac43128a7fcff226eccb6e095a">broad.core.math.Loess</a>
, <a class="el" href="classbroad_1_1core_1_1math_1_1_permutation_distribution.html#a756894b7aa364ed1fd63c44dd315fe13">broad.core.math.PermutationDistribution</a>
, <a class="el" href="classbroad_1_1core_1_1math_1_1_statistics.html#a580590df3a1dd3da73120dee1baa88e6">broad.core.math.Statistics</a>
, <a class="el" href="classbroad_1_1core_1_1motif_1_1misc_1_1_compute_conserved_motifs.html#a1220c0081aaf63c12538ba9d2ae9ffc2">broad.core.motif.misc.ComputeConservedMotifs</a>
, <a class="el" href="classbroad_1_1core_1_1motif_1_1misc_1_1_score_motif_instances.html#abbc985d2c658fd586e8bb2d5732b994c">broad.core.motif.misc.ScoreMotifInstances</a>
, <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix_i_o.html#a3bc6ec02c2c8dc9f665d1b3ec1083651">broad.core.motif.PositionWeightMatrixIO</a>
, <a class="el" href="classbroad_1_1core_1_1motif_1_1_p_w_m_utils.html#a17646e545879c7d2061fbc4cad0bf834">broad.core.motif.PWMUtils</a>
, <a class="el" href="classbroad_1_1core_1_1motif_1_1_sequence_motif.html#a6e112f9e21d5579de3ffa27cc89c8cde">broad.core.motif.SequenceMotif</a>
, <a class="el" href="classbroad_1_1core_1_1motif_1_1_sequence_motif_cluster.html#a2c060687fcab431e1aa54196582ead6b">broad.core.motif.SequenceMotifCluster</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_alignment_extractor.html#a48d2b8b5d5fdbcc0caae3d2e29e049f2">broad.core.multiplealignment.AlignmentExtractor</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_alignment.html#ad326b9ee4d9a163c416a0220a64ebfd5">broad.core.multiplealignment.MAFAlignment</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment_factory.html#a2b51e98b62505bde099c60d1a0e77427">broad.core.multiplealignment.MultipleAlignmentFactory</a>
, <a class="el" href="classbroad_1_1core_1_1overlaputils_1_1_gene_set_intersect.html#a033cf8b0d21c5945f316db8edbf9d1e7">broad.core.overlaputils.GeneSetIntersect</a>
, <a class="el" href="classbroad_1_1core_1_1primer3_1_1_compute_m_i_r_score.html#ae43bbdb355de0144eec67b7a62206f12">broad.core.primer3.ComputeMIRScore</a>
, <a class="el" href="classbroad_1_1core_1_1primer3_1_1is_p_c_r_like.html#ab5392a951ae1926097ad0666c22572b3">broad.core.primer3.isPCRLike</a>
, <a class="el" href="classbroad_1_1core_1_1primer3_1_1_primer_utils.html#a8eeda5cd71eaa1d3db14a83e4038dd23">broad.core.primer3.PrimerUtils</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_extractor.html#ad428faf3d9d13189401ea5ec82d2b2fd">broad.core.sequence.Extractor</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_fasta_quality_index.html#ac885c7baf5be7b8129380877d6d2c7e2">broad.core.sequence.FastaQualityIndex</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_fasta_sequence_i_o.html#aa49fdc72d0f25d8fb26868672544d649">broad.core.sequence.FastaSequenceIO</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_transcribed_regions.html#a4792cac14f26c00b690a0a93a197f3c4">broad.core.sequence.TranscribedRegions</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_transcribed_sequence_composition.html#ac14fe8aa4e8d4daa8429cef414e654a8">broad.core.sequence.TranscribedSequenceComposition</a>
, <a class="el" href="classbroad_1_1core_1_1siphy_1_1_pi_h_m_m.html#a95f6a006e38d7346c711d78afdec2c12">broad.core.siphy.PiHMM</a>
, <a class="el" href="classbroad_1_1core_1_1siphy_1_1tools_1_1conservation_1_1_average_branch_length.html#abf778b26ae852c4da00de07b21d5be05">broad.core.siphy.tools.conservation.AverageBranchLength</a>
, <a class="el" href="classbroad_1_1core_1_1siphy_1_1tools_1_1conservation_1_1_batch_submit_estimate_exon.html#afc2ff32874addd91302bce86915d5bc4">broad.core.siphy.tools.conservation.BatchSubmitEstimateExon</a>
, <a class="el" href="classbroad_1_1core_1_1siphy_1_1tools_1_1conservation_1_1_compute_kmers_for_region.html#adf612a094f26b61397f7750cd5645a44">broad.core.siphy.tools.conservation.ComputeKmersForRegion</a>
, <a class="el" href="classbroad_1_1core_1_1siphy_1_1tools_1_1conservation_1_1_compute_max_percentile_of_elements.html#a505e4a564db2f7eb6207d49594e6bea4">broad.core.siphy.tools.conservation.ComputeMaxPercentileOfElements</a>
, <a class="el" href="classbroad_1_1core_1_1siphy_1_1tools_1_1conservation_1_1_compute_percentage_of_kmers.html#a00d92409a73035d7ea18563e36763b2a">broad.core.siphy.tools.conservation.ComputePercentageOfKmers</a>
, <a class="el" href="classbroad_1_1core_1_1siphy_1_1tools_1_1conservation_1_1_estimate_omega_per_exon.html#a02888701d6e68016d609ee5570a1d7b8">broad.core.siphy.tools.conservation.EstimateOmegaPerExon</a>
, <a class="el" href="classbroad_1_1core_1_1siphy_1_1tools_1_1conservation_1_1_split_file.html#acb9120a544a8787808032eb8411cbc01">broad.core.siphy.tools.conservation.SplitFile</a>
, <a class="el" href="classbroad_1_1core_1_1siphy_1_1_tree_scaler.html#a036762852612940632c8cbaa824157be">broad.core.siphy.TreeScaler</a>
, <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a378ef26bd5ca3d6e143774ca50a9df9a">broad.pda.annotation.BEDFileParser</a>
, <a class="el" href="classbroad_1_1pda_1_1countreads_1_1_annotation_counts.html#a0218f236ebcc2485aa048b38acec5b79">broad.pda.countreads.AnnotationCounts</a>
, <a class="el" href="classbroad_1_1pda_1_1countreads_1_1_fastq_library_stats.html#aeb8b21c540228a98bd7a41bba2a8ec99">broad.pda.countreads.FastqLibraryStats</a>
, <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_batch_norm.html#aaf398d666513fa075cded04291e2b0fa">broad.pda.differentialExpression.BatchNorm</a>
, <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_cis_vs_trans.html#a15bca405efb1258899d23ff59fa94011">broad.pda.differentialExpression.CisVsTrans</a>
, <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_collapse_projections.html#afd9bba513f575c24b9981cf04d6c9256">broad.pda.differentialExpression.CollapseProjections</a>
, <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_convert_to_discrete_matrix.html#a1f9a35c5825119896f3ff4918983ea93">broad.pda.differentialExpression.ConvertToDiscreteMatrix</a>
, <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_fixed_control_differential_expression.html#a0a0018ae22cc989bf1a34659700ffef5">broad.pda.differentialExpression.FixedControlDifferentialExpression</a>
, <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_fold_significance.html#ab0fcd2638e917d36fd6284cb61af2075">broad.pda.differentialExpression.FoldSignificance</a>
, <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_grab_probes.html#adf43cdb478c38d34edb912ae2d67090b">broad.pda.differentialExpression.GrabProbes</a>
, <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_g_s_a_projection.html#a153db78475d8411a07b9ad6fa70459f1">broad.pda.differentialExpression.GSAProjection</a>
, <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_make_positional_gene_sets.html#aff9e7f8580c2c6201db83112380d40ad">broad.pda.differentialExpression.MakePositionalGeneSets</a>
, <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_merge_g_s_e_a_output_into_matrix.html#a183077a2fed3eb1db3dd248cd8bea9b1">broad.pda.differentialExpression.MergeGSEAOutputIntoMatrix</a>
, <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_run_all_g_s_e_a_batch.html#a393c772142bafd80949d883a80243b71">broad.pda.differentialExpression.RunAllGSEABatch</a>
, <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_run_differential_expression.html#a588242a1bb9d0f75a6349cd664892c73">broad.pda.differentialExpression.RunDifferentialExpression</a>
, <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_run_g_s_e_a.html#a4001a06dc0523f7baec455f3a44719f8">broad.pda.differentialExpression.RunGSEA</a>
, <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_run_time_series.html#abe8f9d65ca420bd046ff2d25c98f1ee4">broad.pda.differentialExpression.RunTimeSeries</a>
, <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_split_by_name.html#a83af777ccae0b3a6e67a0c3785e59f87">broad.pda.differentialExpression.SplitByName</a>
, <a class="el" href="classbroad_1_1pda_1_1gene_1_1functional_annotation.html#a7669ae458c2aa99b85c07fd0538a7e58">broad.pda.gene.functionalAnnotation</a>
, <a class="el" href="classbroad_1_1pda_1_1gene_1_1_gene_tools.html#aa133325339cd1c1f462c63c79eb16614">broad.pda.gene.GeneTools</a>
, <a class="el" href="classbroad_1_1pda_1_1gene_1_1_neighbor_analysis.html#a6b06c50aaa94140f80b08a993b7a7054">broad.pda.gene.NeighborAnalysis</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_to_g_c_t.html#a19a14508d238f8becfcac8455933973f">broad.pda.geneexpression.agilent.AgilentToGCT</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1analysis_1_1_process_dataset.html#aafc8e18995266e42bedf8bd4b1c176d1">broad.pda.geneexpression.agilent.analysis.ProcessDataset</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_classify_agilent_linc_probes.html#a87e352f7838c261d30e5505149ec6e6d">broad.pda.geneexpression.agilent.ClassifyAgilentLincProbes</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_collapse_probes_by_min_p_value.html#a2097b70bfb5c1764f7b303d9bf566b55">broad.pda.geneexpression.agilent.CollapseProbesByMinPValue</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_correlation_agilent_nanostring.html#af8f0fd48b0e217aed76490f72df84a99">broad.pda.geneexpression.agilent.CorrelationAgilentNanostring</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_correlation_between_replicates.html#af04fcbd8d6dba821d3fcc2d2a4659d11">broad.pda.geneexpression.agilent.CorrelationBetweenReplicates</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_extract_samples_and_label.html#adfbddf9cd6c9ed062c8620f90b9a60f0">broad.pda.geneexpression.agilent.ExtractSamplesAndLabel</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_filter_matrix_by_min_cutoff.html#af2016ce5ca94836a608a7af64ab65fe8">broad.pda.geneexpression.agilent.FilterMatrixByMinCutoff</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_find_probes_for_gene_set.html#abb20acde160f3a192c745e9a935d9a51">broad.pda.geneexpression.agilent.FindProbesForGeneSet</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_make_c_l_s_files.html#ad35d2092bdab11fa00930058c569c93d">broad.pda.geneexpression.agilent.MakeCLSFiles</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_make_c_l_s_from_description.html#a15f144038a7673393c02908591a2f441">broad.pda.geneexpression.agilent.MakeCLSFromDescription</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_q_c_agilent_results.html#a79a517846de25b324dd6dec0e2ba8831">broad.pda.geneexpression.agilent.QCAgilentResults</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_compute_correlation_of_replicates.html#aca4d392c129c922d844cd2d916d7d8a2">broad.pda.geneexpression.clustering.ComputeCorrelationOfReplicates</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_hierarchical_clustering.html#a4c13521bea7f9bbc7e76a40b9e6865c8">broad.pda.geneexpression.clustering.HierarchicalClustering</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_rank_order.html#a30b1c8bc7dce9788e82908628bfa06ac">broad.pda.geneexpression.clustering.RankOrder</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1_compute_hypergeomtric_enrichment.html#a8ce5ecf302efdf1804f8a34110b041bd">broad.pda.geneexpression.ComputeHypergeomtricEnrichment</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1_count_nums.html#afe3473795aacfeea9d4fda61958f5106">broad.pda.geneexpression.CountNums</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1_filter_g_c_t_file.html#a5ab74b42aa51043336608e645c4faf8d">broad.pda.geneexpression.FilterGCTFile</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1gene_exp_utils.html#a977ddc9ccbb7c26de10de043cf199f78">broad.pda.geneexpression.geneExpUtils</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1_global_stats.html#af5084c20044cacb771295a27d5fd62b8">broad.pda.geneexpression.GlobalStats</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1_grab_specific_columns.html#ae3eb0d6884fe683c7804852e795e650c">broad.pda.geneexpression.GrabSpecificColumns</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1_pa_g_e_java.html#abec7bbb75bff6e228d4781dfc898432f">broad.pda.geneexpression.PaGEJava</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1_preprocess_data_set.html#a677c3cd2ea3a8708e07efcdff29a8680">broad.pda.geneexpression.PreprocessDataSet</a>
, <a class="el" href="classbroad_1_1pda_1_1rap_1_1misc_1_1_find_enriched_depleted_regions.html#ac3175e1af75f2f64d9ca6b678c2440dc">broad.pda.rap.misc.FindEnrichedDepletedRegions</a>
, <a class="el" href="classbroad_1_1pda_1_1rnai_1_1_extract_sequence.html#a64eae6b9137fc522b16e2710dde38db4">broad.pda.rnai.ExtractSequence</a>
, <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_filter_fastq_reads.html#aa564874aaaf58b3008c26590eab3caac">broad.pda.samtools.FilterFastqReads</a>
, <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_filter_paired_reads_by_m_a_q.html#aece7432eeead701f3d75880e45aa7933">broad.pda.samtools.FilterPairedReadsByMAQ</a>
, <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_paired_end_t_d_f.html#a5b1217df4783fb68edb1834de88f7c06">broad.pda.samtools.PairedEndTDF</a>
, <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_s_a_m_fragment_writer.html#a0c8dfc56d2eefdffc5c5073cce9d4adc">broad.pda.samtools.SAMFragmentWriter</a>
, <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_trim_fastq_reads.html#a9a45fa8ea46d4f2b546b20417c7b58f3">broad.pda.samtools.TrimFastqReads</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils.html#a50fe3d7fac421c85d50a737755922da6">broad.pda.seq.alignment.AlignmentUtils</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1chip_1_1_compute_enrichments.html#a66af445380fc238ba17b49be109f4bd7">broad.pda.seq.chip.ComputeEnrichments</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1chip_1_1_find_diffrential_enriched_regions.html#a858019a15b343539847303fda46c7ce9">broad.pda.seq.chip.FindDiffrentialEnrichedRegions</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1chip_1_1_peak_filter.html#ac81624ad94fd32de1e02c3756f37353c">broad.pda.seq.chip.PeakFilter</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_fasta_extend_peaks.html#a254578bf9ef7cc16e998aac6e8e41650">broad.pda.seq.clip.FastaExtendPeaks</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_pileup.html#adb48168b821ea257a6510ed8af7cd09a">broad.pda.seq.clip.Pileup</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_splitter_by_barcode.html#a25fa89de279fd99b719da59cfd89bce2">broad.pda.seq.fastq.FastqSplitterByBarcode</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_split_and_trim_index.html#affffe227b85f7bdeeefc726a6ed23977">broad.pda.seq.fastq.SplitAndTrimIndex</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_split_fastq_by_index.html#a366b68c09f3de3e48680f872495c1286">broad.pda.seq.fastq.SplitFastqByIndex</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_batched_multi_sample_scan_peak_caller.html#a67eab2bdc03b1ac2718fd5c3482af7c4">broad.pda.seq.protection.BatchedMultiSampleScanPeakCaller</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#addead88839c9c7cbc25017192076424e">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_read_simulator.html#ac1e42f4310de34d0ef6f81414e09bd9c">broad.pda.seq.protection.ReadSimulator</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_read_simulator2.html#a6f7810919b0b5fe028d3742764cf7bb0">broad.pda.seq.protection.ReadSimulator2</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_build_ratio_null_distribution.html#a993b3adaa01e6ca8236d95242b9f9482">broad.pda.seq.rap.BuildRatioNullDistribution</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_calculate_coverage_stats.html#acfaa1394e958eb60a3baa83db8b5a874">broad.pda.seq.rap.CalculateCoverageStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_collect_annotation_enrichments.html#a73d43a1211300cc802832977f7de8d5a">broad.pda.seq.rap.CollectAnnotationEnrichments</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_count_probes.html#a699da2067831002738a42f75e8684466">broad.pda.seq.rap.CountProbes</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_count_reads.html#a34343c95343e2c8295bc6ced344d27a1">broad.pda.seq.rap.CountReads</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_distance_to_b_e_d.html#afa76da5ddaf7d7a921ad7737ebd604cf">broad.pda.seq.rap.DistanceToBED</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_g_c_content.html#ad89e50f9e92d9bca5a593f6b5dd28bf8">broad.pda.seq.rap.GCContent</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_plot_aggregate_regions.html#a8a38014e6297abcc1a4a1f06068626f2">broad.pda.seq.rap.PlotAggregateRegions</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_r_a_p_peak_caller.html#ab15edd4dac77fd24e3c0124ee1cd4806">broad.pda.seq.rap.RAPPeakCaller</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_ratio_permutation_peak_caller.html#a9eae71bd208d09027028b53f4098a9df">broad.pda.seq.rap.RatioPermutationPeakCaller</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1rna_1_1_collect_rna_seq_metrics.html#a9016b9b5e6c5d19c2d0d658991c71962">broad.pda.seq.rap.rna.CollectRnaSeqMetrics</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1rna_1_1_count_reads.html#a1748fd683317f43decca98e0586fd046">broad.pda.seq.rap.rna.CountReads</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_shuffle_b_a_m.html#a1acd0f381e3a1478b3062f0edf03a8b0">broad.pda.seq.rap.ShuffleBAM</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_shuffle_b_e_d.html#a989b6ad78711c6085c9f7f8cdaf2204c">broad.pda.seq.rap.ShuffleBED</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_skellam_k_l_peak_caller.html#ae9e70dc2ea97a09395ca2e593ceee908">broad.pda.seq.rap.SkellamKLPeakCaller</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_slide_and_calculate.html#af094b5191043d4dfcbc334e846e6ce05">broad.pda.seq.rap.SlideAndCalculate</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_slide_and_count.html#a83db52ee201e85136f9c32328813b7c7">broad.pda.seq.rap.SlideAndCount</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_unmark_duplicates.html#ae32ac88c212bb9d42f4864b307580685">broad.pda.seq.rap.UnmarkDuplicates</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a8777cdf9066d2f8f1cdd818d9e728d69">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_global_splice_filter.html#a516f3c28151819cf9889ed3f6799e2ad">broad.pda.seq.segmentation.GlobalSpliceFilter</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_precompute_data_alignment_stats.html#aa14baef82753f2755d18332b4b7f186c">broad.pda.seq.segmentation.PrecomputeDataAlignmentStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1utils_1_1_reconstruction_comparer.html#a1e44a1ad1fe857c320525d92e128bed7">broad.pda.seq.segmentation.utils.ReconstructionComparer</a>
, <a class="el" href="classbroad_1_1pda_1_1tools_1_1_p_s_l_x_to_s_a_m.html#a1af5eeda8d2c15e4d051c3825d16093d">broad.pda.tools.PSLXToSAM</a>
, <a class="el" href="classbroad_1_1projection_1_1gsa_1_1_define_gene_signatures.html#a0ea85e571730d95560ddec7bf8177c18">broad.projection.gsa.DefineGeneSignatures</a>
, <a class="el" href="classbroad_1_1projection_1_1gsa_1_1_extract_genes.html#a95622f5f4a11944f457afc60aeb20ca0">broad.projection.gsa.ExtractGenes</a>
, <a class="el" href="classbroad_1_1projection_1_1gsa_1_1_g_s_a_scoring.html#ab594d2e7623ddae67ec238958dc8d067">broad.projection.gsa.GSAScoring</a>
, <a class="el" href="classbroad_1_1projection_1_1gsa_1_1_gsa_utils.html#ad1cc968b8a169b93ed8e61b616205fda">broad.projection.gsa.GsaUtils</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_feature_sequence_alignment.html#a09afca590c8f7c9a6adcb2289b911fa5">umms.core.alignment.FeatureSequenceAlignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_smatch_like.html#a1cf7c64bce53f135ba5900376e5780af">umms.core.alignment.SmatchLike</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a3d87cb62eec12f2643e3da6910e460a9">umms.core.annotation.BasicAnnotation</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ac8aef318ba79841fc69adfbe53ebeede">umms.core.model.JCSAlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1readers_1_1_paired_end_reader.html#a25032e390568784979e93728bdd69fcd">umms.core.readers.PairedEndReader</a>
, <a class="el" href="classumms_1_1core_1_1scripture_1_1_build_scripture_coordinate_space.html#a12f315d4724bfd045035db00ab450200">umms.core.scripture.BuildScriptureCoordinateSpace</a>
, <a class="el" href="classumms_1_1core_1_1scripture_1_1statistics_1_1_connect_disconnected_transcripts.html#af38763966e52886da840f9f3c625a82e">umms.core.scripture.statistics.ConnectDisconnectedTranscripts</a>
, <a class="el" href="classumms_1_1esat_1_1_collapsed_gene_lengths.html#a64640b77e2ba1e1530e27e1dfbdef78e">umms.esat.CollapsedGeneLengths</a>
, <a class="el" href="classumms_1_1esat_1_1_e_s_a_t.html#af92b8b50582e8db464efe59683a5b709">umms.esat.ESAT</a>
, <a class="el" href="classumms_1_1esat_1_1_e_s_a_t_distance_calculator.html#a0ce2ea3dc1e213a0f7fbd3c6dc558e21">umms.esat.ESATDistanceCalculator</a>
</li>
<li>makeBinnedIntervalModel()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_neighbors_null_model.html#a6597b985b4c7a6fe5be986f1d395fbf1">broad.pda.gene.NeighborsNullModel</a>
</li>
<li>MakeCLSFiles()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_make_c_l_s_files.html#acbbf79f2ea46abf7d3857b12b72b8722">broad.pda.geneexpression.agilent.MakeCLSFiles</a>
</li>
<li>MakeCLSFromDescription()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_make_c_l_s_from_description.html#abb7487e724fdeb8359d2c3460e21e7b3">broad.pda.geneexpression.agilent.MakeCLSFromDescription</a>
</li>
<li>makeDartThrowingModel()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_neighbors_null_model.html#a2134fae81a300acd93438baa11809dbe">broad.pda.gene.NeighborsNullModel</a>
</li>
<li>makeDataMatrix()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_utils.html#a750231ef289737258a8ce9d3e6daac54">broad.pda.geneexpression.agilent.AgilentUtils</a>
</li>
<li>makeGenes()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a520840342a23a85f92254df35ddd545c">broad.pda.annotation.BEDFileParser</a>
</li>
<li>makeIntervalTree()
: <a class="el" href="classbroad_1_1core_1_1util_1_1_collapse_by_intersection.html#aaa8e7c2f4714c9ce25ac41c142b7668b">broad.core.util.CollapseByIntersection</a>
</li>
<li>makeIntervalTreeFor3Prime()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#ac9a4b9c9231f57edaad2c6dc33107d0d">broad.pda.annotation.BEDFileParser</a>
</li>
<li>makeIntervalTreeForGeneExons()
: <a class="el" href="classbroad_1_1core_1_1util_1_1_collapse_by_intersection.html#a8ffcc7f3ad85416198a8588bfaa95604">broad.core.util.CollapseByIntersection</a>
</li>
<li>makeIntervalTreeForGenes()
: <a class="el" href="classbroad_1_1core_1_1util_1_1_collapse_by_intersection.html#a725975734775b4289addfab94e943870">broad.core.util.CollapseByIntersection</a>
</li>
<li>makeNaiveRandModel()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_neighbors_null_model.html#a133ed88074e18109890f5b91d37ffaf9">broad.pda.gene.NeighborsNullModel</a>
</li>
<li>makePairs()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_alignment_pair.html#aee8a344eb7c324f23c0b68685f5a3ebf">umms.core.alignment.AlignmentPair</a>
</li>
<li>MakePositionalGeneSets()
: <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_make_positional_gene_sets.html#aed4e8326f4513fb713f691b22a0fcefd">broad.pda.differentialExpression.MakePositionalGeneSets</a>
</li>
<li>makeRandTranscriptList()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1transcripts_null_model.html#aa726b43dff83e7c3c376c79925267f8d">broad.pda.gene.transcriptsNullModel</a>
</li>
<li>manhattanDistance()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_statistics.html#a11b6b83e982e8df2fdaec12be5407078">broad.core.math.Statistics</a>
</li>
<li>MannWhitney()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_mann_whitney.html#a3a06fe081ec04fc21396398f34c7ebdc">broad.core.math.MannWhitney</a>
</li>
<li>MannWhitneyPermutations()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_mann_whitney_permutations.html#a3c28899f88ac9fa037e0fef1c3df90fe">broad.core.math.MannWhitneyPermutations</a>
</li>
<li>mapChildToLargestParent()
: <a class="el" href="classumms_1_1core_1_1utils_1_1_annotation_utils.html#ac8303f589525602b8a1d740ba6b05b30">umms.core.utils.AnnotationUtils</a>
</li>
<li>mapChildToParents()
: <a class="el" href="classumms_1_1core_1_1utils_1_1_annotation_utils.html#ae306f1081d2c5548bf9c320952cd8f68">umms.core.utils.AnnotationUtils</a>
</li>
<li>mapGenesToOverlappers()
: <a class="el" href="classbroad_1_1core_1_1overlaputils_1_1_gene_set_intersect.html#a5d44a2686992401cbd3cf23b0a79f3cb">broad.core.overlaputils.GeneSetIntersect</a>
</li>
<li>mapParentsToChildren()
: <a class="el" href="classumms_1_1core_1_1utils_1_1_annotation_utils.html#aa83a2cd0521f94949b7feac6929438ed">umms.core.utils.AnnotationUtils</a>
</li>
<li>MappingQualityFilter()
: <a class="el" href="classumms_1_1core_1_1read_filters_1_1_mapping_quality_filter.html#a3d2ff3f84d629fcf5fc1b9b73d0e0b46">umms.core.readFilters.MappingQualityFilter</a>
</li>
<li>mapSamRecord()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_read_simulator.html#a0dad6ce668c3cd1fa469550a080a9f03">broad.pda.seq.protection.ReadSimulator</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_read_simulator2.html#a77952d9aaf8a80ac415a7f9f7e00d474">broad.pda.seq.protection.ReadSimulator2</a>
</li>
<li>MarkovModel()
: <a class="el" href="classbroad_1_1core_1_1hmm_1_1_markov_model_3_01_t_01_4.html#a079c48f25bf7da1f0aaf770c33c8165f">broad.core.hmm.MarkovModel&lt; T &gt;</a>
</li>
<li>mask()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#ab2de5da03cf12144aa625a7bdafdb8fb">broad.core.sequence.Sequence</a>
</li>
<li>MaskedGenomicSpace()
: <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_masked_genomic_space.html#acdb954e5aa8cacf86607b8a7e9a29cd4">umms.core.coordinatesystem.MaskedGenomicSpace</a>
</li>
<li>maskSoftmaskedRegions()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a2cbe31f56aa039cf1721466398346e1c">broad.core.sequence.Sequence</a>
</li>
<li>match()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix.html#a3b83d9fb74cd24fa5256f0e66af3d353">broad.core.motif.PositionWeightMatrix</a>
, <a class="el" href="classbroad_1_1core_1_1motif_1_1_sequence_motif.html#ae2940cf15beb430edcb54b937d0212a7">broad.core.motif.SequenceMotif</a>
, <a class="el" href="classbroad_1_1core_1_1motif_1_1_sequence_motif_cluster.html#a8ab9d3f1f5333f4978b8be5dc0ec1ab2">broad.core.motif.SequenceMotifCluster</a>
</li>
<li>matchStartOnFirstSequence()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_smith_waterman_alignment.html#aefe75b54dc89476479283e1f8060062f">umms.core.alignment.SmithWatermanAlignment</a>
</li>
<li>Matrix()
: <a class="el" href="classjaligner_1_1matrix_1_1_matrix.html#adcd6fde8b26088a4ba88d13bad4730db">jaligner.matrix.Matrix</a>
, <a class="el" href="class_jama_1_1_matrix.html#a877dfa361af400fbf817b1046856e239">Jama.Matrix</a>
</li>
<li>MatrixLoaderException()
: <a class="el" href="classjaligner_1_1matrix_1_1_matrix_loader_exception.html#a7d48683e2a49a527a4a15231aea73387">jaligner.matrix.MatrixLoaderException</a>
</li>
<li>MatrixWithHeaders()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a46b7fd9b72db3c4aa6620ab50be42a8a">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>max()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_interval_tree_3_01_v_01_4.html#a166ddc864c75c2ce9bd5459400db5a71">broad.core.datastructures.IntervalTree&lt; V &gt;</a>
, <a class="el" href="classbroad_1_1core_1_1math_1_1_math_util.html#a6912b62c2ce54a09cddc63ad57ba7af2">broad.core.math.MathUtil</a>
, <a class="el" href="classbroad_1_1core_1_1math_1_1_statistics.html#a395e7753ccacf0cb3f885b1e0c0d5341">broad.core.math.Statistics</a>
</li>
<li>maxInt()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_statistics.html#a88857f611a22b3dc9fae555ae6223091">broad.core.math.Statistics</a>
</li>
<li>maxMAD()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_statistics.html#a471cb6d54d7edd179ff6795b12bb8dc1">broad.core.math.Statistics</a>
</li>
<li>maxmean()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_statistics.html#a9129c80b84cba2df7c06ad61090df866">broad.core.math.Statistics</a>
</li>
<li>maxSelfComplementarity()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_hairpin_kmer.html#ada34732d5709df1afba5b4064bd0353a">broad.core.primer3.HairpinKmer</a>
</li>
<li>maxStdev()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_statistics.html#a136ef2aec8d18c2c958e473612fe8b2c">broad.core.math.Statistics</a>
</li>
<li>maxSubSum3()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_maximum_contiguous_subsequence.html#a739cb8ca66abe40ccc3dfdf8764cb920">broad.core.annotation.MaximumContiguousSubsequence</a>
</li>
<li>mayHaveBlocks()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_genomic_annotation.html#a9c77dd64d9f5aa93e5e5a41429581205">broad.core.annotation.BasicGenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1annotation_1_1_b_e_d.html#a95513f9450e6d7d79d9803d1fb0a8762">broad.core.annotation.BED</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_genomic_annotation.html#afb25832a9509f9057a9b7a4ade27fcd2">broad.core.annotation.GenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a3f26953d7f43ec42eafc0456e1896037">broad.core.sequence.SequenceRegion</a>
</li>
<li>mean()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_statistics.html#a6cee05563e8d7390baaf1248fddc8596">broad.core.math.Statistics</a>
</li>
<li>meanNormalizeColumns()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#affab3393a1f0d0a1ec0bbb0fbea64b02">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>meanNormalizeRows()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a4e396629c78164e18a30b0bd3046c19d">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>measure()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_absolute_pearson_function.html#abf100e179e21fa455390e115bd2c8a23">broad.pda.geneexpression.clustering.AbsolutePearsonFunction</a>
, <a class="el" href="interfacebroad_1_1pda_1_1geneexpression_1_1clustering_1_1_cluster_distance_function.html#a5ab124c5bb5838627ca0c2e538201900">broad.pda.geneexpression.clustering.ClusterDistanceFunction</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_euclidean_distance.html#a8f5f662d7cb01210cae193d652818452">broad.pda.geneexpression.clustering.EuclideanDistance</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_pearson_distance.html#af6c631dddd31058852afe3f536f8ee78">broad.pda.geneexpression.clustering.PearsonDistance</a>
</li>
<li>median()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_statistics.html#adb8cf9a05ac40be990320cdbd734935b">broad.core.math.Statistics</a>
, <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_batch_norm.html#acf3302ffab2b17cde49db2081a6824e9">broad.pda.differentialExpression.BatchNorm</a>
</li>
<li>medianCenterColumns()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#af34c92d09322c6a27fb48b1aeb43e2c6">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>medianCenterColumnsThis()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#ac0a2e8cc8c3c17a62fd24b87ab851fbe">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>medianCenterRows()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a8b02d67ec694a0a8f61392fab9ea5bc5">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>medianCenterRowsThis()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#adc7b102b44ffdbe14d71f0cb8b351104">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>medianCollection()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_statistics.html#a1b5c105e0402bc9b1a7d70080c3bc65f">broad.core.math.Statistics</a>
</li>
<li>medianNormalizeColumns()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#acd8a0a2ae2ce4c4211d7bca7b6282bba">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>medianNormalizeRows()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#ac90f92db0b830870eef2aabba1f24f8d">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>MemeMotif()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1meme_1_1_meme_motif.html#a20a7df93fe14459e2764f7d60a135453">broad.core.motif.meme.MemeMotif</a>
</li>
<li>MemeResultReader()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1meme_1_1_meme_result_reader.html#ae99873159732c5ef66a305ca563f24d1">broad.core.motif.meme.MemeResultReader</a>
</li>
<li>memoryStats()
: <a class="el" href="classumms_1_1core_1_1scripture_1_1_build_scripture_coordinate_space.html#a132faf541275464d9bb333b6b7b6ad8b">umms.core.scripture.BuildScriptureCoordinateSpace</a>
</li>
<li>merge()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#a166f1db833577d0454a67ece8db9916e">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
, <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a75d88890a89e815e0dd41134c8eb29e1">broad.pda.annotation.BEDFileParser</a>
, <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_alignments.html#ad139a49c4b30bf32705834f0e79e2789">broad.pda.datastructures.Alignments</a>
, <a class="el" href="classbroad_1_1pda_1_1gene_1_1_gene_tools.html#a2685fed73a7ac2f3844e0e1bec280149">broad.pda.gene.GeneTools</a>
</li>
<li>mergeAllOverlappers()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#ab9c2cce66e01225f9f12dd3a1edde9da">broad.core.annotation.BasicLightweightAnnotation</a>
</li>
<li>mergeByGtfGeneId()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#af406253e663a33e6635ef5056d91221e">broad.pda.annotation.BEDFileParser</a>
, <a class="el" href="classbroad_1_1pda_1_1gene_1_1_gene_tools.html#aee194b8ea30b7cfb58c76bf8beb898c4">broad.pda.gene.GeneTools</a>
</li>
<li>MergeGSEAOutputIntoMatrix()
: <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_merge_g_s_e_a_output_into_matrix.html#a97967d31ddc20639ebe3476801dc866c">broad.pda.differentialExpression.MergeGSEAOutputIntoMatrix</a>
</li>
<li>mergeOverlappingBlocks()
: <a class="el" href="classumms_1_1core_1_1utils_1_1_annotation_utils.html#afa2cbf048459a0670cccff99e9140b6e">umms.core.utils.AnnotationUtils</a>
</li>
<li>mergeOverlappingBlocksSingleOrientation()
: <a class="el" href="classumms_1_1core_1_1utils_1_1_annotation_utils.html#a73c090856164c0c8917b1dad326e515f">umms.core.utils.AnnotationUtils</a>
</li>
<li>mergePeaks()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#aba7af446c1a79202b035afa54bbcc5f5">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>mergeRecords()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_fasta_sequence_i_o.html#af154ff4095e0c7d558b4259bf3752b45">broad.core.sequence.FastaSequenceIO</a>
</li>
<li>mergeSets()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_gene_tools.html#a95b3b562d7d5711676b4cb8aeff56002">broad.pda.gene.GeneTools</a>
</li>
<li>metaAvg()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_statistics.html#a91a5391560b7344a6706d10be8284522">broad.core.math.Statistics</a>
</li>
<li>min()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_interval_tree_3_01_v_01_4.html#a2bc0b97c96c61daeb3c731e6a9864d42">broad.core.datastructures.IntervalTree&lt; V &gt;</a>
, <a class="el" href="classbroad_1_1core_1_1math_1_1_statistics.html#aeb09842c802c009c825df1a0d9395d18">broad.core.math.Statistics</a>
</li>
<li>minInt()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_statistics.html#a725a0ee05b58170f99a60a987a0ee857">broad.core.math.Statistics</a>
</li>
<li>minOverlapper()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_interval_tree_3_01_v_01_4.html#a7793c8a726674cf9cde66f3f8284380a">broad.core.datastructures.IntervalTree&lt; V &gt;</a>
</li>
<li>minus()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#a2b33aa194c4242e1434063c7e5168236">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
, <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#aeffec2cccaeb4cf7b2989c123a756b5a">broad.core.datastructures.MatrixWithHeaders</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#ac95fe5b07675c3796fcdcd971c8cd3c6">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a3c26637d96247aa07ca8fcf6bade2636">broad.pda.annotation.BEDFileParser</a>
, <a class="el" href="class_jama_1_1_matrix.html#a73fe15c75990210496725df14d7c1aad">Jama.Matrix</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#a3b98ec0dfc3dc2c0a78c9a7296e8a733">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4.html#aa210f5981e6f17b0c82501eb726bd343">umms.core.annotation.AnnotationList&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a065db388244cee9269d0ee03a7c76f42">umms.core.annotation.BasicAnnotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_compound_interval.html#a0df0ec9330ced43d0677b39e54a69a98">umms.core.annotation.CompoundInterval</a>
</li>
<li>minusEquals()
: <a class="el" href="class_jama_1_1_matrix.html#a665bb3ba66a0f8fc36b57cc9a3b86fe3">Jama.Matrix</a>
</li>
<li>moveAnnotation()
: <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcriptome_space.html#aab8d4e9081568504dda0d062bebd2fe7">umms.core.coordinatesystem.TranscriptomeSpace</a>
</li>
<li>moveToCoordinate()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#af53a0a8c5f18227b65363f16116a677b">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#af4cbbce6715e864c1155fd105d1d853d">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#aac56df2b60a55cc3d7615ec372584462">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a2e654b6c118a9b3cf6b728737f9a1b47">umms.core.annotation.BasicAnnotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_compound_interval.html#acd2a18ea0afd561b2d15974f7e5821ae">umms.core.annotation.CompoundInterval</a>
</li>
<li>MultipleAlignment()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#a0dba07bb1f121c7cf717509a1eddbc90">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>multiplyColumnsWithConstants()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a276a7e36e9c0939a64033ba0ff1a5e09">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>MultiSampleScanPeakCaller()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#a6d6c011e7c546d363f85ac20f4ce00f7">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>MultiScore()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#aa7db2dce5a45e4e40063bdfd91b60897">umms.core.model.score.MultiScore</a>
</li>
</ul>
</div><!-- contents -->
<!-- start footer part -->
<hr class="footer"/><address class="footer"><small>
Generated on Fri Aug 15 2014 15:11:17 for ESAT by &#160;<a href="http://www.doxygen.org/index.html">
<img class="footer" src="doxygen.png" alt="doxygen"/>
</a> 1.8.7
</small></address>
</body>
</html>
