<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=9"/>
<meta name="generator" content="Doxygen 1.8.7"/>
<title>ESAT: Class Members</title>
<link href="tabs.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="jquery.js"></script>
<script type="text/javascript" src="dynsections.js"></script>
<link href="search/search.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="search/search.js"></script>
<script type="text/javascript">
  $(document).ready(function() { searchBox.OnSelectItem(0); });
</script>
<link href="doxygen.css" rel="stylesheet" type="text/css" />
</head>
<body>
<div id="top"><!-- do not remove this div, it is closed by doxygen! -->
<div id="titlearea">
<table cellspacing="0" cellpadding="0">
 <tbody>
 <tr style="height: 56px;">
  <td style="padding-left: 0.5em;">
   <div id="projectname">ESAT
   </div>
  </td>
 </tr>
 </tbody>
</table>
</div>
<!-- end header part -->
<!-- Generated by Doxygen 1.8.7 -->
<script type="text/javascript">
var searchBox = new SearchBox("searchBox", "search",false,'Search');
</script>
  <div id="navrow1" class="tabs">
    <ul class="tablist">
      <li><a href="index.html"><span>Main&#160;Page</span></a></li>
      <li><a href="pages.html"><span>Related&#160;Pages</span></a></li>
      <li><a href="namespaces.html"><span>Packages</span></a></li>
      <li class="current"><a href="annotated.html"><span>Classes</span></a></li>
      <li><a href="files.html"><span>Files</span></a></li>
      <li>
        <div id="MSearchBox" class="MSearchBoxInactive">
        <span class="left">
          <img id="MSearchSelect" src="search/mag_sel.png"
               onmouseover="return searchBox.OnSearchSelectShow()"
               onmouseout="return searchBox.OnSearchSelectHide()"
               alt=""/>
          <input type="text" id="MSearchField" value="Search" accesskey="S"
               onfocus="searchBox.OnSearchFieldFocus(true)" 
               onblur="searchBox.OnSearchFieldFocus(false)" 
               onkeyup="searchBox.OnSearchFieldChange(event)"/>
          </span><span class="right">
            <a id="MSearchClose" href="javascript:searchBox.CloseResultsWindow()"><img id="MSearchCloseImg" border="0" src="search/close.png" alt=""/></a>
          </span>
        </div>
      </li>
    </ul>
  </div>
  <div id="navrow2" class="tabs2">
    <ul class="tablist">
      <li><a href="annotated.html"><span>Class&#160;List</span></a></li>
      <li><a href="classes.html"><span>Class&#160;Index</span></a></li>
      <li><a href="inherits.html"><span>Class&#160;Hierarchy</span></a></li>
      <li class="current"><a href="functions.html"><span>Class&#160;Members</span></a></li>
    </ul>
  </div>
  <div id="navrow3" class="tabs2">
    <ul class="tablist">
      <li class="current"><a href="functions.html"><span>All</span></a></li>
      <li><a href="functions_func.html"><span>Functions</span></a></li>
      <li><a href="functions_vars.html"><span>Variables</span></a></li>
      <li><a href="functions_eval.html"><span>Enumerator</span></a></li>
    </ul>
  </div>
  <div id="navrow4" class="tabs3">
    <ul class="tablist">
      <li><a href="functions.html#index_a"><span>a</span></a></li>
      <li><a href="functions_b.html#index_b"><span>b</span></a></li>
      <li><a href="functions_c.html#index_c"><span>c</span></a></li>
      <li><a href="functions_d.html#index_d"><span>d</span></a></li>
      <li><a href="functions_e.html#index_e"><span>e</span></a></li>
      <li><a href="functions_f.html#index_f"><span>f</span></a></li>
      <li><a href="functions_g.html#index_g"><span>g</span></a></li>
      <li><a href="functions_h.html#index_h"><span>h</span></a></li>
      <li><a href="functions_i.html#index_i"><span>i</span></a></li>
      <li><a href="functions_j.html#index_j"><span>j</span></a></li>
      <li><a href="functions_k.html#index_k"><span>k</span></a></li>
      <li><a href="functions_l.html#index_l"><span>l</span></a></li>
      <li><a href="functions_m.html#index_m"><span>m</span></a></li>
      <li><a href="functions_n.html#index_n"><span>n</span></a></li>
      <li><a href="functions_o.html#index_o"><span>o</span></a></li>
      <li><a href="functions_p.html#index_p"><span>p</span></a></li>
      <li><a href="functions_q.html#index_q"><span>q</span></a></li>
      <li><a href="functions_r.html#index_r"><span>r</span></a></li>
      <li class="current"><a href="functions_s.html#index_s"><span>s</span></a></li>
      <li><a href="functions_t.html#index_t"><span>t</span></a></li>
      <li><a href="functions_u.html#index_u"><span>u</span></a></li>
      <li><a href="functions_v.html#index_v"><span>v</span></a></li>
      <li><a href="functions_w.html#index_w"><span>w</span></a></li>
      <li><a href="functions_z.html#index_z"><span>z</span></a></li>
    </ul>
  </div>
</div><!-- top -->
<!-- window showing the filter options -->
<div id="MSearchSelectWindow"
     onmouseover="return searchBox.OnSearchSelectShow()"
     onmouseout="return searchBox.OnSearchSelectHide()"
     onkeydown="return searchBox.OnSearchSelectKey(event)">
<a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(0)"><span class="SelectionMark">&#160;</span>All</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(1)"><span class="SelectionMark">&#160;</span>Classes</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(2)"><span class="SelectionMark">&#160;</span>Namespaces</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(3)"><span class="SelectionMark">&#160;</span>Files</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(4)"><span class="SelectionMark">&#160;</span>Functions</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(5)"><span class="SelectionMark">&#160;</span>Variables</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(6)"><span class="SelectionMark">&#160;</span>Enumerator</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(7)"><span class="SelectionMark">&#160;</span>Pages</a></div>

<!-- iframe showing the search results (closed by default) -->
<div id="MSearchResultsWindow">
<iframe src="javascript:void(0)" frameborder="0" 
        name="MSearchResults" id="MSearchResults">
</iframe>
</div>

<div class="contents">
<div class="textblock">Here is a list of all class members with links to the classes they belong to:</div>

<h3><a class="anchor" id="index_s"></a>- s -</h3><ul>
<li>SameOrientationFilter()
: <a class="el" href="classumms_1_1core_1_1read_filters_1_1_same_orientation_filter.html#a1234bd65263e6024b20ba29388cf97cc">umms.core.readFilters.SameOrientationFilter</a>
</li>
<li>SAMFormatFullBED()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils.html#a88b4e4f173dd90784c6fe117156773c6">broad.pda.seq.alignment.AlignmentUtils</a>
</li>
<li>SAMFragmentWriter()
: <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_s_a_m_fragment_writer.html#a1c44db1468f602e735d67529c0eebd7f">broad.pda.samtools.SAMFragmentWriter</a>
</li>
<li>SAMPairedEndFileReader()
: <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_s_a_m_paired_end_file_reader.html#aceab6b49722b6ea070e0e89dcc6e729f">broad.pda.samtools.SAMPairedEndFileReader</a>
</li>
<li>sample()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model.html#a26b4722f3613690bd8d65fd3302021a1">broad.core.siphy.EvolutionaryModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_window_scores.html#a10f405eec7644a33de7c07bb0c8e4caa">broad.pda.seq.protection.WindowScores</a>
</li>
<li>sampleColumns()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_alignment.html#a53c281eb887b5cb250224b1bb1ff25b5">broad.core.multiplealignment.MAFAlignment</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#a8b7d8e493f5c3ea5409d7c12f41c9237">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>SampleData()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#adcfd7ff578f7f31b52d759171e966003">broad.pda.seq.clip.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#acbea9965c1d9f8b120e1510e326c5b96">broad.pda.seq.protection.SampleData</a>
</li>
<li>sampleModel
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#a778cb534f4ecab1514d08f5c5dc47a47">umms.core.model.score.MultiScore</a>
</li>
<li>sampleName
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a7c4644dba1de73c9baf1743c65628617">broad.pda.seq.clip.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a5b0011ca04c2ee52ec32f8508b82ae99">broad.pda.seq.protection.SampleData</a>
</li>
<li>SAVED_MODEL_STATS_EXT
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_stored_alignment_stats.html#a900895cff3a06f322818dd8bf97f1193">broad.pda.seq.alignment.StoredAlignmentStats</a>
</li>
<li>scaleNorm()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a56a1092e224d5e5a09b46efef681d27f">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>scaleRegion()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_tree_scaler.html#a690046ea37206c916a3f3834fda6f203">broad.core.siphy.TreeScaler</a>
</li>
<li>scaleTree()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_tree_scaler.html#a6f7ecab5c4a2cddb25a3540d72ba5d27">broad.core.siphy.TreeScaler</a>
</li>
<li>scaleTreeWithSampling()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_tree_scaler.html#a385c075371190ba318182301e4b907fb">broad.core.siphy.TreeScaler</a>
</li>
<li>scan()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_conserved_p_w_m_scanner.html#a7d0bda7cc27e224befcf4d4f7653ba35">broad.core.siphy.ConservedPWMScanner</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a9055da2c6c85869da10e9bff9c0fbac9">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#ad5e4df7215f9fd180242b6d9b1b8a930">umms.core.annotation.AbstractAnnotationCollection&lt; T extends Annotation &gt;</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#a2120916c9e1d07c8a5acff623534eebc">umms.core.annotation.AnnotationCollection&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a3665a9f334771056e2661622fd756c68">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#a2086f9de68630c7104ab1d43fde8d27a">umms.core.model.JCSAlignmentModel</a>
</li>
<li>scanGenome()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a783abf082b1ebce3361a143d40764b04">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>scanPRate()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#accc819fd09f1e1c607e3b66aab3ae2e0">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#acde220ce2b79ad3a0cb0f0493eb52a2f">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>scanRegion()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_p_w_m_utils.html#a901bd6475f321d8416ddbf47d03d32b4">broad.core.motif.PWMUtils</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a20d56595ce0228db7c4a9ffce432c909">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#afdb0844bc8468e30a3a2b2cd55a13e62">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>ScanStatisticScore()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_scan_statistic_score.html#aa144af17481662d4c3e8b2e562b1b91b">umms.core.model.score.ScanStatisticScore</a>
</li>
<li>score()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_position_weight_matrix_model.html#ac2c476fbc2a41fc8408553365bf4e0bc">broad.core.siphy.PositionWeightMatrixModel</a>
</li>
<li>SCORE
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_genome_scoring_program.html#aa495683afdf659268bd442d9559a09ff">broad.pda.seq.rap.GenomeScoringProgram</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1rna_1_1_transcriptome_scoring_program.html#ae8b26a43c0791ea615a151ffeb6f1656">broad.pda.seq.rap.rna.TranscriptomeScoringProgram</a>
</li>
<li>scoreAnnotations()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_alignment_utils.html#a6c939ca293f7fbfd141d81c1534a8461">broad.pda.seq.alignment.AlignmentUtils</a>
</li>
<li>scoreGene()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a9bdf34088aadf81c3eabae435fe67767">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>scoreGenes()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#af1e5000926013a041e9597fb3a2dc5fe">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>scoreListByTStatistic()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1_score_diff_genes.html#a432bf7e83f4a9d004f876acfd8e892e6">broad.pda.geneexpression.ScoreDiffGenes</a>
</li>
<li>ScoreMotifInstances()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1misc_1_1_score_motif_instances.html#ab64429e0578bbb80408fac9c5359a830">broad.core.motif.misc.ScoreMotifInstances</a>
</li>
<li>scoreMotifs()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1misc_1_1_compute_conserved_motifs.html#a9b7d42e8aa1def6c7f8a7b48ecd8b7a5">broad.core.motif.misc.ComputeConservedMotifs</a>
</li>
<li>scorePaths()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a9e84cfdbc07c6b49f3e35b092726714f">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>scorePathsForOnlyRPKM()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#af620a01c0d3be891bdd277470553e666">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>scoreSegment()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a8cffd238bd6070c45d5bd0bb72b43aeb">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>scoreSegments()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a7009b0bc1a05ca28f02a06831778168b">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>scoreWindow()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#ac476ccc56fab96d17419127a8cf313a1">broad.pda.seq.protection.SampleData</a>
</li>
<li>scoreWindowWithFragmentLengthFilter()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a477fb361142dde07116598599bb7ef52">broad.pda.seq.protection.SampleData</a>
</li>
<li>scramble()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a0820a809e5bd46b6224a31d6dcff27b0">broad.core.sequence.Sequence</a>
</li>
<li>SearchException()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_search_exception.html#a535f7156a0d1bd0dd2fb8b40f6287602">broad.core.motif.SearchException</a>
</li>
<li>selectHighScoringIsoform()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_gene_tools.html#a4411d849943390917a369f9075cb660d">broad.pda.gene.GeneTools</a>
</li>
<li>SelectLongestIntronChainCandidate()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#a11d537e3a80eacc32bb221711bc03dfc">broad.pda.annotation.Locus</a>
</li>
<li>selectRandIsoSubset()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#a5b93959c80e50de0f2dd758c382a74d2">broad.pda.annotation.Locus</a>
, <a class="el" href="classbroad_1_1pda_1_1gene_1_1_gene_with_isoforms.html#a81ed0377315d9630a740515935f554ef">broad.pda.gene.GeneWithIsoforms</a>
</li>
<li>SELF
: <a class="el" href="enumbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t_1_1_edge_source_type.html#ab7b2a9616a3fd5b5b93179d40a94afbf">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT.EdgeSourceType</a>
</li>
<li>SEQ_ID_REC_LENGTH
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_fasta_quality_binary_i_o.html#a3c8ecdb5a93f86f2a1cff925a3603660">broad.core.sequence.FastaQualityBinaryIO</a>
</li>
<li>Sequence()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a3f5ed7a6ae3513ae7eb2bae8789a6f10">broad.core.sequence.Sequence</a>
, <a class="el" href="classjaligner_1_1_sequence.html#a5bb42ea5e75b4fec86bf027e34a1b327">jaligner.Sequence</a>
</li>
<li>sequenceDirectory
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_directory_installed_genome_assembly.html#a529d514f5acc335c587e3a0d86f13762">broad.pda.feature.genome.DirectoryInstalledGenomeAssembly</a>
</li>
<li>SequenceMotif()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_sequence_motif.html#a21d5829daae0736998246dd04054fe0f">broad.core.motif.SequenceMotif</a>
</li>
<li>SequenceMotifCluster()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_sequence_motif_cluster.html#abc01aba2efbdb2c70662910c3c1d86ec">broad.core.motif.SequenceMotifCluster</a>
</li>
<li>SequenceParserException()
: <a class="el" href="classjaligner_1_1util_1_1_sequence_parser_exception.html#aad169b1809166fb80d58cd98ee51236d">jaligner.util.SequenceParserException</a>
</li>
<li>SequenceRegion()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a0b18988250e79bed3b745d6b0172893c">broad.core.sequence.SequenceRegion</a>
</li>
<li>SequenceRegionWindow()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region_window.html#aca389aed2961d7e4f8bf08d45c533385">broad.core.sequence.SequenceRegionWindow</a>
</li>
<li>set()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a36c11d02fe2f922c84cedbec035e8ca3">broad.core.datastructures.MatrixWithHeaders</a>
, <a class="el" href="classjaligner_1_1_cell.html#a05f61e25a9e5e09f15831472817202e8">jaligner.Cell</a>
, <a class="el" href="class_jama_1_1_matrix.html#a46de3040517d1238ae802820acc3b2b5">Jama.Matrix</a>
</li>
<li>setA()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_two_subject_annotation.html#af87993e07bf3337abc59714861e7b85c">broad.core.annotation.BasicTwoSubjectAnnotation</a>
</li>
<li>setAddToReadName()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_splitter_by_barcode.html#a9128e093c39f027a4c8fa9b71df443c3">broad.pda.seq.fastq.FastqSplitterByBarcode</a>
</li>
<li>setAlignedSequences()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_alignment.html#a8275e92f099270ee94ce8682060b8474">broad.core.multiplealignment.MAFAlignment</a>
</li>
<li>setAlignment()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_tree_scaler.html#a957d50f0314cbdbea117d7bdb6595b01">broad.core.siphy.TreeScaler</a>
</li>
<li>setAlignmentChunkSize()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_conserved_p_w_m_scanner.html#a3f966e649abcf8c8215cd049493576e1">broad.core.siphy.ConservedPWMScanner</a>
</li>
<li>setAlignmentDataModel()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#aa2efb2a40faf2b836fc838c5e740836b">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
</li>
<li>setAlignPos()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix.html#a8817e48c123afc0c6c15a30686777a5e">broad.core.motif.PositionWeightMatrix</a>
</li>
<li>setAlpha()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a8c1d4aef55169c5f5cc897edec41c1ff">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>setAlphabetSize()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model.html#ac8fd7b6ca53f99772a505a5d62d77a10">broad.core.siphy.EvolutionaryModel</a>
</li>
<li>setAnnotationList()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_genomic_annotation_reader.html#a2878e5be828b109b925585ef2a23ee0e">broad.core.annotation.BasicGenomicAnnotationReader</a>
</li>
<li>setAsSafeToLoad()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_stored_alignment_stats.html#a1075082a881e90156fc22508ad3b3ff5">broad.pda.seq.alignment.StoredAlignmentStats</a>
</li>
<li>setAsUnsafeToLoad()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_stored_alignment_stats.html#a34b212862579dc340afdedcd9ce5dcb5">broad.pda.seq.alignment.StoredAlignmentStats</a>
</li>
<li>setAtEnd()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_splitter_by_barcode.html#a33b25a986c4cbcc33d87d383c072fc0e">broad.pda.seq.fastq.FastqSplitterByBarcode</a>
</li>
<li>setAttribute()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a4ebdcbbfbfd734c2298f015e4d8cfefb">umms.core.alignment.AbstractPairedEndAlignment</a>
</li>
<li>setAttributes()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_g_f_f.html#a896fe7c925e9513bd06cde675fdfa617">broad.core.annotation.GFF</a>
</li>
<li>setAvergeNumberOfReads()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score.html#afb91d7ac4cc367fecb0312242af171d1">broad.pda.seq.segmentation.GeneScore</a>
</li>
<li>setB()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_two_subject_annotation.html#a4dd0d729f5d02072503264a9e5b0bca7">broad.core.annotation.BasicTwoSubjectAnnotation</a>
</li>
<li>setBackgroundModel()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_conserved_p_w_m_scanner.html#a465fbb69d4ab9d5422016b52abea75cb">broad.core.siphy.ConservedPWMScanner</a>
</li>
<li>setBackgroundNucleotideFreqs()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model_parameters.html#aa189c422aece425c64314dd859a8d606">broad.core.siphy.EvolutionaryModelParameters</a>
</li>
<li>setBedScore()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_gene_with_isoforms.html#a29d8e059388d8c9aff14ebe5d9e66702">broad.pda.gene.GeneWithIsoforms</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a9f1ad94878c6e09f06a6a60e0797b64d">umms.core.annotation.Gene</a>
</li>
<li>setBinomialScore()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#a34f6efc6e8cf3e5eca8c2e42ba418f83">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>setBlastHit()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_hairpin_kmer.html#a3df2b6875a5496c8447a3f389d173d7a">broad.core.primer3.HairpinKmer</a>
</li>
<li>setBlocks()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_alignment.html#adddb5268d348456bc61cf6d66845fe2f">broad.core.multiplealignment.MAFAlignment</a>
</li>
<li>setBlockSizeStarts()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_b_e_d.html#a0b29ae94ceab37657208e52401e6c146">broad.core.annotation.BED</a>
</li>
<li>setBoundariesFromAnnoations()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#a5487f0a8b3da26f5489787aeb030f816">broad.core.annotation.BasicLightweightAnnotation</a>
</li>
<li>setCapacity()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a723236046f4794809e92aef25ca79b89">broad.core.sequence.Sequence</a>
</li>
<li>setCDS()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#aee850a4a9ec48d1cbcb91db4e71eadec">umms.core.annotation.Gene</a>
</li>
<li>setCDSEnd()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a5c135a41f2cf869c6b0ee9efcc4f6c5a">umms.core.annotation.Gene</a>
</li>
<li>setCDSRegion()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a5a3c5a6eddaf623e77ca3966feba6e3e">umms.core.annotation.Gene</a>
</li>
<li>setCDSStart()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#ac5b1e71d199dd1f004d600b51267c2b9">umms.core.annotation.Gene</a>
</li>
<li>setCharAt()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a9d8bf6ecc44a74ff02c8811f29f670ec">broad.core.sequence.Sequence</a>
</li>
<li>setChromosome()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#aeb96f44edbf4809956fe995a5cb3fc36">broad.core.annotation.BasicLightweightAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation.html#ade7adfb7a7597f976d547736ce83a48d">broad.core.annotation.LightweightGenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a0be1547c74c577c331f647a599a5f830">broad.core.sequence.SequenceRegion</a>
</li>
<li>setChromosomeLength()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#ab396983719c5ddc6d2cd9ce108b131ea">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>setChunkSize()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#ad89bc1748b5f6adfa0ae52f15f759806">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a896c96747a76fd0c169bbb694622fd33">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#aad7c99994370aad03b934eb60423656c">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>setClusterDistanceFunction()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_hierarchical_clustering.html#a4b12cc4e37e5ea21f76aab5738c0cfbf">broad.pda.geneexpression.clustering.HierarchicalClustering</a>
</li>
<li>setCol()
: <a class="el" href="classjaligner_1_1_cell.html#a0d19e75ac7bf3c1f326cde5563a842b0">jaligner.Cell</a>
</li>
<li>setColumn()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#af60689c8be349020ca8738952e727469">broad.core.datastructures.MatrixWithHeaders</a>
, <a class="el" href="class_jama_1_1_matrix.html#a9d5823758d0a73cd1e729915d32db5ec">Jama.Matrix</a>
</li>
<li>setColumns()
: <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_location_aware_matrix.html#ae1ebf162e0ebbc8e9e3aae5b44db7596">broad.pda.datastructures.LocationAwareMatrix</a>
</li>
<li>setCoordinateSpace()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#a2978ef854eebbaadd1de11c2e9f06404">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>setCount()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_count_score.html#a1dd7fbb7fe3cd767c9ebde283256ffde">umms.core.model.score.CountScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_end_in_window_count_score.html#a2aa0769ad6490f25b6daffbe413f99d8">umms.core.model.score.EndInWindowCountScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_start_in_window_count_score.html#ae19e8397eb8854112bf315c68b94f0c5">umms.core.model.score.StartInWindowCountScore</a>
</li>
<li>setCountNormalizer()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a94bb6e80732d15b31ff18425e5a8481e">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
</li>
<li>setCountScore()
: <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_alignments.html#a5df291246f8050ba25e2bb7efc7f6b28">broad.pda.datastructures.Alignments</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a5f40ece5a87771c6fc89d79f56ac45e4">umms.core.annotation.Gene</a>
</li>
<li>setCtrlCoordSpace()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#ad82f4f3db08d2c755726216ef3a4f413">umms.core.model.score.MultiScore</a>
</li>
<li>setCtrlCount()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_binomial_enrichment_score.html#af27e39aa46a85ed391bfc2a0638c1eeb">umms.core.model.score.BinomialEnrichmentScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#a7c8b8a89df6913ec391ecbc21328b6b0">umms.core.model.score.MultiScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#a76b7fff9ad89f7581f9ac6c5509c68cb">umms.core.model.score.PoissonEnrichmentScore</a>
</li>
<li>setCtrlRegionCount()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_binomial_enrichment_score.html#a00be31dfd637d3ea6e407f20524db0ac">umms.core.model.score.BinomialEnrichmentScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#a8be96706ce0566b740b6f6b72398d418">umms.core.model.score.MultiScore</a>
</li>
<li>setCtrlRegionTotal()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#a8aff1d02744276269cfed072337fddc1">umms.core.model.score.PoissonEnrichmentScore</a>
</li>
<li>setCtrlTotal()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#a01db0cad7930b22e14f9ee0cef5cca89">umms.core.model.score.PoissonEnrichmentScore</a>
</li>
<li>setCurrentSequenceId()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_abstract_fasta_parser.html#ac75c101613d9c3832d862e2a04eee671">broad.core.sequence.AbstractFastaParser</a>
</li>
<li>setData()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#aba318448932df190ad98f6f13727fe65">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>setDataDimensions()
: <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_location_aware_matrix.html#ad67e04f012ae218f9b8841144dd6a186">broad.pda.datastructures.LocationAwareMatrix</a>
</li>
<li>setDenominatorCount()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#a3d8c6cb4683ab5f88e8e9a36ddfd368f">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#a26d2ff4e83dc5437ee4e1ea9514cb26b">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#a3f247967161862211e93a4726bb565e5">umms.core.model.score.WindowAllScore</a>
</li>
<li>setDenominatorRegionTotal()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#a53d82776cf61bbf4ba0c3dec5abf85e0">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#a84b557325165848135276d69c99d246e">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#a290882f1dd93b974a0c4160f7722220d">umms.core.model.score.WindowAllScore</a>
</li>
<li>setDenominatorScanPVal()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#ab58a0866736371255e8c3c70c921687e">umms.core.model.score.WindowAllScore</a>
</li>
<li>setDenominatorTotal()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#a4ce8d20f5b775f7c805367c75e56dda1">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#a020ab1ef12903c21438c2a2fd676b495">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#a3d0f374e39af3b1f1f73b0aeaac32526">umms.core.model.score.WindowAllScore</a>
</li>
<li>setDescription()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_sequence.html#ab945b06e5aeb8495de038ce68a3eafb9">broad.pda.seq.fastq.FastqSequence</a>
, <a class="el" href="classjaligner_1_1_sequence.html#afe611456a9666b36489172cad9efa137">jaligner.Sequence</a>
</li>
<li>setDiagonal()
: <a class="el" href="classjaligner_1_1_traceback_node.html#a77ef57e58318c665fe107fcb01ba0156">jaligner.TracebackNode</a>
</li>
<li>setDuplicateFlag()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a9b214481a55d504376cf4389f5153102">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#a331c3076e755132f182d225ba9a26f69">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a123904f9fe189195f21b5d04b55fe08a">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>setDuplicateReadFlag()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment_1_1_wrap_sam_record.html#a3a5b72b49f62e442cc56ea8761e0e7b6">umms.core.alignment.SingleEndAlignment.WrapSamRecord</a>
</li>
<li>setEnd()
: <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation.html#afa510429319c0efc147c5da446cab166">broad.core.annotation.LightweightGenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#af7d15e288735f4e08a3b66ad07690297">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#afebca982cb0e0752e8ec6749f1238bc3">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#a149e821da8f19e0fad6c3a52edf61728">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a8abb166f1057b8e7cd4230b4d6b91235">umms.core.annotation.BasicAnnotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_compound_interval.html#a7dc57080ad2356580925c81fcf8eac43">umms.core.annotation.CompoundInterval</a>
</li>
<li>setEndStateTransitionProbability()
: <a class="el" href="classbroad_1_1core_1_1hmm_1_1_markov_model_3_01_t_01_4.html#ae791de048edf2805a41b0ecbefcad618">broad.core.hmm.MarkovModel&lt; T &gt;</a>
</li>
<li>setEnrichment()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score.html#aacb060ae70cc90fee43fa6c60439ba20">broad.pda.seq.segmentation.GeneScore</a>
</li>
<li>setEstimationWindow()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_tree_scaler_i_o.html#af3004c6b98503e588a3ebde5820245f9">broad.core.siphy.TreeScalerIO</a>
</li>
<li>setExonScores()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#ad0922edaa38d4478835466e3bb2a9cbc">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>setExpressionScanPvalueCutoff()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a3f9e817ce92fd84f2b5756bdb5b97c36">broad.pda.seq.clip.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#a1801b112973b23055ad140ba906ab57a">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a2013287af35974ad48c4cd57b099ac4b">broad.pda.seq.protection.SampleData</a>
</li>
<li>setExtend()
: <a class="el" href="classjaligner_1_1_alignment.html#a42a2cd2e2fc936bf3122e58efa3529c7">jaligner.Alignment</a>
</li>
<li>setExtensionFactor()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a9c9d0361b83f9772aa8e8b0536e9ef74">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#ad3f655755e12f50f3511c176c678c338">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a1e3aa5641fa5aa9963d9a3b05870012a">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a7ff259314b1bd4ec3898bccdad3e7433">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>setExtraFields()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_gene_with_isoforms.html#a46a36daba673685dc1948a32e7d12935">broad.pda.gene.GeneWithIsoforms</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#a6af45a42068025dc80837e1df7dbcd49">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a1c88dd4db6a2ad407ff3358344dacd36">umms.core.annotation.Gene</a>
</li>
<li>setFeature()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_g_f_f.html#af47cdf69076679c5c3ef472a2f233f7f">broad.core.annotation.GFF</a>
</li>
<li>setFilterByStrand()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#a74e4548bd7887800f29fc5d487ccd37d">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>setFindMaxContiguous()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#ae9180b76d6c1bc1ea8d6024a2b08c5db">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>setFirstRead()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a956b047171cc65bf166e254d8016eff4">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a9759d16bc1da489233ea978b698f2de1">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>setFirstReadTranscriptionStrand()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#ada766545a50e40b013e0c61d72993b84">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>setFivePrimeBuffer()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_genomic_annotation.html#af0dcab996373305af00c42a58d8f62e9">broad.core.annotation.BasicGenomicAnnotation</a>
</li>
<li>setForwardStrand()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a1126c1510a531a59949482357696d37b">broad.core.sequence.Sequence</a>
</li>
<li>setFragmentFlag()
: <a class="el" href="classumms_1_1core_1_1readers_1_1_paired_end_reader.html#aaeab070b5abd537741ffcd9d3a51ed88">umms.core.readers.PairedEndReader</a>
</li>
<li>setFragmentStrand()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#adb6292c83fa8e285c0350584f196c5fe">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#afd4d444c8d0fffc880aff18481cd39ff">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#aac886365e7cb010840b33c5a493d9e65">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>setFrame()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_g_f_f.html#a8ce81e663aa0881438758e50d4bdfc79">broad.core.annotation.GFF</a>
</li>
<li>setFromCounts()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_column.html#aa264b4ccb3a80288204ac503d629cda3">broad.core.motif.PositionWeightColumn</a>
</li>
<li>setFullyContainedNumberOfReads()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score.html#aaab5b034ea344de92acc299889232dce">broad.pda.seq.segmentation.GeneScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_basic_scan_statistic_score.html#af831bbfb56ac03190d3bb92e7fe154a2">umms.core.model.score.BasicScanStatisticScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_scan_statistic_score.html#a60e610470e2673a071b504431a7bf9a6">umms.core.model.score.ScanStatisticScore</a>
</li>
<li>setGaps()
: <a class="el" href="classjaligner_1_1_alignment.html#a86b472121858e6f1f5f5d53e6516cb66">jaligner.Alignment</a>
</li>
<li>setGene()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#a7f355b3f2c4188e6a979e08b036514d4">umms.core.model.score.MultiScore</a>
</li>
<li>setGeneLength()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score.html#aacdf2e47b32561b897f8fc01ced14e29">broad.pda.seq.segmentation.GeneScore</a>
</li>
<li>setGlobalLength()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_basic_scan_statistic_score.html#a613cd589583ed844d4859ab1d5364ece">umms.core.model.score.BasicScanStatisticScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#a81cf92cb99e356b366d22275dd65de60">umms.core.model.score.PoissonEnrichmentScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_scan_statistic_score.html#aef1bab1d4217739b6c155cfd8745d331">umms.core.model.score.ScanStatisticScore</a>
</li>
<li>setGroup()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_g_f_f.html#aead35b291d127bc093c2ad2b9e5e8469">broad.core.annotation.GFF</a>
</li>
<li>setHasIndel()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#add0609ad85691794579b5d73a6e240ae">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>setHeader()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#aac3271f5fbb76f0150766972b37b3e91">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#a4305d2988fba3c16803f3e999b58a200">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#ad4d52dabe1650f5221b13131bc019df4">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>setId()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_genomic_annotation.html#aab81f801d84164ee7db06ce74d45ea50">broad.core.annotation.BasicGenomicAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_genomic_annotation.html#aeaa90b7ee4396fe088f38ab2570d8b6f">broad.core.annotation.GenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a22ed5d4080bf7946a0d0a782f792e945">broad.core.sequence.Sequence</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#ad8ad1b1aad94d077e5a65cc57bcecbbc">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classjaligner_1_1formats_1_1_format.html#a473fd596efc8f218f904002249e7d2cc">jaligner.formats.Format</a>
, <a class="el" href="classjaligner_1_1_sequence.html#a7a62cfa9181a4602171226b22db21c88">jaligner.Sequence</a>
</li>
<li>setIdentity()
: <a class="el" href="classjaligner_1_1_alignment.html#af64b2e17cdbf370af3ab167fc0fc7c53">jaligner.Alignment</a>
</li>
<li>setIgnoreList()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_conserved_p_w_m_scanner.html#ad8b0d58225f5f42cb4c806dc0490a775">broad.core.siphy.ConservedPWMScanner</a>
</li>
<li>setInitialStateTransitionProbability()
: <a class="el" href="classbroad_1_1core_1_1hmm_1_1_markov_model_3_01_t_01_4.html#aba503eb5544b223a0d59735dce22d352">broad.core.hmm.MarkovModel&lt; T &gt;</a>
</li>
<li>setIOHelper()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#add81a0e357b98982640e70d83b3652fe">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>setIsFirstMate()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a97f8020bc3d1457584313a0405e2f766">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>setIsToComplex()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#ae7e2b65737680bab483a22556e8438ab">broad.pda.seq.graph.Path</a>
</li>
<li>setJnlp()
: <a class="el" href="classjaligner_1_1util_1_1_commons.html#a3bd050e7f48d16e182f8d0acfc433e1c">jaligner.util.Commons</a>
</li>
<li>setLambda()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a6ba57d21a9ba6a7eca2a7f04fa0c4c3d">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
</li>
<li>setLeft()
: <a class="el" href="classjaligner_1_1_traceback_node.html#af75c30d4b1c0f3a58e9f74a3a233956e">jaligner.TracebackNode</a>
</li>
<li>setLine()
: <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_alignments.html#a6d94c020e29525e81a71479ff5459ba8">broad.pda.datastructures.Alignments</a>
</li>
<li>setLinkage()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_hierarchical_clustering.html#a8e710d388508dce2b03f6dc4dde37fdc">broad.pda.geneexpression.clustering.HierarchicalClustering</a>
</li>
<li>setLocalLambda()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#ab5c25356357ea84331de7384dbf44acf">broad.pda.seq.graph.Path</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score.html#a5e775112376674814357454a6087a458">broad.pda.seq.segmentation.GeneScore</a>
</li>
<li>setLocalRate()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a5255b57645f44f82c704a21f6e6932f9">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>setLoggerLevel()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#afca8b084de2774d691172ca3a734e775">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>setMarkupLine()
: <a class="el" href="classjaligner_1_1_alignment.html#a8447b039c854e628939693a1559c2c9e">jaligner.Alignment</a>
</li>
<li>setMaskFileData()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#ac019bf49be802c3b62d3a3055104c365">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>setMatch()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_p_s_l_record.html#ac6b63ce6fb4d1ca38ec813bcf538de03">umms.core.annotation.PSLRecord</a>
</li>
<li>setMatrix()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix.html#a430da298fb10b0fbb57e4e34f49668b8">broad.core.motif.PositionWeightMatrix</a>
, <a class="el" href="classjaligner_1_1_alignment.html#aee8b0ccf455806d2803a39e9c5da87a0">jaligner.Alignment</a>
, <a class="el" href="class_jama_1_1_matrix.html#a80e1f77fbd4ddff32165524cd9c9b0b5">Jama.Matrix</a>
</li>
<li>setMatrixFromRawData()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix.html#a987b71b7f2d6cb595bee270cdf8e682a">broad.core.motif.PositionWeightMatrix</a>
</li>
<li>setMaxAllowableInsert()
: <a class="el" href="classumms_1_1core_1_1writers_1_1_paired_end_writer.html#af311259728eda69307c90e2c7bb01057">umms.core.writers.PairedEndWriter</a>
</li>
<li>setMinContguousSegmentSize()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#ab7ab620d4f4c1b25acc716d83a8d72b6">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>setMinimumMappingQuality()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a14dbe106e617b148c4fe3ce1dd77f172">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a30c869716b5df7ddf3523efb81579fc8">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a13f7d8efa23552e98f798e9deec570de">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a90eb0e32ff0c94bf137992c067e8fd38">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>setMinimumTreeLength()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_tree_scaler.html#a39748e22702a23a62e6806516d4b9c4d">broad.core.siphy.TreeScaler</a>
</li>
<li>setMinNumberOfSplices()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a61ed951795b265ba619f938e715ca1ab">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>setMirScore()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_hairpin_kmer.html#a7c4b98cf635f1314ad769248e0aaa1bd">broad.core.primer3.HairpinKmer</a>
</li>
<li>setMismatch()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_p_s_l_record.html#a532509325fab0bf13280c7ef76664a5f">umms.core.annotation.PSLRecord</a>
</li>
<li>setModel()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_conserved_p_w_m_scanner.html#a089244695d241f1f6d0342f1598e995c">broad.core.siphy.ConservedPWMScanner</a>
, <a class="el" href="classbroad_1_1core_1_1siphy_1_1_tree_scaler.html#a6874a8f4e920f45d2abe3461842138d9">broad.core.siphy.TreeScaler</a>
</li>
<li>setModelParameters()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model.html#ad33990a2042c4e252c1df0957a4d8692">broad.core.siphy.EvolutionaryModel</a>
</li>
<li>setMotif()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_sequence_motif.html#a1e4c678e3e9cf04d053ac2f7538e70f1">broad.core.motif.SequenceMotif</a>
</li>
<li>setName()
: <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation.html#a28cc1992e605bbd4b743e8cf6562fe01">broad.core.annotation.LightweightGenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix.html#a52521dce3a38dcc4eb1734e886b5d870">broad.core.motif.PositionWeightMatrix</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#ab7e684dd06da0b0e01fda2b44847bc9b">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_sequence.html#a27baaec0fecb339510d4f4116ef50c0c">broad.pda.seq.fastq.FastqSequence</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#a91fac76eec98277eca2e0a12d4b4fe77">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a9cde990314fec1ff189b59542cc11809">umms.core.annotation.BasicAnnotation</a>
</li>
<li>setName1()
: <a class="el" href="classjaligner_1_1_alignment.html#acbf8298fb61e5c6b74d413e3d0b4fd98">jaligner.Alignment</a>
</li>
<li>setName2()
: <a class="el" href="classjaligner_1_1_alignment.html#a5c68b2250dd2a7a544106a43c46a4fbf">jaligner.Alignment</a>
</li>
<li>setNcount()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_p_s_l_record.html#a1cff0c12ece80b762281ec6818a5f245">umms.core.annotation.PSLRecord</a>
</li>
<li>setNegativeStranded()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a8ce46d47b18ee30cccb2bc044dda8e92">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#adebf2567676fec9bf5dda66d8bc2da6d">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>setNeutralModel()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_tree_scaler.html#aac84894bf323063b0e023ae5aa23d0e8">broad.core.siphy.TreeScaler</a>
</li>
<li>setNeutralOmegaDistribution()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model.html#ab540e39054069938170aaa3f45142110">broad.core.siphy.EvolutionaryModel</a>
</li>
<li>setNominalPValue()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score.html#a16677af1c9b8dd68bb1ef9d2b772f331">broad.pda.seq.segmentation.GeneScore</a>
</li>
<li>setNormalizationStrategy()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#aec08ae772841ac76b9b628c66a1a822d">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#abcb395b283983b86e7f3f3789c275cf8">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>setNum()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_column.html#a6a093c62c5e480fdb2896978d4fb97cd">broad.core.motif.PositionWeightColumn</a>
</li>
<li>setNumberFormatter()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_tree_scaler_i_o.html#a1877b909d6db65af1d83df131eb88187">broad.core.siphy.TreeScalerIO</a>
</li>
<li>setNumberOfMarkers()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#ac76896063c3a3f4b477649154a3ef41c">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
</li>
<li>setNumberOfReads()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#aae2a2bc92f81bc8d50824672a3f5eb55">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score.html#ac0bc6e6729191cac4dad559a08daab76">broad.pda.seq.segmentation.GeneScore</a>
</li>
<li>setNumeratorCount()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#afe4d4a09361e2f703314d8d905e55a8e">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#acf30a764e757d4b4f615fd1bf5badffd">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#acf6d535853a190576a240e96c23df034">umms.core.model.score.WindowAllScore</a>
</li>
<li>setNumeratorRegionTotal()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#a15d898583a458c1cf1db03757f42a58e">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#a36ba05b15a6e54eaf8fbe20d835b1875">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#a4a64cec6c0ca531671a409d7c47cea15">umms.core.model.score.WindowAllScore</a>
</li>
<li>setNumeratorScanPVal()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#ac9740cfc8bcc63cc0e59c0efc6cde41a">umms.core.model.score.WindowAllScore</a>
</li>
<li>setNumeratorTotal()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_new_ratio_score.html#ab9a973e024d4a60f7d6fe914de852252">umms.core.model.score.NewRatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_ratio_score.html#afb05332cdd6719b45337c0badfe84f7f">umms.core.model.score.RatioScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_all_score.html#aea5c0751bb3ca3195c75e364faf30970">umms.core.model.score.WindowAllScore</a>
</li>
<li>setOmega()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model.html#a0a669885427f163c3382a5a7d77f2573">broad.core.siphy.EvolutionaryModel</a>
</li>
<li>setOmegaDistByTreeLength()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model.html#a7c19df7625f37c6e233846b11f840707">broad.core.siphy.EvolutionaryModel</a>
</li>
<li>setOpen()
: <a class="el" href="classjaligner_1_1_alignment.html#a517634ccc2d6b59a6e0d99734bbf3c13">jaligner.Alignment</a>
</li>
<li>setOrientation()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_genomic_annotation.html#a29df7ed3a39e72b77042d4625b536f1b">broad.core.annotation.BasicGenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#a4fc2bd00d303328100e8bd20d2c6dd49">broad.core.annotation.BasicLightweightAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_genomic_annotation.html#a6155b08fe0145a5ca451a4f49b18d167">broad.core.annotation.GenomicAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation.html#a6bed3a14fbbcb700693200c4a634f69d">broad.core.annotation.LightweightGenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a202b0129ab97723c01d90963f2040a98">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation.html#a2d6aae9b87d1544ef70124ab587d436d">umms.core.annotation.AbstractAnnotation</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#ad919607886c5447f5af908b286fba3ad">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a2956f388c13ae52dda924547002a5393">umms.core.annotation.BasicAnnotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#ae8eb345ba7a1f4e7d15b0fcd8dea258d">umms.core.annotation.Gene</a>
</li>
<li>setOrientedEnd()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a64c04998f8049912cd03687edb2c4012">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#a609c582828b96300a6ae44332c17a8e8">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a350aa437c56b6e7cc3230df3be7097b4">umms.core.annotation.BasicAnnotation</a>
</li>
<li>setOrientedStart()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#ab41d0116c937d9875131e50013a42a4c">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#a6529e3f5bc1531639fe663f8ee5c0be2">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#adb91d1844a2a1543aa854d9c43e5b937">umms.core.annotation.BasicAnnotation</a>
</li>
<li>setOriginalScore()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_hairpin_kmer.html#a294ae5418fc2bd6cbbeb105147e0c157">broad.core.primer3.HairpinKmer</a>
</li>
<li>setOriginalSequence1()
: <a class="el" href="classjaligner_1_1_alignment.html#ad41b3db8eee6829da988657b7588c2c7">jaligner.Alignment</a>
</li>
<li>setOriginalSequence2()
: <a class="el" href="classjaligner_1_1_alignment.html#a4c2c8b94faaf757a90bdbcb1611d20a4">jaligner.Alignment</a>
</li>
<li>setPair1Parser()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_splitter_by_barcode.html#a5c64f9b2979ca3c139a4ffbbd4fcc469">broad.pda.seq.fastq.FastqSplitterByBarcode</a>
</li>
<li>setPair2Parser()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_splitter_by_barcode.html#ae314b8254b7d63a57fac434f66229b62">broad.pda.seq.fastq.FastqSplitterByBarcode</a>
</li>
<li>setPaired()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_splitter_by_barcode.html#afa31dea38fbcd4ed5f0398047280abfc">broad.pda.seq.fastq.FastqSplitterByBarcode</a>
</li>
<li>setParameters()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model.html#aa3bcbca13ff568ed8fc90182c1a71a2e">broad.core.siphy.EvolutionaryModel</a>
</li>
<li>setPeakCutoffMostCommonReplicate()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#ae0a18e8cadb800e2f88cb720e0cbaae2">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>setPeakTrimQuantile()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#aab4e703b4069c5eb67829cebe4b4535a">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>setPeakWindowCountCutoff()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#acb73fe5a22023758262f721784493c36">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>setPeakWindowScanPvalCutoff()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#ae6331bb975e2d00437692308e01e868d">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>setPercentMaskedAllowed()
: <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_genomic_space.html#a2a3d42d66612e6a78e335830a599a88a">umms.core.coordinatesystem.GenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcript_in_genomic_space.html#a82996bfc34f6efd9d1d31e001d4ec56c">umms.core.coordinatesystem.TranscriptInGenomicSpace</a>
</li>
<li>setPercentVariantsCovered()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_hairpin_kmer.html#aad510bc4bf2090b5d528105f223101a9">broad.core.primer3.HairpinKmer</a>
</li>
<li>setPi()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model.html#a161641aefb263800c873b447e4d87d3a">broad.core.siphy.EvolutionaryModel</a>
</li>
<li>setPIDToName()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a4228d8a1040c66f55abbe729f9b9fee5">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>setPositiveStranded()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#a699d89d4a1fdda6b9215fbc45be567a2">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a361c64731d551b0e40d3f90c35051ed5">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>setProgramDescription()
: <a class="el" href="classbroad_1_1core_1_1parser_1_1_command_line_parser.html#a80bd6dcaaa8cac1cdc089ba3d9627f24">broad.core.parser.CommandLineParser</a>
</li>
<li>setProperPairFlag()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a5bc12cc540194c01868fae4e7140eb44">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#ac7f14cb2660aaabeac51f6b75f1ce164">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a3d3da0d6884e3aa32e7e80f983e16a6d">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>setPvalue()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_basic_scan_statistic_score.html#a7c14f611fbcdda2ecc3f09847faa8c00">umms.core.model.score.BasicScanStatisticScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_binomial_enrichment_score.html#ae549727787065b81209f827f7b246808">umms.core.model.score.BinomialEnrichmentScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#a1b0f24ea796da0cd9de002440a4686cc">umms.core.model.score.PoissonEnrichmentScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_scan_statistic_score.html#a6f3939d4183715745840796aba5a3f81">umms.core.model.score.ScanStatisticScore</a>
</li>
<li>setQueryEnd()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_p_s_l_record.html#a571002b83eb3837b3f7b3cad75ad0c6d">umms.core.annotation.PSLRecord</a>
</li>
<li>setQueryGap()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_p_s_l_record.html#a1006f6653f527786d2c9e80641b74eff">umms.core.annotation.PSLRecord</a>
</li>
<li>setQueryGapBases()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_p_s_l_record.html#a44088687e7bab19bfa71d6004b4890c0">umms.core.annotation.PSLRecord</a>
</li>
<li>setQuerySize()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_p_s_l_record.html#a51fd82202e1ee98cf85d0aa88678e768">umms.core.annotation.PSLRecord</a>
</li>
<li>setQueryStart()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_p_s_l_record.html#a8ddc23d04833743e8fbd3c6584df16cd">umms.core.annotation.PSLRecord</a>
</li>
<li>setRateMatrix()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model_parameters.html#a6a774077bdc8eb73e69479ff35044ae6">broad.core.siphy.EvolutionaryModelParameters</a>
</li>
<li>setRead1Trim3p()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_splitter_by_barcode.html#a55cd9f227332e8b25ee4ba3a0a6a5b57">broad.pda.seq.fastq.FastqSplitterByBarcode</a>
</li>
<li>setRead1Trim5p()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_splitter_by_barcode.html#a820e85f24d137a4310f282bd4a4eedb6">broad.pda.seq.fastq.FastqSplitterByBarcode</a>
</li>
<li>setRead2Trim3p()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_splitter_by_barcode.html#ae8008497bf67308fdbba0cc9811a890d">broad.pda.seq.fastq.FastqSplitterByBarcode</a>
</li>
<li>setRead2Trim5p()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_splitter_by_barcode.html#a2f1c8cefd18b213f99447447d30128cd">broad.pda.seq.fastq.FastqSplitterByBarcode</a>
</li>
<li>setReasonForUnsafe()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_stored_alignment_stats.html#a25319ed55ddcf27110c3228505ebb032">broad.pda.seq.alignment.StoredAlignmentStats</a>
</li>
<li>setReferenceId()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#a8147ac5b5ea8641317588c5957006b19">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>setReferenceName()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a8410b134db880c2c1ed53d392a28995b">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#aa6ddff9a7f3bcff152914299092545ff">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#aa116c039168d25b2c9c2f767f2bc44e4">umms.core.annotation.BasicAnnotation</a>
</li>
<li>setRefGapped()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#abec1f2b36b5b61d36491db003ad702cf">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>setRegionEnd()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a1f6c341d0e4aa8f0f7be09ae403167a1">broad.core.sequence.SequenceRegion</a>
</li>
<li>setRegionLength()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_basic_scan_statistic_score.html#add821b493f0413c72b1395537ad655b5">umms.core.model.score.BasicScanStatisticScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_binomial_enrichment_score.html#abda24560294c8a6578c8b59ec879790d">umms.core.model.score.BinomialEnrichmentScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#a612072d77935b81ca8bb0295ee95115d">umms.core.model.score.MultiScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#a5eea269758261a12634b7412ca74f904">umms.core.model.score.PoissonEnrichmentScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_scan_statistic_score.html#a276f9bc08e9d12135b4268dd6399eed1">umms.core.model.score.ScanStatisticScore</a>
</li>
<li>setRegionStart()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a2d73403c15168f6cf0033dd6f861686d">broad.core.sequence.SequenceRegion</a>
</li>
<li>setRegionTotal()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_count_score.html#a4fe9cdf1116853ad26cfdb39ff53617e">umms.core.model.score.CountScore</a>
</li>
<li>setRemoveDuplicatesFlag()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a5ea0c67a41a6d61ae674784525937fd7">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>setRepMatch()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_p_s_l_record.html#a971171293a2eb9b32d1586bc994cdea7">umms.core.annotation.PSLRecord</a>
</li>
<li>setReversedOrientation()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#a3db34c03b25ad0e5b0a7165e9c00b936">broad.core.annotation.BasicLightweightAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_genomic_annotation.html#a01defdbb20c34c88e500ad4009cb4d30">broad.core.annotation.GenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a25bfe99c413ee0e893d687509add2dd8">broad.core.sequence.SequenceRegion</a>
</li>
<li>setRgb()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_b_e_d.html#ae640c0e44d7bece6468367b1f0d939c4">broad.core.annotation.BED</a>
</li>
<li>setRow()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a88ba7d8ba6f897e7e207b05aa8804ba9">broad.core.datastructures.MatrixWithHeaders</a>
, <a class="el" href="classjaligner_1_1_cell.html#aa34e8db83cd9ab01217363eb37c2810c">jaligner.Cell</a>
, <a class="el" href="class_jama_1_1_matrix.html#a4c38cfb681b2d2a64c29467b2a08c045">Jama.Matrix</a>
</li>
<li>setRowDescription()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a06732f037d0c2a4a9610c9b542588a4f">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>setRPKM()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score.html#a5b1c5b65fd25bede453dd30f55808bff">broad.pda.seq.segmentation.GeneScore</a>
</li>
<li>setRPKMConstant()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a6c445a5adf8ec62e7b16f024a561b222">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>setRS8Score()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1_hairpin_kmer.html#a6c5ce386d481e633b2d010ea8585b6c5">broad.core.primer3.HairpinKmer</a>
</li>
<li>setSampleCoordSpace()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#aba6b9a6517ac4e978ef082acdbe470c1">umms.core.model.score.MultiScore</a>
</li>
<li>setSampleCount()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_binomial_enrichment_score.html#ae7ed1a7329037df5041ff3594446a5c0">umms.core.model.score.BinomialEnrichmentScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#aca14502ed9e3480235fc113a4d205541">umms.core.model.score.MultiScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#af40c845e00a0e3e7d56960533d4478c3">umms.core.model.score.PoissonEnrichmentScore</a>
</li>
<li>setSampleName()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a2e2bdce82785743a6814f003b3dc1df0">broad.pda.seq.clip.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#a77cf4e55d644cb09fa376f9d8b41c538">broad.pda.seq.protection.SampleData</a>
</li>
<li>setSampleRegionCount()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_binomial_enrichment_score.html#a76b7022678c5d0c0b48f38cf7e7154a8">umms.core.model.score.BinomialEnrichmentScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_multi_score.html#aed5f43ead6121b5314edbf0861165631">umms.core.model.score.MultiScore</a>
</li>
<li>setSampleRegionTotal()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#ad2783636cbac0d5e6d134c08046f8887">umms.core.model.score.PoissonEnrichmentScore</a>
</li>
<li>setSampleTotal()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_poisson_enrichment_score.html#a750d11633f6f5e66173d1ff2bd249362">umms.core.model.score.PoissonEnrichmentScore</a>
</li>
<li>setSAMString()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#ae08977025f74d398eeecb897efad27f1">umms.core.annotation.Gene</a>
</li>
<li>setScanPvalue()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_gene_score.html#ada83d50ea66182562a60dec6217f8e71">broad.pda.seq.segmentation.GeneScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_basic_scan_statistic_score.html#a5dde9b2f92c48594bfe07ad31cf6d9d9">umms.core.model.score.BasicScanStatisticScore</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_scan_statistic_score.html#a3faddbf93b31f5f9c6b0c5c3530fba06">umms.core.model.score.ScanStatisticScore</a>
</li>
<li>setScore()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#aeb06d309e53ec158338164b8945b9949">broad.core.annotation.BasicLightweightAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation.html#aff1d4a7789e8fee0258a91aaad66111d">broad.core.annotation.LightweightGenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#ab136689f207d5f9d97a335fa1f119d1b">broad.core.multiplealignment.MultipleAlignment</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a42b83bc601a115dbf7e848aa24356ebd">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region_window.html#a6b00804f980b26a9efdfbcd225cbf451">broad.core.sequence.SequenceRegionWindow</a>
, <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_cluster.html#a6fbec99c17be7b4cbf4fbe461f214e12">broad.pda.geneexpression.clustering.Cluster</a>
, <a class="el" href="classjaligner_1_1_alignment.html#a1e3b847b5aac66ef038ed71be6c597b8">jaligner.Alignment</a>
, <a class="el" href="classjaligner_1_1_cell.html#adcc794b3a9cd248461f554c6a00c1859">jaligner.Cell</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#aa46bc7126d4e5f9df374c6525cfcee14">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#ad04c091013feb60b8df504c4feef4cc4">umms.core.annotation.BasicAnnotation</a>
</li>
<li>setScores()
: <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_alignments.html#a513c21f833532890968ad1737a4ca89f">broad.pda.datastructures.Alignments</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_abstract_list_score.html#aa3a95daf6ebcb67546b9fccd018e8bcb">umms.core.model.score.AbstractListScore</a>
</li>
<li>setSecondRead()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#ae298323ad3086061065ecba18f78b9dd">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a33f1d19373f11699e701a22fc857712b">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>setSeed()
: <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_genomic_space.html#a3d4fc4c0f29817931209f7b07f341838">umms.core.coordinatesystem.GenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcript_in_genomic_space.html#aa49d5fc7d3850196506f7b387100209d">umms.core.coordinatesystem.TranscriptInGenomicSpace</a>
</li>
<li>setSentinel()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_interval_tree_3_01_v_01_4.html#a698672fc6e999de06d624f40ef26d0f4">broad.core.datastructures.IntervalTree&lt; V &gt;</a>
</li>
<li>setSequence()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_genomic_annotation.html#a93be2deada3f253081b40fdb92e6f67a">broad.core.annotation.BasicGenomicAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_genomic_annotation.html#a7f3133b1de941460bb22c6d2d3bd64a5">broad.core.annotation.GenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a8428d89d0d76f57b644d03c168febdb3">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classjaligner_1_1_sequence.html#aff2a4b49f5ad221d49972ff94eb54858">jaligner.Sequence</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a7976c9be5a3293b44ce54332610f5092">umms.core.annotation.Gene</a>
</li>
<li>setSequence1()
: <a class="el" href="classjaligner_1_1_alignment.html#ab4d4eeae40a289f641011f699e3e359c">jaligner.Alignment</a>
</li>
<li>setSequence2()
: <a class="el" href="classjaligner_1_1_alignment.html#a0c3ae8867af22b16c950984b0ebdb463">jaligner.Alignment</a>
</li>
<li>setSequenceBases()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a8195f46d431993392dbb892bb85beca1">broad.core.sequence.Sequence</a>
</li>
<li>setSequenceFile()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_chromosome.html#a51712489c54a80d7a571ff99112db94f">broad.pda.feature.genome.Chromosome</a>
</li>
<li>setSequenceFromChromosome()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a40aea7af10672b2b9c85a29ecaca3dad">umms.core.annotation.Gene</a>
</li>
<li>setSimilarity()
: <a class="el" href="classjaligner_1_1_alignment.html#a5237fd2e649ffb1e19b85576e7a303c2">jaligner.Alignment</a>
</li>
<li>setSource()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_g_f_f.html#add3d2dedb1d2dd60bc151c7a8ae3dd99">broad.core.annotation.GFF</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_fasta_sequence_i_o.html#ab7b87ab64cd3b0daa838d666c711815c">broad.core.sequence.FastaSequenceIO</a>
</li>
<li>setSpacing()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_phylip_interleaved_multiple_alignment_i_o.html#a6fcac155d471e8d2a0ddbf0e8ffcebd8">broad.core.multiplealignment.PhylipInterleavedMultipleAlignmentIO</a>
</li>
<li>setSplicedEdges()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#aede37af502c66fb974ebeb31e4634bdf">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>setSpliceWeight()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a4fec10d865ea7836e3a74bb8875229de">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#a348ec20a3d898b018e86c4a619c389b3">broad.pda.seq.graph.Path</a>
</li>
<li>setStart()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#a3f55da6f6c2d257caac12fd7c8ca3770">broad.core.annotation.BasicLightweightAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation.html#aa9663ad49b8a37f51e23928b94702d94">broad.core.annotation.LightweightGenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a999b6323d9f72d3e701608001bf98101">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#ad475cd8839a0836c9939bb17a1235d5b">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#ae077ebc7788964176739ff85bf29b560">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a3c5672991e6679950db460e22d727740">umms.core.annotation.BasicAnnotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_compound_interval.html#af7557cd3e43cbb55482a9eeea6d68629">umms.core.annotation.CompoundInterval</a>
</li>
<li>setStart1()
: <a class="el" href="classjaligner_1_1_alignment.html#a234e5b4ef8032cbefcd96413706aa293">jaligner.Alignment</a>
</li>
<li>setStart2()
: <a class="el" href="classjaligner_1_1_alignment.html#a3fbb3c1b258dd460fd7f486c2ef1384e">jaligner.Alignment</a>
</li>
<li>setStateTransitionProbability()
: <a class="el" href="classbroad_1_1core_1_1hmm_1_1_markov_model_3_01_t_01_4.html#aa8ed4512f906c463dfaf7278cb55a847">broad.core.hmm.MarkovModel&lt; T &gt;</a>
</li>
<li>setStop()
: <a class="el" href="classjaligner_1_1_traceback_node.html#a530103a5fda804219376ca38fa098a67">jaligner.TracebackNode</a>
</li>
<li>setStoredStats()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a3634e42ab60abcfed989990ea8956415">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
</li>
<li>setStrand()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_g_f_f.html#a0b8578bb9313ad95e4affd612318bc15">broad.core.annotation.GFF</a>
</li>
<li>setSubcluster()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1clustering_1_1_cluster.html#a0adc67b078b169fa6c9e54111589e42c">broad.pda.geneexpression.clustering.Cluster</a>
</li>
<li>setTargetGap()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_p_s_l_record.html#ad4f98ee3215b72ce55d25d849649ef05">umms.core.annotation.PSLRecord</a>
</li>
<li>setTargetGapBases()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_p_s_l_record.html#a6651dc11eecfdf92d2cb8ad5449bc628">umms.core.annotation.PSLRecord</a>
</li>
<li>setTargetSize()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_p_s_l_record.html#a8f06cee2b9504e36117fd453cd09300d">umms.core.annotation.PSLRecord</a>
</li>
<li>setThickEnd()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_b_e_d.html#a84b56939d50e48221c25bee0ce905a1d">broad.core.annotation.BED</a>
</li>
<li>setThickStart()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_b_e_d.html#ab50226189f71ccd48f52e5a78143a3e7">broad.core.annotation.BED</a>
</li>
<li>setThreePrimeBuffer()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_genomic_annotation.html#a7af507977197ffb1c8e7c1874cfbd7b9">broad.core.annotation.BasicGenomicAnnotation</a>
</li>
<li>setTotal()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_count_score.html#a7a9fde34261326ae89b9b5260be28648">umms.core.model.score.CountScore</a>
</li>
<li>setTree()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_evolutionary_model_parameters.html#a879f10543f5dc13ebad2d3aff1408433">broad.core.siphy.EvolutionaryModelParameters</a>
</li>
<li>setTrimEnds()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#aa17cb50bb398dc7770859377e8d9771e">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>setTrimQuantile()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#a05c9b50b32fd0cdfee049966ae7b9c1e">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>setType()
: <a class="el" href="classjaligner_1_1_sequence.html#a4ae3de6057a4f6361f3c8e930f8d6fbe">jaligner.Sequence</a>
</li>
<li>setUninformativeNodes()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_conservation_utils.html#a3e67d62373ffaf349c1f2885dbb4a1ac">broad.core.siphy.ConservationUtils</a>
</li>
<li>setUnparsedRow()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix.html#adad83edc20996566232161bb9d5f253f">broad.core.motif.PositionWeightMatrix</a>
</li>
<li>setup()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1rna_1_1_collect_rna_seq_metrics.html#a24459d7847405bc3114e24640edf7365">broad.pda.seq.rap.rna.CollectRnaSeqMetrics</a>
</li>
<li>setUp()
: <a class="el" href="classjaligner_1_1_traceback_node.html#ae33ca35336b16938466da6f720e7982c">jaligner.TracebackNode</a>
</li>
<li>setUpAlignment()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_conservation_utils.html#ab3eecb49e17d4de98098b5b85d53dcf4">broad.core.siphy.ConservationUtils</a>
</li>
<li>setUpMAF()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_conservation_utils.html#a2e8a2956dfe5e74d9cf4fbf09bb29fed">broad.core.siphy.ConservationUtils</a>
</li>
<li>setUpWeightSplices()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_continuous_data_alignment_model.html#af559e50d0cbcc7525afd106c4bb93418">broad.pda.seq.segmentation.ContinuousDataAlignmentModel</a>
</li>
<li>setValue1()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_pair_3_01_t1_01_4.html#a6fd405224a4032e432aa5880d8d43d54">broad.core.datastructures.Pair&lt; T1 &gt;</a>
</li>
<li>setValue2()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_pair_3_01_t1_01_4.html#a249331b264af2be64ae4d0fb4046839c">broad.core.datastructures.Pair&lt; T1 &gt;</a>
</li>
<li>setWeighReadCountsFlag()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a789507649e0152371c17a07d4027ad0b">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>setWindowLength()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_basic_scan_statistic_score.html#a4e1be5bcd2aa52ec9aee678d38c5dde9">umms.core.model.score.BasicScanStatisticScore</a>
</li>
<li>shift()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#a905175697ebd6e928f52403bf4a88832">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a5dc8df1d17984ba670c2b4bba52ca497">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classbroad_1_1core_1_1siphy_1_1_stationary_distribution_i_o.html#a858c5c52b0e6838bb8fba793cf3fcf62">broad.core.siphy.StationaryDistributionIO</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a49adbe8aa021ef96ac89ede1f05694df">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#a3014cb6c724297b8bbccabaffff4bb3a">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#abca261c2fa9c7851c23c2af956b77264">umms.core.annotation.BasicAnnotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_compound_interval.html#af2ab21545b4361433fd942b851d74993">umms.core.annotation.CompoundInterval</a>
</li>
<li>shiftPosition()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_transcribed_regions.html#aeec2d82099da998442a46c749447fdad">broad.core.sequence.TranscribedRegions</a>
</li>
<li>SHORT_ENCODED_a
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#aa8b9e6eb9635659fb61f46a91a2c1831">broad.core.sequence.Sequence</a>
</li>
<li>SHORT_ENCODED_A
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a5e9588156dd0a4c914cf8ea5e46dc288">broad.core.sequence.Sequence</a>
</li>
<li>SHORT_ENCODED_c
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#ade4cf8fa3c5604985c82057049949f18">broad.core.sequence.Sequence</a>
</li>
<li>SHORT_ENCODED_C
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a7eaf6345f01e26bfcf1925adb7b63b6a">broad.core.sequence.Sequence</a>
</li>
<li>SHORT_ENCODED_g
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a317b007c9ec5f87d5e4bae72adb17717">broad.core.sequence.Sequence</a>
</li>
<li>SHORT_ENCODED_G
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a1035184882e0829351d913c8212b8e99">broad.core.sequence.Sequence</a>
</li>
<li>SHORT_ENCODED_GAP
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#af6b23c272152e3450a5ed58c95b31fdb">broad.core.sequence.Sequence</a>
</li>
<li>SHORT_ENCODED_N
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a421b5eb48f2848619737a1a706ff2c55">broad.core.sequence.Sequence</a>
</li>
<li>SHORT_ENCODED_T
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#aa8830afd48b5456f422e0d4ac282c3e4">broad.core.sequence.Sequence</a>
</li>
<li>SHORT_ENCODED_t
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a1b69cede3c60348e6b8f9ab1aec01912">broad.core.sequence.Sequence</a>
</li>
<li>SHORT_READ_FLOW
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a753cb91384665e3a3440ce2f1844fa4a">broad.core.sequence.Sequence</a>
</li>
<li>ShortBED()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_short_b_e_d.html#aa4e5364e3f4f070ebbcade6bef4bede8">broad.core.annotation.ShortBED</a>
</li>
<li>shortBEDFactory
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_factory_factory.html#a94f585b4a297312a21eb818684e81514">broad.core.annotation.AnnotationFactoryFactory</a>
</li>
<li>ShortBEDReader()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_short_b_e_d_reader.html#a8d7aa5d63e299eb76ab40d3c1d822d06">broad.core.annotation.ShortBEDReader</a>
</li>
<li>shuffle()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_math_util.html#a5cc54523bf4f129b9c9dcce4d3e580c4">broad.core.math.MathUtil</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a8434722abf884dd505dab97058388e9f">broad.core.sequence.Sequence</a>
, <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_chromosome.html#a4c4be3e0aa711b3aec171098b3f59bdf">broad.pda.feature.genome.Chromosome</a>
</li>
<li>shuffleAndWriteAnnotations()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_shuffle_b_e_d.html#ab57d670efa1ad1a6aaba06ea3e1316de">broad.pda.seq.rap.ShuffleBED</a>
</li>
<li>ShuffledIterator()
: <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_shuffled_iterator_3_01_t_01extends_01_annotation_01_4.html#a9a35f8f619dde289f3c52653ebe6371b">umms.core.coordinatesystem.ShuffledIterator&lt; T extends Annotation &gt;</a>
</li>
<li>shufflePeaks()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_collect_annotation_enrichments.html#aba7f1f25cb7a629a27fde2c4d3c8105c">broad.pda.seq.rap.CollectAnnotationEnrichments</a>
</li>
<li>shuffleScores()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#ac3f633f85ce2f7edb7df80409f985ff5">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
</li>
<li>shuffleWithinValid()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_chromosome.html#aa9787193ba53f019fc520ec75ca9ccaf">broad.pda.feature.genome.Chromosome</a>
</li>
<li>signalData
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_two_sample_scan_skellam_peak_caller.html#a0e010f18c5cc10693cb6efd34385fb41">broad.pda.seq.protection.TwoSampleScanSkellamPeakCaller</a>
</li>
<li>signalName
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_two_sample_scan_skellam_peak_caller.html#ad9eccd44514f233b9dbaa542e8cdc59d">broad.pda.seq.protection.TwoSampleScanSkellamPeakCaller</a>
</li>
<li>signalProcessor
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_two_sample_scan_skellam_peak_caller.html#a262dec7274dd2c19ec9854f60f873213">broad.pda.seq.protection.TwoSampleScanSkellamPeakCaller</a>
</li>
<li>signalSamples
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#ade4081e64c55bb4a96905b3a5d0926f2">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>SIMILARITY
: <a class="el" href="classjaligner_1_1_markups.html#a7efe6d3f877cee8e68e0dc5dd7ef92e8">jaligner.Markups</a>
</li>
<li>SIMPLE_ARRAY_LEFT
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_count_probes.html#a7ae07f296cfce6d2d8c64f63087a0f1b">broad.pda.seq.rap.CountProbes</a>
</li>
<li>SIMPLE_ARRAY_RIGHT
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_count_probes.html#aef46c558c764f0fe20658d39aa46b7fb">broad.pda.seq.rap.CountProbes</a>
</li>
<li>SingleEndAlignment()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#af6832ccae2d80001aea1e9fab6d071f1">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>SingleInterval()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_single_interval.html#a497bc904cf68273751bd4d5133e63751">umms.core.annotation.SingleInterval</a>
</li>
<li>SingularValueDecomposition()
: <a class="el" href="class_jama_1_1_singular_value_decomposition.html#a3f28de078519f131ef50e32e9f489708">Jama.SingularValueDecomposition</a>
</li>
<li>size()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#addf403e222089681708bbb882411e8ce">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
, <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_interval_tree_3_01_v_01_4.html#a44cc85e85bbdb4f69970c9433039960c">broad.core.datastructures.IntervalTree&lt; V &gt;</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#aa3ee74c7111be122e85a67df4fc44acb">broad.core.sequence.SequenceRegion</a>
</li>
<li>SIZE
: <a class="el" href="classjaligner_1_1matrix_1_1_matrix.html#a3b67208fde3868e8ba53d4c384cde39a">jaligner.matrix.Matrix</a>
</li>
<li>size()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation.html#acfe7f4b4188dabba73d9d53de81f240f">umms.core.annotation.AbstractAnnotation</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#ab59b20bf3914e24babf0df6907bfb557">umms.core.annotation.Annotation</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#ab4d43fbfd436eb2c2016d5c8d1d55745">umms.core.annotation.AnnotationCollection&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4.html#aa3b719763750abc0dd09f44577007f85">umms.core.annotation.AnnotationList&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#aaf519a6b3dfe9f486e96704d21b07e91">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#aab4abcef1912e2e0640854d32e4e0a1e">umms.core.model.JCSAlignmentModel</a>
</li>
<li>sizeFile
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#a83f7fca4b0f3dbcf664e8a65d484c6e1">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>SIZES
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_genome_command_line_program.html#a977f4bba37c07644bd7fe0f02415f4b9">broad.pda.seq.rap.GenomeCommandLineProgram</a>
</li>
<li>skellamCdf()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_distribution.html#a73bea7f039165ac6fcf44985214db058">broad.core.math.Distribution</a>
</li>
<li>skellamDensity()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_distribution.html#afaa8ff22b9120443d6ea3b7002d1003c">broad.core.math.Distribution</a>
</li>
<li>skellamLeftTail()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_distribution.html#ac3e4c327184f1372573b3fb1234b24c6">broad.core.math.Distribution</a>
</li>
<li>skellamPvalue()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_distribution.html#ac6aac09e8f8d9e5ba35e5f660f151170">broad.core.math.Distribution</a>
</li>
<li>skellamRightTail()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_distribution.html#a7257c404df002835e41fa723963ca9db">broad.core.math.Distribution</a>
</li>
<li>SkellamScore()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_skellam_score.html#a10a9ad5e4a20d7ad7efa4f3e012fd852">broad.pda.seq.rap.SkellamScore</a>
</li>
<li>slidePWM()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_conserved_p_w_m_scanner.html#ae52325a0d90721ed2ae2afe3d244f578">broad.core.siphy.ConservedPWMScanner</a>
, <a class="el" href="classbroad_1_1core_1_1siphy_1_1_tree_scaler.html#a45de3c44b31473586a9783d20de81554">broad.core.siphy.TreeScaler</a>
</li>
<li>slideWindowComputeOmega()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1tools_1_1conservation_1_1_estimate_omega_per_exon.html#a6d13fa071903b8908760f11a7186d824">broad.core.siphy.tools.conservation.EstimateOmegaPerExon</a>
</li>
<li>SmatchLike()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_smatch_like.html#a9d7334ec3da8075166fb17ae52ff854e">umms.core.alignment.SmatchLike</a>
</li>
<li>SmithWatermanAlignment()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_smith_waterman_alignment.html#a362dd8843ac06c7a385d31c7a0f8f34b">umms.core.alignment.SmithWatermanAlignment</a>
</li>
<li>solve()
: <a class="el" href="class_jama_1_1_cholesky_decomposition.html#a03d6cb90abf3e377867e6d7758a00521">Jama.CholeskyDecomposition</a>
, <a class="el" href="class_jama_1_1_l_u_decomposition.html#ad8dde76b65b15ce3839648708f4b9b41">Jama.LUDecomposition</a>
, <a class="el" href="class_jama_1_1_matrix.html#a33aeb09e6d0099145841c9d6f82e0576">Jama.Matrix</a>
, <a class="el" href="class_jama_1_1_q_r_decomposition.html#ae6434a5c9faebf49cc163ef7c3ba9048">Jama.QRDecomposition</a>
</li>
<li>solveTranspose()
: <a class="el" href="class_jama_1_1_matrix.html#aac324c690f00efc5b6a1769d10a79722">Jama.Matrix</a>
</li>
<li>SORT_ORDER
: <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_filter_paired_reads_by_m_a_q.html#a9a1d0819235dc343e31ec22d14c3bbac">broad.pda.samtools.FilterPairedReadsByMAQ</a>
</li>
<li>sortColumns()
: <a class="el" href="class_jama_1_1_matrix.html#aefddb41d4a3265377a6e2c2aa8813d15">Jama.Matrix</a>
</li>
<li>sortColumnsKeepingIndex()
: <a class="el" href="class_jama_1_1_matrix.html#a9badd0bece3e57cab71286103370d79e">Jama.Matrix</a>
</li>
<li>sortList()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#afdb4fa388dc0081fdf34784428ec248f">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>source
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_genomic_annotation_reader.html#ab49a4ff8378bbaad2d520ac95b73601d">broad.core.annotation.BasicGenomicAnnotationReader</a>
</li>
<li>spearmanCorrelation()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_statistics.html#ac1fc53b1f7c77248c1d143aa69b66119">broad.core.math.Statistics</a>
</li>
<li>SPLICED
: <a class="el" href="enumbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t_1_1_edge_source_type.html#a331268f2fdc2768b1af5eaf85088c81c">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT.EdgeSourceType</a>
</li>
<li>SplitAndTrimIndex()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_split_and_trim_index.html#a422c5bee37b564849c76569b2ccebedb">broad.pda.seq.fastq.SplitAndTrimIndex</a>
</li>
<li>splitByChr()
: <a class="el" href="classbroad_1_1pda_1_1rnai_1_1_extract_sequence.html#a9c14a04f57f62be107bd1c5f9c365f8d">broad.pda.rnai.ExtractSequence</a>
</li>
<li>SplitByName()
: <a class="el" href="classbroad_1_1pda_1_1differential_expression_1_1_split_by_name.html#a00d4edf89e9f064f975ad0baa59e3c32">broad.pda.differentialExpression.SplitByName</a>
</li>
<li>SplitFastqByIndex()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_split_fastq_by_index.html#a7c2f09a3e6b459723210c28843794c63">broad.pda.seq.fastq.SplitFastqByIndex</a>
</li>
<li>SplitFile()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1tools_1_1conservation_1_1_split_file.html#a0abf3af2fb98486ad2f48e255b46f494">broad.core.siphy.tools.conservation.SplitFile</a>
</li>
<li>standardizeOreintation()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_gene_with_isoforms.html#a77c6049bcb70edc00376b968f0219c5e">broad.pda.gene.GeneWithIsoforms</a>
</li>
<li>start()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_parser.html#a61e15412ca2ceb8d91767e7a1ed0de94">broad.pda.seq.fastq.FastqParser</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_splitter_by_barcode.html#a849b97d6101918f8c0da871f0a3f4c1f">broad.pda.seq.fastq.FastqSplitterByBarcode</a>
</li>
<li>START_CODON_PATRN
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a40234c09d1097c20589ddc5c90b0945d">umms.core.annotation.Gene</a>
</li>
<li>StartInWindowCountScore()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_start_in_window_count_score.html#a8b4e82c682905d6b51301906f4c4a510">umms.core.model.score.StartInWindowCountScore</a>
</li>
<li>startTrack()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#a7e5e29800f6c76b42b51b626d4e919fe">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
</li>
<li>StationaryDistributionIO()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_stationary_distribution_i_o.html#adcd10705e244d6ede5a01dfcf0145bc2">broad.core.siphy.StationaryDistributionIO</a>
</li>
<li>statistic
: <a class="el" href="classbroad_1_1pda_1_1enrichment_1_1_enrichment_metric.html#a9300edd46bda63f9b0b15267c6a840bf">broad.pda.enrichment.EnrichmentMetric</a>
</li>
<li>statisticName
: <a class="el" href="classbroad_1_1pda_1_1enrichment_1_1_enrichment_metric.html#a942a9a1a78e214597d2e96b0ff214771">broad.pda.enrichment.EnrichmentMetric</a>
</li>
<li>STATS
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_count_probes.html#a9dadc2b6bd9a4aeb03283d74dd7d57d7">broad.pda.seq.rap.CountProbes</a>
</li>
<li>stdev()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_statistics.html#abc4dd8ce6829f120079b83b87a1be301">broad.core.math.Statistics</a>
</li>
<li>stepSize
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#a7f46117f3c95d5ed4ae816db5807547d">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#ab5fe21e8281091d1be8de525e17d8a3a">broad.pda.seq.protection.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_two_sample_scan_skellam_peak_caller.html#ad9f06a4eaef0c155a2b6a908bed38223">broad.pda.seq.protection.TwoSampleScanSkellamPeakCaller</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_window_scores.html#a9801a4f13246cf58d640e22a93b02c4d">broad.pda.seq.protection.WindowScores</a>
</li>
<li>stitch()
: <a class="el" href="classbroad_1_1core_1_1siphy_1_1_tree_scaler_i_o.html#a69c1081293f24829302b33be5b1faebf">broad.core.siphy.TreeScalerIO</a>
</li>
<li>stitchList()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#a0ae4c3d5c0d8a32ead8146b275a461be">broad.core.annotation.BasicLightweightAnnotation</a>
</li>
<li>stitchTo()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#a91baf63b1c0a9334fa9a292f8064ed2b">broad.core.annotation.BasicLightweightAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation.html#a5649ce7db12f7377ae3a858884fca5af">broad.core.annotation.LightweightGenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a1d97f790e022d82eedf0d1daba962d01">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#ae3d90b1fa243bbd4a39e95c9ada67f65">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a049964537bfd15dcb46d488f2f14b2f1">umms.core.annotation.BasicAnnotation</a>
</li>
<li>stop()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_splitter_by_barcode.html#a0c70f0b3af633811c92a7cbaa76e7b10">broad.pda.seq.fastq.FastqSplitterByBarcode</a>
</li>
<li>STOP
: <a class="el" href="classjaligner_1_1_directions.html#a24db3cae78d4158c169ebdd937bff01a">jaligner.Directions</a>
</li>
<li>STOP_CODONS
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a4f6beedb7f869c3aac74594c74463f3c">umms.core.annotation.Gene</a>
</li>
<li>store()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_stored_alignment_stats.html#ad8687679eb45fe925e14a8146735819b">broad.pda.seq.alignment.StoredAlignmentStats</a>
</li>
<li>StoredAlignmentStats()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_stored_alignment_stats.html#a515c4d71671662820e8cc8a9d249817e">broad.pda.seq.alignment.StoredAlignmentStats</a>
</li>
<li>StringParser()
: <a class="el" href="classbroad_1_1core_1_1parser_1_1_string_parser.html#aa02fda8dd78bf419b8db9e0a6e523978">broad.core.parser.StringParser</a>
</li>
<li>submatrixByColumnIndex()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a2161f6584f31f29d02b1300ae9284d26">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>submatrixByColumnNames()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a51039cb4346193fba13dd235b10f03bf">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>submatrixByColumns()
: <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_location_aware_matrix.html#a7f4a047ce0579d3ed840c0395cb54baf">broad.pda.datastructures.LocationAwareMatrix</a>
</li>
<li>submatrixByRowDescriptions()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a108627a9a341ddba30ec917b180ad6f1">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>submatrixByRowNames()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a6fe084d8d21e55313ce73c2594e4ac68">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>submitJobs()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_collect_annotation_enrichments.html#ab1badfd95c8ead4c614a17731e449110">broad.pda.seq.rap.CollectAnnotationEnrichments</a>
</li>
<li>subsequence()
: <a class="el" href="classjaligner_1_1_sequence.html#a17bba130f9dfeace50cda783d3be1eff">jaligner.Sequence</a>
</li>
<li>sum()
: <a class="el" href="classbroad_1_1core_1_1math_1_1_statistics.html#a7d4868e44ea8b863197f0324d9aac76b">broad.core.math.Statistics</a>
</li>
<li>SumScore()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_sum_score.html#af49a51996293c1cd5e465a4d7f61fe00">umms.core.model.score.SumScore</a>
</li>
<li>svd()
: <a class="el" href="class_jama_1_1_matrix.html#a64d5b9b2faf9ee4f3fedfcc6423f7706">Jama.Matrix</a>
</li>
<li>SYMMETRIC
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_plot_aggregate_regions.html#aa1d2c8bd57f485aba411349ab9b68248">broad.pda.seq.rap.PlotAggregateRegions</a>
</li>
</ul>
</div><!-- contents -->
<!-- start footer part -->
<hr class="footer"/><address class="footer"><small>
Generated on Fri Aug 15 2014 15:11:16 for ESAT by &#160;<a href="http://www.doxygen.org/index.html">
<img class="footer" src="doxygen.png" alt="doxygen"/>
</a> 1.8.7
</small></address>
</body>
</html>
