<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=9"/>
<meta name="generator" content="Doxygen 1.8.7"/>
<title>ESAT: Class Members</title>
<link href="tabs.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="jquery.js"></script>
<script type="text/javascript" src="dynsections.js"></script>
<link href="search/search.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="search/search.js"></script>
<script type="text/javascript">
  $(document).ready(function() { searchBox.OnSelectItem(0); });
</script>
<link href="doxygen.css" rel="stylesheet" type="text/css" />
</head>
<body>
<div id="top"><!-- do not remove this div, it is closed by doxygen! -->
<div id="titlearea">
<table cellspacing="0" cellpadding="0">
 <tbody>
 <tr style="height: 56px;">
  <td style="padding-left: 0.5em;">
   <div id="projectname">ESAT
   </div>
  </td>
 </tr>
 </tbody>
</table>
</div>
<!-- end header part -->
<!-- Generated by Doxygen 1.8.7 -->
<script type="text/javascript">
var searchBox = new SearchBox("searchBox", "search",false,'Search');
</script>
  <div id="navrow1" class="tabs">
    <ul class="tablist">
      <li><a href="index.html"><span>Main&#160;Page</span></a></li>
      <li><a href="pages.html"><span>Related&#160;Pages</span></a></li>
      <li><a href="namespaces.html"><span>Packages</span></a></li>
      <li class="current"><a href="annotated.html"><span>Classes</span></a></li>
      <li><a href="files.html"><span>Files</span></a></li>
      <li>
        <div id="MSearchBox" class="MSearchBoxInactive">
        <span class="left">
          <img id="MSearchSelect" src="search/mag_sel.png"
               onmouseover="return searchBox.OnSearchSelectShow()"
               onmouseout="return searchBox.OnSearchSelectHide()"
               alt=""/>
          <input type="text" id="MSearchField" value="Search" accesskey="S"
               onfocus="searchBox.OnSearchFieldFocus(true)" 
               onblur="searchBox.OnSearchFieldFocus(false)" 
               onkeyup="searchBox.OnSearchFieldChange(event)"/>
          </span><span class="right">
            <a id="MSearchClose" href="javascript:searchBox.CloseResultsWindow()"><img id="MSearchCloseImg" border="0" src="search/close.png" alt=""/></a>
          </span>
        </div>
      </li>
    </ul>
  </div>
  <div id="navrow2" class="tabs2">
    <ul class="tablist">
      <li><a href="annotated.html"><span>Class&#160;List</span></a></li>
      <li><a href="classes.html"><span>Class&#160;Index</span></a></li>
      <li><a href="inherits.html"><span>Class&#160;Hierarchy</span></a></li>
      <li class="current"><a href="functions.html"><span>Class&#160;Members</span></a></li>
    </ul>
  </div>
  <div id="navrow3" class="tabs2">
    <ul class="tablist">
      <li class="current"><a href="functions.html"><span>All</span></a></li>
      <li><a href="functions_func.html"><span>Functions</span></a></li>
      <li><a href="functions_vars.html"><span>Variables</span></a></li>
      <li><a href="functions_eval.html"><span>Enumerator</span></a></li>
    </ul>
  </div>
  <div id="navrow4" class="tabs3">
    <ul class="tablist">
      <li><a href="functions.html#index_a"><span>a</span></a></li>
      <li><a href="functions_b.html#index_b"><span>b</span></a></li>
      <li><a href="functions_c.html#index_c"><span>c</span></a></li>
      <li><a href="functions_d.html#index_d"><span>d</span></a></li>
      <li><a href="functions_e.html#index_e"><span>e</span></a></li>
      <li><a href="functions_f.html#index_f"><span>f</span></a></li>
      <li><a href="functions_g.html#index_g"><span>g</span></a></li>
      <li><a href="functions_h.html#index_h"><span>h</span></a></li>
      <li class="current"><a href="functions_i.html#index_i"><span>i</span></a></li>
      <li><a href="functions_j.html#index_j"><span>j</span></a></li>
      <li><a href="functions_k.html#index_k"><span>k</span></a></li>
      <li><a href="functions_l.html#index_l"><span>l</span></a></li>
      <li><a href="functions_m.html#index_m"><span>m</span></a></li>
      <li><a href="functions_n.html#index_n"><span>n</span></a></li>
      <li><a href="functions_o.html#index_o"><span>o</span></a></li>
      <li><a href="functions_p.html#index_p"><span>p</span></a></li>
      <li><a href="functions_q.html#index_q"><span>q</span></a></li>
      <li><a href="functions_r.html#index_r"><span>r</span></a></li>
      <li><a href="functions_s.html#index_s"><span>s</span></a></li>
      <li><a href="functions_t.html#index_t"><span>t</span></a></li>
      <li><a href="functions_u.html#index_u"><span>u</span></a></li>
      <li><a href="functions_v.html#index_v"><span>v</span></a></li>
      <li><a href="functions_w.html#index_w"><span>w</span></a></li>
      <li><a href="functions_z.html#index_z"><span>z</span></a></li>
    </ul>
  </div>
</div><!-- top -->
<!-- window showing the filter options -->
<div id="MSearchSelectWindow"
     onmouseover="return searchBox.OnSearchSelectShow()"
     onmouseout="return searchBox.OnSearchSelectHide()"
     onkeydown="return searchBox.OnSearchSelectKey(event)">
<a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(0)"><span class="SelectionMark">&#160;</span>All</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(1)"><span class="SelectionMark">&#160;</span>Classes</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(2)"><span class="SelectionMark">&#160;</span>Namespaces</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(3)"><span class="SelectionMark">&#160;</span>Files</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(4)"><span class="SelectionMark">&#160;</span>Functions</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(5)"><span class="SelectionMark">&#160;</span>Variables</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(6)"><span class="SelectionMark">&#160;</span>Enumerator</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(7)"><span class="SelectionMark">&#160;</span>Pages</a></div>

<!-- iframe showing the search results (closed by default) -->
<div id="MSearchResultsWindow">
<iframe src="javascript:void(0)" frameborder="0" 
        name="MSearchResults" id="MSearchResults">
</iframe>
</div>

<div class="contents">
<div class="textblock">Here is a list of all class members with links to the classes they belong to:</div>

<h3><a class="anchor" id="index_i"></a>- i -</h3><ul>
<li>ic()
: <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix.html#a19c2e267b3adebb5a9e1a0c29c8c2be4">broad.core.motif.PositionWeightMatrix</a>
</li>
<li>IDENTITY
: <a class="el" href="classjaligner_1_1_markups.html#af32496371126c49532d828ba52839ca8">jaligner.Markups</a>
</li>
<li>identity()
: <a class="el" href="class_jama_1_1_matrix.html#aa137f0aa3f5b03e9da21cbf2981e3c20">Jama.Matrix</a>
</li>
<li>IGNORE_STRAND
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_distance_to_b_e_d.html#aab459f090a02c3e7eb185c3207820d59">broad.pda.seq.rap.DistanceToBED</a>
</li>
<li>IllegalTreeOperationException()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_illegal_tree_operation_exception.html#a4a44881fad042bcc09fcd1373bae8487">broad.core.datastructures.IllegalTreeOperationException</a>
</li>
<li>INCLUDE_DUPLICATES
: <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_paired_end_t_d_f.html#a4b7e46dd1f9df8d30c181b4205ae6ffe">broad.pda.samtools.PairedEndTDF</a>
</li>
<li>IncompleteMethodImplementationException()
: <a class="el" href="classumms_1_1core_1_1exception_1_1_incomplete_method_implementation_exception.html#a936015da42cd5b9d71dfd03f95db7338">umms.core.exception.IncompleteMethodImplementationException</a>
</li>
<li>IncrementScoreIfOverlap()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#aae2b5bec9ccbb0c1dca631c44fc03d7a">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
, <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#ae177571878924990e5bdc5cf5726ef92">broad.pda.annotation.BEDFileParser</a>
</li>
<li>INF
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#ad709c721bc2e22de2f16c1af52721d6f">broad.core.sequence.SequenceRegion</a>
</li>
<li>init()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#a427f36c6a67ff8ce3dbd1072478faaeb">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
, <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_column.html#ae153a2a5518ba867a38f580bf936187a">broad.core.motif.PositionWeightColumn</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a040bc3c9f51b2ade134fdfec37c36e7c">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
</li>
<li>initColIndexMaps()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a49a5493e04b7998de5a00eca271510e6">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>initFromGene()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#af73381bf71c4a4a752ff80f5f985bb28">umms.core.annotation.Gene</a>
</li>
<li>initFromIGVMatrix()
: <a class="el" href="classbroad_1_1pda_1_1datastructures_1_1_location_aware_matrix.html#a2c1183b9307cbe047d621bb9c1000977">broad.pda.datastructures.LocationAwareMatrix</a>
</li>
<li>initFromNanostring()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a17270eccd2e5fe75b45d02e921331c13">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>initFromReader()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#ad1dda51b2e8707e56035714a58497111">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>initFromRegularMatrix()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a96c809f62854178409a3f7dcd9623f7c">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>initGCTFromReader()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a8b562e19039440efdb63cf07a4bd4d83">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>initializeFilterRejectWriters()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#a4ea2fd7e43d91432d95e19b23f31a502">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>initNameIndexMaps()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a6a2f525112e1854bfa5c88efabb92b4c">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>initRegion()
: <a class="el" href="classumms_1_1core_1_1model_1_1score_1_1_window_processor_3_01_s_01extends_01_window_score_01_4_1ca0a3751200684ea73fe33d33dcb315e.html#ad7991f2301d4391dd9f15996e0287514">umms.core.model.score.WindowProcessor&lt; S extends WindowScore &gt;.AbstractProcessor&lt; S extends WindowScore &gt;</a>
, <a class="el" href="interfaceumms_1_1core_1_1model_1_1score_1_1_window_processor_3_01_s_01extends_01_window_score_01_4.html#a4a24747b66f15f4da6ed7d08c9061407">umms.core.model.score.WindowProcessor&lt; S extends WindowScore &gt;</a>
</li>
<li>initRowDescrIndexMaps()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#ac0e1a1ef7d279d51d043ddc9ede20e31">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>initRowIndexMaps()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a2d2aa137110ec59c81a845a9c0aea5a6">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>INNER_LENGTH
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_plot_aggregate_regions.html#a5f3aac602c678d4be45f0ea0a5d5480a">broad.pda.seq.rap.PlotAggregateRegions</a>
</li>
<li>inOrientedDegree()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a4d7680c735522fe96b8f19c6901d0d83">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>INPUT
: <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_filter_paired_reads_by_m_a_q.html#a7c04255e908577a13183131023cb85f9">broad.pda.samtools.FilterPairedReadsByMAQ</a>
, <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_paired_end_t_d_f.html#ab11151a3035cc606f914973ba04c1a1f">broad.pda.samtools.PairedEndTDF</a>
, <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_trim_fastq_reads.html#a0f8bf4e7d4271018cf6b4dfc15fe217a">broad.pda.samtools.TrimFastqReads</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_calculate_coverage_stats.html#af99791ff675fe2b297a8eec48211796e">broad.pda.seq.rap.CalculateCoverageStats</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_collect_annotation_enrichments.html#af5ba078f10c4cfcac377cf0469c71407">broad.pda.seq.rap.CollectAnnotationEnrichments</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_count_probes.html#a3d820f4558bfe52a29fd2827b98a4e70">broad.pda.seq.rap.CountProbes</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_shuffle_b_a_m.html#a45600d33be8564cd09136414acc055f8">broad.pda.seq.rap.ShuffleBAM</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_shuffle_b_e_d.html#aeec4441997b1b359679c19422be370c0">broad.pda.seq.rap.ShuffleBED</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_unmark_duplicates.html#af779c21595731e988fb21cefff20353f">broad.pda.seq.rap.UnmarkDuplicates</a>
</li>
<li>INPUT_1
: <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_filter_fastq_reads.html#a90752ab083b3b16496406db95322a5c4">broad.pda.samtools.FilterFastqReads</a>
</li>
<li>INPUT_2
: <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_filter_fastq_reads.html#ac0ab765ce49f2248660d9868c4b96917">broad.pda.samtools.FilterFastqReads</a>
</li>
<li>inReversedOrientation()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#a1d9bc8dac1004a062ae4aca4c8cfbfba">broad.core.annotation.BasicLightweightAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_lightweight_genomic_annotation.html#aacaec867ef129abbfb9e96d2dfb5630f">broad.core.annotation.LightweightGenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#aebbe9fecbd89dfb53c258737e94aab9a">broad.core.sequence.SequenceRegion</a>
</li>
<li>insert()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_chromosome.html#ab295584fcacd051aaf615cffc19376ea">broad.pda.feature.genome.Chromosome</a>
, <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_directory_installed_genome_assembly.html#ad320a404ce1ddb695a305d1a495563b8">broad.pda.feature.genome.DirectoryInstalledGenomeAssembly</a>
</li>
<li>INTER_VERTICAL_SPACING
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_genomic_annotation_reader.html#a7150fc4987cb783f7c26a60312b6cc37">broad.core.annotation.BasicGenomicAnnotationReader</a>
</li>
<li>intersect()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_annotation_reader_3_01_t_01extends_01_genomic_annotation_01_4.html#ad348dc8271285cf288c0ea538199c423">broad.core.annotation.AnnotationReader&lt; T extends GenomicAnnotation &gt;</a>
, <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_lightweight_annotation.html#a9174257fd599f54ae720c92aecb1fed4">broad.core.annotation.BasicLightweightAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a9bd036c43990c3529b4cd6886d1b36eb">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a158b393d669768a427bff7460583d555">broad.pda.annotation.BEDFileParser</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation.html#afa78e8b538cea72c9b378e60662efac5">umms.core.annotation.AbstractAnnotation</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#a3c009278c52746db8adece1e2f7f5d49">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4.html#a1d2f7e819b04092754be9066e33c9765">umms.core.annotation.AnnotationList&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a9dc464c6ce3b4805fde6ddcb5fc9745c">umms.core.annotation.BasicAnnotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_compound_interval.html#a4f4d181aef401608e1132208f67752b3">umms.core.annotation.CompoundInterval</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_single_interval.html#a924417b81926218be5df0a4886c2818d">umms.core.annotation.SingleInterval</a>
</li>
<li>IntersectBlocks()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_b_e_d.html#a300061669e13aa1072380dd85d1c5796">broad.core.annotation.BED</a>
</li>
<li>intersectGeneSets()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_gene_tools.html#a05bf75f8d1e4bf0f75949574d0031a73">broad.pda.gene.GeneTools</a>
</li>
<li>introduceGaps()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#a4ee614853a44ff109c0981a1513ebf46">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>inverse()
: <a class="el" href="class_jama_1_1_matrix.html#a51ae1c3a1dc5f2b9530757782e6e8a3b">Jama.Matrix</a>
</li>
<li>invert()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_chromosome.html#a153ac394d094738f567d7760222d9865">broad.pda.feature.genome.Chromosome</a>
, <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_directory_installed_genome_assembly.html#a6f2fa0e6a784ea7802e369689f25b332">broad.pda.feature.genome.DirectoryInstalledGenomeAssembly</a>
</li>
<li>IS_ADJACENT_AND_EMPTY
: <a class="el" href="interfacebroad_1_1core_1_1datastructures_1_1_interval.html#af0771ee7e0abe266e3cb7a6ee5f0b614">broad.core.datastructures.Interval</a>
</li>
<li>IS_LEFT_OVERHANGING_OVERLAPPER
: <a class="el" href="interfacebroad_1_1core_1_1datastructures_1_1_interval.html#a4c00f998709fa447253d9404c4c2a7b0">broad.core.datastructures.Interval</a>
</li>
<li>IS_RIGHT_OVERHANGING_OVERLAPPER
: <a class="el" href="interfacebroad_1_1core_1_1datastructures_1_1_interval.html#ab67c614b8b6be7c693a963d5ca67f6f6">broad.core.datastructures.Interval</a>
</li>
<li>IS_STRICTLY_GREATER
: <a class="el" href="interfacebroad_1_1core_1_1datastructures_1_1_interval.html#ac35f97a7a7298f01b458e2c79befa059">broad.core.datastructures.Interval</a>
</li>
<li>IS_STRICTLY_LESS
: <a class="el" href="interfacebroad_1_1core_1_1datastructures_1_1_interval.html#adaf50ee698921a6dfde3694c0cd4d485">broad.core.datastructures.Interval</a>
</li>
<li>IS_SUBSET
: <a class="el" href="interfacebroad_1_1core_1_1datastructures_1_1_interval.html#ab96286a8ce295314638b2f698a7f106f">broad.core.datastructures.Interval</a>
</li>
<li>IS_SUPERSET
: <a class="el" href="interfacebroad_1_1core_1_1datastructures_1_1_interval.html#aa24dfd48992d1f74f751a7b2922f0eed">broad.core.datastructures.Interval</a>
</li>
<li>isAddToReadName()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_splitter_by_barcode.html#a10e5fbabd644209fbe126f200f5f0fff">broad.pda.seq.fastq.FastqSplitterByBarcode</a>
</li>
<li>isAdjacent()
: <a class="el" href="interfacebroad_1_1core_1_1datastructures_1_1_interval.html#afd44cbd67e804688cac81aa094bdce1d">broad.core.datastructures.Interval</a>
</li>
<li>isAlignmentPaired()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_stored_alignment_stats.html#a2a8f34c6e7225509065bc31f113d28f8">broad.pda.seq.alignment.StoredAlignmentStats</a>
</li>
<li>isAtEnd()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_splitter_by_barcode.html#a86ab85f75aad995a5c10e162585c0218">broad.pda.seq.fastq.FastqSplitterByBarcode</a>
</li>
<li>isAutosome()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_chromosome.html#a4a192f9d0d97bfac746fd85d42af4507">broad.pda.feature.genome.Chromosome</a>
</li>
<li>isChimera()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a9e2928564d040785988bd1670a50fa62">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#a80e956645e4bc6903d66fe05ebc8f53a">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#ac1acf67653bca7b09d4316c4ac6cd4f0">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>isCodingGene()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#ad2a651cd125ea1ebea3f496cb62853e3">umms.core.annotation.Gene</a>
</li>
<li>isColumnNonTrivial()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_matrix_with_headers.html#a03f51c8a9d593a510cfa0557fcb0b1b2">broad.core.datastructures.MatrixWithHeaders</a>
</li>
<li>isComplete()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_alignment_pair.html#ae7b309f8b45628e21a057aaa45d09d1d">umms.core.alignment.AlignmentPair</a>
</li>
<li>isControl()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1_expression_experiment_info.html#a4c33bc19d44228905f603c3aa5bcefe8">broad.pda.geneexpression.ExpressionExperimentInfo</a>
</li>
<li>isControlProbe()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_probe.html#af0fd5c8c29a0217733387a1128f5a17d">broad.pda.geneexpression.agilent.AgilentProbe</a>
</li>
<li>isDiagonal()
: <a class="el" href="class_jama_1_1_matrix.html#a1e03a02fb132e843e6ec9b08c9674fdd">Jama.Matrix</a>
</li>
<li>isDirect()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_two_subject_annotation.html#a4997cb1a61a0bdde91a29211d3b4b463">broad.core.annotation.BasicTwoSubjectAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_two_subject_annotation.html#a06572a7c14612d2563b23d6e6b4ce57e">broad.core.annotation.TwoSubjectAnnotation</a>
</li>
<li>isDuplicate()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#aa13e1931258062454882ba057a6a2ea4">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#a43825498ddbd33f21d6e2a2186a1e471">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a4eb89311179898e19bf4319e133f3e62">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>isEmpty()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_interval_tree_3_01_v_01_4.html#a8b579056a7540902bb270ecf9f8b47fa">broad.core.datastructures.IntervalTree&lt; V &gt;</a>
, <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_pair_3_01_t1_01_4.html#a7dced7e2243229ef6b4b58ae7920b248">broad.core.datastructures.Pair&lt; T1 &gt;</a>
, <a class="el" href="classbroad_1_1core_1_1motif_1_1_position_weight_matrix.html#a4088289b8a3cd37348a5bfadb1f73677">broad.core.motif.PositionWeightMatrix</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_m_a_f_alignment.html#ac9afcd9e472b40be045969b06ae38ab6">broad.core.multiplealignment.MAFAlignment</a>
, <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#a4c4058ed47c1cf0ccf504de279f16f41">broad.core.multiplealignment.MultipleAlignment</a>
, <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#ae81836c47db766ac166e4c6a7b4054f9">broad.pda.annotation.BEDFileParser</a>
</li>
<li>isEnough()
: <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_genomic_annotation_filter_3_01_t_01extends_01_lightweight_genomic_annotation_01_4.html#a98ff6734ee1846a62d2510612d707fe9">broad.core.annotation.GenomicAnnotationFilter&lt; T extends LightweightGenomicAnnotation &gt;</a>
</li>
<li>IsExactOverlappingIsoform()
: <a class="el" href="classbroad_1_1pda_1_1gene_1_1_gene_with_isoforms.html#ae0b37109038d6e424519403bb56ac6a9">broad.pda.gene.GeneWithIsoforms</a>
</li>
<li>isExpressed()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_genome_space_sample_data.html#afe48c74821821d5bdc1feb0dce339646">broad.pda.seq.clip.GenomeSpaceSampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1clip_1_1_sample_data.html#a20bd1f5c944414c4305844a15c35f8a1">broad.pda.seq.clip.SampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_genome_space_sample_data.html#abf6af4a1de83e2cfd80b682ef1f4dbe9">broad.pda.seq.protection.GenomeSpaceSampleData</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#a0b1b91b4c484ea9357695e7f570ccc31">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_sample_data.html#af2a3f7b805b2fea8605b821b53f56958">broad.pda.seq.protection.SampleData</a>
</li>
<li>isExpressedInAllControlSamples()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1protection_1_1_multi_sample_scan_peak_caller.html#a363342cd71f370bcb563741291a2d436">broad.pda.seq.protection.MultiSampleScanPeakCaller</a>
</li>
<li>isFlagged()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_probe.html#ab6fd6a397a5b909e011860cf43127861">broad.pda.geneexpression.agilent.AgilentProbe</a>
</li>
<li>isFlankedBy()
: <a class="el" href="classbroad_1_1core_1_1annotation_1_1_basic_genomic_annotation.html#a2b03867be090a45c5526631d5e661916">broad.core.annotation.BasicGenomicAnnotation</a>
, <a class="el" href="interfacebroad_1_1core_1_1annotation_1_1_genomic_annotation.html#ad4fa97f4e5f130966714abc7398554e9">broad.core.annotation.GenomicAnnotation</a>
, <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a33988f05eed5e8868dbd8aadeb9e073a">broad.core.sequence.SequenceRegion</a>
</li>
<li>isForwardStrand()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#ad3843c99ca1bdd29ff3a8d054c2714fc">broad.core.sequence.Sequence</a>
</li>
<li>isFullRank()
: <a class="el" href="class_jama_1_1_q_r_decomposition.html#ab7a81bd679ece0dd255aa3a786e2efed">Jama.QRDecomposition</a>
</li>
<li>isFullyComaptible()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#aa1a696f876a12b75cfab2f26340e6c1f">broad.pda.annotation.Locus</a>
</li>
<li>isFullyCompatible()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#ac1f84639046dcf50ab5c6ff82416cfaa">umms.core.annotation.Gene</a>
</li>
<li>isGap()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence.html#a0311924cfc4add3eee42b023f71b42c8">broad.core.sequence.Sequence</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#ab0d12e3df82e16fecdf3fa3a15f1988d">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>isGenomicSpace()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1rap_1_1_genome_command_line_program.html#a219dff1983f3e6c2b4c002c740074f9c">broad.pda.seq.rap.GenomeCommandLineProgram</a>
</li>
<li>isInAllGaps()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a328f7944789466e63a8f6816409bcd41">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>isInExon()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a6327a3395c20606b0b944cca1c9bee84">umms.core.annotation.Gene</a>
</li>
<li>isJnlp()
: <a class="el" href="classjaligner_1_1util_1_1_commons.html#aadca0c521881611256cc7aa24cab4cdd">jaligner.util.Commons</a>
</li>
<li>isMapped()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#af47f73adc6769845e2bb1811dea02f8c">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#aeb2dbb3707f8f1eb9de40688fdda0ed6">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a7f83aba9bd2a412269f8a9c99b98e260">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>isNegativeStrand()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#a4a4fda6b7990a9d6f27679ae3a58a717">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#af37e974bc3b52387663422c762994f94">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#ac6754f0229f8ed7e7b44d584a07f6dae">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_abstract_annotation.html#a4e1da04a01deed1a4b6036a36e24b86e">umms.core.annotation.AbstractAnnotation</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#aa72d3cd6ea67627915291f116b777be3">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a64f4da8582b66df16ffdf4f10d189e06">umms.core.annotation.Gene</a>
</li>
<li>isNegativeStranded()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#af12a10e456797cda37eeef9c19e7f99d">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a77358e51336c799241530fb90e0c1bf3">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>isNonsingular()
: <a class="el" href="class_jama_1_1_l_u_decomposition.html#aa5a58918751ee6b2abc858542fb5ae47">Jama.LUDecomposition</a>
</li>
<li>isoformMapContains()
: <a class="el" href="classumms_1_1core_1_1scripture_1_1_build_scripture_coordinate_space.html#affbce5fbd3aaa984b84581aeb4890039">umms.core.scripture.BuildScriptureCoordinateSpace</a>
</li>
<li>isOverlapCompatible()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_b_e_d_file_parser.html#a80bdec41f0a829196cee03fc498b6eb3">broad.pda.annotation.BEDFileParser</a>
</li>
<li>isPaired()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_splitter_by_barcode.html#abf92378a2b0e63fe01228f308ff0cdd4">broad.pda.seq.fastq.FastqSplitterByBarcode</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model_stats.html#a407275e05e013a63813920aa391e24bb">broad.pda.seq.segmentation.AlignmentDataModelStats</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a3c5cbd4ef536d5c63105eca0fde0d141">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#aed4467c1090c04d0a76946b0b65a946d">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a8fceb6ed7f1a65205bf787e4a5a69420">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>isPairedEndFormat()
: <a class="el" href="classumms_1_1core_1_1readers_1_1_paired_end_reader.html#a90d2c33b061d7e411d931933247ab9c3">umms.core.readers.PairedEndReader</a>
</li>
<li>isPartialPolyA()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_sequence.html#ad97e971df9d823122223a236ead320d0">broad.pda.seq.fastq.FastqSequence</a>
</li>
<li>isPCRLike()
: <a class="el" href="classbroad_1_1core_1_1primer3_1_1is_p_c_r_like.html#a201fedf69d09db6e441701283948cf8b">broad.core.primer3.isPCRLike</a>
</li>
<li>isPolyA()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1fastq_1_1_fastq_sequence.html#ae96731dcb1c3e8c7c03cc592bfb05101">broad.pda.seq.fastq.FastqSequence</a>
</li>
<li>isPositiveStranded()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#add40b2ab02323c98f487bdf550bcf2ab">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#ae494c574fcc812082712b4436f4761f1">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>isProbeFound()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_probe.html#ae13421376722c2a6cdb6fd7c3c5085bc">broad.pda.geneexpression.agilent.AgilentProbe</a>
</li>
<li>isProbeNonUniform()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_probe.html#aea7bd1ea385e66fa6cf6d4143406992b">broad.pda.geneexpression.agilent.AgilentProbe</a>
</li>
<li>isProbePopulationOutlier()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_probe.html#a27fe598abff650cbae102f1a0ade64c7">broad.pda.geneexpression.agilent.AgilentProbe</a>
</li>
<li>isProbeWellAboveBackground()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_probe.html#abf9f43f7f9af0c6b270b8540524356e4">broad.pda.geneexpression.agilent.AgilentProbe</a>
</li>
<li>isProperPair()
: <a class="el" href="classumms_1_1core_1_1alignment_1_1_abstract_paired_end_alignment.html#a3b74b26023376f9451c4243674bf33ce">umms.core.alignment.AbstractPairedEndAlignment</a>
, <a class="el" href="interfaceumms_1_1core_1_1alignment_1_1_alignment.html#a0cfc2e0a3ee1e18b1a103d600e29bba0">umms.core.alignment.Alignment</a>
, <a class="el" href="classumms_1_1core_1_1alignment_1_1_single_end_alignment.html#a244a56ea5a881e4067a3d193f727d28f">umms.core.alignment.SingleEndAlignment</a>
</li>
<li>isRefGapped()
: <a class="el" href="classbroad_1_1core_1_1multiplealignment_1_1_multiple_alignment.html#a8bbb7981b4e9286756b65a983b8ec7a1">broad.core.multiplealignment.MultipleAlignment</a>
</li>
<li>isSafeToLoad()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1alignment_1_1_stored_alignment_stats.html#a797882c7239d6c3c704ed036ddaa35b8">broad.pda.seq.alignment.StoredAlignmentStats</a>
</li>
<li>isSaturated()
: <a class="el" href="classbroad_1_1pda_1_1geneexpression_1_1agilent_1_1_agilent_probe.html#aaceac16b43cdc2bf5594996d56592273">broad.pda.geneexpression.agilent.AgilentProbe</a>
</li>
<li>isSexChromosome()
: <a class="el" href="classbroad_1_1pda_1_1feature_1_1genome_1_1_chromosome.html#a2056c7b3c6f4dd1a7d7951c10235aef0">broad.pda.feature.genome.Chromosome</a>
</li>
<li>isSpanningGap()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#ac8a3ce5c61ee9ab7759b97b809b9099f">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
</li>
<li>isSPD()
: <a class="el" href="class_jama_1_1_cholesky_decomposition.html#ad84aa22ce8f3e9646d7c5e932b6d28d8">Jama.CholeskyDecomposition</a>
</li>
<li>isStopCodon()
: <a class="el" href="classumms_1_1core_1_1annotation_1_1_gene.html#a58800b7ac03f5205cb858c171beb7d28">umms.core.annotation.Gene</a>
</li>
<li>isStranded()
: <a class="el" href="interfacebroad_1_1pda_1_1seq_1_1segmentation_1_1_alignment_data_model.html#ab10cf271be5d0c0dbebff5b7c5088085">broad.pda.seq.segmentation.AlignmentDataModel</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1segmentation_1_1_generic_alignment_data_model.html#a86e1f6ed98d7f04a2b86a861fc91c8a7">broad.pda.seq.segmentation.GenericAlignmentDataModel</a>
</li>
<li>isSymmetric()
: <a class="el" href="class_jama_1_1_matrix.html#a081f88a952b4efe65ca8ab758f6893fb">Jama.Matrix</a>
</li>
<li>isTerminiCoverageBias()
: <a class="el" href="classbroad_1_1pda_1_1annotation_1_1_locus.html#a647a1e88afae9412ccedfc6284f1d979">broad.pda.annotation.Locus</a>
</li>
<li>isToComplex()
: <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_path.html#a00daa0a104035222bb606b658ed21b78">broad.pda.seq.graph.Path</a>
</li>
<li>isUnoriented()
: <a class="el" href="classbroad_1_1core_1_1sequence_1_1_sequence_region.html#ab8be6edb1adc3a624f8361daa51ecbc8">broad.core.sequence.SequenceRegion</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation.html#a796b34bf340d8ebe30937eb5b0b5d8d2">umms.core.annotation.Annotation</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_basic_annotation.html#a60bb0d08360fd4bce2974c8aeae80cab">umms.core.annotation.BasicAnnotation</a>
</li>
<li>isValid()
: <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#aff0080eefe7199b9db895a2b9ef628b0">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#af00895782a6099bc876a4226499e102d">umms.core.model.JCSAlignmentModel</a>
</li>
<li>isValidWindow()
: <a class="el" href="interfaceumms_1_1core_1_1coordinatesystem_1_1_coordinate_space.html#aaa00b0f98960c55359dd483367da6c46">umms.core.coordinatesystem.CoordinateSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_genomic_space.html#ac9bb06d851f3ce97d435b2af47a05b7e">umms.core.coordinatesystem.GenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_masked_genomic_space.html#aa9b95648e84134a9ddf6229dda7b66c3">umms.core.coordinatesystem.MaskedGenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcript_in_genomic_space.html#a2b0985bdebf24dc9c6840cbff3533682">umms.core.coordinatesystem.TranscriptInGenomicSpace</a>
, <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_transcriptome_space.html#a28ab0ddf98f75313937219c61a5cff37">umms.core.coordinatesystem.TranscriptomeSpace</a>
</li>
<li>iterator()
: <a class="el" href="classbroad_1_1core_1_1datastructures_1_1_interval_tree_3_01_v_01_4.html#a9727995889413a674272f79899f4a932">broad.core.datastructures.IntervalTree&lt; V &gt;</a>
, <a class="el" href="classbroad_1_1pda_1_1samtools_1_1_s_a_m_paired_end_file_reader.html#ac5a42e248431b503fb9e03126fb688ef">broad.pda.samtools.SAMPairedEndFileReader</a>
, <a class="el" href="classbroad_1_1pda_1_1seq_1_1graph_1_1_chromosome_with_bubbles_j_graph_t.html#a8cfa2b58cf5ca0466c67439292d04442">broad.pda.seq.graph.ChromosomeWithBubblesJGraphT</a>
, <a class="el" href="interfaceumms_1_1core_1_1annotation_1_1_annotation_collection_3_01_t_01extends_01_annotation_01_4.html#ab7b15f648793ccec506ec3ac64168500">umms.core.annotation.AnnotationCollection&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1annotation_1_1_annotation_list_3_01_t_01extends_01_annotation_01_4.html#af0a29e6cf7a93ac69b3470fd966812b0">umms.core.annotation.AnnotationList&lt; T extends Annotation &gt;</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_alignment_model.html#a605283647c0e4797f19a57b20fb870bc">umms.core.model.AlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1model_1_1_j_c_s_alignment_model.html#ae41f39ab7b1d13fd6915e1d0405c78a7">umms.core.model.JCSAlignmentModel</a>
, <a class="el" href="classumms_1_1core_1_1readers_1_1_paired_end_reader.html#ab050bf528b5fe9e7b2011ba41d41e9cc">umms.core.readers.PairedEndReader</a>
</li>
<li>itr
: <a class="el" href="classumms_1_1core_1_1coordinatesystem_1_1_shuffled_iterator_3_01_t_01extends_01_annotation_01_4.html#a7bc14e133f323959b7c98ccd5e93a140">umms.core.coordinatesystem.ShuffledIterator&lt; T extends Annotation &gt;</a>
</li>
</ul>
</div><!-- contents -->
<!-- start footer part -->
<hr class="footer"/><address class="footer"><small>
Generated on Fri Aug 15 2014 15:11:16 for ESAT by &#160;<a href="http://www.doxygen.org/index.html">
<img class="footer" src="doxygen.png" alt="doxygen"/>
</a> 1.8.7
</small></address>
</body>
</html>
